---
title: 'SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics'
keywords:
- covid-19
- coronavirus
- pandemic
- viral infection
lang: en-US
date-meta: '2021-09-27'
author-meta:
- Halie M. Rando
- Casey S. Greene
- Michael P. Robson
- Simina M. Boca
- Nils Wellhausen
- Ronan Lordan
- Christian Brueffer
- Sandipan Ray
- Lucy D\'Agostino McGowan
- Anthony Gitter
- Anna Ada Dattoli
- Ryan Velazquez
- John P. Barton
- Jeffrey M. Field
- Bharath Ramsundar
- Adam L. MacLean
- Alexandra J. Lee
- Immunology Institute of the Icahn School of Medicine
- Fengling Hu
- Nafisa M. Jadavji
- Elizabeth Sell
- Vincent Rubinetti
- Jinhui Wang
- Diane N. Rafizadeh
- Ashwin N. Skelly
- Marouen Ben Guebila
- Likhitha Kolla
- David Manheim
- Soumita Ghosh
- James Brian Byrd
- YoSon Park
- Vikas Bansal
- Stephen Capone
- John J. Dziak
- Yuchen Sun
- Yanjun Qi
- Lamonica Shinholster
- Temitayo Lukan
- Sergey Knyazev
- Dimitri Perrin
- Serghei Mangul
- Shikta Das
- Gregory L Szeto
- Tiago Lubiana
- David Mai
- COVID-19 Review Consortium
- Rishi Raj Goel
- Joel D Boerckel
- Amruta Naik
- Yusha Sun
- Daniel S. Himmelstein
- Jeremy P. Kamil
header-includes: |-
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta name="citation_title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta property="og:title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta property="twitter:title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta name="dc.date" content="2021-09-27" />
  <meta name="citation_publication_date" content="2021-09-27" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Halie M. Rando" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, Colorado, United States of America" />
  <meta name="citation_author_institution" content="Center for Health AI, University of Colorado School of Medicine, Aurora, Colorado, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-7688-1770" />
  <meta name="twitter:creator" content="@tamefoxtime" />
  <meta name="citation_author" content="Casey S. Greene" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, Colorado, United States of America" />
  <meta name="citation_author_institution" content="Center for Health AI, University of Colorado School of Medicine, Aurora, Colorado, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-8713-9213" />
  <meta name="twitter:creator" content="@GreeneScientist" />
  <meta name="citation_author" content="Michael P. Robson" />
  <meta name="citation_author_institution" content="Department of Computing Sciences, Villanova University, Villanova, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-4859-0033" />
  <meta name="citation_author" content="Simina M. Boca" />
  <meta name="citation_author_institution" content="Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, District of Columbia, United States of America" />
  <meta name="citation_author_institution" content="Early Biometrics &amp; Statistical Innovation, Data Science &amp; Artificial Intelligence, R &amp; D, AstraZeneca, Gaithersburg, Maryland, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-1400-3398" />
  <meta name="citation_author" content="Nils Wellhausen" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-8955-7582" />
  <meta name="citation_author" content="Ronan Lordan" />
  <meta name="citation_author_institution" content="Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104-5158, USA" />
  <meta name="citation_author_orcid" content="0000-0001-9668-3368" />
  <meta name="twitter:creator" content="@el_ronan" />
  <meta name="citation_author" content="Christian Brueffer" />
  <meta name="citation_author_institution" content="Department of Clinical Sciences, Lund University, Lund, Sweden" />
  <meta name="citation_author_orcid" content="0000-0002-3826-0989" />
  <meta name="twitter:creator" content="@cbrueffer" />
  <meta name="citation_author" content="Sandipan Ray" />
  <meta name="citation_author_institution" content="Department of Biotechnology, Indian Institute of Technology Hyderabad, Kandi, Sangareddy 502285, Telangana, India" />
  <meta name="citation_author_orcid" content="0000-0002-9960-5768" />
  <meta name="citation_author" content="Lucy D\&#39;Agostino McGowan" />
  <meta name="citation_author_institution" content="Department of Mathematics and Statistics, Wake Forest University, Winston-Salem, North Carolina, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-7297-9359" />
  <meta name="twitter:creator" content="@LucyStats" />
  <meta name="citation_author" content="Anthony Gitter" />
  <meta name="citation_author_institution" content="Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin, United States of America" />
  <meta name="citation_author_institution" content="Morgridge Institute for Research, Madison, Wisconsin, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-5324-9833" />
  <meta name="twitter:creator" content="@anthonygitter" />
  <meta name="citation_author" content="Anna Ada Dattoli" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology &amp; Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0003-1462-831X" />
  <meta name="twitter:creator" content="@aadattoli" />
  <meta name="citation_author" content="Ryan Velazquez" />
  <meta name="citation_author_institution" content="Azimuth1, McLean, Virginia, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-3655-3403" />
  <meta name="citation_author" content="John P. Barton" />
  <meta name="citation_author_institution" content="Department of Physics and Astronomy, University of California-Riverside, Riverside, California, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-1467-421X" />
  <meta name="twitter:creator" content="@_jpbarton" />
  <meta name="citation_author" content="Jeffrey M. Field" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0001-7161-7284" />
  <meta name="citation_author" content="Bharath Ramsundar" />
  <meta name="citation_author_institution" content="The DeepChem Project, https://deepchem.io/" />
  <meta name="citation_author_orcid" content="0000-0001-8450-4262" />
  <meta name="twitter:creator" content="@rbhar90" />
  <meta name="citation_author" content="Adam L. MacLean" />
  <meta name="citation_author_institution" content="Department of Quantitative and Computational Biology, University of Southern California, Los Angeles, California, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-0689-7907" />
  <meta name="twitter:creator" content="@adamlmaclean" />
  <meta name="citation_author" content="Alexandra J. Lee" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-0208-3730" />
  <meta name="citation_author" content="Immunology Institute of the Icahn School of Medicine" />
  <meta name="citation_author_institution" content="Immunology Institute of the Icahn School of Medicine" />
  <meta name="citation_author" content="Fengling Hu" />
  <meta name="citation_author_institution" content="Department of Biostatistics, Epidemiology and Informatics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-1081-5038" />
  <meta name="twitter:creator" content="@hufengling" />
  <meta name="citation_author" content="Nafisa M. Jadavji" />
  <meta name="citation_author_institution" content="Biomedical Science, Midwestern University, Glendale, AZ, United States of America" />
  <meta name="citation_author_institution" content="Department of Neuroscience, Carleton University, Ottawa, Ontario, Canada" />
  <meta name="citation_author_orcid" content="0000-0002-3557-7307" />
  <meta name="twitter:creator" content="@nafisajadavji" />
  <meta name="citation_author" content="Elizabeth Sell" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-9658-1107" />
  <meta name="citation_author" content="Vincent Rubinetti" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Center for Health AI, University of Colorado School of Medicine, Aurora, Colorado, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-4655-3773" />
  <meta name="citation_author" content="Jinhui Wang" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-5796-8130" />
  <meta name="citation_author" content="Diane N. Rafizadeh" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Department of Chemistry, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-2838-067X" />
  <meta name="citation_author" content="Ashwin N. Skelly" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Institute for Immunology, University of Pennsylvania Perelman School of Medicine, Philadelphia, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-1565-3376" />
  <meta name="citation_author" content="Marouen Ben Guebila" />
  <meta name="citation_author_institution" content="Department of Biostatistics, Harvard School of Public Health, Boston, Massachusetts, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-5934-966X" />
  <meta name="twitter:creator" content="@marouenbg" />
  <meta name="citation_author" content="Likhitha Kolla" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-1169-906X" />
  <meta name="twitter:creator" content="@lkolla2018" />
  <meta name="citation_author" content="David Manheim" />
  <meta name="citation_author_institution" content="1DaySooner, Delaware, United States of America" />
  <meta name="citation_author_institution" content="Risk and Health Communication Research Center, School of Public Health, University of Haifa, Haifa, Israel" />
  <meta name="citation_author_orcid" content="0000-0001-8599-8380" />
  <meta name="twitter:creator" content="@davidmanheim" />
  <meta name="citation_author" content="Soumita Ghosh" />
  <meta name="citation_author_institution" content="Institute of Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-2783-2750" />
  <meta name="citation_author" content="James Brian Byrd" />
  <meta name="citation_author_institution" content="University of Michigan School of Medicine, Ann Arbor, Michigan, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-0509-3520" />
  <meta name="twitter:creator" content="@thebyrdlab" />
  <meta name="citation_author" content="YoSon Park" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-0465-4744" />
  <meta name="twitter:creator" content="@__yoson__" />
  <meta name="citation_author" content="Vikas Bansal" />
  <meta name="citation_author_institution" content="Biomedical Data Science and Machine Learning Group, German Center for Neurodegenerative Diseases, Tübingen 72076, Germany" />
  <meta name="citation_author_orcid" content="0000-0002-0944-7226" />
  <meta name="twitter:creator" content="@VikasBansal1989" />
  <meta name="citation_author" content="Stephen Capone" />
  <meta name="citation_author_institution" content="St. George&#39;s University School of Medicine, St. George&#39;s, Grenada" />
  <meta name="citation_author_orcid" content="0000-0001-7231-1535" />
  <meta name="citation_author" content="John J. Dziak" />
  <meta name="citation_author_institution" content="Edna Bennett Pierce Prevention Research Center, The Pennsylvania State University, University Park, PA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-0762-5495" />
  <meta name="citation_author" content="Yuchen Sun" />
  <meta name="citation_author_institution" content="Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America" />
  <meta name="citation_author" content="Yanjun Qi" />
  <meta name="citation_author_institution" content="Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-5796-7453" />
  <meta name="citation_author" content="Lamonica Shinholster" />
  <meta name="citation_author_institution" content="Mercer University, Macon, GA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-6285-005X" />
  <meta name="citation_author" content="Temitayo Lukan" />
  <meta name="citation_author_institution" content="University of Pennsylvania, Philadelphia, PA, United States of America" />
  <meta name="citation_author" content="Sergey Knyazev" />
  <meta name="citation_author_institution" content="Georgia State University, Atlanta, GA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-0385-1831" />
  <meta name="twitter:creator" content="@SeKnyaz" />
  <meta name="citation_author" content="Dimitri Perrin" />
  <meta name="citation_author_institution" content="School of Computer Science, Queensland University of Technology, Brisbane, Australia" />
  <meta name="citation_author_institution" content="Centre for Data Science, Queensland University of Technology, Brisbane, Australia" />
  <meta name="citation_author_orcid" content="0000-0002-4007-5256" />
  <meta name="twitter:creator" content="@dperrin" />
  <meta name="citation_author" content="Serghei Mangul" />
  <meta name="citation_author_institution" content="Department of Clinical Pharmacy, School of Pharmacy, University of Southern California, Los Angeles, CA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-4770-3443" />
  <meta name="twitter:creator" content="@serghei_mangul" />
  <meta name="citation_author" content="Shikta Das" />
  <meta name="citation_author_institution" content="C4X Discovery, London, United Kingdom" />
  <meta name="citation_author_institution" content="Medical Research Council LHA, Institute of Cardiovascular Studies, University College London, London, United Kingdom" />
  <meta name="citation_author_orcid" content="0000-0002-8291-2788" />
  <meta name="twitter:creator" content="@shikta_das" />
  <meta name="citation_author" content="Gregory L Szeto" />
  <meta name="citation_author_institution" content="Allen Institute for Immunology, Seattle, WA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-7604-1333" />
  <meta name="twitter:creator" content="@greg_szeto" />
  <meta name="citation_author" content="Tiago Lubiana" />
  <meta name="citation_author_institution" content="Department of Clinical and Toxicological Analyses, School of Pharmaceutical Sciences, University of São Paulo, São Paulo, Brazil" />
  <meta name="citation_author_orcid" content="0000-0003-2473-2313" />
  <meta name="twitter:creator" content="@lubianat" />
  <meta name="citation_author" content="David Mai" />
  <meta name="citation_author_institution" content="Department of Bioengineering, University of Pennsylvania, Philadelphia, PA, USA" />
  <meta name="citation_author_orcid" content="0000-0002-9238-0164" />
  <meta name="twitter:creator" content="@daveomai" />
  <meta name="citation_author" content="COVID-19 Review Consortium" />
  <meta name="citation_author" content="Rishi Raj Goel" />
  <meta name="citation_author_institution" content="Institute for Immunology, University of Pennsylvania, Philadelphia, PA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-1715-5191" />
  <meta name="twitter:creator" content="@rishirajgoel" />
  <meta name="citation_author" content="Joel D Boerckel" />
  <meta name="citation_author_institution" content="Department of Orthopaedic Surgery, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, United States of America" />
  <meta name="citation_author_institution" content="Department of Bioengineering, University of Pennsylvania, Philadelphia, PA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-3126-3025" />
  <meta name="twitter:creator" content="@jboerckel" />
  <meta name="citation_author" content="Amruta Naik" />
  <meta name="citation_author_institution" content="Children&#39;s Hospital of Philadelphia, Philadelphia, PA, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-0673-2643" />
  <meta name="citation_author" content="Yusha Sun" />
  <meta name="citation_author_institution" content="Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0003-4835-3000" />
  <meta name="citation_author" content="Daniel S. Himmelstein" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Related Sciences" />
  <meta name="citation_author_orcid" content="0000-0002-3012-7446" />
  <meta name="twitter:creator" content="@dhimmel" />
  <meta name="citation_author" content="Jeremy P. Kamil" />
  <meta name="citation_author_institution" content="Department of Microbiology and Immunology, Louisiana State University Health Sciences Center Shreveport, Shreveport, Louisiana, USA" />
  <meta name="citation_author_orcid" content="0000-0001-8422-7656" />
  <link rel="canonical" href="https://greenelab.github.io/covid19-review/" />
  <meta property="og:url" content="https://greenelab.github.io/covid19-review/" />
  <meta property="twitter:url" content="https://greenelab.github.io/covid19-review/" />
  <meta name="citation_fulltext_html_url" content="https://greenelab.github.io/covid19-review/" />
  <meta name="citation_pdf_url" content="https://greenelab.github.io/covid19-review/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://greenelab.github.io/covid19-review/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://greenelab.github.io/covid19-review/v/caa4d5ac0458eb37498ef0e7d86c1a3e1e39b228/" />
  <meta name="manubot_html_url_versioned" content="https://greenelab.github.io/covid19-review/v/caa4d5ac0458eb37498ef0e7d86c1a3e1e39b228/" />
  <meta name="manubot_pdf_url_versioned" content="https://greenelab.github.io/covid19-review/v/caa4d5ac0458eb37498ef0e7d86c1a3e1e39b228/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <meta property="og:image" content="https://github.com/greenelab/covid19-review/raw/caa4d5ac0458eb37498ef0e7d86c1a3e1e39b228/content/images/thumbnail.png" />
  <meta property="twitter:image" content="https://github.com/greenelab/covid19-review/raw/caa4d5ac0458eb37498ef0e7d86c1a3e1e39b228/content/images/thumbnail.png" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
bibliography:
- content/manual-references-2021-02-08.json
- content/manual-references.bib
- content/manual-references.json
manubot-output-bibliography: output/references.json
manubot-output-citekeys: output/citations.tsv
manubot-requests-cache-path: ci/cache/requests-cache
manubot-clear-requests-cache: false
...






<small><em>
This manuscript
([permalink](https://greenelab.github.io/covid19-review/v/caa4d5ac0458eb37498ef0e7d86c1a3e1e39b228/))
was automatically generated
from [greenelab/covid19-review@caa4d5a](https://github.com/greenelab/covid19-review/tree/caa4d5ac0458eb37498ef0e7d86c1a3e1e39b228)
on September 27, 2021.
It is also available as a [PDF](https://greenelab.github.io/covid19-review/v/caa4d5ac0458eb37498ef0e7d86c1a3e1e39b228/manuscript.pdf).
Snapshots of individual sections have been published [@individual-pathogenesis; @individual-nutraceuticals; @individual-pharmaceuticals; @individual-methods].
</em></small>

<!-- include the Font Awesome library, per: https://fontawesome.com/start -->
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css">
[
[]{.fas .fa-info-circle .fa-lg} **This in progress manuscript is not intended for the general public.**<br>
This is a review paper that is authored by scientists for an audience of scientists to discuss research that is in progress.
If you are interested in guidelines on testing, therapies, or other issues related to your health, you should not use this document.
Instead, you should collect information from your local health department, the [CDC's guidance](https://www.cdc.gov/coronavirus/2019-ncov/index.html), or your own government.
]{.banner .lightred}

## Authors {.unnumbered}



+ **Halie M. Rando**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-7688-1770](https://orcid.org/0000-0001-7688-1770) · ![GitHub icon](images/github.svg){.inline_icon}
    [rando2](https://github.com/rando2) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [tamefoxtime](https://twitter.com/tamefoxtime)<br>
  <small>
     Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, Colorado, United States of America; Center for Health AI, University of Colorado School of Medicine, Aurora, Colorado, United States of America
     · Funded by the Gordon and Betty Moore Foundation (GBMF 4552); the National Human Genome Research Institute (R01 HG010067)
  </small>

+ **Casey S. Greene**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-8713-9213](https://orcid.org/0000-0001-8713-9213) · ![GitHub icon](images/github.svg){.inline_icon}
    [cgreene](https://github.com/cgreene) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [GreeneScientist](https://twitter.com/GreeneScientist)<br>
  <small>
     Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Philadelphia, Pennsylvania, United States of America; Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, Colorado, United States of America; Center for Health AI, University of Colorado School of Medicine, Aurora, Colorado, United States of America
     · Funded by the Gordon and Betty Moore Foundation (GBMF 4552); the National Human Genome Research Institute (R01 HG010067)
  </small>

+ **Michael P. Robson**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-4859-0033](https://orcid.org/0000-0002-4859-0033) · ![GitHub icon](images/github.svg){.inline_icon}
    [mprobson](https://github.com/mprobson)<br>
  <small>
     Department of Computing Sciences, Villanova University, Villanova, Pennsylvania, United States of America
  </small>

+ **Simina M. Boca**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-1400-3398](https://orcid.org/0000-0002-1400-3398) · ![GitHub icon](images/github.svg){.inline_icon}
    [SiminaB](https://github.com/SiminaB)<br>
  <small>
     Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, District of Columbia, United States of America; Early Biometrics & Statistical Innovation, Data Science & Artificial Intelligence, R & D, AstraZeneca, Gaithersburg, Maryland, United States of America
  </small>

+ **Nils Wellhausen**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-8955-7582](https://orcid.org/0000-0001-8955-7582) · ![GitHub icon](images/github.svg){.inline_icon}
    [nilswellhausen](https://github.com/nilswellhausen)<br>
  <small>
     Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
  </small>

+ **Ronan Lordan**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-9668-3368](https://orcid.org/0000-0001-9668-3368) · ![GitHub icon](images/github.svg){.inline_icon}
    [RLordan](https://github.com/RLordan) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [el_ronan](https://twitter.com/el_ronan)<br>
  <small>
     Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104-5158, USA
  </small>

+ **Christian Brueffer**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-3826-0989](https://orcid.org/0000-0002-3826-0989) · ![GitHub icon](images/github.svg){.inline_icon}
    [cbrueffer](https://github.com/cbrueffer) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [cbrueffer](https://twitter.com/cbrueffer)<br>
  <small>
     Department of Clinical Sciences, Lund University, Lund, Sweden
  </small>

+ **Sandipan Ray**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-9960-5768](https://orcid.org/0000-0002-9960-5768) · ![GitHub icon](images/github.svg){.inline_icon}
    [rays1987](https://github.com/rays1987)<br>
  <small>
     Department of Biotechnology, Indian Institute of Technology Hyderabad, Kandi, Sangareddy 502285, Telangana, India
  </small>

+ **Lucy D\'Agostino McGowan**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-7297-9359](https://orcid.org/0000-0001-7297-9359) · ![GitHub icon](images/github.svg){.inline_icon}
    [LucyMcGowan](https://github.com/LucyMcGowan) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [LucyStats](https://twitter.com/LucyStats)<br>
  <small>
     Department of Mathematics and Statistics, Wake Forest University, Winston-Salem, North Carolina, United States of America
  </small>

+ **Anthony Gitter**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-5324-9833](https://orcid.org/0000-0002-5324-9833) · ![GitHub icon](images/github.svg){.inline_icon}
    [agitter](https://github.com/agitter) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [anthonygitter](https://twitter.com/anthonygitter)<br>
  <small>
     Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin, United States of America; Morgridge Institute for Research, Madison, Wisconsin, United States of America
     · Funded by John W. and Jeanne M. Rowe Center for Research in Virology
  </small>

+ **Anna Ada Dattoli**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-1462-831X](https://orcid.org/0000-0003-1462-831X) · ![GitHub icon](images/github.svg){.inline_icon}
    [aadattoli](https://github.com/aadattoli) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [aadattoli](https://twitter.com/aadattoli)<br>
  <small>
     Department of Systems Pharmacology & Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
  </small>

+ **Ryan Velazquez**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-3655-3403](https://orcid.org/0000-0002-3655-3403) · ![GitHub icon](images/github.svg){.inline_icon}
    [rdvelazquez](https://github.com/rdvelazquez)<br>
  <small>
     Azimuth1, McLean, Virginia, United States of America
  </small>

+ **John P. Barton**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-1467-421X](https://orcid.org/0000-0003-1467-421X) · ![GitHub icon](images/github.svg){.inline_icon}
    [johnbarton](https://github.com/johnbarton) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [_jpbarton](https://twitter.com/_jpbarton)<br>
  <small>
     Department of Physics and Astronomy, University of California-Riverside, Riverside, California, United States of America
  </small>

+ **Jeffrey M. Field**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-7161-7284](https://orcid.org/0000-0001-7161-7284) · ![GitHub icon](images/github.svg){.inline_icon}
    [Jeff-Field](https://github.com/Jeff-Field)<br>
  <small>
     Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
  </small>

+ **Bharath Ramsundar**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-8450-4262](https://orcid.org/0000-0001-8450-4262) · ![GitHub icon](images/github.svg){.inline_icon}
    [rbharath](https://github.com/rbharath) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [rbhar90](https://twitter.com/rbhar90)<br>
  <small>
     The DeepChem Project, https://deepchem.io/
  </small>

+ **Adam L. MacLean**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-0689-7907](https://orcid.org/0000-0003-0689-7907) · ![GitHub icon](images/github.svg){.inline_icon}
    [adamlmaclean](https://github.com/adamlmaclean) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [adamlmaclean](https://twitter.com/adamlmaclean)<br>
  <small>
     Department of Quantitative and Computational Biology, University of Southern California, Los Angeles, California, United States of America
  </small>

+ **Alexandra J. Lee**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-0208-3730](https://orcid.org/0000-0002-0208-3730) · ![GitHub icon](images/github.svg){.inline_icon}
    [ajlee21](https://github.com/ajlee21)<br>
  <small>
     Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
     · Funded by the Gordon and Betty Moore Foundation (GBMF 4552)
  </small>

+ **Immunology Institute of the Icahn School of Medicine**<br> · ![GitHub icon](images/github.svg){.inline_icon}
    [ismms-himc](https://github.com/ismms-himc)<br>
  <small>
     Immunology Institute of the Icahn School of Medicine
  </small>

+ **Fengling Hu**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-1081-5038](https://orcid.org/0000-0003-1081-5038) · ![GitHub icon](images/github.svg){.inline_icon}
    [hufengling](https://github.com/hufengling) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [hufengling](https://twitter.com/hufengling)<br>
  <small>
     Department of Biostatistics, Epidemiology and Informatics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
  </small>

+ **Nafisa M. Jadavji**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-3557-7307](https://orcid.org/0000-0002-3557-7307) · ![GitHub icon](images/github.svg){.inline_icon}
    [nafisajadavji](https://github.com/nafisajadavji) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [nafisajadavji](https://twitter.com/nafisajadavji)<br>
  <small>
     Biomedical Science, Midwestern University, Glendale, AZ, United States of America; Department of Neuroscience, Carleton University, Ottawa, Ontario, Canada
     · Funded by the American Heart Association (20AIREA35050015)
  </small>

+ **Elizabeth Sell**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-9658-1107](https://orcid.org/0000-0002-9658-1107) · ![GitHub icon](images/github.svg){.inline_icon}
    [esell17](https://github.com/esell17)<br>
  <small>
     Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
  </small>

+ **Vincent Rubinetti**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-4655-3773](https://orcid.org/0000-0002-4655-3773) · ![GitHub icon](images/github.svg){.inline_icon}
    [vincerubinetti](https://github.com/vincerubinetti)<br>
  <small>
     Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Center for Health AI, University of Colorado School of Medicine, Aurora, Colorado, United States of America
  </small>

+ **Jinhui Wang**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-5796-8130](https://orcid.org/0000-0002-5796-8130) · ![GitHub icon](images/github.svg){.inline_icon}
    [jinhui2](https://github.com/jinhui2)<br>
  <small>
     Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
  </small>

+ **Diane N. Rafizadeh**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-2838-067X](https://orcid.org/0000-0002-2838-067X) · ![GitHub icon](images/github.svg){.inline_icon}
    [dianerafi](https://github.com/dianerafi)<br>
  <small>
     Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Department of Chemistry, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
     · Funded by NIH Medical Scientist Training Program T32 GM07170
  </small>

+ **Ashwin N. Skelly**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-1565-3376](https://orcid.org/0000-0002-1565-3376) · ![GitHub icon](images/github.svg){.inline_icon}
    [anskelly](https://github.com/anskelly)<br>
  <small>
     Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Institute for Immunology, University of Pennsylvania Perelman School of Medicine, Philadelphia, United States of America
     · Funded by NIH Medical Scientist Training Program T32 GM07170
  </small>

+ **Marouen Ben Guebila**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-5934-966X](https://orcid.org/0000-0001-5934-966X) · ![GitHub icon](images/github.svg){.inline_icon}
    [marouenbg](https://github.com/marouenbg) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [marouenbg](https://twitter.com/marouenbg)<br>
  <small>
     Department of Biostatistics, Harvard School of Public Health, Boston, Massachusetts, United States of America
  </small>

+ **Likhitha Kolla**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-1169-906X](https://orcid.org/0000-0002-1169-906X) · ![GitHub icon](images/github.svg){.inline_icon}
    [likhithakolla](https://github.com/likhithakolla) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [lkolla2018](https://twitter.com/lkolla2018)<br>
  <small>
     Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
     · Funded by NIH Medical Scientist Training Program T32 GM07170
  </small>

+ **David Manheim**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-8599-8380](https://orcid.org/0000-0001-8599-8380) · ![GitHub icon](images/github.svg){.inline_icon}
    [davidmanheim](https://github.com/davidmanheim) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [davidmanheim](https://twitter.com/davidmanheim)<br>
  <small>
     1DaySooner, Delaware, United States of America; Risk and Health Communication Research Center, School of Public Health, University of Haifa, Haifa, Israel
  </small>

+ **Soumita Ghosh**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-2783-2750](https://orcid.org/0000-0002-2783-2750) · ![GitHub icon](images/github.svg){.inline_icon}
    [soumitagh](https://github.com/soumitagh)<br>
  <small>
     Institute of Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
  </small>

+ **James Brian Byrd**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-0509-3520](https://orcid.org/0000-0002-0509-3520) · ![GitHub icon](images/github.svg){.inline_icon}
    [byrdjb](https://github.com/byrdjb) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [thebyrdlab](https://twitter.com/thebyrdlab)<br>
  <small>
     University of Michigan School of Medicine, Ann Arbor, Michigan, United States of America
     · Funded by NIH K23HL128909; FastGrants
  </small>

+ **YoSon Park**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-0465-4744](https://orcid.org/0000-0002-0465-4744) · ![GitHub icon](images/github.svg){.inline_icon}
    [ypar](https://github.com/ypar) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [__yoson__](https://twitter.com/__yoson__)<br>
  <small>
     Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
     · Funded by NHGRI R01 HG10067
  </small>

+ **Vikas Bansal**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-0944-7226](https://orcid.org/0000-0002-0944-7226) · ![GitHub icon](images/github.svg){.inline_icon}
    [bansalvi](https://github.com/bansalvi) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [VikasBansal1989](https://twitter.com/VikasBansal1989)<br>
  <small>
     Biomedical Data Science and Machine Learning Group, German Center for Neurodegenerative Diseases, Tübingen 72076, Germany
  </small>

+ **Stephen Capone**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-7231-1535](https://orcid.org/0000-0001-7231-1535) · ![GitHub icon](images/github.svg){.inline_icon}
    [scapone01](https://github.com/scapone01)<br>
  <small>
     St. George's University School of Medicine, St. George's, Grenada
  </small>

+ **John J. Dziak**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-0762-5495](https://orcid.org/0000-0003-0762-5495) · ![GitHub icon](images/github.svg){.inline_icon}
    [dziakj1](https://github.com/dziakj1)<br>
  <small>
     Edna Bennett Pierce Prevention Research Center, The Pennsylvania State University, University Park, PA, United States of America
  </small>

+ **Yuchen Sun**<br> · ![GitHub icon](images/github.svg){.inline_icon}
    [kevinsunofficial](https://github.com/kevinsunofficial)<br>
  <small>
     Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America
  </small>

+ **Yanjun Qi**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-5796-7453](https://orcid.org/0000-0002-5796-7453) · ![GitHub icon](images/github.svg){.inline_icon}
    [qiyanjun](https://github.com/qiyanjun)<br>
  <small>
     Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America
  </small>

+ **Lamonica Shinholster**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-6285-005X](https://orcid.org/0000-0001-6285-005X) · ![GitHub icon](images/github.svg){.inline_icon}
    [LSH2126](https://github.com/LSH2126)<br>
  <small>
     Mercer University, Macon, GA, United States of America
     · Funded by the Center for Global Genomics and Health Equity at the University of Pennsylvania
  </small>

+ **Temitayo Lukan**<br> · ![GitHub icon](images/github.svg){.inline_icon}
    [tlukan](https://github.com/tlukan)<br>
  <small>
     University of Pennsylvania, Philadelphia, PA, United States of America
  </small>

+ **Sergey Knyazev**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-0385-1831](https://orcid.org/0000-0003-0385-1831) · ![GitHub icon](images/github.svg){.inline_icon}
    [Sergey-Knyazev](https://github.com/Sergey-Knyazev) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [SeKnyaz](https://twitter.com/SeKnyaz)<br>
  <small>
     Georgia State University, Atlanta, GA, United States of America
  </small>

+ **Dimitri Perrin**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-4007-5256](https://orcid.org/0000-0002-4007-5256) · ![GitHub icon](images/github.svg){.inline_icon}
    [SystemsResearch](https://github.com/SystemsResearch) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [dperrin](https://twitter.com/dperrin)<br>
  <small>
     School of Computer Science, Queensland University of Technology, Brisbane, Australia; Centre for Data Science, Queensland University of Technology, Brisbane, Australia
  </small>

+ **Serghei Mangul**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-4770-3443](https://orcid.org/0000-0003-4770-3443) · ![GitHub icon](images/github.svg){.inline_icon}
    [smangul1](https://github.com/smangul1) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [serghei_mangul](https://twitter.com/serghei_mangul)<br>
  <small>
     Department of Clinical Pharmacy, School of Pharmacy, University of Southern California, Los Angeles, CA, United States of America
  </small>

+ **Shikta Das**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-8291-2788](https://orcid.org/0000-0002-8291-2788) · ![GitHub icon](images/github.svg){.inline_icon}
    [shiktadas](https://github.com/shiktadas) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [shikta_das](https://twitter.com/shikta_das)<br>
  <small>
     C4X Discovery, London, United Kingdom; Medical Research Council LHA, Institute of Cardiovascular Studies, University College London, London, United Kingdom
  </small>

+ **Gregory L Szeto**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-7604-1333](https://orcid.org/0000-0001-7604-1333) · ![GitHub icon](images/github.svg){.inline_icon}
    [gregszetoAI](https://github.com/gregszetoAI) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [greg_szeto](https://twitter.com/greg_szeto)<br>
  <small>
     Allen Institute for Immunology, Seattle, WA, United States of America
  </small>

+ **Tiago Lubiana**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-2473-2313](https://orcid.org/0000-0003-2473-2313) · ![GitHub icon](images/github.svg){.inline_icon}
    [lubianat](https://github.com/lubianat) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [lubianat](https://twitter.com/lubianat)<br>
  <small>
     Department of Clinical and Toxicological Analyses, School of Pharmaceutical Sciences, University of São Paulo, São Paulo, Brazil
  </small>

+ **David Mai**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-9238-0164](https://orcid.org/0000-0002-9238-0164) · ![GitHub icon](images/github.svg){.inline_icon}
    [davemai](https://github.com/davemai) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [daveomai](https://twitter.com/daveomai)<br>
  <small>
     Department of Bioengineering, University of Pennsylvania, Philadelphia, PA, USA
  </small>

+ **COVID-19 Review Consortium**<br><br>
  <small>
  </small>

+ **Rishi Raj Goel**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-1715-5191](https://orcid.org/0000-0003-1715-5191) · ![GitHub icon](images/github.svg){.inline_icon}
    [rishirajgoel](https://github.com/rishirajgoel) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [rishirajgoel](https://twitter.com/rishirajgoel)<br>
  <small>
     Institute for Immunology, University of Pennsylvania, Philadelphia, PA, United States of America
  </small>

+ **Joel D Boerckel**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-3126-3025](https://orcid.org/0000-0003-3126-3025) · ![GitHub icon](images/github.svg){.inline_icon}
    [jboerckel](https://github.com/jboerckel) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [jboerckel](https://twitter.com/jboerckel)<br>
  <small>
     Department of Orthopaedic Surgery, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, United States of America; Department of Bioengineering, University of Pennsylvania, Philadelphia, PA, United States of America
  </small>

+ **Amruta Naik**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-0673-2643](https://orcid.org/0000-0003-0673-2643) · ![GitHub icon](images/github.svg){.inline_icon}
    [NAIKA86](https://github.com/NAIKA86)<br>
  <small>
     Children's Hospital of Philadelphia, Philadelphia, PA, United States of America
  </small>

+ **Yusha Sun**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-4835-3000](https://orcid.org/0000-0003-4835-3000) · ![GitHub icon](images/github.svg){.inline_icon}
    [yusha-sun](https://github.com/yusha-sun)<br>
  <small>
     Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
  </small>

+ **Daniel S. Himmelstein**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-3012-7446](https://orcid.org/0000-0002-3012-7446) · ![GitHub icon](images/github.svg){.inline_icon}
    [dhimmel](https://github.com/dhimmel) · ![Twitter icon](images/twitter.svg){.inline_icon}
    [dhimmel](https://twitter.com/dhimmel)<br>
  <small>
     Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Related Sciences
     · Funded by GBMF4552
  </small>

+ **Jeremy P. Kamil**<br> ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-8422-7656](https://orcid.org/0000-0001-8422-7656)<br>
  <small>
     Department of Microbiology and Immunology, Louisiana State University Health Sciences Center Shreveport, Shreveport, Louisiana, USA
  </small>



**COVID-19 Review Consortium:**
Vikas Bansal, John P. Barton, Simina M. Boca, Joel D Boerckel, Christian Brueffer, James Brian Byrd, Stephen Capone, Shikta Das, Anna Ada Dattoli, John J. Dziak, Jeffrey M. Field, Soumita Ghosh, Anthony Gitter, Rishi Raj Goel, Casey S. Greene, Marouen Ben Guebila, Daniel S. Himmelstein, Fengling Hu, Nafisa M. Jadavji, Jeremy P. Kamil, Sergey Knyazev, Likhitha Kolla, Alexandra J. Lee, Ronan Lordan, Tiago Lubiana, Temitayo Lukan, Adam L. MacLean, David Mai, Serghei Mangul, David Manheim, Lucy D\'Agostino McGowan, Amruta Naik, YoSon Park, Dimitri Perrin, Yanjun Qi, Diane N. Rafizadeh, Bharath Ramsundar, Halie M. Rando, Sandipan Ray, Michael P. Robson, Vincent Rubinetti, Elizabeth Sell, Lamonica Shinholster, Ashwin N. Skelly, Yuchen Sun, Yusha Sun, Gregory L Szeto, Ryan Velazquez, Jinhui Wang, Nils Wellhausen


Authors are ordered arbitrarily.


## Pathogenesis, Symptomatology, and Transmission of SARS-CoV-2 through Analysis of Viral Genomics and Structure

### Abstract

The novel coronavirus SARS-CoV-2, which emerged in late 2019, has since spread around the world and infected hundreds of millions of people with coronavirus disease 2019 (COVID-19).
While this viral species was unknown prior to January 2020, its similarity to other coronaviruses that infect humans has allowed for rapid insight into the mechanisms that it uses to infect human hosts, as well as the ways in which the human immune system can respond.
Here, we contextualize SARS-CoV-2 among other coronaviruses and identify what is known and what can be inferred about its behavior once inside a human host.
Because the genomic content of coronaviruses, which specifies the virus's structure, is highly conserved, early genomic analysis provided a significant head start in predicting viral pathogenesis and in understanding potential differences among variants.
The pathogenesis of the virus offers insights into symptomatology, transmission, and individual susceptibility.
Additionally, prior research into interactions between the human immune system and coronaviruses has identified how these viruses can evade the immune system's protective mechanisms.
We also explore systems-level research into the regulatory and proteomic effects of SARS-CoV-2 infection and the immune response.
Understanding the structure and behavior of the virus serves to contextualize the many facets of the COVID-19 pandemic and can influence efforts to control the virus and treat the disease.

### Importance

COVID-19 involves a number of organ systems and can present with a wide range of symptoms.
From how the virus infects cells to how it spreads between people, the available research suggests that these patterns are very similar to those seen in the closely related viruses SARS-CoV-1 and possibly MERS-CoV.
Understanding the pathogenesis of the SARS-CoV-2 virus also contextualizes how the different biological systems affected by COVID-19 connect.
Exploring the structure, phylogeny, and pathogenesis of the virus therefore helps to guide interpretation of the broader impacts of the virus on the human body and on human populations.
For this reason, an in-depth exploration of viral mechanisms is critical to a robust understanding of SARS-CoV-2 and, potentially, future emergent HCoV.

### Introduction

The current coronavirus disease 2019 (COVID-19) pandemic, caused by the _Severe acute respiratory syndrome-related coronavirus 2_ (SARS-CoV-2) virus, represents an acute global health crisis.
Symptoms of the disease can range from mild to severe or fatal [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html] and can affect a variety of organs and systems [@doi:10.23750/abm.v91i1.9397].
Outcomes of infection can include acute respiratory distress (ARDS) and acute lung injury, as well as damage to other organ systems [@doi:10.23750/abm.v91i1.9397; @doi:10.1002/ctm2.16].
Understanding the progression of the disease, including these diverse symptoms, depends on understanding how the virus interacts with the host.
Additionally, the fundamental biology of the virus can provide insights into how it is transmitted among people, which can, in turn, inform efforts to control its spread.
As a result, a thorough understanding of the pathogenesis of SARS-CoV-2 is a critical foundation on which to build an understanding of COVID-19 and the pandemic as a whole.

The rapid identification and release of the genomic sequence of the virus in January 2020 [@doi:10.46234/ccdcw2020.017] provided early insight into the virus in a comparative genomic context.
The viral genomic sequence clusters with known coronaviruses (order *Nidovirales*, family *Coronaviridae*, subfamily *Orthocoronavirinae*).
Phylogenetic analysis of the coronaviruses reveals four major subclades, each corresponding to a genus: the alpha, beta, gamma, and delta coronaviruses.
Among them, alpha and beta coronaviruses infect mammalian species, gamma coronaviruses infect avian species, and delta coronaviruses infect both mammalian and avian species [@doi:10.1146/annurev-virology-110615-042301].
The novel virus now known as SARS-CoV-2 was identified as a beta coronavirus belonging to the B lineage based on phylogenetic analysis of a polymerase chain reaction (PCR) amplicon fragment from five patients along with the full genomic sequence [@doi:10/ggjs7j].
This lineage also includes the _Severe acute respiratory syndrome-related coronavirus_ (SARS-CoV-1) that caused the 2002-2003 outbreak of Severe Acute Respiratory Syndrome (SARS) in humans [@doi:10/ggjs7j].
(Note that these subclades are not to be confused with variants of concern within SARS-CoV-2 labeled with Greek letters; i.e., the Delta variant of SARS-CoV-2 is still a beta coronavirus.)

Because viral structure and mechanisms of pathogenicity are highly conserved within the order, this phylogenetic analysis provided a basis for forming hypotheses about how the virus interacts with hosts, including which tissues, organs, and systems would be most susceptible to SARS-CoV-2 infection.
Coronaviruses that infect humans (HCoV) are not common, but prior research into other HCoV such as SARS-CoV-1 and _Middle East respiratory syndrome-related coronavirus_ (MERS-CoV), as well as other viruses infecting humans such as a variety of influenza species, established a strong foundation that accelerated the pace of SARS-CoV-2 research.

Coronaviruses are large viruses that can be identified by their distinctive "crown-like" shape (Figure @fig:genome-structure).
Their spherical virions are made from lipid envelopes ranging from 100 to 160 nanometers in which peplomers (protruding structures) of two to three spike (S) glycoproteins are anchored, creating the crown [@isbn:978-0781760607; @doi:10/dvvg2h].
These spikes, which are critical to both viral pathogenesis and to the response by the host immune response, have been visualized using cryo-electron microscopy [@doi:10.1126/science.abb2507].
Because they induce the human immune response, they are also the target of many proposed therapeutic agents [@individual-pharmaceuticals; @individual-nutraceuticals].
Viral pathogenesis is typically broken down into three major components: entry, replication, and spread [@isbn:0-9631172-1-1 {chap. 45}].
However, in order to draw a more complete picture of pathogenesis, it is also necessary to examine how infection manifests clinically, identify systems-level interactions between the virus and the human body, and consider the possible effects of variation or evolutionary change on pathogenesis and virulence.
Thus, clinical medicine and traditional biology are both important pieces of the puzzle of SARS-CoV-2 presentation and pathogenesis.

### Coronavirus Structure and Pathogenesis

#### Structure of Coronaviruses

Genome structure is highly conserved among coronaviruses, meaning that the relationship between the SARS-CoV-2 genome and its pathogenesis can be inferred from prior research in related viral species.
The genomes of viruses in the *Nidovirales* order share several fundamental characteristics.
They are non-segmented, which means the viral genome is a single continuous strand of RNA, and are enveloped, which means that the genome and capsid are encased by a lipid bilayer.
Coronaviruses have large positive-sense RNA (ssRNA+) genomes ranging from 27 to 32 kilobases in length [@doi:10.1007/978-1-4939-2438-7_1; @doi:10/ggjr43].
The SARS-CoV-2 genome lies in the middle of this range at 29,903 bp [@doi:10/ggjr43].
Genome organization is highly conserved within the order [@doi:10.1007/978-1-4939-2438-7_1].
There are three major genomic regions: one containing the replicase gene, one containing the genes encoding structural proteins, and interspersed accessory genes [@doi:10.1007/978-1-4939-2438-7_1] (Figure @fig:genome-structure).
The replicase gene comprises about two-thirds of the genome and consists of two open reading frames that are translated with ribosomal frameshifting [@doi:10.1007/978-1-4939-2438-7_1].
This polypeptide is then translated into 16 non-structural proteins (nsp), except in gammacoronaviruses where nsp1 is absent, that form the replication machinery used to synthesize viral RNA [@doi:10.1002/jmv.25681].
The remaining third of the genome encodes structural proteins, including the spike (S), membrane, envelope, and nucleocapsid proteins.
Additional accessory genes are sometimes present between these two regions, depending on the species or strain.
Much attention has been focused on the S protein, which is a critical structure involved in cell entry.

![**Structure of SARS-CoV-2 capsid and genome.**
A) The genomic structure of coronaviruses is highly conserved and includes three main regions.
Open reading frames (ORF) 1a and 1b contain two polyproteins that encode the non-structural proteins (nsp).
The nsp include enzymes such as RNA-dependent RNA Polymerase (RdRp).
The last third of the genome encodes structural proteins, including the spike (S), envelope (E), membrane (M) and nucleocapsid (N) proteins.
Accessory genes can also be interspersed throughout the genome [@doi:10.1007/978-1-4939-2438-7_1].
B) The physical structure of the coronavirus virion, including the components determined by the conserved structural proteins S, E, M and N.
](images/genome-structure.png){#fig:genome-structure secno=1}

#### Pathogenic Mechanisms of Coronaviruses

While it is possible that SARS-CoV-1 and SARS-CoV-2, like most viruses, enter cells through endocytosis, a process conserved among coronaviruses enables them to target cells for entry through fusion with the plasma membrane [@doi:10.1038/cr.2008.15; @doi:10.1146/annurev-biochem-060208-104626].
Cell entry proceeds in three steps: binding, cleavage, and fusion.
First, the viral spike protein binds to a host cell via a recognized receptor or entry point.
Coronaviruses can bind to a range of host receptors [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009], with binding conserved only at the genus level [@doi:10.1146/annurev-virology-110615-042301].
Viruses in the beta coronavirus genus, to which SARS-CoV-2 belongs, are known to bind to the CEACAM1 protein, 5-N-acetyl-9-O-acetyl neuraminic acid, and to angiotensin-converting enzyme 2 (ACE2) [@doi:10.3390/v4061011].
This recognition is driven by domains in the S1 subunit [@doi:10.1016/j.cell.2020.02.058].
SARS-CoV-2 has a high affinity for human ACE2, which is expressed in the vascular epithelium, other epithelial cells, and cardiovascular and renal tissues [@doi:10.1007/s00134-020-05985-9; @doi:10.1007/978-0-387-33012-9_85], as well as many others [@doi:10.15252/msb.20209610].
The binding process is guided by the molecular structure of the spike protein, which is structured in three segments: an ectodomain, a transmembrane anchor, and an intracellular tail [@doi:10.1128/JVI.02615-14].
The ectodomain forms the crown-like structures on the viral membrane and contains two subdomains known as the S1 and S2 subunits [@doi:10.1038/nrmicro2090].
The S1 (N-terminal) domain forms the head of the crown and contains the receptor binding motif, and the S2 (C-terminal) domain forms the stalk that supports the head [@doi:10.1038/nrmicro2090].
The S1 subunit guides the binding of the virus to the host cell, and the S2 subunit guides the fusion process [@doi:10.1128/JVI.02615-14].

After the binding of the S1 subunit to an entry point, the spike protein of coronaviruses is often cleaved at the S1/S2 boundary into the S1 and S2 subunits by a host protease [@doi:10.1016/j.cell.2020.02.058; @doi:10/cf8chz; @doi:10.1016/j.antiviral.2020.104792].
This proteolytic priming is important because it prepares the S protein for fusion [@doi:10/cf8chz; @doi:10.1016/j.antiviral.2020.104792].
The two subunits remain bound by van der Waals forces, with the S1 subunit stabilizing the S2 subunit throughout the membrane fusion process [@doi:10.1016/j.cell.2020.02.058].
Cleavage at a second site within S2 (S2') activates _S_ for fusion by inducing conformational changes [@doi:10.1016/j.cell.2020.02.058].
Similar to SARS-CoV-1, SARS-CoV-2 exhibits redundancy in which host proteases can cleave the S protein [@doi:10.1016/j.cell.2020.02.052].
Both transmembrane protease serine protease-2 (TMPRSS-2) and cathepsins B/L have been shown to mediate SARS-CoV-2 S protein proteolytic priming, and small molecule inhibition of these enzymes fully inhibited viral entry _in vitro_ [@doi:10.1016/j.cell.2020.02.052; @doi:10.1038/s41467-020-15562-9].
Other proteases known to cleave the S1/S2 boundary in coronaviruses include TMPRSS-4, trypsin, furin, cathepsins, and human airway trypsin-like protease (HAT) [@doi:10.1038/s41467-020-15562-9].

Unlike in SARS-CoV-1, a second cleavage site featuring a furin-like binding motif is also present near the S1/S2 boundary in SARS-CoV-2 [@doi:10.1016/j.antiviral.2020.104742].
This site is found in HCoV belonging to the A and C lineages of beta coronavirus, including MERS-CoV, but not in the other known members of the B lineage of beta coronavirus that contains SARS-CoV-1 and SARS-CoV-2 [@doi:10.1016/j.antiviral.2020.104742].
It is associated with increased virulence in other viral species [@doi:10.1016/j.antiviral.2020.104742] and may facilitate membrane fusion of SARS-CoV-2 in the absence of other proteases that prime the S1/S2 site [@doi:10.1038/s41392-020-0184-0].
However, given that proteases such as HAT are likely to be present in targets like the human airway, the extent to which this site has had a real-world effect on the spread of SARS-CoV-2 was initially unclear [@doi:10.1038/s41392-020-0184-0].
Subsequent research has supported this site as an important contributor to pathogenesis: _in vitro_ analyses have reported that it bolsters pathogenicity specifically in cell lines derived from human airway cells (Calu3 cell line) [@doi:10.1101/2020.08.26.268854; @doi:10.1038/s41586-021-03237-4; @doi:10.1038/s41564-021-00908-w] and that furin inhibitors reduced pathogenic effects in VeroE6 cells [@doi:10.1016/j.celrep.2020.108254].

Electron microscopy suggests that in some coronaviruses, including SARS-CoV-1 and MERS-CoV, a six-helix bundle separates the two subunits in the postfusion conformation, and the unusual length of this bundle facilitates membrane fusion through the release of additional energy [@doi:10.1146/annurev-virology-110615-042301].
The viral membrane can then fuse with the endosomal membrane to release the viral genome into the host cytoplasm.
Once the virus enters a host cell, the replicase gene is translated and assembled into the viral replicase complex.
This complex then synthesizes the double-stranded RNA (dsRNA) genome from the genomic ssRNA(+).
The dsRNA genome is transcribed and replicated to create viral mRNAs and new ssRNA(+) genomes [@doi:10.1007/978-1-4939-2438-7_1; @url:https://viralzone.expasy.org/30].
From there, the virus can spread into other cells.
In SARS-CoV-2, the insertion of the furin-like binding site near the S1/S2 boundary is also thought to increase cell-cell adhesion, making it possible for the viral genome to spread directly from cell to cell rather than needing to propagate the virion itself [@doi:10.1016/j.isci.2020.101212].
In this way, the genome of SARS-CoV-2 provides insight into the pathogenic behavior of the virus.

Evidence also suggests that SARS-CoV-2 may take advantage of the specific structure of endothelial cells to enter the circulatory system.
Endothelial cells are specialized epithelial cells [@doi:10.1055/s-0030-1253446] that form a barrier between the bloodstream and surrounding tissues.
The endothelium facilitates nutrient, oxygen, and cellular exchange between the blood and vascularized tissues [@PMID:362947].
The luminal (interior) surface of the endothelium is lined with glycocalyx, a network of both membrane-bound and soluble proteins and carbohydrates, primarily proteoglycans and glycoproteins [@doi:10.1007/s00424-007-0212-8; @doi:10.1016/j.cell.2020.09.033].
The glycocalyx varies in thickness from 0.5 microns in the capillaries to 4.5 microns in the carotid arteries and forms a meshwork that localizes both endothelial- and plasma-derived signals to the inner vessel wall [@doi:10.1007/s00424-007-0212-8].
Heparan sulfate is the dominant proteoglycan in the glycocalyx, representing 50-90% of glycocalyx proteoglycan content [@doi:10/bp82dx].
The SARS-CoV-2 spike protein can bind directly to heparan sulfate, which serves in part as a scaffolding molecule to facilitate ACE2 binding and entry into endothelial cells [@doi:10.1016/j.cell.2020.09.033].
A heparan sulfate binding site has also been identified near the ACE2 binding site on the viral receptor binding domain (RBD), and modeling has suggested that heparan sulfate binding yields an open conformation that facilitates binding to ACE2 on the cell surface [@doi:10.1016/j.cell.2020.09.033].
Degrading or removing heparan sulfate was associated with decreased binding [@doi:10.1016/j.cell.2020.09.033].
Heparan sulfate may also interact with the S1/S2 proteolytic cleavage site and other binding sites to promote binding affinity [@doi:10.1101/2020.05.10.087288].
Notably, treatment with soluble heparan sulfate or even heparin (a commonly used anti-coagulant and vasodilator that is similar in structure to heparan sulfate [@doi:10.1007/978-3-642-23056-1_8]) potently blocked spike protein binding and viral infection [@doi:10.1016/j.cell.2020.09.033].
This finding is particularly interesting because degradation of heparan sulfate in the glycocalyx has previously been identified as an important contributor to ARDS and sepsis [@doi:10.1016/bs.ctm.2018.08.005], two common and severe outcomes of COVID-19, and suggests that heparan sulfate could be a target for pharmaceutical inhibition of cell entry by SARS-CoV-2 [@doi:10.1038/s41421-020-00222-5; @doi:10.1016/j.ebiom.2020.102969; @doi:10.1016/j.amjcard.2020.08.005; @doi:10.3390/nu13020328; @doi:10.3389/fimmu.2021.607306].
Together, this evidence suggests that heparan sulfate can serve as an important adhesion molecule for SARS-CoV-2 cell entry.
It may represent a therapeutic target but has not been pursued as much as other candidate targets [@individual-pharmaceuticals].

#### Immune Evasion Strategies

Research in other HCoV provides some indication of how SARS-CoV-2 infection can proceed despite human immune defenses.
Infecting the epithelium can help viruses such as SARS-CoV-1 bypass the physical barriers, such as mucus, that comprise the immune system's first line of defense [@doi:10.1038/mi.2015.127].
Once the virus infiltrates host cells, it is adept at evading detection.
CD163+ and CD68+ macrophage cells are especially crucial for the establishment of SARS-CoV-1 in the body [@doi:10.1038/mi.2015.127].
These cells most likely serve as viral reservoirs that help shield SARS-CoV-1 from the innate immune response.
According to a study on the viral dissemination of SARS-CoV-1 in Chinese macaques, viral RNA could be detected in some monocytes throughout the process of differentiation into dendritic cells [@doi:10.1038/mi.2015.127].
This lack of active viral replication allows SARS-CoV-1 to escape the innate immune response because reduced levels of detectable viral RNA allow the virus to avoid both natural killer cells and Toll-like receptors [@doi:10.1038/mi.2015.127].
Even during replication, SARS-CoV-1 is able to mask its dsRNA genome from detection by the immune system.
Although dsRNA is a pathogen-associated molecular pattern that would typically initiate a response from the innate immune system [@doi:10.3389/fimmu.2018.00829], _in vitro_ analysis of nidoviruses including SARS-CoV-1 suggests that these viruses can induce the development of double-membrane vesicles that protect the dsRNA signature from being detected by the host immune system [@doi:10.1128/jvi.02501-05].
This protective envelope can therefore insulate these coronaviruses from the innate immune system's detection mechanism [@doi:10.1016/j.jpha.2020.03.001].

HCoVs are also known to interfere with the host immune response, rather than just evade it.
For example, the virulence of SARS-CoV-2 is increased by nsp1, which can suppress host gene expression by stalling mRNA translation and inducing endonucleolytic cleavage and mRNA degradation [@doi:10.1126/science.abc8665].
SARS-CoV-1 also evades the immune response by interfering with type I IFN induction signaling, which is a mechanism that leads to cellular resistance to viral infections.
SARS-CoV-1 employs methods such as ubiquitination and degradation of RNA sensor adaptor molecules MAVS and TRAF3/6 [@doi:10.12932/AP-200220-0772].
Also, MERS-CoV downregulates antigen presentation via MHC class I and MHC class II, which leads to a reduction in T cell activation [@doi:10.12932/AP-200220-0772].
These evasion mechanisms, in turn, may facilitate systemic infection.
Coronaviruses such as SARS-CoV-1 are also able to evade the humoral immune response through other mechanisms, such as inhibiting certain cytokine pathways or down-regulating antigen presentation by the cells [@doi:10.1128/jvi.02501-05].

#### Host Cell Susceptibility

ACE2 and TMPRSS-2 have been identified as the primary entry portal and as a critical protease, respectively, in facilitating the entry of SARS-CoV-1 and SARS-CoV-2 into a target cell [@doi:10.1038/nature02145; @doi:10.1126/science.abb2507; @doi:10.1128/JVI.01542-10; @doi:10.1016/j.cell.2020.02.052; @doi:10.1128/JVI.02232-10].
This finding has led to a hypothesized role for the expression of these molecules in determining which cells, tissues, and organs are most susceptible to SARS-CoV-2 infection.
ACE2 is expressed in numerous organs, such as the heart, kidney, and intestine, but it is most prominently expressed in alveolar epithelial cells; this pattern of expression is expected to contribute to the virus' association with lung pathology [@doi:10.1002/jmv.26139; @doi:10.1007/s00134-020-05985-9; @doi:10.1016/j.phrs.2020.104833] as well as that of SARS [@doi:10.1002/path.1570].
A retrospective observational study reported indirect evidence that certain antineoplastic therapies, such as the chemotherapy drug gemcitabine, may reduce risk of SARS-CoV-2 infection in patients with cancer, possibly via decreased ACE2 expression [@doi:10.1001/jamaoncol.2021.3585].
Additionally, the addition of the furin site insertion at the S1/S2 boundary means that SARS-CoV-2 does not require TMPRSS-2 when furin, an ubiquitously expressed endoprotease [@doi:10.1038/nrm934], is present, enabling cell-cell fusion independent of TMPRSS-2 availability [@doi:10.1016/j.molcel.2020.04.022].

Clinical investigations of COVID-19 patients have detected SARS-CoV-2 transcripts in bronchoalveolar lavage fluid (BALF) (93% of specimens), sputum (72%), nasal swabs (63%), fibrobronchoscopy brush biopsies (46%), pharyngeal swabs (32%), feces (29%), and blood (1%) [@doi:10.1001/jama.2020.3786].
Two studies reported that SARS-CoV-2 could not be detected in urine specimens [@doi:10.1001/jama.2020.3786; @doi:10.1001/jama.2020.3204]; however, a third study identified four urine samples (out of 58) that were positive for SARS-CoV-2 nucleic acids [@doi:10.1097/CM9.0000000000000774].
Although respiratory failure remains the leading cause of death for COVID-19 patients [@doi:10.1111/his.14134], SARS-CoV-2 infection can damage many other organ systems including the heart [@doi:10.1001/jamacardio.2020.0950], kidneys [@doi:10.1038/s41585-020-0319-7; @doi:10.1186/s13054-020-02872-z], liver [@doi:10/ggpx6s], and gastrointestinal tract [@doi:10.1053/j.gastro.2020.02.055; @doi:10.1111/1751-2980.12851].
As it becomes clear that SARS-CoV-2 infection can damage multiple organs, the scientific community is pursuing multiple avenues of investigation in order to build a consensus about how the virus affects the human body.

### Clinical Presentation of COVID-19

SARS-CoV-2 pathogenesis is closely linked with the clinical presentation of the COVID-19 disease.
Reports have described diverse symptom profiles associated with COVID-19, with a great deal of variability both within and between institutions and regions.
Definitions for non-severe, severe, and critical COVID-19, along with treatment recommendations, are available from the World Health Organization living guidelines [@doi:10.1136/bmj.m3379].
A large study from Wuhan, China conducted early in the pandemic identified fever and cough as the two most common symptoms that patients reported at hospital admission [@doi:10.1056/NEJMoa2002032], while a retrospective study in China described the clinical presentations of patients infected with SARS-CoV-2 as including lower respiratory tract infection with fever, dry cough, and dyspnea (shortness of breath) [@doi:10/ggnxb3].
This study [@doi:10/ggnxb3] noted that upper respiratory tract symptoms were less common, suggesting that the virus preferentially targets cells located in the lower respiratory tract.
However, data from the New York City region [@doi:10.1001/jama.2020.6775; @doi:10.1056/NEJMc2010419] showed variable rates of fever as a presenting symptom, suggesting that symptoms may not be consistent across individuals.
For example, even within New York City, one study [@doi:10.1001/jama.2020.6775] identified low oxygen saturation (<90% without the use of supplemental oxygen or ventilation support) in 20.4% of patients upon presentation, with fever being present in 30.7%, while another study [@doi:10.1056/NEJMc2010419] reported cough (79.4%), fever (77.1%), and dyspnea (56.5%) as the most common presenting symptoms; both of these studies considered only hospitalized patients.
A later study reported radiographic findings such as ground-glass opacity and bilateral patchy shadowing in the lungs of many hospitalized patients, with most COVID-19 patients having lymphocytopenia, or low levels of lymphocytes (a type of white blood cell) [@doi:10.1056/NEJMoa2002032].
Patients may also experience loss of smell, myalgias (muscle aches), fatigue, or headache.
Gastrointestinal symptoms can also present [@doi:10.15585/mmwr.mm6928a2], and the CDC includes nausea and vomiting, as well as congestion and runny nose, on its list of symptoms consistent with COVID-19 [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html].
An analysis of an app-based survey of 500,000 individuals in the U.S. found that among those tested for SARS-CoV-2, a loss of taste or smell, fever, and a cough were significant predictors of a positive test result [@doi:10.1038/s41562-020-00944-2].
It is important to note that in this study, the predictive value of symptoms may be underestimated if they are not specific to COVID-19.
This underestimation could occur because the outcome measured was a positive, as opposed to a negative, COVID-19 test result, meaning an association would be more easily identified for symptoms that were primarily or exclusively found with COVID-19.
At the time the surveys were conducted, due to limits in U.S. testing infrastructure, respondents typically needed to have some symptoms known to be specific to COVID-19 in order to qualify for testing.
Widespread testing of asymptomatic individuals may therefore provide additional insight into the range of symptoms associated with COVID-19.

Consistent with the wide range of symptoms observed and the pathogenic mechanisms described above, COVID-19 can affect a variety of systems within the body in addition to causing respiratory problems [@doi:10.1038/s41591-020-0968-3].
For example, COVID-19 can lead to acute kidney injury, especially in patients with severe respiratory symptoms or certain preexisting conditions [@doi:10.1016/j.kint.2020.05.006].
Some patients are at risk for collapsing glomerulopathy [@doi:10.1038/s41581-020-0332-3].

COVID-19 can also cause neurological complications [@doi:10.1016/j.bbi.2020.03.031; @doi:10/d259; @doi:10.1001/jamapsychiatry.2021.0500], potentially including stroke, seizures or meningitis [@doi:10.1002/jmv.26000; @doi:10.1002/ana.25807].
One study on autopsy samples suggested that SARS-CoV-2 may be able to enter the central nervous system via the neural–mucosal interface [@doi:10.1038/s41593-020-00758-5].
However, a study of 41 autopsied brains [@doi:10.1093/brain/awab148] found no evidence that the virus can actually infect the central nervous system.
Although there was viral RNA in some brain samples, it was only found in very small amounts, and no viral protein was found.
The RNA may have been in the blood vessels or blood components and not in the brain tissue itself.
Instead, the neuropathological effects of COVID-19 are more likely to be caused indirectly by hypoxia, coagulopathy, or inflammatory processes rather than by infection in the brain [@doi:10.1093/brain/awab148].
COVID-19 has been associated with an increased incidence of large vessel stroke, particularly in patients under the age of 40 [@doi:10.1056/NEJMc2009787], and other thrombotic events including pulmonary embolism and deep vein thrombosis [@doi:10.1016/j.thromres.2020.04.013].
The mechanism behind these complications has been suggested to be related to coagulopathy, with reports indicating the presence of antiphospholipid antibodies [@doi:10.1056/NEJMc2007575] and elevated levels of d-dimer and fibrinogen degradation products in deceased patients [@doi:10.1111/jth.14768].
Other viral infections have been associated with coagulation defects and changes to the coagulation cascade; notably, SARS was also found to lead to disseminated intravascular coagulation and was associated with both pulmonary embolism and deep vein thrombosis [@doi:10.1002/jmv.23354].
The mechanism behind these insults has been suggested to be related to inflammation-induced increases in the von Willebrand factor clotting protein, leading to a pro-coagulative state [@doi:10.1002/jmv.23354].
Abnormal clotting (thromboinflammation or coagulopathy) has been increasingly discussed recently as a possible key mechanism in many cases of severe COVID-19, and may be associated with the high d-dimer levels often observed in severe cases [@doi:10/ggvd74; @doi:10.1186/s13054-020-03060-9; @doi:10.1007/s11239-020-02134-3].
This excessive clotting in lung capillaries has been suggested to be related to a dysregulated activation of the complement system, part of the innate immune system [@doi:10.1172/jci.insight.140711; @doi:10.1172/JCI140183].

Finally, concerns have been raised about long-term sequelae of COVID-19.
Some COVID-19 patients have reported that various somatic symptoms (such as shortness of breath, fatigue, chest pain) and psychological (depression, anxiety or mild cognitive impairment) symptoms can last for months after infection [@doi:10.1038/s41591-021-01283-z].
Such long-term affects occur both in adults [@doi:10.1007/s11606-021-07032-9] and children [@doi:10.1111/apa.15870].
Sustained symptoms affecting a variety of biological systems have been reported across many studies (e.g., [@doi:10.1038/s41591-021-01283-z; @doi:10.1016/j.eclinm.2021.101019; @doi:10/ghstsk]).
The phenomenon of "long COVID" is not fully understood although various possible explanations have been proposed, including damage caused by immune response to infection as well as by the infection itself, in addition to negative consequences of the experience of lengthy illness and hospitalization.
However, a lack of consistency among definitions used in different studies makes it difficult to develop precise definitions or identify specific symptoms associated with long-term effects of COVID-19 [@doi:10.1101/2021.03.20.21253896; @doi:10.1101/2021.06.23.21259416].
Patient and family support groups for "long haulers" have been formed online, and patient-driven efforts to collect data about post-acute COVID-19 provide valuable sources of information (e.g., [@doi:10.1016/j.eclinm.2021.101019]).
The specific relationship between viral pathogenesis and these reported sequelae remains to be uncovered, however.

#### Pediatric Presentation

The presentation of COVID-19 infection can vary greatly among pediatric patients and, in some cases, manifests in distinct ways from COVID-19 in adults.
Evidence suggests that children and adolescents tend to have mostly asymptomatic infections and that those who are symptomatic typically exhibit mild illness [@doi:10.1101/2020.06.25.20140178; @doi:10.1056/NEJMc2005073; @doi:10.1111/apa.15270; @doi:10.1126/science.abe5765].
One review examined symptoms reported in 17 studies of children infected with COVID-19 during the early months of the COVID-19 epidemic in China and one study from Singapore [@doi:10.1001/jamapediatrics.2020.1467].
In the more than a thousand cases described, the most common reports were for mild symptoms such as fever, dry cough, fatigue, nasal congestion and/or runny nose, while three children were reported to be asymptomatic.
Severe lower respiratory infection was described in only one of the pediatric cases reviewed.
Gastrointestinal symptoms such as vomiting or diarrhea were occasionally reported.
Radiologic findings were not always reported in the case studies reviewed, but when they were mentioned they included bronchial thickening, ground-glass opacities, and/or inflammatory lesions [@doi:10.1001/jamapediatrics.2020.1467].
Neurological symptoms have also been reported [@doi:10.1001/jamaneurol.2020.2687].

These analyses indicate that most pediatric cases of COVID-19 are not severe.
Indeed, it is estimated that less than 1% of pediatric cases result in critical illness [@doi:10.1056/NEJMc2007617; @doi:10.1111/apa.15270], although reporting suggests that pediatric hospitalizations may be greater with the emergence of the Delta variant of concern (VOC) [@url:https://covid.cdc.gov/covid-data-tracker; @doi:10.1136/bmj.n1513; @url:https://www.nytimes.com/2021/08/09/health/coronavirus-children-delta.html].
Serious complications and, in relatively rare cases, deaths have occurred [@doi:10.1016/j.eclinm.2020.100433].
Of particular interest, children have occasionally experienced a serious inflammatory syndrome, multisystem inflammatory syndrome in children (MIS-C), following COVID-19 infection [@doi:10.1111/apa.16051].
This syndrome is similar in some respects to Kawasaki disease, including Kawasaki disease shock syndrome [@doi:10.1093/jpids/piaa069; @doi:10.1001/jama.2020.10369; @doi:10.1016/j.ajem.2020.05.117], and is thought to be a distinct clinical manifestation of SARS-CoV-2 due to its distinct cytokine profile and the presence of burr cells in peripheral blood smears [@doi:10.1172/JCI140970; @doi:10.1016/j.cell.2020.09.016].
MIS-C has been associated with heart failure in some cases [@doi:10.1161/CIRCULATIONAHA.120.048360].
A small number of case studies have identified presentations similar to MIS-C in adults associated with SARS-CoV-2 [@doi:10/gg4sd6; @doi:10.1136/bcr-2021-243888; @doi:10.1016/j.ajem.2020.06.053; @doi:10.3389/fimmu.2021.680567].
However, not all cases of severe COVID-19 in children are characterizable as MIS-C.
A recent study [@doi:10.1001/jama.2021.2091] described demographic and clinical variables associated with MIS-C in comparison with non-MIS-C severe acute COVID-19 in young people in the United States.
Efforts to characterize long-term sequelae of SARS-CoV-2 infection in children face the same challenges as in adults, but long-term effects remain a concern in pediatric patients [@doi:10.1101/2021.01.23.21250375; @doi:10.1111/apa.15870; @doi:10.1002/ppul.25521], although some early studies have suggested that they may be less of a concern than in adults [@doi:10/gj9p7b; @doi:10.1101/2021.05.16.21257255; @doi:10.1001/jama.2021.11880].
Research is ongoing into the differences between the pediatric and adult immune responses to SARS-CoV-2, and future research may shed light on the factors that lead to MIS-C; it is also unknown whether the relative advantages of children against severe COVID-19 will remain in the face of current and future variants [@doi:10.1038/d41586-021-02423-8].

#### Cytokine Release Syndrome

The inflammatory response was identified early on as a potential driver of COVID-19 outcomes due to existing research in SARS and emerging research in COVID-19.
While too low of an inflammatory response is a concern because it will fail to eliminate the immune threat [@doi:10.1111/imr.12671], excessive pro-inflammatory cytokine activity can cascade [@doi:10.1159/000087104] and cause cell damage, among other problems [@doi:10.18632/oncotarget.23208].
A dysregulated immune response can cause significant damage to the host [@isbn:9780815332183; @isbn:9780071283663; @doi:10.1016/j.chom.2020.05.009] including pathogenesis associated with sepsis.
Sepsis, which can lead to multi-organ failure and death [@doi:10.1128/MMBR.05015-11; @pmid:22142805], is traditionally associated with bacterial infections.
However, sepsis associated with viral infections may be underidentified [@doi:10.1183/16000617.0038-2020], and sepsis has emerged as a major concern associated with SARS-CoV-2 infection [@doi:10/ggsps7].
Hyperactivity of the pro-inflammatory response due to lung infection is commonly associated with acute lung injury and more rarely with the more severe manifestation, ARDS, which can arise from pneumonia, SARS, and COVID-19 [@doi:10.1159/000087104; @doi:10.1128/MMBR.05015-11].
Damage to the capillary endothelium can cause leaks that disrupt the balance between pro-inflammatory cytokines and their regulators [@doi:10.1164/ajrccm.164.10.2104013], and heightened inflammation in the lungs can also serve as a source for systemic inflammation, or sepsis, and potentially multi-organ failure [@doi:10.1128/MMBR.05015-11].
The shift from local to systemic inflammation is a phenomenon often referred to broadly as a cytokine storm [@doi:10.1128/MMBR.05015-11] or, more precisely, as cytokine release syndrome [@doi:10.1186/s40425-018-0343-9].

Cytokine dysregulation is therefore a significant concern in the context of COVID-19.
In addition to the known role of cytokines in ARDS and lung infection more broadly, immunohistological analysis at autopsy of deceased SARS patients revealed that ACE2-expressing cells that were infected by SARS-CoV-1 showed elevated expression of the cytokines IL-6, IL-1&beta;, and TNF-&alpha; [@doi:10.1002/path.2067].
Similarly, the introduction of the S protein from SARS-CoV-1 to mouse macrophages was found to increase production of IL-6 and TNF-&alpha; [@doi:10.1016/j.virusres.2007.02.007].
For SARS-CoV-2 infection leading to COVID-19, early reports described a cytokine storm syndrome-like response in patients with particularly severe infections [@doi:10.1002/jmv.26139; @doi:10/ggnzmc; @doi:10.1016/j.immuni.2020.06.017].
Sepsis has been identified as a major contributor to COVID-19-related death.
Among patients hospitalized with COVID-19 in Wuhan, China, 112 out of 191 (59%) developed sepsis, including all 54 of the non-survivors [@doi:10/ggnxb3].

While IL-6 is sometimes used as a biomarker for cytokine storm activity in sepsis [@doi:10.1128/MMBR.05015-11], the relationship between cytokine profiles and the risks associated with sepsis may be more complex.
One study of patients with and at risk for ARDS, specifically those who were intubated for medical ventilation, found that shortly after the onset of ARDS, anti-inflammatory cytokine concentration in BALF increased relative to the concentration of pro-inflammatory cytokines [@doi:10.1164/ajrccm.164.10.2104013].
The results suggest that an increase in pro-inflammatory cytokines such as IL-6 may signal the onset of ARDS, but recovery depends on an increased anti-inflammatory response [@doi:10.1164/ajrccm.164.10.2104013].
However, patients with severe ARDS were excluded from this study.
Another analysis of over 1,400 pneumonia patients in the United States reported that IL-6, tumor necrosis factor (TNF), and IL-10 were elevated at intake in patients who developed severe sepsis and/or ultimately died [@doi:10.1001/archinte.167.15.1655].
However, unlike the study analyzing pro- and anti-inflammatory cytokines in ARDS patients [@doi:10.1164/ajrccm.164.10.2104013], this study reported that unbalanced pro-/anti-inflammatory cytokine profiles were rare.
This discrepancy could be related to the fact that the sepsis study measured only three cytokines.
Although IL-6 has traditionally been considered pro-inflammatory, its pleiotropic effects via both classical and trans-signaling allow it to play an integral role in both the inflammatory and anti-inflammatory responses [@doi:10.1016/j.bbamcr.2011.01.034], leading it to be associated with both healthy and pathological responses to viral threat [@doi:10.3389/fmicb.2019.01057].
While the cytokine levels observed in COVID-19 patients fall outside of the normal range, they are not as high as typically found in patients with ARDS [@doi:10.1001/jamainternmed.2020.3313].
Regardless of variation in the anti-inflammatory response, prior work has therefore made it clear that pulmonary infection and injury are associated with systemic inflammation and with sepsis.
Inflammation has received significant interest both in regards to the pathology of COVID-19 as well as potential avenues for treatment, as the relationship between the cytokine storm and the pathophysiology of COVID-19 has led to the suggestion that a number of immunomodulatory pharmaceutical interventions could hold therapeutic value for the treatment of COVID-19 [@doi:10.1016/j.jaut.2020.102452; @individual-pharmaceuticals].

### Insights from Systems Biology

Systems biology provides a cross-disciplinary analytical paradigm through which the host response to an infection can be analyzed.
This field integrates the "omics" fields (genomics, transcriptomics, proteomics, metabolomics, etc.) using bioinformatics and other computational approaches.
Over the last decade, systems biology approaches have been used widely to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].
Omics-based studies have also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].
Though the complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not yet fully understood, omics technologies offer the opportunity for discovery-driven analysis of biological changes associated with SARS-CoV-2 infection.

#### Transcriptomics

Through transcriptomic analysis, the effect of a viral infection on gene expression can be assessed.
Transcriptomic analyses, whether _in vivo_ or _in situ_, can potentially reveal insights into viral pathogenesis by elucidating the host response to the virus.
For example, infection by some viruses, including by the coronaviruses SARS-CoV-2, SARS-CoV-1, and MERS-CoV, is associated with the upregulation of ACE2 in human embryonic kidney cells and human airway epithelial cells [@doi:10.1002/jmv.26139].
This finding suggests that SARS-CoV-2 facilitates the positive regulation of its own transmission between host cells [@doi:10.1002/jmv.26139].
The host immune response also likely plays a key role in mediating infection-associated pathologies.
Therefore, transcriptomics is one critical tool for characterizing the host response in order to gain insight into viral pathogenesis.
For this reason, the application of omics technologies to the process of characterizing the host response is expected to provide novel insights into how hosts respond to SARS-CoV-2 infection and how these changes might influence COVID-19 outcomes.

Several studies have examined the cellular response to SARS-CoV-2 _in vitro_ in comparison to other viruses.
One study [@doi:10.1016/j.cell.2020.04.026] compared the transcriptional responses of three human cell lines to SARS-CoV-2 and to other respiratory viruses, including MERS-CoV, SARS-CoV-1, *Human parainfluenza virus 3*, *Respiratory syncytial virus*, and *Influenza A virus*.
The transcriptional response differed between the SARS-CoV-1 infected cells and the cells infected by other viruses, with changes in differential expression specific to each infection type.
Where SARS-CoV-2 was able to replicate efficiently, differential expression analysis revealed that the transcriptional response was significantly different from the response to all of the other viruses tested.
A unique pro-inflammatory cytokine signature associated with SARS-CoV-2 was present in cells exposed to both high and low doses of the virus, with the cytokines IL-6 and IL1RA uniquely elevated in response to SARS-CoV-2 relative to other viruses.
However, one cell line showed significant IFN-I or IFN-III expression when exposed to high, but not low, doses of SARS-CoV-2, suggesting that IFN induction is dependent on the extent of exposure.
These results suggest that SARS-CoV-2 induces a limited antiviral state with low IFN-I or IFN-III expression and a moderate IFN-stimulated gene response, in contrast to other viruses.
Other respiratory viruses have been found to encode antagonists to the IFN response [@doi:10.1016/j.mib.2010.05.009; @doi:10.1371/journal.pone.0112014], including SARS-CoV-1 [@doi:10.1128/JVI.01782-06] and MERS-CoV [@doi:10.1128/JVI.01845-13].

The analysis of SARS-CoV-2 suggested that this transcriptional state was specific to cells expressing ACE2, as it was not observed in cells lacking expression of this protein except with ACE2 supplementation and at very high (10-fold increase) level of SARS-CoV-2 exposure [@doi:10.1016/j.cell.2020.04.026].
In another study, direct stimulation with inflammatory cytokines such as type I interferons (e.g., IFN&beta;) was also associated with the upregulation of ACE2 in human bronchial epithelial cells, with treated groups showing four-fold higher ACE2 expression than control groups at 18 hours post-treatment [@doi:10.1101/2020.02.24.963348].
This hypothesis was further supported by studies showing that several nsp in SARS-CoV-2 suppress interferon activity [@doi:10.1080/22221751.2020.1780953] and that the SARS-CoV-2 *ORF3b* gene suppresses IFNB1 promoter activity (IFN-I induction) more efficiently than the SARS-CoV-1 *ORF3b* gene [@doi:10.1016/j.celrep.2020.108185].
Taken together, these findings suggest that a unique cytokine profile is associated with the response to the SARS-CoV-2 virus, and that this response differs depending on the magnitude of exposure.

Susceptibility and IFN induction may also vary by cell type.
Using poly(A) bulk RNA-seq to analyzed dynamic transcriptional responses to SARS-CoV-2 and SARS-CoV-1 revealed negligible susceptibility of cells from the H1299 line (< 0.08 viral read percentage of total reads) compared to those from the Caco-2 and Calu-3 lines (>10% of viral reads) [@doi:10.1101/2020.05.05.079194].
This finding suggests that the risk of infection varies among cell types, and that cell type could influence which hosts are more or less susceptible.
Based on visual inspection of microscopy images alongside transcriptional profiling, the authors also showed distinct responses among the host cell lines evaluated [@doi:10.1101/2020.05.05.079194].
In contrast to Caco-2, Calu-3 cells infected with SARS-CoV-2 showed signs of impaired growth and cell death at 24 hours post infection, as well as moderate IFN induction with a strong up-regulation of IFN-stimulated genes.
Interestingly, the results were similar to those reported in Calu-3 cells exposed to much higher levels of SARS-CoV-2 [@doi:10.1016/j.cell.2020.04.026], as described above.
This finding suggests that IFN induction in Calu-3 cells is not dependent on the level of exposure, in contrast to A549-ACE2 cells.
The discrepancy could be explained by the observations that Calu-3 cells are highly susceptible to SARS-CoV-2 and show rapid viral replication [@doi:10.1038/s41467-020-15562-9], whereas A549 cells are incompatible with SARS-CoV-2 infection [@doi:10.1101/2020.03.02.972935].
This discrepancy raises the concern that _in vitro_ models may vary in their similarity to the human response, underscoring the importance of follow-up studies in additional models.

As a result, transcriptional analysis of patient tissue is an important application of omics technology to understanding COVID-19.
Several studies have collected blood samples from COVID-19 patients and analyzed them using RNA-Seq [@doi:10.1186/s13073-020-00823-5; @doi:10.1016/j.immuni.2020.11.017; @doi:10.1080/22221751.2020.1747363; @doi:10.1016/j.cell.2021.02.018; @doi:10.3389/fimmu.2021.694243; @doi:10.1126/science.abc6261].
Analyzing gene expression in the blood is valuable to understanding host-pathogen interactions because of the potential to identify alterations associated with the immune response and to gain insights into inflammation, among other potential insights [@doi:10.1186/s13073-020-00823-5].
One study compared gene expression in 39 COVID-19 inpatients admitted with community-acquired pneumonia to that of control donors using whole blood cell transcriptomes [@doi:10.1186/s13073-020-00823-5].
They also evaluated the effect of mild versus severe disease.
A greater number of differentially expressed genes were found in severe patients compared to controls than in mild patients compared to controls.
They also identified that the transcriptional profiles clustered into five groups and that the groups could not be explained by disease severity.
Most severe cases fell into two clusters associated with increased inflammation and granulocyte and neutrophil activation.
The presence of these clusters suggests the possibility that personalized medicine could be useful in the treatment of COVID-19 [@doi:10.1186/s13073-020-00823-5].
Longitudinal analysis of granulocytes from patients with mild versus severe COVID-19 revealed that granulocyte activation-associated factors differentiated the disease states, with greater numbers of differentially expressed genes early in disease course [@doi:10.1186/s13073-020-00823-5].
This study therefore revealed distinct patterns associated with COVID-19 and identified genes and pathways associated with each cluster.

Many other studies have also identified transcriptomic signatures associated with the immune response and inflammation.
Other studies have profiled the transcriptome of BALF [@doi:10.1080/22221751.2020.1747363] and the nasopharynx [@doi:10.1084/jem.20210583].
One study used single-cell transcriptomics techniques to investigate cell types including brain and choroid plexus cells compared to healthy controls and controls with influenza; among other signals of neuroinflammation, this study reported cortical T cells only in COVID-19 patients [@doi:10.1038/s41586-021-03710-0].
Transcriptomic analysis can thus provide insight into the pathogenesis of SARS-CoV-2 and may also be useful in identifying candidate therapeutics [@doi:10.1186/s13073-020-00823-5].

#### Proteomics

Proteomics analysis offers an opportunity to characterize the response to a pathogen at a level above transcriptomics.
Especially early on, this primarily involved evaluating the effect of the virus on cell lines.
One early proteomics study investigated changes associated with _in vitro_ SARS-CoV-2 infection using Caco-2 cells [@doi:10.1038/s41586-020-2332-7].
This study reported that SARS-CoV-2 induced alterations in multiple vital physiological pathways, including translation, splicing, carbon metabolism and nucleic acid metabolism in the host cells.
Another area of interest is whether SARS-CoV-2 is likely to induce similar changes to other HCoV.
For example, because of the high level of sequence homology between SARS-CoV-2 and SARS-CoV-1, it has been hypothesized that sera from convalescent SARS-CoV-1 patients might show some efficacy in cross-neutralizing SARS-CoV-2-S-driven entry [@doi:10.1016/j.cell.2020.02.052].
However, despite the high level of sequence homology, certain protein structures might be immunologically distinct, which would be likely to prohibit effective cross-neutralization across different SARS species [@doi:10.1101/2020.03.21.990770].
Consequently, proteomic analyses of SARS-CoV-1 might also provide some essential information regarding the new pathogen [@doi:10.1073/pnas.0407992101; @doi:10.1016/j.bbrc.2004.02.074].

Proteomics research has been able to get ahead of the timeline for development of omics-level big data sets specific to SARS-CoV-2 by adopting a comparative bioinformatics approach.
Data hubs such as UniProt [@doi:10.1093/nar/gky1049], NCBI Genome Database [@url:https://www.ncbi.nlm.nih.gov/genome], The Immune Epitope Database and Analysis Resource [@doi:10.1093/nar/gky1006], and The Virus Pathogen Resource [@doi:10.1093/nar/gkr859] contain a wealth of data from studies in other viruses and even HCoV.
Such databases facilitate the systems-level reconstruction of protein-protein interaction networks, providing opportunities to generate hypotheses about the mechanism of action of SARS-CoV-2 and identify potential drug targets.
In an initial study [@doi:10.1038/s41586-020-2286-9], 26 of the 29 SARS-CoV-2 proteins were cloned and expressed in HEK293T kidney cells, allowing for the identification of 332 high-confidence human proteins interacting with them.
Notably, this study suggested that SARS-CoV-2 interacts with innate immunity pathways.
Ranking pathogens by the similarity between their interactomes and that of SARS-CoV-2 suggested *West Nile virus*, *Mycobacterium tuberculosis*, and *human papillomavirus* infections as the top three hits.
The fact that the host-pathogen interactome of the bacterium *Mycobacterium tuberculosis* was found to be similar to that of SARS-CoV-2 suggests that changes related to lung pathology might comprise a significant contributor to these expression profiles.
Additionally, it was suggested that the envelope protein, E, could disrupt host bromodomain-containing proteins, i.e., BRD2 and BRD4, that bind to histones, and the spike protein could likely intervene in viral fusion by modulating the GOLGA7-ZDHHC5 acyl-transferase complex to increase palmitoylation, which is a post-translational modification that affects how proteins interact with membranes [@doi:10.3389/fimmu.2017.02003].

An example of an application of this _in silico_ approach comes from another study [@doi:10.1101/2020.03.31.019216], which used patient-derived peripheral blood mononuclear cells to identify 251 host proteins targeted by SARS-CoV-2.
This study also reported that more than 200 host proteins were disrupted following infection.
In particular, a network analysis showed that nsp9 and nsp10 interacted with NF-Kappa-B-Repressing Factor, which encodes a transcriptional repressor that mediates repression of genes responsive to Nuclear Factor kappa-light-chain-enhancer of activated B-cells.
These genes are important to pro-, and potentially also anti-, inflammatory signaling [@doi:10.1101/cshperspect.a001651].
This finding could explain the exacerbation of the immune response that shapes the pathology and the high cytokine levels characteristic of COVID-19, possibly due to the chemotaxis of neutrophils mediated by IL-8 and IL-6.
Finally, it was suggested [@doi:10.1101/2020.04.09.033522] that the E protein of both SARS-CoV-1 and SARS-CoV-2 has a conserved Bcl-2 Homology 3-like motif, which could inhibit anti-apoptosis proteins, e.g., BCL2, and trigger the apoptosis of T cells.
Several compounds are known to disrupt the host-pathogen protein interactome, largely through the inhibition of host proteins.
Therefore, this research identifies candidate targets for intervention and suggests that drugs modulating protein-level interactions between virus and host could be relevant to treating COVID-19.

As with other approaches, analyzing the patterns found in infected versus healthy human subjects is also important.
COVID-19 infection has been associated with quantitative changes in transcripts, proteins, metabolites, and lipids in patient blood samples [@doi:10.1016/j.cels.2020.10.003].
One longitudinal study [@doi:10.15252/emmm.202114167] compared COVID-19 patients to symptomatic controls who were PCR-negative for SARS-CoV-2.
The longitudinal nature of this study allowed it to account for differences in the scale of inter- versus intraindividual changes.
At the time of first sampling, common functions of proteins upregulated in COVID-19 patients relative to controls were related to immune system mediation, coagulation, lipid homeostasis, and protease inhibition.
They compared these data to the patient-specific timepoints associated with the highest levels of SARS-CoV-2 antibodies and found that the actin-binding protein gelsolin, which is involved in recovery from disease, showed the steepest decline between those two time points.
Immunoglobulins comprised the only proteins that were significantly different between the COVID-19 and control patients at both of these timepoints.
The most significantly downregulated proteins between these time points were related to inflammation, while the most significantly upregulated proteins were immunoglobulins.
Proteins related to coagulation also increased between the two timepoints.
The selection of a symptomatic control cohort rather than healthy comparisons also suggests that the results are more likely to highlight the response to SARS-CoV-2 and COVID-19 specifically, rather than to disease more broadly.
This study also compared the disease course in patients who ultimately survived to those who died and found that ITIH4, a protein associated with the inflammatory response to trauma, may be a biomarker useful to identifying patients at risk of death.
Thus, these results indicate the value of studying patients in a longitudinal manner over the disease course.
By revealing which genes are perturbed during SARS-CoV-2 infection, proteomics-based analyses can thus provide novel insights into host-virus interaction and serve to generate new avenues of investigation for therapeutics.

### Viral Virulence

Like that of SARS-CoV-1, the entry of SARS-CoV-2 into host cells is mediated by interactions between the viral spike glycoprotein, S, and human ACE2 (hACE2) [@doi:10.1126/science.1116480; @doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2762; @doi:10.1038/s41586-020-2179-y; @doi:10.1101/2020.02.19.956235; @doi:10.1016/j.cell.2020.02.052; @doi:10.1016/j.cell.2020.03.045; @doi:10.1128/jvi.00127-20].
Differences in how the S proteins of the two viruses interact with hACE2 could partially account for the increased transmissibility of SARS-CoV-2.
Studies have reported conflicting binding constants for the S-hACE2 interaction, though they have agreed that the SARS-CoV-2 S protein binds with equal, if not greater, affinity than the SARS-CoV-1 S protein does [@doi:10.1016/j.cell.2020.02.058; @doi:10.1101/2020.02.19.956235; @doi:10.1126/science.abb2507].
The C-terminal domain of the SARS-CoV-2 S protein in particular was identified as the key region of the virus that interacts with hACE2, and the crystal structure of the C-terminal domain of the SARS-CoV-2 S protein in complex with hACE2 reveals stronger interaction and a higher affinity for receptor binding than that of SARS-CoV-1 [@doi:10.1016/j.cell.2020.03.045].
Among the 14 key binding residues identified in the SARS-CoV-1 S protein, eight are conserved in SARS-CoV-2, and the remaining six are semi-conservatively substituted, potentially explaining variation in binding affinity [@doi:10.1016/j.cell.2020.02.058; @doi:10.1101/2020.02.19.956235].
Studies of crystal structure have shown that the RBD of the SARS-CoV-2 S protein, like that of other coronaviruses, undergoes stochastic hinge-like movement that flips it from a “closed” conformation, in which key binding residues are hidden at the interface between protomers, to an “open” one [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507].
Spike proteins cleaved at the furin-like binding site are substantially more likely to take an open conformation (66%) than those that are uncleaved (17%) [@doi:10.1038/s41594-020-0468-7].
Because the RBD plays such a critical role in viral entry, blocking its interaction with ACE2 could represent a promising therapeutic approach.
Nevertheless, despite the high structural homology between the SARS-CoV-2 RBD and that of SARS-CoV-1, monoclonal antibodies targeting SARS-CoV-1 RBD failed to bind to SARS-CoV-2-RBD [@doi:10.1126/science.abb2507].
However, in early research, sera from convalescent SARS patients were found to inhibit SARS-CoV-2 viral entry _in vitro_, albeit with lower efficiency than it inhibited SARS-CoV-1 [@doi:10.1016/j.cell.2020.02.052].

Comparative genomic analysis reveals that several regions of the coronavirus genome are likely critical to virulence.
The S1 domain of the spike protein, which contains the receptor binding motif, evolves more rapidly than the S2 domain [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009].
However, even within the S1 domain, some regions are more conserved than others, with the receptors in S1's N-terminal domain (S1-NTD) evolving more rapidly than those in its C-terminal domain (S1-CTD) [@doi:10.1016/j.jmb.2020.04.009].
Both S1-NTD and S1-CTD are involved in receptor binding and can function as RBDs to bind proteins and sugars [@doi:10.3390/v4061011], but RBDs in the S1-NTD typically bind to sugars, while those in the S1-CTD recognize protein receptors [@doi:10.1146/annurev-virology-110615-042301].
Viral receptors show higher affinity with protein receptors than sugar receptors [@doi:10.1146/annurev-virology-110615-042301], which suggests that positive selection on or relaxed conservation of the S1-NTD might reduce the risk of a deleterious mutation that would prevent binding.
The SARS-CoV-2 S protein also contains an RRAR furin recognition site at the S1/S2 junction [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507], setting it apart from both bat coronavirus RaTG13, with which it shares 96% genome sequence identity, and SARS-CoV-1 [@doi:10.1038/s41586-020-2012-7].
Such furin cleavage sites are commonly found in highly virulent influenza viruses [@doi:10/bvgh5b; @doi:10.1006/viro.1999.9716].
The furin recognition site at the S1/S2 junction is likely to increase pathogenicity via destabilization of the spike protein during fusion to ACE2 and the facilitation of cell-cell adhesion [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507; @doi:10.1038/s41594-020-0468-7; @doi:10/bvgh5b; @doi:10.1006/viro.1999.9716; @doi:10.1016/j.isci.2020.101212].
These factors may influence the virulence of SARS-CoV-2 relative to other beta coronaviruses.
Additionally, a major concern has been the emergence of SARS-CoV-2 variants with increased virulence.
The extent to which evolution within SARS-CoV-2 may affect pathogenesis is reviewed below.

### Molecular Signatures, Transmission, and Variants of Concern

Genetic variation in SARS-CoV-2 has been used to elucidate patterns over time and space.
Many mutations are neutral in their effect and can be used to trace transmission patterns.
Such signatures within SARS-CoV-2 have provided insights during outbreak investigations [@doi:10.1186/s12967-020-02344-6; @doi:10.1093/nsr/nwaa036; @doi:10.1016/j.meegid.2020.104351].
Similar mutations observed in several patients may indicate that the patients belong to the same transmission group.
The tracking of SARS-CoV-2 mutations is recognized as an essential tool for controlling future outbreaks and tracing the path of the spread of SARS-CoV-2.
In the first months of the pandemic in early 2020, early genomic surveillance efforts in Guangdong, China revealed that local transmission rates were low and that most cases arising in the province were imported [@doi:10.1016/j.cell.2020.04.023].
Since then, efforts have varied widely among countries: for example, the U.K. has coordinated a national database of viral genomes [@doi:10/d5mg], but efforts to collect this type of data in the United States have been more limited [@url:https://www.cdc.gov/coronavirus/2019-ncov/variants/spheres.html].
Studies have applied phylogenetic analyses of viral genomes to determine the source of local COVID-19 outbreaks in Connecticut (USA), [@doi:10.1016/j.cell.2020.04.021], the New York City area (USA) [@doi:10.1126/science.abc1917], and Iceland [@doi:10.1056/NEJMoa2006100].
There has been an ongoing effort to collect SARS-CoV-2 genomes throughout the COVID-19 outbreak, and as of summer 2021, millions of genome sequences have been collected from patients.
The sequencing data can be found at GISAID [@https://www.gisaid.org], NCBI [@https://www.ncbi.nlm.nih.gov/sars-cov-2], and COVID-19 data portal [@https://www.covid19dataportal.org].

Ongoing evolution can be observed in genomic data collected through molecular surveillance efforts.
In some cases, mutations can produce functional changes that can impact pathogenesis.
One early example is the spike protein mutation D614G, which appeared in March 2020 and became dominant worldwide by the end of May 2020 [@doi:10.1016/j.cell.2020.06.043; @doi:10.1016/j.cell.2020.09.032].
This variant was associated with increased infectivity and increased viral load, but not with more severe disease outcomes [@doi:10.1016/j.cell.2020.06.043; @doi:10.1038/s41586-020-2895-3].
This increased virulence is likely achieved by altering the conformation of the S1 domain to facilitate binding to ACE2 [@doi:10.1038/s41586-020-2895-3].
Similarly, the N439K mutation within the RBD of the spike protein is likely associated with increased transmissibility and enhanced binding affinity for hACE2, although it is also not thought to affect disease outcomes [@doi:10.1016/j.cell.2021.01.037].
In contrast, a mutation in ORF8 that was identified in Singapore in the early months of 2020 was associated with cases of COVID-19 that were less likely to require treatment with supplemental oxygen [@doi:10/d6x7], and a deletion surrounding the furin site insertion at the S1/S2 boundary has been identified only rarely in clinical settings [@doi:10.1128/JVI.00790-20], suggesting that these mutations may disadvantage viral pathogenesis in human hosts.
Thus, mutations have been associated with both virological and clinical differences in pathogenesis.

Several VOCs have also been identified and designated through molecular surveillance efforts [@doi:10.1038/s41576-021-00408-x].
The Alpha variant (lineage B.1.1.7) was first observed in the U.K. in October 2020 before it quickly spread around the world [@doi:10.3390/v13030392].
Other variants meriting further investigation have also been identified, including the Beta variant (B.1.351 lineage) first identified in South Africa and the Gamma variant (P.1 lineage) initially associated with outbreaks in Brazil.
These lineages share independently acquired mutations that may affect pathogenicity [@doi:10.1101/2020.12.14.422555; @url:https://cov-lineages.org/global_report_B.1.351.html; @doi:10/gsfg; @doi:10.1016/j.csbj.2021.06.037; @doi:10.1101/2020.12.30.20249034].
For example, they are all associated with a greater binding affinity for hACE2 than that of the wildtype variant [@doi:10/gsfg; @doi:10.1016/j.jmb.2021.167058; @doi:10.1016/j.medidd.2021.100086], but they were not found to have more efficient cell entry than the wildtype virus [@doi:10.1016/j.cell.2021.03.036].
A fourth VOC, the Delta variant (B.1.617.2 and AY.1, AY.2, and AY.3 lineages), was identified in India in late 2020 [@url:https://www.who.int/en/activities/tracking-SARS-CoV-2-variants].
Some of the mutations associated with this lineage may alter fusogenicity and enhance furin cleavage, among other effects associated with increased pathogenicity [@doi:10.1101/2021.06.17.448820].
The changes in these VOC demonstrate how ongoing evolution in SARS-CoV-2 can drive changes in how the virus interacts with host cells.

### Quantifying Viral Presence

Assessing whether a virus is present in a sample is a more complex task than it initially seems.
Many diagnostic tests rely on real-time polymerase chain reaction (RT-PCR) to test for the presence versus absence of a virus [@individual-diagnostics].
They may report the cycle threshold (C~t~) indicating the number of doubling cycles required for the target (in this case, SARS-CoV-2) to become detectable.
A lower C~t~ therefore corresponds to a higher viral load.
The C~t~ that corresponds to a positive can vary widely, but is often around 35.
This information is sufficient to answer many questions, since an amplicon must be present in order to be duplicated in RT-PCR.
For example, if a patient is presenting with COVID-19 symptoms, a positive RT-PCR test can confirm the diagnosis.

However, RT-PCR analysis alone cannot provide the information needed to determine whether a virus is present at sufficient levels to be infectious [@doi:10.2807/1560-7917.ES.2020.25.32.2001483].
Some studies have therefore taken the additional step of cultivating samples _in vitro_ in order to observe whether cells become infected with SARS-CoV-2.
One study collected upper respiratory tract samples from COVID-19 patients, analyzed them with RT-PCR to determine the cycle threshold, and then attempted to cultivate the SARS-CoV-2 virus in VeroE6 cells [@doi:10.2807/1560-7917.ES.2020.25.32.2001483].
This study found that out of 246 samples, less than half (103) produced a positive culture.
Moreover, at a C~t~ of 35, only 5 out of 60 samples grew _in vitro_.
Therefore, the RT-PCR-confirmed presence of SARS-CoV-2 in a sample does not necessarily indicate that the virus is present at a high enough concentration to grow and/or spread. 

### Mechanisms of Transmission

When a human host is infected with a virus and is contagious, person-to-person viral transmission can occur through several possible mechanisms.
When a contagious individual sneezes, coughs, or exhales, they produce respiratory droplets that can contain a large number of viral particles [@pubmed:23771256].
Viral particles can enter the body of a new host when they then come in contact with the oral, nasal, eye, or other mucus membranes [@pubmed:23771256].
The primary terms typically used to discuss the transmission of viruses via respiratory droplets are droplet, aerosol, and contact transmission [@doi:10.1097/QCO.0000000000000563].
The distinction between droplet and aerosol transmission is typically anchored on whether a particle containing the virus is larger or smaller than 5 micrometers (&mu;m) [@doi:10.1016/j.jhin.2020.12.022; @{url:https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3829873}].
Droplet transmission typically refers to contact with large droplets that fall quickly to the ground at close range, such as breathing in droplets produced by a sneeze [@pubmed:23771256; @doi:10.1016/j.jhin.2020.12.022].
Aerosol transmission typically refers to much smaller particles (less than 5 &mu;m) produced by sneezing, coughing, or exhaling [@doi:10.1097/QCO.0000000000000563; @pubmed:23771256] that can remain suspended over a longer period of time and potentially to be moved by air currents [@pubmed:23771256].
It is also possible that viral particles deposited on surfaces via large respiratory droplets could later be aerosolized [@pubmed:23771256].
The transmission of viral particles that have settled on a surface is typically referred to as contact or fomite transmission [@url:https://www.who.int/news-room/commentaries/detail/transmission-of-sars-cov-2-implications-for-infection-prevention-precautions; @pubmed:23771256].
Any respiratory droplets that settle on a surface could contribute to fomite transmission [@pubmed:23771256].
Droplet and contact transmission are both well-accepted modes of transmission for many viruses associated with common human illnesses, including influenza and rhinovirus [@pubmed:23771256].

The extent to which aerosol transmission contributes to the spread of respiratory viruses is more widely debated.
In influenza A, for example, viral particles can be detected in aerosols produced by infected individuals, but it is not clear to what extent these particles drive the spread of influenza A infection [@pubmed:23771256; @doi:10.1097/QCO.0000000000000563; @doi:10.3201/eid1301.061202; @doi:10.1038/s41598-019-38825-y; @doi:10.1093/femsre/fuv039].
Regardless of its role in the spread of influenza A, however, aerosol transmission likely played a role in outbreaks such as the 1918 Spanish Influenza (H1N1) and 2009 "swine flu" (pH1N1) [@doi:10.1093/femsre/fuv039].
All three of these mechanisms have been identified as possible contributors to the transmission of HCoVs [@pubmed:23771256], including the highly pathogenic coronaviruses SARS-CoV-1 and MERS-CoV [@doi:10.1016/B978-0-12-385885-6.00009-2; @doi:10.1038/nrmicro.2016.81].
Transmission of SARS-CoV-1 is thought to proceed primarily through droplet transmission, but aerosol transmission is also considered possible [@pubmed:23771256; @doi:10.1111/j.1600-0668.2004.00317.x; @doi:10.1086/429634], and fomite transmission may have also played an important role in some outbreaks [@doi:10.1371/journal.pone.0181558].
Similarly, the primary mechanism of MERS transmission is thought to be droplets because inter-individual transmission appears to be associated with close interpersonal contact (e.g., household or healthcare settings), but aerosolized particles of the MERS virus have been reported to persist much more robustly than influenza A under a range of environmental conditions [@doi:10.2807/1560-7917.es2013.18.38.20590; @doi:10.1186/s12985-015-0439-5].
However, few of these analyses have sought to grow positive samples in culture and thus to confirm their potential to infect new hosts.

Contact, droplet, and aerosol transmission are therefore all worth evaluating when considering possible modes of transmission for a respiratory virus like SARS-CoV-2.
The stability of the SARS-CoV-2 virus both in aerosols and on a variety of surfaces was found to be similar to that of SARS-CoV-1 [@doi:10.1056/NEJMc2004973].
Droplet-based and contact transmission were initially put forward as the greatest concern for the spread of SARS-CoV-2 [@doi:10.1038/s41368-020-0075-9], with droplet transmission considered the dominant mechanism driving the spread of the virus [@doi:10.1001/jama.2020.12458] because the risk of fomite transmission under real-world conditions is likely to be substantially lower than the conditions used for experimental analyses [@doi:10/gg6br7].
The COVID-19 pandemic has, however, exposed significant discrepancies in how terms pertaining to airborne viral particles are interpreted in different contexts [@doi:10.1016/j.jhin.2020.12.022].
The 5-&mu;m distinction between "droplets" and "aerosols" is typical in the biological literature but is likely an artifact of historical science rather than a meaningful boundary in biology or physics [@{url:https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3829873}].
Additionally, various ambient conditions such as air flow can influence how particles of different sizes fall or spread [@doi:10.1016/j.jhin.2020.12.022].
Despite initial skepticism about airborne transmission of SARS-CoV-2 through small particles [@{url:https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3829873}], evidence now suggests that small particles can contribute to SARS-CoV-2 transmission [@doi:10.1126/science.abc6197; @doi:10.1056/NEJMc2004973; @doi:10.1093/cid/ciaa939; @doi:10/gjqmvq].
For example, one early study detected SARS-CoV-2 viral particles in air samples taken from hospitals treating COVID-19 patients, although the infectivity of these samples was not assessed [@doi:10.1038/s41586-020-2271-3].
Subsequently, other studies have been successful in growing SARS-CoV-2 in culture with samples taken from the air [@doi:10.1016/j.ijid.2020.09.025; @doi:10.1101/2020.07.13.20041632] while others have not [@doi:10.1038/s41598-020-69286-3; @doi:10.1093/cid/ciaa905] (see [@doi:10.1002/rmv.2184] for a systematic review of available findings as of July 2020).
The fact that viable SARS-CoV-2 may exist in aerosolized particles calls into question whether some axioms of COVID-19 prevention, such as 2-meter social distancing, are sufficient [@doi:10.1001/jama.2020.4756; @{url:https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3829873}; @doi:10.1016/j.ijid.2020.09.025].

#### Symptoms and Viral Spread

Other aspects of pathogenesis are also important to understanding how the virus spreads, especially the relationship between symptoms, viral shedding, and contagiousness.
Symptoms associated with reported cases of COVID-19 range from mild to severe [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html], but some individuals who contract COVID-19 remain asymptomatic throughout the duration of the illness [@doi:10.1007/s11427-020-1661-4].
The incubation period, or the time period between exposure and the onset of symptoms, has been estimated at five to eight days, with means of 4.91 (95% confidence interval (CI) 4.35-5.69) and 7.54 (95% CI 6.76-8.56) reported in two different Asian cities and a median of 5 (IQR 1 to 6) reported in a small number of patients in a Beijing hospital [@doi:10.7554/eLife.57149; @doi:10.1164/rccm.202003-0524LE].

However, the exact relationship between contagiousness and viral shedding remains unclear.
Estimates suggest that viral shedding can, in some cases, begin as early as 12.3 days (95% CI 5.9-17.0) before the onset of symptoms, although this was found to be very rare, with less than 0.1% of transmission events occurring 7 or more days before symptom onset [@doi:10.1038/s41591-020-0869-5].
Transmissibility appeared to peak around the onset of symptoms (95% CI -0.9 - 0.9 days), and only 44% (95% CI 30–57%) of transmission events were estimated to occur from presymptomatic contacts [@doi:10.1038/s41591-020-0869-5].
A peak in viral load corresponding to the onset of symptoms was also confirmed by another study [@doi:10.2807/1560-7917.ES.2020.25.32.2001483].
As these trends became apparent, concerns arose due to the potential for individuals who did not yet show symptoms to transmit the virus [@url:https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/how-covid-spreads.html].
Recovered individuals may also be able to transmit the virus after their symptoms cease.
A study of the communicable period based on twenty-four individuals who tested positive for SARS-CoV-2 prior to or without developing symptoms estimated that individuals may be contagious for one to twenty-one days, but they note that this estimate may be low [@doi:10.1007/s11427-020-1661-4].
In an early study, viral nucleic acids were reported to remain at observable levels in the respiratory specimens of recovering hospitalized COVID-19 patients for a median of 20 days and with a maximum observed duration through 37 days, when data collection for the study ceased [@doi:10/ggnxb3].

As more estimates of the duration of viral shedding were released, they converged around approximately three weeks from first positive PCR test and/or onset of symptoms (which, if present, are usually identified within three days of the initial PCR test).
For example, in some studies, viral shedding was reported for up to 28 days following symptom onset [@doi:10.1038/s41586-020-2196-x] and for one to 24 days from first positive PCR test, with a median of 12 days [@doi:10.1001/jama.2020.3204].
On the other hand, almost 70% of patients were reported to still have symptoms at the time that viral shedding ceased, although all symptoms reduced in prevalence between onset and cessation of viral shedding [@doi:10.1016/j.jcv.2020.104577].
The median time that elapsed between the onset of symptoms and cessation of viral RNA shedding was 23 days and between first positive PCR test and cessation of viral shedding was 17 days [@doi:10.1016/j.jcv.2020.104577].
The fact that this study reported symptom onset to predate the first positive PCR test by an average of three days, however, suggests that there may be some methodological differences between it and related studies.
Furthermore, an analysis of residents of a nursing home with a known SARS-CoV-2 case measured similar viral load in residents who were asymptomatic regardless of whether they later developed symptoms, and the load in the asymptomatic residents was comparable to that of residents who displayed either typical or atypical symptoms [@doi:10.1056/NEJMoa2008457].
Taken together, these results suggest that the presence or absence of symptoms are not reliable predictors of viral shedding or of SARS-CoV-2 status (e.g, [@doi:10.1001/jama.2020.6887]).
However, it should be noted that viral shedding is not necessarily a robust indicator of contagiousness.
The risk of spreading the infection was low after ten days from the onset of symptoms, as viral load in sputum was found to be unlikely to pose a significant risk based on efforts to culture samples _in vitro_ [@doi:10.1038/s41586-020-2196-x].
The relationship between symptoms, detectable levels of the virus, and risk of viral spread is therefore complex.

The extent to which asymptomatic or presymptomatic individuals are able to transmit SARS-CoV-2 has been a question of high scientific and community interest.
Early reports (February and March 2020) described transmission from presymptomatic SARS-CoV-2-positive individuals to close family contacts [@doi:10.1001/jama.2020.2565; @doi:10.1016/j.ijid.2020.03.027].
One of these reports [@doi:10.1016/j.ijid.2020.03.027] also included a description of an individual who tested positive for SARS-CoV-2 but never developed symptoms.
Later analyses also sought to estimate the proportion of infections that could be traced back to a presymptomatic or asymptomatic individual (e.g., [@doi:10.2139/ssrn.3678556]).
Estimates of the proportion of individuals with asymptomatic infections have varied widely.
The proportion of asymptomatic individuals on board the Diamond Princess cruise ship, which was the site of an early COVID-19 outbreak, was estimated at 17.9% [@doi:10.2807/1560-7917.ES.2020.25.10.2000180].
In contrast, a model using the prevalence of antibodies among residents of Wuhan, China estimated a much higher rate of asymptomatic cases, at approximately 7 in 8, or 87.5% [@doi:10.1093/pcmedi/pbaa032].
An analysis of the populations of care homes in London found that, among the residents (median age 85), the rate of asymptomatic infection was 43.8%, and among the caretakers (median age 47), the rate was 49.1% [@doi:10.1016/j.eclinm.2020.100533].
The duration of viral shedding may also be longer in individuals with asymptomatic cases of COVID-19 compared to those who do show symptoms [@doi:10.1038/s41591-020-0965-6].
As a result, the potential for individuals who do not know they have COVID-19 to spread the virus raises significant concerns.
In Singapore and Tianjin, two cities studied to estimate incubation period, an estimated 40-50% and 60-80% of cases, respectively, were considered to be caused by contact with asymptomatic individuals [@doi:10.7554/eLife.57149].
An analysis of viral spread in the Italian town of Vo', which was the site of an early COVID-19 outbreak, revealed that 42.5% of cases were asymptomatic and that the rate was similar across age groups [@doi:10.1038/s41586-020-2488-1].
The argument was thus made that the town's lockdown was imperative for controlling the spread of COVID-19 because it isolated asymptomatic individuals.
While more models are likely to emerge to better explore the effect of asymptomatic individuals on SARS-CoV-2 transmission, these results suggest that strategies for identifying and containing asymptomatic but contagious individuals are important for managing community spread.

#### Estimating the Fatality Rate

Estimating the occurrence of asymptomatic and mild COVID-19 cases is important to identifying the mortality rate associated with COVID-19.
The mortality rate of greatest interest would be the total number of fatalities as a fraction of the total number of people infected.
One commonly reported metric is the case fatality rate (CFR), which compares the number of COVID-19 related deaths to the number of confirmed or suspected cases.
However, in locations without universal testing protocols, it is impossible to identify all infected individuals because so many asymptomatic or mild cases go undetected.
Therefore, a more informative metric is the infection fatality rate (IFR), which compares the known deaths to the estimated number of cases.
It thus requires the same numerator as CFR, but divides by an approximation of the total number of cases rather than only the observed/suspected cases.
IFR varies regionally, with some locations observed to have IFRs as low as 0.17% while others are as high as 1.7% [@doi:10.1016/j.ijid.2020.09.1464].
Estimates of CFR at the national and continental level and IFR at the continent level is maintained by the Centre for Evidence-Based Medicine [@url:https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates].
Several meta-analyses have also sought to estimate IFR at the global scale.
These estimates have varied; one peer-reviewed study aggregated data from 24 other studies and estimated IFR at 0.68% (95% CI 0.53%–0.82%), but a preprint that aggregated data from 139 countries calculated a global IFR of 1.04% (95% CI 0.77%-1.38%) when false negatives were considered in the model [@doi:10.1101/2020.05.11.20098780; @doi:10.1016/j.ijid.2020.09.1464].
A similar prevalence estimate was identified through a repeated cross-sectional serosurvey conducted in New York City that estimated the IFR as 0.97% [@doi:10.1038/s41586-020-2912-6].
Examination of serosurvey-based estimates of IFR identified convergence on a global IFR estimate of 0.60% (95% CI 0.42%–0.77%) [@doi:10.1016/j.ijid.2020.09.1464].
All of these studies note that IFR varies widely by location, and it is also expected to vary with demographic and health-related variables such as age, sex, prevalence of comorbidities, and access to healthcare and testing [@url:https://ourworldindata.org/covid-mortality-risk].
Estimates of infection rates are becoming more feasible as more data becomes available for modeling and will be bolstered as serological testing becomes more common and more widely available.
However, this research may be complicated due to the emergence of variants over time, as well as the varying availability and acceptance of vaccines in different communities and locations.

### Dynamics of Transmission

Disease spread dynamics can be estimated using R~0~, the basic reproduction number, and R~t~, the effective reproduction number.
Accurate estimates of both are crucial to understanding the dynamics of infection and to predicting the effects of different interventions.
R~0~ is the average number of new (secondary) infections caused by one infected person, assuming a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x], and is one of the most important epidemiological parameters [@isbn:9780691116174].
A simple mechanistic model used to describe infectious disease dynamics is a susceptible-infected-recovered compartmental model [@doi:10.1098/rspa.1927.0118; @doi:10.1038/280361a0].
In this model, individuals move through three states: susceptible, infected, and recovered; two parameters, $\gamma$ and $\beta$, specify the rate at which the infectious recover, and the infection transmission rate, respectively, and R~0~ is estimated as the ratio of $\beta$ and $\gamma$ [@doi:10.1126/science.abb5659; @isbn:9780691116174].
A pathogen can invade a susceptible population only if R~0~ > 1 [@isbn:9780691116174; @isbn:9780199209996].
The spread of an infectious disease at a particular time t can be quantified by R~t~, the effective reproduction number, which assumes that part of the population has already recovered (and thus gained immunity to reinfection) or that mitigating interventions have been put into place.
For example, if only a fraction S~t~ of the population is still susceptible, R~t~ = S~t~ x R~0~.
When R~t~ is greater than 1, an epidemic grows (i.e., the proportion of the population that is infectious increases); when R~t~ is less than 1, the proportion of the population that is infectious decreases.
R~0~ and R~t~ can be estimated directly from epidemiological data or inferred using susceptible-infected-recovered-type models.
To capture the dynamics of SARS-CoV-2 accurately, the addition of a fourth compartment, i.e. a susceptible-exposed-infectious-recovered model, may be appropriate because such models account for the relative lengths of incubation and infectious periods [@doi:10.1038/s41598-020-76563-8].

Original estimates of R~0~ for COVID-19 lie in the range R~0~=1.4-6.5 [@doi:10/ggjvr7; @doi:10.1093/jtm/taaa021; @doi:10.1126/science.abb3221].
Variation in R~0~ is expected between different populations, and the estimated values of R~0~ discussed below are for specific populations in specific environments.
The different estimates of R~0~ should not necessarily be interpreted as a range of estimates of the same underlying parameter.
In one study of international cases, the predicted value was R~0~=1.7 [@doi:10.1101/2020.03.21.20040329].
In China (both Hubei province and nationwide), the value was predicted to lie in the range R~0~=2.0-3.6 [@doi:10.2139/ssrn.3524675; @doi:10.1101/2020.01.25.919787; @doi:10/ggjvr7].
Another estimate based on a cruise ship where an outbreak occurred predicted R~0~=2.28 [@doi:10.1016/j.ijid.2020.02.033].
Susceptible-exposed-infectious-recovered model-derived estimates of R~0~ range from 2.0 - 6.5 in China [@doi:10.3390/jcm9020462; @doi:10.1101/2020.01.27.20018952; @doi:10.1101/2020.01.23.916726; @doi:10.1101/2020.01.23.20018549] to R~0~=4.8 in France [@doi:10.1101/2020.03.22.20040915].
Using the same model as for the French population, a study estimated R~0~=2.6 in South Korea [@doi:10.1101/2020.03.22.20040915], which is consistent with other studies [@tag:Park2020_distancing].
From a meta-analysis of studies estimating R~0~, [@doi:10.1093/jtm/taaa021] the median R~0~ was estimated to be 2.79 (IQR 1.16) based on twelve studies published between January 1 and February 7, 2020.

Inference of the effective reproduction number can provide insight into how populations respond to an infection and the effectiveness of interventions.
In China, R~t~ was predicted to lie in the range 1.6-2.6 in January 2020, before travel restrictions [@doi:10/ggptcf].
R~t~ decreased from 2.35 one week before travel restrictions were imposed (January 23, 2020), to 1.05 one week after.
Using their model, the authors also estimated the probability of new outbreaks occurring.
Assuming individual-level variation in transmission comparable to that of MERS or SARS, the probability of a single individual exporting the virus and causing a large outbreak is 17-25%, and assuming variation like that of SARS and transmission patterns like those observed for COVID-19 in Wuhan, the probability of a large outbreak occurring after ≥4 infections exist at a new location is greater than 50%.
An independent study came to similar conclusions, finding R~t~=2.38 in the two-week period before January 23 with a decrease to R~t~ = 1.34 (using data from January 24 to February 3) or R~t~=0.98 (using data from January 24 to February 8) [@doi:10.1126/science.abb3221].
In South Korea, R~t~ was inferred for February through March 2020 in two cities, Daegu (the center of the outbreak) and Seoul [@tag:Park2020_distancing].
Metro data was also analyzed to estimate the effects of social distancing measures.
R~t~ decreased in Daegu from around 3 to <1 over the period that social distancing measures were introduced.
In Seoul, R~t~ decreased slightly, but remained close to 1 (and larger than R~t~ in Daegu).
These findings indicate that social distancing measures appeared to be effective in containing the infection in Daegu, but in Seoul, R~t~ remained above 1, meaning secondary outbreaks remained possible.
The study also shows the importance of region-specific analysis: the large decline in case load nationwide was mainly due to the Daegu region and could mask persistence of the epidemic in other regions, such as Seoul and Gyeonggi-do.
In Iran, estimates of R~t~ declined from 4.86 in the first week to 2.1 by the fourth week after the first cases were reported [@doi:10.1101/2020.03.20.20038422].
In Europe, analysis of 11 countries inferred the dynamics of R~t~ over a time range from the beginning of the outbreak until March 28, 2020, by which point most countries had implemented major interventions (such as stay-at-home orders, public gathering bans, and school closures) [@doi:10.25561/77731].
Across all countries, the mean R~t~ before interventions began was estimated as 3.87; R~t~ varied considerably, from below 3 in Norway to above 4.5 in Spain.
After interventions, R~t~ decreased by an average of 64% across all countries, with mean R~t~=1.43.
The lowest predicted value was 0.97 for Norway and the highest was 2.64 for Sweden, which could be related to the fact that Sweden did not implement social distancing measures on the same scale as other countries.
The study concludes that while large changes in R~t~ are observed, it is too early to tell whether the interventions put into place are sufficient to decrease R~t~ below 1.

Evolution within SARS-CoV-2 has also driven changes in the estimated reproduction number for different populations at different times.
As of June 2021, the reproduction number had increased globally relative to 2020, and increased transmissibility over the wildtype variant was observed for the Alpha, Beta, Gamma, and Delta VOC [@doi:10.2807/1560-7917.ES.2021.26.24.2100509].
In the U.S. between December 2020 and January 2021, B.1.1.7 (Alpha) was estimated to have an increased transmission of 35 to 45% relative to common SARS-CoV-2 variants at the time, with B.1.1.7 the dominant SARS-CoV-2 variant in some places at some timepoints [@doi:10.1101/2021.02.06.21251159].
This lineage was estimated to have increased transmissibility of 43 to 90% in the U.K. [@doi:10.1126/science.abg3055].
An estimate of the reproduction number of B.1.1.7 in the U.K. from September to December 2020 yielded 1.59 overall and between 1.56 and 1.95 in different regions of the country [@doi:10.1101/2020.12.30.20249034].
The Delta variant is particularly transmissible, and it has been estimated to be twice as transmissible than the wildtype variant of SARS-CoV-2 [@doi:10.2807/1560-7917.ES.2021.26.24.2100509].
A review of the literature describing the Delta variant identified a mean estimated R~0~ of 5.08 [@doi:10.1093/jtm/taab124].
Such differences can affect fitness and therefore influence the relative contributions of different lineages to a given viral gene pool over time [@doi:10.2807/1560-7917.ES.2021.26.27.2100570].
Therefore, the evolution of the virus can result in shifts in the reproduction rate.

More generally, population-level epidemic dynamics can be both observed and modeled [@doi:10.1126/science.abb5659].
Data and empirically determined biological mechanisms inform models, while models can be used to try to understand data and systems of interest or to make predictions about possible future dynamics, such as the estimation of capacity needs [@doi:10.1073/pnas.2004064117] or the comparison of predicted outcomes among prevention and control strategies [@doi:10/ggp3xq; @doi:10.1073/pnas.2004978117].
Many current efforts to model R~t~ have also led to tools that assist the visualization of estimates in real time or over recent intervals [@url:https://epiforecasts.io/covid; @url:https://rt.live].
These are valuable resources, yet it is also important to note that the estimates arise from models containing many assumptions and are dependent on the quality of the data they use, which varies widely by region.

### Conclusions

The novel coronavirus SARS-CoV-2 is the third HCoV to emerge in the 21st century, and research into previous HCoVs has provided a strong foundation for characterizing the pathogenesis and transmission of SARS-CoV-2.
Critical insights into how the virus interacts with human cells have been gained from previous research into HCoVs and other viral infections.
With the emergence of three devastating HCoV over the past twenty years, emergent viruses are likely to represent an ongoing threat.
Contextualizing SARS-CoV-2 alongside other viruses serves not only to provide insights that can be immediately useful for combating this virus itself but may also prove valuable in the face of future viral threats.

Host-pathogen interactions provide a basis not only for understanding COVID-19, but also for developing a response.
As with other HCoVs, the immune response to SARS-CoV-2 is likely driven by detection of its spike protein, which allows it to enter cells through ACE2.
Epithelial cells have also emerged as the major cellular target of the virus, contextualizing the respiratory and gastrointestinal symptoms that are frequently observed in COVID-19.
Many of the mechanisms that facilitate the pathogenesis of SARS-CoV-2 are currently under consideration as possible targets for the treatment or prevention of COVID-19 [@individual-pharmaceuticals; @individual-nutraceuticals].
Research in other viruses also provides a foundation for understanding the transmission of SARS-CoV-2 among people and can therefore inform efforts to control the virus's spread.
Airborne forms of transmission (droplet and aerosol transmission) have emerged as the primary modes by which the virus spreads to new hosts.
Asymptomatic transmission was also a concern in the SARS outbreak of 2002-03 and, as in the current pandemic, presented challenges for estimating rates of infection [@doi:10.1098/rstb.2004.1490].
These insights are important for developing a public health response, such as the CDC's shift in its recommendations surrounding masking [@url:https://www.latimes.com/science/story/2021-07-27/timeline-cdc-mask-guidance-during-covid-19-pandemic].

Even with the background obtained from research in SARS and MERS, COVID-19 has revealed itself to be a complex and difficult-to-characterize disease that has many possible presentations that vary with age.
Variability in presentation, including cases with no respiratory symptoms or with no symptoms altogether, were also reported during the SARS epidemic at the beginning of the 21st century [@doi:10.1098/rstb.2004.1490].
The variability of both which symptoms present and their severity have presented challenges for public health agencies seeking to provide clear recommendations regarding which symptoms indicate SARS-CoV-2 infection and should prompt isolation.
Asymptomatic cases add complexity both to efforts to estimate statistics such as R~0~ and R~t~, which are critical to understanding the transmission of the virus, and IFR, which is an important component of understanding its impact on a given population.
The development of diagnostic technologies over the course of the pandemic has facilitated more accurate identification, including of asymptomatic cases [@individual-diagnostics].
As more cases have been diagnosed, the health conditions and patient characteristics associated with more severe infection have also become more clear, although there are likely to be significant sociocultural elements that also influence these outcomes [@individual-inequality].
While many efforts have focused on adults, and especially older adults because of the susceptibility of this demographic, additional research is needed to understand the presentation of COVID-19 and MIS-C in pediatric patients.
As more information is uncovered about the pathogenesis of HCoV and SARS-CoV-2 specifically, the diverse symptomatology of COVID-19 has and likely will continue to conform with the ever-broadening understanding of how SARS-CoV-2 functions within a human host.

While the SARS-CoV-2 virus is very similar to other HCoV in several ways, including in its genomic structure and the structure of the virus itself, there are also some differences that may account for differences in the COVID-19 pandemic compared to the SARS and MERS epidemics of the past two decades.
The R~0~ of SARS-CoV-2 has been estimated to be similar to SARS-CoV-1 but much higher than that of MERS-CoV [@doi:10.1093/ajcp/aqaa029], although a higher R~0~ has been estimated for some VOC.
While the structures of the viruses are very similar, evolution among these species may account for differences in their transmissibility and virulence.
For example, the acquisition of a furin cleavage site the S1/S2 boundary within the SARS-CoV-2 S protein may be associated with increased virulence.
Additionally, concerns have been raised about the accumulation of mutations within the SARS-CoV-2 species itself, and whether these could influence virulence [@individual-evolution].
These novel variants may be resistant to vaccines and antibody treatments such as Bamlanivimab that were designed based on the wildtype spike protein [@individual-pharmaceuticals; @individual-vaccines].
As a consequence of reliance on targeting the SARS-CoV-2 spike protein for many therapeutic and prophylactic strategies, increased surveillance is required to rapidly identify and prevent the spread of novel SARS-CoV-2 variants with alterations to the spike protein.
The coming of age of genomic technologies has made these types of analyses feasible, and genomics research characterizing changes in SARS-CoV-2 along with temporal and spatial movement is likely to provide additional insights into whether within-species evolution influences the effect of the virus on the human host.
Additionally, the rapid development of sequencing technologies over the past decade has made it possible to rapidly characterize the host response to the virus.
For example, proteomics analysis of patient-derived cells revealed candidate genes whose regulation is altered by SARS-CoV-2 infection, suggesting possible approaches for pharmaceutical invention and providing insight into which systems are likely to be disrupted in COVID-19 [@doi:10.1101/2020.03.31.019216].
As more patient data becomes available, the biotechnological advances of the 2000s are expected to allow for more rapid identification of potential drug targets than was feasible during the SARS, or even MERS, pandemic.

Thus, the COVID-19 crisis continues to evolve, but the insights acquired over the past 20 years of HCoV research have provided a solid foundation for understanding the SARS-CoV-2 virus and the disease it causes.
As the scientific community continues to respond to COVID-19 and to elucidate more of the relationships between pathogenesis, transmission, host regulatory responses, and symptomatology, this understanding will no doubt continue to evolve and to reveal additional connections among virology, pathogenesis, and health.
This review represents a collaboration between scientists from diverse backgrounds to contextualize this virus at the union of many different biological disciplines [@individual-methods].
At present, understanding the SARS-CoV-2 virus and its pathogenesis is critical to a holistic understanding of the COVID-19 pandemic.
In the future, interdisciplinary work on SARS-CoV-2 and COVID-19 may guide a response to a new viral threat.


## Evolutionary and Genomic Analysis of SARS-CoV-2

### Abstract

### Introduction

### Initial Characterization of SARS-CoV-2

The first genome sequence of the virus was released on January 3, 2020. 
It revealed that the cluster of pneumonia cases seen in Wuhan were caused by a novel coronavirus [@doi:10.46234/ccdcw2020.017].
Multiple research groups have drafted the genome sequence of SARS-CoV-2 based on sequences developed from clinical samples collected from the lower respiratory tract, namely bronchoalveolar lavage fluid (BALF), and the upper respiratory tract, in the form of throat and nasopharyngeal swabs [@doi:10.1038/s41586-020-2008-3; @doi:10.1038/s41586-020-2012-7; @doi:10/ggjr43].
Analysis of the SARS-CoV-2 genome revealed significant sequence homology with two coronaviruses known to infect humans, with about 79% shared sequence identity with SARS-CoV-1 and 50% with MERS-CoV [@doi:10/ggjr43].
However, in this analysis, the highest degree of similarity was observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21) [@doi:10/ggjr43].
Other analyses have reported even greater similarity between SARS-CoV-2 and the bat coronavirus BatCoV-RaTG13, with shared sequence identity as high as 96.2% [@doi:10.1038/s41586-020-2012-7; @doi:10.1093/nsr/nwaa036], and the closely related pangolin coronavirus [@doi:10.1101/2020.02.19.950253].
This evidence therefore suggests the SARS-CoV-2 virus is the result of zoonotic transfer of a virus from bats to humans.
Nevertheless, some fragments between SARS-CoV-2 and RATG13 differ by up to 17%, suggesting a complex natural selection process during zoonotic transfer.
While the _S_ region is highly similar to that of viruses found in pangolins [@doi:10.1101/2020.02.19.950253], there is no consensus about the origin of _S_ in SARS-CoV-2, as it could potentially be the result either of recombination or coevolution [@doi:10.1093/nsr/nwaa036; @doi:10.1126/sciadv.abb9153].
Though the intermediate host serving as the source for the zoonotic introduction of SARS-CoV-2 to human populations has not yet been identified, the SARS-CoV-2 virus has been placed within the coronavirus phylogeny through comparative genomic analyses.
Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-2 is unlikely to have originated in a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi:10.1038/s41591-020-0820-9].
Indeed, the World Health Organization (WHO) have published their intentions to thoroughly investigate the origins of SARS-CoV-2 [@url:https://www.who.int/docs/default-source/coronaviruse/20200802-tors-chn-and-who-agreed-final-version.pdf?sfvrsn=5323728d_2&download=true].
While the position of the SARS-CoV-2 virus within the coronavirus phylogeny has been largely resolved, the functional consequences of molecular variation between this virus and other viruses, such as its bat and pangolin sister taxa or SARS-CoV-1, remain unknown [@doi:10.1038/s41591-020-0820-9, @doi:10.1038/s41586-020-2012-7].
Fortunately, the basic genome structure of coronaviruses is highly conserved, and insight into the mechanisms the virus uses to enter cells, replicate, and spread is available from prior research on coronaviruses, which has been instrumental in the mobilization of global research to understand the biology of SARS-CoV-2. 

Additionally, worldwide sequencing of viral samples has provided some preliminary insights into possible mechanisms of adaptation in the virus and the detection of novel variants, and omics-based analysis of patient samples has elucidated some of the biological changes the virus induces in its human hosts.

### Coronaviruses and Animal Hosts

Coronaviruses have long been known to infect animals and have been the subject of veterinary medical investigations and vaccine development efforts due to their effect on the health of companion and agricultural animals [@doi:10/ckfn8b].
<!-- TO DO: Expand on the relationship between coronaviruses and animals -- which taxa are most often affected? how severe are veterinary infections? how has vaccine development worked for these diseases?--> 

<!-- TO DO: Be care to check overlap between this section and 20.pharmaceuticals.md. Trim to address just the important information about human-coronavirus interactions that feeds into an evolutionary perspective of how these host/virus interactions evolve across taxa and over time -->
Most coronaviruses show little to no transmission in humans.
However, it is thought that approximately one-third of common cold infections are caused by four seasonal human coronaviruses (HCoV): _Human coronavirus 229E_ (HCoV-229E), _Human coronavirus NL63_ (HCoV-NL63), _Human coronavirus OC43_ (HCoV-OC43), and _Human coronavirus HKU1_ (HCoV-HKU1) [@doi:10.1038/s41579-018-0118-9; @doi:10.3390/diseases4030026; @doi:10.1016/j.jcv.2018.01.019].
The first HCoV were identified in the 1960s: HCoV-229E in 1965 [@doi:10.3181/00379727-121-30734] and HCoV-OC43 in 1967 [@doi:10.1073/pnas.57.4.933].
Both of these viruses typically cause cold-like symptoms, including upper and lower respiratory infections [@doi:10.1186/1743-422X-1-7; @pmid:20700397; @doi:10.1186/s40779-020-00279-z], but they have also been associated with gastrointestinal symptoms [@doi:10.1111/j.1440-1754.2007.01246.x].
Two additional HCoV were subsequently identified [@doi:10.1038/nm1024; @doi:10.1128/JVI.79.2.884-895.2005].
In 2003, HCoV-NL63 [@doi:10.1038/nm1024] was first identified in a 7-month-old infant and then in clinical specimens collected from seven additional patients, five of whom were infants younger than 1 year old and the remainder of whom were adults.
CoV-HKU1 was identified in samples collected from a 71-year-old pneumonia patient in 2004 and then found in samples collected from a second adult patient [@doi:10.1128/JVI.79.2.884-895.2005].
These viruses are associated with respiratory diseases of varying severity, ranging from common cold to severe pneumonia, with severe symptoms mostly observed in immunocompromised individuals [@doi:10.1086/377612], and also have gastrointestinal involvement in some cases [@doi:10.1111/j.1440-1754.2007.01246.x].
In addition to these relatively mild HCoV, however, highly pathogenic human coronaviruses have been identified, including _Severe acute respiratory syndrome-related coronavirus_ (SARS-CoV or SARS-CoV-1) and _Middle East respiratory syndrome-related coronavirus_ (MERS-CoV) [@doi:10.1038/nrmicro.2016.81; @doi:10.1038/s41579-018-0118-9; @doi:10.1016/bs.aivir.2018.01.001].

At the time that SARS-CoV-1 emerged in the early 2000s, no HCoV had been identified in almost 40 years [@doi:10.1038/nrmicro.2016.81].
The first case of SARS was reported in November 2002 in the Guangdong Province of China, and over the following month, the disease spread more widely within China and then into several countries across multiple continents [@doi:10.1093/ajcp/aqaa029; @doi:10.1038/nrmicro.2016.81].
Unlike previously identified HCoV, SARS was much more severe, with an estimated death rate of 9.5% [@doi:10.1093/ajcp/aqaa029].
It was also highly contagious via droplet transmission, with a basic reproduction number (R~0~) of 4 (i.e., each person infected was estimated to infect four other people) [@doi:10.1093/ajcp/aqaa029].
However, the identity of the virus behind the infection remained unknown until April of 2003, when the SARS-CoV-1 virus was identified through a worldwide scientific effort spearheaded by the WHO [@doi:10.1038/nrmicro.2016.81].
SARS-CoV-1 belonged to a distinct lineage from the two other HCoV known at the time [@doi:10.1093/ajcp/aqaa029].
By July 2003, the SARS outbreak was officially determined to be under control, with the success credited to infection management practices [@doi:10.1038/nrmicro.2016.81].
A decade later, a second outbreak of severe respiratory illness associated with a coronavirus emerged, this time in the Arabian Peninsula.
This disease, known as Middle East respiratory syndrome (MERS), was linked to another novel coronavirus, MERS-CoV.
The fatality rate associated with MERS is much higher than that of SARS, at almost 35%, but the disease is much less easily transmitted, with an R~0~ of 1 [@doi:10.1093/ajcp/aqaa029].
Although MERS is still circulating, its low reproduction number has allowed for its spread to be contained [@doi:10.1093/ajcp/aqaa029].
The COVID-19 pandemic is thus associated with the seventh HCoV to be identified and the fifth since the turn of the millennium, though additional HCoVs may be in circulation but remain undetected.  
Possible evidence for a rare eighth HCoV was later reported by [@doi:10.1093/cid/ciab456].

SARS-CoV-1 and MERS-CoV were ultimately managed largely through infection management practices (e.g., mask wearing) and properties of the virus itself (i.e., low rate of transmission), respectively [@doi:10.1038/nrmicro.2016.81; @doi:10.1093/ajcp/aqaa029].
Vaccines were not used to control either outbreak, although vaccine development programs were established for SARS-CoV-1 [@doi:10.3390/v11010059].
In general, care for SARS and MERS patients focuses on supportive care and symptom management [@doi:10.1093/ajcp/aqaa029].
Clinical treatments for SARS and MERS developed during the outbreaks generally do not have strong evidence supporting their use.
Common treatments included Ribavirin, an antiviral, often in combination with corticosteroids or sometimes interferon (IFN) medications, which would both be expected to have immunomodulatory effects [@doi:10.1038/nrmicro.2016.81].
However, retrospective and _in vitro_ analyses have reported inconclusive results of these treatments on SARS and the SARS-CoV-1 virus, respectively [@doi:10.1038/nrmicro.2016.81].
IFNs and Ribavirin have shown promise in _in vitro_ analyses of MERS, but their clinical effectiveness remains unknown [@doi:10.1038/nrmicro.2016.81].
Therefore, only limited strategies can be adopted for the pharmaceutical management of COVID-19 from previous severe HCoV infections.
Research in response to prior outbreaks of HCoV-borne infections, such as SARS and MERS, have, however, provided a strong foundation for hypotheses about the pathogenesis of SARS-CoV-2 as well as potential diagnostic and therapeutic approaches.

### Zoonotic Transfer of Coronaviruses

<!-- TO DO: How does zoonotic transfer happen? What role has this played in previous HCoV outbreaks? How do viruses evolve during host species changes?-->

If not addressed, economic and environmental stressors are likely to cause future zoonotic transfer of diseases in the future [@doi:10.1126/science.abc3189].

### Evolution of the SARS-CoV-2 Virus

#### Emergence of SARS-CoV-2

<!-- TO DO: Which zoonotic viruses have been identified as likely relatives of SARS-CoV-2? How do we know this viral species evolved naturally (as opposed to the conspiracy theories? What differs between SARS-CoV-2 and its close relatives?-->

#### Evolution of SARS-CoV-2 Variants

Evolution in SARS-CoV-2 has also been observed over a short timescale.
After zoonotic transfer, SARS-CoV-2 continued evolving in the human population [@doi:10.1186/s12967-020-02344-6].
The SARS-CoV-2 mutation rate is moderate compared to other RNA viruses [@doi:10.1016/j.meegid.2020.104351], which likely restricts the pace of evolution in SARS-CoV-2.
Nevertheless, genomic analyses have yielded statistical evidence of ongoing evolution.
Initially, two known variants of the spike protein emerged that differed by a single amino acid at position 614 (G614 and D614), and there is evidence that G614 had become more prevalent than D614 by June 2020 [@doi:10.1016/j.cell.2020.06.043].
While there is a hypothesis that this genomic change increased the SARS-CoV-2 infectivity and virulence, this hypothesis has not yet been tested due to a lack of data [@doi:10.1016/j.cell.2020.06.040].
Another study [@doi:10.1016/j.meegid.2020.104351] identified 198 recurrent mutations in a dataset of 7,666 curated sequences, all of which defined non-synonymous protein-level modifications.
This pattern of convergent evolution at some sites could indicate that certain mutations confer an adaptive advantage.
While it is evident that SARS-CoV-2 exhibits moderate potential for ongoing and future evolution, the relationship between mutations and pathogenicity is not yet known.
Additional data is needed in order to understand patterns of evolutionary change and the mechanisms by which they might affect virulence.

Several factors could promote the evolution of SARS-CoV-2, including host immunodeficiency and transient exposure to antibodies directed against SARS-CoV-2 proteins. 
A single case study of SARS-CoV-2 infection in an immunocompromised female with chronic lymphocytic leukemia and hypogammaglobulinemia [@doi:10.1016/j.cell.2020.10.049] suggested that an accelerated evolution of the virus could occur in conditions of immunodeficiency. 
A first administration of convalescent plasma did not clear the virus, and an ensuing increase in the genomic diversity in the samples was observed, suggesting an accelerated evolution due to selection pressure. 
A second administration of convalescent plasma cleared the virus from the host 105 days after the initial diagnosis. 
However, throughout the duration of infection, the patient was asymptomatic but contagious.
A second single case study in a 45-year old male with antiphospholipid syndrome [@doi:10.1056/NEJMc2031364] confirmed the earlier results, providing evidence of persistent COVID-19 symptoms in an immunocompromised patient for 154 days following diagnosis, ultimately leading to the death of patient. 
The treatments administered included remdesivir and the Regeneron anti-spike protein antibody cocktail. 
Genomic analyses of the patient's nasopharyngeal swabs confirmed an accelerated evolution of the virus through mutations in the spike gene and the receptor-binding domain. 
In summary, these two case studies suggested an accelerated evolution and persistent shedding of the virus in conditions of immunodeficiency. 
In particular, the first case highlighted the role of convalescent plasma in creating escape variants. 
In fact, one study [@doi:10.1101/2020.12.28.424451] exposed the SARS-CoV-2 virus to convalescent plasma _in vitro_ repeatedly to see how much plasma was required to neutralize the virus.
The results of the first six exposures were similar, but they reported that after the seventh exposure (on day 45), the amount of plasma required began to increase.
In analyzing the viral variants present, they found that this viral escape was promoted by the sudden accumulation of mutations, especially in the receptor-binding domain (RBD) and N-terminal domain (NTD), that quickly rose in frequency.
By the thirteenth exposure (day 85), the virus had evolved three mutations and could no longer be neutralized by the plasma used, even though the plasma was comprised of polyclonal serum that targeted a variety of epitopes.
Taken together, these observations suggest that evolutionary analyses of SARS-CoV-2 can provide crucial information about the conditions that promote resistance in SARS-CoV-2 and the kinetics of how resistance develops, information which will be important for understanding the implications of how vaccine regimens are designed and whether/when next-generation vaccines will be needed.

When variants occur, they can rise in frequency by chance or through an adaptive process that confers a competitive advantage to the virus.
Variants that had the D614G mutation in the spike glycoprotein seemed to spread faster. <!-- TO DO: add more history of this variant?-->
However, it has been suggested that the mutation rose in frequency due to early chance events rather than by adaptive events [@doi:10.1001/jama.2020.27124].
Another mutation, Y453F, that occurred in the receptor binding domain of _S_, was first detected in mink; however, the transmission to humans has been established.
In mink, this mutation conferred an advantage by increasing the affinity towards ACE2 [@doi:10.1126/science.abe5901].
Similarly, N501Y mutation induces an increased affinity towards human ACE2 and has been involved in the dominance of B.1.1.7 by outcompeting other variants [@doi:10.1101/2020.12.24.20248822].
Therefore, genomic surveillance is essential to prevent the emergence of super-spreaders [@doi:10.1126/science.abf2946].

Emerging methods are being applied to this problem in an effort to understand which mutations are most likely to be of significant concern.
Novel machine learning methods were developed to predict the mutations in the sequence that promote viral escape.
While they preserve the pathogenicity of the virus, escape mutations change the virus's sequence to evade detection by the immune system. 
By using tools from natural language processing (NLP), viral escape was modeled as an NLP problem [@doi:10.1126/science.abd7331] where a modification makes a sentence grammatically correct but semantically different.
Therefore, language models of viruses could predict mutations that change the presentation of the virus to the immune system but preserve its infectivity.

#### Variants of concern and variants under surveillance

Viral replication naturally leads to the occurrence of mutations, and thus to genetic variation [@doi:10.1038/s41564-020-0690-4].
However, due to an intrinsic RNA proof-reading process in the SARS-CoV-2 virus, the pace of evolution of SARS-CoV-2 is moderate in comparison to other viruses [@doi:10.1186/1471-2148-4-21].
The declaration of the first SARS-CoV-2 variant of concern (VOC) B.1.1.7 in December 2020 has attracted significant media attention.
While the B.1.1.7 lineage garnered attention in November 2020, two genomes of the lineage were detected as early as September 20th, 2020 from routine genomic data sampled in Kent (U.K.) by the COVID-19 Genomics UK Consortium (COG-UK). 
The following day, a second B.1.1.7 genome was reported in greater London [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf; @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563; @doi:10.1101/2020.12.30.20249034; @doi:10.1001/jama.2020.27124]
Since then, B.1.1.7 has spread across the UK and internationally, and it has now been detected in at least 62 countries [@url:https://cov-lineages.org/global_report_B.1.1.7.html], despite several countries imposing travel restrictions on travelers from the UK. 
Of the twenty-three mutations that define B.1.1.7 from the original strain isolated in Wuhan (lineage A), fourteen are lineage-specific and three appear to be biologically consequential mutations associated with the spike protein, namely N501Y, P681H, and 69-70del [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf;  @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563].
The latter is a 6-bp deletion that leads to the loss of two amino acids and has consequences for immune recognition; it may, in conjunction with N501Y, be responsible for the increased transmissibility of the B.1.1.7 VOC due to changes in the RBD that increase binding affinity with ACE2 [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf; @doi:10.1101/2020.12.14.422555]. 
B.1.1.7 has increased transmissibility by up to 56%, leading to an R~0~ of approximately 1.4.
Additionally, this VOC has been shown to be associated with increased disease severity and increased mortality [@doi:10.1038/s41586-021-03426-1]. 
Other variants also express the 69-70del mutation [@url:https://virological.org/t/identification-of-a-novel-sars-cov-2-spike-69-70-deletion-lineage-circulating-in-the-united-states/577; @doi:10.1101/2020.12.24.20248814], and public health officials in the United States and the UK have been able to use RT-PCR-based assays (ThermoFisher TaqPath COVID-19 assay) to identify sequences with this deletion because it occurs where the qPCR probe binds [@doi:10.1101/2020.12.30.20249034]. 
In the UK, B.1.1.7 is present in more than 97% of diagnostic tests that return negative for S-gene targets and positive for the other targets; thus, the frequency of S-gene target failure can be used as a proxy for the detection of B.1.1.7 [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf; https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/950823/Variant_of_Concern_VOC_202012_01_Technical_Briefing_3_-_England.pdf]._
The FDA has highlighted that the performance of three diagnostic tests may be affected by the B.1.1.7 lineage because it could cause false negative tests [@url:https://www.fda.gov/medical-devices/letters-health-care-providers/genetic-variants-sars-cov-2-may-lead-false-negative-results-molecular-tests-detection-sars-cov-2].

While B.1.1.7 is currently the main VOC, other genetic variants also currently designated as VOCs have been detected, including B.1.351 and P.1, both of which emerged independently [@url:https://www.cdc.gov/coronavirus/2019-ncov/more/science-and-research/scientific-brief-emerging-variants.html; @url:https://www.cdc.gov/coronavirus/2019-ncov/cases-updates/variant-surveillance/variant-info.html].
B.1.351 was first detected in October 2020 in South Africa, was later detected in the EU on December 28th, 2020 and has now spread to at least 26 countries [@url:https://www.gov.za/speeches/minister-zweli-mkhize-confirms-8-725-more-cases-coronavirus-covid-19-18-dec-2020-0000; @url:https://cov-lineages.org/global_report_B.1.351.html; @url:https://virological.org/t/tracking-the-international-spread-of-sars-cov-2-lineages-b-1-1-7-and-b-1-351-501y-v2/592]. 
B.1.351 contains several mutations at the RBD including K417N, E484K, and N501Y.
While the biological significance of these mutations are still under investigation, it does appear that this lineage may be associated with increased transmissibility [@doi:10.1101/2020.12.21.20248640] due to the N501Y mutation [@doi:10.1101/2020.12.14.422555; @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563].
Additionally, an analysis of a pseudovirus expressing the 501Y.V2 spike protein (B.1.351) showed that this variant demonstrates increased resistance to neutralization by convalescent plasma, even though total binding activity remained mostly intact [@doi:10.1038/s41591-021-01285-x].
Further, using a live virus neutralization assay (LVNA), it was shown that 501Y.V2 (B.1.351) is poorly neutralized by convalescent plasma obtained from individuals who responded to non-501Y.V2 variants [@doi:10.1038/s41586-021-03471-w]. 
However, 501Y.V2 infection-elicited plasma was able to cross-neutralize earlier non-501Y.V2 variants, suggesting that vaccines targeting VOCs may be effective against other mutant lineages [@doi:10.1038/s41586-021-03471-w].

The P.1 variant is a sublineage of the B.1.1.28 lineage that was first detected in Japan in samples obtained from four travelers from Brazil during a screening at a Tokyo airport on January 10, 2021 [@url:https://www.ecdc.europa.eu/sites/default/files/documents/COVID-19-risk-related-to-spread-of-new-SARS-CoV-2-variants-EU-EEA-first-update.pdf].
Shortly thereafter, it was established that there was a concentration of cases of the P.1 variant in Manaus, Brazil.
In a small number of samples (n=31) sequenced in Manaus, 42% were identified as the P.1 variant as early as mid-December, but the variant seemed to be absent in genome surveillance testing prior to December [@url:https://virological.org/t/genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-manaus-preliminary-findings/586].
To date, at least eight countries have detected the P.1 lineage [@url:https://cov-lineages.org/global_report_P.1.html].
While the majority of P.1 cases detected internationally have been linked to travel originating from Brazil, the UK has also reported evidence of community transmission detected via routine community sequencing [@url:https://cov-lineages.org/global_report_P.1.html; @url:https://www.reuters.com/article/us-health-coronavirus-britain-variants/uk-detects-77-cases-of-south-african-covid-variant-nine-of-brazilian-idUSKBN29T07P].  
P.1 has eight lineage-specific mutations along with three concerning spike protein mutations in the RBD, including K417T, E484K, and N501Y [@doi:10.1101/2020.12.21.20248640].
 
There have been multiple different SARS-CoV-2 lineages detected that have mostly been of no more clinical concern than the original devastating lineage originating in Wuhan [@url:https://cov-lineages.org/lineages.html].
However, the spotlight has been cast on other variants of unknown clinical relevance due to the increase of cases observed that have been associated with B.1.1.7 in particular.  
Although early in its ascendency, B.1.427/429 are SARS-CoV-2 variants that was detected in California, USA and also known as CAL.20C [@doi:10.1101/2021.01.18.21249786].
It was first detected in July 2020 but was not detected again until October 2020. 
In December 2020, B.1.427/429 accounted for ~24% of the total cases in Southern California and ~36% of total cases in the Los Angeles area. 
B.1.427/429 have now been detected in several U.S. states and at least 38 countries worldwide [@url:https://www.gisaid.org/hcov19-variants/; @doi:10.1101/2021.01.18.21249786].
This variant is characterized by five key lineage-specific mutations (ORF1a: I4205V, ORF1b:D1183Y, S: S13I;W152C;L452R).
The latter spike mutation, L452R, is found in an area of the RBD known to resist monoclonal antibodies to the spike protein [@doi:10.1016/j.cell.2020.07.012], and it is hypothesized that this mutation may resist polyclonal sera in convalescent patients or in individuals post-vaccination [@doi:10.1101/2020.12.31.425021; @doi:10.1101/2021.01.18.21249786].
B.1.427/429 are now designated VOCs [@url:https://www.cdc.gov/coronavirus/2019-ncov/cases-updates/variant-surveillance/variant-info.html]; however, further research is still required to determine the implications of the mutations encoded in this genetic variant.  
Another notable variant has recently been discovered in 35 patients in a Bavarian hospital in Germany; however, the sequencing data has not been published to date and it remains to be determined whether this variant is of any further concern [@url:https://www.br.de/nachrichten/bayern/neue-coronavirus-mutation-35-faelle-in-garmisch-partenkirchen,SMQ1V6u].

There are several shared mutations and deletions between the three lineages, P.1, B.1.1.7, and B.1.315 and indeed other variants of SARS-CoV-2 that are under investigation [@url:https://virological.org/t/genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-manaus-preliminary-findings/586].
For example, N501Y, which appears to have occurred independently in each of the three lineages.  
E484K is present in both B.1.351 and P.1 [@url:https://cov-lineages.org/global_report.html].
The mutations N501Y and E484K are found in the RBD within the receptor-binding motif responsible for forming an interface with the ACE2 receptor, which seems to be consequential for ACE2 binding affinity [@doi:10.1016/j.cell.2020.08.012].
Indeed, N501Y is associated with increased virulence and infectivity in mouse models [@doi:10.1126/science.abc4730].
E484K has also been associated with evasion from neutralizing antibodies [@doi:10.1016/j.chom.2020.11.007; @doi:10.1101/2020.12.28.424451; @doi:10.1101/2020.12.31.425021].
The del69-70 (del:11288:9) is also shared between P.1 and B.1.1.7 and happens to be a common deletion found in the N terminal mutation of the spike protein. 
This deletion has also been associated with several RBD mutations [@doi:10.1101/2020.11.19.389916; @doi:10.1101/2020.12.14.422555; @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563].
There is concern that mutations in the spike protein of variants may lead to clinical consequences for transmissibility, disease severity, re-infection, therapeutics, and vaccinations [@doi:10.1126/science.abg4312; @url:https://virological.org/t/sars-cov-2-reinfection-by-the-new-variant-of-concern-voc-p-1-in-amazonas-brazil/596; @doi:10.1101/2020.12.28.424451; @doi:10.1101/2020.12.31.425021; @doi:10.1038/d41586-021-00121-z; @doi:10.1126/science.abf9302; @doi:10.1126/science.371.6527.329]. 

Vaccine producers are working to determine whether the vaccines are still effective against the novel genetic variants.
Moderna recently published data for their mRNA-1273 vaccine that showed no significant impact of neutralization against the B.1.1.7 variant upon vaccination in humans and non-human primates. 
On the other hand, Moderna reported a reduced but significant neutralization against the B.1.351 variant upon vaccination [@doi:10.1101/2021.01.25.427948].
Indeed, Pfizer–BioNTech reported that sera from twenty participants vaccinated with the BNT162b COVID-19 vaccine in previous clinical trials [@doi:10.1056/NEJMoa2034577; @doi:10.1056/NEJMoa2027906] elicited equivalent neutralizing titers against isogenic Y501 SARS-CoV-2 on an N501Y genetic background _in vitro_ [@doi:10.1101/2021.01.07.425740].
Another study has reported that the plasma neutralizing activity against SARS-CoV-2 variants encoding the combination of K417N:E484K:N501Y or E484K or N501Y was variably and significantly reduced in the sera of twenty participants who received either the Pfizer–BioNTech BNT162b (n = 6) vaccine or the Moderna’s mRNA-1273 vaccine (n =14) [@doi:10.1101/2021.01.15.426911]. 
In a study focusing on serum samples from a combination of convalescent individuals, those who obtained the mRNA-1273 vaccine, and those who obtained Novavax, in comparison to the D614G variant, the B.1.419 variant was 2-3 times less sensitive to neutralization while the B.1.351 variant was 9-14 times less sensitive [@doi:10.1056/NEJMc2103740].
Indeed, the E484K substitution seen in the P.1 and B.1.315 variants of the B.1.1.7 lineage are broadly reported to substantially reduce the efficacy of mRNA-based vaccines [@doi:10/f53t; @doi:10.1038/s41586-021-03412-7; @doi:10.1056/NEJMc2103740]. 
For now, the consensus appears to be that the FDA-approved vaccines still seem to be generally effective against the genetic variants of SARS-CoV-2 and their accompanying mutations, albeit with a lower neutralizing capacity [@doi:10.1073/pnas.2008281117; @doi:10.1101/2021.01.15.426911; @doi:10.1101/2021.01.07.425740; @doi:10.1101/2021.01.25.427948], though select VOCs may present challenges. 
Further research is required to discern the clinical, prophylactic, and therapeutic consequences of these genetic SARS-CoV-2 variants as the pandemic evolves. 

### Conclusions

As of October 2020 the SARS-CoV-2 virus remains a serious worldwide threat.
The scientific community has responded by rapidly collecting and disseminating information about the SARS-CoV-2 virus and the associated illness, COVID-19.
The rapid identification of the genomic sequence of the virus allowed for early contextualization of SARS-CoV-2 among other known respiratory viruses.
The pathogen is a coronavirus that is closely related to SARS-CoV-1, which caused the SARS pandemics of the early 2000s.
Knowing the phylogenetic context and genomic sequence of the virus then allowed for rapid insights into its structure and pathogenesis.


## Diagnostics

### Abstract

### Importance

### Introduction

Since the emergence of _Severe acute respiratory syndrome-like coronavirus 2_ (SARS-CoV-2) in late 2019, significant international concern has focused on how to manage the spread of the virus.
Identifying individuals who have contracted coronavirus disease 2019 (COVID-19) is crucial to slowing down the global pandemic.
Given the high transmissibility of SARS-CoV-2 and the potential for asymptomatic or presymptomatic individuals to be contagious [@individual-pathogenesis], the development of rapid, reliable, and affordable methods to detect SARS-CoV-2 infection is and was vitally important for understanding and controlling spread.
For instance, test-trace-isolate procedures were an early cornerstone of many nations' efforts to control the outbreak [@doi:10.1503/cmaj.202120; @doi:10/ftzx; @doi:10.1016/j.lanwpc.2020.100044].
<!-- To Do: possibly add text on what goes into developing new diagnostics?-->

The genetic sequence of the virus was first released by Chinese officials on January 10, 2020, and the first test to detect the virus was released about 13 days later [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
This information is important to the development of diagnostic approaches using a variety of approaches.
There are two main classes of diagnostic tests: molecular tests, which can diagnose an active infection by identifying the presence of SARS-CoV-2, and serological tests, which can assess whether an individual was infected in the past via the presence or absence of antibodies against SARS-CoV-2.
Molecular tests are essential for identifying individuals for treatment and alerting their contacts to quarantine and be alert for possible symptoms.
Here, a patient sample is evaluated to determine the presence or absence of a viral target.
These techniques depend on knowledge of the viral genomic sequence for the development of targeted primers.
On the other hand, serological tests are useful for collecting population-level information for epidemiological analysis, as they can be used to estimate the extent of the infection in a given area.
Thus, they may be useful in efforts to better understand the percent of cases that manifest as severe versus mild and for guiding public health and economic decisions regarding resource allocation and counter-disease measures.
These tests typically detect the presence of antibodies in blood plasma samples.
In such enzyme-linked immunosorbent assay (ELISA) approaches, the detection of the antibodies depends on knowledge of a specific antibody-antigen interaction.
<!-- To Do: How did we discover the identity of these to start binding them?-->
As the pandemic has evolved throughout 2020 and 2021, a variety of technological implementations have emerged within these two categories.

<!-- To Do: Discuss HCoV broadly and the potential risk of detecting other coronaviruses-->
<!-- To Do: Trade offs in cost vs precision (throughput)-->
<!-- To Do: Discuss viral load vs contagiousness vs symptoms vs antibody development? -->

<!-- To:Write / complete - #### Extant Work

Prior narrative reviews [@doi:10.7326/M20-1301] and systematic reviews [@doi:10.1080/14737159.2020.1816466; @doi:10.1016/j.ajic.2020.07.011] of diagnostic technology for COVID-19 have been perfomed, and these contain detailed review of technologies and specific tests or techniques that exist.
Other reviews provided interim status and reviewed challenges [@doi:10.1016/j.isci.2020.101406], or high level overviews [@doi:10.1126/scitranslmed.abc1931].
Policy discussions have also reviewed both the critical importance of diagnostic in the developing world [@doi:10.1038/d41586-020-01265-0], and how they can best be used [@http://documents.worldbank.org/curated/en/426741613767908998/Five-Ways-that-COVID-19-Diagnostics-Can-Save-Lives-Prioritizing-Uses-of-Tests-to-Maximize-Cost-Effectiveness], and unfortunately, the need for triage when insufficient supplies are available [@doi:10.1016/j.jacc.2020.05.006] -->

### Molecular Tests

Molecular tests are used to identify distinct genomic subsequences of a viral molecule in a sample and thus to diagnose an active viral infection.
An important first step is identifying which biospecimens are likely to contain the virus in infected individuals and then acquiring these samples from the patient(s) to be tested.
Common sampling sources for molecular tests include nasopharyngeal cavity samples, such as throat washes,  throat swabs, and saliva [@doi:10/ggp4qx], and stool samples [@doi:10.1002/jmv.25742].
Once a sample from an appropriate source is acquired from a patient, molecular tests can utilize a number of different steps, described below, to analyze a sample and identify whether evidence of SARS-CoV-2 is present.
When testing for RNA viruses like SARS-CoV-2, pre-processing is necessary in order to create DNA from the RNA sample.
The DNA can then be amplified with PCR.
Some tests use the results of the PCR itself to determine whether the pathogen is present, but in other cases, it may be necessary to sequence the amplified DNA.
For sequencing, an additional pre-processing step, library preparation, therefore must be undertaken.
Library preparation is the process of preparing the sample for sequencing, typically by fragmenting the sequences and adding adapters [@doi:10.1016/j.biotechadv.2020.107537].
In some cases, library preparation can involve other modifications of the sample, such as adding barcodes to identify a particular sample within the sequence data.
Barcoding can therefore be used to pool samples from multiple sources.
There are different reagents used for library preparation that are specific to identifying one or more target sections with PCR [@doi:10.1021/acsnano.0c02624].
Sequential pattern matching is then used to identify unique subsequences of the virus, and if sufficient subsequences are found, the test is considered positive.
Therefore, tests that utilize sequencing require a number of additional molecular and analytical steps relative to tests that use PCR alone.
<!-- To Do: What is the advantage of the sequencing-based tests?-->

#### RT-PCR

<!-- To Do: Check that all of these RT-PCR definitions are consistent, they seem to vary a little across sections... real time or reverse transcriptase?-->
Real-time polymerase chain reaction (RT-PCR) tests determine whether a target is present by measuring the rate of amplification during PCR compared to a standard.
When the target is RNA, such as in the case of RNA viruses, the RNA must be converted into complementary DNA during pre-processing.
The first test developed and validated for the detection of SARS-CoV-2 uses RT-PCR with reverse transcription [@doi:10.2807/1560-7917.ES.2020.25.3.2000045] to detect several regions of the viral genome: the _ORF1b_ of the RNA-dependent RNA polymerase (RdRP), the Envelope protein gene (_E_), and the Nucleocapsid protein gene (_N_).
The publication reporting this text was released on January 23, 2020, less than two weeks after the sequence of the virus was first reported [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
In collaboration with several other labs in Europe and in China, the researchers confirmed the specificity of this test with respect to other coronaviruses against specimens from 297 patients infected with a broad range of respiratory agents.
Specifically this test utilizes two probes against RdRP, one of which is specific to SARS-CoV-2 [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
Importantly, this assay was not found to return false positive results.
<!-- To Do: How has this changed in the past year? Are we worried about viral evolution? Mention this approach is specific to SARS-CoV-2 rather than other HCoV-->

#### RT-qPCR

Another test was also announced during January 2020.
Chinese researchers developed a reverse transcriptase quantitative real-time PCR (RT-qPCR) test to identify two gene regions of the viral genome, _ORF1b_ and _N_ [@doi:10.1093/clinchem/hvaa029].
<!-- To Do: how does RT-qPCR differ from RT-PCR?-->
This assay was tested on samples from two COVID-19 patients and a panel of positive and negative controls consisting of RNA extracted from several cultured viruses.
The assay uses the _N_ gene to screen patients, while the _ORF1b_ gene region is used to confirm the infection [@doi:10.1093/clinchem/hvaa029].
In this case the test was designed to detect sequences conserved across sarbecoviruses, or viruses within the same subgenus as SARS-CoV-2.
Considering that SARS-CoV-1 and SARS-CoV-2 are the only sarbecoviruses currently known to infect humans, a positive test can be assumed to indicate that the patient is infected with SARS-CoV-2.
However, this test is not able to discriminate the genetics of viruses within the sarbecovirus clade.
<!-- To Do: this means it's probably super robust to any mutations though?-->

##### dPCR

Digital PCR (dPCR) is a new generation of PCR technologies offering an alternative to traditional real-time quantitative PCR.
In dPCR, a sample is partitioned into thousands of compartments, such as nanodroplets (droplet dPCR or ddPCR) or nanowells, and a PCR reaction takes place in each compartment.
This design allows for a digital read-out where each partition is either positive or negative for the nucleic acid sequence being tested for, allowing for much higher throughput.
While dPCR equipment is not yet as common as that for RT-PCR, dPCR for DNA targets generally achieves higher sensitivity than other PCR technologies while maintaining high specificity, though sensitivity is slightly lower for RNA targets [@doi:10.3390/s18041271].
High sensitivity is particularly relevant for SARS-CoV-2 detection, since low viral load in clinical samples can lead to false negatives.
Suo et al. [@doi:10.1080/22221751.2020.1772678] performed a double-blind evaluation of ddPCR for SARS-CoV-2 detection on 57 samples, comprised by 43 samples from suspected positive patients and 14 from supposed convalescents, that had all tested negative for SARS-CoV-2 using RT-PCR.
Despite the initial negative results, 33 out of 35 (94.3%) patients were later clinically confirmed positive.
All of these individuals tested positive using ddPCR.
Additionally, of 14 supposed convalescents who had received two consecutive negative RT-PCR tests, nine (64.2%) tested positive for SARS-CoV-2 using ddPCR.
Two symptomatic patients tested negative with both RT-PCR and ddPCR, but were later clinically diagnosed positive, and 5 of the 14 suspected convalescents tested negative by ddPCR.
While this study did not provide a complete head-to-head comparison to RT-PCR in all aspects, e.g., no samples testing positive using RT-PCR were evaluated by ddPCR, the study shows the potential of dPCR for viral detection even in highly diluted samples.

A second study [@doi:10.1016/j.talanta.2020.121726] confirmed that RT-ddPCR is able to detect SARS-CoV-2 at a lower threshold for viral load relative to RT-PCR.
This study analyzed 196 samples, including 103 samples from suspected patients, 77 from contacts and close contacts, and 16 from suspected convalescents, using both RT-qPCR and RT-ddPCR.
Of the 103 suspected patient samples, RT-qPCR identified 29 as positive, 25 as negative, and 49 as suspected.
Of the RT-qPCR negative or suspected samples, a total of 61 (19 negative and 42 suspected) were confirmed to be positive by RT-ddPCR.
All 103 of the suspected patients were later confirmed to be SARS-CoV-2 positive through a combination of symptom development and RT-qPCR resampling, indicating that RT-ddPCR improved the overall detection rate among these patients from 28.2% to 87.4%.
Of 77 patient samples from contacts and close contacts, 48 tested negative with both methods, and these patients were observed to remain healthy over a period of 14 days.
Within the remaining 29 patient samples, 12 tested positive, 1 negative, and 16 suspected with RT-qPCR.
Fifteen out of 16 suspected results and the negative result were overturned by positive RT-ddPCR results, decreasing the rate of suspected cases from 21% to 1%.
All 16 patients identified as positive by RT-ddPCR were subsequently determined, both clinically and through repeated sampling, to be positive for SARS-CoV-2.
Importantly, all samples that tested positive using RT-qPCR also tested positive using ddPCR.
Among the 16 convalescent patients, RT-qPCR identified 12 as positive, three as suspect, and one as negative, but RT-dPCR identified all 16 as positive.
This evidence further indicates that the lower limit of detection made possible by ddPCR may be useful for identifying when COVID-19 patients are cleared of the virus.
Overall, these studies suggest that ddPCR is a promising tool for overcoming the problem of false-negative SARS-CoV-2 testing.
<!-- To Do: Should we mention the differences in viral load threshhold for these two methods? We could tie this in to how we don't actually know how the viral load relates to infectiousness-->

#### Pooled and Automated PCR Testing

Due to limited supplies and the need for more tests, several labs have found ways to pool or otherwise strategically design tests to increase throughput.
The first such result came from Yelin et al. [@doi:10.1101/2020.03.26.20039438], who found they could pool up to 32 samples in a single qPCR run.
This was followed by larger-scale pooling with slightly different methods [@doi:10.1101/2020.04.02.022186].
Although these approaches are also PCR based, they allow for more rapid scaling and higher efficiency for testing than the initial PCR-based methods developed.
Technology based on CRISPR (clustered regularly interspaced short palindromic repeats) has also been instrumental in scaling up testing protocols.
<!--To Do: Expand this section to discuss implementation of this strategy using PCR?-->

##### CRISPR-based Detection

Two CRISPR-associated nucleases, Cas12 and Cas13, have been used for nucleic acid detection.
Multiple assays exploiting these nucleases have emerged as potential diagnostic tools for the rapid detection of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection.
The SHERLOCK method (Specific High-sensitivity Enzymatic Reporter unLOCKing) from Sherlock Biosciences relies on Cas13a to discriminate between inputs that differ by a single nucleotide at very low concentrations [@doi:10.1126/science.aam9321].
The target RNA is amplified by RT-RPA and T7 transcription, and the amplified product activates Cas13a.
The nuclease then cleaves a reporter RNA, which liberates a fluorescent dye from a quencher.
Several groups have used the SHERLOCK method to detect SARS-CoV-2 viral RNA.
An early study reported that the method could detect 7.5 copies of viral RNA in all 10 replicates, 2.5 copies in 6 out of 10, and 1.25 copies in 2 out of 10 runs [@doi:10.1101/2020.02.22.20025460].
It also reported 100% specificity and sensitivity on 114 RNA samples from clinical respiratory samples (61 suspected cases, among which 52 were confirmed and nine were ruled out by metagenomic next-generation sequencing, 17 nCoV-/HCoV+ cases and 36 samples from healthy subjects), and a reaction turnaround time of 40 minutes.
A separate study screened four designs of SHERLOCK and extensively tested the best-performing assay.
They determined the limit of detection to be 10 copies/&mu;l using both fluorescent and lateral flow detection [@doi:10.1101/2020.02.26.967026].
Lateral flow test strips are simple to use and read, but there are limitations in terms of availability and cost per test.
Another group therefore proposed the CREST protocol (Cas13-based, Rugged, Equitable, Scalable Testing), which uses a P51 cardboard fluorescence visualizer powered by a 9-volt battery, for the detection of Cas13 activity instead of immunochromatography [@doi:10.1101/2020.04.20.052159].
CREST can be run from RNA sample to result in approximately 2 hours, with no need for AC power or a dedicated facility and with minimal handling.
Testing was performed on 14 nasopharyngeal swabs.
CREST picked up the same positives as the CDC-recommended TaqMan assay with the exception of one borderline sample that displayed low-quality RNA.
This approach may therefore represent a rapid, accurate, and affordable procedure for detecting SARS-CoV-2.

The DETECTR method (DNA Endonuclease-Targeted CRISPR Trans Reporter) from Mammoth Biosciences involves purification of RNA extracted from patient specimens, amplification of extracted RNAs by loop-mediated amplification, which is a rapid, isothermal nucleic acid amplification technique, and application of their Cas12-based technology.
In this assay, guide RNAs (gRNAs) were designed to recognize portions of sequences corresponding to the SARS-CoV-2 genome, specifically the N2 and E regions [@doi:10.1038/s41587-020-0513-4].
In the presence of SARS-CoV-2 genetic material, sequence recognition by the gRNAs results in double-stranded DNA cleavage by Cas12, as well as cleavage of a single-stranded DNA molecular beacon.
The cleavage of this molecular beacon acts as a colorimetric reporter that is subsequently read out in a lateral flow assay and indicates the positive presence of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection.
The 40-minute assay is considered positive if there is detection of both the E and N genes or presumptive positive if there is detection of either of them.
The assay had 95% positive predictive agreement and 100% negative predictive agreement with the US Centers for Disease Control and Prevention SARS-CoV-2 real-time RT–PCR assay.
The estimated limit of detection was 10 copies per &mu;l reaction, versus 1 copy per &mu;l reaction for the CDC assay.
These results have been confirmed by other DETECTR approaches.
Using real-time recombinase polymerase amplification (RT-RPA) for amplification, another group detected 10 copies of synthetic SARS-CoV-2 RNA per &mu;l of input within 60 minutes of RNA sample preparation in a proof-of-principle evaluation [@doi:10.1101/2020.02.29.971127].
The DETECTR protocol was improved by combining RT-RPA and CRISPR-based detection in a one-pot reaction that incubates at a single temperature, and by using dual CRISPR RNAs (which increases sensitivity).
This new assay, known as All-In-One Dual CRISPR-Cas12a (AIOD-CRISPR), detected 4.6 copies of SARS-CoV-2 RNA per &mu;l of input in 40 minutes [@doi:10.1101/2020.03.19.998724].
Another single-tube, constant-temperature approach using Cas12b instead of Cas12a achieved a detection limit of 5 copies/&mu;l in 40-60 minutes [@doi:10.1101/2020.04.10.023358].
It was also reported that electric field gradients can be used to control and accelerate CRISPR assays by co-focusing Cas12-gRNA, reporters, and target [@doi:10.1101/2020.05.21.109637].
The authors generated an appropriate electric field gradient using a selective ionic focusing technique known as isotachophoresis (ITP) implemented on a microfluidic chip.
They also used ITP for automated purification of target RNA from raw nasopharyngeal swab samples.
Combining this ITP purification with loop-mediated isothermal amplification, their ITP-enhanced assay to achieved detection of SARS-CoV-2 RNA (from raw sample to result) in 30 minutes.

There is an increasing body of evidence that CRISPR-based assays offer a practical solution for rapid, low-barrier testing in areas that are at greater risk of infection, such as airports and local community hospitals.
In the largest study to date, DETECTR was compared to RT-qPCR on 378 patient samples [@doi:10.1101/2020.07.27.20147249].
The authors reported a 95% reproducibility.
Both techniques were equally sensitive in detecting SARS-CoV-2.
Lateral flow strips showed a 100% correlation to the high-throughput DETECTR assay.
Importantly, DETECTR was 100% specific for SARS-CoV-2 and did not detect other human coronaviruses.

#### Limitations of Molecular Tests

Tests that identify SARS-CoV-2 using nucleic-acid-based technologies will identify only individuals with current infections and are not appropriate for identifying individuals who have recovered from a previous infection.
Within this category, different types of tests have different limitations.
For example, PCR-based test can be highly sensitive, but in high-throughput settings they can show several problems:
First of all, there is a risk of false-negative responses, which can present a significant problem to large-scale testing.
To reduce occurrence of false negatives, correct execution of the analysis is crucial [@doi:10.1038/d41587-020-00002-2].
Additionally, the emerging nature of the COVID-19 pandemic has introduced some challenges related to Uncertainty surrounding interactions between SARS-CoV-2 and its human hosts.
For example, viral shedding kinetics are still not well understood, but are expected to introduce a significant effect of timing of sample collection on test results [@doi:10.1038/d41587-020-00002-2].
Similarly, the type of specimen could also influence outcomes, as it is not clear which clinical samples are best for detecting the virus [@doi:10.1038/d41587-020-00002-2].
There are also significant practical and logistical concerns.
Much of the technology used for molecular tests is expensive, and while it might be available in major hospitals and/or diagnostic centers, it is often not available to smaller facilities [@doi:10.1126/science.abb8400].
At times during the first year of the pandemic, the availability of supplies for testing, including swabs and testing media, has also been limited [@doi:10.1128/JCM.00512-20].
Similarly, processing times can be long, and tests might take up to 4 days to return results [@doi:10.1126/science.abb8400].
Finally, with CRISPR-based testing strategies, the gRNA can recognize other interspersed sequences on the patient’s genome, false positives and a loss of specificity can occur.
As noted above, false negatives are a significant concern for several reason.
Importantly, clinical reports indicate that it is imperative to exercise caution when interpreting the results of molecular tests for SARS-CoV-2 because negative results do not necessarily mean a patient is virus-free [@doi:10.1128/JCM.00297-20].

### Serological Tests

Although diagnostic tests based on the detection of genetic material can be quite sensitive, they provide information only about active infection, and therefore offer just a snapshot-in-time perspective on the spread of a disease.
Most importantly, they would not work on a patient who has fully recovered from the virus at the time of sample collection.
In this context, serological tests, which use serum to test for the presence of antibodies against SARS-CoV-2, are significantly more informative.
Serological tests can provide insight into population-level dynamics and can also offer a glimpse into the development of antibodies by individual patients during the course of a disease.
Therefore, they can be useful to developing strategies for the management of viral spread.
Furthermore, serological tests hold significant interest because of the possibility that they could provide information relevant to advancing economic recovery and allowing reopenings.
For instance, early in the course of the COVID-19 pandemic, it was hypothesized that people who had developed antibodies might be able to return to work [@doi:10.1101/2020.04.14.20065771], although this strategy would have relied on recovered individuals acquiring long-term immunity.
Some infectious agents can be controlled through "herd immunity", which is when a critical mass within the population acquires immunity through vaccination and/or infection, preventing an infectious agent from spreading widely.
<!--To Do: Because we split out the pathogenesis/transmission section, we need to redefine these concepts-->
A simple SIR model predicts that to achieve the required level of exposure for herd immunity to be effective, at least (1-(1/R~0~)) fraction of the population must be immune or, equivalently, less than (1/R~0~) fraction of the population susceptible [@isbn:9780199209996].
However, for SARS-CoV-2 and COVID-19, the R~0~ and mortality rates that have been observed suggest that relying on herd immunity without some combination of vaccines, proven treatment options, and strong non-pharmaceutical measures of prevention and control would likely result in a significant loss of life.<!--To Do: citation? Scandinavian international comparisons?-->

<~--Include some of this info somewhere?
Understanding the fundamental organization of the human immune response to viral threats is critical to understanding the varied response to SARS-CoV-2.
The human immune system utilizes a variety of innate and adaptive responses to protect against the pathogens it encounters.
The innate immune system consists of barriers, such as the skin, mucous secretions, neutrophils, macrophages, and dendritic cells.
It also includes cell-surface receptors that can recognize the molecular patterns of pathogens.
The adaptive immune system utilizes antigen-specific receptors that are expressed on B and T lymphocytes.
These components of the immune system typically act together; the innate response acts first, and the adaptive response begins to act several days after initial infection following the clonal expansion of T and B cells [@doi:10.1016/j.jaci.2005.09.034].
After a virus enters into a host cell, its antigen is presented by major histocompatibility complex 1 (MHC 1) molecules and is then recognized by cytotoxic T lymphocytes.
-->

#### Sustained Immunity to COVID-19

In the process of mounting a response to a pathogen, the immune system produces antibodies specific to the pathogen.
Understanding the acquisition and retention of antibodies is important both to the diagnosis of prior (inactive) infections and to the development of vaccines.
The two immunoglobulin classes most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies found in the blood.
Prior research is available about the development of antibodies to _Severe acute respiratory syndrome-related coronavirus 1_ (SARS-CoV-1) during the course of the associated disease, severe acute respiratory syndrome (SARS).
Following SARS infection, IgM and IgG antibodies were detected in the second week post-infection.
IgM titers peaked by the first month post-infection, and then declined to undetectable levels after day 180.
IgG titers peaked by day 60, and persisted in all donors through the two-year duration of study [@doi:10.1111/j.1440-1843.2006.00783.x].
A two-year longitudinal study following convalesced SARS patients with a mean age of 29 found that IgG antibodies were detectable in all 56 patients surveyed for at least 16 months, and remained detectable in all but 4 (11.8%) of patients through the full two-year study period [@doi:10.1086/500469].
These results suggest that immunity to SARS-CoV-1 is sustained for at least a year.

The persistence of antibodies to SARS-CoV-2 remains under investigation.
Circulating antibody titers to other coronaviruses have been reported to decline significantly after 1 year [@doi:10.1017/s0950268800048019].
Autopsies of lymph nodes and spleens from severe acute COVID-19 patients showed a loss of T follicular helper cells and germinal centers that may explain some of the impaired development of antibody responses [@doi:10.1016/j.cell.2020.08.025].
An early study (initially released on _medRxiv_ on February 25, 2020) presented a chemiluminescence immunoassay to a synthetic peptide derived from the amino acid sequence of the SARS-CoV-2 _S_ protein [@doi:10.1093/infdis/jiaa243].
This method was highly specific to SARS-CoV-2 and detected IgM in 57.2% and IgG in 71.4% and 57.2% of sera samples from 276 confirmed COVID-19 patients.
It reported that IgG could be detected within two days of the onset of fever but that IgM could not be detected any earlier, a pattern they compared to findings in another disease caused by a HCoV, Middle East respiratory syndrome (MERS).
Since then, several trials have reported the potential protective effect of antibodies in convalescent plasma obtained from recovered COVID-19 patients to treat critically ill COVID-19 patients [@doi:10.1172/jci138745; @doi:10.1001/jama.2020.4783; @doi:10.1056/NEJMoa2031893].

Evidence to date suggests that sustained immunity to the SARS-CoV-2 virus remains for a period of at least 6 to 8 months [@doi:10.3201/eid2703.204543; @doi:10.1126/science.abf4063; @pmid:33443036; @doi:10.1016/j.medj.2021.02.001].
Dan et al. assessed sustained immunity using 254 blood samples from 188 COVID-19 positive patients [@doi:10.1126/science.abf4063]. 
The samples were collected at various time points between 6 and 240 days post-symptom onset, meaning some patients were assessed longitudinally. 
Of the samples, 43 were collected at least 6 months after symptom onset.
After 1 month, 98% of patients were seropositive for IgG to the spike protein, S. 
Moreover, S IgG titers were stable and heterogeneous among patients over a period of 6 to 8 months post-symptom onset, with 90% of subjects seropositive at 6 months.
Similarly, at 6 to 8 months 88% of patients were seropositive for receptor binding domain (RBD) IgG and 90% were seropositive for SARS-CoV-2 neutralizing antibodies.

The findings of Dan et al. are in accordance with a study by Sherina et al. that examined 119 samples from 88 donors who had recovered from mild to severe cases of COVID-19 [@doi:10.1016/j.medj.2021.02.001]. 
They observed a relatively stable level of IgG and plasma neutralizing antibodies up to 6 months post diagnosis.
Significantly lower but considerable levels of anti-SARS-CoV-2 IgG antibodies were still present in 80% of samples obtained 6-8 months post symptom-onset.
A study by Gaebler et al. also found that titers of IgM and IgG antibodies against the RBD decreased from 1.3 to 6.2 months post infection in a study of 87 individuals [@doi:10.1038/s41586-021-03207-w]. 
However, the decline of IgA (15%) activity was less pronounced than that of IgM (53%) or IgG (32%).
It was noted that higher levels of anti-RBD IgG and anti-N total antibodies were detected in individuals that reported persistent post-acute symptoms at both study visits.<!--To Do: confirm we define N in this paper-->
Moreover, plasma neutralizing activity decreased five-fold between 1.3 and 6.2 months in an assay of HIV-1 virus pseudotyped with SARS-CoV-2 S protein, and this neutralizing activity was directly correlated with IgG anti-RBD titers [@doi:10.1038/s41586-021-03207-w].
These findings are in accordance with other studies that show that the majority of seroconverters have detectable, albeit decreasing, levels of neutralizing antibodies at least 3-6 months post infection [@doi:10.1126/science.abd7728; @doi:10.1038/s41564-020-00813-8; @doi:10.1038/s41467-021-21444-5].
Determining the potency of anti-RBD antibodies early in the course of an infection may be important moving forward, as their neutralizing potency may be prognostic for disease severity and survival [@doi:10.1016/j.cell.2020.12.015].

The development of memory B cells and memory T cells has also been assessed in several studies.
Dan et al. showed that SARS-CoV-2 S-specific memory B cell levels rose steadily over the first 120 days following symptom onset [@doi:10.1126/science.abf4063].
RBD-specific memory B cells had been detected in COVID-19 patients 90 days post-symptom onset in previous studies [@doi:10.1016/j.cell.2020.11.029; @pmid:33443036], and this study confirms these finding and shows that levels of these cells increased over the 4-5 months post-symptom onset [@doi:10.1126/science.abf4063].
Gaebler et al. have shown that memory B cells specific to the RBD remain unaltered and exhibit clonal turnover and antibody sequence evolution 6 months post infection, indicative of prolonged germinal cell reactions [@doi:10.1038/s41586-021-03207-w]. 
The same study showed that antibodies expressed by these memory B cells have resistance to RBD mutations, greater somatic hypermutations, and increased potency, which the authors suggest might be evidence of continued evolution of humoral immunity [@doi:10.1038/s41586-021-03207-w].
Indeed, Wheatley et al. showed that S-specific IgG^+^ memory B cells consistently increase over time and by 4 months comprise approximately 0.8% of all IgG^+^ memory B cells, which may indicate cellular immune memory to even mild-to-moderate COVID-19 infection [@doi:10.1038/s41467-021-21444-5]. 
Dan et al. showed that N-specific memory B cells steadily increased up to 4-5 months post-symptom onset [@doi:10.1126/science.abf4063].
SARS-CoV-2 memory CD8^+^ T cells were also detected in 70% of 169 COVID-19 patients after 1 month [@doi:10.1126/science.abf4063], which is consistent with previous research [@doi:10.1016/j.cell.2020.05.015].
However, SARS-CoV-2 memory CD8^+^ T cells were slightly decreased (50%) 6 months post-symptom onset. 
In this same subset of COVID-19 patients, 93% of subjects had detectable levels of SARS-CoV-2 memory CD4^+^ T cells, of which 42% had more than 1% SARS-CoV-2-specific CD4^+^ T cells.
At 6 months, 92% of patients were positive for SARS-CoV-2 memory CD4^+^ T cells.
Indeed, the abundance of S-specific memory CD4^+^ T cells over time was similar to that of SARS-CoV-2-specific CD4^+^ T cells overall [@doi:10.1126/science.abf4063].
T cell immunity to SARS-CoV-2 at 6 to 8 months following symptom onset has also been confirmed by other studies [@doi:10.1101/2020.11.01.362319; @doi:10.1101/2020.12.08.416636; @doi:10.1016/j.medj.2021.02.001].
In another study, T cell reactivity to SARS-CoV-2 epitopes was also detected in some individuals never been exposed to SARS-CoV-2. 
This finding suggests the potential for cross-reactive T cell recognition between SARS-CoV-2 and pre-existing circulating HCoV that are responsible for the “common cold” [@doi:10.1016/j.cell.2020.05.015], but further research is required.

Concerns have also been raised that immunity may wane over time.
Several reported cases of reinfection have been confirmed via genomic analysis that revealed distinct variants of SARS-CoV-2 within a single patient [@doi:10/ghfgkt; @doi:10.1093/cid/ciaa1451; @doi:10.1093/cid/ciaa1275].
Further research is required to determine the full extent to which sustained immunity can be and is typically achieved following SARS-CoV-2 infection.
Furthermore, it is unclear whether previous infection may carry repercussions in terms of disease severity for patients [@doi:10/fscx] and what implications the possibility of reinfection holds for vaccine development and the long-term efficacy of vaccines [@doi:10.1038/s41591-020-1121-z; @doi:10.3389/fimmu.2020.571481].
<!--To Do: add https://www.nature.com/articles/s41591-020-0965-6#Sec9 and evalute reports of recurrence.-->

#### Current Approaches

<!-- To Do: Update for 2021-->
Several countries are now focused on implementing antibody tests, and in the United States, the FDA recently approved a serological test by Cellex for use under emergency conditions [@url:https://www.fda.gov/media/136625/download].
Specifically, the Cellex qSARS-CoV-2 IgG/IgM Rapid Test is a chromatographic immunoassay designed to qualitatively detect IgM and IgG antibodies against SARS-CoV-2 in the plasma (from a blood sample) of patients suspected to have developed the SARS-CoV-2 infection [@url:https://www.fda.gov/media/136625/download].
Such tests illuminate the progression of viral disease, as IgM are the first antibodies produced by the body and indicate that the infection is active.
Once the body has responded to the infection, IgG are produced and gradually replace IgM, indicating that the body has developed immunogenic memory [@doi:10.1002/jmv.25820].
The Cellex test cassette contains a pad of SARS-CoV-2 antigens and a nitrocellulose strip with lines for each of IgG and IgM, as well as a control (goat IgG) [@url:https://www.fda.gov/media/136625/download].
In a specimen that contains antibodies against the SARS-CoV-2 antigen, the antibodies will bind to the strip and be captured by the IgM and/or IgG line(s), resulting in a change of color [@url:https://www.fda.gov/media/136625/download].
With this particular assay, results can be read within 15 to 20 minutes [@url:https://www.fda.gov/media/136625/download].
Other research groups, such as the Krammer lab of the Icahn School of Medicine at Mount Sinai, proposed an ELISA test that detects IgG and IgM that react against the RBD of the spike proteins (S) of the virus [@doi:10.1101/2020.03.17.20037713].
The authors are now working to get the assay into clinical use [@url:https://www.livescience.com/coronavirus-tests-available.html].

#### Limitations of Serological Tests

Like molecular tests, serological tests carry a number of limitations that influence their utility in different situations.
Importantly, false positives can occur due to cross-reactivity with other antibodies according to the clinical condition of the patient [@url:https://www.fda.gov/media/136625/download].
Therefore, such tests must be used in combination with RNA detection tests if intended for diagnostic purposes, and while serological tests may be of interest to individuals who wish to confirm they were infected with SARS-CoV-2 in the past, their potential for false positives means that they are not currently recommended for this use.
<!-- To Do: These are also single-throughput (can't be multiplexed), or can they also be high-throughput??-->
Due to the long incubation times and delayed immune responses of infected patients, serological tests are insufficiently sensitive for a diagnosis in the early stages of an infection.
The limitations due to timing make serological tests far less useful for enabling test-and-trace strategies.
<!--To Do: Add a sentence explaining the utility of serosurveys?-->

### Possible Alternatives to Current Practices for Identifying Active Cases

COVID-19 can present with symptoms similar to other types of pneumonia, and symptoms can vary widely among COVID-19 patients; therefore, clinical presentation is often insufficient as a sole diagnostic criterion.
In addition, identifying and isolating mild or asymptomatic cases is critical to efforts to manage outbreaks.
Even among mildly symptomatic patients, a predictive model based on clinical symptoms had a sensitivity of only 56% and a specificity of 91% [@doi:10.2807/1560-7917.ES.2020.25.16.2000508].
More problematic is that clinical symptom-based tests are only able to identify already symptomatic cases, not presymptomatic or asymptomatic cases.
They may still be important for clinical practice, and for reducing tests needed for patients deemed unlikely to have COVID-19.

Similarly, X-ray diagnostics have been reported to have high sensitivity but low specificity in some studies [@doi:10.1148/radiol.2020200642].
Other studies have shown that specificity varies between radiologists [@doi:10.1148/radiol.2020200823], though the sensitivity reported here was lower than that published in the previous paper.
However, preliminary machine-learning results have shown far higher sensitivity and specificity from analyzing chest X-rays than was possible with clinical examination [@doi:10.1007/s13246-020-00865-4].
X-ray tests with machine learning can potentially detect asymptomatic or presymptomatic infections that show lung manifestations.
This approach would still not recognize entirely asymptomatic cases.
Given the above, the widespread use of X-ray tests on otherwise healthy adults is likely inadvisable.

### Strategies and Considerations for Determining Whom to Test

Early in the COVID-19 pandemic, testing was typically limited to individuals considered high risk for developing serious illness [@url:https://www.cdc.gov/coronavirus/2019-nCoV/hcp/clinical-criteria.html].
This approach often involved limiting testing to people with severe symptoms and people showing mild symptoms that had been in contact with a person who had tested positive.
Individuals who were asymptomatic (i.e., potential spreaders) and individuals who were able to recover at home were therefore often unaware of their status.
However, this method of testing administration misses a high proportion of infections and does not allow for test-and-trace methods to be used.
For instance, a recent study from Imperial College estimates that in Italy, the true number of infections was around 5.9 million in a total population ~60 million, compared to the 70,000 detected as of March 28th [@doi:10.25561/77731].
Another analysis, which examined New York state, indicated that as of May 2020, approximately 300,000 cases had been reported in a total population of approximately 20 million [@url:https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf].
This corresponded to ~1.5% of the population, but ~12% of individuals sampled statewide were estimated as positive through antibody tests (along with indications of spatial heterogeneity at higher resolution) [@url:https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf].
Technological advancements that facilitate widespread, rapid testing will therefore improve the potential to accurately assess the rate of infection and aid in controlling the virus' spread.
<!-- To Do: update this section because there have plenty of efforts to do this in the past 12 months, e.g., countries, universities, etc.-->

### Conclusions

<!-- To Do: This section has less discussion than other sections, not sure if we want to try to expand it?
Potentially mention that there are longitudinal studies on how long sustained immunity lasts, but time makes it easier to get an idea of the distribution without much time elapsing-->
Major advancements have been made in identifying diagnostic approaches.
The development of diagnostic technologies have been rapid, beginning with the release of the SARS-CoV-2 viral genome sequence in January.
As of October 2020, a range of diagnostic tests have become available.
One class of tests uses PCR (RT-PCR or RT-qPCR) to assess the presence of SARS-CoV-2 RNA, while another typically uses ELISA to test for the presence of antibodies to SARS-CoV-2.
The former approach is useful for identifying active infections, while the latter measures hallmarks of the immune response and therefore can detect either active infections or immunity gained from prior infection.
Combining these tests leads to extremely accurate detection of SARS-CoV-2 infection (98.6%), but when used alone, PCR-based tests are recommended before 5.5 days after the onset of the illness and antibody tests after 5.5 days [@doi:10.1001/jama.2020.8259].
Other strategies for testing can also influence the tests' accuracy, such as the use of nasopharyngeal swabs versus bronchoalveolar lavage fluid [@doi:10.1001/jama.2020.8259], which allow for trade-offs between patient's comfort and test sensitivity.
Additionally, technologies such as digital PCR may allow for scale-up in the throughput of diagnostic testing, facilitating widespread testing.
One major question that remains is whether people who recover from SARS-CoV-2 develop sustained immunity, and over what period this immunity is expected to last.
Some reports have suggested that some patients may develop COVID-19 reinfections (e.g., [@doi:10/ghfgkt]), but the rates of reinfection are currently unknown.
Serologic testing combined with PCR testing will be critical to confirming purported cases of reinfection and to identifying the duration over which immunity is retained and to understanding reinfection risks.


## Identification and Development of Therapeutics for COVID-19

### Abstract

After emerging in China in late 2019, the novel coronavirus SARS-CoV-2 spread worldwide and as of mid-2021 remains a significant threat globally.
Only a few coronaviruses are known to infect humans, and only two cause infections similar in severity to SARS-CoV-2: _Severe acute respiratory syndrome-related coronavirus_, a closely related species of SARS-CoV-2 that emerged in 2002, and _Middle East respiratory syndrome-related coronavirus_, which emerged in 2012.
Unlike the current pandemic, previous epidemics were controlled rapidly through public health measures, but the body of research investigating severe acute respiratory syndrome and Middle East respiratory syndrome has proven valuable for identifying approaches to treating and preventing novel coronavirus disease 2019 (COVID-19).
Building on this research, the medical and scientific communities have responded rapidly to the COVID-19 crisis to identify many candidate therapeutics.
The approaches used to identify candidates fall into four main categories: adaptation of clinical approaches to diseases with related pathologies, adaptation based on virological properties, adaptation based on host response, and data-driven identification of candidates based on physical properties or on pharmacological compendia.
To date, a small number of therapeutics have already been authorized by regulatory agencies such as the Food and Drug Administration (FDA), while most remain under investigation.
The scale of the COVID-19 crisis offers a rare opportunity to collect data on the effects of candidate therapeutics.
This information provides insight not only into the management of coronavirus diseases, but also into the relative success of different approaches to identifying candidate therapeutics against an emerging disease.

### Importance

The COVID-19 pandemic is a rapidly evolving crisis.
With the worldwide scientific community shifting focus onto the SARS-CoV-2 virus and COVID-19, a large number of possible pharmaceutical approaches for treatment and prevention have been proposed.
What was known about each of these potential interventions evolved rapidly throughout 2020 and 2021.
This fast-paced area of research provides important insight into how the ongoing pandemic can be managed and also demonstrates the power of interdisciplinary collaboration to rapidly understand a virus and match its characteristics with existing or novel pharmaceuticals.
As illustrated by the continued threat of viral epidemics during the current millennium, a rapid and strategic response to emerging viral threats can save lives.
In this review, we explore how different modes of identifying candidate therapeutics have borne out during COVID-19.

### Introduction

The novel coronavirus _Severe acute respiratory syndrome-related coronavirus 2_ (SARS-CoV-2) emerged in late 2019 and quickly precipitated the worldwide spread of novel coronavirus disease 2019 (COVID-19).
COVID-19 is associated with symptoms ranging from mild or even asymptomatic to severe, and up to 2% of patients diagnosed with COVID-19 die from COVID-19-related complications such as acute respiratory disease syndrome (ARDS) [@individual-pathogenesis].
As a result, public health efforts have been critical to mitigating the spread of the virus.
However, as of mid-2021, COVID-19 remains a significant worldwide concern (Figure @fig:csse-deaths), with 2021 cases in some regions surging far above the numbers reported during the initial outbreak in early 2020.
While a number of vaccines have been developed and approved in different countries starting in late 2020 [@individual-vaccines], vaccination efforts have not proceeded at the same pace throughout the world and are not yet close to ending the pandemic.

Due to the continued threat of the virus and the severity of the disease, the identification and development of therapeutic interventions have emerged as significant international priorities.
Prior developments during other recent outbreaks of emerging diseases, especially those caused by human coronaviruses (HCoV), have guided biomedical research into the behavior and treatment of this novel coronavirus infection.
However, previous emerging HCoV-related disease threats were controlled much more quickly than SARS-CoV-2 through public health efforts (Figure @fig:csse-deaths).
The scale of the COVID-19 pandemic has made the repurposing and development of pharmaceuticals more urgent than in previous coronavirus epidemics.

#### Lessons from Prior HCoV Outbreaks

![
**Cumulative global incidence of COVID-19 and SARS.**
As of September 25, 2021, 231,515,976 COVID-19 cases and 4,743,196 COVID-19 deaths had been reported worldwide since January 22, 2020.
A total of 8,432 cases and 813 deaths were reported for SARS from March 17 to July 11, 2003.
SARS-CoV-1 was officially contained on July 5, 2003, within 9 months of its appearance [@doi:10.3390/ijerph17113973].
In contrast, SARS-CoV-2 remains a significant global threat nearly two years after its emergence.
COVID-19 data are from the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University [@https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series; @doi:10/ggnsjk].
SARS data are from the WHO [@url:https://www.who.int/csr/sars/country/en] and were obtained from a dataset on GitHub [@url:https://github.com/imdevskp/sars-2003-outbreak-data-webscraping-code].
See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily.
](https://github.com/greenelab/covid19-review/raw/0e36a517bc88e54459c2b6ed3a4803e621267045/csse/csse-deaths.png "Global incidence of COVID-19 and SARS"){#fig:csse-deaths secno=1}

At first, SARS-CoV-2's rapid shift from an unknown virus to a significant worldwide threat closely paralleled the emergence of _Severe acute respiratory syndrome-related coronavirus_ (SARS-CoV-1), which was responsible for the 2002-03 SARS epidemic.
The first documented case of COVID-19 was reported in Wuhan, China in November 2019, and the disease quickly spread worldwide in the early months of 2020.
In comparison, the first case of SARS was reported in November 2002 in the Guangdong Province of China, and it spread within China and then into several countries across continents during the first half of 2003 [@doi:10.1093/ajcp/aqaa029; @doi:10.1038/nrmicro.2016.81; @doi:10.3390/ijerph17113973].
In fact, genome sequencing quickly revealed the virus causing COVID-19 to be a novel betacoronavirus closely related to SARS-CoV-1 [@doi:10.46234/ccdcw2020.017].

While similarities between these two viruses are unsurprising given their close phylogenetic relationship, there are also some differences in how the viruses affect humans.
SARS-CoV-1 infection is severe, with an estimated case fatality rate (CFR) for SARS of 9.5% [@doi:10.1093/ajcp/aqaa029], while estimates of the CFR associated with COVID-19 are much lower, at up to 2% [@individual-pathogenesis].
SARS-CoV-1 is highly contagious and spread primarily by droplet transmission, with a basic reproduction number (R~0~) of 4 (i.e., each person infected was estimated to infect four other people) [@doi:10.1093/ajcp/aqaa029].
There is still some controversy whether SARS-CoV-2 is primarily spread by droplets or is primarily airborne [@doi:10.1001/jama.2020.12458; @doi:10/gg6br7; @doi:10/gjqmvq; @doi:10.1136/bmj.n913]. 
Most estimates of its R~0~ fall between 2.5 and 3 [@individual-pathogenesis].
Therefore, SARS is thought to be a deadlier and more transmissible disease than COVID-19.

With the 17-year difference between these two outbreaks, there were major differences in the tools available to efforts to organize international responses.
At the time that SARS-CoV-1 emerged, no new HCoV had been identified in almost 40 years [@doi:10.1038/nrmicro.2016.81].
The identity of the virus underlying the SARS disease remained unknown until April of 2003, when the SARS-CoV-1 virus was characterized through a worldwide scientific effort spearheaded by the World Health Organization (WHO) [@doi:10.1038/nrmicro.2016.81].
In contrast, the SARS-CoV-2 genomic sequence was released on January 3, 2020 [@doi:10.46234/ccdcw2020.017], only days after the international community became aware of the novel pneumonia-like illness now known as COVID-19.
While SARS-CoV-1 belonged to a distinct lineage from the two other HCoVs known at the time of its discovery [@doi:10.1093/ajcp/aqaa029], SARS-CoV-2 is closely related to SARS-CoV-1 and is a more distant relative of another HCoV characterized in 2012, _Middle East respiratory syndrome-related coronavirus_ [@doi:10/ggjr43; @doi:10.1056/NEJMoa1211721].
Significant efforts had been dedicated towards understanding SARS-CoV-1 and MERS-CoV and how they interact with human hosts.
Therefore, SARS-CoV-2 emerged under very different circumstances than SARS-CoV-1 in terms of scientific knowledge about HCoVs and the tools available to characterize them.

Despite the apparent advantages for responding to SARS-CoV-2 infections, COVID-19 has caused many orders of magnitude more deaths than SARS did (Figure @fig:csse-deaths).
The SARS outbreak was officially determined to be under control in July 2003, with the success credited to infection management practices such as mask wearing [@doi:10.1038/nrmicro.2016.81].
_Middle East respiratory syndrome-related coronavirus_ (MERS-CoV) is still circulating and remains a concern; although the fatality rate is very high at almost 35%, the disease is much less easily transmitted, as its R~0~ has been estimated to be 1 [@doi:10.1093/ajcp/aqaa029].
The low R~0~ in combination with public health practices allowed for its spread to be contained [@doi:10.1093/ajcp/aqaa029].
Neither of these trajectories are comparable to that of SARS-CoV-2, which remains a serious threat worldwide over a year and a half after the first cases of COVID-19 emerged (Figure @fig:csse-deaths).

#### Potential Approaches to the Treatment of COVID-19

Therapeutic interventions can utilize two approaches: they can either mitigate the effects of an infection that harms an infected person, or they can hinder the spread of infection within a host by disrupting the viral life cycle.
The goal of the former strategy is to reduce the severity and risks of an active infection, while for the latter, it is to inhibit the replication of a virus once an individual is infected, potentially freezing disease progression.
Additionally, two major approaches can be used to identify interventions that might be relevant to managing an emerging disease or a novel virus: drug repurposing and drug development.
Drug repurposing involves identifying an existing compound that may provide benefits in the context of interest [@doi:10.1038/nrd.2018.168].
This strategy can focus on either approved or investigational drugs, for which there may be applicable preclinical or safety information [@doi:10.1038/nrd.2018.168].
Drug development, on the other hand, provides an opportunity to identify or develop a compound specifically relevant to a particular need, but it is often a lengthy and expensive process characterized by repeated failure [@doi:10.1016/j.trci.2017.10.005].
Drug repurposing therefore tends to be emphasized in a situation like the COVID-19 pandemic due to the potential for a more rapid response.

Even from the early months of the pandemic, studies began releasing results from analyses of approved and investigational drugs in the context of COVID-19.
The rapid timescale of this response meant that, initially, most evidence came from observational studies, which compare groups of patients who did and did not receive a treatment to determine whether it may have had an effect.
This type of study can be conducted rapidly but is subject to confounding.
In contrast, randomized controlled trials (RCTs) are the gold-standard method for assessing the effects of an intervention.
Here, patients are prospectively and randomly assigned to treatment or control conditions, allowing for much stronger interpretations to be drawn; however, data from these trials take much longer to collect.
Both approaches have proven to be important sources of information in the development of a rapid response to the COVID-19 crisis, but as the pandemic draws on and more results become available from RCTs, more definitive answers are becoming available about proposed therapeutics.
Interventional clinical trials are currently investigating or have investigated a large number of possible therapeutics and combinations of therapeutics for the treatment of COVID-19 (Figure @fig:ebm-trials).

![
**COVID-19 clinical trials.**
Trials data are from the University of Oxford Evidence-Based Medicine Data Lab's COVID-19 TrialsTracker [@doi:10.5281/zenodo.3732709].
As of December 31, 2020, there were 6,987 COVID-19 clinical trials of which 3,962 were interventional.
The study types include only types used in at least five trials.
Only interventional trials are analyzed in the figures depicting status, phase, and intervention.
Of the interventional trials, 98 trials had reported results as of December 31, 2020.
Recruitment status and trial phase are shown only for interventional trials in which the status or phase is recorded.
Common interventions refers to interventions used in at least ten trials.
Combinations of interventions, such as hydroxychloroquine with azithromycin, are tallied separately from the individual interventions.
See https://greenelab.github.io/covid19-review/ for the most recent version of this figure, which is updated daily.
](https://github.com/greenelab/covid19-review/raw/0e36a517bc88e54459c2b6ed3a4803e621267045/ebmdatalab/ebmdatalab-trials.png "COVID-19 clinical trials"){#fig:ebm-trials secno=1}

The purpose of this review is to provide an evolving resource tracking the status of efforts to repurpose and develop drugs for the treatment of COVID-19.
We highlight four strategies that provide different paradigms for the identification of potential pharmaceutical treatments.
The WHO guidelines [@doi:10.1136/bmj.m3379] and a systematic review [@doi:10.1136/bmj.m2980] are complementary living documents that summarize COVID-19 therapeutics.


### Repurposing Drugs for Symptom Management

A variety of symptom profiles with a range of severity are associated with COVID-19 [@individual-pathogenesis].
In many cases, COVID-19 is not life threatening.
A study of COVID-19 patients in a hospital in Berlin, Germany reported that the highest risk of death was associated with infection-related symptoms, such as sepsis, respiratory symptoms such as ARDS, and cardiovascular failure or pulmonary embolism [@doi:10.1101/2020.06.15.20131540].
Similarly, an analysis in Wuhan, China reported that respiratory failure (associated with ARDS) and sepsis/multi-organ failure accounted for 69.5% and 28.0% of deaths, respectively, among 82 deceased patients [@doi:10.1371/journal.pone.0235458].
COVID-19 is characterized by two phases.
The first is the acute response, where an adaptive immune response to the virus is established and in many cases can mitigate viral damage to organs [@doi:10.1038/s41418-020-0530-3].
The second phase characterizes more severe cases of COVID-19.
Here, patients experience a cytokine storm, whereby excessive production of cytokines floods into circulation, leading to systemic inflammation, immune dysregulation, and multiorgan dysfunction that can cause multiorgan failure and death if untreated [@doi:10.1056/NEJMra2026131].
ARDS-associated respiratory failure can occur during this phase.
Cytokine dysregulation was also identified in patients with SARS [@doi:10/c8mmbg; @doi:10.1186/cc2452]. 

In the early days of the COVID-19 pandemic, physicians sought to identify potential treatments that could benefit patients, and in some cases shared their experiences and advice with the medical community on social media sites such as Twitter [@doi:10.7759/cureus.13594].
These on-the-ground treatment strategies could later be analyzed retrospectively in observational studies or investigated in an interventional paradigm through RCTs.
Several notable cases involved the use of small-molecule drugs, which are synthesized compounds of low molecular weight, typically less than 1 kilodalton (kDa) [@url:https://www.nuventra.com/resources/blog/small-molecules-versus-biologics].
Small-molecule pharmaceutical agents have been a backbone of drug development since the discovery of penicillin in the early twentieth century [@doi:10.1126/science.287.5460.1960].
It and other antibiotics have long been among the best known applications of small molecules to therapeutics, but biotechnological developments such as the prediction of protein-protein interactions (PPIs) have facilitated advances in precise targeting of specific structures using small molecules [@doi:10.1126/science.287.5460.1960].
Small molecule drugs today encompass a wide range of therapeutics beyond antibiotics, including antivirals, protein inhibitors, and many broad-spectrum pharmaceuticals.

Many treatments considered for COVID-19 have relied on a broad-spectrum approach.
These treatments do not specifically target a virus or particular host receptor, but rather induce broad shifts in host biology that are hypothesized to be potential inhibitors of the virus.
This approach relies on the fact that when a virus enters a host, the host becomes the virus's environment.
Therefore, the state of the host can also influence the virus's ability to replicate and spread.
The administration and assessment of broad-spectrum small-molecule drugs on a rapid time course was feasible because they are often either available in hospitals, or in some cases may also be prescribed to a large number of out-patients.
One of the other advantages is that these well-established compounds, if found to be beneficial, are often widely available, in contrast to boutique experimental drugs.

In some cases, prior data was available from experiments examining the response of other HCoVs or HCoV infections to a candidate drug.
In addition to non-pharmaceutical interventions such as encouraging non-intubated patients to adopt a prone position [@doi:10.1001/jamainternmed.2020.3030], knowledge about interactions between HCoVs and the human body, many of which emerged from SARS and MERS research over the past two decades, led to the suggestion that a number of common drugs might benefit COVID-19 patients.
However, the short duration and low case numbers of prior outbreaks were less well-suited to the large-scale study of clinical applications than the COVID-19 pandemic is.
As a result, COVID-19 has presented the first opportunity to robustly evaluate treatments that were common during prior HCoV outbreaks to determine their clinical efficacy.
The first year of the COVID-19 pandemic demonstrated that there are several different trajectories that these clinically suggested, widely available candidates can follow when assessed against a widespread, novel viral threat.

One approach to identifying candidate small molecule drugs was to look at the approaches used to treat SARS and MERS.
Treatment of SARS and MERS patients prioritized supportive care and symptom management [@doi:10.1093/ajcp/aqaa029].
Among the clinical treatments for SARS and MERS that were explored, there was generally a lack of evidence indicating whether they were effective.
Most of the supportive treatments for SARS were found inconclusive in meta-analysis [@doi:10.1371/journal.pmed.0030343], and a 2004 review reported that not enough evidence was available to make conclusions about most treatments [@doi:10.1007/s10156-003-0296-9].
However, one strategy adopted from prior HCoV outbreaks is currently the best-known treatment for severe cases of COVID-19.
Corticosteroids represent broad-spectrum treatments and are a well-known, widely available treatment for pneumonia [@doi:10.1002/14651858.CD007720.pub3; @doi:10.1002/14651858.CD007720.pub2; @doi:10.1183/09031936.00154107; @doi:10.1007/s00408-007-9020-3; @doi:10.1371/journal.pone.0047926; @doi:10.1186/cc10103] that have also been debated as a possible treatment for ARDS [@doi:10/ggpxzc; @doi:10/gcv2; @doi:10.1378/chest.06-1743; @doi:10.1378/chest.07-0714; @doi:10.1056/NEJMoa051693; @doi:10.1136/bmj.39537.939039.BE].
Corticosteroids were also used and subsequently evaluated as possible supportive care for SARS and MERS.
In general, studies and meta-analyses did not identify support for corticosteroids to prevent mortality in these HCoV infections [@doi:10.1136/thx.2003.017665; @doi:10.1016/j.jinf.2006.01.005; @doi:10.1038/s41375-020-0848-3]; however, one found that the effects might be masked by variability in treatment protocols, such as dosage and timing [@doi:10.1007/s10156-003-0296-9].
While the corticosteroids most often used to treat SARS were methylprednisolone and hydrocortisone, availability issues for these drugs at the time led to dexamethasone also being used in North America [@doi:10.1056/NEJMp030072].

Dexamethasone (9&alpha;-fluoro-16&alpha;-methylprednisolone) is a synthetic corticosteroid that binds to glucocorticoid receptors [@doi:10.1021/cr068203e; @url:https://www.cebm.net/covid-19/dexamethasone].
It functions as an anti-inflammatory agent by binding to glucocorticoid receptors with higher affinity than endogenous cortisol [@doi:10.1016/B978-0-323-48110-6.00034-X].
Dexamethasone and other steroids are widely available and affordable, and they are often used to treat community-acquired pneumonia [@doi:10.1038/s41392-020-0127-9] as well as chronic inflammatory conditions such as asthma, allergies, and rheumatoid arthritis [@doi:10.1021/ja01545a061; @doi:10.1001/jama.1960.03030070009002; @dailymed-dexamethasone].
Immunosuppressive drugs such as steroids are typically contraindicated in the setting of infection [@doi:10.1177/2040622313485275], but because COVID-19 results in hyperinflammation that appears to contribute to mortality via lung damage, immunosuppression may be a helpful approach to treatment [@doi:10/ggnzmc].
A clinical trial that began in 2012 recently reported that dexamethasone may improve outcomes for patients with ARDS [@doi:10/ggpxzc], but a meta-analysis of a small amount of available data about dexamethasone as a treatment for SARS suggested that it may, in fact, be associated with patient harm [@doi:10/ggks86].
However, the findings in SARS may have been biased by the fact that all of the studies examined were observational and a large number of inconclusive studies were not included [@doi:10/ggq356].
The questions of whether and when to counter hyperinflammation with immunosuppression in the setting of COVID-19 (as in SARS [@doi:10.1186/cc2452]) was an area of intense debate, as the risks of inhibiting antiviral immunity needed to be weighed against the beneficial anti-inflammatory effects [@doi:10/ggq8hs].
As a result, guidelines early in the pandemic typically recommended avoiding treating COVID-19 patients with corticosteroids such as dexamethasone [@doi:10/ggks86].

Despite this initial concern, dexamethasone was evaluated as a potential treatment for COVID-19 (Appendix 1).
Dexamethasone treatment comprised one arm of the multi-site Randomized Evaluation of COVID-19 Therapy (RECOVERY) trial in the United Kingdom [@doi:10.1101/2020.06.22.20137273].
This study found that the 28-day mortality rate was lower in patients receiving dexamethasone than in those receiving standard of care (SOC).
However, this finding was driven by differences in mortality among patients who were receiving mechanical ventilation or supplementary oxygen at the start of the study.
The report indicated that dexamethasone reduced 28-day mortality relative to SOC in patients who were ventilated (29.3% versus 41.4%) and among those who were receiving oxygen supplementation (23.3% versus 26.2%) at randomization, but not in patients who were breathing independently (17.8% versus 14.0%).
These findings also suggested that dexamethasone may have reduced progression to mechanical ventilation, especially among patients who were receiving oxygen support at randomization.
Other analyses have supported the importance of disease course in determining the efficacy of dexamethasone: additional results suggest greater potential for patients who have experienced symptoms for at least seven days and patients who were not breathing independently [@doi:10.1056/NEJMoa2021436].
A meta-analysis that evaluated the results of the RECOVERY trial alongside trials of other corticosteroids, such as hydrocortisone, similarly concluded that corticosteroids may be beneficial to patients with severe COVID-19 who are receiving oxygen supplementation [@doi:10.1053/j.jvca.2020.11.057].
Thus, it seems likely that dexamethasone is useful for treating inflammation associated with immunopathy or cytokine release syndrome (CRS), which is a condition caused by detrimental overactivation of the immune system [@individual-pathogenesis].
In fact, corticosteroids such as dexamethasone are sometimes used to treat CRS [@doi:10.1182/blood-2014-05-552729].
Guidelines were quickly updated to encourage the use of dexamethasone in severe cases [@doi:10.1001/jama.2020.16747], and this affordable and widely available treatment rapidly became a valuable tool against COVID-19 [@doi:10.1016/j.ejphar.2021.173854], with demand surging within days of the preprint's release [@doi:10.1136/bmj.m2512].

### Approaches Targeting the Virus

Therapeutics that directly target the virus itself hold the potential to prevent people infected with SARS-CoV-2 from developing potentially damaging symptoms (Figure @fig:therapeutics).
Such drugs typically fall into the broad category of antivirals.
Antiviral therapies hinder the spread of a virus within the host, rather than destroying existing copies of the virus, and these drugs can vary in their specificity to a narrow or broad range of viral targets.
This process requires inhibiting the replication cycle of a virus by disrupting one of six fundamental steps [@isbn:978-1405136457].
In the first of these steps, the virus attaches to and enters the host cell through endocytosis.
Then the virus undergoes uncoating, which is classically defined as the release of viral contents into the host cell.
Next, the viral genetic material enters the nucleus where it gets replicated during the biosynthesis stage.
During the assembly stage, viral proteins are translated, allowing new viral particles to be assembled.
In the final step new viruses are released into the extracellular environment.
Although antivirals are designed to target a virus, they can also impact other processes in the host and may have unintended effects.
Therefore, these therapeutics must be evaluated for both efficacy and safety.
As the technology to respond to emerging viral threats has also evolved over the past two decades, a number of candidate treatments have been identified for prior viruses that may be relevant to the treatment of COVID-19.

![
**Mechanisms of Action for Potential Therapeutics**
Potential therapeutics currently being studied can target the SARS-CoV-2 virus or modify the host environment through many different mechanisms.
Here, the relationships between the virus, host cells, and several therapeutics are visualized.
Drug names are color-coded according to the grade assigned to them by the Center for Cytokine Storm Treatment & Laboratory's CORONA Project [@url:https://cdcn.org/corona-data-viewer] (Green = A, Lime = B, Orange = C, and Red = D).
](images/N001-LifeCyclePlusDrugs.png "SARS-CoV-2 life cycle and therapeutic mechanisms"){#fig:therapeutics secno=1}

Many antiviral drugs are designed to inhibit the replication of viral genetic material during the biosynthesis step.
Unlike DNA viruses, which can use the host enzymes to propagate themselves, RNA viruses like SARS-CoV-2 depend on their own polymerase, the RNA-dependent RNA polymerase (RdRP), for replication [@doi:10.1007/978-1-4939-2438-7; @doi:10.1002/jmv.25761].
RdRP is therefore a potential target for antivirals against RNA viruses. 
Disruption of RdRP is the proposed mechanism underlying the treatment of SARS and MERS with ribavirin [@doi:10.1080/17460441.2019.1581171].
Ribavirin is an antiviral drug effective against other viral infections that was often used in combination with corticosteroids and sometimes interferon (IFN) medications to treat SARS and MERS [@doi:10.1038/nrmicro.2016.81].
However, analyses of its effects in retrospective and _in vitro_ analyses of SARS and the SARS-CoV-1 virus, respectively, have been inconclusive [@doi:10.1038/nrmicro.2016.81].
While IFNs and ribavirin have shown promise in _in vitro_ analyses of MERS, their clinical effectiveness remains unknown [@doi:10.1038/nrmicro.2016.81].
The current COVID-19 pandemic has provided an opportunity to assess the clinical effects of these treatments.
As one example, ribivarin was also used in the early days of COVID-19, but a retrospective cohort study comparing patients who did and did not receive ribivarin revealed no effect on the mortality rate [@doi:10.1016/j.ijantimicag.2020.106114].

Since nucleotides and nucleosides are the natural building blocks for RNA synthesis, an alternative approach has been to explore nucleoside and nucleotide analogs for their potential to inhibit viral replication.
Analogs containing modifications to nucleotides or nucleosides can disrupt key processes including replication [@doi:10.1016/j.antiviral.2018.04.004].
A single incorporation does not influence RNA transcription; however, multiple events of incorporation lead to the arrest of RNA synthesis [@doi:10.1371/journal.pone.0068347].
One candidate antiviral considered for the treatment of COVID-19 is favipiravir (Avigan), also known as T-705, which was discovered by Toyama Chemical Co., Ltd. [@url:https://www.drugbank.ca/drugs/DB12466].
It was previously found to be effective at blocking viral amplification in several influenza subtypes as well as other RNA viruses, such as _Flaviviridae_ and _Picornaviridae_, through a reduction in plaque formation [@doi:10.1128/AAC.46.4.977-981.2002] and viral replication in Madin-Darby canine kidney cells [@doi:10.1128/AAC.01051-06].
Favipiravir (6-fluoro-3-hydroxy-2-pyrazinecarboxamide) acts as a purine and purine nucleoside analogue that inhibits viral RNA polymerase in a dose-dependent manner across a range of RNA viruses, including influenza viruses [@doi:10.1128/AAC.49.3.981-986.2005; @doi:10.1128/AAC.01074-08; @doi:10.1128/AAC.00356-07; @doi:10.1016/j.bbrc.2012.07.034; @doi:10.1128/AAC.01219-10].
Biochemical experiments showed that favipiravir was recognized as a purine nucleoside analogue and incorporated into the viral RNA template.
In 2014, the drug was approved in Japan for the treatment of influenza that was resistant to conventional treatments like neuraminidase inhibitors [@doi:10.2183/pjab.93.027].
Though initial analyses of favipiravir in observational studies of its effects on COVID-19 patients were promising, recent results of two small RCTs suggest that it is unlikely to affect COVID-19 outcomes (Appendix 1).

In contrast, another nucleoside analog, remdesivir, is one of the few treatments against COVID-19 that has received FDA approval.
Remdesivir (GS-5734) is an intravenous antiviral that was proposed by Gilead Sciences as a possible treatment for Ebola virus disease.
It is metabolized to GS-441524, an adenosine analog that inhibits a broad range of polymerases and then evades exonuclease repair, causing chain termination [@doi:10.1074/jbc.AC120.013056; @doi:10.1128/mBio.00221-18; @doi:10.1038/s41422-020-0282-0].
Gilead received an emergency use authorization (EUA) for remdesivir from the FDA early in the pandemic (May 2020) and was later found to reduce mortality and recovery time in a double-blind, placebo-controlled, phase III clinical trial performed at 60 trial sites, 45 of which were in the United States [@doi:10.1056/NEJMoa2007764; @clinicaltrials:NCT04280705; @clinicaltrials:NCT04292899; @doi:10.1056/NEJMoa2007016].
Subsequently, the WHO Solidarity trial, a large-scale, open-label trial enrolling 11,330 adult in-patients at 405 hospitals in 30 countries around the world, reported no effect of remdesivir on in-hospital mortality, duration of hospitalization, or progression to mechanical ventilation [@doi:10.1056/NEJMoa2023184].
Therefore, additional clinical trials of remdesivir in different patient pools and in combination with other therapies may be needed to refine its use in the clinic and determine the forces driving these differing results.
Remdesivir offers proof of principle that SARS-CoV-2 can be targeted at the level of viral replication, since remdesivir targets the viral RNA polymerase at high potency.
Identification of such candidates depends on knowledge about the virological properties of a novel threat.
However, the success and relative lack of success, respectively, of remdesivir and favipiravir underscore the fact that drugs with similar mechanisms will not always produce similar results in clinical trials.

### Disrupting Host-Virus Interactions

#### Interrupting Viral Colonization of Cells

Some of the most widely publicized examples of efforts to repurpose drugs for COVID-19 are broad-spectrum, small-molecule drugs where the mechanism of action made it seem that the drug might disrupt interactions between SARS-CoV-2 and human host cells (Figure @fig:therapeutics).
However, the exact outcomes of such treatments are difficult to predict _a priori_, and there are several examples where early enthusiasm was not borne out in subsequent trials.
One of the most famous examples of an analysis of whether a well-known medication could provide benefits to COVID-19 patients came from the assessment of chloroquine (CQ) and hydroxychloroquine (HCQ), which are used for the treatment and prophylaxis of malaria as well as the treatment of lupus erythematosus and rheumatoid arthritis in adults [@dailymed-hcq].
These drugs are lysosomotropic agents, meaning they are weak bases that can pass through the plasma membrane.
It was thought that they might provide benefits against SARS-CoV-2 by interfering with the digestion of antigens within the lysosome and inhibiting CD4 T-cell stimulation while promoting the stimulation of CD8 T-cells [@doi:10.1111/j.1529-8019.2007.00131.x].
These compounds also have anti-inflammatory properties [@doi:10.1111/j.1529-8019.2007.00131.x] and can decrease the production of certain key cytokines involved in the immune response, including interleukin-6 (IL-6) and inhibit the stimulation of Toll-like receptors (TLR) and TLR signaling [@doi:10.1111/j.1529-8019.2007.00131.x].

_In vitro_ analyses reported that CQ inhibited cell entry of SARS-CoV-1 [@doi:10.1186/1743-422X-2-69] and that both CQ and HCQ inhibited viral replication within cultured cells [@doi:10.1093/cid/ciaa237], leading to early hope that it might provide similar therapeutic or protective effects in patients.
However, while the first publication on the clinical application of these compounds to the inpatient treatment of COVID-19 was very positive [@doi:10.1016/j.ijantimicag.2020.105949], it was quickly discredited [@url:https://www.isac.world/news-and-publications/official-isac-statement].
Over the following months, extensive evidence emerged demonstrating that CQ and HCQ offered no benefits for COVID-19 patients and, in fact, carried the risk of dangerous side effects (Appendix 1).
The nail in the coffin came when findings from the large-scale RECOVERY trial were released on October 8, 2020.
This study enrolled 11,197 hospitalized patients whose physicians believed it would not harm them to participate and used a randomized, open-label design to study the effects of HCQ compared to standard of care (SOC) at 176 hospitals in the United Kingdom [@doi:10.1056/NEJMoa2022926].
Rates of COVID-19-related mortality did not differ between the control and HCQ arms, but patients receiving HCQ were slightly more likely to die due to cardiac events.
Patients who received HCQ also had a longer duration of hospitalization than patients receiving usual care and were more likely to progress to mechanical ventilation or death (as a combined outcome).
As a result, enrollment in the HCQ arm of the RECOVERY trial was terminated early [@url:https://www.recoverytrial.net/news/statement-from-the-chief-investigators-of-the-randomised-evaluation-of-covid-19-therapy-recovery-trial-on-hydroxychloroquine-5-june-2020-no-clinical-benefit-from-use-of-hydroxychloroquine-in-hospitalised-patients-with-covid-19].
The story of CQ/HCQ therefore illustrates how initial promising _in vitro_ analyses can fail to translate to clinical usefulness.

A similar story has arisen with the broad-spectrum, small-molecule anthelmintic ivermectin, which is a synthetic analog of avermectin, a bioactive compound produced by a microorganism known as _Streptomyces avermectinius_ and _Streptomyces avermitilis_ [@doi:10.1038/nrmicro1048; @doi:10.1128/AAC.15.3.361].
Avermectin disrupts the ability of parasites to avoid the host immune response by blocking glutamate-gated chloride ion channels in the peripheral nervous system from closing, leading to hyperpolarization of neuronal membranes, disruption of neural transmission, and paralysis [@doi:10.2183/pjab.87.13; @doi:10.1038/nrmicro1048; @doi:10.1038/ja.2017.11].
Ivermectin has been used since the early 1980s to treat endo- and ecto-parasitic infections by helminths, insects, and arachnids in veterinary contexts [@doi:10.1038/nrmicro1048; @doi:10.1128/AAC.15.3.372] and since the late 1980s to treat human parasitic infections as well [@doi:10.1038/nrmicro1048; @doi:10.2183/pjab.87.13].
More recent research has indicated that ivermectin might function as a broad-spectrum antiviral by disrupting the trafficking of viral proteins by both RNA and DNA viruses [@doi:10.1038/s41429-020-0336-z; @doi:10.1038/ja.2017.11; @doi:10.1186/s12941-020-00368-w], although most of these studies have demonstrated this effect _in vitro_ [@doi:10.1186/s12941-020-00368-w].
The potential for antiviral effects on SARS-CoV-2 were investigated _in vitro_, and ivermectin was found to inhibit viral replication in a cell line derived from Vero cells (Vero-hSLAM) [@doi:10.1016/j.antiviral.2020.104787]. 
However, inhibition of viral replication was achieved at concentrations that were much higher than that explored by existing dosage guidelines [@doi:10.4269/ajtmh.20-0271; @doi:10.1002/cpt.1889], which are likely to be associated with significant side effects due to the increased potential that the compound could cross the mammalian blood-brain barrier [@doi:10.1080/13102818.2020.1775118; @doi:10.1371/journal.pntd.0001883].

Retrospective studies and small RCTs began investigating the effects of standard doses of this low-cost, widely available drug.
One retrospective study reported that ivermectin reduced all-cause mortality [@doi:10.1016/j.chest.2020.10.009] while another reported no difference in clinical outcomes or viral clearance [@doi:10.1371/journal.pone.0242184].
Small RCTs enrolling less than 50 patients per arm have also reported a wide array of positive [@doi:10.4103/1995-7645.318304; @doi:10.1093/qjmed/hcab035; @doi:10.1016/j.toxrep.2021.03.003; @doi:10.1016/j.ijid.2020.11.191; @doi:10.1016/j.clinthera.2021.04.007] and negative results [@doi:10.1016/j.eclinm.2020.100720; @doi:10.21203/rs.3.rs-191648/v1].
A slightly larger RCT enrolling 115 patients in two arms reported inconclusive results [@doi:10.18433/jpps32105].
Hope for the potential of ivermectin peaked with the release of a preprint reporting results of a multicenter, double-blind RCT where a four-day course of ivermectin was associated with clinical improvement and earlier viral clearance in 400 symptomatic patients and 200 close contacts [@doi:10.21203/rs.3.rs-100956/v3]; however, concerns were raised about both the integrity of the data and the paper itself [@url:https://grftr.news/why-was-a-major-study-on-ivermectin-for-covid-19-just-retracted; @url:https://steamtraen.blogspot.com/2021/07/Some-problems-with-the-data-from-a-Covid-study.html], and this study was removed by the preprint server Research Square [@doi:10.21203/rs.3.rs-100956/v4].
A similarly sized RCT suggested no effect on the duration of symptoms among 400 patients split evenly across the intervention and control arms [@doi:10.1001/jama.2021.3071], and although meta-analyses have reported both null [@doi:10.1101/2021.05.21.21257595; @doi:10.1101/2021.01.26.21250420] and beneficial [@doi:10.1093/ofid/ofab394; @doi:10.1016/j.dsx.2021.102186; @doi:10.1097/MJT.0000000000001402; @doi:10.1002/rmv.2265; @doi:10.31219/osf.io/k37ft; @doi:10.1097/MJT.0000000000001377; @doi:10.1007/s43440-021-00245-z; @doi:10.1101/2021.04.30.21256415] effects of ivermectin on COVID-19 outcomes, the certainty is likely to be low [@doi:10.1016/j.dsx.2021.102186].
These findings are potentially biased by a small number of low-quality studies, including the preprint that has been taken down [@url:https://gidmk.medium.com/does-ivermectin-work-for-covid-19-1166126c364a], and the authors of one [@doi:10.1093/ofid/ofab358] have issued a notice [@doi:10.1093/ofid/ofab394] that they will revise their study with the withdrawn study removed.
Thus, much like HCQ/CQ, enthusiasm for research that either has not or should not have passed peer review has led to large numbers of patients worldwide receiving treatments that might not have any effect or could even be harmful.
Additionally, comments on the now-removed preprint include inquiries into how best to self-administer veterinary ivermectin as a prophylactic [@doi:10.21203/rs.3.rs-100956/v4], and the FDA has posted information explaining why veterinary ivermectin should not be taken by humans concerned about COVID-19 [@url:https://www.fda.gov/animal-veterinary/product-safety-information/faq-covid-19-and-ivermectin-intended-animals].
Ivermectin is now one of several candidate therapeutics being investigated in the large-scale TOGETHER [@clinicaltrials:NCT04727424] and PRINCIPLE [@url:https://www.principletrial.org/news/ivermectin-to-be-investigated-as-a-possible-treatment-for-covid-19-in-oxford2019s-principle-trial] clinical trials.
The TOGETHER trial, which previously demonstrated no effect of HCQ and lopinavir-ritonavir [@doi:10.1001/jamanetworkopen.2021.6468], released preliminary results in early August 2021 suggesting that ivermectin also has no effect on COVID-19 outcomes [@url:https://rethinkingclinicaltrials.org/news/august-6-2021-early-treatment-of-covid-19-with-repurposed-therapies-the-together-adaptive-platform-trial-edward-mills-phd-frcp].

While CQ/HCQ and ivermectin are well-known medications that have long been prescribed in certain contexts, investigation of another well-established type of pharmaceutical was facilitated by the fact that it was already being taken by a large number of COVID-19 patients.
Angiotensin-converting enzyme inhibitors (ACEIs) and angiotensin II receptor blockers (ARBs) are among today's most commonly prescribed medications, often being used to control blood pressure [@url:https://clincalc.com/DrugStats/Drugs/Lisinopril; @doi:10.1056/NEJMp1901657].
In the United States, for example, they are prescribed well over 100,000,000 times annually [@doi:10.18553/jmcp.2010.16.9.671].
Prior to the COVID-19 pandemic, the relationship between ACE2, ACEIs, and SARS had been considered as possible evidence that ACE2 could serve as a therapeutic target [@doi:10.1016/j.tips.2004.04.001], and the connection had been explored through _in vitro_ and molecular docking analysis [@doi:10.1161/01.HYP.0000146120.29648.36] but ultimately was not pursued clinically [@doi:10/d85vmw].
Data from some animal models suggest that several, but not all, ACEIs and several ARBs increase ACE2 expression in the cells of some organs [@doi:10.1093/cvr/cvaa097], but clinical studies have not established whether plasma ACE2 expression is increased in humans treated with these medications [@doi:10.1093/europace/euw246].
In this case, rather than introducing ARBs/ACEIs, a number of analyses have investigated whether discontinuing use affects COVID-19 outcomes.
An initial observational study of the association of exposure to ACEIs or ARBs with outcomes in COVID-19 was retracted from the _New England Journal of Medicine_ [@doi:10.1056/NEJMoa2007621] due to concerns related to data availability [@doi:10.1056/nejmc2021225].
As RCTs have become available, they have demonstrated no effect of continuing versus discontinuing ARBs/ACEIs on patient outcomes [@doi:10/fvgt; @doi:10.1001/jama.2020.25864] (Appendix 1).
Thus, once again, despite a potential mechanistic association with the pathology of SARS-CoV-2 infection, these medications were not found to influence the trajectory of COVID-19 illness.

For medications that are widely known and common, clinical research into their efficacy against a novel threat can be developed very quickly.
This feasibility can present a double-edged sword.
For example, HCQ and CQ were incorporated into SOC in many countries early in the pandemic and had to be discontinued once their potential to harm COVID-19 patients became apparent [@url:https://www.fda.gov/media/138946/download; @url:https://www.ema.europa.eu/en/news/covid-19-chloroquine-hydroxychloroquine-only-be-used-clinical-trials-emergency-use-programmes].
Dexamethasone remains the major success story from this category of repurposed drugs and is likely to have saved a large number of lives since summer 2020 [@doi:10.1016/j.ejphar.2021.173854].

#### Manipulating the Host Immune Response

Treatments based on understanding a virus and/or how a virus interacts with the human immune system can fall into two categories: they can interact with the innate immune response, which is likely to be a similar response across viruses, or they can be specifically designed to imitate the adaptive immune response to a particular virus. 
In the latter case, conservation of structure or behavior across viruses enables exploring whether drugs developed for one virus can treat another.
During the COVID-19 pandemic, a number of candidate therapeutics have been explored in these categories, with varied success.

Knowledge gained from trying to understand SARS-CoV-1 and MERS-CoV from a fundamental biological perspective and characterize how they interact with the human immune system provides a theoretical basis for identifying candidate therapies.
Biologics are a particularly important class of drugs for efforts to address HCoV through this paradigm.
They are produced from components of living organisms or viruses, historically primarily from animal tissues, but have become increasingly feasible to produce as recombinant technologies have advanced  [@doi:10.1016/j.copbio.2009.10.006].

There are many differences on the development side between biologics and synthesized pharmaceuticals, such as small molecule drugs.
Typically, biologics are orders of magnitude larger than small molecule drugs and are catabolized by the body to their amino acid components [@doi:10.2174/138920006774832604].
They are often heat sensitive, and their toxicity can vary, as it is not directly associated with the primary effects of the drug; in general, their physiochemical properties are much less understood compared to small molecules [@doi:10.2174/138920006774832604].
Biologics include significant medical breakthroughs such as insulin for the management of diabetes and vaccines, as well monoclonal antibodies (mAbs) and interferons (IFNs), which can be used to target the host immune response after infection.

mAbs have revolutionized the way we treat human diseases and have become some of the best-selling drugs in the pharmaceutical market in recent years [@doi:10.1186/s12929-019-0592-z].
There are currently 79 FDA approved mAbs on the market, including antibodies for viral infections (e.g. Ibalizumab for _Human immunodeficiency virus_ and Palivizumab for _Respiratory syncytial virus_) [@doi:10.1186/s12929-019-0592-z; @doi:10.1146/annurev-immunol-032712-095916].
Virus-specific neutralizing antibodies commonly target viral surface glycoproteins or host structures, thereby inhibiting viral entry through receptor binding interference [@doi:10.3389/fmicb.2017.02323; @doi:10.1080/21645515.2017.1337614].
This interference is predicted to reduce the viral load, mitigate disease, and reduce overall hospitalization.
mAbs can be designed for a particular virus, and significant advances have been made in the speed at which new mAbs can be identified and produced.
At the time of the SARS and MERS epidemics, interest in mAbs to reduce infection was never realized [@doi:10.1093/infdis/jiw080; @doi:10.1016/j.antiviral.2017.03.025], but this allowed for mAbs to quickly be considered among the top candidates against COVID-19.

##### Biologics and the Innate Immune Response

Deaths from COVID-19 often occur when inflammation becomes dysregulated following an immune response to the SARS-CoV-2 virus.
Therefore, one potential approach to reducing COVID-19 mortality rates is to manage the inflammatory response in severely ill patients.
One candidate therapeutic identified that uses this mechanism is tocilizumab (TCZ).
TCZ is a mAb that was developed to manage chronic inflammation caused by the continuous synthesis of the cytokine IL-6 [@doi:10.1101/cshperspect.a016295].
IL-6 is a pro-inflammatory cytokine belonging to the interleukin family, which is comprised by immune system regulators that are primarily responsible for immune cell differentiation.
Often used to treat chronic inflammatory conditions such as rheumatoid arthritis [@doi:10.1101/cshperspect.a016295], TCZ has become a pharmaceutical of interest for the treatment of COVID-19 because of the role IL-6 plays in this disease.
It has also been approved to treat CRS caused by CAR-T treatments [@dailymed-tocilizumab].
While the secretion of IL-6 can be associated with chronic conditions, IL-6 is a key player in the innate immune response and is secreted by macrophages in response to the detection of pathogen-associated molecular patterns and damage-associated molecular patterns [@doi:10.1101/cshperspect.a016295].
An analysis of 191 in-patients at two Wuhan hospitals revealed that blood concentrations of IL-6 differed between patients who did and did not recover from COVID-19.
Patients who ultimately died had higher IL-6 levels at admission than those who recovered [@doi:10/ggnxb3].
Additionally, IL-6 levels remained higher throughout the course of hospitalization in the patients who ultimately died [@doi:10/ggnxb3].

Currently, TCZ is being administered either as a monotherapy or in combination with other treatments in 73 interventional COVID-19 clinical trials (Figure @fig:ebm-trials).
A number of retrospective studies have been conducted in several countries [@doi:10/d2pk; @doi:10.1016/j.chest.2020.06.006; @doi:10.1016/j.ejim.2020.05.009; @doi:10.1016/j.medmal.2020.05.001; @doi:10.1093/qjmed/hcaa206; @doi:10.1073/pnas.2005615117].
In general, these studies have reported a positive effect of TCZ on reducing mortality in COVID-19 patients, although due to their retrospective designs, significant limitations are present in all of them (Appendix 1).
It was not until February 11, 2021 that a preprint describing preliminary results of the first RCT of TCZ was released as part of the RECOVERY trial [@doi:10.1101/2021.02.11.21249258].
TCZ was found to reduce 28-day mortality from 33% in patients receiving SOC alone to 29% in those receiving TCZ.
Combined analysis of the RECOVERY trial data with data from smaller RCTs suggested a 13% reduction in 28-day mortality [@doi:10.1101/2021.02.11.21249258].
While this initial report did not include the full results expected from the RECOVERY trial, this large-scale, RCT provides strong evidence that TCZ may offer benefits for COVID-19 patients.
The RECOVERY trial along with results from several other RCTs [@doi:10.1056/NEJMoa2028700; @doi:10.1056/NEJMoa2030340; @clinicaltrials:NCT04409262; @clinicaltrials:NCT04320615; @clinicaltrials:NCT04372186] were cited as support for the EUA issued for TCZ in June 2021 [@url:https://www.fda.gov/media/150319/download].
However, the fact that TCZ suppresses the immune response means that it does carry risks for patients, especially a potential risk of secondary infection (Appendix 1).

TCZ is just one example of a candidate drug targeting the host immune response and specifically excessive inflammation.
For example, interferons (IFNs) have also been investigated; these are a family of cytokines critical to activating the innate immune response against viral infections.
Synairgen has been investigating a candidate drug, SNG001, which is an IFN-𝛽-1a formulation to be delivered to the lungs via inhalation [@clinicaltrials:NCT04385095] that they reported reduced progression to ventilation in a double-blind, placebo-controlled, multi-center study of 101 patients with an average age in the late 50s [@synairgen-SNG001-July-2020; @doi:10/ghjzm4].
However, these findings were not supported by the large-scale WHO Solidarity trial, which reported no significant effect of IFN-&beta;-1a on patient survival during hospitalization [@doi:10.1056/NEJMoa2023184], although differences in the designs of the two studies, and specifically the severity of illness among enrolled patients, may have influenced their divergent outcomes (Appendix 1).
Other biologics influencing inflammation are also being explored (Appendix 1).
It is also important that studies focused on inflammation as a possible therapeutic target consider the potential differences in baseline inflammation among patients from different backgrounds, which may be caused by differing life experiences (see [@individual-inequality]).

##### Biologics and the Adaptive Immune Response

While TCZ is an example of an mAb focused on managing the innate immune response, other treatments are more specific, targeting the adaptive immune response after an infection.
In some cases, treatments can utilize biologics obtained directly from recovered individuals.
From the very early days of the COVID-19 pandemic, polyclonal antibodies from convalescent plasma were investigated as a potential treatment for COVID-19 [@doi:10/ggqr7s; @doi:10.1001/jama.2020.4940].
Convalescent plasma was used in prior epidemics including SARS, Ebola Virus Disease, and even the 1918 Spanish Influenza [@doi:10/ggqr7s; @doi:10.1002/jmv.25961].
Use of convalescent plasma transfusion (CPT) over more than a century has aimed to reduce symptoms and improve mortality in infected people [@doi:10.1002/jmv.25961], possibly by accelerating viral clearance [@doi:10/ggqr7s].
However, it seems unlikely that this classic treatment confers any benefit for COVID-19 patients.
Several systematic reviews have investigated whether CPT reduced mortality in COVID-19 patients, and although findings from early in the pandemic (up to April 19, 2020) did support use of CPT [@doi:10.1002/jmv.25961], the tide has shifted as the body of available literature has grown [@doi:10.1001/jama.2021.2747].
While titer levels were suggested as a possible determining factor in the success of CPT against COVID-19 [@doi:10.1056/NEJMoa2031893], the large-scale RECOVERY trial evaluated the effect of administering high-titer plasma specifically and found no effect on mortality or hospital discharge over a 28-day period [@doi:10.1101/2021.03.09.21252736].
These results thus suggest that, despite initial optimism and an EUA from the FDA, CPT is unlikely to be an effective therapeutic for COVID-19.

A different narrative is shaping up around the use of mAbs specifically targeting SARS-CoV-2. 
During the first SARS epidemic in 2002, neutralizing antibodies (nAbs) were found in SARS-CoV-1-infected patients [@doi:10.1111/j.1469-0691.2004.01009.x; @doi:10.1086/423286].
Several studies following up on these findings identified various S-glycoprotein epitopes as the major targets of nAbs against SARS-CoV-1 [@doi:10.1517/14712590902763755].
Coronaviruses use trimeric spike (S) glycoproteins on their surface to bind to the host cell, allowing for cell entry [@doi:10.1016/j.cell.2020.02.052; @doi:10.1016/j.cell.2020.02.058].
Each S glycoprotein protomer is comprised of an S1 domain, also called the receptor binding domain (RBD), and an S2 domain.
The S1 domain binds to the host cell while the S2 domain facilitates the fusion between the viral envelope and host cell membranes [@doi:10.1517/14712590902763755].
The genomic identity between the RBD of SARS-CoV-1 and SARS-CoV-2 is around 74% [@doi:10.1101/2020.02.16.951723].
Due to this high degree of similarity, preexisting antibodies against SARS-CoV-1 were initially considered candidates for neutralizing activity against SARS-CoV-2.
While some antibodies developed against the SARS-CoV-1 spike protein showed cross-neutralization activity with SARS-CoV-2 [@doi:10.1016/j.tips.2020.07.004; @doi:10.1038/s41586-020-2349-y], others failed to bind to SARS-CoV-2 spike protein at relevant concentrations [@doi:10.1126/science.abb2507].
Cross-neutralizing activities were dependent on whether the epitope recognized by the antibodies were conserved between SARS-CoV-1 and SARS-CoV-2 [@doi:10.1016/j.tips.2020.07.004].

Technological advances in antibody drug design as well as in structural biology massively accelerated the discovery of novel antibody candidates and the mechanisms by which they interact with the target structure.
Within just a year of the structure of the SARS-CoV-2 spike protein being published, an impressive pipeline of monoclonal antibodies targeting SARS-CoV-2 entered clinical trials, with hundreds more candidates in preclinical stages.
The first human monoclonal neutralizing antibody specifically against the SARS-CoV-2 S glycoprotein was developed using hybridoma technology [@doi:10.1101/2020.03.11.987958], where antibody-producing B-cells developed by mice are inserted into myeloma cells to produce a hybrid cell line (the hybridoma) that is grown in culture.
The 47D11 antibody clone was able to cross-neutralize SARS-CoV-1 and SARS-CoV-2.
This antibody (now ABVV-47D11) has recently entered clinical trials in collaboration with AbbVie.
Additionally, an extensive monoclonal neutralizing antibody pipeline has been developed to combat the ongoing pandemic, with over 50 different antibodies in clinical trials [@doi:10.1016/j.bsheal.2021.02.001].
Thus far, the monotherapy sotrovimab and two antibody cocktails (bamlanivimab/estesevimab and casirivimab/imdevimab) have been granted EUAs by the FDA.

One of the studied antibody cocktails consists of bamlanivimab and estesevimab.
Bamlanivimab (Ly-CoV555) is a human mAb that was derived from convalescent plasma donated by a recovered COVID-19 patient, evaluated in research by the National Institute of Allergy and Infectious Diseases (NIAID), and subsequently developed by AbCellera and Eli Lilly.
The neutralizing activity of bamlanivimab was initially demonstrated _in vivo_ using a nonhuman primate model [@doi:10.1101/2020.09.30.318972].
Based on these positive preclinical data, Eli Lilly initiated the first human clinical trial for a monoclonal antibody against SARS-CoV-2.
The phase 1 trial, which was conducted in hospitalized COVID-19 patients, was completed in August 2020 [@clinicaltrials:NCT04411628].
Estesevimab (LY-CoV016 or JS-016) is also a monoclonal neutralizing antibody against the spike protein of SARS-CoV-2.
It was initially developed by Junshi Biosciences and later licensed and developed through Eli Lilly.
A phase 1 clinical trial to assess the safety of etesevimab was completed in October 2020 [@clinicaltrials:NCT04441931].
Etesevimab was shown to bind a different epitope on the spike protein than bamlanivimab, suggesting that the two antibodies used as a combination therapy would further enhance their clinical use compared to a monotherapy [@doi:10.1001/jama.2021.0202].
To assess the efficacy and safety of bamlanivimab alone or in combination with etesevimab for the treatment of COVID-19, a phase 2/3 trial (BLAZE-1) [@clinicaltrials:NCT04427501] was initiated.
The interim analysis of the phase 2 portion suggested that bamlanivimab alone was able to accelerate the reduction in viral load [@doi:10.1056/NEJMoa2029849].
However, more recent data suggests that only the bamlanivimab/etesevimab combination therapy is able to reduce viral load in COVID-19 patients [@doi:10.1001/jama.2021.0202].
Based on this data, the combination therapy received an EUA for COVID-19 from the FDA in February 2021 [@url:https://www.fda.gov/media/145801/download].

A second therapy is comprised of casirivimab and imdevimab (REGN-COV2).
Casirivimab (REGN10933) and imdevimab (REGN10987) are two monoclonal antibodies against the SARS-CoV-2 spike protein.
They were both developed by Regeneron in a parallel high-throughput screening (HTS) to identify neutralizing antibodies from either humanized mice or patient-derived convalescent plasma [@doi:10.1126/science.abd0827].
In these efforts, multiple antibodies were characterized for their ability to bind and neutralize the SARS-CoV-2 spike protein.
The investigators hypothesized that an antibody cocktail, rather than each individual antibody, could increase the therapeutic efficacy while minimizing the risk for virus escape.
Therefore, the authors tested pairs of individual antibodies for their ability to simultaneously bind the RBD of the spike protein.
Based on this data, casirivimab and imdevimab were identified as the lead antibody pair, resulting in the initiation of two clinical trials [@clinicaltrials:NCT04426695; @clinicaltrials:NCT04425629].
Data from this phase 1-3 trial published in the _New England Journal of Medicine_ shows that the REGN-COV2 antibody cocktail reduced viral load, particularly in patients with high viral load or whose endogenous immune response had not yet been initiated [@doi:10.1056/NEJMoa2035002].
However, in patients who already initiated an immune response, exogenous addition of REGN-COV2 did not improve the endogenous immune response.
Both doses were well tolerated with no serious events related to the antibody cocktail.
Based on this data, the FDA granted an EUA for REGN-COV2 in patients with mild to moderate COVID-19 who are at risk of developing severe disease [@url:https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-monoclonal-antibodies-treatment-covid-19].
Ongoing efforts are trying to evaluate the efficacy of REGN-COV2 to improve clinical outcomes in hospitalized patients [@clinicaltrials:NCT04426695].

Sotrovimab is the most recent mAb to receive an EUA.
It was identified in the memory B cells of a 2003 survivor of SARS [@doi:10.1038/nm1080] and was found to be cross-reactive with SARS-CoV-2 [@doi:10.1038/s41586-020-2349-y].
This cross-reactivity is likely attributable to conservation within the epitope, with 17 out of 22 residues conserved between the two viruses, four conservatively substituted, and one semi-conservatively substituted [@doi:10.1038/s41586-020-2349-y].
In fact, these residues are highly conserved among sarbecoviruses, a clade that includes SARS-CoV-1 and SARS-CoV-2 [@doi:10.1038/s41586-020-2349-y].
This versatility has led to it being characterized as a "super-antibody" [@doi:10.1038/s41587-021-00980-x], a potent, broadly neutralizing antibody [@doi:10.1038/nri.2017.148].
Interim analysis of data from a clinical trial [@clinicaltrials:NCT04545060] reported high safety and efficacy of this mAb in 583 COVID-19 patients [@doi:10.1101/2021.05.27.21257096].
Compared to placebo, sotrovimab was found to be 85% more effective in reducing progression to the primary endpoint, which was the proportion of patients who, within 29 days, were either hospitalized for more than 24 hours or died.
Additionally, rates of adverse events were comparable, and in some cases lower, among patients receiving sotrovimab compared to patients receiving a placebo.
Sotrovimab therefore represents a mAb therapeutic that is effective against SARS-CoV-2 and may also be effective against other sarbecoviruses.

Several potential limitations remain in the application of mAbs to the treatment of COVID-19.
One of the biggest challenges is identifying antibodies that not only bind to their target, but also prove to be beneficial for disease management.
Currently, use of mAbs is limited to people with mild to moderate disease that are not hospitalized, and it has yet to be determined whether they can be used as a successful treatment option for severe COVID-19 patients.
While preventing people from developing severe illness provides significant benefits, patients with severe illness are at the greatest risk of death, and therefore therapeutics that provide benefits against severe illness are particularly desirable.
It remains to be seen whether mAbs confer any benefits for patients in this category.

Another concern about therapeutics designed to amplify the response to a specific viral target is that they may need to be modified as the virus evolves.
With the ongoing global spread of new SARS-CoV-2 variants, there is a growing concern that mutations in SARS-CoV-2 spike protein could escape antibody neutralization, thereby reducing the efficacy of monoclonal antibody therapeutics and vaccines.
A comprehensive mutagenesis screen recently identified several amino acid substitutions in the SARS-CoV-2 spike protein that can prevent antibody neutralization [@doi:10.1016/j.chom.2021.01.014].
While some mutations result in resistance to only one antibody, others confer broad resistance to multiple mAbs as well as polyclonal human sera, suggesting that some amino acids are "hotspots" for antibody resistance.
However, it was not investigated whether the resistance mutations identified result in a fitness advantage.
Accordingly, an impact on neutralizing efficiency has been reported for the B.1.1.7 (Alpha) variant first identified in the UK and the B.1.351 (Beta) variant first identified in in South Africa [@doi:10.21203/rs.3.rs-228079/v1; @doi:10.1101/2021.02.03.429355; @doi:10.1101/2021.01.25.428137].
As of June 25, 2021, the CDC recommended a pause in the use of bamlanivimab and etesevimab due to decreased efficacy against the P.1 (Gamma) and B.1.351 (Beta) variants of SARS-CoV-2 [@url:https://www.phe.gov/emergency/events/COVID19/investigation-MCM/Bamlanivimab-etesevimab/Pages/bamlanivimab-etesevimab-distribution-pause.aspx].
While the reported impact on antibody neutralization needs to be confirmed _in vivo_, it suggests that some adjustments to therapeutic antibody treatments may be necessary to maintain the efficacy that was reported in previous clinical trials.

Several strategies have been employed to try to mitigate the risk of diminished antibody neutralization.
Antibody cocktails such as those already holding an EUA may help overcome the risk for attenuation of the neutralizing activity of a single monoclonal antibody.
These cocktails consist of antibodies that recognize different epitopes on the spike protein, decreasing the likelihood that a single amino acid change can cause resistance to all antibodies in the cocktail.
However, neutralizing resistance can emerge even against an antibody cocktail if the individual antibodies target subdominant epitopes [@doi:10.1101/2021.02.03.429355].
Another strategy is to develop broadly neutralizing antibodies that target structures that are highly conserved, as these are less likely to mutate [@doi:10.1016/j.immuni.2007.11.018; @doi:10.1126/science.1171491] or to target epitopes that are insensitive to mutations [@doi:10.1016/j.vaccine.2006.04.054].
Sotrovimab, one such "super-antibody", is thought to be somewhat robust to neutralization escape [@doi:10.1086/651022] and has been found to be effective against all variants assessed as of August 12, 2021 [@url:https://covdb.stanford.edu/page/susceptibility-data].
Another antibody (ADG-2) targets a highly conserved epitope that overlaps the hACE2 binding site of all clade 1 sarbecoviruses [@doi:10.1126/science.abf4830].
Prophylactic administration of ADG-2 in an immunocompetent mouse model of COVID-19 resulted in protection against viral replication in the lungs and respiratory burden.
Since the epitope targeted by ADG-2 represents an Achilles’ heel for clade 1 sarbecoviruses, this antibody, like sotrovimab, might be a promising candidate against all circulating variants as well as emerging SARS-related coronaviruses.
To date, it has fared well against the Alpha, Beta, Gamma, and Delta variants [@url:https://covdb.stanford.edu/page/susceptibility-data].

The development of mAbs against SARS-CoV-2 has made it clear that this technology is rapidly adaptable and offers great potential for the response to emerging viral threats.
However, additional investigation may be needed to adapt mAb treatments to SARS-CoV-2 as it evolves and potentially to pursue designs that confer benefits for patients at the greatest risk of death.
While polyclonal antibodies from convalescent plasma have been evaluated as a treatment for COVID-19, these studies have suggested fewer potential benefits against SARS-CoV-2 than mAbs; convalescent plasma therapy has been thoroughly reviewed elsewhere [@doi:10/ggqr7s; @doi:10.1001/jama.2020.4940].
Thus, advances in biologics for COVID-19 illustrate that an understanding of how the host and virus interact can guide therapeutic approaches.
The FDA authorization of two combination mAb therapies, in particular, underscores the potential for this strategy to allow for a rapid response to a novel pathogen.
Additionally, while TCZ is not yet as established, this therapy suggests that the strategy of using biologics to counteract the cytokine storm response may provide therapies for the highest-risk patients.

### High-Throughput Screening for Drug Repurposing

The drug development process is slow and costly, and developing compounds specifically targeted to an emerging viral threat is not a practical short-term solution.
Screening existing drug compounds for alternative indications is a popular alternative [@doi:10.1016/j.drudis.2019.06.014; @doi:10.1111/bph.13895; @doi:10.1039/D0CS01065K; @doi:10.1016/j.coi.2013.08.004].
HTS has been a goal of pharmaceutical development since at least the mid-1980s [@doi:10.1038/sj.bjp.0707373].
Traditionally, phenotypic screens were used to test which compounds would induce a desired change in _in vitro_ or _in vivo_ models, focusing on empirical, function-oriented exploration naïve to molecular mechanism [@doi:10.1126/scitranslmed.aab1201; @doi:10.1038/clpt.2012.236; @doi:10.1038/nrd4366].
In many cases, these screens utilize large libraries that encompass a diverse set of agents varying in many pharmacologically relevant properties (e.g., [@url:https://www.chembridge.com/screening_libraries/diversity_libraries]).
The compounds inducing a desired effect could then be followed up on.
Around the turn of the millennium, advances in molecular biology allowed for HTS to shift towards screening for compounds interacting with a specific molecular target under the hypothesis that modulating that target would have a desired effect.
These approaches both offer pros and cons, and today a popular view is that they are most effective in combination [@doi:10.1038/nrd4366; @doi:10.1016/j.chembiol.2015.11.008; @doi:10.1126/scitranslmed.aab1201].

Today, some efforts to screen compounds for potential repurposing opportunities are experimental, but others use computational HTS approaches [@doi:10.1016/j.drudis.2019.06.014; @doi:10.1016/j.tim.2018.04.004].
Computational drug repurposing screens can take advantage of big data in biology [@doi:10.1038/nrd.2018.168] and as a result are much more feasible today than during the height of the SARS and MERS outbreaks in the early 2000s and early 2010s, respectively.
Advancements in robotics also facilitate the experimental component of HTS [@doi:10.1039/D0CS01065K].
For viral diseases, the goal of drug repurposing is typically to identify existing drugs that have an antiviral effect likely to impede the virus of interest.
While both small molecules and biologics can be candidates for repurposing, the significantly lower price of many small molecule drugs means that they are typically more appealing candidates [@doi:10.1089/adt.2019.950].

Depending on the study design, screens vary in how closely they are tied to a hypothesis.
As with the candidate therapeutics described above, high-throughput experimental or computational screens can proceed based on a hypothesis.
Just as remdesivir was selected as a candidate antiviral because it is a nucleoside analog [@doi:10.1021/acs.jcim.0c00861], so too can high-throughput screens select libraries of compounds based on a molecular hypothesis.
Likewise, when the library of drugs is selected without basis in a potential mechanism, a screen can be considered hypothesis free [@doi:10.1021/acs.jcim.0c00861].
Today, both types of analyses are common both experimentally and computationally.
Both strategies have been applied to identifying candidate therapeutics against SARS-CoV-2.

#### Hypothesis-Driven Screening

Hypothesis-driven screens often select drugs likely to interact with specific viral or host targets or drugs with desired clinical effects, such as immunosuppressants.
There are several properties that might identify a compound as a candidate for an emerging viral disease.
Drugs that interact with a target that is shared between pathogens (i.e., a viral protease or a polymerase) or between a viral pathogen and another illness (i.e., a cancer drug with antiviral potential) are potential candidates, as are drugs that are thought to interact with additional molecular targets beyond those they were developed for [@doi:10.1016/j.tim.2018.04.004].
Such research can be driven by _in vitro_ or _in silico_ experimentation.
Computational analyses depend on identifying compounds that modulate pre-selected proteins in the virus or host.
As a result, they build on experimental research characterizing the molecular features of the virus, host, and candidate compounds [@doi:10.1016/j.coi.2013.08.004].

One example of the application of this approach to COVID-19 research comes from work on protease inhibitors.
Studies have shown that viral proteases play an important role in the life cycle of viruses, including coronaviruses, by modulating the cleavage of viral polyprotein precursors [@doi:10.2174/138161207780162971].
Several FDA-approved drugs target proteases, such as lopinavir and ritonavir for HIV infection and simeprevir for hepatitis C virus infection.
Serine protease inhibitors were previously suggested as possible treatments for SARS and MERS [@doi:10.1016/j.antiviral.2015.01.011].
One early study [@doi:10.1016/j.cell.2020.02.052] suggested that camostat mesylate, a protease inhibitor, could block the entry of SARS-CoV-2 into lung cells _in vitro_.
Two polyproteins encoded by the SARS-CoV-2 replicase gene, pp1a and pp1ab, are critical for viral replication and transcription [@doi:10.1038/s41586-020-2223-y].
These polyproteins must undergo proteolytic processing, which is usually conducted by main protease (M^Pro^), a 33.8-kDa SARS-CoV-2 protease that is therefore fundamental to viral replication and transcription.
Therefore, it was hypothesized that compounds targeting M^Pro^ could be used to prevent or slow the replication of the SARS-CoV-2 virus.

Both computational and experimental approaches facilitated the identification of compounds that might inhibit SARS-CoV-2 M^Pro^.
In 2005, computer-aided design facilitated the development of a Michael acceptor inhibitor, now known as N3, to target M^Pro^ of SARS-like coronaviruses [@doi:10.1371/journal.pbio.0030324].
N3 binds in the substrate binding pocket of M^Pro^ in several HCoV [@doi:10.1007/s13238-013-2841-3; @doi:10.1371/journal.pbio.0030324; @doi:10.1038/srep22677; @doi:10.1128/JVI.02114-07].
The structure of N3-bound SARS-CoV-2 M^Pro^ has been solved, confirming the computational prediction that N3 would similarly bind in the substrate binding pocket of SARS-CoV-2 [@doi:10.1038/s41586-020-2223-y].
N3 was tested _in vitro_ on SARS-CoV-2-infected Vero cells, which belong to a line of cells established from the kidney epithelial cells of an African green monkey, and was found to inhibit SARS-CoV-2 [@doi:10.1038/s41586-020-2223-y].
A library of approximately 10,000 compounds was screened in a fluorescence resonance energy transfer assay constructed using SARS-CoV-2 M^Pro^ expressed in _Escherichia coli_ [@doi:10.1038/s41586-020-2223-y].

Six leads were identified in this hypothesis-driven screen.
_In vitro_ analysis revealed that ebselen had the strongest potency in reducing the viral load in SARS-CoV-2-infected Vero cells [@doi:10.1038/s41586-020-2223-y].
Ebselen is an organoselenium compound with anti-inflammatory and antioxidant properties [@doi:10.1007/s11033-014-3417-x].
Molecular dynamics analysis further demonstrated the potential for ebselen to bind to M^Pro^ and disrupt the protease's enzymatic functions [@doi:10.1126/sciadv.abd0345]. 
However, ebselen is likely to be a promiscuous binder, which could diminish its therapeutic potential [@doi:10.1038/s41586-020-2223-y; @doi:10.1039/c8cc04258f], and compounds with higher specificity may be needed to translate this mechanism effectively to clinical trials.
In July 2020, phase II clinical trials commenced to assess the effects of SPI-1005, an investigational drug from Sound Pharmaceuticals that contains ebselen [@url:https://www.contractpharma.com/contents/view_breaking-news/2020-08-31/fda-clears-spis-ebselen-for-phase-ii-covid-19-trials], on 60 adults presenting with each of moderate [@clinicaltrials:NCT04484025] and severe [@clinicaltrials:NCT04483973] COVID-19.
Other M^Pro^ inhibitors are also being evaluated in clinical trials [@clinicaltrials:NCT04535167; @clinicaltrials:NCT04960202; @clinicaltrials:NCT04960202].
Pending the results of clinical trials, N3 remains a computationally interesting compound based on both computational and experimental data, but whether these potential effects will translate to the clinic remains unknown.

#### Hypothesis-Free Screening

Hypothesis-free screens use a discovery-driven approach, where screens are not targeted to specific viral proteins, host proteins, or desired clinical modulation.
Hypothesis-free drug screening began twenty years ago with the testing of libraries of drugs experimentally.
Today, like many other areas of biology, _in silico_ analyses have become increasingly popular and feasible through advances in biological big data [@doi:10.1021/acs.jcim.0c00861; @doi:10.1080/23808993.2019.1617632].
Many efforts have collected data about interactions between drugs and SARS-CoV-2 and about the host genomic response to SARS-CoV-2 exposure, allowing for hypothesis-free computational screens that seek to identify new candidate therapeutics.
Thus, they utilize a systems biology paradigm to extrapolate the effect of a drug against a virus based on the host interactions with both the virus and the drug [@doi:10.1016/j.coi.2013.08.004].

Resources such as the COVID-19 Drug and Gene Set Library, which at the time of its publication contained 1,620 drugs sourced from 173 experimental and computational drug sets and 18,676 human genes sourced from 444 gene sets [@doi:10.1016/j.patter.2020.100090], facilitate such discovery-driven approaches.
Analysis of these databases indicated that some drugs had been identified as candidates across multiple independent analyses, including high-profile candidates such as CQ/HCQ and remdesivir [@doi:10.1016/j.patter.2020.100090].
Computational screening efforts can then mine databases and other resources to identify potential PPIs among the host, the virus, and established and/or experimental drugs [@doi:10.1038/s41467-020-17189-2].
Subject matter expertise from human users may be integrated to varying extents depending on the platform (e.g,. [@doi:10.1038/s41467-020-17189-2; @doi:10/grs9]).
These resources have allowed studies to identify potential therapeutics for COVID-19 without an _a priori_ reason for selecting them.

One example of a hypothesis-free screen for COVID-19 drugs comes from a PPI-network-based analysis that was published early in the pandemic [@doi:10.1038/s41586-020-2286-9].
Here, researchers cloned the proteins expressed by SARS-CoV-2 _in vitro_ and quantified 332 viral-host PPI using affinity purification mass spectrometry [@doi:10.1038/s41586-020-2286-9].
They identified two SARS-CoV-2 proteins (Nsp6 and Orf9c) that interacted with host Sigma-1 and Sigma-2 receptors.
Sigma receptors are located in the endoplasmic reticulum of many cell types, and type 1 and 2 Sigma receptors have overlapping but distinct affinities for a variety of ligands [@doi:10.1016/j.pharmthera.2009.07.001].
Molecules interacting with the Sigma receptors were then analyzed and found to have an effect on viral infectivity _in vitro_ [@doi:10.1038/s41586-020-2286-9].
A follow-up study evaluated the effect of perturbing these 332 proteins in two cells lines, A549 and Caco-2, using knockdown and knockout methods, respectively, and found that the replication of SARS-CoV-2 in cells from both lines was dependent on the expression of _SIGMAR1_, which is the gene that encodes the Sigma-1 receptor [@pubmed:33060197].
Following these results, drugs interacting with Sigma receptors were suggested as candidates for repurposing for COVID-19 (e.g., [@doi:10.3389/fphar.2020.582310]).
Because many well-known and affordable drugs interact with the Sigma receptors [@doi:10.1038/s41586-020-2286-9; @doi:10.2174/157015905774322516], they became a major focus of drug repurposing efforts.
Some of the drugs suggested by the apparent success of Sigma receptor-targeting drugs were already being investigated at the time.
HCQ, for example, forms ligands with both Sigma-1 and Sigma-2 receptors and was already being explored as a candidate therapeutic for COVID-19 [@doi:10.1038/s41586-020-2286-9].
Thus, this computational approach yielded interest in drugs whose antiviral activity was supported by initial _in vitro_ analyses.

Follow-up research, however, called into question whether the emphasis on drugs interacting with Sigma receptors might be based on a spurious association [@doi:10.1126/science.abi4708].
This study built on the prior work by examining whether antiviral activity among compounds correlated with their affinity for the Sigma receptors and found that it did not.
The study further demonstrated that cationic amphiphilicity was a shared property among many of the candidate drugs identified through both computational and phenotypic screens and that it was likely to be the source of many compounds' proposed antiviral activity [@doi:10.1126/science.abi4708].
Cationic amphiphilicity is associated with the induction of phospholipidosis, which is when phospholipids accumulate in the lysosome [@doi:10.1515/hsz-2019-0270]. 
Phospholipidosis can disrupt viral replication by inhibiting lipid processing [@doi:10.1080/01913123.2020.1850966] (see discussion of HCQ in Appendix 1).
However, phospholipidosis is known to translate poorly from _in vitro_ models to _in vivo_ models or clinical applications.
Thus, this finding suggested that these screens were identifying compounds that shared a physiochemical property rather than a specific target [@doi:10.1126/science.abi4708].
The authors further demonstrated that antiviral activity against SARS-CoV-2 _in vitro_ was correlated with the induction of phospholipidosis for drugs both with and without cationic amphiphilicity [@doi:10.1126/science.abi4708].
This finding supports the idea that the property of cationic amphicility was being detected as a proxy for the shared effect of phospholipidosis [@doi:10.1126/science.abi4708].
They demonstrated that phospholipidosis-inducing drugs were not effective at preventing viral propagation _in vivo_ in a murine model of COVID-19 [@doi:10.1126/science.abi4708].
Therefore, removing hits that induce phospholipidosis from computational and _in vitro_ experimental repurposing screens (e.g., [@doi:10.1016/j.celrep.2021.108959]) may help emphasize those that are more likely to provide clinical benefits.
This work illustrates the importance of considering confounding variables in computational screens, a principle that has been incorporated into more traditional approaches to drug development [@doi:10.1126/science.abj9488].

One drug that acts on Sigma receptors does, however, remain a candidate for the treatment of COVID-19.
Several psychotropic drugs target Sigma receptors in the central nervous system and thus attracted interest as potential COVID-19 therapeutics following the findings of two host-virus PPI studies [@doi:10.1007/s00406-020-01231-x]. 
For several of these drugs, the _in vitro_ antiviral activity [@pubmed:33060197] was not correlated with their affinity for the Sigma-1 receptor [@doi:10.1007/s00406-020-01231-x; @doi:10.1126/science.abi4708] but was correlated with phospholipidosis [@doi:10.1126/science.abi4708].
However, fluvoxamine, a selective serotonin reuptake inhibitor that is a particularly potent Sigma-1 receptor agonist [@doi:10.1007/s00406-020-01231-x], has shown promise as a preventative of severe COVID-19 in a preliminary analysis of data from the large-scale TOGETHER trial [@url:https://rethinkingclinicaltrials.org/news/august-6-2021-early-treatment-of-covid-19-with-repurposed-therapies-the-together-adaptive-platform-trial-edward-mills-phd-frcp].
As of August 6, 2021, this trial had collected data from over 1,400 patients in the fluvoxamine arm of their study, half of whom received a placebo [@url:https://rethinkingclinicaltrials.org/news/august-6-2021-early-treatment-of-covid-19-with-repurposed-therapies-the-together-adaptive-platform-trial-edward-mills-phd-frcp].
Only 74 patients in the fluvoxamine group had progressed to hospitalization for COVID-19 compared to 107 in the placebo group, corresponding to a relative risk of 0.69; additionally, the relative risk of mortality between the two groups was calculated at 0.71.
These findings support the results of small clinical trials that have found fluvoxamine to reduce clinical deterioration relative to a placebo [@doi:10.1001/jama.2020.22760; @doi:10.1093/ofid/ofab050].
However, the ongoing therapeutic potential of fluvoxamine does not contradict the finding that hypothesis-free screening hits can be driven by confounding factors.
The authors point out that its relevance would not just be antiviral as it has a potential immunomodulatory mechanism [@doi:10.1001/jama.2020.22760].
It has been found to be protective against septic shock in an _in vivo_ mouse model [@doi:10.1126/scitranslmed.aau5266].
It is possible that fluvoxamine also exerts an antiviral effect [@doi:10.3389/fphar.2021.652688].
Thus, Sigma-1 receptor activity may contribute to fluvoxamine's potential effects in treating COVID-19, but is not the only mechanism by which this drug can interfere with disease progression.

#### Potential and Limitations of High-Throughput Analyses

Computational screening allows for a large number of compounds to be evaluated to identify those most likely to display a desired behavior or function.
This approach can be guided by a hypothesis or can aim to discover underlying characteristics that produce new hypotheses about the relationship between a host, a virus, and candidate pharmaceuticals.
The examples outlined above illustrate that HTS-based evaluations of drug repurposing can potentially provide valuable insights. 
Computational techniques were used to design compounds targeting M^Pro^ based on an understanding of how this protease aids viral replication, and M^Pro^ inhibitors remain promising candidates [@doi:10.1039/D0CS01065K], although the clinical trial data is not yet available.
Similarly, computational analysis correctly identified the Sigma-1 receptor as a protein of interest.
Although the process of identifying which drugs might modulate the interaction led to an emphasis on candidates that ultimately have not been supported, fluvoxamine remains an appealing candidate.
The difference between the preliminary evidence for fluvoxamine compared to other drugs that interact with Sigma receptors underscores a major critique of hypothesis-free HTS in particular: while these approaches allow for brute force comparison of a large number of compounds against a virus of interest, they lose the element of expertise that is associated with most successes in drug repurposing [@doi:10.1021/acs.jcim.0c00861].

There are also practical limitations to these methods.
One concern is that computational analyses inherently depend on the quality of the data being evaluated.
The urgency of the COVID-19 pandemic led many research groups to pivot towards computational HTS research without familiarity with best practices in this area [@doi:10.1039/D0CS01065K].
As a result, there is an excessive amount of information available from computational studies [@url:https://blogs.sciencemag.org/pipeline/archives/2021/07/19/too-many-papers], but not all of it is high-quality.
Additionally, the literature used to identify and validate targets can be difficult to reproduce [@doi:10.1038/nrd3439-c1], which may pose challenges to target-based experimental screening and to _in silico_ screens. 
Some efforts to repurpose antivirals have focused on host, rather than viral, proteins [@doi:10.1016/j.coi.2013.08.004], which might be expected to translate poorly _in vivo_ if the targeted proteins serve essential functions in the host.
Concerns about the practicality of hypothesis-free screens to gain novel insights are underscored by the fact that very few or possibly no success stories have emerged from hypothesis-free screens over the past twenty years [@doi:10.1021/acs.jcim.0c00861].
These findings suggest that data-driven research can be an important component of the drug repurposing ecosystem, but that drug repurposing efforts that proceed without a hypothesis, an emphasis on biological mechanisms, or an understanding of confounding effects may not produce viable candidates.

### Considerations in Balancing Different Approaches

The approaches described here offer a variety of advantages and limitations in responding to a novel viral threat and building on existing bodies of knowledge in different ways.
Medicine, pharmacology, basic science (especially virology and immunology), and biological data science can all provide different insights and perspectives for addressing the challenging question of which existing drugs might provide benefits against an emerging viral threat.
A symptom management-driven approach allows clinicians to apply experience with related diseases or related symptoms to organize a rapid response aimed at saving the lives of patients already infected with a new disease.
Oftentimes, the pharmaceutical agents that are applied are small-molecule, broad-spectrum pharmaceuticals that are widely available and affordable to produce, and they may already be available for other purposes, allowing clinicians to administer them to patients quickly either with an EUA or off-label.
In this vein, dexamethasone has emerged as the strongest treatment against severe COVID-19 (Table @tbl:summary-table).

Alternatively, many efforts to repurpose drugs for COVID-19 have built on information gained through basic scientific research of HCoV.
Understanding how related viruses function has allowed researchers to identify possible pharmacological strategies to disrupt pathogenesis (Figure @fig:therapeutics).
Some of the compounds identified through these methods include small-molecule antivirals, which can be boutique and experimental medications like remdesivir (Table @tbl:summary-table).
Other candidate drugs that intercept host-pathogen interactions include biologics, which imitate the function of endogenous host compounds.
Most notably, several mAbs that have been developed (casirivimab, imdevimab, bamlanivimab and etesevimab) or repurposed (sotrovimab, tocilizumab) have now been granted EUAs (Table @tbl:summary-table).
Although not discussed here, several vaccine development programs have also met huge success using a range of strategies [@individual-vaccines].

| Treatment | Grade | Category | FDA Status | Evidence Available | Suggested Effectiveness |
|-----------|-------|----------|-----------------|------------------|-------------------------|
| Dexamethasone | A | Small molecule, broad spectrum | Used off-label | RCT | Supported: RCT shows improved outcomes over SOC, especially in severe cases such as CRS |
|Remdesivir | A | Small molecule, antiviral, adenosine analog | Approved for COVID-19 (and EUA for combination with baricitinib) | RCT | Mixed: Conflicting evidence from large WHO-led Solidarity trial vs US-focused RCT and other studies |
| Tocilizumab |A | Biologic, monoclonal antibody | EUA | RCT | Mixed: It appears that TCZ may work well in combination with dexamethasone in severe cases, but not as monotherapy |
| Sotrovimab | N/A | Biological, monoclonal antibody | EUA | RCT | Supported: Phase 2/3 clinical trial showed reduced hospitalization/death
| Bamlanivimab and etesevimab | B & N/A | Biologic, monoclonal antibodies | EUA | RCT | Supported: Phase 2 clinical trial showed reduction in viral load, but FDA pause recommended because may be less effective against Delta variant|
| Casirivimab and imdevimab | N/A | Biologic, monoclonal antibodies | EUA | RCT | Supported: Reduced viral load at interim analysis |
| Fluvoxamine | B | Small-molecule, Sigma-1 receptor agonist | N/A | RCT | Supported: Support from two small RCTs and preliminary support from interim analysis of TOGETHER
| SNG001 | B | Biologic, interferon | None | RCT | Mixed: Support from initial RCT but no effect found in WHO's Solidarity trial |
| M^Pro^ Protease Inhibitors | N/A | Small molecule, protease inhibitor | None | Computational prediction, _in vitro_ studies | Unknown |
| ARBs & ACEIs | C | Small molecule, broad spectrum | None | Observational studies and some RCTs | Not supported: Observational study retracted, RCTs suggest no association |
| Favipiravir | D | Small molecule, antiviral, nucleoside analog | None | RCT | Not supported: RCTs do not show significant improvements for individuals taking this treatment, good safety profile |
| HCQ/CQ | D | Small molecule, broad spectrum | None | RCT | Not supported, possibly harmful: Non-blinded RCTs showed no improvement over SOC, safety profile may be problematic |
| Convalescent plasma transfusion | D | Biologic, polyclonal antibodies | EUA | RCT | Mixed: Supported in small trials but not in large-scale RECOVERY trial
| Ivermectin | D | Small molecule, broad spectrum | None | RCT | Mixed: Mixed results from small RCTs, major supporting RCT now withdrawn, preliminary results of large RCT (TOGETHER) suggest no effect on emergency room visits or hospitalization for COVID-19

Table: Summary table of candidate therapeutics examined in this manuscript.
"Grade" is the rating given to each treatment by the Systematic Tracker of Off-label/Repurposed Medicines Grades (STORM) maintained by the Center for Cytokine Storm Treatment & Laboratory (CSTL) at the University of Pennsylvania [@url:https://cdcn.org/corona-data-viewer].
A grade of A indicates that a treatment is considered effective, B that all or most RCTs have shown positive results, C that RCT data are not yet available, and D that multiple RCTs have produced negative results.
Treatments not in the STORM database are indicated as N/A.
FDA status is also provided where available.
The evidence available is based on the progression of the therapeutic through the pharmaceutical development pipeline, with RCTs as the most informative source of evidence.
The effectiveness is summarized based on the current available evidence; large trials such as RECOVERY and Solidarity are weighted heavily in this summary.
This table was last updated on August 20, 2021.
{#tbl:summary-table} 

All of the small-molecule drugs evaluated and most of the biologics are repurposed, and thus hinge on a theoretical understanding of how the virus interacts with a human host and how pharmaceuticals can be used to modify those interactions rather than being designed specifically against SARS-CoV-2 or COVID-19.
As a result, significant attention has been paid to computational approaches that automate the identification of potentially desirable interactions.
However, work in COVID-19 has made it clear that relevant compounds can also be masked by confounds, and spurious associations can drive investment in candidate therapeutics that are unlikely to translate to the clinic.
Such spurious hits are especially likely to impact hypothesis-free screens.
However, hypothesis-free screens may still be able to contribute to the drug discovery or repurposing ecosystem, assuming the computational arm of HTS follows the same trends seen in its experimental arm.
In 2011, a landmark study in drug discovery demonstrated that although more new drugs were discovered using target-based rather than phenotypic approaches, the majority of drugs with a novel molecular mechanism of action (MMOA) were identified in phenotypic screens [@doi:10.1038/nrd3480].
This pattern applied only to first-in-class drugs, with most follower drugs produced by target-based screening [@doi:10.1038/clpt.2012.236].
These findings suggest that target-based drug discovery is more successful when building on a known MMOA, and that modulating a target is most valuable when the target is part of a valuable MMOA [@doi:10.1038/nrd4366].
Building on this, many within the field suggested that mechanism-informed phenotypic investigations may be the most useful approach to drug discovery [@doi:10.1038/nrd4366; @doi:10.1016/j.chembiol.2015.11.008; @doi:10.1126/scitranslmed.aab1201].
As it stands, data-driven efforts to identify patterns in the results of computational screens allowed researchers to notice the shared property of cationic amphicility among many of the hits from computational screening analyses [@doi:10.1126/science.abi4708].
While easier said than done, efforts to fill in the black box underlying computational HTS and recognize patterns among the identified compounds aid in moving data-oriented drug repurposing efforts in this direction.

The unpredictable nature of success and failure in drug repurposing for COVID-19 thus highlights one of the tenets of phenotypic screening: there are a lot of "unknown unknowns", and a promising mechanism at the level of an MMOA will not necessarily propagate up to the pathway, cellular, or organismal level [@doi:10.1126/scitranslmed.aab1201].
Despite the fact that apparently mechanistically relevant drugs may exist, identifying effective treatments for a new viral disease is extremely challenging.
Targets of repurposed drugs are often non-specific, meaning that the MMOA can appear to be relevant to COVID-19 without a therapeutic or prophylactic effect being observed in clinical trials.
The difference in the current status of remdesivir and favipiravir as treatments for COVID-19 (Table @tbl:summary-table) underscores how difficult to predict whether a specific compound will produce a desired effect, even when the mechanisms are similar.
Furthermore, the fact that many candidate COVID-19 therapeutics were ultimately identified because of their shared propensity to induce phospholipidosis underscores how challenging it can be to identify a mechanism _in silico_ or _in vitro_ that will translate to a successful treatment.
While significant progress has been made thus far in the pandemic, the therapeutic landscape is likely to continue to evolve as more results become available from clinical trials and as efforts to develop novel therapeutics for COVID-19 progress.

### Towards the Next HCoV Threat

Only very limited testing of candidate therapies was feasible during the SARS and MERS epidemics, and as a result, few treatments were available at the outset of the COVID-19 pandemic.
Even corticosteroids, which were used to treat SARS patients, were a controversial therapeutic prior to the release of the results of the large RECOVERY trial. 
The scale and duration of the COVID-19 pandemic has made it possible to conduct large, rigorous RCTs such as RECOVERY, Solidarity, TOGETHER, and others.
As results from these trials have continued to emerge, it has become clear that small clinical trials often produce spurious results.
In the case of HCQ/CQ, the therapeutic had already attracted so much attention based on small, preliminary (and in some cases, methodologically concerning) studies that it took the results of multiple large studies before attention began to be redirected to more promising candidates [@doi:10.1126/science.368.6496.1166].
In fact, most COVID-19 clinical trials lack the statistical power to reliably test their hypotheses [@doi:10.1038/d41573-021-00037-3; @doi:10.2196/26718].
In the face of an urgent crisis like COVID-19, the desire to act quickly is understandable, but it is imperative that studies maintain strict standards of scientific rigor [@doi:10.1126/science.abj9488; @doi:10.1039/D0CS01065K], especially given the potential dangers of politicization, as illustrated by HCQ/CQ [@doi:10.4269/ajtmh.20-1320].
Potential innovations in clinical trial structure, such as adaptable clinical trials with master protocols [@doi:10.1016/j.cct.2021.106438] or the sharing of data among small clinical trials [@doi:10.2196/26718] may help to address future crises and to bolster the results from smaller studies, respectively. 

In the long-term, new drugs specific for treatment of COVID-19 may also enter development.
Development of novel drugs is likely to be guided by what is known about the pathogenesis and molecular structure of SARS-CoV-2.
For example, understanding the various structural components of SARS-CoV-2 may allow for the development of small molecule inhibitors of those components.
Crystal structures of the SARS-CoV-2 main protease have been resolved [@doi:10.1038/s41586-020-2223-y; @url:https://www.diamond.ac.uk/covid-19/for-scientists/Main-protease-structure-and-XChem.html].
Much work remains to be done to determine further crystal structures of other viral components, understand the relative utility of targeting different viral components, perform additional small molecule inhibitor screens, and determine the safety and efficacy of the potential inhibitors.
While still nascent, work in this area is promising.
Over the longer term, this approach and others may lead to the development of novel therapeutics specifically for COVID-19 and SARS-CoV-2.
Such efforts are likely to prove valuable in managing future emergent HCoV, just as research from the SARS and MERS pandemic has provided a basis for the COVID-19 response.



## Appendix: Identification and Development of Therapeutics for COVID-19

### Dexamethasone

In order to understand how dexamethasone reduces inflammation, it is necessary to consider the stress response broadly.
In response to stress, corticotropin‐releasing hormone stimulates the release of neurotransmitters known as catecholamines, such as epinephrine, and steroid hormones known as glucocorticoids, such as cortisol [@doi:10.1016/j.cyto.2015.01.008; @doi:10.1111/j.1749-6632.2002.tb04229.x].
While catecholamines are often associated with the fight-or-flight response, the specific role that glucocorticoids play is less clear, although they are thought to be important to restoring homeostasis [@doi:10.1159/000054578].
Immune challenge is a stressor that is known to interact closely with the stress response.
The immune system can therefore interact with the central nervous system; for example, macrophages can both respond to and produce catecholamines [@doi:10.1016/j.cyto.2015.01.008].
Additionally, the production of both catecholamines and glucocorticoids is associated with inhibition of proinflammatory cytokines such as IL-6, IL-12, and tumor necrosis factor-&alpha; (TNF‐&alpha;) and the stimulation of anti-inflammatory cytokines such as IL-10, meaning that the stress response can regulate inflammatory immune activity [@doi:10.1111/j.1749-6632.2002.tb04229.x].
Administration of dexamethasone has been found to correspond to dose-dependent inhibition of IL-12 production, but not to affect IL-10 [@PMID:8902882]; the fact that this relationship could be disrupted by administration of a glucocorticoid-receptor antagonist suggests that it is regulated by the receptor itself [@PMID:8902882].
Thus, the administration of dexamethasone for COVID-19 is likely to simulate the release of glucocorticoids endogenously during stress, resulting in binding of the synthetic steroid to the glucocorticoid receptor and the associated inhibition of the production of proinflammatory cytokines.
In this model, dexamethasone reduces inflammation by stimulating the biological mechanism that reduces inflammation following a threat such as immune challenge.

Initial support for dexamethasone as a treatment for COVID-19 came from the United Kingdom's RECOVERY trial [@doi:10.1101/2020.06.22.20137273], which assigned over 6,000 hospitalized COVID-19 patients to the standard of care (SOC) or treatment (dexamethasone) arms of the trial at a 2:1 ratio.
At the time of randomization, some patients were ventilated (16%), others were on non-invasive oxygen (60%), and others were breathing independently (24%).
Patients in the treatment arm were administered dexamethasone either orally or intravenously at 6 mg per day for up to 10 days.
The primary end-point was the patient's status at 28-days post-randomization (mortality, discharge, or continued hospitalization), and secondary outcomes analyzed included the progression to invasive mechanical ventilation over the same period.
The 28-day mortality rate was found to be lower in the treatment group than in the SOC group (21.6% vs 24.6%, _p_ < 0.001).
However, the effect was driven by improvements in patients receiving mechanical ventilation or supplementary oxygen.
One possible confounder is that patients receiving mechanical ventilation tended to be younger than patients who were not receiving respiratory support (by 10 years on average) and to have had symptoms for a longer period.
However, adjusting for age did not change the conclusions, although the duration of symptoms was found to be significantly associated with the effect of dexamethasone administration.
Thus, this large, randomized, and multi-site, albeit not placebo-controlled, study suggests that administration of dexamethasone to patients who are unable to breathe independently may significantly improve survival outcomes.
Additionally, dexamethasone is a widely available and affordable medication, raising the hope that it could be made available to COVID-19 patients globally.

It is not surprising that administration of an immunosuppressant would be most beneficial in severe cases where the immune system was dysregulated towards inflammation.
However, it is also unsurprising that care must be taken in administering an immunosuppressant to patients fighting a viral infection.
In particular, the concern has been raised that treatment with dexamethasone might increase patient susceptibility to concurrent (e.g., nosocomial) infections [@doi:10.23812/20-EDITORIAL_1-5].
Additionally, the drug could potentially slow viral clearance and inhibit patients' ability to develop antibodies to SARS-CoV-2 [@doi:10.23812/20-EDITORIAL_1-5; @doi:10/ggks86], with the lack of data about viral clearance being put forward as a major limitation of the RECOVERY trial [@doi:10/ftk4].
Furthermore, dexamethasone has been associated with side effects that include psychosis, glucocorticoid-induced diabetes, and avascular necrosis [@doi:10/ggks86], and the RECOVERY trial did not report outcomes with enough detail to be able to determine whether they observed similar complications.
The effects of dexamethasone have also been found to differ among populations, especially in high-income versus middle- or low-income countries [@doi:10/gg42kx].
However, since the RECOVERY trial's results were released, strategies have been proposed for administering dexamethasone alongside more targeted treatments to minimize the likelihood of negative side effects [@doi:10.23812/20-EDITORIAL_1-5].
Given the available evidence, dexamethasone is currently the most promising treatment for severe COVID-19.

### Favipiravir

The effectiveness of favipiravir for treating patients with COVID-19 is currently under investigation.
Evidence for the drug inhibiting viral RNA polymerase are based on time-of-drug addition studies that found that viral loads were reduced with the addition of favipiravir in early times post-infection [@doi:10.1128/AAC.49.3.981-986.2005; @doi:10.1016/j.bbrc.2012.07.034; @doi:10.1128/AAC.01219-10].
An open-label, nonrandomized, before-after controlled study for COVID-19 was recently conducted [@doi:10.1016/j.eng.2020.03.007].
The study included 80 COVID-19 patients (35 treated with favipiravir, 45 control) from the isolation ward of the National Clinical Research Center for Infectious Diseases (The Third People’s Hospital of Shenzhen), Shenzhen, China.
The patients in the control group were treated with other antivirals, such as lopinavir and ritonavir.
It should be noted that although the control patients received antivirals, two subsequent large-scale analyses, the WHO Solidarity trial and the Randomized Evaluation of COVID-19 Therapy (RECOVERY) trial, identified no effect of lopinavir or of a lopinavir-ritonavir combination, respectively, on the metrics of COVID-19-related mortality that each assessed [@doi:10.1056/NEJMoa2023184; @doi:10/fnx2; @doi:10.1056/NEJMe2034294].
Treatment was applied on days 2-14; treatment stopped either when viral clearance was confirmed or at day 14.
The efficacy of the treatment was measured by, first, the time until viral clearance using Kaplan-Meier survival curves, and, second, the improvement rate of chest computed tomography (CT) scans on day 14 after treatment.
The study found that favipiravir increased the speed of recovery, measured as viral clearance from the patient by RT-PCR, with patients receiving favipiravir recovering in four days compared to 11 days for patients receiving antivirals such as lopinavir and ritonavir.
Additionally, the lung CT scans of patients treated with favipiravir showed significantly higher improvement rates (91%) on day 14 compared to control patients (62%, _p_ = 0.004).
However, there were adverse side effects in 4 (11%) favipiravir-treated patients and 25 (56%) control patients.
The adverse side effects included diarrhea, vomiting, nausea, rash, and liver and kidney injury.
Despite the study reporting clinical improvement in favipiravir-treated patients, several study design issues are problematic and lower confidence in the overall conclusions.
For example, the study was neither randomized nor blinded.
Moreover, the selection of patients did not take into consideration important factors such as previous clinical conditions or sex, and there was no age categorization.
Additionally, it should be noted that this study was temporarily retracted and then restored without an explanation [@url:https://retractionwatch.com/retracted-coronavirus-covid-19-papers].

In late 2020 and early 2021, the first randomized controlled trials of favipiravir for the treatment of COVID-19 released results [@doi:10.1016/j.ejps.2020.105631; @doi:10.1007/s00705-021-04956-9; @doi:10.1093/cid/ciaa1176].
The first [@doi:10.1016/j.ejps.2020.105631] used a randomized, controlled, open-label design to compare two drugs, favipiravir and baloxavir marboxil, to SOC alone.
Here, SOC included antivirals such as lopinavir/ritonavir and was administered to all patients.
The primary endpoint analyzed was viral clearance at day 14.
The sample size for this study was very small, with 29 total patients enrolled, and no significant effect of the treatments was found for the primary or any of the secondary outcomes analyzed, which included mortality.
The second study [@doi:10.1007/s00705-021-04956-9] was larger, with 96 patients enrolled, and included only individuals with mild to moderate symptoms who were randomized into two groups: one receiving chloroquine (CQ) in addition to SOC, and the other receiving favipiravir in addition to SOC.
This study reported a non-significant trend for patients receiving favipiravir to have a shorter hospital stay (13.29 days compared to 15.89 for CQ, _p_ = 0.06) and less likelihood of progressing to mechanical ventilation (_p_ = 0.118) or to an oxygen saturation < 90% (_p_ = 0.129).
These results, combined with the fact that favipiravir was being compared to CQ, which is now widely understood to be ineffective for treating COVID-19, thus do not suggest that favipiravir was likely to have had a strong effect on these outcomes.
On the other hand, another trial of 60 patients reported a significant effect of favipiravir on viral clearance at four days (a secondary endpoint), but not at 10 days (the primary endpoint) [@doi:10.1093/cid/ciaa1176].
This study, as well as a prior study of favipiravir [@doi:10/ftgm], also reported that the drug was generally well-tolerated.
Thus, in combination, these small studies suggest that the effects of favipiravir as a treatment for COVID-19 cannot be determined based on the available evidence, but additionally, none raise major concerns about the safety profile of the drug.

### Remdesivir

At the outset of the COVID-19 pandemic, remdesivir did not have any have any FDA-approved use.
A clinical trial in the Democratic Republic of Congo found some evidence of effectiveness against ebola virus disease (EVD), but two antibody preparations were found to be more effective, and remdesivir was not pursued [@doi:10.1056/NEJMoa1910993].
Remdesivir also inhibits polymerase and replication of the coronaviruses MERS-CoV and SARS-CoV-1 in cell culture assays with submicromolar IC50s [@doi:10.1126/scitranslmed.aal3653].
It has also been found to inhibit SARS-CoV-2, showing synergy with CQ _in vitro_ [@doi:10.1038/s41422-020-0282-0].

Remdesivir was first used on some COVID-19 patients under compassionate use guidelines [@doi:10.1126/science.abb7243, @doi:10.1056/NEJMoa2001191; @doi:10.1101/2020.03.09.20032896].
All were in late stages of COVID-19 infection, and initial reports were inconclusive about the drug's efficacy.
Gilead Sciences, the maker of remdesivir, led a recent publication that reported outcomes for compassionate use of the drug in 61 patients hospitalized with confirmed COVID-19.
Here, 200 mg of remdesivir was administered intravenously on day 1, followed by a further 100 mg/day for 9 days [@doi:10.1056/NEJMoa2007016].
There were significant issues with the study design, or lack thereof.
There was no randomized control group.
The inclusion criteria were variable: some patients only required low doses of oxygen, while others required ventilation.
The study included many sites, potentially with variable inclusion criteria and treatment protocols.
The patients analyzed had mixed demographics.
There was a short follow-up period of investigation.
Eight patients were excluded from the analysis mainly due to missing post-baseline information; thus, their health was unaccounted for.
Therefore, even though the study reported clinical improvement in 68% of the 53 patients ultimately evaluated, due to the significant issues with study design, it could not be determined whether treatment with remdesivir had an effect or whether these patients would have recovered regardless of treatment.
Another study comparing 5- and 10-day treatment regimens reported similar results but was also limited because of the lack of a placebo control [@doi:10.1056/NEJMoa2015301].
These studies did not alter the understanding of the efficacy of remdesivir in treating COVID-19, but the encouraging results provided motivation for placebo-controlled studies.

The double-blind placebo-controlled ACTT-1 trial [@doi:10.1056/NEJMoa2007764; @clinicaltrials:NCT04280705] recruited 1,062 patients and randomly assigned them to placebo treatment or treatment with remdesivir.
Patients were stratified for randomization based on site and the severity of disease presentation at baseline [@doi:10.1056/NEJMoa2007764].
The treatment was 200 mg on day 1, followed by 100 mg on days 2 through 10.
Data was analyzed from a total of 1,059 patients who completed the 29-day course of the trial, with 517 assigned to remdesivir and 508 to placebo [@doi:10.1056/NEJMoa2007764].
The two groups were well matched demographically and clinically at baseline.
Those who received remdesivir had a median recovery time of 10 days, as compared with 15 days in those who received placebo (rate ratio for recovery, 1.29; 95% confidence interval (CI), 1.12 to 1.49; _p_ < 0.001).
The Kaplan-Meier estimates of mortality by 14 days were 6.7% with remdesivir and 11.9% with placebo, with a hazard ratio (HR) for death of 0.55 and a 95% CI of 0.36 to 0.83, and at day 29, remdesivir corresponded to 11.4% and the placebo to 15.2% (HR: 0.73; 95% CI, 0.52 to 1.03).
Serious adverse events were reported in 131 of the 532 patients who received remdesivir (24.6%) and in 163 of the 516 patients in the placebo group (31.6%).
This study also reported an association between remdesivir administration and both clinical improvement and a lack of progression to more invasive respiratory intervention in patients receiving non-invasive and invasive ventilation at randomization [@doi:10.1056/NEJMoa2007764].
Largely on the results of this trial, the FDA reissued and expanded the EUA for remdesivir for the treatment of hospitalized COVID-19 patients ages twelve and older [@url:https://www.fda.gov/media/137564/download].
Additional clinical trials [@doi:10.1038/s41422-020-0282-0; @clinicaltrials:NCT04292730; @url:https://www.clinicaltrialsregister.eu/ctr-search/trial/2020-000936-23/FR; @clinicaltrials:NCT04252664; @clinicaltrials:NCT04257656] are currently underway to evaluate the use of remdesivir to treat COVID-19 patients at both early and late stages of infection and in combination with other drugs (Figure @fig:ebm-trials).
As of October 22, 2020, remdesivir received FDA approval based on three clinical trials [@url:https://www.fda.gov/news-events/press-announcements/fda-approves-first-treatment-covid-19].

However, results suggesting no effect of remdesivir on survival were reported by the WHO Solidarity trial [@doi:10.1056/NEJMoa2023184].
Patients were randomized in equal proportions into four experimental conditions and a control condition, corresponding to four candidate treatments for COVID-19 and SOC, respectively; no placebo was administered.
The 2,750 patients in the remdesivir group were administered 200 mg intravenously on the first day and 100 mg on each subsequent day until day 10 and assessed for in-hospital death (primary endpoint), duration of hospitalization, and progression to mechanical ventilation.
There were also 2,708 control patients who would have been eligible and able to receive remdesivir were they not assigned to the control group.
A total of 604 patients among these two cohorts died during initial hospitalization, with 301 in the remdesivir group and 303 in the control group.
The rate ratio of death between these two groups was therefore not significant (0.95, _p_ = 0.50), suggesting that the administration of remdesivir did not affect survival.
The two secondary analyses similarly did not find any effect of remdesivir.
Additionally, the authors compared data from their study with data from three other studies of remdesivir (including [@doi:10.1056/NEJMoa2007764]) stratified by supplemental oxygen status.
A meta-analysis of the four studies yielded an overall rate ratio for death of 0.91 (_p_ = 0.20).
These results thus do not support the previous findings that remdesivir reduced median recovery time and mortality risk in COVID-19 patients.

In response to the results of the Solidarity trial, Gilead, which manufactures remdesivir, released a statement pointing to the fact that the Solidarity trial was not placebo-controlled or double-blind and at the time of release, the statement had not been peer reviewed [@url:https://www.gilead.com/news-and-press/company-statements/gilead-sciences-statement-on-the-solidarity-trial]; these sentiments have been echoed elsewhere [@doi:10.1093/eurheartj/ehaa934].
Other critiques of this study have noted that antivirals are not typically targeted at patients with severe illness, and therefore remdesivir could be more beneficial for patients with mild rather than severe cases [@url:https://www.sciencemag.org/news/2020/10/very-very-bad-look-remdesivir-first-fda-approved-covid-19-drug; @doi:10.1056/NEJMe2034294].
However, the publication associated with the trial sponsored by Gilead did purport an effect of remdesivir on patients with severe disease, identifying an 11 versus 18 day recovery period (rate ratio for recovery: 1.31, 95% CI 1.12 to 1.52) [@doi:10.1056/NEJMoa2007764].
Additionally, a smaller analysis of 598 patients, of whom two-thirds were randomized to receive remdesivir for either 5 or 10 days, reported a small effect of treatment with remdesivir for five days relative to standard of care in patients with moderate COVID-19 [@doi:10.1001/jama.2020.16349].
These results suggest that remdesivir could improve outcomes for patients with moderate COVID-19, but that additional information would be needed to understand the effects of different durations of treatment.
Therefore, the Solidarity trial may point to limitations in the generalizability of other research on remdesivir, especially since the broad international nature of the Solidarity clinical trial, which included countries with a wide range of economic profiles and a variety of healthcare systems, provides a much-needed global perspective in a pandemic [@doi:10.1056/NEJMe2034294].
On the other hand, only 62% of patients in the Solidarity trial were randomized on the day of admission or one day afterwards [@doi:10.1056/NEJMoa2023184], and concerns have been raised that differences in disease progression could influence the effectiveness of remdesivir [@doi:10.1056/NEJMe2034294].
Despite the findings of the Solidarity trial, remdesivir remains available for the treatment of COVID-19 in many places.
Remdesivir has also been investigated in combination with other drugs, such as baricitinib, which is an inhibitor of Janus kinase 1 and 2 [@doi:10.1056/NEJMoa2031994]; the FDA has issued an EUA for the combination of remdesivir and baricitinib in adult and pediatric patients [@url:https://www.fda.gov/media/143822/download].
Follow-up studies are needed and, in many cases, are underway to further investigate remdesivir-related outcomes.

Similarly, the extent to which the remdesivir dosing regimen could influence outcomes continues to be under consideration.
A randomized, open-label trial compared the effect of remdesivir on 397 patients with severe COVID-19 over 5 versus 10 days [@doi:10.1056/NEJMoa2015301; @ClinicalTrials:NCT04292899], complementing the study that found that a 5-day course of remdesivir improved outcomes for patients with moderate COVID-19 but a 10-day course did not [@doi:10.1001/jama.2020.16349].
Patients in the two groups were administered 200 mg of remdesivir intravenously on the first day, followed by 100 mg on the subsequent four or nine days, respectively.
The two groups differed significantly in their clinical status, with patients assigned to the 10-day group having more severe illness.
This study also differed from most because it included not only adults, but also pediatric patients as young as 12 years old.
It reported no significant differences across several outcomes for patients receiving a 5-day or 10-day course, when correcting for baseline clinical status.
The data did suggest that the 10-day course might reduce mortality in the most severe patients at day 14, but the representation of this group in the study population was too low to justify any conclusions [@doi:10.1056/NEJMoa2015301].
Thus, additional research is also required to determine whether the dosage and duration of remdesivir administration influences outcomes.

In summary, remdesivir is the first FDA approved anti-viral against SARS-CoV-2 as well as the first FDA approved COVID-19 treatment.
Early investigations of this drug established proof of principle that drugs targeting the virus can benefit COVID-19 patients.
Moreover, one of the most successful strategies for developing therapeutics for viral diseases is to target the viral replication machinery, which are typically virally encoded polymerases.
Small molecule drugs targeting viral polymerases are the backbones of treatments for other viral diseases including human immunodeficiency virus (HIV) and herpes.
Notably, the HIV and herpes polymerases are a reverse transcriptase and a DNA polymerase, respectively, whereas SARS-CoV-2 encodes an RdRP, so most of the commonly used polymerase inhibitors are not likely to be active against SARS-CoV-2.
In clinical use, polymerase inhibitors show short term benefits for HIV patients, but for long term benefits they must be part of combination regimens.
They are typically combined with protease inhibitors, integrase inhibitors, and even other polymerase inhibitors.
Remdesivir provides evidence that a related approach may be beneficial for the treatment of COVID-19.

### Hydroxychloroquine and Chloroquine

CQ and hydroxychloroquine (HCQ) increase cellular pH by accumulating in their protonated form inside lysosomes [@doi:10.1186/1743-422X-8-163; @doi:10.1111/j.1529-8019.2007.00131.x].
This shift in pH inhibits the breakdown of proteins and peptides by the lysosomes during the process of proteolysis [@doi:10.1111/j.1529-8019.2007.00131.x].
Interest in CQ and HCQ for treating COVID-19 was catalyzed by a mechanism observed in _in vitro_ studies of both SARS-CoV-1 and SARS-CoV-2.
In one study, CQ inhibited viral entry of SARS-CoV-1 into Vero E6 cells, a cell line that was derived from Vero cells in 1968, through the elevation of endosomal pH and the terminal glycosylation of ACE2 [@doi:10.1186/1743-422X-2-69].
Increased pH within the cell, as discussed above, inhibits proteolysis, and terminal glycosylation of ACE2 is thought to interfere with virus-receptor binding.
An _in vitro_ study of SARS-CoV-2 infection of Vero cells found both HCQ and CQ to be effective in inhibiting viral replication, with HCQ being more potent [@doi:10.1093/cid/ciaa237].
Additionally, an early case study of three COVID-19 patients reported the presence of antiphospholipid antibodies in all three patients [@doi:10.1056/NEJMc2007575].
Antiphospholipid antibodies are central to the diagnosis of the antiphospholipid syndrome, a disorder that HCQ has often been used to treat [@doi:10.1182/blood.V128.22.5023.5023; @doi:10.1016/j.autrev.2014.01.053; @doi:10.1182/asheducation-2016.1.714].
Because the 90% effective concentration (EC~90~) of CQ in Vero E6 cells (6.90 μM) can be achieved in and tolerated by rheumatoid arthritis (RA) patients, it was hypothesized that it might also be possible to achieve the effective concentration in COVID-19 patients [@doi:10.1093/jac/dkaa114].
Additionally, clinical trials have reported HCQ to be effective in treating HIV [@doi:10/cq2hx9] and chronic Hepatitis C [@doi:10.1002/jmv.24575].
Together, these studies triggered initial enthusiasm about the therapeutic potential for HCQ and CQ against COVID-19.
HCQ/CQ has been proposed both as a treatment for COVID-19 and a prophylaxis against SARS-CoV-2 exposure, and trials often investigated these drugs in combination with azithromycin (AZ) and/or zinc supplementation.
However, as more evidence has emerged, it has become clear that HCQ/CQ offer no benefits against SARS-CoV-2 or COVID-19.

#### Trials Assessing Therapeutic Administration of HCQ/CQ

The initial study evaluating HCQ as a treatment for COVID-19 patients was published on March 20, 2020 by Gautret et al. [@doi:10.1016/j.ijantimicag.2020.105949].
This non-randomized, non-blinded, non-placebo clinical trial compared HCQ to SOC in 42 hospitalized patients in southern France.
It reported that patients who received HCQ showed higher rates of virological clearance by nasopharyngeal swab on days 3-6 when compared to SOC.
This study also treated six patients with both HCQ + AZ and found this combination therapy to be more effective than HCQ alone.
However, the design and analyses used showed weaknesses that severely limit interpretability of results, including the small sample size and the lack of: randomization, blinding, placebo (no "placebo pill" given to SOC group), Intention-To-Treat analysis, correction for sequential multiple comparisons, and trial pre-registration.
Furthermore, the trial arms were entirely confounded by the hospital and there were false negative outcome measurements (see [@doi:10.5014/ajot.2013.006015]).
Two of these weaknesses are due to inappropriate data analysis and can therefore be corrected _post hoc_ by recalculating the p-values (lack of Intention-To-Treat analysis and multiple comparisons).
However, all other weaknesses are fundamental design flaws and cannot be corrected for.
Thus, the conclusions cannot be generalized outside of the study.
The International Society of Antimicrobial Chemotherapy, the scientific organization that publishes the journal where the article appeared, subsequently announced that the article did not meet its expected standard for publications [@url:https://www.isac.world/news-and-publications/official-isac-statement], although it has not been officially retracted.

Because of the preliminary data presented in this study, HCQ treatment was subsequently explored by other researchers.
About one week later, a follow-up case study reported that 11 consecutive patients were treated with HCQ + AZ using the same dosing regimen [@doi:10.1016/j.medmal.2020.03.006].
One patient died, two were transferred to the intensive care unit (ICU), and one developed a prolonged QT interval, leading to discontinuation of HCQ + AZ administration.
As in the Gautret et al. study, the outcome assessed was virological clearance at day 6 post-treatment, as measured from nasopharyngeal swabs.
Of the ten living patients on day 6, eight remained positive for SARS-CoV-2 RNA.
Like in the original study, interpretability was severely limited by the lack of a comparison group and the small sample size.
However, these results stand in contrast to the claims by Gautret et al. that all six patients treated with HCQ + AZ tested negative for SARS-CoV-2 RNA by day 6 post-treatment.
This case study illustrated the need for further investigation using robust study design to evaluate the efficacy of HCQ and/or CQ.

On April 10, 2020, a randomized, non-placebo trial of 62 COVID-19 patients at the Renmin Hospital of Wuhan University was released [@doi:10.1101/2020.03.22.20040758].
This study investigated whether HCQ decreased time to fever break or time to cough relief when compared to SOC [@doi:10.1101/2020.03.22.20040758].
This trial found HCQ decreased both average time to fever break and average time to cough relief, defined as mild or no cough.
While this study improved on some of the methodological flaws in Gautret et al. by randomizing patients, it also had several flaws in trial design and data analysis that prevent generalization of the results.
These weaknesses include the lack of placebo, lack of correction for multiple primary outcomes, inappropriate choice of outcomes, lack of sufficient detail to understand analysis, drastic disparities between pre-registration [@cctr-hcq] and published protocol (including differences in the inclusion and exclusion criteria, the number of experimental groups, the number of patients enrolled, and the outcome analyzed), and small sample size.
The choice of outcomes may be inappropriate as both fevers and cough may break periodically without resolution of illness.
Additionally, for these outcomes, the authors reported that 23 of 62 patients did not have a fever and 25 of 62 patients did not have a cough at the start of the study, but the authors failed to describe how these patients were included in a study assessing time to fever break and time to cough relief.
It is important to note here that the authors claimed "neither the research performers nor the patients were aware of the treatment assignments."
This blinding seems impossible in a non-placebo trial because at the very least, providers would know whether they were administering a medication or not, and this knowledge could lead to systematic differences in the administration of care.
Correction for multiple primary outcomes can be adjusted _post hoc_ by recalculating p-values, but all of the other issues were design and statistical weaknesses that cannot be corrected for.
Additionally, disparities between the pre-registered and published protocols raise concerns about experimental design.
The design limitations mean that the conclusions cannot be generalized outside of the study.

A second randomized trial, conducted by the Shanghai Public Health Clinical Center, analyzed whether HCQ increased rates of virological clearance at day 7 in respiratory pharyngeal swabs compared to SOC [@doi:10/drbx].
This trial was published in Chinese along with an abstract in English, and only the English abstract was read and interpreted for this review.
The trial found comparable outcomes in virological clearance rate, time to virological clearance, and time to body temperature normalization between the treatment and control groups.
The small sample size is one weakness, with only 30 patients enrolled and 15 in each arm.
This problem suggests the study is underpowered to detect potentially useful differences and precludes interpretation of results.
Additionally, because only the abstract could be read, other design and analysis issues could be present.
Thus, though these studies added randomization to their assessment of HCQ, their conclusions should be interpreted very cautiously.
These two studies assessed different outcomes and reached differing conclusions about the efficacy of HCQ for treating COVID-19; the designs of both studies, especially with respect to sample size, meant that no general conclusions can be made about the efficacy of the drug.

Several widely reported studies on HCQ also have issues with data integrity and/or provenance.
A Letter to the Editor published in _BioScience Trends_ on March 16, 2020 claimed that numerous clinical trials have shown that HCQ is superior to control treatment in inhibiting the exacerbation of COVID-19 pneumonia [@doi:10.5582/bst.2020.01047].
This letter has been cited by numerous primary literature, review articles, and media alike [@doi:10.7150/ijbs.45498; @doi:10.7150/ijbs.45053].
However, the letter referred to 15 pre-registration identifiers from the Chinese Clinical Trial Registry.
When these identifiers are followed back to the registry, most trials claim they are not yet recruiting patients or are currently recruiting patients.
For all of these 15 identifiers, no data uploads or links to publications could be located on the pre-registrations.
At the very least, the lack of availability of the primary data means the claim that HCQ is efficacious against COVID-19 pneumonia cannot be verified.
Similarly, a recent multinational registry analysis [@doi:10/ggwzsb] analyzed the efficacy of CQ and HCQ with and without a macrolide, which is a class of antibiotics that includes Azithromycin, for the treatment of COVID-19.
The study observed 96,032 patients split into a control and four treatment conditions (CQ with and without a macrolide; HCQ with and without a macrolide).
They concluded that treatment with CQ or HCQ was associated with increased risk of _de novo_ ventricular arrhythmia during hospitalization.
However, this study has since been retracted by _The Lancet_ due to an inability to validate the data used [@doi:10/ggzqng].
These studies demonstrate that increased skepticism in evaluation of the HCQ/CQ and COVID-19 literature may be warranted, possibly because of the significant attention HCQ and CQ have received as possible treatments for COVID-19 and the politicization of these drugs.

Despite the fact that the study suggesting that CQ/HCQ increased risk of ventricular arrhythmia in COVID-19 patients has now been retracted, previous studies have identified risks associated with HCQ/CQ.
A patient with systemic lupus erythematosus developed a prolonged QT interval that was likely exacerbated by use of HCQ in combination with renal failure [@doi:10.1155/2016/4626279].
A prolonged QT interval is associated with ventricular arrhythmia [@doi:10.1016/j.hrthm.2008.05.008].
Furthermore, a separate study [@doi:10.1101/2020.04.08.20054551] investigated the safety associated with the use of HCQ with and without macrolides between 2000 and 2020.
The study involved 900,000 cases treated with HCQ and 300,000 cases treated with HCQ + AZ.
The results indicated that short-term use of HCQ was not associated with additional risk, but that HCQ + AZ was associated with an enhanced risk of cardiovascular complications (such as a 15% increased risk of chest pain, calibrated HR = 1.15, 95% CI, 1.05 to 1.26) and a two-fold increased 30-day risk of cardiovascular mortality (calibrated HR = 2.19; 95% CI, 1.22 to 3.94).
Therefore, whether studies utilize HCQ alone or HCQ in combination with a macrolide may be an important consideration in assessing risk.
As results from initial investigations of these drug combinations have emerged, concerns about the efficacy and risks of treating COVID-19 with HCQ and CQ have led to the removal of CQ/HCQ from SOC practices in several countries [@doi:10.1101/2020.04.07.20056424; @url:https://www.cnn.com/2020/04/13/health/chloroquine-risks-coronavirus-treatment-trials-study/index.html].
As of May 25, 2020, WHO had suspended administration of HCQ as part of the worldwide Solidarity Trial [@raw:WHO_briefing_2020_5_25], and later the final results of this large-scale trial that compared 947 patients administered HCQ to 906 controls revealed no effect on the primary outcome, mortality during hospitalization (rate ratio: 1.19; _p_ = 0.23)

Additional research has emerged largely identifying HCQ/CQ to be ineffective against COVID-19 while simultaneously revealing a number of significant side effects.
A randomized, open-label, non-placebo trial of 150 COVID-19 patients was conducted in parallel at 16 government-designated COVID-19 centers in China to assess the safety and efficacy of HCQ [@doi:10.1101/2020.04.10.20060558].
The trial compared treatment with HCQ in conjunction with SOC to SOC alone in 150 infected patients who were assigned randomly to the two groups (75 per group).
The primary endpoint of the study was the negative conversion rate of SARS-CoV-2 in 28 days, and the investigators found no difference in this parameter between the groups (estimated difference between SOC plus HCQ and SOC 4.1%; 95% CI, –10.3% to 18.5%).
The secondary endpoints were an amelioration of the symptoms of the disease such as axillary temperature ≤36.6°C, SpO2 >94% on room air, and disappearance of symptoms like shortness of breath, cough, and sore throat.
The median time to symptom alleviation was similar across different conditions (19 days in HCQ + SOC versus 21 days in SOC, _p_ = 0.97).
Additionally, 30% of the patients receiving SOC+HCQ reported adverse outcomes compared to 8.8% of patients receiving only SOC, with the most common adverse outcome in the SOC+HCQ group being diarrhea (10% versus 0% in the SOC group, _p_ = 0.004).
However, there are several factors that limit the interpretability of this study.
Most of the enrolled patients had mild-to-moderate symptoms (98%), and the average age was 46.
SOC in this study included the use of antivirals (Lopinavir-Ritonavir, Arbidol, Oseltamivir, Virazole, Entecavir, Ganciclovir, and Interferon alfa), which the authors note could influence the results.
Thus, they note that an ideal SOC would need to exclude the use of antivirals, but that ceasing antiviral treatment raised ethical concerns at the time that the study was conducted.
In this trial, the samples used to test for the presence of the SARS-CoV-2 virus were collected from the upper respiratory tract, and the authors indicated that the use of upper respiratory samples may have introduced false negatives (e.g., [@doi:10.1001/jama.2020.3786]).
Another limitation of the study that the authors acknowledge was that the HCQ treatment began, on average, at a 16-day delay from the symptom onset.
The fact that this study was open-label and lacked a placebo limits interpretation, and additional analysis is required to determine whether HCQ reduces inflammatory response.
Therefore, despite some potential areas of investigation identified in _post hoc_ analysis, this study cannot be interpreted as providing support for HCQ as a therapeutic against COVID-19.
This study provided no support for HCQ against COVID-19, as there was no difference between the two groups in either negative seroconversion at 28 days or symptom alleviation, and in fact, more severe adverse outcomes were reported in the group receiving HCQ.

Additional evidence comes from a retrospective analysis [@doi:10.1101/2020.04.16.20065920] that examined data from 368 COVID-19 patients across all United States Veteran Health Administration medical centers.
The study retrospectively investigated the effect of the administration of HCQ (n=97), HCQ + AZ (n=113), and no HCQ (n=158) on 368 patients.
The primary outcomes assessed were death and the need for mechanical ventilation.
Standard supportive care was rendered to all patients.
Due to the low representation of women (N=17) in the available data, the analysis included only men, and the median age was 65 years.
The rate of death was 27.8% in the HCQ-only treatment group, 22.1% in the HCQ + AZ treatment group, and 14.1% in the no-HCQ group.
These data indicated a statistically significant elevation in the risk of death for the HCQ-only group compared to the no-HCQ group (adjusted HR: 2.61, _p_ = 0.03), but not for the HCQ + AZ group compared to the no-HCQ group (adjusted HR: 1.14; _p_ = 0.72).
Further, the risk of ventilation was similar across all three groups (adjusted HR: 1.43, _p_ = 0.48 (HCQ) and 0.43, _p_ = 0.09 (HCQ + AZ) compared to no HCQ).
The study thus showed evidence of an association between increased mortality and HCQ in this cohort of COVID-19 patients but no change in rates of mechanical ventilation among the treatment conditions.
The study had a few limitations: it was not randomized, and the baseline vital signs, laboratory tests, and prescription drug use were significantly different among the three groups.
All of these factors could potentially influence treatment outcome.
Furthermore, the authors acknowledge that the effect of the drugs might be different in females and pediatric subjects, since these subjects were not part of the study.
The reported result that HCQ + AZ is safer than HCQ contradicts the findings of the previous large-scale analysis of twenty years of records that found HCQ + AZ to be more frequently associated with cardiac arrhythmia than HCQ alone [@doi:10.1101/2020.04.08.20054551]; whether this discrepancy is caused by the pathology of COVID-19, is influenced by age or sex, or is a statistical artifact is not presently known.

Finally, findings from the RECOVERY trial were released on October 8, 2020.
This study used a randomized, open-label design to study the effects of HCQ compared to SOC in 11,197 patients at 176 hospitals in the United Kingdom [@doi:10.1056/NEJMoa2022926].
Patients were randomized into either the control group or one of the treatment arms, with twice as many patients enrolled in the control group as any treatment group.
Of the patients eligible to receive HCQ, 1,561 were randomized into the HCQ arm, and 3,155 were randomized into the control arm.
The demographics of the HCQ and control groups were similar in terms of average age (65 years), proportion female (approximately 38%), ethnic make-up (73% versus 76% white), and prevalence of pre-existing conditions (56% versus 57% overall).
In the HCQ arm of the study, patients received 800 mg at baseline and again after 6 hours, then 400 mg at 12 hours and every subsequent 12 hours.
The primary outcome analyzed was all-cause mortality, and patient vital statistics were reported by physicians upon discharge or death, or else at 28 days following HCQ administration if they remained hospitalized.
The secondary outcome assessed was the combined risk of progression to invasive mechanical ventilation or death within 28 days.
By the advice of an external data monitoring committee, the HCQ arm of the study was reviewed early, leading to it being closed due a lack of support for HCQ as a treatment for COVID-19.
COVID-19-related mortality was not affected by HCQ in the RECOVERY trial (rate ratio, 1.09; 95% CI, 0.97 to 1.23; _p_ = 0.15), but cardiac events were increased in the HCQ arm (0.4 percentage points), as was the duration of hospitalization (rate ratio for discharge alive within 28 days: 0.90; 95% CI, 0.83 to 0.98) and likelihood of progression to mechanical ventilation or death (risk ratio 1.14; 95% CI, 1.03 to 1.27).
This large-scale study thus builds upon studies in the United States and China to suggest that HCQ is not an effective treatment, and in fact may negatively impact COVID-19 patients due to its side effects.
Therefore, though none of the studies have been blinded, examining them together makes it clear that the available evidence points to significant dangers associated with the administration of HCQ to hospitalized COVID-19 patients, without providing any support for its efficacy.

#### HCQ for the Treatment of Mild Cases

One additional possible therapeutic application of HCQ considered was the treatment of mild COVID-19 cases in otherwise healthy individuals.
This possibility was assessed in a randomized, open-label, multi-center analysis conducted in Catalonia (Spain) [@doi:10.1093/cid/ciaa1009].
This analysis enrolled adults 18 and older who had been experiencing mild symptoms of COVID-19 for fewer than five days.
Participants were randomized into an HCQ arm (N=136) and a control arm (N=157), and those in the treatment arm were administered 800 mg of HCQ on the first day of treatment followed by 400 mg on each of the subsequent six days.
The primary outcome assessed was viral clearance at days 3 and 7 following the onset of treatment, and secondary outcomes were clinical progression and time to complete resolution of symptoms.
No significant differences between the two groups were found: the difference in viral load between the HCQ and control groups was 0.01 (95% CI, -0.28 to 0.29) at day 3 and -0.07 (95% CI -0.44 to 0.29) at day 7, the relative risk of hospitalization was 0.75 (95% CI, 0.32 to 1.77), and the difference in time to complete resolution of symptoms was -2 days (_p_ = 0.38).
This study thus suggests that HCQ does not improve recovery from COVID-19, even in otherwise healthy adult patients with mild symptoms.

#### Prophylactic Administration of HCQ

An initial study of the possible prophylactic application of HCQ utilized a randomized, double-blind, placebo-controlled design to analyze the administration of HCQ prophylactically [@doi:10.1056/NEJMoa2016638].
Asymptomatic adults in the United States and Canada who had been exposed to SARS-CoV-2 within the past four days were enrolled in an online study to evaluate whether administration of HCQ over five days influenced the probability of developing COVID-19 symptoms over a 14-day period.
Of the participants, 414 received HCQ and 407 received a placebo.
No significant difference in the rate of symptomatic illness was observed between the two groups (11.8% HCQ, 14.3% placebo, _p_ = 0.35).
The HCQ condition was associated with side effects, with 40.1% of patients reporting side effects compared to 16.8% in the control group (_p_ < 0.001).
However, likely due to the high enrollment of healthcare workers (66% of participants) and the well-known side effects associated with HCQ, a large number of participants were able to correctly identify whether they were receiving HCQ or a placebo (46.5% and 35.7%, respectively).
Furthermore, due to a lack of availability of diagnostic testing, only 20 of the 107 cases were confirmed with a PCR-based test to be positive for SARS-CoV-2.
The rest were categorized as "probable" or "possible" cases by a panel of four physicians who were blind to the treatment status.
One possible confounder is that a patient presenting one or more symptoms, which included diarrhea, was defined as a "possible" case, but diarrhea is also a common side effect of HCQ.
Additionally, four of the twenty PCR-confirmed cases did not develop symptoms until after the observation period had completed, suggesting that the 14-day trial period may not have been long enough or that some participants also encountered secondary exposure events.
Finally, in addition to the young age of the participants in this study, which ranged from 32 to 51, there were possible impediments to generalization introduced by the selection process, as 2,237 patients who were eligible but had already developed symptoms by day 4 were enrolled in a separate study.
It is therefore likely that asymptomatic cases were over-represented in this sample, which would not have been detected based on the diagnostic criteria used.
Therefore, while this study does represent the first effort to conduct a randomized, double-blind, placebo-controlled investigation of HCQ's effect on COVID-19 prevention after SARS-CoV-2 exposure in a large sample, the lack of PCR tests and several other design flaws significantly impede interpretation of the results.
However, in line with the results from therapeutic studies, once again no evidence was found suggesting an effect of HCQ against COVID-19.

A second study [@doi:10.1001/jamainternmed.2020.6319] examined the effect of administering HCQ to healthcare workers as a pre-exposure prophylactic.
The primary outcome assessed was the conversion from SARS-CoV-2 negative to SARS-CoV-2 positive status over the 8 week study period.
This study was also randomized, double-blind, and placebo-controlled, and it sought to address some of the limitations of the first prophylactic study.
The goal was to enroll 200 healthcare workers, preferentially those working with COVID-19 patients, at two hospitals within the University of Pennsylvania hospital system in Philadelphia, PA.
Participants were randomized 1:1 to receive either 600 mg of HCQ daily or a placebo, and their SARS-CoV-2 infection status and antibody status were assessed using RT-PCR and serological testing, respectively, at baseline, 4 weeks, and 8 weeks following the beginning of the treatment period.
The statistical design of the study accounted for interim analyses at 50 and 100 participants in case efficacy or futility of HCQ for prophylaxis became clear earlier than completion of enrollment.
The 139 individuals enrolled comprised a study population that was fairly young (average age 33) and made of largely of people who were white, women, and without pre-existing conditions.
At the second interim analysis, more individuals in the treatment group than the control group had contracted COVID-19 (4 versus 3), causing the estimated z-score to fall below the pre-established threshold for futility.
As a result, the trial was terminated early, offering additional evidence against the use of HCQ for prophylaxis.

#### Summary of HCQ/CQ Research Findings

Early _in vitro_ evidence indicated that HCQ could be an effective therapeutic against SARS-CoV-2 and COVID-19, leading to significant media attention and public interest in its potential as both a therapeutic and prophylactic.
Initially it was hypothesized that CQ/HCQ might be effective against SARS-CoV-2 in part because CQ and HCQ have both been found to inhibit the expression of CD154 in T-cells and to reduce TLR signaling that leads to the production of pro-inflammatory cytokines [@doi:10.1038/s41584-020-0372-x].
Clinical trials for COVID-19 have more often used HCQ rather than CQ because it offers the advantages of being cheaper and having fewer side effects than CQ.
However, research has not found support for a positive effect of HCQ on COVID-19 patients.
Multiple clinical studies have already been carried out to assess HCQ as a therapeutic agent for COVID-19, and many more are in progress.
To date, none of these studies have used randomized, double-blind, placebo-controlled designs with a large sample size, which would be the gold standard.
Despite the design limitations (which would be more likely to produce false positives than false negatives), initial optimism about HCQ has largely dissipated.
The most methodologically rigorous analysis of HCQ as a prophylactic [@doi:10.1056/NEJMoa2016638] found no significant differences between the treatment and control groups, and the WHO's global Solidarity trial similarly reported no effect of HCQ on mortality [@doi:10.1056/NEJMoa2023184].
Thus, HCQ/CQ are not likely to be effective therapeutic or prophylactic agents against COVID-19.
One case study identified drug-induced phospholipidosis as the cause of death for a COVID-19 patient treated with HCQ [@doi:10.1080/01913123.2020.1850966], suggesting that in some cases, the proposed mechanism of action may ultimately be harmful.
Additionally, one study identified an increased risk of mortality in older men receiving HCQ, and administration of HCQ and HCQ + AZ did not decrease the use of mechanical ventilation in these patients [@doi:10.1101/2020.04.16.20065920].
HCQ use for COVID-19 could also lead to shortages for anti-malarial or anti-rheumatic use, where it has documented efficacy.
Despite significant early attention, these drugs appear to be ineffective against COVID-19.
Several countries have now removed CQ/HCQ from their SOC for COVID-19 due to the lack of evidence of efficacy and the frequency of adverse effects.

### ACE Inhibitors and Angiotensin II Receptor Blockers

Several clinical trials testing the effects of ACEIs or ARBs on COVID-19 outcomes are ongoing [@clinicaltrials:NCT04338009; @clinicaltrials:NCT04353596; @clinicaltrials:NCT04311177; @clinicaltrials:NCT04312009; @clinicaltrials:NCT04330300; @clinicaltrials:NCT04366050; @clinicaltrials:NCT04364893].
Clinical trials are needed because the findings of the various observational studies bearing on this topic cannot be interpreted as indicating a protective effect of the drug [@doi:10.1161/CIRCRESAHA.120.317205; @doi:10.2215/CJN.03530320].
Two analyses [@clinicaltrials:NCT04338009; @clinicaltrials:NCT04364893] have reported no effect of continuing or discontinuing ARBs and ACEIs on patients admitted to the hospital for COVID-19.
The first, known as REPLACE COVID [@doi:10/fvgt], was a randomized, open-label study that enrolled patients who were admitted to the hospital for COVID-19 and were taking an ACEI at the time of admission.
They enrolled 152 patients at 20 hospitals across seven countries and randomized them into two arms, continuation (n=75) and discontinuation (n=77).
The primary outcome evaluated was a global rank score that integrated several dimensions of illness.
The components of this global rank score, such as time to death and length of mechanical ventilation, were evaluated as secondary endpoints.
This analysis reported no differences between the two groups in the primary or any of the secondary outcomes.

Similarly, a second study [@doi:10.1001/jama.2020.25864] used a randomized, open-label design to examine the effects of continuing versus discontinuing ARBs and ACEIs on patients hospitalized for mild to moderate COVID-19 at 29 hospitals in Brazil.
This study enrolled 740 patients but had to exclude one trial site from all analyses due to the discovery of violations of Good Clinical Trial practice and data falsification.
After this exclusion, 659 patients remained, with 334 randomized to discontinuation and 325 to continuation.
In this study, the primary endpoint analyzed was the number of days that patients were alive and not hospitalized within 30 days of enrollment.
The secondary outcomes included death (including in-hospital death separately), number of days hospitalized, and specific clinical outcomes such as heart failure or stroke.
Once again, no significant differences were found between the two groups.
Initial studies of randomized interventions therefore suggest that ACEIs and ARBs are unlikely to affect COVID-19 outcomes.
These results are also consistent with findings from observational studies (summarized in [@doi:10/fvgt]).
Additional information about ACE2, observational studies of ACEIs and ARBs in COVID-19, and clinical trials on this topic have been summarized [@url:http://www.nephjc.com/news/covidace2].
Therefore, despite the promising potential mechanism, initial results have not provided support for ACEIs and ARBs as therapies for COVID-19.

### Tocilizumab

Human IL-6 is a 26-kDa glycoprotein that consists of 184 amino acids and contains two potential N-glycosylation sites and four cysteine residues.
It binds to a type I cytokine receptor (IL-6R&alpha; or glycoprotein 80) that exists in both membrane-bound (IL-6R&alpha;) and soluble (sIL-6R&alpha;) forms [@doi:10.3389/fimmu.2016.00604].
It is not the binding of IL-6 to the receptor that initiates pro- and/or anti-inflammatory signaling, but rather the binding of the complex to another subunit, known as IL-6R&beta; or glycoprotein 130 (gp130) [@doi:10.7150/ijbs.4989; @doi:10.3389/fimmu.2016.00604].
Unlike membrane-bound IL-6R&alpha;, which is only found on hepatocytes and some types of leukocytes, gp130 is found on most cells [@doi:10.1007/s00430-006-0019-9].
When IL-6 binds to sIL-6R&alpha;, the complex can then bind to a gp130 protein on any cell [@doi:10.1007/s00430-006-0019-9].
The binding of IL-6 to IL-6R&alpha; is termed classical signaling, while its binding to sIL-6R&alpha; is termed trans-signaling [@doi:10.1016/j.cytogfr.2012.04.001; @doi:10.1042/bj3000281; @doi:10.1007/s00430-006-0019-9].
These two signaling processes are thought to play different roles in health and illness.
For example, trans-signaling may play a role in the proliferation of mucosal T-helper TH2 cells associated with asthma, while an earlier step in this proliferation process may be regulated by classical signaling [@doi:10.1007/s00430-006-0019-9].
Similarly, IL-6 is known to play a role in Crohn's Disease via trans-, but not classical, signaling [@doi:10.1007/s00430-006-0019-9].
Both classical and trans-signaling can occur through three independent pathways: the Janus-activated kinase-STAT3 pathway, the Ras/Mitogen-Activated Protein Kinases pathway and the Phosphoinositol-3 Kinase/Akt pathway [@doi:10.3389/fimmu.2016.00604].
These signaling pathways are involved in a variety of different functions, including cell type differentiation, immunoglobulin synthesis, and cellular survival signaling pathways, respectively [@doi:10.3389/fimmu.2016.00604].
The ultimate result of the IL-6 cascade is to direct transcriptional activity of various promoters of pro-inflammatory cytokines, such as IL-1, TFN, and even IL-6 itself, through the activity of NF-κB [@doi:10.3389/fimmu.2016.00604].
IL-6 synthesis is tightly regulated both transcriptionally and post-transcriptionally, and it has been shown that viral proteins can enhance transcription of the IL-6 gene by strengthening the DNA-binding activity between several transcription factors and IL-6 gene-cis-regulatory elements [@doi:10.2492/inflammregen.33.054].
Therefore, drugs inhibiting the binding of IL-6 to IL-6R&alpha; or sIL-6R&alpha; are of interest for combating the hyperactive inflammatory response characteristic of cytokine release syndrome (CRS) and cytokine storm syndrome (CSS).
TCZ is a humanized monoclonal antibody that binds both to the insoluble and soluble receptor of IL-6, providing de facto inhibition of the IL-6 immune cascade.
Interest in TCZ as a possible treatment for COVID-19 was piqued by early evidence indicating that COVID-19 deaths may be induced by the hyperactive immune response, often referred to as CRS or CSS [@doi:10/ggnxb3], as IL-6 plays a key role in this response [@doi:10.1128/mmbr.05015-11].
The observation of elevated IL-6 in patients who died relative to those who recovered [@doi:10/ggnxb3] could reflect an over-production of proinflammatory interleukins, suggesting that TCZ could potentially palliate some of the most severe symptoms of COVID-19 associated with increased cytokine production.

This early interest in TCZ as a possible treatment for COVID-19 was bolstered by a very small retrospective study in China that examined 20 patients with severe symptoms in early February 2020 and reported rapid improvement in symptoms following treatment with TCZ [@doi:10.1073/pnas.2005615117].
Subsequently, a number of retrospective studies have been conducted in several countries.
Many studies use a retrospective, observational design, where they compare outcomes for COVID-19 patients who received TCZ to those who did not over a set period of time.
For example, one of the largest retrospective, observational analyses released to date [@doi:10/d2pk], consisting of 1,351 patients admitted to several care centers in Italy, compared the rates at which patients who received TCZ died or progressed to invasive medical ventilation over a 14-day period compared to patients receiving only SOC.
Under this definition, SOC could include other drugs such as HCQ, azithromycin, lopinavir-ritonavir or darunavir-cobicistat, or heparin.
While this study was not randomized, a subset of patients who were eligible to receive TCZ were unable to obtain it due to shortages; however, these groups were not directly compared in the analysis.
After adjusting for variables such as age, sex, and SOFA (sequential organ failure assessment) score, they found that patients treated with TCZ were less likely to progress to invasive medical ventilation and/or death (adjusted HR = 0.61, CI 0.40-0.92, _p_ = 0.020); analysis of death and ventilation separately suggests that this effect may have been driven by differences in the death rate (20% of control versus 7% of TCZ-treated patients).
The study reported particular benefits for patients whose PaO~2~/FiO~2~ ratio, also known as the Horowitz Index for Lung Function, fell below a 150 mm Hg threshold.
They found no differences between groups administered subcutaneous versus intravenous TCZ.

Another retrospective observational analysis of interest examined the charts of patients at a hospital in Connecticut, USA where 64% of all 239 COVID-19 patients in the study period were administered TCZ based on assignment by a standardized algorithm [@doi:10.1016/j.chest.2020.06.006].
They found that TCZ administration was associated with more similar rates of survivorship in patients with severe versus nonsevere COVID-19 at intake, defined based on the amount of supplemental oxygen needed.
They therefore proposed that their algorithm was able to identify patients presenting with or likely to develop CRS as good candidates for TCZ.
This study also reported higher survivorship in Black and Hispanic patients compared to white patients when adjusted for age.
The major limitation with interpretation for these studies is that there may be clinical characteristics that influenced medical practitioners decisions to administer TCZ to some patients and not others.
One interesting example therefore comes from an analysis of patients at a single hospital in Brescia, Italy, where TCZ was not available for a period of time [@doi:10.1016/j.ejim.2020.05.009].
This study compared COVID-19 patients admitted to the hospital before and after March 13, 2020, when the hospital received TCZ.
Therefore, patients who would have been eligible for TCZ prior to this arbitrary date did not receive it as treatment, making this retrospective analysis something of a natural experiment.
Despite this design, demographic factors did not appear to be consistent between the two groups, and the average age of the control group was older than the TCZ group.
The control group also had a higher percentage of males and a higher incidence of comorbidities such as diabetes and heart disease.
All the same, the multivariate HR, which adjusted for these clinical and demographic factors, found a significant difference between survival in the two groups (HR=0.035, CI=0.004-0.347, _p_ = 0.004).
The study reported improvement of survival outcomes after the addition of TCZ to the SOC regime, with 11 of 23 patients (47.8%) admitted prior to March 13th dying compared to 2 of 62 (3.2%) admitted afterwards (HR=0.035; 95% CI, 0.004 to 0.347; _p_ = 0.004).
They also reported a reduced progression to mechanical ventilation in the TCZ group.
However, this study also holds a significant limitation: the time delay between the two groups means that knowledge about how to treat the disease likely improved over this timeframe as well.
All the same, the results of these observational retrospective studies provide support for TCZ as a pharmaceutical of interest for follow-up in clinical trials.

Other retrospective analyses have utilized a case-control design to match pairs of patients with similar baseline characteristics, only one of whom received TCZ for COVID-19.
In one such study, TCZ was significantly associated with a reduced risk of progression to intensive care unit (ICU) admission or death [@doi:10.1016/j.medmal.2020.05.001].
This study examined only 20 patients treated with TCZ (all but one of the patients treated with TCZ in the hospital during the study period) and compared them to 25 patients receiving SOC.
For the combined primary endpoint of death and/or ICU admission, only 25% of patients receiving TCZ progressed to an endpoint compared to 72% in the SOC group (_p_ = 0.002, presumably based on a chi-square test based on the information provided in the text).
When the two endpoints were examined separately, progression to invasive medical ventilation remained significant (32% SOC compared to 0% TCZ, _p_ = 0.006) but not for mortality (48% SOC compared to 25% TCZ, _p_ = 0.066).
In contrast, a study that compared 96 patients treated with TCZ to 97 patients treated with SOC only in New York City found that differences in mortality did not differ between the two groups, but that this difference did become significant when intubated patients were excluded from the analysis [@doi:10.1093/qjmed/hcaa206].
Taken together, these findings suggest that future clinical trials of TCZ may want to include intubation as an endpoint.
However, these studies should be approached with caution, not only because of the small number of patients enrolled and the retrospective design, but also because they performed a large number of statistical tests and did not account for multiple hypothesis testing.
In general, caution must be exercised when interpreting subgroup analyses after a primary combined endpoint analysis.
These last findings highlight the need to search for a balance between impairing a harmful immune response, such as the one generated during CRS/CSS, and preventing the worsening of the clinical picture of the patients by potential new viral infections.
Early meta-analyses and systematic reviews have investigated the available data about TCZ for COVID-19.
One meta-analysis [@doi:10.2139/ssrn.3642653] evaluated 19 studies published or released as preprints prior to July 1, 2020 and found that the overall trends were supportive of the frequent conclusion that TCZ does improve survivorship, with a significant HR of 0.41 (_p_ < 0.001).
This trend improved when they excluded studies that administered a steroid alongside TCZ, with a significant HR of 0.04 (_p_ < 0.001).
They also found some evidence for reduced invasive ventilation or ICU admission, but only when excluding all studies except a small number whose estimates were adjusted for the possible bias introduced by the challenges of stringency during the enrollment process.
A systematic analysis of sixteen case-control studies of TCZ estimated an odds ratio of mortality of 0.453 (95% CI 0.376–0.547, _p_ < 0.001), suggesting possible benefits associated with TCZ treatment [@doi:10.7717/peerj.10322].
Although these estimates are similar, it is important to note that they are drawing from the same literature and are therefore likely to be affected by the same potential biases in publication.
A different systematic review of studies investigating TCZ treatment for COVID-19 analyzed 31 studies that had been published or released as pre-prints and reported that none carried a low risk of bias [@doi:10.1016/j.pulmoe.2020.07.003].
Therefore, the present evidence is not likely to be sufficient for conclusions about the efficacy of TCZ.

On February 11, 2021, a preprint describing the first randomized control trial of TCZ was released as part of the RECOVERY trial [@doi:10.1101/2021.02.11.21249258].
Of the 21,550 patients enrolled in the RECOVERY trial at the time, 4,116 adults hospitalized with COVID-19 across the 131 sites in the United Kingdom were assigned to the arm of the trial evaluating the effect of TCZ.
Among them, 2,022 were randomized to receive TCZ and 2,094 were randomized to SOC, with 79% of patients in each group available for analysis at the time that the initial report was released.
The primary outcome measured was 28-day mortality, and TCZ was found to reduce 28-day mortality from 33% of patients receiving SOC alone to 29% of those receiving TCZ, corresponding to a rate ratio of 0.86 (95% CI 0.77-0.96; _p_ = 0.007).
TCZ was also significantly associated with the probability of hospital discharge within 28 days for living patients, which was 47% in the SOC group and 54% in the TCZ group (rate ratio 1.22, 95% CI 1.12-1.34, _p_ < 0.0001).
A potential statistical interaction between TCZ and corticosteroids was observed, with the combination providing greater mortality benefits than TCZ alone, but the authors note that caution is advisable in light of the number of statistical tests conducted.
Combining the RECOVERY trial data with data from seven smaller randomized control trials indicates that TCZ is associated with a 13% reduction in 28-day mortality (rate ratio 0.87, 95% CI 0.79-0.96, _p_ = 0.005) [@doi:10.1101/2021.02.11.21249258].

There are possible risks associated with the administration of TCZ for COVID-19.
TCZ has been used for over a decade to treat RA [@doi:10.1177/1759720X18798462], and a recent study found the drug to be safe for pregnant and breastfeeding women [@doi:10.1093/rheumatology/kez100].
However, TCZ may increase the risk of developing infections [@doi:10.1177/1759720X18798462], and RA patients with chronic hepatitis B infections had a high risk of hepatitis B virus reactivation when TCZ was administered in combination with other RA drugs [@doi:10.1111/1756-185X.13010].
As a result, TCZ is contraindicated in patients with active infections such as tuberculosis [@doi:10.1186/s12967-020-02339-3].
Previous studies have investigated, with varying results, a possible increased risk of infection in RA patients administered TCZ [@doi:10.1093/rheumatology/keq343; @doi:10.1136/annrheumdis-2018-214367], although another study reported that the incidence rate of infections was higher in clinical practice RA patients treated with TCZ than in the rates reported by clinical trials [@doi:10.1093/rheumatology/ker223].
In the investigation of 544 Italian COVID-19 patients, the group treated with TCZ was found to be more likely to develop secondary infections, with 24% compared to 4% in the control group (_p_ < 0.0001) [@doi:10/d2pk].
Reactivation of hepatitis B and herpes simplex virus 1 was also reported in a small number of patients in this study, all of whom were receiving TCZ.
A July 2020 case report described negative outcomes of two COVID-19 patients after receiving TCZ, including one death; however, both patients were intubated and had entered septic shock prior to receiving TCZ [@doi:10.1016/j.chest.2020.04.024], likely indicating a severe level of cytokine production.
Additionally, D-dimer and sIL2R levels were reported by one study to increase in patients treated with TCZ, which raised concerns because of the potential association between elevated D-dimer levels and thrombosis and between sIL2R and diseases where T-cell regulation is compromised [@doi:10.1016/j.chest.2020.06.006].
An increased risk of bacterial infection was also identified in a systematic review of the literature, based on the unadjusted estimates reported [@doi:10.2139/ssrn.3642653].
In the RECOVERY trial, however, only three out of 2,022 participants in the group receiving TCZ developed adverse reactions determined to be associated with the intervention, and no excess deaths were reported [@doi:10.1101/2021.02.11.21249258].
TCZ administration to COVID-19 patients is not without risks and may introduce additional risk of developing secondary infections; however, while caution may be prudent when treating patients who have latent viral infections, the results of the RECOVERY trial indicate that adverse reactions to TCZ are very rare among COVID-19 patients broadly.

In summary, approximately 33% of hospitalized COVID-19 patients develop ARDS [@doi:10.1186/s13054-020-03240-7], which is caused by an excessive early response of the immune system which can be a component of CRS/CSS [@doi:10.1016/j.chest.2020.06.006; @doi:10.1186/s12967-020-02339-3].
This overwhelming inflammation is triggered by IL-6.
TCZ is an inhibitor of IL-6 and therefore may neutralize the inflammatory pathway that leads to the cytokine storm.
The mechanism suggests TCZ could be beneficial for the treatment of COVID-19 patients experiencing excessive immune activity, and the RECOVERY trial reported a reduction in 28-day mortality.
Interest in TCZ as a treatment for COVID-19 was also supported by two meta-analyses [@doi:10.1101/2020.07.10.20150938; @doi:10.2139/ssrn.3642653], but a third meta-analysis found that all of the available literature at that time carried a risk of bias [@doi:10.1016/j.pulmoe.2020.07.003].
Additionally, different studies used different dosages, number of doses, and methods of administration.
Ongoing research may be needed to optimize administration of TCZ [@doi:10.1016/j.jcv.2020.104443], although similar results were reported by one study for intravenous and subcutaneous administration [@doi:10/d2pk].
Clinical trials that are in progress are likely to provide additional insight into the effectiveness of this drug for the treatment of COVID-19 along with how it should be administered.

### Interferons

IFNs are a family of cytokines critical to activating the innate immune response against viral infections.
Interferons are classified into three categories based on their receptor specificity: types I, II and III [@doi:10.1128/mmbr.05015-11].
Specifically, IFNs I (IFN-𝛼 and 𝛽) and II (IFN-𝛾) induce the expression of antiviral proteins [@doi:10.1016/j.jcv.2003.11.013].
Among these IFNs, IFN-𝛽 has already been found to strongly inhibit the replication of other coronaviruses, such as SARS-CoV-1, in cell culture, while IFN-𝛼 and 𝛾 were shown to be less effective in this context [@doi:10.1016/j.jcv.2003.11.013].
There is evidence that patients with higher susceptibility to ARDS indeed show deficiency in IFN-𝛽.
For instance, infection with other coronaviruses impairs IFN-𝛽 expression and synthesis, allowing the virus to escape the innate immune response [@doi:10.1016/j.virusres.2014.07.024].
On March 18 2020, Synairgen plc received approval to start a phase II trial for SNG001, an IFN-𝛽-1a formulation to be delivered to the lungs via inhalation [@clinicaltrials:NCT04385095].
SNG001, which contains recombinant interferon beta-1a, was previously shown to be effective in reducing viral load in an _in vivo_ model of swine flu and _in vitro_ models of other coronavirus infections [@synairgen-SNG001-March-2020].
In July 2020, a press release from Synairgen stated that SNG001 reduced progression to ventilation in a double-blind, placebo-controlled, multi-center study of 101 patients with an average age in the late 50s [@synairgen-SNG001-July-2020].
These results were subsequently published in November 2020 [@doi:10/ghjzm4].
The study reports that the participants were assigned at a ratio of 1:1 to receive either SNG001 or a placebo that lacked the active compound, by inhalation for up to 14 days.
The primary outcome they assessed was the change in patients' score on the WHO Ordinal Scale for Clinical Improvement (OSCI) at trial day 15 or 16.
SNG001 was associated with an odds ratio of improvement on the OSCI scale of 2.32 (95% CI 1.07 – 5.04, _p_ = 0.033) in the intention-to-treat analysis and 2.80 (95% CI 1.21 – 6.52, _p_ = 0.017) in the per-protocol analysis, corresponding to significant improvement in the SNG001 group on the OSCI at day 15/16.
Some of the secondary endpoints analyzed also showed differences: at day 28, the OR for clinical improvement on the OSCI was 3.15 (95% CI 1.39 – 7.14, _p_ = 0.006), and the odds of recovery at day 15/16 and at day 28 were also significant between the two groups.
Thus, this study suggested that IFN-𝛽1 administered via SNG001 may improve clinical outcomes.

In contrast, the WHO Solidarity trial reported no significant effect of IFN-&beta;-1a on patient survival during hospitalization [@doi:10.1056/NEJMoa2023184].
Here, the primary outcome analyzed was in-hospital mortality, and the rate ratio for the two groups was 1.16 (95% CI, 0.96 to 1.39; _p_ = 0.11) administering IFN-&beta;-1a to 2050 patients and comparing their response to 2,050 controls.
However, there are a few reasons that the different findings of the two trials might not speak to the underlying efficacy of this treatment strategy.
One important consideration is the stage of COVID-19 infection analyzed in each study.
The Synairgen trial enrolled only patients who were not receiving invasive ventilation, corresponding to a less severe stage of disease than many patients enrolled in the SOLIDARITY trial, as well as a lower overall rate of mortality [@doi:10/ftmj].
Additionally, the methods of administration differed between the two trials, with the SOLIDARITY trial administering IFN-&beta;-1a subcutaneously [@doi:10/ftmj].
The differences in findings between the studies suggests that the method of administration might be relevant to outcomes, with nebulized IFN-&beta;-1a more directly targeting receptors in the lungs.
A trial that analyzed the effect of subcutaneously administered IFN-&beta;-1a on patients with ARDS between 2015 and 2017 had also reported no effect on 28-day mortality [@doi:10.1001/jama.2019.22525], while a smaller study analyzing the effect of subcutaneous IFN administration did find a significant improvement in 28-day mortality for COVID-19 [@doi:10.1128/AAC.01061-20].
At present, several ongoing clinical trials are investigating the potential effects of IFN-&beta;-1a, including in combination with therapeutics such as remdesivir [@clinicaltrials:NCT04492475] and administered via inhalation [@clinicaltrials:NCT04385095].
Thus, as additional information becomes available, a more detailed understanding of whether and under which circumstances IFN-&beta;-1a is beneficial to COVID-19 patients should develop.

### Potential Avenues of Interest for Therapeutic Development

Given what is currently known about these therapeutics for COVID-19, a number of related therapies beyond those explored above may also prove to be of interest.
For example, the demonstrated benefit of dexamethasone and the ongoing potential of tocilizumab for treatment of COVID-19 suggests that other anti-inflammatory agents might also hold value for the treatment of COVID-19.
Current evidence supporting the treatment of severe COVID-19 with dexamethasone suggests that the need to curtail the cytokine storm inflammatory response transcends the risks of immunosuppression, and other anti-inflammatory agents may therefore benefit patients in this phase of the disease.
While dexamethasone is considered widely available and generally affordable, the high costs of biologics such as tocilizumab therapy may present obstacles to wide-scale distribution of this drug if it proves of value.
At the doses used for RA patients, the cost for tocilizumab ranges from $179.20 to $896 per dose for the IV form and $355 for the pre-filled syringe [@url:https://www.ncbi.nlm.nih.gov/books/NBK349513/table/T43].
Several other anti-inflammatory agents used for the treatment of autoimmune diseases may also be able to counter the effects of the cytokine storm induced by the virus, and some of these, such as cyclosporine, are likely to be more cost-effective and readily available than biologics [@url:https://escholarship.umassmed.edu/meyers_pp/385].
While tocilizumab targets IL-6, several other inflammatory markers could be potential targets, including TNF-&alpha;.
Inhibition of TNF-&alpha; by a compound such as Etanercept was previously suggested for treatment of SARS-CoV-1 [@doi:10.1185/030079903125002757] and may be relevant for SARS-CoV-2 as well.
Another anti-IL-6 antibody, sarilumab, is also being investigated [@url:http://www.news.sanofi.us/2020-03-16-Sanofi-and-Regeneron-begin-global-Kevzara-R-sarilumab-clinical-trial-program-in-patients-with-severe-COVID-19; @clinicaltrials:NCT04327388].
Baricitinib and other small molecule inhibitors of the Janus-activated kinase pathway also curtail the inflammatory response and have been suggested as potential options for SARS-CoV-2 infections [@doi:10/dph5].
Baricitinib, in particular, may be able to reduce the ability of SARS-CoV-2 to infect lung cells [@doi:10/ggnrsx].
Clinical trials studying baricitinib in COVID-19 have already begun in the US and in Italy [@url:https://investor.lilly.com/news-releases/news-release-details/lilly-begins-clinical-testing-therapies-covid-19; @clinicaltrials:NCT04320277].
Identification and targeting of further inflammatory markers that are relevant in SARS-CoV-2 infection may be of value for curtailing the inflammatory response and lung damage.

In addition to immunosuppressive treatments, which are most beneficial late in disease progression, much research is focused on identifying therapeutics for early-stage patients.
For example, although studies of HCQ have not supported the early theory-driven interest in this antiviral treatment, alternative compounds with related mechanisms may still have potential.
Hydroxyferroquine derivatives of HCQ have been described as a class of bioorganometallic compounds that exert antiviral effects with some selectivity for SARS-CoV-1 _in vitro_ [@doi:10.1021/jm0601856].
Future work could explore whether such compounds exert antiviral effects against SARS-CoV-2 and whether they would be safer for use in COVID-19.

Another potential approach is the development of antivirals, which could be broad-spectrum, specific to coronaviruses, or targeted to SARS-CoV-2.
Development of new antivirals is complicated by the fact that none have yet been approved for human coronaviruses.
Intriguing new options are emerging, however.
Beta-D-N4-hydroxycytidine is an orally bioavailable ribonucleotide analog showing broad-spectrum activity against RNA viruses, which may inhibit SARS-CoV-2 replication _in vitro_ and _in vivo_ in mouse models of HCoVs [@doi:10.1126/scitranslmed.abb5883].
A range of other antivirals are also in development.
Development of antivirals will be further facilitated as research reveals more information about the interaction of SARS-CoV-2 with the host cell and host cell genome, mechanisms of viral replication, mechanisms of viral assembly, and mechanisms of viral release to other cells; this can allow researchers to target specific stages and structures of the viral life cycle.
Finally, antibodies against viruses, also known as antiviral monoclonal antibodies, could be an alternative as well and are described in detail in an above section.
The goal of antiviral antibodies is to neutralize viruses through either cell-killing activity or blocking of viral replication [@doi:10.1016/j.tim.2015.07.005].
They may also engage the host immune response, encouraging the immune system to hone in on the virus.
Given the cytokine storm that results from immune system activation in response to the virus, which has been implicated in worsening of the disease, a neutralizing antibody (nAb) may be preferable.
Upcoming work may explore the specificity of nAbs for their target, mechanisms by which the nAbs impede the virus, and improvements to antibody structure that may enhance the ability of the antibody to block viral activity.

Some research is also investigating potential therapeutics and prophylactics that would interact with components of the innate immune response.
For example, TLRs are pattern recognition receptors that recognize pathogen- and damage-associated molecular patterns and contribute to innate immune recognition and, more generally, promotion of both the innate and adaptive immune responses [@ISBN:9780815332183].
In mouse models, poly(I:C) and CpG, which are agonists of Toll-like receptors TLR3 and TLR9, respectively, showed protective effects when administered prior to SARS-CoV-1 infection [@doi:10.1128/JVI.01410-12].
Therefore, TLR agonists hold some potential for broad-spectrum prophylaxis.



## Vaccine Development Strategies for SARS-CoV-2

### Abstract

Vaccines have revolutionized the relationship between people and disease.
In the 21^st^ century, a number of emergent viruses have emphasized the importance of rapid and scalable vaccine development programs.
During the pandemic caused by _Severe acute respiratory syndrome coronavirus 2_ (SARS-CoV-2), recent biotechnological advances in vaccine design provided the circumstances for the development and deployment of vaccines at an unprecedented pace.
The genome sequence of SARS-CoV-2 was released on January 10^th^, 2020, allowing for global efforts in vaccine development to begin within two weeks of the international community becoming aware of the new viral threat.
Both pre-existing vaccine platforms and novel vaccine technologies have been explored against SARS-CoV-2.
Although historically a slow process, vaccine development in the face of COVID-19 accelerated so much that less than a year into the pandemic, some vaccine candidates had reported interim phase III clinical trial data and were being administered in countries around the world.
In this review, we examine the strategies used to develop the leading vaccine candidates and where these candidates currently stand in terms of efficacy, safety, and approval in light of the ongoing pandemic and threat from emerging SARS-CoV-2 variants.
We also discuss the patterns of distribution around the world.
Vaccine development began almost five centuries ago, but the SARS-CoV-2 pandemic provides an exceptional illustration of how rapidly vaccine development technology has evolved in the last two decades.

### Importance

The SARS-CoV-2 pandemic has caused untold damage to the global population, but it also presented some unique opportunities for vaccine development.
As of September 25, 2021, SARS-CoV-2 has infected over 231,515,976 and cost the lives of 4,743,196 people globally.
The development, production, and distribution of vaccines is imperative to saving lives, preventing illness, and reducing the economic and social burdens caused by the COVID-19 pandemic.
Now that promising candidates exist, effective deployment will provide an opportunity to move into a new phase of the pandemic where the susceptibility of worldwide populations is significantly reduced.
This review provides a historical context for vaccine devleopment and highlights the main strategies utilized for the development of the COVID-19 vaccines, their clinical appraisal, and their distribution.
These technologies have revolutionized the timescale at which countries can mount a response to an emerging viral threat and provide potential for mitigating future threats before their damage reaches the levels caused by SARS-CoV-2.
As the SARS-CoV-2 virus has evolved, they also provide insight into how these technologies have to adapt on incredibly short timescales. 

### Introduction

The development of vaccines is widely considered one of the most important medical advances in recent human history.
Over the past 150 years, approaches to vaccine developments have undergone a transformation and the diversity of available strategies has grown [@doi:10.1073/pnas.1400472111].
Since the turn of the millennium, particular interest has emerged in the potential to develop vaccines as a rapid response to emerging threats.
Severe acute respiratory syndrome (SARS), "swine flu" (H1N1), Middle East respiratory syndrome (MERS), and Ebola all underscored the importance of a rapid global response to a new infectious virus, but the vaccine development process has historically been slow, and vaccines fail to provide immediate prophylactic protection or treat ongoing infections [@doi:10.3390/v10120680].
However, the _Severe acute respiratory syndrome-related coronavirus 2_ (SARS-CoV-2) pandemic has highlighted a confluence of circumstances that positioned vaccine development as a key player in efforts to control the virus and mitigate its damage.
Examining the vaccine development programs tackling the COVID-19 pandemic alongside other 21^st^ century efforts to control emerging viral threats demonstrates the significant biotechnological advances in this field.

### Historical Vaccine Development

The first vaccination strategy in human history is widely considered to be the practice of variolation, which makes the history of vaccine development almost 500 years long [@doi:10.1016/j.jinf.2005.07.021; @doi:10.1007/978-1-4419-1339-5_2].
Famously employed as a strategy to improve survival of smallpox by, for example, exposing a healthy individual to pus from smallpox pustules [@doi:10.1377/hlthaff.24.3.611; @doi:10.1016/j.jinf.2005.07.021; @doi:10.1007/978-1-4419-1339-5_2], variolation provides a mechanism for infecting a healthy individual with a mild case of a disease.
This strategy aims to confer adaptive immunity, but it also carries a number of risks for the vaccine recipient [@doi:10.1016/j.virol.2015.03.032].
This approach was (debatably) the first example of a live-attenuated virus being used to induce immunity [@doi:10.1016/j.virol.2015.03.032; 10.1073/pnas.1400472111].
Many subsequent efforts to develop live-attenuated viral vaccines relied on either the identification of related zoonotic viruses that are less virulent in humans (e.g., cowpox/horsepox or rotavirus vaccines) or efforts to attenuate the virus through culturing it _in vitro_ [@doi:10.1073/pnas.1400472111; @doi:10.1377/hlthaff.24.3.611].
This approach still carried risks, however [@doi:10.1073/pnas.1400472111].

Efforts to overcome the limitations of live-virus vaccines led to the development of approaches to inactivate viruses (circa 1900) and to purify proteins from viruses cultured in eggs (circa 1920) [@doi:10.1073/pnas.1400472111; @doi:10.1073/pnas.1402981111].
Inactivated viral vaccines still raised some concerns, however, including that back-mutations could potentially lead the inactivated vaccines to become virulent or that recombination could occur between the inactivated virus and other viruses in circulation [@doi:10.3389/fimmu.2019.00594].
For example, errors in the manufacturing process that produced polio vaccines containing live polio virus led to a polio outbreak in the United States [@doi:10.1056/NEJMp048180].
Concerns about contamination are shared across several vaccination platforms, including those that use live and attenuated viruses [@doi:10.1377/hlthaff.24.3.611].
Additionally, one of the major limitations of inactivated whole-virus vaccines is their susceptibility to lose efficacy due to mutations in the epitopes of the circulating virus [@doi:10.3389/fimmu.2019.00594].
This loss of specificity over time is likely to be influenced by the evolution of the virus, and specifically by the rate of evolution in the region of the genome that codes for the antigen.

With the increased feasibility of genetic research in the 1980s came the application of genetic engineering to vaccine development, which allowed for the growth of the gene sequences of specific viral antigens _in vitro_ [@doi:10.1073/pnas.1400472111].
While vaccine development strategies such as live-attenuation and inactivation remain in use in the 21st century [@doi:10.1073/pnas.1400472111], approaches built off of the principles of genetic engineering offer some of the most dynamic opportunities in the field of modern vaccine development.

### 21^st^ Century Advances in Vaccine Development

![
**Vaccine Development Strategies.**
Several different strategies can and are being employed for the development of vaccines today.
Each approach capitalizes on different features of the SARS-CoV-2 virus and delivery through a different platform.
All of these approaches are being explored in the current pandemic.<!--To Do: define LNP, define mRNA, define S protein. Swap order so viral particles are on top, then DNA, then viral vector, then RNA panels? Not sure if adenovirus/lentivirus is consistent with the rest of the text, need to double-check before finalizing figure. Change viral particle to inactivated whole virus-->
](images/N002-Vaccines.png){#fig:vaccines secno=1}

While traditional methods of vaccine development such as inactivated whole viruses are still used today (Figure @fig:vaccines), biomedical research in the 21st century has been significantly influenced by the genomic revolution, and vaccine development is no exception.
<!--To Do: note which methods had produced successful vaccines in the 1900s and which were new in the genomic era.-->
Building on the advances in vaccine development that came out of genetic engineering, several newer approaches to vaccine development today utilize information available in the DNA sequence of a virus (Figure @fig:vaccines).

In traditional DNA vaccines, the sequence encoding the antigen(s) against which an immune response is sought can be cultivated in a plasmid and delivered directly to an appropriate tissue [@url:https://www.who.int/biologicals/areas/vaccines/dna/en/].
Plasmids are not the only vector that can be used to deliver sequences associated with viral antigens: genetic material from the target virus can also be delivered using a second virus as a vector.
Once the plasmid or viral vector brings the DNA sequence to an antigen presenting cell (APC), the host machinery can be used to construct antigen(s) from the transported genetic material, and the body can then synthesize antibodies in response [@doi:10.1038/nrg2432].

More recently, interest has also emerged the potential for viral RNA to induce an immune response (Figure @fig:vaccines).
This approach operates one level above the DNA: instead of directly furnishing the gene sequence associated with an antigen to the host, it provides the messenger RNA (mRNA) transcribed from the DNA sequence.
Some of the potential advantages of mRNA compared to DNA include safety, as it cannot be integrated by the host and the half life can be regulated, it avoids any issues of a host immune response against the vector, and it holds the potential to dramatically accelerate vaccine manufacturing and development [@doi:10.1038/nrd.2017.243].

These two appraoches are based on a similar underlying principle: utilizing a vector to deliver the information the host needs to produce an antigen, which in turn triggers an immune response to the antigen without introducing an infectious agent.
In addition to lacking an infectious agent, both of these approaches are likely to offer several advantages over more traditional immunization platforms because they can stimulate both B- and T-cell responses [@doi:10.1016/j.coi.2016.05.008; @doi:10.1038/nrg2432].
Current methods in vaccine development can therefore be framed in terms of the central dogma of genetics: instead of directly providing the proteins from the infectious agents, vaccines developers are exploring the potential for the delivery of DNA or RNA to induce the cell to produce proteins from the virus that in turn induce a host immune response.

All of these technologies are fairly new.
<!--To Do: When did DNA vaccines really hit market? + same for viral vector vaccine, ebola was first-->
Prior to 2020, no mRNA vaccines had been approved for use in humans<!--To Do: Cite-->, despite significant advances in the development of this technology [@doi:10.1038/nrd.2017.243].
Vaccine development using mRNA is a recent frontier, with challenges in its execution arising from the instability of mRNA molecules, the design requirements of an efficient delivery system, and the potential for mRNA to elicit a very strong immune response [@doi:10.1038/nrd.2017.243].
Therefore, while these technologies elegantly capitalize on decades of research in vaccine development as well as the tools of the genomic revolution, it was largely unknown prior to the SARS-CoV-2 pandemic whether this potential could be realized in a real-world vaccination effort.

### The Pursuit of Rapid, Scalable Vaccine Development

The requirements for a successful vaccine trial and deployment are complex and may require coordination between government, industry, academia, and philanthropic entities [@doi:10.1126/science.abc5312].
Flu-like illnesses caused by viruses are a common target of vaccine development programs, and influenza vaccine technology in particular has made many strides.
Beyond the seasonal flu, however, a number of emergent viral threats over the past 20 years have challenged the vaccine development pipeline to respond more rapidly to previously unknown viruses.
<!--To Do: SARS-->
During the H1N1 influenza outbreak, vaccine development was accelerated because of the existing infrastructure, along with the fact that regulatory agencies had already decided that vaccines produced using egg- and cell-based platforms could be licensed under the regulations used for a strain change.
Critiques of the production and distribution of the H1N1 vaccine have stressed the need for alternative development-and-manufacturing platforms that can be readily adapted to new pathogens.
Although a monovalent H1N1 vaccine was not available before the pandemic peaked in the United States and Europe, it was available soon afterward as a stand-alone vaccine that was eventually incorporated into commercially available seasonal influenza vaccines [@doi:10.1056/NEJMp2005630].
If H1N1 vaccine development provides any indication, considering developing and manufacturing platforms for promising COVID-19 vaccine trials early could hasten the emergence of an effective prophylactic vaccine against SARS-CoV-2.
<!--To Do: MERS, then Ebola-->
<!--To Do: add a sentence about adjuvants and link to appendix-->
The potential for technologies such as DNA and RNA vaccines to benefit the field of oncology has encouraged vaccine developers to invest in next-generation approaches to vaccine development, which have led to the great diversity of vaccine development programs [@url:https://www.the-scientist.com/news-opinion/newer-vaccine-technologies-deployed-to-develop-covid-19-shot-67152; @doi:10.1038/nrg2432].

### Challenges and Opportunities in Developing a Vaccine against SARS-CoV-2

The emergence of SARS-CoV-2 in late 2019 rapidly induced a global public health crisis.
This viral threat did not follow the same trajectory as other emergent viruses of recent note, such as SARS-CoV-1, MERS-CoV, and ebola virus, none of which reached the level of pandemic.
Spread of the SARS-CoV-2 virus has remained out of control in many parts of the world well into 2021, especially with the emergence of variants that have increased rates of transmission [@individual-pathogenesis].
While, for a variety of reasons, SARS-CoV-2 was not controlled as rapidly as the viruses underlying prior 21^st^ century epidemics, vaccine development technology had also progressed based on these and other prior viral threats to the point that a rapid international vaccine development response was possible.
The first critical step towards developing a vaccine against SARS-CoV-2 was characterizing the viral target, which happened extremely early in the COVID-19 outbreak with the sequencing and dissemination of the viral genome in early January 2020 [@url:https://www.who.int/csr/don/12-january-2020-novel-coronavirus-china/en/] (Figure @fig:virus).
<!--To Do: Focus on S protein, and specifically RBD-- is there anything else being examined we should talk about?-->
The _S_ protein (Figure @fig:virus) is an antigen and induces an immune response [@doi:10/fmq2; @doi:10.3389/fimmu.2020.576622].
<--To Do: insert discussion of pre vs post fusion conformation here->

![
**Structure of the SARS-CoV-2 virus.**
The development of vaccines depends on the immune system recognizing the virus.
Here, the structure of SARS-CoV-2 is represented both in the abstract and against a visualization of the virion.
The abstracted visualization was made using BioRender [@url:https://biorender.com/] and the microscopy was conducted by the National Institute of Allergy and Infectious Diseases [@url:https://www.niaid.nih.gov/news-events/novel-coronavirus-sarscov2-images].
](images/SARS_CoV_2.png){#fig:virus secno=1}

For a highly infectious virus like SARS-CoV-2, a vaccine would hold particular value because it could bolster the immune response to the virus of the population broadly, thereby driving a lower rate of infection and likely significantly reducing fatalities.
The Coalition for Epidemic Preparedness Innovations (CEPI) is coordinating global health agencies and pharmaceutical companies to develop vaccines against SARS-CoV-2.
As of September 2020, there were over 180 vaccine candidates against SARS-CoV-2 in development [@doi:10.1038/s41586-020-2798-3].
While little is currently known about immunity to SARS-CoV-2, vaccine development typically tests for serum neutralizing activity, as this has been established as a biomarker for adaptive immunity in other respiratory illnesses [@doi:10.1056/NEJMoa2022483].
However, unlike in efforts to develop vaccines for prior viral threats, the duration of the COVID-19 pandemic has made it possible to test vaccine in phase III trials, where the effect of the vaccine on a cohort's likelihood of contracting SARS-CoV-2 can be evaluated.
Unlike many global vaccine development programs previously, such as with H1N1, the vaccine development landscape for COVID-19 includes vaccines produced by a wide array of technologies.
These diverse technology platforms include DNA, RNA, virus-like particle, recombinant protein, both replicating and non-replicating viral vectors, live attenuated virus, and inactivated virus approaches (Figure @fig:vaccines).

### Live-Attenuated Viruses

Live-attenuated vaccines (LAV), or replication-competent vaccines, use a weakened living version of a disease-causing virus or a version of a virus that is modified to induce an immune response [@doi:10.1038/s41586-020-2798-3]. 
The virus can be attenuated by passaging it in a foreign host until, as a consequence of selection pressure, the virus loses its efficacy in the original host. 
Alternatively, selective gene deletion or codon de-optimization can be utilized to attenuate the virus [@doi:10.1038/s41586-020-2798-3]. 
LAVs are used globally to prevent diseases caused by viruses such as measles, rubella, polio, influenza, varicella zoster, and the yellow fever virus [@doi:10.1016/B978-0-323-35761-6.00002-X].
It is generally recognized that LAVs induce an immune response similar to natural infection, and they are favored because they induce long-lasting and robust immunity that can protect from disease. 
This strong protective effect is induced in part by the immune response to the range of viral antigens available from LAV, which tend to be more immunogenic than those from non-replicating vaccines [@url:https://www.hhs.gov/immunization/basics/types/index.html; @doi:10.1016/j.virol.2015.03.032; @doi:10.3389/fimmu.2020.602256]. 
LAVs are also favored because they tend to be restricted to viral replication in the tissues around the location of inoculation [@doi:10.1128/9781555818951], and some can be administered intranasally [@doi:10.1038/s41586-020-2798-3]. 

The first deliberate attempt to utilize an attenuated viral vaccine dates back to Louis Pasteur in 1885, despite his not knowing that the disease-causing agent he was experimenting with was a virus. 
Indeed, the next intentional LAVs developed were intended to prevent illness caused by the yellow fever virus in 1935, followed by the first influenza vaccine in 1936 [@doi:10.1128/9781555818951]. 
Although LAV development strategies have the longest history, this strategy has not been widely utilized against SARS-CoV-2 and COVID-19. 
There is general concern that LAV strategies may risk causing disease in individuals who are immunocompromised [@doi:10.1016/j.copbio.2007.10.010], which is an even greater concern when dealing with a novel virus and disease. 
Previously, there have been numerous attempts to develop both SARS-CoV-1 and MERS-CoV LAVs [@doi:10.3389/fimmu.2020.602256], but no vaccines were approved for use in humans.
While safety in production was a major concern in the past, nowadays manufacturers of LAVs use safe and reliable methods to produce large quantities of vaccines once they have undergone rigorous preclinical studies and clinical trials to evaluate their safety and efficacy.

There are at least five COVID-19 LAV candidates at various stages of vaccine development.
A single-dose LAV candidate referred to as YF-SO used live-attenuated yellow fever 17D (YF17D) as a vector for a noncleavable prefusion conformation of the SARS-CoV-2 antigen.
This LAV has been assessed in hamsters, mice, and macaques [@doi:10.1038/s41586-020-3035-9].
YF-SO induced a robust immune response in all three animal models and prevented SARS-CoV-2 infection in macaques and hamsters [@doi:10.1038/s41586-020-3035-9].
Other LAVs being investigated for the prophylaxis of COVID-19 include a Bacillus Calmette-Guerin (BCG) vaccine sponsored by institutes in Australia in collaboration with the Bill and Melinda Gates Foundation, which is in phase III clinical testing [@clinicaltrials:NCT04327206]. 
A second investigation using a BCG vaccine is also ongoing, which is led by Texas A&M University in collaboration with numerous other U.S. institutions [@clinicaltrials:NCT04348370].
The purpose of the BCG vaccine is to prevent tuberculosis; however, it is known to exert protective non-specific effects against other respiratory tract infections in _in vitro_ and _in vivo_ studies [@doi:10.1016/j.cmi.2019.04.020], hence the interest many have for its potential use against COVID-19 [@doi:10.1038/s41577-020-0337-y].

In April 2020, it was announced that the Indian Immunologicals Ltd. and Griffith University Australia had partnered to develop codon de-optimized LAV [@url:https://news.griffith.edu.au/2020/04/23/griffith-university-researchers-on-the-road-to-covid-19-vaccine]; however, there have been no updates on the findings of their preclinical testing. 
Another codon de-optimized LAV is being developed by Mehmet Ali Aydinlar University and Acibadem Labmed Health Services A.S., which also has yet to report the findings of its preclinical tests [@url:https://covid-19tracker.milkeninstitute.org].
Following successful preclinical investigation [@doi:10.1073/pnas.2102775118], an intranasally administered deoptimized SARS-CoV-2 LAV known as COVI-VAC was developed by both New York-based Codagenix and the Serum Institute of India. 
COVI-VAC entered phase I human trials and dosed its first participants in January 2021 [@clinicaltrials:NCT04619628; @url:https://covid-19tracker.milkeninstitute.org].  
It is anticipated that the COVI-VAC phase I human trials will be completed by May 2022.
Similarly,  Meissa Vaccines in Kansas, U.S.A., which also develops vaccines for _Respiratory syncytial virus_ (RSV), began enrollment for phase I human trials on an intranasal live-attenuated chimeric vaccine candidate in March 2021 for which recruitment is ongoing [@clinicaltrials:NCT04798001; @url:https://covid-19tracker.milkeninstitute.org].

Despite the long and trusted history of LAV development, this vaccine strategy does not appear to be favored for vaccine development against COVID-19.
Modern technologies such as mRNA vaccines and vectored vaccines seem to have been favored due to their expediency and safety versus the time-consuming nature of developing LAVs using a novel virus.  

### Inactivated Whole-Virus Vaccines

Another well-established technology, inactivated whole-virus vaccines, is under development against SARS-CoV-2.
This platform has been a valuable tool in efforts to control many viruses, and some well-known whole virus vaccines targets include influenza viruses, poliovirus, and hepatitis A virus.
These types of vaccines use full virus particles that have been rendered non-infectious by chemical (i.e., formaldehyde or &beta;-propiolactone [@doi:10.1080/14760584.2017.1357471]) or physical (i.e., heat) means.
Though these virus particles are inactivated, they still have the capacity to prime the immune system.
The size of the virus particle makes it ideal for uptake by APC, which leads to stimulation of helper T-cells [@doi:10.1038/nri2868].
Additionally, the array of epitopes on the surface of the virus increases antibody binding efficiency [@doi:10.1038/nri2868].
The native conformation of the surface proteins, which is also important for eliciting an immune response, is preserved using these techniques.
Membrane proteins, which support B-cell responses to surface proteins, are also included using this method [@doi:10.1084/jem.164.4.1114].
Overall, these vaccines are able to mimic the key properties of the virus that stimulate a robust immune response, but the risk of adverse reactions is reduced because the virus is inactivated and thus unable to replicate.

<!--To Do: Were these investigated for SARS, MERS, ebola, H1N1?-->

One prominent inactivated whole-virus vaccine against SARS-CoV-2 is being developed by Sinovac, a Beijing-based biopharmaceutical company.
Their CoronaVac vaccine uses an inactivated whole virus with the addition of an aluminum adjuvant [@doi:10.3389/fimmu.2020.585354] and is currently in Phase III clinical trials in Brazil [@doi:10.1186/s13063-020-04775-4].
<!--To Do: Expand slightly (1-2 sentences) on how it is made -- how do they inactivate it?-->
Phase I and II clinical trials indicated a strong immunogenic response in animal models and the development of neutralizing antibodies in human participants [@doi:10/fx8z; @doi:10.1126/science.abc1932; @doi:10/fpcx].
<!--To Do: Check if there are any specifics available about immunogenicity that we can compare across vaccines...-->
Safety analysis of the vaccine during the phase II trial revealed that most adverse reactions were either mild (grade 1) or moderate (grade 2) in severity.<!--To Do: check this, original text said "all" but that is in conflict with 2 lines below-->
The most common symptom was pain at the injection site (9%) and fever (3%), and only 2% (n=7) of participants participants reported severe adverse events, though these were determined to be unrelated to the vaccine.
While data from the phase III study will be required to evaluate the efficacy of the virus under real-world conditions of exposure, the current results suggest that this vaccine is likely to adapt a well-established approach to vaccine development for the prevention of COVID-19.<!--To Do: these vaccines are not doing so well against variants supposedly-->

<!--To Do: reorganize text from paragraph below, maybe some moves to appendix-->
India, the biggest producer of vaccines globally, has developed COVAXIN®, which is an indigenous COVID-19 vaccine researched and manufactured by Bharat Biotech International Ltd. in collaboration with the Indian Council of Medical Research (ICMR) - National Institute of Virology (NIV). 
Bharat Biotech reported 80.6% vaccine efficacy for its whole virion inactivated  BBV152 (COVAXIN®) vaccine candidate in 25,800 participants in phase III clinical trials [@url:https://www.bharatbiotech.com/images/press/covaxin-phase3-efficacy-results.pdf; @url:https://www.biogenetech.co.th/wp-content/uploads/2021/03/5-Ocugen.pdf]. 
It was reported in _The Lancet_ that the BBV152 vaccine candidate adjuvanted with alum and a Toll-like receptor 7/8 (TLR7/8) agonist is safe, immunogenic, and induces Th1-skewed memory T-cell responses upon immunization [@doi:10/gh7597].
Importantly, sera from individuals vaccinated with COVAXIN^®^ efficiently neutralized the B.1.1.7 variant and the heterologous strain of SARS-CoV-2 [@doi:10.1101/2021.01.26.426986].
U.S.-based Ocugen Inc., a  co-development partner of Bharat Biotech, is leading the application for an Emergency Use Authorization (EUA) for COVAXIN™ intended for the U.S. market. 
As of April 1, 2021 COVAXIN® has been approved for emergency use in Iran, Zimbabwe, and Nepal, and Mauritius and Paraguay have received a commercial supply of the vaccine.
In Asia, China and India are the main COVID-19 vaccination developers and providers.
Thus far, over 63 million people have been vaccinated against COVID-19 in India [@url:https://www.mygov.in/covid-19/].
A broad range of COVID-19 vaccine candidates are under investigation in order to respond to the COVID-19 pandemic.
In India, the Covaxin vaccine produced by Bharat Biotech received emergency authorization on January 3rd, 2021, despite the lack of phase III data until March 3rd [@url:https://www.bharatbiotech.com/images/press/covaxin-phase3-efficacy-results.pdf].
Following the release of the phase III data indicating 81% efficacy, Zimbabwe authorized the use of Covaxin [@url:http://www.xinhuanet.com/english/2021-03/04/c_139783893.htm].
In addition to COVAXIN®, Bharat Biotech has also developed an adenovirus vector-based intranasal BBV154 vaccine candidate that induces neutralizing IgG, mucosal IgA, and T cell responses [@url:https://www.bharatbiotech.com/intranasal-vaccine.html]. 
In February, 2021, Bharat Biotech received approval from Indian officials to commence a phase I study of this intranasal chimpanzee-adenovirus (ChAd) vectored SARS-CoV-2-S vaccine [@url:https://www.bharatbiotech.com/intranasal-vaccine.html]. 
Another Indian pharmaceutical industry, Zydus Cadila is developing India’s first indigenous DNA vaccine candidate ZyCoV-D [@doi:10.1101/2021.01.26.428240]  which has completed phase I and II clinical trials and has received approval from Drugs Controller General of India (DCGI) to conduct a phase III clinical trial [@url:https://vaccine.icmr.org.in/covid-19-vaccine].
Moreover, Indian-based Biological E.Limited has partnered with U.S.-based biopharmaceutical company Dynavax Technologies Corporation and Baylor College of Medicine, Texas, to initiate a phase I/II clinical trial in India for its COVID-19 subunit vaccine, which consists of the RBD of the spike protein of SARS-CoV-2 [@url:https://vaccine.icmr.org.in/covid-19-vaccine]. 
Notably, Novavax has signed an agreement with the Serum Institute of India allowing them to produce up to 2 billion doses a year [@url:https://www.reuters.com/article/health-coronavirus-novavax-idUSKBN2661PI].
Novavax has also signed agreements with the U.K., Canada, Australia, and South Korea [@url:https://ir.novavax.com/news-releases/news-release-details/novavax-announces-expanded-collaboration-and-license-agreement] and has projected that they will supply 1.1 billion doses to COVAX who will distribute the vaccines to countries with disadvantaged access to vaccine supplies [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html].
India has vaccinated approximately 24 million people [@url:https://www.bloomberg.com/graphics/covid-vaccine-tracker-global-distribution/].
This has been achieved by mainly using the AstraZeneca-University of Oxford vaccine, known as Covishield in India, which is also produced by the Serum Institute of India, and using India’s own Covaxin vaccine [@url:https://www.bbc.com/news/world-asia-india-55748124].
India has also shipped approximately 58 million COVID-19 vaccines to 66 countries [@url:https://www.mea.gov.in/vaccine-supply.htm]
Considering India produces approximately 60% of the world's vaccines prior to the pandemic, it is no surprise that several other vaccine candidates are under development.
These include ZyCov-Di, a DNA vaccine produced by Zydus Cadila, HGCO19, India’s first mRNA vaccine produced by Genova and HDT Biotech Corporation (of the U.S.), and the Bio E subunit vaccine produced by Biological E in collaboration with U.S.-based Dynavax and the Baylor College of Medicine [@url:https://www.bbc.com/news/world-asia-india-55748124].

CoronaVac has been approved for use in China and has been granted emergency use in Azerbaijan, Brazil, Cambodia, Chile, Colombia, Ecuador, Hong Kong, Indonesia, Laos, Malaysia, Mexico, Philippines, Thailand, Turkey, Ukraine, and Uruguay [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#sinovac].<!--To Do: clean citation-->
Sinovac has reported that their platform now has the capacity to provide up to a billion doses [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#sinovac].<!--To Do: clean citation-->
<!--To Do: A sentence or two comparing to Sinopharm (2 candidates), https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7561304/ BBIBP-CorV; https://pubmed.ncbi.nlm.nih.gov/32778225/ --->
The Sinopharm-Beijing Institute vaccine is currently approved for use in Bahrain, China, and the United Arab Emirates, but has been granted emergency use in Argentina, Cambodia, Egypt, Guyana, Hungary, Iran, Iraq, Jordan, Nepal, Pakistan, Peru, Venezuela, and Zimbabwe, with limited use in both Serbia and the Seychelles [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#sinopharm].<!--To do: clean citation-->
In contrast, the Sinopharm-Wuhan vaccine, which has been approved for use in China since February 25th, 2021, has been distributed almost exclusively within China, with limited supplies distributed to the United Arab Emirates [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#wuhan].<!--To Do: clean citation-->
<!--To Do: This should be slower to produce, I think? How does this compare to other platforms?-->
Delays in vaccine distribution have also caused issues, particularly in Turkey where 10 million doses of Sinovac were due to arrive by December 2020, but instead only 3 million were delivered in early January [@url:https://www.nytimes.com/2021/01/25/business/china-covid-19-vaccine-backlash.html].
Similar delays and shortages of doses promised have been reported by officials in the Philippines, Egypt, Morocco, and the United Arab Emirates [@url:https://abcnews.go.com/Health/wireStory/philippines-receive-covid-19-vaccine-delays-76163594; @url:https://www.wsj.com/articles/chinas-covid-19-vaccine-makers-struggle-to-meet-demand-11612958560].
This will be concerning to China who have vaccine contracts for millions of doses with Indonesia (>100 million), Brazil (100 million), Chile (60 million), Turkey (50 million), Egypt (40 million) and many others [@url:https://www.wsj.com/articles/chinas-covid-19-vaccine-makers-struggle-to-meet-demand-11612958560].

### Protein Subunit Vaccines

<!--To Do: Add to figure-->
Compared to the inactivated whole virus vaccines, these protein subunit vaccines isolate a single protein of the virus and use it to stimulate the immune system.
These proteins, also known as antigens, are usually those located on the surface of the viral particle and are therefore key targets of the immune system.
These proteins are typically grown in yeast and then harvested.
This vaccine can stimulate antibodies and CD4^+^ T-cell response [@doi:10.1007/978-1-4615-1823-5].
The main advantage of this method is that protein subunit vaccines are considered very safe, as the antigen alone cannot cause an infection.
However, the immune response is weaker and an adjuvant is usually needed to boost the response [@doi:10.1517/14712598.2011.573624] (see Appendix).
<!--To Do: Application/advancements in prior 21st century epidemics?-->

NVX-CoV2373, produced by U.S. company Novavax, is a protein nanoparticle vaccine candidate against SARS-CoV-2 that is constructed from a mutated prefusion SARS-CoV-2 spike protein in combination with a specialized adjuvant to elicit an immune response against SARS-CoV-2.
The spike protein is recombinantly expressed in Sf9 insect cells [@doi:10.1038/s41467-020-20653-8], which have previously been used for several other FDA-approved protein therapeutics [@doi:10.1021/acs.iecr.8b00985] and contains mutations in the furin cleavage site (682-RRAR-685 to 682-QQAQ-685) along with two proline substitutions (K986P and V987P) that improve thermostability [@doi:10.1038/s41467-020-20653-8].
In preclinical mouse models, Novavax-CoV2373 elicited high anti-spike IgG titers 21 to 28 days post-vaccination that could neutralize the SARS-CoV-2 virus and protect the animals against virus challenge, with particularly strong effects when administered with the proprietary adjuvant Matrix-M^TM^ [@doi:10.1038/s41467-020-20653-8].
In a phase I/II trial, a two-dose regimen of NVX-CoV2373 was found to induce anti-spike IgG levels and neutralizing antibody-titers exceeding those observed in convalescent plasma donated by symptomatic patients [@doi:10.1056/NEJMoa2026920].
In line with the preclinical studies, the use of Matrix-M adjuvant further increased anti-spike immunoglobulin levels and induced a Th1 response.
Although the phase III trial data has not been published yet, Novavax announced an efficacy of 89.3% based on their phase III trial in the UK and also noted that 90% of cases occurring in their phase IIb study in South Africa were caused by a variant of concern, B.1.351 [@url:https://ir.novavax.com/news-releases/news-release-details/novavax-covid-19-vaccine-demonstrates-893-efficacy-uk-phase-3].
Despite these very preliminary results, Novavax has signed an agreement with the Serum Institute of India allowing them to produce up to 2 billion doses a year [@url:https://www.reuters.com/article/health-coronavirus-novavax-idUSKBN2661PI] and has also signed agreements with the U.K., Canada, Australia, and South Korea [@url:https://ir.novavax.com/news-releases/news-release-details/novavax-announces-expanded-collaboration-and-license-agreement] as well as projecting that they will supply 1.1 billion doses to COVAX for distribution to countries with limited access to vaccine supplies [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html].
<!--To Do: closing sentence contextualizing role protein subunit vaccines may play in this pandemic relative to previous ones-->

### Vaccines Delivering DNA

The delivery and presentation of antigens is fundamental to inducing immunity against a virus such as SARS-CoV-2.
DNA vaccines offer an approach to delivering foreign substances into the body in a way that induces both a humoral and cellular immune response [@doi:10.1038/nrg2432].
Delivering a DNA sequence to host cells allows the host to synthesize an antigen without exposure to a viral threat [@doi:10.1038/nrg2432].
Host-synthesized antigens can then be presented in complex with major histocompatibility complex (MHC) I and II, which can activate either T- or B-cells [@doi:10.1038/nrg2432].
While these vaccines encode specific proteins, providing many of the benefits of a protein subunit vaccine, they do not carry any risk of DNA being live, replicating, or spreading, and their manufacturing process lends itself to scalability [@doi:10.1038/nrg2432].
Many of the safety concerns raised about DNA vaccines were not found to be an issue during preclinical and phase I testing, although antibiotic resistance introduced during the plasmid selection process remained a concern during this initial phase of development [@doi:10.1038/nrg2432].
However, the immunogenicity of these vaccines has also not reached expectations [@doi:10.1038/nrg2432].
<!--To Do: Expand. I have a bunch of papers downloaded about why this such a groundbreaking technology, need to read and summarize-->

#### DNA Vaccines

In the 1990s and 2000s, DNA vaccines delivered via plasmids sparked significant scientific interest, leading to a large number of preclinical trials [@doi:10.1038/nrg2432].
Early preclinical trials primarily focused on long-standing disease threats, including viral diseases such as rabies and bacterial diseases such as malaria, and promising results led to phase I testing of the application of this technology to HIV, influenza, malaria, and other diseases of concern during this period [@doi:10.1038/nrg2432].
Although they were well-tolerated, these early attempts to develop vaccines were generally not very successful in inducing immunity to the target pathogen, with either limited T-cell or limited neutralizing antibody responses observed [@doi:10.1038/nrg2432].
<!--To Do: Plasmid DNA vaccines for other 21st century epidemics, and the coming-of-age of this technology in the late 00s-->

<!--To Do: Add background on how the INO-4800 vaccine is designed-->
Currently, a Phase I safety and immunogenicity clinical trial of INO-4800, a prophylactic vaccine against SARS-CoV-2, is underway [@clinicaltrials:NCT04336410].
The vaccine developer Inovio Pharmaceuticals Technology is overseeing administration of INO-4800 by intradermal injection followed by electroporation with the CELLECTRA® device to healthy volunteers.
Electroporation is the application of brief electric pulses to tissues in order to permeabilize cell membranes in a transient and reversible manner.
It has been shown that electroporation can enhance vaccine efficacy by up to 100-fold, as measured by increases in antigen-specific antibody titers [@doi:10.1016/j.coi.2011.03.008].
The safety of the CELLECTRA® device has been studied for over seven years, and these studies support the further development of electroporation as a safe vaccine delivery method [@doi:10.4161/hv.24702].
The temporary formation of pores through electroporation facilitates the successful transportation of macromolecules into cells, allowing cells to robustly take up INO-4800 for the production of an antibody response.
Approved by the United States (U.S.) FDA on April 6, 2020, the phase I study is enrolling up to 40 healthy adult volunteers in Philadelphia, PA at the Perelman School of Medicine and at the Center for Pharmaceutical Research in Kansas City, MO.
The trial has two experimental arms corresponding to the two locations.
Participants in Experimental Group 1 will receive one intradermal injection of 1.0 milligram (mg) of INO-4800 followed by electroporation using the CELLECTRA® 2000 device twice, administered at Day 0 and Week 4.
Participants in Experimental Group 2 will receive two intradermal injections of 1.0 mg (total 2.0 mg per dosing visit) of INO-4800 followed by electroporation using the CELLECTRA® 2000 device, administered at Day 0 and Week 4.
Safety data and the initial immune responses of participants from the trial are expected by the end of the summer of 2021.
The development of a DNA vaccine against SARS-CoV-2 by Inovio could be an important step forward in the world's search for a COVID-19 vaccine.
Although exciting, the cost of vaccine manufacturing and electroporation may make scaling the use of this technology for prophylactic use for the general public difficult.

#### Viral-Vector DNA Vaccines

Viral vectors have emerged as a safe and efficient method to furnish the nucleotide sequences of an antigen to the immune system using a second virus as a vector [@doi:10.1128/CVI.00298-16].
The genetic content of the vector virus is often altered to prevent it from replicating, but replication-competent viruses can also be used under certain circumstances [@doi:10.1016/j.coi.2016.05.014].
The vaccine then uses the host machinery to construct antigen(s) from the transported genetic material, for which the body synthesizes antibodies in response.
One of the early viral vectors explored was adenovirus, with serotype 5 (Ad5) being particularly effective [@doi:10.1038/nrg2432].
This technology rose in popularity during the 2000s due to its being more immunogenic in humans and non-human primates than plasmid-vectored DNA vaccines [@doi:10.1038/nrg2432].
In the 2000s, interest also arose in utilizing simian adenoviruses as vectors because of the reduced risk that human vaccine recipients would have prior exposure resulting in adaptive immunity [@doi:10.1038/nrg2432; @doi:10.1038/mt.2013.284], and chimpanzee adenoviruses were explored as a potential vector in the development of a vaccine against _Middle East respiratory syndrome-related coronavirus_ (MERS-CoV) [@doi:10.3390/v12080861].
Today, various viral-vector platforms including poxviruses [@doi:10/cnw6vw; @doi:10.4161/hv.28974], adenoviruses [@doi:10.1038/mt.2009.130], and vesicular stomatitis viruses [@pmid:10196265; @doi:10.1016/j.molmed.2004.03.003] are being developed,
Viral-vector vaccines are able to induce both an antibody and cellular response; however, the response is limited due to the immunogenicity of the viral vector used [@doi:10.1016/j.coi.2011.03.006; @doi:10.1038/mt.2009.130].
An important consideration in identifying potential vectors is the immune response to the vector.
Both the innate and adaptive immune responses can potentially respond to the vector, limiting the ability of the vaccine to transfer information to the immune system [@doi:10.1038/gt.2009.148].
Different vectors are associated with different levels of reactogenicity; for example, adenoviruses elicit a much stronger innate immune response than replication deficient adeno-associated viruses derived from parvoviruses [@doi:10.1038/gt.2009.148].
Additionally, using a virus circulating widely in human populations as a vector presents additional challenges because vaccine recipients may already have developed an immune response to the vector [@doi:10.3390/vaccines2030624].

There are several viral vector vaccines that are available for veterinary use [@doi:10.3390/vaccines8040680; @doi:10.1038/nrg2432], but prior to the COVID-19 pandemic, only one viral vector vaccine was approved by the FDA for use in humans.
This vaccine is vectored with a recombinant vesicular stomatitis virus and targeted against the ebola virus [doi:10.1016/j.cell.2020.03.011].
Additionally, several phase I and phase II clinical trials for other vaccines are ongoing [@doi:10.1128/CVI.00298-16], and the technology is currently being explored for its potential against numerous infectious diseases including malaria [@doi:10.1016/j.ymthe.2016.11.003; @doi:10.1016/j.vaccine.2006.07.035], ebola [@doi:10.1056/NEJMoa1410863; @doi:10.1093/infdis/jir349; @doi:10.1586/14760584.2014.885841], and human immunodeficiency virus (HIV) [@doi:10.1097/COH.0b013e328363d389; @doi:10.1016/j.virol.2014.09.004].
The threat of MERS and SARS initiated interest in the application of viral vector vaccines to human coronaviruses [@doi:10.3390/v12080861], but efforts to apply this technology to these pathogens had not yet led to a successful vaccine candidate.
In the mid-to-late 00s, adenoviral vectored vaccines against SARS were found to induce SARS-CoV-specific IgA in the lungs of mice [@doi:10.1099/vir.0.81579-0], but were later found to offer incomplete protection in ferret models [@doi:10.1099/vir.0.2008/001891-0].
Gamaleya National Center of Epidemiology and Microbiology in Moscow sought to use an adenovirus platform for the development of vaccines for _Middle East respiratory syndrome-related coronavirus_ and Ebola virus, although neither of the previous vaccines were internationally licensed [@url:https://www.who.int/immunization/sage/meetings/2018/october/2_Ebola_SAGE2018Oct_BgDoc_20180919.pdf].

In 2017, results were published from an initial investigation of two vaccine candidates against MERS-CoV containing the MERS-CoV S gene vectored with chimpanzee adenovirus, Oxford University #1 (ChAdOx1), a replication-deficient chimpanzee adenovirus [@doi:10.1016/j.vaccine.2017.05.032].
This study reported that a candidate containing the complete spike protein sequence induced a stronger neutralizing antibody response in mice than candidates vectored with modified vaccinia virus Ankara.
It was pursued in additional research, and in the summer of 2020 results of two studies were published.
The first reported that a single dose of ChAdOx1 MERS induced an immune response and inhibited viral replication in macaques [@doi:10.1126/sciadv.aba8399].
The second reported  promising results from a phase I trial that administered the vaccine to adults and measured safety/tolerability and immune response (as indicated through immune assays following vaccination) [@doi:10/ggtxgp].

While not all of these results were available at the time that vaccine development programs against SARS-CoV-2 began, at least three viral vector vaccines have also been developed against this hCoV.
First, collaboration between AstraZeneca and researchers at the University of Oxford has successfully applied a viral vector approach to the development of a vaccine against SARS-CoV-2 using the replication-deficient ChAdOx1 vector modified to encode the spike protein of SARS-CoV-2 [@doi:10.1038/s41586-020-2608-y].
In phase I and I/II trials, respectively, the immunogenic potential of vaccine candidate ChAdOx1 nCoV-19 was demonstrated through the immune challenge of two animal models, mice and rhesus macaques [@doi:10.1038/s41586-020-2608-y] and patients receiving the ChAdOx1 nCoV-19 vaccine developed antibodies to the SARS-CoV-2 spike protein that peaked by day 28, with these levels remaining stable until a second observation at day 56 [@doi:10/gg5gwk].
In December 2020, preliminary results of the phase III trial were released detailing randomized control trials conducted in the U.K., Brazil, and South Africa between April and November 2020 [@doi:10/fmq2].
These trials again compared ChAdOx1 nCoV-19 to a control, but the design of each study varied; pooling data across studies indicated an overall efficacy of 70.4%. <!--To Do: symptomatic?-->
ChAdOx1 nCoV-19 was first approved for emergency use on December 30, 2020 in the United Kingdom [@url:https://www.astrazeneca.com/media-centre/press-releases/2020/astrazenecas-covid-19-vaccine-authorised-in-uk.html] and has since then been approved for emergency use in several dozen countries, in addition to receiving full approval in Brazil.

Second, a viral vector approach was also applied by Gamaleya to develop Sputnik V, a replication-deficient recombinant adenovirus (rAd) vaccine that combines two adenovirus vectors, rAd26-S and rAd5-S, that express the full-length SARS-CoV-2 spike glycoprotein.
The two vectors are administered intramuscularly administered sequentially, following a prime-boost regimen.
Despite a lack of data from clinical trials, President Vladimir Putin of Russia announced the approval of the Sputnik V vaccines on August 11th, 2020 [@doi:10/ft7j] and it has subsequently been administered in Russia and other countries.
Subsequently, the phase I/II clinical trial was published and indicated that the vaccine was safe, with the most common adverse events being mild pain at the injection site (58%), hypothermia (50%), headaches (42%), fatigue (28%), and joint and muscle pain (24%), and immunogenic, with seroconversion observed in all participants three weeks after the second dose and with all participants producing antibodies to the SARS-CoV-2 glycoprotein [@doi:10/gg96hq].
In February 2021, six months after its approval in Russia, interim results of the phase III trial were released, indicating an overall vaccine efficacy of 91.6% for symptomatic COVID-19 [@doi:10/ghxj4g].
As of early January, Sputnik V had been administered to as many as 1.5 million Russians [@url:https://www.brusselstimes.com/news-contents/world/149039/1-5-million-people-have-received-sputnik-v-vaccine-russia-says-russian-direct-investment-fund-mikhail-murashko/] , and doses of Sputnik V have also been distributed to other parts of Europe, such as Belarus, Bosnia-Herzegovina, Hungary, San Marino, Serbia, and Slovakia [@url:https://www.euronews.com/2021/02/12/hungary-to-begin-using-russia-s-sputnik-v-vaccine-today; @url:https://www.euronews.com/2021/02/24/san-marino-buys-russia-s-sputnik-v-after-eu-vaccine-delivery-delays; @url:https://www.themoscowtimes.com/2020/12/29/belarus-starts-coronavirus-vaccination-with-sputnik-v-a72512], with the Czech Republic and Austria also having expressed interest in its procurement [@url:https://www.cnbc.com/2021/03/02/russias-sputnik-vaccine-is-luring-eastern-europe-worrying-the-eu.html].

Third, Janssen Pharmaceuticals, Inc., a subsidiary of Johnson & Johnson, also developed a viral vector vaccine in collaboration with and funded by the United States's “Operation Warp Speed” [@url:https://www.gao.gov/products/gao-21-319; @url:https://www.jnj.com/johnson-johnson-announces-a-lead-vaccine-candidate-for-covid-19-landmark-new-partnership-with-u-s-department-of-health-human-services-and-commitment-to-supply-one-billion-vaccines-worldwide-for-emergency-pandemic-use].
The vaccine candidate JNJ-78436735, formerly known as Ad26.COV2-S, is a monovalent vaccine that is composed of a replication-deficient adenovirus serotype 26 (Ad26) vector expressing the stabilized prefusion S protein of SARS-CoV-2 [@doi:10.1056/NEJMoa2034201;@doi:10.1038/s41541-020-00243-x]. 
Unlike the other two viral vector vaccines available to date, JNJ-78436735 requires only a single dose, a characteristic that is expected to aid in global deployment [@doi:10.1038/s41586-020-2607-z].
JNJ-78436735 was selected from among a number of initial candidate designs [@doi:10.1038/s41541-020-00243-x] and tested _in vivo_ in Syrian golden hamsters and Rhesus macaques to assess safety and immunogenicity [@doi:10.1038/s41541-020-00243-x; @doi:10.1038/s41586-020-2607-z; @doi:10.1038/s41591-020-1070-6; @doi:10.1101/2020.11.17.368258].
The JNJ-78436735 candidate was selected for its favorable immunogenicity profile and ease of manufacturability [@doi:10.1038/s41541-020-00243-x; @doi:10.1038/s41586-020-2607-z; @doi:10.1038/s41591-020-1070-6; @doi:10.1101/2020.11.17.368258] and was found to confer protection against SARS-CoV-2 in macaques even after six months [@doi:10.1101/2021.01.30.428921].
The one- versus two-dose regimen was tested in volunteers through a phase I/IIa trial [@doi:10.1056/NEJMoa2034201], although these results are not yet available; however, the study did report that the vaccine was well-tolerated and that most participants demonstrated seroconversion in a neutralization assay 29 days after immunization [@doi:10.1056/NEJMoa2034201].
The phase III trial is ongoing across several countries (Argentina, Brazil, Chile, Colombia, Mexico, Peru, South Africa, and the U.S.), but interim results were reported in a press release on January 29th, 2021 [@url:https://www.nih.gov/news-events/news-releases/janssen-investigational-covid-19-vaccine-interim-analysis-phase-3-clinical-data-released; @url:https://www.janssen.com/emea/sites/www_janssen_com_emea/files/johnson_johnson_announces_single-shot_janssen_covid-19_vaccine_candidate_met_primary_endpoints_in_interim_analysis_of_its_phase_3_ensemble_trial.pdf].
The vaccine was well-tolerated, and across all regions studied, it was found to be 66% effective after 28 days for the prevention of moderate to severe COVID-19 and to be 85% effective for the prevention of laboratory-confirmed severe COVID-19 as well as 100% protection against COVID-19-related hospitalization and death.
However, the reported efficacy ranged from 57% in South Africa to 72% in the United States, suggesting that these observations might be influenced by the prominent viral strains circulating in each country at the time of the trial; at the time, several variants of concern including B.1.351, which was first identified in South Africa [@url:https://cov-lineages.org/global_report_B.1.351.html], were being monitored.

The three viral-vector vaccines described above have demonstrated the potential for this technology to facilitate a quick response to an emerging pathogen.
However, two of the three vaccines have faced a number of criticisms surrounding the implementation of their clinical trials.
<--To Do: Suggestion to move some of the Sputnik controversy here, along with describing the issues with the AstraZeneca trial-->
Additionally, though the vaccines are developed using similar principles, there are some differences that might influence their efficacy as SARS-CoV-2 evolves.
In the Janssen vaccine, the S protein immunogen is stabilized in its prefusion conformation, while in the Sputnik V and AstraZeneca vaccines, it is not.
The prefusion conformation of the SARS-CoV-2 S protein is metastable [@doi:10.1016/j.lfs.2020.118056], and the release of energy during membrane fusion drives this process forward following destabilization [@doi:10.1146/annurev-virology-110615-042301; @doi:10.1038/nature17200].
Due to the significant conformational changes that occur during membrane fusion [@doi:10.1016/j.cell.2020.02.058; @doi:10.1016/j.str.2020.10.001; @doi:10.1038/s41586-020-2665-2], S protein immunogens that are stabilized in the prefusion conformation are of particular interest, especially because a prefusion stabilized MERS-CoV S antigen was found to elicit an improved antibody response [@doi:10.1073/pnas.1707304114].
Moreover, the prefusion conformation offers an opportunity to target S2, a region of the S protein that accumulates mutations at a slower rate [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009; @doi:10.1073/pnas.1707304114] (see also [@individual-pathogenesis]).
Vaccine developers can use versions of the spike protein that contain mutations that stabilize the prefusion conformer, essentially locking them in this position [@doi:10.1126/science.abd0826].
The immune response to the spike protein when it is stabilized in this conformation is improved over other S structures [@doi:10.1038/s41541-020-00243-x].
Thus, vaccines that use this prefusion stabilized conformation, including the Janssen viral-vector vaccine, the Novavax-CoV2373 protein nanoparticle vaccine, as well as the Moderna and Pfizer/BioNTech mRNA vaccines, are expected to not only offer improved immunogenicity, but also be more resilient to the accumulation of mutations in SARS-CoV-2.
How these differences in design influence the efficacy of these three viral-vector vaccines over time remains to be seen.

### RNA Vaccines

Building on DNA Vaccine technology, RNA vaccines are an even more recent advancement for vaccine development.
RNA vaccines are nucleic-acid based modalities that code for viral antigens against which the human body elicits a humoral and cellular immune response.
The mRNA technology is transcribed _in vitro_ and delivered to cells via lipid nanoparticles (LNP) [@doi:10.1002/eji.1830230749].
They are recognized by ribosomes _in vivo_ and then translated and modified into functional proteins [@doi:10.3389/fimmu.2019.00594].
The resulting intracellular viral proteins are displayed on surface MHC proteins, provoking a strong CD8+ T cell response as well as a CD4+ T cell and B cell-associated antibody responses [@doi:10.3389/fimmu.2019.00594].
Naturally, mRNA is not very stable and can degrade quickly in the extracellular environment or the cytoplasm.
The LNP covering protects the mRNA from enzymatic degradation outside of the cell [@doi:10.4155/tde-2016-0006].
Codon optimization to prevent secondary structure formation and modifications of the poly-A tail as well as the 5’ untranslated region to promote ribosomal complex binding can increase mRNA expression in cells.
Furthermore, purifying out double-stranded RNA and immature RNA with FPLC (fast performance liquid chromatography) and HPLC (high performance liquid chromatography) technology will improve translation of the mRNA in the cell [@doi:10.3389/fimmu.2019.00594; @doi:10.1080/14760584.2017.1355245].

There are three types of RNA vaccines: non-replicating, _in vivo_ self-replicating, and _in vitro_ dendritic cell non-replicating [@url:https://www.phgfoundation.org/briefing/rna-vaccines].
Non-replicating mRNA vaccines consist of a simple open reading frame (ORF) for the viral antigen flanked by the 5’ UTR and 3’ poly-A tail.
_In vivo_ self-replicating vaccines encode a modified viral genome derived from single-stranded, positive sense RNA alphaviruses [@doi:10.3389/fimmu.2019.00594; @doi:10.1080/14760584.2017.1355245].
The RNA genome encodes the viral antigen along with proteins of the genome replication machinery, including an RNA polymerase.
Structural proteins required for viral assembly are not included in the engineered genome [@doi:10.3389/fimmu.2019.00594].
Self-replicating vaccines produce more viral antigens over a longer period of time, thereby evoking a more robust immune response [@url:https://www.phgfoundation.org/briefing/rna-vaccines].
Finally, _in vitro_ dendritic cell non-replicating RNA vaccines limit transfection to dendritic cells.
Dendritic cells are potent antigen-presenting immune cells that easily take up mRNA and present fragments of the translated peptide on their MHC proteins, which can then interact with T cell receptors.
Ultimately, primed T follicular helper cells can stimulate germinal center B cells that also present the viral antigen to produce antibodies against the virus [@doi:10.1016/j.immuni.2014.10.004].
These cells are isolated from the patient, grown and transfected _ex vivo_, and reintroduced to the patient [@doi:10.1038/nrd.2017.243].

Vaccines based on mRNA delivery confer many advantages over traditional viral vectored vaccines and DNA vaccines.
In comparison to live attenuated viruses, mRNA vaccines are non-infectious and can be synthetically produced in an egg-free, cell-free environment, thereby reducing the risk of a detrimental immune response in the host [@doi:10.1016/j.immuni.2020.03.007].
Unlike DNA vaccines, mRNA technologies are naturally degradable and non-integrating, and they do not need to cross the nuclear membrane in addition to the plasma membrane for their effects to be seen [@doi:10.3389/fimmu.2019.00594].
Furthermore, mRNA vaccines are easily, affordably, and rapidly scalable.
Although mRNA vaccines have been developed for therapeutic and prophylactic purposes, none have previously been licensed or commercialized.
Nevertheless, they have shown promise in animal models and preliminary clinical trials for several indications, including rabies, coronavirus, influenza, and cytomegalovirus [@clinicaltrials:NCT01669096].
Preclinical data previously identified effective antibody generation against full-length FPLC-purified influenza hemagglutinin stalk-encoding mRNA in mice, rabbits, and ferrets [@doi:10.1038/s41467-018-05482-0].
Similar immunological responses for mRNA vaccines were observed in humans in Phase I and II clinical trials operated by the pharmaceutical-development companies Curevac and Moderna for rabies, flu, and zika [@doi:10.1080/14760584.2017.1355245].
Positively charged bilayer LNPs carrying the mRNA attract negatively charged cell membranes, endocytose into the cytoplasm [@doi:10.4155/tde-2016-0006], and facilitate endosomal escape.
LNPs can be coated with modalities recognized and engulfed by specific cell types, and LNPs that are 150 nm or less effectively enter into lymphatic vessels [@doi:10.4155/tde-2016-0006; @doi:10.1038/s41467-018-06936-1].
Therefore, this technology holds great potential for targeted delivery of modified mRNA.
<!-- To Do: development of this tech during previous epidemics?-->

Given the potential for this technology to be quickly adapted for a new pathogen, it has held significant interest for the treatment of COVID-19.
In the vaccines developed under this approach, the mRNA coding for a stabilized prefusion spike protein, which is immunogenic [@doi:10.1016/j.immuni.2020.05.002], can be furnished to the immune system in order to train its response.
Two vaccine candidates in this category emerged with promising phase III results at the end of 2020.
Both require two doses approximately one month apart.
The first was Pfizer/BioNTech's BNT162b2, which contains the full prefusion stabilized, membrane-anchored SARS-CoV-2 spike protein in a vaccine formulation based on modified mRNA (modRNA) technology [@doi:10.1101/cshperspect.a014035; @url:https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-announce-early-positive-data-ongoing-0].
<!--To Do: brief history of phase I & phase II (like 1-2 sentences)-->
In the phase II/III multinational trial, this vaccine was associated with a 95% efficacy against laboratory-confirmed COVID-19 and with mild-to-moderate local and systemic effects but a low risk of serious adverse effects [@doi:10.1056/NEJMoa2034577].
Similarly, ModernaTX developed mRNA-1273, which, despite being the second mRNA vaccine to release phase III results, was the first mRNA vaccine to enter phase I clinical trials.
mRNA-1273 is comprised by a conventional lipid nanoparticle encapsulated RNA encoding a full-length prefusion stabilized S protein for SARS-CoV-2 [@clinicaltrials:NCT04283461].
In the phase I trial, neutralizing activity reached similar levels to that observed in convalescent plasma samples by day 7 after the second dose of RNA-1273 [@doi:10.1056/NEJMoa2022483].
A few months later, interim results from the phase III trial indicated 94.5% efficacy of the vaccine in preventing symptomatic COVID-19 in adults who received the vaccine at 99 sites around the United States [@doi:10.1056/NEJMoa2035389].
Similar to BNT162b2, the vaccine was associated with mild-to-moderate adverse effects but with a low risk of serious adverse events [@doi:10.1056/NEJMoa2035389]. 
In late 2020, both vaccines both received approval from the United States's Food and Drug Administration (FDA) under an emergency use authorization [@url:https://www.fda.gov/news-events/press-announcements/fda-takes-key-action-fight-against-covid-19-issuing-emergency-use-authorization-first-covid-19; @url:https://www.cdc.gov/mmwr/volumes/69/wr/mm695152e1.htm], and these vaccines have been widely distributed, primarily in North America and the European Union [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html].
As the first mRNA vaccines to make it to market, these two highly efficacious vaccines demonstrate the power of this emerging technology, which has previously attracted scientific interest because of its potential to be used to treat non-infectious as well as infectious diseases.

### Discussion (Probably to be fleshed out into multiple sections)

Given the wide range of vaccines under development, it is possible that some vaccine products may eventually be shown to be more effective in certain subpopulations, such as children, pregnant women, immunocompromised patients, the elderly, etc.
However, the vaccine development process has historically been slow, and vaccines fail to provide immediate prophylactic protection or treat ongoing infections [@doi:10.3390/v10120680].

<!--To Do: Add table of different technologies and which vaccines fall under each, ability to respond to VOC-->

Concerns: diversity of volunteer pools, variants, and distribution
Another benefit of vaccines is lower population size in SARS-CoV-2 = less risk of VOC emerging that are less susceptible to the vaccine


## Vaccine Development Strategies for SARS-CoV-2

#### Appendix: Sinovac's CoronaVac

Pre-clinical trials were performed using BALB/c mice and rhesus macaques [@doi:10.1126/science.abc1932].
The SARS-CoV-2 strains used in this trial isolated from 11 hospitalized patients (5 from China, 3 from Italy, 1 from the UK, 1 from Spain, 1 from Switzerland).
A phylogenetic analysis demonstrated that the strains were representative of the current circulating variants.
One of the strains, CN2, from China was used as the inactivated and purified virus while the other 10 strains were used to challenge.
The CN2 was grown in Vero cells.
An ELISA assay was used to assess the immunogenicity of the vaccine.
10 mice were injected with the vaccine on day 0 and day 7 with varying doses (0, 1.5, 3 or 6 &mu;g), and 10 mice were treated with physiological saline as the control.
IgG developed in the serum of all vaccinated mice.
Using the same setup, immunogenicity was also assessed in macaques.
Four macaques were assigned to each of four groups: treatment with 3 &mu;g at day 0, 7 and 14, treatment with a high dose of 6 &mu;g at day 0, 7 and 14, administration of a placebo vaccine, and administration of only the adjuvant.
All vaccinated macaques induced IgGs and neutralizing antibodies.
After challenge with SARS-CoV-2 strain CN1, vaccinated macaques were protected compared to control macaques (placebo or adjuvant only) based on histology and viral loads collected from different regions of the lung.

Phase I/II clinical trials were conducted in adults 18-59 years old [@doi:10/fpcx] and adults over 60 years old [@doi:10/fx8z] in China.
In the case of adults 18-59 years old, a single center, randomized, double-blind, placebo-controlled phase I/II trial was conducted in April 2020.
Patients in this study were recruited from the community in Suining County of Jiangsu province, China.
For the phase I trial, 144 (of 185 screened) participants were enrolled, with 72 enrolled in the 14-day interval cohort (i.e., treated on day 0 and day 14) and 72 in the 28-day interval cohort.
This group of 72 participants was split into 2 blocks for a low-dose (3 &mu;g) and high-dose (6 &mu;g) vaccine.
Within each block, participants were randomly assigned vaccination with CoronaVac or placebo (aluminum diluent without the virus) at a 2:1 ratio.
Both the vaccine and placebo were prepared in a Good Manufacturing Practice-accredited facility of Sinovac Life Sciences (Beijing, China).

The phase II trial followed the same organization of participants, this time using 300 enrolled participants in the 14-day and another 300 enrolled in the 28-day groups.
One change of note was that the vaccine was produced using a highly automated bioreactor (ReadyToProcess WAVE 25, GE, Ume&aring;, Sweden) to increase vaccine production capacity.
This change resulted in a higher intact spike protein content.
The authors of this study were not aware of this antigen-level difference between the vaccine batches for the phase I/II when the ethical approval for the trials occurred.

To assess adverse responses, participants were asked to record any events up to 7 days post-treatment. 
The reported adverse events were graded according to the China National Medical Products Administration guidelines.
In the phase I trial, the overall incidence of adverse reactions was 29-38% of patients in the 0 to 14 day group and 13-17% in the 0 to 28 day vaccination group.
The most common symptom was pain at the injection site, which was reported by 17-21% of patients in the 0 to 14 day cohort and 13% in the 0 to 28 day cohort.
Most adverse reactions were mild (grade 1) where patients recovered within 48 hours.
A single case of acute hypersensitivity with manifestation of urticaria 48 hours following the first dose of study drug was reported in the 6 &mu;g group
Most adverse reactions were mild (grade 1) in severity and participants recovered within 48 hours. 
There was a single case, from the 6 &mu;g group, of acute hypersensitivity with manifestation of urticaria 48 hours after the first dose.
Both the 14-day and 28-day cohorts had a strong neutralizing Ab response.
The neutralizing Ab response was measured using a micro cytopathogenic effect assay, which assesses the minimum dilution of neutralizing Ab to be 50% protective against structural changes in host cells in response to viral infection [@doi:10.1016/j.vaccine.2012.03.010].
Additionally IgG antibody titers against the receptor binding domain were also measured using ELISA.

Another phase I/II study was performed with older patients (older than 60 years) [@doi:10/fx8z].
The study conducted a single-center, randomized, double-blind, placebo-controlled trial.
The phase I trial looked at dose escalation using 3 dosages: 1.5, 3 and 6 &mu;g.
The mean age of participants was 65.8 years (std = 4.8).
Of 95 screened participants, 72 were enrolled.
These 72 participants were split into low (3 &mu;g) and high (6 &mu;g) dose groups.
Within each group, 24 participants received the treatment and 12 the placebo.
A neutralizing antibody response against live SARS-CoV-2 was detected compared to baseline using the same micro cytopathogenic effect assay.
This response was similar across the two dose concentrations.
Additionally, they did not observe a difference in response between age groups (60–64 years, 65–69 years, and ≥70 years).

In phase II the mean age was 66.6 years (standard deviation = 4.7).
499 participants were screened and 350 were enrolled.
300 were evenly split into 1.5, 3 and 6 &mu;g dose groups, and the remaining 50 were assigned to the placebo group.
Again, they found a neutralizing antibody response in phase II.
There wasn't a significant different between the response to 3 &mu;g versus 6 &mu;g, but the response was higher than that to 1.5 &mu;g.

Participants were required to record adverse reaction events within the first 7 days after each dose.
The safety results were combined across phage I and II.
All adverse reactions were either mild (grade 1) or moderate (grade 2) in severity.
The most common symptom was pain at the injection site (9%) and fever (3%).
2% (7 participants) reported severe adverse events (4 from the 1.5 &mu;g group, 1 from the 3 &mu;g group, 2 from the 6 &mu;g group), though these were found to be unrelated to the vaccine.

Overall, the results from the pre-clinical and phase I/II clinical trials are promising.
It was also very hopeful to see that the immune response was consistent in older adults (> 60 years).
Currently, phase III trials are being conducted in Brazil [@doi:10.1186/s13063-020-04775-4].
This is a randomized, multicenter, endpoint driven, double-blind, placebo-controlled clinical trial.
They are expecting 13,060 participants with 11,800 ages 18 to 59 years and 1,260 age 60+.
Participants will be health care professionals.

### RNA Vaccines

RNA vaccines are nucleic-acid based modalities that code for viral antigens against which the human body elicits a humoral and cellular immune response.
The mRNA technology is transcribed _in vitro_ and delivered to cells via lipid nanoparticles (LNP) [@doi:10.1002/eji.1830230749].
They are recognized by ribosomes _in vivo_ and then translated and modified into functional proteins [@doi:10.3389/fimmu.2019.00594].
The resulting intracellular viral proteins are displayed on surface MHC proteins, provoking a strong CD8+ T cell response as well as a CD4+ T cell and B cell-associated antibody responses [@doi:10.3389/fimmu.2019.00594].
Naturally, mRNA is not very stable and can degrade quickly in the extracellular environment or the cytoplasm.
The LNP covering protects the mRNA from enzymatic degradation outside of the cell [@doi:10.4155/tde-2016-0006].
Codon optimization to prevent secondary structure formation and modifications of the poly-A tail as well as the 5’ untranslated region to promote ribosomal complex binding can increase mRNA expression in cells.
Furthermore, purifying out double-stranded RNA and immature RNA with FPLC (fast performance liquid chromatography) and HPLC (high performance liquid chromatography) technology will improve translation of the mRNA in the cell [@doi:10.3389/fimmu.2019.00594; @doi:10.1080/14760584.2017.1355245].
Vaccines based on mRNA delivery confer many advantages over traditional viral vectored vaccines and DNA vaccines.
In comparison to live attenuated viruses, mRNA vaccines are non-infectious and can be synthetically produced in an egg-free, cell-free environment, thereby reducing the risk of a detrimental immune response in the host [@doi:10.1016/j.immuni.2020.03.007].
Unlike DNA vaccines, mRNA technologies are naturally degradable and non-integrating, and they do not need to cross the nuclear membrane in addition to the plasma membrane for their effects to be seen [@doi:10.3389/fimmu.2019.00594].
Furthermore, mRNA vaccines are easily, affordably, and rapidly scalable.

Although mRNA vaccines have been developed for therapeutic and prophylactic purposes, none have previously been licensed or commercialized.
Nevertheless, they have shown promise in animal models and preliminary clinical trials for several indications, including rabies, coronavirus, influenza, and cytomegalovirus [@clinicaltrials:NCT01669096].
Preclinical data previously identified effective antibody generation against full-length FPLC-purified influenza hemagglutinin stalk-encoding mRNA in mice, rabbits, and ferrets [@doi:10.1038/s41467-018-05482-0].
Similar immunological responses for mRNA vaccines were observed in humans in Phase I and II clinical trials operated by the pharmaceutical-development companies Curevac and Moderna for rabies, flu, and zika [@doi:10.1080/14760584.2017.1355245].
Positively charged bilayer LNPs carrying the mRNA attract negatively charged cell membranes, endocytose into the cytoplasm [@doi:10.4155/tde-2016-0006], and facilitate endosomal escape.
LNPs can be coated with modalities recognized and engulfed by specific cell types, and LNPs that are 150 nm or less effectively enter into lymphatic vessels [@doi:10.4155/tde-2016-0006; @doi:10.1038/s41467-018-06936-1].
Therefore, this technology holds great potential for targeted delivery of modified mRNA.

There are three types of RNA vaccines: non-replicating, _in vivo_ self-replicating, and _in vitro_ dendritic cell non-replicating [@url:https://www.phgfoundation.org/briefing/rna-vaccines].
Non-replicating mRNA vaccines consist of a simple open reading frame (ORF) for the viral antigen flanked by the 5’ UTR and 3’ poly-A tail.
_In vivo_ self-replicating vaccines encode a modified viral genome derived from single-stranded, positive sense RNA alphaviruses [@doi:10.3389/fimmu.2019.00594; @doi:10.1080/14760584.2017.1355245].
The RNA genome encodes the viral antigen along with proteins of the genome replication machinery, including an RNA polymerase.
Structural proteins required for viral assembly are not included in the engineered genome [@doi:10.3389/fimmu.2019.00594].
Self-replicating vaccines produce more viral antigens over a longer period of time, thereby evoking a more robust immune response [@url:https://www.phgfoundation.org/briefing/rna-vaccines].
Finally, _in vitro_ dendritic cell non-replicating RNA vaccines limit transfection to dendritic cells.
Dendritic cells are potent antigen-presenting immune cells that easily take up mRNA and present fragments of the translated peptide on their MHC proteins, which can then interact with T cell receptors.
Ultimately, primed T follicular helper cells can stimulate germinal center B cells that also present the viral antigen to produce antibodies against the virus [@doi:10.1016/j.immuni.2014.10.004].
These cells are isolated from the patient, grown and transfected _ex vivo_, and reintroduced to the patient [@doi:10.1038/nrd.2017.243].

Given the potential for this technology to be quickly adapted for a new pathogen, it has held significant interest for the treatment of COVID-19.
In the vaccines developed under this approach, the spike protein, which is immunogenic [@doi:10.1016/j.immuni.2020.05.002], can be furnished to the immune system in order to train its response.
The vaccine candidates developed against SARS-CoV-2 using mRNA vectors utilize similar principles and technologies, although there are slight differences in implementation among candidates such as the formulation of the platform and the specific components of the spike protein encapsulated (e.g., the full Spike protein vs. the RBD alone) [@doi:10.3389/fphar.2020.00937].
The results of the interim analyses of two mRNA vaccine candidates became available at the end of 2020 and provided strong support for this emerging approach to vaccination.
Below we describe the results available as of February 2021 for two such candidates, mRNA-1273 produced by ModernaTX and BNT162b2 produced by Pfizer, Inc. and BioNTech.

#### ModernaTX mRNA Vaccine

ModernaTX's mRNA-1273 vaccine was the first COVID-19 vaccine to enter a phase I clinical trial in the United States.
In this trial, Moderna spearheaded an investigation on the immunogenicity and reactogenicity of mRNA-1273, a conventional lipid nanoparticle encapsulated RNA encoding a full-length prefusion stabilized S protein for SARS-CoV-2 [@clinicaltrials:NCT04283461].
An initial report described the results of enrolling forty-five participants who were administered intramuscular injections of mRNA-1273 in their deltoid muscle on day 1 and day 29, with the goal of following patients for the next twelve months [@doi:10.1056/NEJMoa2022483].
Healthy males and non-pregnant females aged 18-55 years were recruited for this study and divided into three groups receiving 25, 100, or 250 micrograms (&mu;g) of mRNA-1273.
IgG ELISA assays on patient serology samples were used to examine the immunogenicity of the vaccine [@clinicaltrials:NCT04283461].
Binding antibodies were observed at two weeks after the first dose at all concentrations.
At the time point one week after the second dose was administered on day 29, the pseudotyped lentivirus reporter single-round-of-infection neutralization assay (PsVNA), which was used to assess neutralizing activity, reached a median level similar to the median observed in convalescent plasma samples.
Participants reported mild and moderate systemic adverse events after the day 1 injection, and one severe local event was observed in each of the two highest dose levels.
The second injection led to severe systemic adverse events for three of the participants at the highest dose levels, with one participant in the group being evaluated at an urgent care center on the day after the second dose.
The reported localized adverse events from the second dose were similar to those from the first.

Several months later, a press release from ModernaTX described the results of the first interim analysis of the vaccine [@url:https://investors.modernatx.com/news-releases/news-release-details/modernas-covid-19-vaccine-candidate-meets-its-primary-efficacy].
On November 16, 2020, a report was released describing the initial results from Phase III testing, corresponding to the first 95 cases of COVID-19 in the 30,000 enrolled participants [@url:https://investors.modernatx.com/news-releases/news-release-details/modernas-covid-19-vaccine-candidate-meets-its-primary-efficacy], with additional data released to the FDA on December 17, 2020 [@url:https://www.fda.gov/advisory-committees/advisory-committee-calendar/vaccines-and-related-biological-products-advisory-committee-december-17-2020-meeting-announcement#event-materials].
These results were subsequently published in a peer-reviewed journal (_The New England Journal of Medicine_) on December 30, 2020 [@doi:10.1056/NEJMoa2035389].
The first group of 30,420 study participants were randomized to receive the vaccine or a placebo at a ratio of 1:1 [@doi:10.1056/NEJMoa2035389].
Administration occurred at 99 sites within the United States in two sessions, spaced 28 days apart [@url:https://www.fda.gov/media/144434/download; @doi:10.1056/NEJMoa2035389].
Patients reporting COVID-19 symptoms upon follow-up were tested for SARS-CoV-2 using a nasopharyngeal swab that was evaluated with RT-PCR [@url:https://www.fda.gov/media/144434/download].
The initial preliminary analysis reported the results of the cases observed up until a cut-off date of November 11, 2020.
Of these first 95 cases reported, 90 occurred in participants receiving the placebo compared to 5 cases in the group receiving the vaccine [@url:https://investors.modernatx.com/news-releases/news-release-details/modernas-covid-19-vaccine-candidate-meets-its-primary-efficacy].
These results suggested the vaccine is 94.5% effective in preventing COVID-19.
Additionally, eleven severe cases of COVID-19 were observed, and all eleven occurred in participants receiving the placebo.
The publication reported the results through an extended cut-off date of November 21, 2020, corresponding to 196 cases [@doi:10.1056/NEJMoa2035389].
Of these, 11 occurred in the vaccine group and 185 in the placebo group, corresponding to an efficacy of 94.1%.
Once again, all of the severe cases of COVID-19 observed (n=30) occurred in the placebo group, including one death.
Thus, as more cases are reported, the efficacy of the vaccine has remained above 90%, and no cases of severe COVID-19 have yet been reported in participants receiving the vaccine.

These findings suggest the possibility that the vaccine might bolster immune defenses even for subjects who do still develop a SARS-CoV-2 infection.
The study was designed with an explicit goal of including individuals at high risk for COVID-19, including older adults, people with underlying health conditions, and people of color [@url:https://investors.modernatx.com/news-releases/news-release-details/moderna-has-completed-case-accrual-first-planned-interim].
The Phase III trial population was comprised by approximately 25.3% adults over age 65 in the initial report and 24.8% in the publication [@url:https://www.fda.gov/media/144434/download].
Among the cases reported by both interim analyses, 16-17% occurred in older adults [@url:https://investors.modernatx.com/news-releases/news-release-details/modernas-covid-19-vaccine-candidate-meets-its-primary-efficacy; @doi:10.1056/NEJMoa2035389]..
Additionally, approximately 10% of participants identified a Black or African-American background and 20% identified Hispanic or Latino ethnicity [@url:https://www.fda.gov/media/144434/download; @doi:10.1056/NEJMoa2035389].
Among the first 95 cases, 12.6% occurred in participants identifying a Hispanic or Latino background and 4% in participants reporting a Black or African-American background [@url:https://investors.modernatx.com/news-releases/news-release-details/modernas-covid-19-vaccine-candidate-meets-its-primary-efficacy]; in the publication, they indicated only that 41 of the cases reported in the placebo group and 1 case in the treatment group occurred in "communities of color", corresponding to 21.4% of all cases [@doi:10.1056/NEJMoa2035389].
While the sample size in both analyses is small relative to the study population of over 30,000, these results suggest that the vaccine is likely to be effective in people from a variety of backgrounds.
By all indications, this vaccine is likely to be highly useful in mitigating the damage of SARS-CoV-2.

In-depth safety data was released by ModernaTX as part of their application for an EUA from the FDA and summarized in the associated publication [@doi:10.1056/NEJMoa2035389; @url:https://www.fda.gov/media/144434/download].
Because the detail provided in the report is greater than that provided in the publication, here we emphasize the results observed at the time of the first analysis.
Overall, a large percentage of participants reported adverse effects when solicited, and these reports were higher in the vaccine group than in the placebo group (94.5% versus 59.5%, respectively, at the time of the initial analysis) [@url:https://www.fda.gov/media/144434/download].
Some of these events met the criteria for grade 3 (local or systemic) or grade 4 (systemic only) toxicity [@url:https://www.fda.gov/media/144434/download], but most were grade 1 or grade 2 and lasted 2-3 days [@doi:10.1056/NEJMoa2035389].
The most common local adverse reaction was pain at the injection site, reported by 83.7% of participants receiving the first dose of the vaccine and 88.4% upon receiving the second dose, compared to 19.8% and 19.8% and 17.0%, respectively, of patients in the placebo condition [@url:https://www.fda.gov/media/144434/download].
Fewer than 5% of vaccine recipients reported grade 3 pain at either administration.
Other frequent local reactions included erythema, swelling, and lymphadenopathy [@url:https://www.fda.gov/media/144434/download].
For systemic adverse reactions, fatigue was the most common [@url:https://www.fda.gov/media/144434/download].
Among participants receiving either dose of the vaccine, 68.5% reported fatigue compared to 36.1% participants receiving the placebo [@url:https://www.fda.gov/media/144434/download].
The level of fatigue experienced was usually fairly mild, with only 9.6% and 1.3% of participants in the vaccine and placebo conditions, respectively, reporting grade 3 fatigue [@url:https://www.fda.gov/media/144434/download], which corresponds to significant interference with daily activity [@url:https://www.fda.gov/media/73679/download].
Based on the results of the report, an EUA was issued on December 18, 2020 to allow distribution of this vaccine in the United States [@url:https://www.cdc.gov/mmwr/volumes/69/wr/mm695152e1.htm], and it was shortly followed by an Interim Order authorizing distribution of the vaccine in Canada [@url:https://investors.modernatx.com/news-releases/news-release-details/health-canada-authorizes-moderna-covid-19-vaccine-canada] and a conditional marketing authorization by the European Medicines Agency to facilitate distribution in the European Union [@url:https://www.ema.europa.eu/en/news/ema-recommends-covid-19-vaccine-moderna-authorisation-eu].

#### Pfizer/BioNTech BNT162b2

ModernaTX was, in fact, the second company to release news of a successful interim analysis of an mRNA vaccine and receive an EUA.
The first report came from Pfizer and BioNTech's mRNA vaccine BNT162b2 on November 9, 2020 [@url:https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-announce-vaccine-candidate-against], and a preliminary report was published in the _New England Journal of Medicine_ one month later [@doi:10.1056/NEJMoa2034577].
The vaccine candidate is contains the full prefusion stabilized, membrane-anchored SARS-CoV-2 spike protein in a vaccine formulation based on modified mRNA (modRNA) technology [@doi:10.1101/cshperspect.a014035; @url:https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-announce-early-positive-data-ongoing-0].
This vaccine candidate should not be confused with a similar candidate from Pfizer/BioNTech, BNT162b1, that delivered only the RBD of the spike protein [@doi:10.1038/s41586-020-2639-4; @doi:10.1038/s41586-020-2814-7], which was not advanced to a stage III trial because of the improved reactogenicity/immunogenicity profile of BNT162b2 [@doi:10.1056/NEJMoa2027906].

During the Phase III trial of BNT162b2, 43,538 participants were enrolled 1:1 in the placebo and the vaccine candidate and received two 30-μg doses 21 days apart [@doi:10.1056/NEJMoa2034577].
Of these enrolled participants, 21,720 received BNT162b2 and 21,728 received a placebo [@doi:10.1056/NEJMoa2034577].
Recruitment occurred at 135 sites across six countries: Argentina, Brazil, Germany, South Africa, Turkey, and the United States.
An initial press release described the first 94 cases, which were consistent with 90% efficacy of the vaccine at 7 days following the second dose [@url:https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-announce-vaccine-candidate-against].
The release of the full trial information covered a longer period and analyzed the first 170 cases occurring at least 7 days after the second dose, 8 of which occurred in patients who had received BNT162b2.
The press release characterized the study population as diverse, reporting that 42% of the participants worldwide came from non-white backgrounds, including 10% Black and 26% Hispanic or Latino [@url:https://www.pfizer.com/science/coronavirus/vaccine].
Within the United States, 10% and 13% of participants, respectively, identified themselves as having Black or Hispanic/Latino backgrounds [@url:https://www.pfizer.com/science/coronavirus/vaccine].
Additionally, 41% of participants worldwide were 56 years of age or older [@url:https://www.pfizer.com/science/coronavirus/vaccine], and they reported that the efficacy of the vaccine in adults over 65 was 94% [@url:https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-conclude-phase-3-study-covid-19-vaccine].
The primary efficacy analysis of the Phase III study was concluded on November 18, 2020 [@url:https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-conclude-phase-3-study-covid-19-vaccine], and the final results indicted 94.6% efficacy of the vaccine [@doi:10.1056/NEJMoa2034577].

The safety profile of the vaccine was also assessed [@doi:10.1056/NEJMoa2034577].
A subset of patients were followed for reactogenicity using electronic diaries, with the data collected from these 8,183 participants comprising the solicited safety events analyzed.
Much like those who received the ModernaTX vaccine candidate, a large proportion of participants reported experiencing site injection pain within 7 days of vaccination.
While percentages are broken down by age group in the publication, these proportions correspond to approximately 78% and 73% of all participants after the first and second doses, respectively, overall.
Only a small percentage of these events (less than 1%) were rated as serious, with the rest being mild or moderate, and none reached grade 4.
Some participants also reported redness or swelling, and the publication indicates that in most cases, such events resolved within 1 to 2 days.
Participants also experienced systemic effects, including fever (in most cases lower than 38.9°C and more common after dose 2), fatigue (25-50% of participants depending on age group and dose), headache (25-50% of participants depending on age group and dose), chills, and muscle or joint pain; more rarely, patients could experience gastrointestinal effects such as vomiting or diarrhea.
As with the local events, these events were almost always grade I or II.
While some events were reported by the placebo groups, these events were much rarer than in the treatment group even though compliance was similar.
Based on the efficacy and safety information released, the vaccine was approved in early December by the United Kingdom's Medicines and Healthcare Products Regulatory Agency with administration outside of a clinical trial beginning on December 8, 2020 [@doi:10.1136/bmj.m4714; @url:https://www.bbc.com/news/uk-55227325].
As of December 11, 2020, the United States FDA approved this vaccine under an emergency use authorization [@url:https://www.fda.gov/news-events/press-announcements/fda-takes-key-action-fight-against-covid-19-issuing-emergency-use-authorization-first-covid-19].

### Viral Vector Vaccines

#### ChAdOx1 nCoV-19 (AstraZeneca)

As discussed above, prior analyses of viral vector vaccines against hCoV had indicated that this approach showed potential for inducing an immune response, but little information was available about the effect on real-world immunity.
In the first phase of development, a candidate ChAdOx1 nCoV-19 was evaluated through the immune challenge of two animal models, mice and rhesus macaques [@doi:10.1038/s41586-020-2608-y].
Animals in the treatment condition were observed to develop neutralizing antibodies specific to SARS-CoV-2 (both macaques and mice) and to show reduced clinical scores when exposed to SARS-CoV-2 (macaques) [@doi:10.1038/s41586-020-2608-y].
Next, a phase I/II trial was undertaken using a single-blind, randomized controlled design [@doi:10/gg5gwk].
ChAdOx1 nCoV-19 and a control, the meningococcal conjugate vaccine MenACWY, were administered intramuscularly to adults ages 18 to 55 at five sites within the United Kingdom (U.K.) at a 1:1 ratio (n=543 and n=534, respectively).
All but ten participants received a single dose; this small group received a booster 28 days after their first dose of ChAdOx1 nCoV-19.
Commonly reported local adverse reactions included mild-to-moderate pain and tenderness at the injection site over the course of seven days, while the most common systemic adverse reactions were fatigue and headache; some patients reported severe adverse systemic effects.
The study also reported that many common reactions could be reduced through the administration of paracetamol (acetaminophen), and paracetamol was not found to reduce immunogenicity.
Patients receiving the ChAdOx1 nCoV-19 vaccine developed antibodies to the SARS-CoV-2 spike protein that peaked by day 28, with these levels remaining stable until a second observation at day 56 except in the ten patients who received a booster dose at day 28, in whom they increased by day 56.
Analysis of serum indicated that participants developed antibodies to both S and the RBD, and that 100% of them achieved neutralizing titers by day 28.
By day 35, the neutralization titers of vaccinated patients was comparable to that observed with plasma from convalescents.
This initial study therefore suggested that the vaccine was likely to confer protection against SARS-CoV-2, although analysis of its efficacy in preventing COVID-19 was not reported.

In December 2020, preliminary results of the phase III trial were released detailing randomized control trials conducted in the U.K., Brazil, and South Africa between April and November 2020 [@doi:10/fmq2].
These trials again compared ChAdOx1 nCoV-19 to a control, but the design of each study varied.
For example, in South Africa, the trial was double-blinded, whereas in the U.K. and Brazil it was single-blinded, and one of the two trials carried out in the U.K. examined two dosing regimens (low dose or standard dose, both followed by standard dose).
Some of the trials used MenACWY as a control, while others used saline.
Data was pooled across countries for analysis.
The primary outcome assessed was symptomatic, laboratory-confirmed COVID-19.
There were 131 cases observed among the 11,636 participants eligible for the primary efficacy analysis, corresponding to an overall efficacy of 70.4% (30 out of 5807 in the vaccine arm and 101 out of 5829 in the control arm); the 95.8% CI was reported as 54.8 to 80.6.
However, a higher efficacy was reported in the subgroup of patients who received a low-dose followed by a standard dose (90.0%, 95% CI 67.4 to 97·0).
A total of ten cases of severe COVID-19 resulting in hospitalization were observed among trial participants, and all of these occurred in patients in the control arm of the study.
In line with the previously reported safety profiling for this vaccine, serious adverse events were reported to be comparable across the two arms of the study, with only three events identified as potentially associated with the vaccine itself.
The U.K. Medicines and Healthcare Products Regulatory Agency (MHRA) approved ChAdOx1 nCoV-19 for emergency use on December 30, 2020 [@url:https://www.astrazeneca.com/media-centre/press-releases/2020/astrazenecas-covid-19-vaccine-authorised-in-uk.html].
Additional data about the efficacy of this vaccine became available in a preprint released on March 2, 2021 [@doi:10.1101/2021.03.01.21252652].
This report provided data describing the efficacy of ChAdOx1 nCoV-19, along with Pfizer/BioNTech's BNT162b2, in the U.K. between December 8, 2020 and February 19, 2021 and specifically sought to evaluate the efficacy of the vaccine in the presence of a potentially more contagious variant of concern, B.1.1.7.
All participants in this study were age 70 or older and the efficacy was estimated to increase from 60% at 28 days after vaccination to 73% at 35 days after vaccination, although the standard error also increased over this time.
Therefore, preliminary results suggest that in a number of samples, this vaccine confers a high level of protection against SARS-CoV-2.
<!--To Do: Add discussion of all the drama-->

#### Sputnik-V (Gam-COVID-Vac and Gam-COVID-Vac-Lyo)

The vaccine Gam-COVID-Vac, nicknamed Sputnik V in reference to the space race and "V for vaccine", was developed by the Gamaleya National Center of Epidemiology and Microbiology in Moscow.
Gamaleya is an organization with prior experience using the adenovirus platform for the development of vaccines for _Middle East respiratory syndrome-related coronavirus_ and Ebola virus, although neither of the previous vaccines were internationally licensed [@url:https://www.who.int/immunization/sage/meetings/2018/october/2_Ebola_SAGE2018Oct_BgDoc_20180919.pdf].
The development of Sputnik V was financed by the Russian Direct Investment Fund (RDIF) [@doi:10/ft7j; @doi:10/ghs3sn].
Sputnik V is a replication-deficient recombinant adenovirus (rAd) vaccine that combines two adenovirus vectors, rAd26-S and rAd5-S, that express the full-length SARS-CoV-2 spike glycoprotein.
These vectors are intramuscularly administered individually using two separate vaccines in a prime-boost regimen.
The rAd26-S is administered first, followed by rAd5-S 21 days later.
Both vaccines deliver 10^11^ viral particles per dose.
This approach is designed to overcome any potential pre-existing immunity to adenovirus in the population [@doi:10/ghx7xz], as some individuals may possess immunity to Ad5 [@doi:10.1016/j.vaccine.2011.05.025].
Sputnik V is the only recombinant adenovirus vaccine to utilize two vectors.
Other vaccines, such as the Oxford-AstraZeneca vaccine, utilize the chimpanzee adenovirus vector (ChAdOx1 nCoV-19) for both doses [@doi:10/ghpghz].
The Sputnik V vaccines are available in both a lyophilized (Gam-COVID-Vac-Lyo) and frozen form (Gam-COVID-Vac), which are stored at 2-8°C and -18°C respectively [@doi:10/gg96hq].
The lyophilized vaccine is convenient for distribution and storage, particularly to remote or disadvantaged areas [@doi:10.3390/v13010054].

In the race to develop vaccines against SARS-CoV-2, President Vladimir Putin of Russia announced the approval of the Sputnik V vaccines on August 11th, 2020 in the absence of clinical evidence [@doi:10/ft7j].
Consequently, many international scientific agencies and public health bodies expressed concern that due diligence to the clinical trial process was subverted for the sake of expediency, leading many to question the safety and efficacy of Sputnik V [@doi:10.1126/science.abe3147; @url:https://www.cnbc.com/2020/08/11/scientists-worry-whether-russias-sputnik-v-coronavirus-vaccine-is-safe-and-effective.html; @doi:10/ft7j].
Despite regulatory, safety, and efficacy concerns, pre-orders for 1 billion doses of the Sputnik V were reported within days of the vaccine's approval in Russia [@doi:10/ft7j].
Almost a month later, the phase I/II trial data was published [@doi:10/gg96hq].

In the phase I/II trial study conducted between late June and early August 2020, 76 participants (18-60 years old) were split into two groups of 38 participants, which were non-randomized in two hospitals in Russia.
In phase I, 9 patients received rAd26 and 9 patients received rAd5-S to assess safety over 28 days.
In phase II, at least 5 days after the completion of phase I, 20 patients received a prime-boost vaccination of rAd26-S on day 0 and rAd5-S on day 2, which was administered intramuscularly.
The phase I/II trial reported that both vaccines were deemed safe and well tolerated.
The most common adverse events reported were mild, such as pain at the injection site (58%), hypothermia (50%), headaches (42%), fatigue (28%), and joint and muscle pain (24%).
Seroconversion was observed in all participants three weeks post the second vaccination (day 42), and all participants produced antibodies to the SARS-CoV-2 glycoprotein.
RBD-specific IgG levels were high in both the frozen and lyophilized versions of the vaccine (14,703 and 11,143 respectively), indicating a sufficient immune response to both.
Three weeks post the second vaccination, the virus-neutralizing geometric mean antibody titers were 49.25 and 45.95 from the frozen and lyophilized vaccines, respectively.
At 28 days, median cell proliferation of 1.3% CD4^+^ and 1.1% CD8^+^ were reported for the lyophilized vaccine and 2.5% CD4^+^ and 1.3% CD8^+^ for the vaccine stored frozen.
These results indicated that both forms of Sputnik V appeared to be safe and induce a humoral and cellular response in human subjects [@doi:10/gg96hq], which may be robust enough to persist and not wane rapidly [@doi:10/ghx7xz].

A press release on November 11th, 2020 indicated positive results from an interim analysis of the phase III Sputnik V trials, which reported 92% efficacy in 16,000 volunteers [@doi:10.1126/science.abf6791].
However, this release came only two days after both Pfizer and BioNTech reported that their vaccines had an efficacy over 90%, which led to significant skepticism of the Russian findings for a myriad of reasons including the lack of a published protocol and the “reckless” approval of the vaccine in Russia months prior to the publication of the interim results of the phase III trial [@doi:10.1126/science.abf6791; @doi:10.1038/d41586-020-03209-0].
In February 2021, the interim results of the phase III randomized, double-blind, placebo-controlled trial were eventually published in _The Lancet_ [@doi:10/ghxj4g].
The participants were randomly assigned to receive either a 0.5 mL/dose of vaccine or placebo, which was comprised of the vaccine buffer composition, that was delivered intramuscularly using the same prime-boost regimen as in the phase I/II trials.
From September 7th to Nov 24th, 19,866 participants completed the trial.
Of the 14,964 participants who received the vaccine, 16 (0.1%) were confirmed to have COVID-19, whereas 62 of the 4,902 participants (1.3%) in the placebo group were confirmed to have COVID-19.
Of these participants, no moderate or severe cases of COVID-19 were reported in the vaccine group, juxtaposed with 20 in the placebo group.
However, only symptomatic individuals were confirmed for SARS-CoV-2 infection in this trial.
Therefore, asymptomatic infections were not detected, thus potentially inflating the efficacy estimate.
Overall, a vaccine efficacy of 91.6% (95% CI 85.6-95.2) was reported, where an efficacy of 91.8% was reported for those over 60 years old and 92.7% for those who were 51-60 years old.
Indeed, 14 days after the first dose, 87.6% efficacy was achieved and the immunity required to prevent disease occurred within 18 days of vaccination.
Based on these results, scientists are investigating the potential for a single dose regimen of the rAd26-S sputnik V vaccine [@doi:10.1136/bmj.n309].
By the end of the trial, 7,485 participants reported adverse events, of which 94% were grade I.
Of the 68 participants who experienced serious adverse events during the trial, 45 from the vaccine group and 23 from the placebo groups, none were reported to be associated with the vaccination.
Likewise, 4 deaths occurred during the trial period that were not related to the vaccine [@doi:10/ghxj4g].
The interim findings of the phase III trial indicate that the Sputnik V vaccine regimen appears to be safe with 91.6% efficacy.
Gamaleya had intended to reach a total of 40,000 participants for the completion of their phase III trial.
However, the trial has stopped enrolling participants and the numbers have been cut to 31,000 as many individuals in the placebo group dropped out of the study to obtain the vaccine [@url:https://abcnews.go.com/Health/wireStory/russia-cuts-size-covid-19-vaccine-study-stops-74885458].
Indeed, other trials involving Sputnik V are currently underway in Belarus, India, the United Arab Emirates, and Venezuela [@url:https://sputnikvaccine.com/about-vaccine/].

Preliminary results of a trial of Argentinian healthcare workers in Buenos Aires who were vaccinated with the Sputnik V rAd26-R vector-based vaccine seems to support the short term safety of the first vaccination [@url:https://doi.org/10.1101/2021.02.03.21251071].
Of the 707 vaccinated healthcare workers, 71.3% of the 96.6% of respondents reported at least one adverse event attributed to the vaccine.
Of these individuals, 68% experienced joint and muscle pain, 54% had injection site pain, 11% reported redness and swelling, 40% had a fever, and 5% reported diarrhea.
Only 5% of the vaccinated participants experienced serious adverse events that required medical attention, of which one was monitored as an inpatient.

Additionally, an Independent assessment of Sputnik V in a phase II clinical trial in India found the vaccine to be effective, but the data is not yet publicly available [@url:https://www.reuters.com/article/health-coronavirus-india-vaccine-idUSL4N2JM2XA].
On December 21st, 2020, Gamaleya, AstraZeneca, R-Pharm, and the Russian Direct Investment Fund agreed to assess the safety and immunogenicity of the combined use of components of the AstraZeneca and University of Oxford AZD1222 (ChAdOx1) vaccine and the rAd26-S component of the Sputnik V vaccine in clinical trials [@url:https://rdif.ru/Eng_fullNews/6220/].
This agreement hopes to establish scientific and business relations between the entities with an aim to co-develop a vaccine providing long-term immunization.
The trial, which will begin enrollment soon, will include 100 participants in a phase II open-label study and is hoped to be complete within 6 months.
Participants will first receive an intramuscular dose of AZD1222 on day 1, followed by a dose of rAd26 on day 29.
Participants will be monitored from day 1 for 180 days in total.
The primary outcomes measured will include incidence of serious adverse events post first dose until the end of the study.
Secondary outcome measures will include incidence of local and systemic adverse events 7 days post each dose, a time course of antibody responses for the Spike protein and the presence of anti-SARS-CoV-2 neutralizing antibodies [@clinicaltrials:NCT04686773].

Overall, there is hesitancy surrounding the management of the Sputnik V vaccine approval process and concerns over whether the efficacy data may be inflated due to a lack of asymptomatic testing within the trial.
However, the interim results of the phase III study were promising and further trials are underway, which will likely shed light on the overall efficacy and safety of the Sputnik V vaccine regimen.
There may be some advantage to the Sputnik V approach including the favorable storage conditions afforded by choice between a frozen and lyophilized vaccine.
Furthermore, the producers of Gam-COVID-Vac state that they can produce the vaccine at a cost of less than $10 per dose or less than $20 per patient [@url:https://sputnikvaccine.com].

#### Janssen’s JNJ-78436735

The Johnson & Johnson (J&J) vaccine developed by Janssen Pharmaceuticals, Inc., a subsidiary of J&J, was conducted in collaboration with and funded by “Operation Warp Speed” [@url:https://www.gao.gov/products/gao-21-319; @url:https://www.jnj.com/johnson-johnson-announces-a-lead-vaccine-candidate-for-covid-19-landmark-new-partnership-with-u-s-department-of-health-human-services-and-commitment-to-supply-one-billion-vaccines-worldwide-for-emergency-pandemic-use].
The vaccine candidate JNJ-78436735, formerly known as Ad26.COV2-S, is a monovalent vaccine that is composed of a replication-deficient adenovirus serotype 26 (Ad26) vector expressing the stabilized pre-fusion S protein of SARS-CoV-2 [@doi:10.1056/NEJMoa2034201;@doi:10.1038/s41541-020-00243-x].
The vaccine was developed using Janssen’s AdVac^®^ and PER.C6 platforms that were previously utilized to develop the European Commission-approved Ebola vaccine (Ad26 ZEBOV and MVN-BN-Filo) and their Zika, respiratory syncytial (RSV), and HIV investigational vaccine candidates [@url:https://www.jnj.com/johnson-johnson-initiates-pivotal-global-phase-3-clinical-trial-of-janssens-covid-19-vaccine-candidate].

The development of a single-dose vaccine was desirable by J&J from the outset, with global deployment being a key priority [@doi:10.1038/s41586-020-2607-z].
Using their AdVac^®^ technology, the vaccine can remain stable for up to two years between -15℃ and -25℃ and at least three months at 2-8℃ [@url:https://www.jnj.com/johnson-johnson-initiates-pivotal-global-phase-3-clinical-trial-of-janssens-covid-19-vaccine-candidate].
This allows the vaccine to be distributed easily without the requirement for very low temperature storage, unlike many of the other COVID-19 vaccine candidates.
J&J screened numerous potential vaccine candidates _in vitro_ and in animal models using varying different designs of the S protein, heterologous signal peptides, and prefusion-stabilizing substitutions [@doi:10.1038/s41541-020-00243-x].
A select few candidates were further investigated as a single dose regimen in Syrian golden hamsters, a single dose regimen in rhesus macaques, and a single- and two-dose regimen in both adult and aged rhesus macaques [@doi:10.1038/s41541-020-00243-x; @doi:10.1038/s41586-020-2607-z; @doi:10.1038/s41591-020-1070-6; @doi:10.1101/2020.11.17.368258].
From these studies, the JNJ-78436735 candidate was selected for its favorable immunogenicity profile and ease of manufacturability [@doi:10.1038/s41541-020-00243-x; @doi:10.1038/s41586-020-2607-z; @doi:10.1038/s41591-020-1070-6; @doi:10.1101/2020.11.17.368258].
A SARS-CoV-2 challenge study in rhesus macaques showed that vaccine doses as low as 2 x 10^9^ viral particles/mL was sufficient to induce strong protection in bronchoalveolar lavage but that doses higher than 1.125 x 10^10^ were required to close achieve close to complete protection in nasal swabs [@doi:10.1101/2021.01.27.428380].
Indeed, six months post-immunization, levels of S-binding and neutralizing antibodies in rhesus macaques indicated that the JNJ-78436735 vaccine conferred durable protection against SARS-CoV-2 [@doi:10.1101/2021.01.30.428921].

Following selection of the JNJ-78436735 vaccine, J&J began phase I/IIa trials.
The interim phase I/IIa data was placed on the _medRxiv_ preprint server on September 25th, 2020 [@doi:10.1101/2020.09.23.20199604] and was later published in the _New England Journal of Medicine_ on January 13th, 2021 [@doi:10.1056/NEJMoa2034201].
The phase I/IIa multi-center, randomized, placebo-controlled trial enrolled 402 healthy participants between 18-55 years old and a further 403 healthy older participants ≥ 65 years old [@doi:10.1056/NEJMoa2034201].
Patients were administered either a placebo, a low dose (5 x 10^10^ viral particles per mL), or a high dose (1 X 10^11^ viral particles per mL) intramuscularly as part of either a single- or two-dose regimen.
All patients received injections 56 days apart, but participants in the single-dose condition received the placebo at the second appointment.
Those who received only one dose of either vaccine received a placebo dose at their second vaccination visit.
A comparison of the single versus double dose regimen has yet to be published.
The primary endpoints of both the trial were safety and reactogenicity of each dose.
Fatigue, headache, myalgia, and pain at the injection site were the most frequent solicited adverse events reported by participants.
Although less common, particularly for those in the elderly cohort and those on the low dose regimen, the most frequent systemic adverse effect was fever.
Overall, immunization was well tolerated, particularly at the lower dose concentration.
In terms of reactogenicity, over 90% of those who received either the low or high dose demonstrated seroconversion in a neutralization assay using wild-type SARS-CoV-2, 29 days after immunization [@doi:10.1056/NEJMoa2034201].
Neutralizing geometric mean ratio of antibody titers (GMT) between 224-354 were detected regardless of age.
By day 57, 100% of the 18-55 year old participants had neutralizing GMT (288-488), which remained stable until day 71.
In the ≥ 65 years old cohort, the incidence of seroconversion for the low- and high-dose was 96% and 88% respectively by day 29.

GMTs for the low and high doses were slightly lower for participants ≥ 65 years old (196 and 127 respectively), potentially indicating slightly lower immunogenicity.
Seroconversion of the S antibodies was detected in 99% of individuals between 18-55 years old for the low and high doses (GMTs 528 and 695 respectively), with similar findings reported for the ≥ 65 years old.
Indeed, both dose concentrations also induced robust Th1 cytokine-producing S-specific CD4^+^ T cells and CD8^+^ T cell responses in both age groups.
The findings of the phase I/IIa study supported further investigation of a single immunization using the low dose vaccine.
Therefore, 25 patients were enrolled for a second randomized double-blind, placebo-controlled phase 1 clinical trial currently being conducted in Boston, Massachusetts for 2 years [@doi:10.1001/jama.2021.3645].
Participants received either a single dose followed by a placebo, or a double dose of either a low dose (5 x 10^10^ viral particles/mL) or a high dose (1 x 10^11^ viral particles/mL) vaccine administered intramuscularly on day 1 or day 57.
Placebo-only recipients received a placebo dose on day 1 and 57.
Interim analyses conducted on day 71 indicated that binding and neutralizing antibodies developed 8 days after administration in 90% and 25% of vaccine recipients, respectively.
Binding and neutralizing antibodies were detected in 100% of vaccine recipients by day 57 after a single dose immunization.
Spike-specific antibodies were highly prevalent (GMT 2432 to 5729) as were neutralizing antibodies (GMT 242 to 449) in the vaccinated groups.
Indeed, CD4^+^ and CD8^+^ T-cell responses were also induced, which may provide additional protection, particularly if antibodies wane or poorly respond to infection [@doi:10.1038/s41586-020-03041-6].

On September 23rd, 2020, J&J launched its phase III trial ENSEMBLE and released the study protocol to the public [@url:https://www.jnj.com/johnson-johnson-initiates-pivotal-global-phase-3-clinical-trial-of-janssens-covid-19-vaccine-candidate; @clinicaltrials:NCT04505722].
The trial intended to enroll 60,000 volunteers to assess the safety and efficacy of the single vaccine dose versus placebo with primary endpoints of 14 and 28 days post-immunization [@url:https://www.jnj.com/johnson-johnson-initiates-pivotal-global-phase-3-clinical-trial-of-janssens-covid-19-vaccine-candidate].
The trial was conducted in Argentina, Brazil, Chile, Colombia, Mexico, Peru, South Africa, and the U.S.
The trial was paused briefly in October 2020 to investigate a “serious medical event”, but resumed shortly after [@url:https://www.jnj.com/our-company/johnson-johnson-prepares-to-resume-phase-3-ensemble-trial-of-its-janssen-covid-19-vaccine-candidate-in-the-us].
An interim analysis was reported via press release on January 29th, 2021 [@url:https://www.nih.gov/news-events/news-releases/janssen-investigational-covid-19-vaccine-interim-analysis-phase-3-clinical-data-released; @url:https://www.janssen.com/emea/sites/www_janssen_com_emea/files/johnson_johnson_announces_single-shot_janssen_covid-19_vaccine_candidate_met_primary_endpoints_in_interim_analysis_of_its_phase_3_ensemble_trial.pdf].
The interim data included 43,783 participants who accrued 468 symptomatic cases of COVID-19.
It was reported that the JNJ-78436735 vaccine was 66% effective across all regions studied for the prevention of moderate to severe COVID-19 28 days post-vaccination in those aged 18 years and older.
Notably, JNJ-78436735 was 85% effective for the prevention of laboratory-confirmed severe COVID-19 and 100% protection against COVID-19-related hospitalization and death 28 days post-vaccination across all study sites.
Efficacy of the vaccine against severe COVID-19 increased over time, and there were no cases of COVID-19 reported in immunized participants after day 49.
The trial also determined that the vaccine candidate has a favorable safety profile as determined by an independent Data and Safety Monitoring Board.
The vaccine was well tolerated, consistent with previous vaccines produced using the AdVac^®^ platform.
Fever occurred in 9% of vaccine recipients, with grade 3 fever occurring in only 0.2% of recipients.
Serious adverse events were reportedly higher in the placebo group than the vaccine group, and no anaphylaxis was reported [@url:https://www.janssen.com/emea/sites/www_janssen_com_emea/files/johnson_johnson_announces_single-shot_janssen_covid-19_vaccine_candidate_met_primary_endpoints_in_interim_analysis_of_its_phase_3_ensemble_trial.pdf].

At the time the phase III trial was being conducted, several concerning variants, including B.1.1.7 [@url:https://cov-lineages.org/global_report_B.1.1.7.html] and B.1.351 [@url:https://cov-lineages.org/global_report_B.1.351.html], were spreading across the globe.
In particular, B.1.351 was first identified in South Africa, which was one of the JNJ-78436735 vaccine trial sites.
Therefore, the J&J investigators also analyzed the efficacy of the JNJ-78436735 vaccine associated with their various trial sites to determine any potential risk of reduced efficacy as a result of the novel variants.
It was determined that JNJ-78436735 was 72% effective in the U.S., 66% effective in Latin America, and 57% effective in South Africa 28 days post-vaccination.
These findings underpin the importance of monitoring for the emergence of novel SARS-CoV-2 variants and determining their effects on vaccine efficacy.

Looking forward, Janssen are also running a phase III randomized, double-blind, placebo-controlled clinical trial, Ensemble 2, which aims to assess the efficacy, safety, and immunogenicity of a two-dose regimen of JNJ-78436735 administered 57 days apart.
This trial will enroll 30,000 participants ≥ 18 years old from Belgium, Colombia, France, Germany, Philippines, South Africa, Spain, U.K., and the U.S. [@url:https://www.jnj.com/johnson-johnson-initiates-second-global-phase-3-clinical-trial-of-its-janssen-covid-19-vaccine-candidate].
This trial will also include participants with and without comorbidities associated with an increased risk of COVID-19.

#### Overall Status of Viral-Vector Vaccines

The three viral-vector vaccines described above have demonstrated the potential for this technology to facilitate a quick response to an emerging pathogen.
However, two of the three vaccines have faced a number of criticisms surrounding the implementation of their clinical trials.
<--To Do: Suggestion to move some of the Sputnik controversy here, along with describing the issues with the AstraZeneca trial-->

Additionally, though the vaccines are built using similar principles, there are some differences that might influence their efficacy as SARS-CoV-2 evolves.
<--To Do: suggestion to discuss prefusion conformation (J&J) vs not (the other two)-->

### Sinovac's CoronaVac

The CoronaVac vaccine is being developed by Sinovac, a Beijing-based biopharmaceutical company.
The vaccine is using an inactivate whole virus with the addition of an aluminum adjuvant [@doi:10.3389/fimmu.2020.585354].
The vaccine is currently in Phase III clinical trials.

Pre-clinical trials were performed using BALB/c mice and rhesus macaques [@doi:10.1126/science.abc1932].
The SARS-CoV-2 strains used in this trial isolated from 11 hospitalized patients (5 from China, 3 from Italy, 1 from the UK, 1 from Spain, 1 from Switzerland).
A phylogenetic analysis demonstrated that the strains were representative of the current circulating variants.
One of the strains, CN2, from China was used as the inactivated and purified virus while the other 10 strains were used to challenge.
The CN2 was grown in Vero cells.
An ELISA assay was used to assess the immunogenicity of the vaccine.
10 mice were injected with the vaccine on day 0 and day 7 with varying doses (0, 1.5, 3 or 6 &mu;g), and 10 mice were treated with physiological saline as the control.
IgG developed in the serum of all vaccinated mice.
Using the same setup, immunogenicity was also assessed in macaques.
Four macaques were assigned to each of four groups: treatment with 3 &mu;g at day 0, 7 and 14, treatment with a high dose of 6 &mu;g at day 0, 7 and 14, administration of a placebo vaccine, and administration of only the adjuvant.
All vaccinated macaques induced IgGs and neutralizing antibodies.
After challenge with SARS-CoV-2 strain CN1, vaccinated macaques were protected compared to control macaques (placebo or adjuvant only) based on histology and viral loads collected from different regions of the lung.

Phase I/II clinical trials were conducted in adults 18-59 years old [@doi:10/fpcx] and adults over 60 years old [@doi:10/fx8z] in China.
In the case of adults 18-59 years old, a single center, randomized, double-blind, placebo-controlled phase I/II trial was conducted in April 2020.
Patients in this study were recruited from the community in Suining County of Jiangsu province, China.
For the phase I trial, 144 (of 185 screened) participants were enrolled, with 72 enrolled in the 14-day interval cohort (i.e., treated on day 0 and day 14) and 72 in the 28-day interval cohort.
This group of 72 participants was split into 2 blocks for a low-dose (3 &mu;g) and high-dose (6 &mu;g) vaccine.
Within each block, participants were randomly assigned vaccination with CoronaVac or placebo (aluminum diluent without the virus) at a 2:1 ratio.
Both the vaccine and placebo were prepared in a Good Manufacturing Practice-accredited facility of Sinovac Life Sciences (Beijing, China).

The phase II trial followed the same organization of participants, this time using 300 enrolled participants in the 14-day and another 300 enrolled in the 28-day groups.
One change of note was that the vaccine was produced using a highly automated bioreactor (ReadyToProcess WAVE 25, GE, Ume&aring;, Sweden) to increase vaccine production capacity.
This change resulted in a higher intact spike protein content.
The authors of this study were not aware of this antigen-level difference between the vaccine batches for the phase I/II when the ethical approval for the trials occurred.

To assess adverse responses, participants were asked to record any events up to 7 days post-treatment.
The reported adverse events were graded according to the China National Medical Products Administration guidelines.
In the phase I trial, the overall incidence of adverse reactions was 29-38% of patients in the 0 to 14 day group and 13-17% in the 0 to 28 day vaccination group.
The most common symptom was pain at the injection site, which was reported by 17-21% of patients in the 0 to 14 day cohort and 13% in the 0 to 28 day cohort.
Most adverse reactions were mild (grade 1) where patients recovered within 48 hours.
A single case of acute hypersensitivity with manifestation of urticaria 48 hours following the first dose of study drug was reported in the 6 &mu;g group
Most adverse reactions were mild (grade 1) in severity and participants recovered within 48 hours.
There was a single case, from the 6 &mu;g group, of acute hypersensitivity with manifestation of urticaria 48 hours after the first dose.
Both the 14-day and 28-day cohorts had a strong neutralizing Ab response.
The neutralizing Ab response was measured using a micro cytopathogenic effect assay, which assesses the minimum dilution of neutralizing Ab to be 50% protective against structural changes in host cells in response to viral infection [@doi:10.1016/j.vaccine.2012.03.010].
Additionally IgG antibody titers against the receptor binding domain were also measured using ELISA.

Another phase I/II study was performed with older patients (older than 60 years) [@doi:10/fx8z].
The study conducted a single-center, randomized, double-blind, placebo-controlled trial.
The phase I trial looked at dose escalation using 3 dosages: 1.5, 3 and 6 &mu;g.
The mean age of participants was 65.8 years (std = 4.8).
Of 95 screened participants, 72 were enrolled.
These 72 participants were split into low (3 &mu;g) and high (6 &mu;g) dose groups.
Within each group, 24 participants received the treatment and 12 the placebo.
A neutralizing antibody response against live SARS-CoV-2 was detected compared to baseline using the same micro cytopathogenic effect assay.
This response was similar across the two dose concentrations.
Additionally, they did not observe a difference in response between age groups (60–64 years, 65–69 years, and ≥70 years).

In phase II the mean age was 66.6 years (standard deviation = 4.7).
499 participants were screened and 350 were enrolled.
300 were evenly split into 1.5, 3 and 6 &mu;g dose groups, and the remaining 50 were assigned to the placebo group.
Again, they found a neutralizing antibody response in phase II.
There wasn't a significant different between the response to 3 &mu;g versus 6 &mu;g, but the response was higher than that to 1.5 &mu;g.

Participants were required to record adverse reaction events within the first 7 days after each dose.
The safety results were combined across phage I and II.
All adverse reactions were either mild (grade 1) or moderate (grade 2) in severity.
The most common symptom was pain at the injection site (9%) and fever (3%).
2% (7 participants) reported severe adverse events (4 from the 1.5 &mu;g group, 1 from the 3 &mu;g group, 2 from the 6 &mu;g group), though these were found to be unrelated to the vaccine.

Overall, the results from the pre-clinical and phase I/II clinical trials are promising.
It was also very hopeful to see that the immune response was consistent in older adults (> 60 years).
Currently, phase III trials are being conducted in Brazil [@doi:10.1186/s13063-020-04775-4].
This is a randomized, multicenter, endpoint driven, double-blind, placebo-controlled clinical trial.
They are expecting 13,060 participants with 11,800 ages 18 to 59 years and 1,260 age 60+.
Participants will be health care professionals.

<!--- #### Sinopharm (2 candidates) --->

<!--- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7561304/ BBIBP-CorV --->
<!--- https://pubmed.ncbi.nlm.nih.gov/32778225/ --->

### Protein Subunit Vaccines

<!--Do we need to add this to the figure? Or does it fall in one of the other categories?-->
Compared to the inactivated whole virus vaccines, these protein subunit vaccines isolate a single protein of the virus and use it to stimulate the immune system.
These proteins, also referred to as antigens, are usually those located on the surface of the viral particle and are therefore key targets of the immune system.
These proteins are typically grown in yeast and then harvested.
This vaccine can stimulate antibodies and CD4^+^ T-cell response [@doi:10.1007/978-1-4615-1823-5].
The main advantage of this method is that they are considered very safe because the antigen alone cannot cause an infection; however, the immune response is weaker and an adjuvant is usually needed to boost the response [@doi:10.1517/14712598.2011.573624].

#### Novavax NVX-CoV2373

Novavax-CoV2373 is a protein nanoparticle vaccine candidate against SARS-CoV-2.
The vaccine is constructed from a mutated SARS-CoV-2 spike protein in combination with a specialized adjuvant to elicit an immune response against SARS-CoV-2.
The spike protein is recombinantly expressed in Sf9 insect cells [@doi:10.1038/s41467-020-20653-8], which have previously been used for several other FDA-approved protein therapeutics [@doi:10.1021/acs.iecr.8b00985].
The expressed spike protein contains mutations in the furin cleavage site (682-RRAR-685 to 682-QQAQ-685) to avoid cleavage of the spike protein as well as two proline substitutions (K986P and V987P) to improve thermostability [@doi:10.1038/s41467-020-20653-8].
The improved stability caused by the proline substitutions is particularly critical to facilitating global distribution, particularly to regions where local refrigerator/freezer capacities are limited.
Importantly, these amino acid substitutions did not affect the ability of the spike protein to bind the hACE2 receptor (the target receptor of SARS-CoV-2 spike protein).
The Novavax-CoV2373 vaccine candidate uses a proprietary, saponin-based Matrix-M^TM^ adjuvant that contains two different 40nm-sized particles formed by formulating purified saponin with cholesterol and phospholipids [@doi:10.1007/s12026-018-8991-x].
In preclinical models, the use of the Matrix-M adjuvant potentiated the cellular and humoral immune responses to influenza vaccines [@doi:10.1016/j.vaccine.2013.01.039; @doi:10.1016/j.vaccine.2009.09.044; @doi:10.1111/j.1750-2659.2011.00256.x; @doi:10.1007/s12026-018-8991-x].
Importantly, Matrix-M adjuvant-containing vaccines have shown acceptable safety profiles in human clinical trials [@doi:10.1016/j.vaccine.2011.08.042].

In preclinical mouse models, Novavax-CoV2373 elicited high anti-spike IgG titers 21-28 days post-vaccination that could neutralize the SARS-CoV-2 virus and protect the animals against virus challenge [@doi:10.1038/s41467-020-20653-8].
Antibody titers were significantly elevated in groups receiving the vaccine with the Matrix-M adjuvant compared to the groups without adjuvant.
Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses and generate high frequencies of follicular helper T-cells and B-cell germinal centers after vaccination.
These findings were subsequently evaluated in a baboon primate model, in which Novavax-CoV2373 also elicited high antibody titers against the SARS-CoV-2 spike protein, as well as an antigen specific T-cell response.
Based on this data Novavax initiated a Phase 1/2 clinical trial to evaluate the safety and immunogenicity of Novavax-CoV2373 with Matrix-M [@clinicaltrials:NCT04368988; @doi:10.1056/NEJMoa2026920].

The phase I/II trial was a randomized, placebo-controlled study with 131 healthy adult participants in 5 treatment arms [@doi:10.1056/NEJMoa2026920].
Participants that received the recombinant SARS-CoV-2 vaccine with or without the Matrix-M adjuvant got two injections, 21 days apart.
Primary outcomes that were assessed include reactogenicity, lab-values (serum chemistry and hematology), and anti-spike IgG levels.
Secondary outcomes measured included virus neutralization, T-cell responses, and unsolicited adverse events.
The authors reported that no serious treatment-related adverse events occurred in any of the treatment arms.
Reactogenicity was mostly absent and of short duration.
The two-dose vaccine regimen induced anti-spike IgG levels and neutralizing antibody-titers exceeding those in the convalescent plasma of symptomatic patients.
In line with the preclinical studies, the use of Matrix-M adjuvant further increased anti-spike immunoglobulin levels and induced a Th1 response.
The outcomes of this trial suggest that Novavax-CoV2373 has an acceptable safety profile and is able to induce a strong immune response with high neutralizing antibody titers.
The phase II component of this phase I/II trial was recently uploaded to an open-access repository [@doi:10.1101/2021.02.26.21252482].
This part of the trial was designed to identify which dosing regimen should move forward into late phase clinical trials.
Both younger (18-59 years) and older patients (60-84 years) were randomly assigned to receive either 5 &mu;g or 25 &mu;g Novavax-CoV2373 or placebo in two doses, 21 days apart.<!--To Do: What is placebo?-->
In line with the phase I data, reactogenicity remained mild to moderate and of short duration.
Both dose levels were able to induce high anti-spike IgG titers as well as neutralizing antibody responses after the second dose.
Based on both safety and efficacy data, the 5 &mu;g dosing regimen was selected as the optimal dose regiment for the ongoing phase III trial.
Although the phase III trial data has not been published yet, Novavax announced an efficacy of 89.3% based on their phase 3 trial in the UK and South Africa.
This trial included over 15,000 participants in the UK and 4,000 participants in South Africa with occurrence of a PCR-confirmed symptomatic case as the primary endpoint.
In the first interim analysis (U.K.), 56 cases of COVID-19 were observed in the placebo group compared to 6 cases in the treatment group.
Importantly, the vaccine candidate also shows significant clinical efficacy against the prevalent UK and South African variants.
The company has also initiated the development of new constructs to select candidates that can be used as a booster against new strains and plans to initiate clinical trials for these new constructs in the second quarter of 2021.

### Vaccine Development Summary

### Complementary Approaches to Vaccine Development

#### Adjuvants for Vaccines

Adjuvants include a variety of molecules or larger microbial-related products that have an effect on the immune system or an immune response of interest.
They can either be comprised of or contain immunostimulants or immunomodulators.
Adjuvants are sometimes included within vaccines, especially vaccines other than live-attenuated and inactivated viruses, in order to enhance the immune response.
A review on the development of SARS-CoV-2 vaccines [@doi:10.3390/vaccines8020153] also included a brief summary of the potential of adjuvants for these vaccines, including a brief description of some already commonly used adjuvants.
Different adjuvants can regulate different types of immune responses, so the type or combination of adjuvants used in a vaccine will depend on both the type of vaccine and concern related to efficacy and safety.
A variety of possible mechanisms for adjuvants have been researched [@doi:10.1038/nri2510; @doi:10.3389/fimmu.2018.02619; @doi:10.3390/vaccines8010128], including the following: induction of DAMPs that can be recognized by certain PRRs of the innate immune system; functioning as PAMP that can also be recognized by certain PRRs; and more generally enhancing the humoral or cellular immune responses.
Selection of one or more adjuvants requires considering how to promote the advantageous effects of the components and/or immune response and, likewise, to inhibit possible deleterious effects.
There are also considerations related to the method of delivering (or co-delivering) the adjuvant and antigen components of a vaccine.

#### Trained Immunity

Another approach that is being investigated explores the potential for vaccines that are not made from the SARS-CoV-2 virus to confer what has been termed trained immunity.
In a recent review [@doi:10.1038/s41577-020-0285-6], trained immunity was defined as forms of memory that are temporary (e.g., months or years) and reversible.
It is induced by exposure to whole-microorganism vaccines or other microbial stimuli that generates heterologous protective effects.
Trained immunity can be displayed by innate immune cells or innate immune features of other cells, and it is characterized by alterations to immune responsiveness to future immune challenges due to epigenetic and metabolic mechanisms.
These alterations can take the form of either an increased or decreased response to immune challenge by a pathogen.
Trained immunity elicited by non-SARS-CoV-2 whole-microorganism vaccines could potentially improve SARS-CoV-2 susceptibility or severity [@doi:10.1016/j.cell.2020.04.042].

One type of stimulus which research indicates can induce trained immunity is bacillus Calmette-Guerin (BCG) vaccination.
BCG is an attenuated form of bacteria _Mycobacterium bovis_.
The vaccine is most commonly administered for the prevention of tuberculosis in humans.
Clinical trials in non-SARS-CoV-2-infected adults have been designed to assess whether BCG vaccination could have prophylactic effects against SARS-CoV-2 by reducing susceptibility, preventing infection, or reducing disease severity.
A number of trials are now evaluating the effects of the BCG vaccine or the related vaccine VPM1002 [@doi:10.1016/j.cell.2020.04.042; @clinicaltrials:NCT04327206; @clinicaltrials:NCT04328441; @clinicaltrials:NCT04348370; @clinicaltrials:NCT04350931; @clinicaltrials:NCT04362124; @clinicaltrials:NCT04369794; @clinicaltrials:NCT04373291; @clinicaltrials:NCT04379336; @clinicaltrials:NCT04384549; @clinicaltrials:NCT04387409; @clinicaltrials:NCT04414267; @clinicaltrials:NCT04417335; @clinicaltrials:NCT04435379; @clinicaltrials:NCT04439045].

The ongoing trials are using a number of different approaches.
Some trials enroll healthcare workers, other trials hospitalized elderly adults without immunosuppression who get vaccinated with placebo or BCG at hospital discharge, and yet another set of trials older adults (>50 years) under chronic care for conditions like hypertension and diabetes.
One set of trials, for example, uses time until first infection as the primary study endpoint; more generally, outcomes measured in some of these trials are related to incidence of disease and disease severity or symptoms.
Some analyses have suggested a possible correlation at the country level between the frequency of BCG vaccination (or BCG vaccination policies) and the severity of COVID-19 [@doi:10.1016/j.cell.2020.04.042].
Currently it is unclear whether this correlation has any connection to trained immunity.
Many possible confounding factors are also likely to vary among countries, such as age distribution, detection efficiency, stochastic epidemic dynamic effects, differences in healthcare capacity over time in relation to epidemic dynamics, and these have not been adequately accounted for in current analyses. <!-- TO DO: add in study that debunks this-->
It is unclear whether there is an effect of the timing of BCG vaccination, both during an individual's life cycle and relative to the COVID-19 pandemic.
Additionally, given that severe SARS-CoV-2 may be associated with a dysregulated immune response, it is unclear what alterations to the immune response would be most likely to be protective versus pathogenic (e.g., [@doi:10.1016/j.chom.2020.04.009; @doi:10.1016/j.chom.2020.05.009; @doi:10.1016/j.cell.2020.04.042; @doi:10.1016/j.chom.2020.05.008]).
The article [@doi:10.1016/j.cell.2020.04.042] proposes that trained immunity might lead to an earlier and stronger response, which could in turn reduce viremia and the risk of later, detrimental immunopathology.
While trained immunity is an interesting possible avenue to complement vaccine development efforts through the use of an existing vaccine, additional research is required to assess whether the BCG vaccine is likely to confer trained immunity in the case of SARS-CoV-2.

### Viral evolution and vaccine protection

With these vaccines in place, one concern is how the virus's continued evolution will affect their efficacy.
Since the start of this pandemic, we have already seen multiple variants emerge: B.1.1.7, which emerged in the UK, B.1.351, which emerged in South Africa, and P.1, which emerged in Brazil.

Viruses evolve or mutate at different rates.
Mutation rate is measured as the number of substitutions per nucleotide per cell infected (&mu;~s/n/c~) [@doi:10.1128/JVI.00694-10].
RNA viruses tend to have mutation rates between 10^-6^ to 10^-4^ [@doi:10.1128/JVI.00694-10].
As a reference, influenza A virus has a mutation rate of 10^-5^, whereas the mutation rate of SARS-CoV-2 is lower, with the mutation rate estimated at 10^-6^ [@doi:10.24875/BMHIM.20000183].
The accumulation of mutations allows the virus to escape recognition by the immune system [@doi:10.1093/protein/gzw017].

The efficacy of vaccines depends on their ability to train the immune system to recognize the virus.
Therefore, viruses can develop resistance to vaccines through the accumulation of mutations that affect recognition.
The lower mutation rate of SARS-CoV-2 suggests the possibility of SARS-CoV-2 vaccines having a more long-lasting effect compared to vaccines targeting the influenza A virus.

The current SARS-CoV-2 vaccines in distribution have been reported to provide similar efficacy against the B.1.1.7 variant compared to the variants common at the time they were developed but reduced efficacy against the B.1.351 variant [@doi:10.1056/NEJMc2102017].
Pfizer and Moderna announced that they are working on developing a booster shot to improve efficacy against the B.1.351 variant [@url:https://www.pharmaceutical-technology.com/comment/covid-19-vaccine-effectiveness-affected-by-variants/].
The WHO continues to monitor the emergence of variants and their impact on vaccine efficacy [@url:https://www.who.int/news-room/feature-stories/detail/the-effects-of-virus-variants-on-covid-19-vaccines].
Previous research in the computational prediction of the efficacy of vaccines targeting the influenza A virus might complement efforts to monitor these types of viral outbreaks [@doi:10.1093/cid/ciy323].
To adapt, future vaccines may need to account for multiple variants and strains of SARS-CoV-2, and booster shots may be required [@doi:10.1038/s41591-021-01230-y].

### Global Vaccine Status and Distribution

The unprecedented development of COVID-19 vaccines in under a year since the beginning of the pandemic now requires rapid global vaccine production and distribution plans.
The development of vaccines is costly and complicated, but vaccine distribution can be just as challenging.
Logistical considerations such as transport, storage, equipment (e.g., syringes), the workforce to administer the vaccines, and a continual supply from the manufacturers to meet global demands all must be accounted for and will vary globally due to economic, geographic, and sociopolitical reasons [@doi:10.1016/j.eclinm.2020.100674; @doi:10.1126/science.abe2803; @doi:10.1126/sciadv.abf1374].
Deciding on the prioritization and allocation of the COVID-19 vaccines is also a challenging task due to ethical and operational considerations.
Various frameworks, models, and methods have been proposed to tackle these issues with many countries, regions or states as is the case in the U.S., devising their own distribution and administration plans [@doi:10.1126/science.abe6959; @doi:10.1038/s41598-020-78447-3; @doi:10.1016/j.vaccine.2020.12.059; @doi:10.1136/medethics-2020-107036; @doi:10.1101/2021.01.12.21249694].
The majority of the distribution plans prioritize offering vaccines to key workers such as health care workers, and those who are clinically vulnerable such as the elderly, the immunocompromised, and individuals with comorbidities, before targeting the rest of the population, who are less likely to experience severe outcomes from COVID-19 [@url:https://ourworldindata.org/covid-vaccination-policy].
As of March 6th, 2021, approximately 319 million vaccine doses have been administered in at least 118 countries worldwide using 10 different vaccines [@url:https://www.nytimes.com/interactive/2021/world/covid-vaccinations-tracker.html; @url:https://www.bloomberg.com/graphics/covid-vaccine-tracker-global-distribution/].
The global vaccination rate is currently ~8.1 million doses per day, which at the current rate would take almost 4 years to vaccinate 75% of the world's population according to media estimates of a two-dose regimen [@url:https://www.bloomberg.com/graphics/covid-vaccine-tracker-global-distribution/].
Vaccine production and distribution varies from region to region and seems to depend on the availability of the vaccines and potentially a country's resources and wealth [@url:https://www.nytimes.com/2020/12/25/business/coronavirus-vaccines-global-economy.html].

In North America, the majority of vaccines distributed until March 2021 have been produced by Pfizer-BioNTech and Moderna.
In Canada, the vaccine approval process is conducted by Health Canada, which uses a fast-tracked process whereby vaccine producers can submit data as it becomes available to allow for rapid review.
An approval may be granted following reviews of the available phase III clinical data.
This is followed by a period of pharmacovigilance in the population using their post-market surveillance system, which will monitor the long-term safety and efficacy of any vaccines [@url:https://www.canada.ca/en/health-canada/services/drugs-health-products/covid19-industry/drugs-vaccines-treatments/vaccines/development-approval-infographic.html; @url:https://www.canada.ca/en/public-health/services/diseases/2019-novel-coronavirus-infection/prevention-risks/covid-19-vaccine-treatment/safety-after-authorization.html].
Health Canada has authorized the use of the Pfizer (December 9th, 2020), Moderna (December 23rd, 2020), Oxford-AstraZeneca (February 26th, 2021), and the Janssen (March 5th, 2021) vaccines, and the Novavax Inc vaccine is also under consideration [@url:https://www.canada.ca/en/health-canada/services/drugs-health-products/covid19-industry/drugs-vaccines-treatments/authorization/applications.html#wb-auto-4].
While Canada initially projected that by the end of September 2021 a vaccine would be available for all Canadian adults, they now predict that it may be possible earlier as more vaccines have been approved and become available [@url:https://www.cbc.ca/news/politics/trudeau-possible-vaccination-campaign-ends-sooner-1.5934994].

In the U.S., vaccines are required to have demonstrated safety and efficacy in phase III trials before manufacturers apply for an emergency use authorization (EUA) from the FDA.
If an EUA is granted, an additional evaluation of the safety and efficacy of the vaccines is conducted by the CDC’s Advisory Committee on Immunization Practices (ACIP) who also provide guidance on vaccine prioritization.
On December 1st, 2020, ACIP provided an interim phase 1a recommendation that healthcare workers and long-term care facility residents should be the first to be offered any vaccine approved [@doi:10.15585/mmwr.mm6949e1].
This was shortly followed by an EUA on December 11th, 2020 for the use of the Pfizer-BioNTech COVID vaccine [@doi:10.15585/mmwr.mm6950e2], which was distributed and administered to the first healthcare workers on December 14th, 2020 [@{url:https://abcnews.go.com/US/us-administer-1st-doses-pfizer-coronavirus-vaccine/story?id=74703018}].
Shortly thereafter, an EUA for the Moderna vaccine was issue on December 18th, 2020 [@doi:10.15585/mmwr.mm695152e1].
On December 20th, 2020, ACIP updated their initial recommendations to suggest that vaccinations should be offered to people aged 75 years old and older and to non-healthcare frontline workers in phase 1b [@doi:10.15585/mmwr.mm695152e2].
On the same date, it was recommended that phase 1c should include people aged 65-74 years old, individuals between the ages of 16-74 years old at high-risk due to health conditions, and essential workers ineligible in phase 1b [@doi:10.15585/mmwr.mm695152e2].
On the following day, December 21st, 2020, the first Moderna vaccines used outside of clinical trials were administered to American healthcare workers, which was the same day that President-elect Biden and Dr. Biden received their first doses of the Pfizer-BioNTech vaccine live on television to instill confidence in the approval and vaccination process [@url:https://www.cnn.com/2020/12/21/health/us-coronavirus-monday/index.html].

On February 27th, 2020, the FDA issued an EUA for the Janssen COVID-19 Vaccine [@url:https://www.fda.gov/emergency-preparedness-and-response/coronavirus-disease-2019-covid-19/janssen-covid-19-vaccine].
This was followed by an update on recommendations by ACIP for the use of the Janssen COVID-19 vaccine for those over 18 years old [@doi:10.15585/mmwr.mm7009e4].
The Janssen vaccine was first distributed to healthcare facilities on March 1st, 2021.
On March 12, 2021, the WHO added the Janssen vaccine to the list of safe and effective emergency tools for COVID-19 [@url:https://www.who.int/news/item/12-03-2021-who-adds-janssen-vaccine-to-list-of-safe-and-effective-emergency-tools-against-covid-19].
While the CDC’s ACIP can provide recommendations, it is up to the public health authorities of each state, territory, and tribe to interpret the guidance and determine who will be vaccinated first [@url:https://www.hhs.gov/coronavirus/covid-19-vaccines/distribution/index.html].
Prior to distribution of the Janssen vaccine, over 103 million doses of the Moderna and Pfizer-BioNTech vaccines were delivered across the U.S., with almost 79 million doses administered.
Of the total population, 15.6% have received at least one dose and 7.9% have received a second dose of either the Moderna (~38.3 million) or the Pfizer-BioNTech (~40.2 million) vaccines by February 28th, 2021 [@url:https://covid.cdc.gov/covid-data-tracker/#vaccinations].
President Biden’s administration has predicted that by the end of May 2021 there may be enough vaccine supply available for all adults in the U.S. [@{url:https://www.cnn.com/2021/03/02/politics/biden-merck-johnson--johnson-vaccine/index.html}; @url:https://www.bbc.com/news/world-us-canada-55721437].
However, vaccine production, approval, and distribution was not straightforward in the U.S., as information was initially sparse and the rollout of vaccines was complicated by poor planning and leadership due to political activities prior to the change of administration in January 2021 [@{url:https://ssrn.com/abstract=3770368}].
These political complications highlight the importance of the transparent vaccine approval process conducted by the FDA [@doi:10.1001/jama.2021.1961].

Outside the U.S., the Moderna and Pfizer-BioNTech vaccines have been administered in 29 and 69 other countries, respectively, mainly in Europe and North America [@url:https://www.nytimes.com/interactive/2021/world/covid-vaccinations-tracker.html].
The Janssen vaccine has so far only been administered in South Africa and the U.S. [@url:https://www.nytimes.com/interactive/2021/world/covid-vaccinations-tracker.html; @url:https://www.biopharma-reporter.com/Article/2021/02/18/South-Africa-starts-administering-Janssen-COVID-19-vaccine-to-health-workers], but it has also been approved in Bahrain, the European Union (E.U.), Iceland, Liechtenstein, and Norway [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html].
On March 11th, 2021, Johnson & Johnson received approval from the European Medicines Agency (EMA) for conditional marketing authorization of their vaccine [@url:https://www.ema.europa.eu/en/news/ema-receives-application-conditional-marketing-authorisation-covid-19-vaccine-janssen].
Notably, on March 2nd, 2021, rivals Johnson & Johnson and Merck announced that they entered an agreement to increase production of the Janssen vaccine to meet global demand [@url:https://www.washingtonpost.com/health/2021/03/02/merck-johnson-and-johnson-covid-vaccine-partnership/].

The U.K. was the first country to approve use of the Pfizer-BioNTech vaccine on December 2nd, 2020 [@doi:10.1038/d41586-020-03441-8], and it was later approved by EMA on December 21st, 2020 [@url:https://www.ema.europa.eu/en/news/ema-recommends-first-covid-19-vaccine-authorisation-eu].
The U.K. was also the first to administer the Pfizer-BioNTech vaccine, making it the first COVID-19 vaccine supported by phase III data to be administered outside of clinical trials on December 8th, 2020.
The Oxford-AstraZeneca vaccine, was approved by the Medicines and Healthcare Products Regulatory Agency (MHRA) in the U.K. and by EMA in the E.U. on December 30th (2020) [@url:https://www.gov.uk/government/publications/regulatory-approval-of-covid-19-vaccine-astrazeneca] and January 29th (2021) [@url:https://www.ema.europa.eu/en/news/ema-recommends-covid-19-vaccine-astrazeneca-authorisation-eu] respectively.
The Oxford-AstraZeneca vaccine was first administered in the UK on January 4th, 2021 [@url:https://www.bbc.com/news/uk-55525542], and it is now being used in 53 countries in total, including Brazil, India, Pakistan, Mexico, and spanning most of Europe [@url:https://www.nytimes.com/interactive/2021/world/covid-vaccinations-tracker.html].
The Moderna vaccine was authorized for use in the E.U. by EMA on January 6th, 2021 [@url:https://www.ema.europa.eu/en/medicines/human/EPAR/covid-19-vaccine-moderna#authorisation-details-section] and in the U.K. by MHRA on January 8th, 2021 [@url:https://www.gov.uk/government/publications/regulatory-approval-of-covid-19-vaccine-moderna].
As of March 5th, 2021, 22 million people in the U.K. had received at least one vaccine dose [@url:https://ourworldindata.org/covid-vaccinations].

While the Pfizer-BioNTech vaccine was the first to be distributed following phase III clinical trials, the first COVID-19 vaccine to be widely administered to people prior to the completion of phase III clinical trials was Sputnik V.
Sputnik V was administered to as many as 1.5 million Russians by early January [@url:https://www.brusselstimes.com/news-contents/world/149039/1-5-million-people-have-received-sputnik-v-vaccine-russia-says-russian-direct-investment-fund-mikhail-murashko/] due to the establishment of mass vaccination clinics in December 2020, prior to which only approximately 100,000 Russians had already been vaccinated [@url:https://www.bloomberg.com/news/articles/2020-12-02/within-hours-of-u-k-putin-orders-start-of-mass-covid-19-shots;@url:https://www.forbes.com/sites/jamesrodgerseurope/2020/12/05/facing-record-covid-19-case-rise-russia-rolls-out-sputnik-v-vaccine/?sh=50650e4e62e1].
Doses of Sputnik V have also been distributed to other parts of Europe, such as Belarus, Bosnia-Herzegovina, Hungary, San Marino, Serbia, and Slovakia [@url:https://www.euronews.com/2021/02/12/hungary-to-begin-using-russia-s-sputnik-v-vaccine-today; @url:https://www.euronews.com/2021/02/24/san-marino-buys-russia-s-sputnik-v-after-eu-vaccine-delivery-delays; @url:https://www.themoscowtimes.com/2020/12/29/belarus-starts-coronavirus-vaccination-with-sputnik-v-a72512], with the Czech Republic and Austria also having expressed interest in its procurement [@url:https://www.cnbc.com/2021/03/02/russias-sputnik-vaccine-is-luring-eastern-europe-worrying-the-eu.html].
Hungary was the first E.U. member country to approve and distribute Sputnik V outside of Russia [@url:https://www.cnbc.com/2021/03/02/russias-sputnik-vaccine-is-luring-eastern-europe-worrying-the-eu.html], despite the EMA stating that they had neither approved nor received a request for approval of Sputnik V [@url:https://www.ema.europa.eu/en/news/clarification-sputnik-v-vaccine-eu-approval-process].
Hungary is also in talks with China to procure the Sinopharm vaccines, which have been approved by Hungarian health authorities but also have not received approval by EMA in the E.U. [@url:https://www.cnbc.com/2021/03/02/russias-sputnik-vaccine-is-luring-eastern-europe-worrying-the-eu.html].
In Latin America, production facilities in both Brazil and Argentina will allow for increased production capacity of Sputnik V and doses have been distributed to Mexico, Argentina, Bolivia, Nicaragua, Paraguay, and Venezuela [@url:https://fortune.com/2021/02/08/international-sputnik-russia-demand/].
Guinea was the first African nation to administer Sputnik V in December 2020, and the Central African Republic, Zimbabwe, and the Ivory Coast have all registered their interest in purchasing doses of the vaccine [@url:https://fortune.com/2021/02/08/international-sputnik-russia-demand/].
In the Middle East, Iran has received its first doses of Sputnik V and the United Arab Emirates is conducting phase III trials [@url:https://fortune.com/2021/02/08/international-sputnik-russia-demand/].
In Asia, while China’s vaccine candidates are favored, the Philippines, Nepal, and Uzbekistan have sought Sputnik V doses [@url:https://rdif.ru/Eng_fullNews/5858/; @url:https://rdif.ru/Eng_fullNews/5858/].
In total, the RDIF claims to have received orders totalling 1.2 billion doses by over 50 countries worldwide [@url:https://rdif.ru/Eng_fullNews/5858/] and at least 18 countries are currently administering Sputnik V around the globe [@url:https://www.nytimes.com/interactive/2021/world/covid-vaccinations-tracker.html].
Sputnik V has been an attractive vaccine for many countries due to its relatively low price, high efficacy, and its favorable storage conditions.
For some countries, Russia and China have also been more palatable politically than vaccine suppliers in the West [@url:https://fortune.com/2021/02/08/international-sputnik-russia-demand/; @url:https://www.politico.com/news/2021/02/25/global-vaccine-public-relations-war-471665].
For others, the delays in the distribution of the other, more-favored candidates has been a motivating factor for pursuing the Sputnik V and Chinese alternatives [@url:https://www.politico.com/news/2021/02/25/global-vaccine-public-relations-war-471665; @url:https://www.euronews.com/2021/02/24/san-marino-buys-russia-s-sputnik-v-after-eu-vaccine-delivery-delays].
Additionally, Germany has stated that if Sputnik V were approved by EMA, it would be considered by the E.U. [@url:https://www.france24.com/en/live-news/20210203-germany-moves-to-bring-russian-vaccine-into-eu-orbit].
Russia is developing other vaccine candidates and has approved a third vaccine, CoviVac, which is an inactivated vaccine produced by the Chumakov Centre in Moscow, despite the fact the clinical trials have yet to begin [@url:https://www.reuters.com/article/us-health-coronavirus-russia-vaccine-idUSKBN2AK07H].

In Asia, China and India are the main COVID-19 vaccination developers and providers.
In India, the Covaxin vaccine produced by Bharat Biotech received emergency authorization on January 3rd, 2021, despite the lack of phase III data until March 3rd [@url:https://www.bharatbiotech.com/images/press/covaxin-phase3-efficacy-results.pdf].
Following the release of the phase III data indicating 81% efficacy, Zimbabwe authorized the use of Covaxin [@url:http://www.xinhuanet.com/english/2021-03/04/c_139783893.htm].
In February, 2021, Bharat Biotech received approval from Indian officials to commence a phase I study of an intranasal chimpanzee-adenovirus (ChAd) vectored SARS-CoV-2-S vaccine called BBV154 [@url:https://www.bharatbiotech.com/intranasal-vaccine.html].
Notably, Novavax has signed an agreement with the Serum Institute of India allowing them to produce up to 2 billion doses a year [@url:https://www.reuters.com/article/health-coronavirus-novavax-idUSKBN2661PI].
Novavax has also signed agreements with the U.K., Canada, Australia, and South Korea [@url:https://ir.novavax.com/news-releases/news-release-details/novavax-announces-expanded-collaboration-and-license-agreement] and has projected that they will supply 1.1 billion doses to COVAX who will distribute the vaccines to countries with disadvantaged access to vaccine supplies [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html].
India has vaccinated approximately 24 million people [@url:https://www.bloomberg.com/graphics/covid-vaccine-tracker-global-distribution/].
This has been achieved by mainly using the AstraZeneca-University of Oxford vaccine, known as Covishield in India, which is also produced by the Serum Institute of India, and using India’s own Covaxin vaccine [@url:https://www.bbc.com/news/world-asia-india-55748124].
India has also shipped approximately 58 million COVID-19 vaccines to 66 countries [@url:https://www.mea.gov.in/vaccine-supply.htm]
Considering India produces approximately 60% of the world's vaccines prior to the pandemic, it is no surprise that several other vaccine candidates are under development.
These include ZyCov-Di, a DNA vaccine produced by Zydus Cadila, HGCO19, India’s first mRNA vaccine produced by Genova and HDT Biotech Corporation (of the U.S.), and the Bio E subunit vaccine produced by Biological E in collaboration with U.S.-based Dynavax and the Baylor College of Medicine [@url:https://www.bbc.com/news/world-asia-india-55748124].

In China, the Sinopharm-Beijing Institute vaccine, the Sinopharm-Wuhan Institute of Biological Products vaccine, the Sinovac Biotech (CoronaVac) vaccine, and CanSino Biologics vaccine are the main vaccines being distributed.
The Sinopharm-Beijing vaccine has been distributed to at least 16 countries.
This vaccine is currently approved for use in Bahrain, China, and the United Arab Emirates, but has been granted emergency use in Argentina, Cambodia, Egypt, Guyana, Hungary, Iran, Iraq, Jordan, Nepal, Pakistan, Peru, Venezuela, and Zimbabwe, with limited use in both Serbia and the Seychelles [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#sinopharm].<!--To do: clean citation-->
The Sinovac vaccine, CoronaVac, has been approved for use in China, and has been granted emergency use in Azerbaijan, Brazil, Cambodia, Chile, Colombia, Ecuador, Hong Kong, Indonesia, Laos, Malaysia, Mexico, Philippines, Thailand, Turkey, Ukraine, and Uruguay [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#sinovac].<!--To Do: clean citation-->
Sinovac has reported that their platform now has the capacity to provide up to a billion doses [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#sinovac].<!--To Do: clean citation-->
Indeed, Sinovac and Sinopharm have estimated that they will be able to produce 2 billion doses by the end of 2021, and they have been able to distribute vaccines as aid to the Philippines and Pakistan [@url:https://www.nytimes.com/2021/01/25/business/china-covid-19-vaccine-backlash.html].
In contrast, the Sinopharm-Wuhan vaccine, which has been approved for use in China since February 25th, 2021, has been distributed almost exclusively within China, with limited supplies distributed to the United Arab Emirates [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#wuhan].<!--To Do: clean citation-->
On the same date, the CanSino vaccine was approved for use in China and has been granted emergency use in Mexico and Pakistan, which were two participating countries in the CanSino phase III trials [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#cansino].
However, the vaccine approval and distribution processes in China have come under increased scrutiny from other nations.
China was criticized for administering vaccines to thousands of government officials and state-owned businesses in September 2020, prior to the completion of phase III clinical trials [@doi:10.1001/jama.2021.1961].
The behavior of Chinese officials has also come into question due to misinformation campaigns questioning the safety of Western vaccine candidates such as Moderna and Pfizer-BioNTech in a way that is intended to highlight the benefits of their own vaccine candidates [@url:https://www.nytimes.com/2021/01/25/business/china-covid-19-vaccine-backlash.html].
Furthermore, delays in vaccine distribution have also caused issues, particularly in Turkey where 10 million doses of Sinovac were due to arrive by December 2020, but instead only 3 million were delivered in early January [@url:https://www.nytimes.com/2021/01/25/business/china-covid-19-vaccine-backlash.html].
Similar delays and shortages of doses promised have been reported by officials in the Philippines, Egypt, Morocco, and the United Arab Emirates [@url:https://abcnews.go.com/Health/wireStory/philippines-receive-covid-19-vaccine-delays-76163594; @url:https://www.wsj.com/articles/chinas-covid-19-vaccine-makers-struggle-to-meet-demand-11612958560].
This will be concerning to China who have vaccine contracts for millions of doses with Indonesia (>100 million), Brazil (100 million), Chile (60 million), Turkey (50 million), Egypt (40 million) and many others [@url:https://www.wsj.com/articles/chinas-covid-19-vaccine-makers-struggle-to-meet-demand-11612958560].

Globally, North America currently leads the world vaccination rates (13.8 per 100 people) followed by Europe (8.2 per 100), South America (3.1 per 100), Asia (1.9 per 100), Africa (0.3 per 100), and Oceania (0.1 per 100) are trailing behind [@url:https://www.nytimes.com/interactive/2021/world/covid-vaccinations-tracker.html].
Considering the wealthy nations of North America and Europe have secured most of the limited COVID-19 vaccine stocks [@url:https://www.nytimes.com/2020/12/15/us/coronavirus-vaccine-doses-reserved.html], it is likely that low- and middle-income countries will face further competition with Western countries for vaccine availability.
While South Africa and Zimbabwe have their own vaccination programs, many other African nations will be reliant on the COVID-19 Vaccines Global Access (COVAX) Facility, who have promised 600 million doses to the continent [@url:https://www.bbc.com/news/56100076].
COVAX is a multilateral initiative as part of the Access to COVID-19 Tools (ACT) Accelerator coordinated by the WHO, Gavi The Vaccine Alliance, and the Coalition for Epidemic Preparedness Innovations (CEPI), the latter two of which are supported by the Bill and Melinda Gates Foundation.
Their intention is to accelerate the development of COVID-19 vaccines, diagnostics, and therapeutics and to ensure the equitable distribution of vaccines to low- and middle-income countries [@doi:10.1111/1468-0009.12503; @url:https://www.who.int/publications/m/item/fair-allocation-mechanism-for-covid-19-vaccines-through-the-covax-facility].
COVAX invested in several vaccine programs to ensure they would have access to successful vaccine candidates [@doi:10.1126/science.369.6503.489].
The COVAX plan ensured that all participating countries would be allocated vaccines in proportion to their population sizes.
Once each country has received vaccine doses to account for 20% of their population, the country’s risk profile will determine its place in subsequent phases of vaccine distribution.
However, several limitations of this framework exist, including that the COVAX scheme seems to go against the WHO’s own ethical principles of human well-being, equal respect, and global equity, and that other frameworks might have been more suitable, as is discussed elsewhere [@doi:10.1136/bmj.m4853].
Furthermore, COVAX is supposed to allow poorer countries access to affordable vaccines, but the vaccines are driven by publicly traded companies that are required to make a profit [@url:https://www.nytimes.com/2020/12/25/business/coronavirus-vaccines-global-economy.html].
In any case, COVAX provides access to COVID-19 vaccines that may otherwise have been difficult for some countries to obtain.
COVAX aims to distribute 2 billion vaccine doses globally by the end of 2021 [@url:https://www.who.int/initiatives/act-accelerator/covax].
COVAX may also receive additional donations of doses from Western nations who purchased surplus vaccines in the race to vaccinate their populations, which will be a welcome boost to the vaccination programs of low- and middle-income countries [@doi:10.1126/science.abh4476].
As of March, 2021, 9 African countries have received vaccines and at least 11 other nations have begun vaccinations via COVAX, aid from other countries, or their own agreements with producers [@url:https://www.bbc.com/news/56100076; @url:https://www.who.int/news/item/01-03-2021-first-covid-19-covax-vaccine-doses-administered-in-africa].
However, much further progress is required when only 0.3 per 100 people have been vaccinated in Africa [@url:https://www.nytimes.com/interactive/2021/world/covid-vaccinations-tracker.html].

### Discussion

Additionally, major advances in vaccines using mRNA and adenoviruses that have led to three vaccines becoming available or close to becoming available in late 2020 (Figure @fig:therapeutics). <!-- Update depending on where this goes and specify countries etc.-->

Though some concerns remain about the duration of sustained immunity for convalescents, vaccine development efforts are ongoing and show initial promising results.
<!-- TO DO: Needs to be updated with vaccine info about duration, this is changing rapidly so leaving for now-->
The Moderna trial, for example, reported that the neutralizing activity in participants who received two doses of the vaccine was similar to that observed in convalescent plasma.

One of the two mRNA vaccines, Pfizer and BioNTech's BNT162b2, has been issued an EUA for patients as young as 16 [@url:https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-submit-emergency-use-authorization], while ModernaTX has begun a clinical trial to assess its mRNA vaccine in adolescents ages 12 to 18 [@url:https://investors.modernatx.com/news-releases/news-release-details/moderna-announces-first-participants-dosed-phase-23-study-covid].


## Dietary Supplements and Nutraceuticals Under Investigation for COVID-19 Prevention and Treatment

### Abstract

Coronavirus disease 2019 (COVID-19) has caused global disruption and a significant loss of life.
Existing treatments that can be repurposed as prophylactic and therapeutic agents could reduce the pandemic's devastation.
Emerging evidence of potential applications in other therapeutic contexts has led to the investigation of dietary supplements and nutraceuticals for COVID-19.
Such products include vitamin C, vitamin D, omega 3 polyunsaturated fatty acids, probiotics, and zinc, all of which are currently under clinical investigation.
In this review, we critically appraise the evidence surrounding dietary supplements and nutraceuticals for the prophylaxis and treatment of COVID-19.
Overall, further study is required before evidence-based recommendations can be formulated, but nutritional status plays a significant role in patient outcomes, and these products could help alleviate deficiencies. 
For example, evidence indicates that vitamin D deficiency may be associated with greater incidence of infection and severity of COVID-19, suggesting that vitamin D supplementation may hold prophylactic or therapeutic value.
A growing number of scientific organizations are now considering recommending vitamin D supplementation to those at high risk of COVID-19.
Because research in vitamin D and other nutraceuticals and supplements is preliminary, here we evaluate the extent to which these nutraceutical and dietary supplements hold potential in the COVID-19 crisis.

### Importance

Sales of dietary supplements and nutraceuticals have increased during the pandemic due to their perceived "immune-boosting" effects.
However, little is known about the efficacy of these dietary supplements and nutraceuticals against the novel coronavirus (SARS-CoV-2) or the disease it causes, COVID-19.
This review provides a critical overview of the potential prophylactic and therapeutic value of various dietary supplements and nutraceuticals from the evidence available to date.
These include vitamin C, vitamin D, and zinc, which are often perceived by the public as treating respiratory infections or supporting immune health.
Consumers need to be aware of misinformation and false promises surrounding some supplements, which may be subject to limited regulation by authorities.
However, considerably more research is required to determine whether dietary supplements and nutraceuticals exhibit prophylactic and therapeutic value against SARS-CoV-2 infection and COVID-19.
This review provides perspective on which nutraceuticals and supplements are involved in biological processes that are relevant to recovery from or prevention of COVID-19.

### Introduction

The year 2020 saw scientists and the medical community scrambling to repurpose or discover novel host-directed therapies against the coronavirus disease 2019 (COVID-19) pandemic caused by the spread of the novel _Severe acute respiratory syndrome-related coronavirus 2_ (SARS-CoV-2).
This rapid effort led to the identification of some promising pharmaceutical therapies for hospitalized patients, such as remdesivir and dexamethasone.
Furthermore, most societies have adopted non-pharmacological preventative measures such as utilizing public health strategies that reduce the transmission of SARS-CoV-2.
However, during this time, many individuals sought additional protections via the consumption of various dietary supplements and nutraceuticals that they believed to confer beneficial effects.
While a patient's nutritional status does seem to play a role in COVID-19 susceptibility and outcomes [@doi:10.3390/nu12051466; @doi:10.1101/2020.10.19.20214395; @doi:10.1093/advances/nmaa125; @doi:10.1016/j.ijid.2020.08.018; @doi:10.3390/nu12041181], the beginning of the pandemic saw sales of vitamins and other supplements soar despite a lack of any evidence supporting their use against COVID-19.
In the United States, for example, dietary supplement and nutraceutical sales have shown modest annual growth in recent years (approximately 5%, or a $345 million increase in 2019), but during the six-week period preceding April 5, 2020, they increased by 44% ($435 million) relative to the same period in 2019 [@url:https://www.nutritionaloutlook.com/view/peak-dietary-supplement-sales-leveling-during-covid-19-pandemic-growth-still-remains-strong].
While growth subsequently leveled off, sales continued to boom, with a further 16% ($151 million) increase during the six weeks preceding May 17, 2020 relative to 2019 [@url:https://www.nutritionaloutlook.com/view/peak-dietary-supplement-sales-leveling-during-covid-19-pandemic-growth-still-remains-strong].
In France, New Zealand, India, and China, similar trends in sales were reported [@doi:10.3390/nu12061699; @url:https://www.nutraingredients-asia.com/Article/2020/07/06/Lockdown-impact-Grocery-stores-bolstered-NZ-supplements-sales-as-pharmacies-slumped; @url:https://www.nutritioninsight.com/news/covid-19-temporarily-bolsters-european-interest-in-supplements.html; @url:https://www.nutraingredients.com/Article/2020/07/21/India-s-immune-health-surge-Nation-leads-APAC-in-number-of-new-product-launches-new-data].
The increase in sales was driven by a consumer perception that dietary supplements and nutraceuticals would protect consumers from infection and/or mitigate the impact of infection due to the various "immune-boosting" claims of these products [@doi:10.1016/j.tifs.2020.09.001; @url:https://kerry.com/insights/kerrydigest/2020/5-food-and-beverage-trends-in-europe-during-covid-19].

Due to the significant interest from the general public in dietary additives, whether and to what extent nutraceuticals or dietary supplements can provide any prophylactic or therapeutic benefit remains a topic of interest for the scientific community.
Nutraceuticals and dietary supplements are related but distinct non-pharmaceutical products.
Nutraceuticals are classified as supplements with health benefits beyond their basic nutritional value [@doi:10.1080/10408390902841529; @doi:10.1038/nrcardio.2016.103].
The key difference between a dietary supplement and a nutraceutical is that nutraceuticals should not only supplement the diet, but also aid in the prophylaxis and/or treatment of a disorder or disease [@doi:10.1208/ps050325].
However, dietary supplements and nutraceuticals, unlike pharmaceuticals, are not subject to the same regulatory protocols that protect consumers of medicines.
Indeed, nutraceuticals do not entirely fall under the responsibility of the Food and Drug Administration (FDA), but they are monitored as dietary supplements according to the Dietary Supplement, Health and Education Act 1994 (DSHEA) [@url:https://ods.od.nih.gov/About/DSHEA_Wording.aspx] and the Food and Drug Administration Modernization Act 1997 (FDAMA) [@url:https://www.fda.gov/regulatory-information/selected-amendments-fdc-act/food-and-drug-administration-modernization-act-fdama-1997].
Due to increases in sales of dietary supplements and nutraceuticals, in 1996 the FDA established the Office of Dietary Supplement Programs (ODSP) to increase surveillance.
Novel products or nutraceuticals must now submit a new dietary ingredient notification to the ODSP for review.
There are significant concerns that these legislations do not adequately protect the consumer as they ascribe responsibility to the manufacturers to ensure the safety of the product before manufacturing or marketing [@doi:10.1080/17512433.2018.1464911].
Manufacturers are not required to register or even seek approval from the FDA to produce or sell food supplements or nutraceuticals.
Health or nutrient content claims for labeling purposes are approved based on an authoritative statement from the Academy of Sciences or relevant federal authorities once the FDA has been notified and on the basis that the information is known to be true and not deceptive [@doi:10.1080/17512433.2018.1464911].
Therefore, there is often a gap between perceptions by the American public about a nutraceutical or dietary supplement and the actual clinical evidence surrounding its effects.

Despite differences in regulations, similar challenges exist outside of the United States.
In Europe, where the safety of supplements is monitored by the European Union (EU) under Directive 2002/46/EC [@url:https://eur-lex.europa.eu/legal-content/EN/ALL/?uri=celex%3A32002L0046].
However, nutraceuticals are not directly mentioned.
Consequently, nutraceuticals can be generally described as either a medicinal product under Directive 2004/27/EC [@url:https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX:32004L0027] or as a 'foodstuff' under Directive 2002/46/EC of the European council.
In order to synchronize the various existing legislations, Regulation EC 1924/2006 on nutrition and health claims was put into effect to assure customers of safety and efficacy of products and to deliver understandable information to consumers.
However, specific legislation for nutraceuticals is still elusive.
Health claims are permitted on a product label only following compliance and authorization according to the European Food Safety Authority (EFSA) guidelines on nutrition and health claims [@url:https://ec.europa.eu/food/safety/labelling_nutrition/claims/register/public].
EFSA does not currently distinguish between food supplements and nutraceuticals for health claim applications of new products, as claim authorization is dependent on the availability of clinical data in order to substantiate efficacy [@doi:10.1111/bcp.13496].
These guidelines seem to provide more protection to consumers than the FDA regulations but potentially at the cost of innovation in the sector [@doi:10.1080/09637486.2016.1212816].
The situation becomes even more complicated when comparing regulations at a global level, as countries such as China and India have existing regulatory frameworks for traditional medicines and phytomedicines not commonly consumed in Western society [@doi:10.3390/nu10010041].
Currently, there is debate among scientists and regulatory authorities surrounding the development of a widespread regulatory framework to deal with the challenges of safety and health claim substantiation for nutraceuticals [@doi:10.1111/bcp.13496; @doi:10.1080/17512433.2018.1464911], as these products do not necessarily follow the same rigorous clinical trial frameworks used to approve the use of pharmaceuticals.
Such regulatory disparities have been highlighted by the pandemic, as many individuals and companies have attempted to profit from the vulnerabilities of others by overstating claims in relation to the treatment of COVID-19 using supplements and nutraceuticals.
The FDA has written several letters to prevent companies marketing or selling products based on false hyperbolic promises about preventing SARS-CoV-2 infection or treating COVID-19 [@url:https://www.fda.gov/inspections-compliance-enforcement-and-criminal-investigations/warning-letters/noetic-nutraceuticals-607572-05152020; @url:https://beta.regulations.gov/document/FDA-2020-S-0023-0068; @url:https://www.fda.gov/inspections-compliance-enforcement-and-criminal-investigations/warning-letters/spartan-enterprises-inc-dba-watershed-wellness-center-610876-10302020].
These letters came in response to efforts to market nutraceutical prophylactics against COVID-19, some of which charged the consumer as much as $23,000 [@url:https://www.ftc.gov/news-events/press-releases/2020/07/ftc-sues-california-marketer-23000-covid-19-treatment-plan].
There have even been some incidents highlighted in the media because of their potentially life threatening consequences; for example, the use of oleandrin was touted as a potential "cure" by individuals close to the former President of the United States despite its high toxicity [@url:https://cen.acs.org/biological-chemistry/natural-products/oleandrin-compound-touted-possible-COVID/98/web/2020/08].
Thus, heterogeneous and at times relaxed regulatory standards have permitted high-profile cases of the sale of nutraceuticals and dietary supplements that are purported to provide protection against COVID-19, despite a lack of research into these compounds.

Notwithstanding the issues of poor safety, efficacy, and regulatory oversight, some dietary supplements and nutraceuticals have exhibited therapeutic and prophylactic potential.
Some have been linked with reduced immunopathology, antiviral and anti-inflammatory activities, or even the prevention of acute respiratory distress syndrome (ARDS) [@doi:10/ggkd3b; @doi:10.1016/j.tifs.2020.09.001; @doi:10.3390/nu12061718].
A host of potential candidates have been highlighted in the literature that target various aspects of the COVID-19 viral pathology, while others are thought to prime the host immune system.
These candidates include vitamins and minerals along with extracts and omega-3 polyunsaturated fatty acids (n-3 PUFA) [@doi:10.1002/jmv.25707].
_In vitro_ and _in vivo_ studies suggest that nutraceuticals containing phycocyanobilin, N-acetylcysteine, glucosamine, selenium or phase 2 inductive nutraceuticals (e.g. ferulic acid, lipoic acid, or sulforaphane) can prevent or modulate RNA virus infections via amplification of the signaling activity of mitochondrial antiviral-signaling protein (MAVS) and activation of Toll-like receptor 7 [@doi:10.1016/j.pcad.2020.02.007].
Phase 2 inductive molecules used in the production of nutraceuticals are known to activate nuclear factor erythroid 2–related factor 2 (Nrf2), which is a protein regulator of antioxidant enzymes that leads to the induction of several antioxidant enzymes, such as gamma-glutamylcysteine synthetase.
While promising, further animal and human studies are required to assess the therapeutic potential of these various nutrients and nutraceuticals against COVID-19.
For the purpose of this review, we have highlighted some of the main dietary supplements and nutraceuticals that are currently under investigation for their potential prophylactic and therapeutic applications.
These include n-3 PUFA, zinc, vitamins C and D, and probiotics.

### n-3 PUFA

One category of supplements that has been explored for beneficial effects against various viral infections are the n-3 PUFAs [@doi:10.1002/jmv.25707], commonly referred to as omega-3 fatty acids, which include eicosapentaenoic acid (EPA) and docosahexaenoic acid (DHA).
EPA and DHA intake can come from a diet high in fish or through dietary supplementation with fish oils or purified oils [@doi:10.1039/c9fo01742a].
Other, more sustainable sources of EPA and DHA include algae [@doi:10.3389/fphys.2020.00752; @doi:10.3390/fermentation4020046], which can also be exploited for their rich abundance of other bioactive compounds such as angiotensin converting enzyme inhibitor peptides and antiviral agents including phycobiliproteins, sulfated polysaccharides, and calcium-spirulan [@doi:10.1016/j.nut.2020.111089].
n-3 PUFAs have been investigated for many years for their therapeutic potential [@doi:10.1007/bf03257413].
Supplementation with fish oils is generally well tolerated [@doi:10.1007/bf03257413], and intake of n-3 PUFAs through dietary sources or supplementation is specifically encouraged for vulnerable groups such as pregnant and lactating women [@url:https://www.acog.org/clinical/clinical-guidance/practice-advisory/articles/2017/01/update-on-seafood-consumption-during-pregnancy; @pmc:PMC2621042].
As a result, these well-established compounds have drawn significant interest for their potential immune effects and therapeutic potential.

Particular interest has arisen in n-3 PUFAs as potential therapeutics against diseases associated with inflammation.
n-3 PUFAs have been found to modulate inflammation by influencing processes such as leukocyte chemotaxis, adhesion molecule expression, and the production of eicosanoids [@doi:10.1111/j.1365-2125.2012.04374.x; @doi:10.1016/j.bbalip.2014.08.010].
This and other evidence indicates that n-3 PUFAs may have the capacity to modulate the adaptive immune response [@doi:10.1111/j.1365-2125.2012.04374.x; @doi:10.1039/c9fo01742a; @doi:10.1038/nrcardio.2016.103]; for example, they have been found to influence antigen presentation and the production of CD4(+) Th1 cells, among other relevant effects [@doi:10.1016/j.ejphar.2015.03.100].
Certainly, preliminary evidence from banked blood samples from 100 COVID-19 patients suggests that patients with a higher omega-3 index, a measure of the amount of EPA and DHA in red blood cells, had a lower risk of death due to COVID-19 [@doi:10.1016/j.plefa.2021.102250].
Interest has also arisen as to whether nutritional status related to n-3 PUFAs can also affect inflammation associated with severe disease, such as ARDS or sepsis [@doi:10.1007/s11745-016-4129-x; @doi:10.1016/j.nut.2019.02.016].
ARDS and sepsis hold particular concern in the treatment of severe COVID-19; an analysis of 82 deceased COVID-19 patients in Wuhan during January to February 2020 reported that respiratory failure (associated with ARDS) was the cause of death in 69.5% of cases, and sepsis or multi-organ failure accounted for 28.0% of deaths [@doi:10.1371/journal.pone.0235458].
Research in ARDS prior to current pandemic suggests that n-3 PUFAs may hold some therapeutic potential.
One study randomized 16 consecutive ARDS patients to receive either a fish oil-enriched lipid emulsion or a control lipid emulsion (comprised of 100% long-chain triglycerides) under a double-blinded design [@doi:10.1186/1743-7075-8-22].
They reported a statistically significant reduction in leukotriene B4 levels in the group receiving the fish oil-enriched emulsion, suggesting that the fish oil supplementation may have reduced inflammation.
However, they also reported that most of their tests were not statistically significant, and therefore it seems that additional research using larger sample sizes is required.
A recent meta-analysis of 10 randomized controlled trials (RCTs) examining the effects of n-3 PUFAs on ARDS patients did not find evidence of any effect on mortality, although the effect on secondary outcomes could not be determined due to a low quality of evidence [@doi:10.4187/respcare.06965].
However, another meta-analysis that examined 24 RCTs studying the effects of n-3 fatty acids on sepsis, including ARDS-induced sepsis, did find support for an effect on mortality when n-3 fatty acids were administered via enteral nutrition, although a paucity of high-quality evidence again limited conclusions [@doi:10.1186/s12937-018-0356-8].
Therefore, despite theoretical support for an immunomodulatory effect of n-3 PUFAs in COVID-19, evidence from existing RCTs is insufficient to determine whether supplementation offers an advantage in a clinical setting that would be relevant to COVID-19.

Another potential mechanism that has led to interest in n-3 PUFAs as protective against viral infections including COVID-19 is its potential as a precursor molecule for the biosynthesis of endogenous specialized proresolving mediators (SPM), such as protectins and resolvins, that actively resolve inflammation and infection [@doi:10.1016/j.immuni.2014.02.009].
SPM have exhibited beneficial effects against a variety of lung infections, including some caused by RNA viruses [@doi:10.1038/nri.2015.4; @doi:10.1002/biof.1691].
Several mechanisms for SPM have been proposed, including preventing the release of pro-inflammatory cytokines and chemokines or increasing phagocytosis of cellular debris by macrophages [@doi:10.1038/nature13479].
In influenza, SPM promote antiviral B lymphocytic activities [@doi:10.4049/jimmunol.1302795], and protectin D1 has been shown to increase survival from H1N1 viral infection in mice by affecting the viral replication machinery [@doi:10.1016/j.cell.2013.02.027].
It has thus been hypothesized that SPM could aid in the resolution of the cytokine storm and pulmonary inflammation associated with COVID-19 [@doi:10.1007/s10555-020-09889-4; @doi:10.1016/j.mehy.2020.110340].
Another theory is that some comorbidities, such as obesity, could lead to deficiencies of SPM, which could in turn be related to the occurrence of adverse outcomes for COVID-19 [@doi:10.3389/fimmu.2020.01997].
However, not all studies are in agreement that n-3 PUFAs or their resulting SPM are effective against infections [@doi:10.3945/jn.109.108027].
At a minimum, the effectiveness of n-3 PUFAs against infections would be dependent on the dosage, timing, and the specific pathogens responsible [@doi:10.1016/j.jinf.2016.10.001].
On another level, there is still the question of whether fish oils can raise the levels of SPM levels upon ingestion and in response to acute inflammation in humans [@doi:10.1194/jlr.M060392].
Currently, Karolinska University Hospital is running a trial that will measure the levels of SPM as a secondary outcome following intravenous supplementation of n-3 PUFAs in hospitalized COVID-19 patients to determine whether n-3 PUFAs provides therapeutic value [@clinicaltrials:NCT04647604; @doi:10.3389/fphys.2020.624657].
Therefore, while this mechanism provides theoretical support for a role for n-3 PUFAs against COVID-19, experimental support is still needed.

A third possible mechanism by which n-3 PUFAs could benefit COVID-19 patients arises from the fact that some COVID-19 patients, particularly those with comorbidities, are at a significant risk of thrombotic complications including arterial and venous thrombosis [@doi:10.1182/blood.2020006000; @doi:10.1007/s11239-020-02134-3].
Therefore, the use of prophylactic and therapeutic anticoagulants and antithrombotic agents is under consideration [@doi:10.1016/j.jacc.2020.04.031; @doi:10.3389/fnut.2020.583080].
Considering that there is significant evidence that n-3 fatty acids and other fish oil-derived lipids possess antithrombotic properties and anti-inflammatory properties [@doi:10.1016/j.prostaglandins.2018.09.005; @doi:10.1016/j.blre.2020.100694; @doi:10.1039/C9FO01742A], they may have therapeutic value against the prothrombotic complications of COVID-19.
In particular, concerns have been raised within the medical community about using investigational therapeutics on COVID-19 patients who are already on antiplatelet therapies due to pre-existing comorbidities because the introduction of such therapeutics could lead to issues with dosing and drug choice and/or negative drug-drug interactions [@doi:10.1016/j.jacc.2020.04.031].
In such cases, dietary sources of n-3 fatty acids or other nutraceuticals with antiplatelet activities could hold particular value for reducing the risk of thrombotic complications in patients already receiving pharmaceutical antiplatelet therapies.
A new clinical trial [@clinicaltrials:NCT04412018] is currently recruiting COVID-19 positive patients to investigate the anti-inflammatory activity of a recently developed, highly purified nutraceutical derivative of EPA known as icosapent ethyl (Vascepa^TM^) [@doi:10.1056/NEJMoa1812792].
Other randomized controlled trials that are in the preparatory stages intend to investigate the administration of EPA and other bioactive compounds to COVID-19 positive patients in order to observe whether anti-inflammatory effects or disease state improvements occur [@clinicaltrials:NCT04335032; @clinicaltrials:NCT04323228].
Finally, while there have been studies investigating the therapeutic value of n-3 fatty acids against ARDS in humans, there is still limited evidence of their effectiveness [@doi:10.3390/ijms21093104].
It should be noted that the overall lack of human studies in this area means there is limited evidence as to whether these supplements could affect COVID-19 infection.
Consequently, the clinical trials that are underway and those that have been proposed will provide valuable insight into whether the anti-inflammatory potential of n-3 PUFAs and their derivatives can be beneficial to the treatment of COVID-19.
All the same, while the evidence is not present to draw conclusions about whether n-3 PUFAs will be useful in treating COVID-19, there is likely little harm associated with a diet rich in fish oils, and interest in n-3 PUFA supplementation by the general public is unlikely to have negative effects.

### Zinc

Zinc is nutrient supplement that may exhibit some benefits against RNA viral infections.
Zinc is a trace metal obtained from dietary sources or supplementation and is important for the maintenance of immune cells involved in adaptive and innate immunity [@doi:10.1016/j.abb.2016.03.022].
Supplements can be administered orally as a tablet or as a lozenge and are available in many forms, such as zinc picolinate, zinc acetate, and zinc citrate.
Zinc is also available from dietary sources including meat, seafood, nuts, seeds, legumes, and dairy.
The role of zinc in immune function has been extensively reviewed [@doi:10.1016/j.abb.2016.03.022].
Zinc is an important signaling molecule, and zinc levels can alter host defense systems.
In inflammatory situations such as an infection, zinc can regulate leukocyte immune responses and modulate the nuclear factor kappa-light-chain-enhancer of activated B cells, thus altering cytokine production [@doi:10.4049/jimmunol.179.6.4180; @doi:10.1067/mlc.2001.118108].
In particular, zinc supplementation can increase natural killer cell levels, which are important cells for host defense against viral infections [@doi:10.1126/science.1198687; @doi:10.1016/j.abb.2016.03.022].
As a result of these immune-related functions, zinc is also under consideration for possible benefits against COVID-19.

Adequate zinc intake has been associated with reduced incidence of infection [@doi:10.1093/ajcn/85.3.837] and antiviral immunity [@doi:10.1093/advances/nmz013].
A randomized, double-blind, placebo-controlled trial that administered zinc supplementation to elderly subjects over the course of a year found that zinc supplementation decreased susceptibility to infection and that zinc deficiency was associated with increased susceptibility to infection [@doi:10.1093/ajcn/85.3.837].
Clinical trial data supports the utility of zinc to diminish the duration and severity of symptoms associated with common colds when it is provided within 24 hours of the onset of symptoms [@doi:10.1331/1544-3191.44.5.594.Hulisz; @doi:10.2174/1874306401105010051].
An observational study showed that COVID-19 patients had significantly lower zinc levels in comparison to healthy controls and that zinc-deficient COVID-19 patients (those with levels less than 80 μg/dl) tended to have more complications (70.4% vs 30.0%, _p_ = 0.009) and potentially prolonged hospital stays (7.9 vs 5.7 days, _p_ = 0.048) relative to patients who were not zinc deficient [@doi:10.1016/j.ijid.2020.09.014].
In coronaviruses specifically, _in vitro_ evidence has demonstrated that the combination of zinc (Zn^2+^) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP (a fluorescently tagged SARS-CoV-1) and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015].
Currently, there are over twenty clinical trials registered with the intention to use zinc in a preventative or therapeutic manner for COVID-19.
However, many of these trials proposed the use of zinc in conjunction with hydroxychloroquine and azithromycin [@clinicaltrials:NCT04370782; @clinicaltrials:NCT04377646; @clinicaltrials:NCT04395768; @clinicaltrials:NCT04373733], and it is not known how the lack of evidence supporting the use of hydroxychloroquine will affect investigation of zinc.
One retrospective observational study of New York University Langone hospitals in New York compared outcomes among hospitalized COVID-19 patients administered hydroxychloroquine and azithromycin with zinc sulfate (n = 411) versus hydroxychloroquine and azithromycin alone (n = 521).
Notably, zinc is the only treatment that was used in this trial that is still under consideration as a therapeutic agent due to the lack of efficacy and potential adverse events associated with hydroxychloroquine and azithromycin against COVID-19 [@doi:10.1056/NEJMoa2019014; @doi:10.1371/journal.pone.0244778; @doi:10.1016/j.cmi.2020.08.022].
While the addition of zinc sulfate did not affect the duration of hospitalization, the length of ICU stays or patient ventilation duration, univariate analyses indicated that zinc did increase the frequency of patients discharged and decreased the requirement for ventilation, referrals to the ICU, and mortality [@doi:10.1099/jmm.0.001250].
However, a smaller retrospective study at Hoboken University Medical Center New Jersey failed to find an association between zinc supplementation and survival of hospitalized patients [@doi:10.1016/j.chest.2020.06.082].
Therefore, whether zinc contributes to COVID-19 recovery remains unclear.
Other trials are now investigating zinc in conjunction with other supplements such as vitamin C or n-3 PUFA [@clinicaltrials:NCT04342728; @clinicaltrials:NCT04323228].
Though there is, overall, encouraging data for zinc supplementation against the common cold and viral infections, there is currently limited evidence to suggest zinc supplementation has any beneficial effects against the current novel COVID-19; thus, the clinical trials that are currently underway will provide vital information on the efficacious use of zinc in COVID-19 prevention and/or treatment.
However, given the limited risk and the potential association between zinc deficiency and illness, maintaining a healthy diet to ensure an adequate zinc status may be advisable for individuals seeking to reduce their likelihood of infection.

### Vitamin C

Vitamins B, C, D, and E have also been suggested as potential nutrient supplement interventions for COVID-19 [@doi:10.1002/jmv.25707; @doi:10.20944/preprints202003.0347.v1].
In particular vitamin C has been proposed as a potential therapeutic agent against COVID-19 due to its long history of use against the common cold and other respiratory infections [@doi:10.3389/fimmu.2020.574029; @doi:10.3390/nu12113286].
Vitamin C can be obtained via dietary sources such as fruits and vegetables or via supplementation.
Vitamin C plays a significant role in promoting immune function due to its effects on various immune cells.
It affects inflammation by modulating cytokine production, decreasing histamine levels, enhancing the differentiation and proliferation of T- and B-lymphocytes, increasing antibody levels, and protecting against the negative effects of reactive oxygen species, among other effects related to COVID-19 pathology [@doi:10.1155/2014/426740; @doi:10.1007/s00210-013-0880-1; @doi:10.3390/nu9111211].
Vitamin C is utilized by the body during viral infections, as evinced by lower concentrations in leukocytes and lower concentrations of urinary vitamin C.
Post-infection, these levels return to baseline ranges [@doi:10.1177/003693307301800102; @doi:10.1111/j.1749-6632.1975.tb29312.x; @doi:10/fd22sv; @doi:10.3390/nu9040339; @doi:10.1079/bjn19920004].
It has been shown that as little as 0.1 g/d of vitamin C can maintain normal plasma levels of vitamin C in healthy individuals, but higher doses of at least 1-3 g/d are required for critically ill patients in ICUs [@doi:10.3390/nu11040708].
Indeed, vitamin C deficiency appears to be common among COVID-19 patients [@doi:10.1016/j.medidd.2020.100064; @doi:10.1186/s13054-020-03249-y].
COVID-19 is also associated with the formation of microthrombi and coagulopathy [@doi:10.1172/JCI140183] that contribute to its characteristic lung pathology [@doi:10.1183/16000617.0240-2020], but these symptoms can be ameliorated by early infusions of vitamin C to inhibit endothelial surface P-selectin expression and platelet-endothelial adhesion [@doi:10.3390/antiox6030049].
Intravenous vitamin C also reduced D-dimer levels in a case study of 17 COVID-19 patients [@doi:10.1080/14787210.2020.1794819].
D-dimer levels are an important indicator of thrombus formation and breakdown and are notably elevated in COVID-19 patients [@doi:10.1186/s13054-020-03060-9; @doi:10/ggvd74]. 
There is therefore preliminary evidence suggesting that vitamin C status and vitamin C administration may be relevant to COVID-19 outcomes.

Larger-scale studies of vitamin C, however, have provided mixed results.
A recent meta-analysis found consistent support for regular vitamin C supplementation reducing the duration of the common cold, but that supplementation with vitamin C (> 200 mg) failed to reduce the incidence of colds [@doi:10.1002/14651858.CD000980.pub4].
Individual studies have found Vitamin C to reduce the susceptibility of patients to lower respiratory tract infections, such as pneumonia [@doi:10.1097/00006454-199709000-00003].
Another meta-analysis demonstrated that in twelve trials, vitamin C supplementation reduced the length of stay of patients in intensive care units (ICUs) by 7.8% (95% CI: 4.2% to 11.2%; _p_ = 0.00003).
Furthermore, high doses (1-3 g/day) significantly reduced the length of an ICU stay by 8.6% in six trials (_p_ = 0.003).
Vitamin C also shortened the duration of mechanical ventilation by 18.2% in three trials in which patients required intervention for over 24 hours (95% CI 7.7% to 27%; p = 0.001) [@doi:10.3390/nu11040708].
Despite these findings, an RCT of 167 patients known as CITRUS ALI failed to show a benefit of a 96-hour infusion of vitamin C to treat ARDS [@doi:10.1001/jama.2019.11825].
Clinical trials specifically investigating vitamin C in the context of COVID-19 have now begun, as highlighted by Carr et al. [@doi:10.3390/nu12113286].
These trials intend to investigate the use of intravenous vitamin C in hospitalized COVID-19 patients.
The first trial to report initial results took place in Wuhan, China [@doi:10.1136/bmjopen-2020-039519].
These initial results indicated that the administration of 12 g/12 hr of intravenous vitamin C for 7 days in 56 critically ill COVID-19 patients resulted in a promising reduction of 28-day mortality (_p_ = 0.06) in univariate survival analysis [@doi:10.21203/rs.3.rs-52778/v2].
Indeed, the same study reported a significant decrease in IL-6 levels by day 7 of vitamin C infusion (_p_ = 0.04) [@doi:10.21203/rs.3.rs-52778/v1].
Additional studies that are being conducted in Canada, China, Iran, and the USA will provide additional insight into whether vitamin C supplementation affects COVID-19 outcomes on a larger scale.

Even though evidence supporting the use of vitamin C is beginning to emerge, we will not know how effective vitamin C is as a therapeutic for quite some time.
Currently (as of January 2021) over fifteen trials are registered with clinicaltrials.gov that are either recruiting, active or are currently in preparation.
When completed, these trials will provide crucial evidence on the efficacy of vitamin C as a therapeutic for COVID-19 infection.
However, the majority of supplementation studies investigate the intravenous infusion of vitamin C in severe patients.
Therefore, there is a lack of studies investigating the potential prophylactic administration of vitamin C via oral supplementation for healthy individuals or potentially asymptomatic SARS-CoV-2 positive patients.
Once again, vitamin C intake is part of a healthy diet and the vitamin likely presents minimal risk, but its potential prophylactic or therapeutic effects against COVID-19 are yet to be determined.
To maintain vitamin C status, it would be prudent for individuals to ensure that they consume the recommended dietary allowance of vitamin C to maintain a healthy immune system [@doi:10.3390/nu12051466].
The recommended dietary allowance according to the FDA is 75-90 mg/d, whereas EFSA recommends 110 mg/d [@doi:10.17226/9810].

### Vitamin D

Of all of the supplements currently under investigation, vitamin D has become a leading prophylactic and therapeutic candidate against SARS-CoV-2.
Vitamin D can modulate both the adaptive and innate immune system and is associated with various aspects of immune health and antiviral defense [@doi:10.3390/nu9070651; @doi:10.3390/ijms19082419; @doi:10.3390/nu12072097; @doi:10.1111/j.1365-2249.2009.04001.x; @doi:10.1016/j.ecl.2017.07.010].
Vitamin D can be sourced through diet or supplementation, but it is mainly biosynthesized by the body on exposure to ultraviolet light (UVB) from sunlight.
Vitamin D deficiency is associated with an increased susceptibility to infection [@doi:10.2310/JIM.0b013e31821b8755].
In particular, vitamin D deficient patients are at risk of developing acute respiratory infections [@doi:10.1016/j.jsbmb.2012.11.017] and ARDS [@doi:10.1016/j.jsbmb.2012.11.017].
1,25-dihydroxyvitamin D3 is the active form of vitamin D that is involved in adaptive and innate responses; however, due to its low concentration and a short half life of a few hours, vitamin D levels are typically measured by the longer lasting and more abundant precursor 25-hydroxyvitamin D.
The vitamin D receptor is expressed in various immune cells, and vitamin D is an immunomodulator of antigen presenting cells, dendritic cells, macrophages, monocytes, and T- and B-lymphocytes [@doi:10.2310/JIM.0b013e31821b8755; @doi:10.1016/j.coph.2010.04.001].
Due to its potential immunomodulating properties, vitamin D supplementation may be advantageous to maintain a healthy immune system.

Early in the pandemic it was postulated that an individual’s vitamin D status could significantly affect their risk of developing COVID-19 [@doi:10.3390/nu12040988].
This hypothesis was derived from the fact that the current pandemic emerged in Wuhan China during winter, when 25-hydroxyvitamin D concentrations are at their lowest due to a lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer and higher 25-hydroxyvitamin D concentrations would be higher, the number of cases was low.
This led researchers to question whether there was a seasonal component to the SARS-CoV-2 pandemic and whether vitamin D levels might play a role [@doi:10.3390/nu12040988; @doi:10.1111/joim.13149; @doi:10.1016/j.ajic.2020.06.193; @doi:10.1111/apt.15777].
Though it is assumed that COVID-19 is seasonal, multiple other factors that can affect vitamin D levels should also be considered.
These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others.
Indeed, it has been estimated that each degree of latitude north of 28 degrees corresponded to a 4.4% increase of COVID-19 mortality, indirectly linking a persons vitamin D levels via exposure to UVB light to COVID-19 mortality [@doi:10.1111/joim.13149].

As the pandemic has evolved, additional research of varying quality has investigated some of the potential links identified early in the pandemic [@doi:10.3390/nu12040988] between vitamin D and COVID-19.
Indeed, studies are beginning to investigate whether there is any prophylactic and/or therapeutic relationship between vitamin D and COVID-19.
A study in Switzerland demonstrated that 27 SARS-CoV-2 positive patients exhibited 25-hydroxyvitamin D plasma concentrations that were significantly lower (11.1 ng/ml) than those of SARS-CoV-2 negative patients (24.6 ng/ml; _p_ = 0.004), an association that held when stratifying patients greater than 70 years old [@doi:10.3390/nu12051359].
These findings seem to be supported by a Belgian observational study of 186 SARS-CoV-2 positive patients exhibiting symptoms of pneumonia, where 25-hydroxyvitamin D plasma concentrations were measured and CT scans of the lungs were obtained upon hospitalization [@doi:10.1101/2020.05.01.20079376].
A significant difference in 25-hydroxyvitamin D levels was observed between the SARS-CoV-2 patients and 2,717 season-matched hospitalized controls.
It is not clear from the study which diseases caused the control subjects to be admitted at the time of their 25-hydroxyvitamin D measurement, which makes it difficult to assess the observations reported.
Both female and male patients possessed lower median 25-hydroxyvitamin D concentrations than the control group as a whole (18.6 ng/ml versus 21.5 ng/ml; _p_ = 0.0016) and a higher rate of vitamin D deficiency (58.6% versus 42.5%).
However, when comparisons were stratified by sex, evidence of sexual dimorphism became apparent, as female patients had equivalent levels of 25-hydroxyvitamin D to females in the control group, whereas male patients were deficient in 25-hydroxyvitamin D relative to male controls (67% versus 49%; _p_ = 0.0006).
Notably, vitamin D deficiency was progressively lower in males with advancing radiological disease stages (_p_ = 0.001).
However, these studies are supported by several others that indicate that vitamin D status may be an independent risk factor for the severity of COVID-19 [@doi:10.1371/journal.pone.0239799; @doi:10.1016/j.jiph.2020.06.021; @doi:10.1111/febs.15495; @doi:10.1001/jamanetworkopen.2020.19722] and in COVID-19 patients relative to population-based controls [@doi:10.1210/clinem/dgaa733].
Indeed, serum concentrations of 25-hydroxyvitamin D above 30 ng/ml, which indicate vitamin D sufficiency, seems to be associated with a reduction in serum C-reactive protein, an inflammatory marker, along with increased lymphocyte levels, which suggests that vitamin D levels may modulate the immune response by reducing risk for cytokine storm in response to SARS-CoV-2 infection [@doi:10.1210/clinem/dgaa733].
A study in India determined that COVID-19 fatality was higher in patients with severe COVID-19 and low serum 25-hydroxyvitamin D (mean level 6.2 ng/ml; 97% vitamin D deficient) levels versus asymptomatic non-severe patients with higher levels of vitamin D (mean level 27.9 ng/ml; 33% vitamin D deficient) [@doi:10.1038/s41598-020-77093-z].
In the same study, vitamin D deficiency was associated with higher levels of inflammatory markers including IL-6, ferritin, and tumor necrosis factor &alpha;.
Collectively, these studies add to a multitude of observational studies reporting potential associations between low levels of 25-hydroxyvitamin D and COVID-19 incidence and severity [@doi:10.3390/nu12051359; @doi:10.3390/nu12123773; @doi:10.1007/s40618-020-01370-x; @doi:10.1210/clinem/dgaa733; @doi:10.3390/nu12092757; @doi:10.3390/nu12092775; @doi:10.1080/07315724.2020.1826005; @doi:10.1001/jamanetworkopen.2020.19722; @doi:10.1016/j.intimp.2020.107001; @doi:10.1093/jn/nxaa332].

Despite the large number of studies establishing a link between vitamin D status and COVID-19 severity, an examination of data from the UK Biobank did not support this thesis [@doi:10.1016/j.dsx.2020.04.050; @doi:10.1007/s00394-020-02372-4].
These analyses examined 25-hydroxyvitamin D concentrations alongside SARS-CoV-2 positivity and COVID-19 mortality in over 340,000 UK Biobank participants.
However, these studies have caused considerable debate that will likely be settled following further studies [@doi:10.1111/cen.14276; @doi:10.1016/j.dsx.2020.05.046].
Overall, while the evidence suggests that there is likely an association between low serum 25-hydroxyvitamin D and COVID-19 incidence, these studies must be interpreted with caution, as there is the potential for reverse causality, bias, and other confounding factors including that vitamin D deficiency is also associated with numerous pre-existing conditions and risk factors that can increase the risk for severe COVID-19 [@doi:10.3390/nu12051466; @doi:10.1038/s41371-020-00398-z; @doi:10.1111/joim.13149; @doi:10.1007/s40520-020-01677-y].

While these studies inform us of the potential importance of vitamin D sufficiency and the risk of SARS-CoV-2 infection and severe COVID-19, they fail to conclusively determine whether vitamin D supplementation can therapeutically affect the clinical course of COVID-19.
In one study, 40 vitamin D deficient asymptomatic or mildly symptomatic participants patients were either randomized to receive 60,000 IU of cholecalciferol daily for at least 7 days (n = 16) or a placebo (n = 24) with a target serum 25-hydroxyvitamin D level >50 ng/ml.
At day 7, 10 patients achieved >50 ng/ml, followed by another 2 by day 14.
By the end of the study, the treatment group had a greater proportion of vitamin D-deficient participants that tested negative for SARS-CoV-2 RNA, and they had a significantly lower fibrinogen levels, potentially indicating a beneficial effect [@doi:10.1136/postgradmedj-2020-139065].
A pilot study in Spain determined that early administration of high dose calcifediol (~21,000 IU days 1-2 and ~11,000 IU days 3-7 of hospital admission) with hydroxychloroquine and azithromycin to 50 hospitalized COVID-19 patients significantly reduced ICU admissions and may have reduced disease severity versus hydroxychloroquine and azithromycin alone [@doi:10.1016/j.jsbmb.2020.105751].
Although this study received significant criticism from the National Institute for Health and Care Excellence (NICE) in the UK [@url:https://www.nice.org.uk/advice/es28/chapter/Key-messages], an independent follow-up statistical analysis supported the findings of the study with respect to the results of cholecalciferol treatment [@doi:10.1101/2020.11.08.20222638].
Another trial of 986 patients hospitalized for COVID-19 in three UK hospitals administered cholecalciferol (≥ 280,000 IU in a time period of 7 weeks) to 151 patients and found an association with a reduced risk of COVID-19 mortality, regardless of baseline 25-hydroxyvitamin D levels [@doi:10.3390/nu12123799].
However, a double-blind, randomized, placebo-controlled trial of 240 hospitalized COVID-19 patients in São Paulo, Brazil administered a single 200,000 IU oral dose of vitamin D.
At the end of the study, there was a 24 ng/mL difference of 25-hydroxyvitamin D levels in the treatment group versus the placebo group (_p_ = 0.001), and 87% of the treatment group were vitamin D sufficient versus ~11% in the placebo group.
Supplementation was well tolerated. 
However, there was no reduction in the length of hospital stay or mortality, and no change to any other relevant secondary outcomes were reported [@doi:10.1101/2020.11.16.20232397].
These early findings are thus still inconclusive with regards to the therapeutic value of vitamin D supplementation.
However, other trials are underway, including one trial that is investigating the utility of vitamin D as an immune-modulating agent by monitoring whether administration of vitamin D precipitates an improvement of health status in non-severe symptomatic COVID-19 patients and whether vitamin D prevents patient deterioration [@clinicaltrials:NCT04334005].
Other trials are examining various factors including mortality, symptom recovery, severity of disease, rates of ventilation, inflammatory markers such as C-reactive protein and IL-6, blood cell counts, and the prophylactic capacity of vitamin D administration [@clinicaltrials:NCT04385940; @clinicaltrials:NCT04334005; @clinicaltrials:NCT04411446; @clinicaltrials:NCT04344041].
Concomitant administration of vitamin D with pharmaceuticals such as aspirin [@clinicaltrials:NCT04363840] and bioactive molecules such as resveratrol [@clinicaltrials:NCT04400890] is also under investigation.

The effectiveness of vitamin D supplementation against COVID-19 remains open for debate.
All the same, there is no doubt that vitamin D deficiency is a widespread issue and should be addressed not only because of its potential link to SARS-CoV-2 incidence [@doi:10.1530/eje-18-0736], but also due to its importance for overall health.
There is a possibility that safe exposure to sunlight could improve endogenous synthesis of vitamin D, potentially strengthening the immune system.
However, sun exposure is not sufficient on its own, particularly in the winter months.
Indeed, while the possible link between vitamin D status and COVID-19 is further investigated, preemptive supplementation of vitamin D and encouraging people to maintain a healthy diet for optimum vitamin D status is likely to raise serum levels of 25-hydroxyvitamin D while being unlikely to carry major health risks.
These principles seem to be the basis of a number of guidelines issued by some countries and scientific organizations that have advised supplementation of vitamin D during the pandemic.
The Académie Nationale de Médecine in France recommends rapid testing of 25-hydroxyvitamin D for people over 60 years old to identify those most at risk of vitamin D deficiency and advises them to obtain a bolus dose of 50,000 to 100,000 IU vitamin D to limit respiratory complications.
It has also recommended that those under 60 years old should take 800 to 1,000 IU daily if they receive a SARS-CoV-2 positive test [@url:https://www.academie-medecine.fr/communique-de-lacademie-nationale-de-medecine-vitamine-d-et-covid-19/].
In Slovenia, doctors have been advised to provide nursing home patients with vitamin D [@doi:10.1136/bmj.m1820].
Both Public Health England and Public Health Scotland have advised members of the Black, Asian, and minority ethnic communities to supplement for vitamin D in light of evidence that they may be at higher risk for vitamin D deficiency along with other COVID-19 risk factors, a trend that has also been observed in the United States [@doi:10.1136/bmj.m2475; @doi:10.1136/bmjnph-2020-000096].
However, other UK scientific bodies including the NICE recommend that individuals supplement for vitamin D as per usual UK government advice but warn that people should not supplement for vitamin D solely to prevent COVID-19.
All the same, the NICE has provided guidelines for research to investigate the supplementation of vitamin D in the context of COVID-19 [@url:https://www.nice.org.uk/guidance/ng187/resources/covid19-rapid-guideline-vitamin-d-pdf-66142026720709].
Despite vitamin D deficiency being a widespread issue in the United States [@doi:10.7759/cureus.2741], the National Institutes of Health have stated that there is "insufficient data to recommend either for or against the use of vitamin D for the prevention or treatment of COVID-19" [@url:https://www.covid19treatmentguidelines.nih.gov/adjunctive-therapy/vitamin-d/].
These are just some examples of how public health guidance has responded to the emerging evidence regarding vitamin D and COVID-19.
Outside of official recommendations, there is also evidence that individuals may be paying increased attention to their vitamin D levels, as a survey of Polish consumers showed that 56% of respondents used vitamin D during the pandemic [@doi:10.3390/nu13010054].
However, some companies have used the emerging evidence surrounding vitamin D to sell products that claim to prevent and treat COVID-19, which in one incident required a federal court to intervene and issue an injunction barring the sale of vitamin-D-related products due to the lack of clinical data supporting these claims [@url:https://www.justice.gov/opa/pr/court-orders-georgia-defendants-stop-selling-vitamin-d-products-treatments-covid-19-and-other].
It is clear that further studies and clinical trials are required to conclusively determine the prophylactic and therapeutic potential of vitamin D supplementation against COVID-19.
Until such time that sufficient evidence emerges, individuals should follow their national guidelines surrounding vitamin D intake to achieve vitamin D sufficiency.

### Probiotics

Probiotics are “live microorganisms that, when administered in adequate amounts, confer a health benefit on the host” [@doi:10.1038/nrgastro.2014.66].
Some studies suggest that probiotics are beneficial against common viral infections, and there is modest evidence to suggest that they can modulate the immune response [@doi:10.4082/kjfm.2013.34.1.2; @doi:10.2174/1381612824666180116163411].
As a result, it has been hypothesized that probiotics may have therapeutic value worthy of investigation against SARS-CoV-2 [@doi:10.3389/fpubh.2020.00186].
Probiotics and next-generation probiotics, which are more akin to pharmacological-grade supplements, have been associated with multiple potential beneficial effects for allergies, digestive tract disorders, and even metabolic diseases through their anti-inflammatory and immunomodulatory effects [@doi:10.1038/nmicrobiol.2017.57; @doi:10.1093/advances/nmy063].
However, the mechanisms by which probiotics affect these various conditions would likely differ among strains, with the ultimate effect of the probiotic depending on the heterogeneous set of bacteria present [@doi:10.1093/advances/nmy063].
Some of the beneficial effects of probiotics include reducing inflammation by promoting the expression of anti-inflammatory mediators, inhibiting Toll-like receptors 2 and 4, competing directly with pathogens, synthesizing antimicrobial substances or other metabolites, improving intestinal barrier function, and/or favorably altering the gut microbiota and the brain-gut axis [@doi:10.1016/j.earlhumdev.2019.05.010; @doi:10.1093/advances/nmy063; @doi:10.1159/000342079].
It is also thought that lactobacilli such as _Lactobacillus paracasei, Lactobacillus plantarum_ and _Lactobacillus rhamnosus_ have the capacity to bind to and inactivate some viruses via adsorptive and/or trapping mechanisms [@doi:10.1016/j.ijfoodmicro.2006.10.044].
Other probiotic lactobacilli and even non-viable bacterium-like particles have been shown to reduce both viral attachment to host cells and viral titers, along with reducing cytokine synthesis, enhancing the antiviral IFN-α response, and inducing various other antiviral mechanisms [@doi:10.1111/lam.12160; @doi:10.1016/j.anaerobe.2011.04.022; @doi:10.4049/jimmunol.1800927; @doi:10.1016/j.ijfoodmicro.2006.10.044; @doi:10.3390/medicina55090519; @doi:10.1038/s41598-019-39602-7; @doi:10.3920/bm2016.0160; @doi:10.1016/j.intimp.2013.12.020; @doi:10.1016/j.intimp.2019.106115].
These antiviral and immunobiotic mechanisms and others have been reviewed in detail elsewhere [@doi:10.3390/nu12061718; @doi:10.1038/s41538-020-00078-9; @doi:10.3389/fpubh.2020.00186].
However, there is also a bi-directional relationship between the lungs and gut microbiota known as the gut-lung axis [@doi:10.1038/mi.2011.55], whereby gut microbial metabolites and endotoxins may affect the lungs via the circulatory system and the lung microbiota in return may affect the gut [@doi:10.1111/cmi.12966].
Therefore, the gut-lung axis may play role in our future understanding of COVID-19 pathogenesis and become a target for probiotic treatments [@doi:10.1016/j.virusres.2020.198018].
Moreover, as microbial dysbiosis of the respiratory tract and gut may play a role in some viral infections, it has been suggested that SARS-CoV-2 may interact with our commensal microbiota [@doi:10.3390/nu12061718; @doi:10.3389/fmicb.2020.01840; 10.3389/fmicb.2020.01840] and that the lung microbiome could play a role in developing immunity to viral infections [@doi:10.1016/j.humic.2020.100073].
These postulations, if correct, could lead to the development of novel probiotic and prebiotic treatments.
However, significant research is required to confirm these associations and their relevance to patient care, if any.

Probiotic therapies and prophylactics may also confer some advantages for managing symptoms of COVID-19 or risks associated with its treatment.
Probiotics have tentatively been associated with the reduction of risk and duration of viral upper respiratory tract infections [@doi:10.1007/s10096-014-2086-y; @doi:10.1002/14651858.CD006895.pub3; @doi:10.1016/j.ijantimicag.2008.11.005].
Some meta-analyses that have assessed the efficacy of probiotics in viral respiratory infections have reported moderate reductions in the incidence and duration of infection [@doi:10.1002/14651858.CD006895.pub3; @doi:10.1017/S0007114514000075].
Indeed, randomized controlled trials have shown that administering _Bacillus subtilis_ and _Enterococcus faecalis_ [@doi:10.1007/s00134-016-4303-x], _Lactobacillus rhamnosus GG_ [@doi:10.1164/rccm.200912-1853OC], or _Lactobacillus casei_ and _Bifidobacterium breve_ with galactooligosaccharides [@doi:10.1186/s13054-018-2167-x] via the nasogastric tube to ventilated patients reduced the occurrence of ventilator-associated pneumonia in comparison to the respective control groups in studies of viral infections and sepsis.
These findings were also supported by a recent meta-analysis [@doi:10.4187/respcare.07097].
Additionally, COVID-19 patients carry a significant risk of ventilator-associated bacterial pneumonia [@doi:10.6084/m9.figshare.12340496], but it can be challenging for clinicians to diagnose this infection due to the fact that severe COVID-19 infection presents with the symptoms of pneumonia [@doi:10.1186/s13054-020-03013-2].
Therefore, an effective prophylactic therapy for ventilator-associated pneumonia in severe COVID-19 patients would carry significant therapeutic value.
Additionally, in recent years, probiotics have become almost synonymous with the treatment of gastrointestinal issues due to their supposed anti-inflammatory and immunomodulatory effects [@doi:10/d2qp].
Notably, gastrointestinal symptoms commonly occur in COVID-19 patients [@doi:10.1053/j.gastro.2020.03.020], and angiotensin-converting enzyme 2, the portal by which SARS-CoV-2 enters human cells, is highly expressed in enterocytes of the ileum and colon, suggesting that these organs may be a potential route of infection [@doi:10.1101/2020.01.30.927806; @doi:10.1038/cr.2016.152].
Indeed, SARS-CoV-2 viral RNA has been detected in human feces [@doi:10/ggq8zp; @doi:10.1053/j.gastro.2020.02.055], and fecal-oral transmission of the virus has not yet been ruled out [@doi:10/ggpx7s].
Rectal swabs of some SARS-CoV-2 positive pediatric patients persistently tested positive for several days despite negative nasopharyngeal tests, indicating the potential for fecal viral shedding [@doi:10.1038/s41591-020-0817-4].
However, there is conflicting evidence for the therapeutic value of various probiotics against the incidence or severity of gastrointestinal symptoms in viral or bacterial infections such as gastroenteritis [@doi:10.1007/s00203-017-1400-3; @doi:10.1056/NEJMoa1802597].
Nevertheless, it has been proposed that the administration of probiotics to COVID-19 patients and healthcare workers may prevent or ameliorate the gastrointestinal symptoms of COVID-19, a hypothesis that several clinical trials are now preparing to investigate [@clinicaltrials:NCT04420676; @clinicaltrials:NCT04366180].
Other studies are investigating whether probiotics may affect patient outcomes following SARS-CoV-2 infection [@clinicaltrials:NCT04390477].

Generally, the efficacy of probiotic use is a controversial topic among scientists.
In Europe, EFSA has banned the term probiotics on products labels, which has elicited either criticism for EFSA or support for probiotics from researchers in the field [@doi:10.1016/j.bpg.2015.12.001; @doi:10.1017/S000711451100287X; @doi:10.1038/nrgastro.2014.66].
This regulation is due to the hyperbolic claims placed on the labels of various probiotic products, which lack rigorous scientific data to support their efficacy.
Overall, the data supporting probiotics in the treatment or prevention of many different disorders and diseases is not conclusive, as the quality of the evidence is generally considered low [@doi:10.1007/s10096-014-2086-y].
However, in the case of probiotics and respiratory infections, the evidence seems to be supportive of their potential therapeutic value.
Consequently, several investigations are underway to investigate the prophylactic and therapeutic potential of probiotics for COVID-19.
The blind use of conventional probiotics for COVID-19 is currently cautioned against until the pathogenesis of SARS-CoV-2 can be further established [@doi:10/d2qq].
Until clinical trials investigating the prophylactic and therapeutic potential of probiotics for COVID-19 are complete, it is not possible to provide an evidence-based recommendation for their use.
Despite these concerns, complementary use of probiotics as an adjuvant therapeutic has been proposed by the Chinese National Health Commission and National Administration of Traditional Chinese Medicine [@doi:10.1111/1751-2980.12851].
While supply issues prevented the probiotics market from showing the same rapid response to the COVID-19 as some other supplements, many suppliers are reporting growth during the pandemic [@url:https://www.bloomberg.com/press-releases/2020-08-03/probiotics-market-worth-76-7-billion-by-2027-exclusive-report-covering-pre-and-post-covid-19-market-analysis-by-meticulous].
Therefore, the public response once again seems to have adopted supplements promoted as bolstering the immune response despite a lack of evidence suggesting they are beneficial for preventing or mitigating COVID-19.

### Discussion

In this review, we report the findings to date of analyses of several dietary supplements and nutraceuticals.
While existing evidence suggests potential benefits of n-3 PUFA and probiotic supplementation for COVID-19 treatment and prophylaxis, clinical data is still lacking, although trials are underway.
Both zinc and vitamin C supplementation in hospitalized patients seem to be associated with positive outcomes; however, further clinical trials are required.
In any case, vitamin C and zinc intake are part of a healthy diet and likely present minimal risk when supplemented, though their potential prophylactic or therapeutic effects against COVID-19 are yet to be determined.
On the other hand, mounting evidence from observational studies indicates that there is an association between vitamin D deficiency and COVID-19 incidence has also been supported by meta-analysis [@doi:10.1080/10408398.2020.1841090].
Indeed, scientists are working to confirm these findings and to determine whether a patient's serum 25-hydroxyvitamin D levels are also associated with COVID-19 severity.
Clinical trials are required to determine whether preemptive vitamin D supplementation may mitigate against severe COVID-19.
In terms of the therapeutic potential of vitamin D, initial evidence from clinical trials is conflicting but seems to indicate that vitamin D supplementation may reduce COVID-19 severity [@doi:10.1016/j.jsbmb.2020.105751].
The various clinical trials currently underway will be imperative to provide information on the efficacious use of vitamin D supplementation for COVID-19 prevention and/or treatment.

The purported prophylactic and therapeutic benefits of dietary supplements and nutraceuticals for multiple disorders, diseases, and infections has been the subject of significant research and debate for the last few decades.
Inevitably, scientists are also investigating the potential for these various products to treat or prevent COVID-19.
This interest also extends to consumers, which led to a remarkable increase of sales of dietary supplements and nutraceuticals throughout the pandemic due to a desire to obtain additional protections from infection and disease.
The nutraceuticals discussed in this review, namely vitamin C, vitamin D, n-3 PUFA, zinc, and probiotics, were selected because of potential biological mechanisms that could beneficially affect viral and respiratory infections and because they are currently under clinical investigation.
Specifically, these compounds have all been found to influence cellular processes related to inflammation.
Inflammation is particularly relevant to COVID-19 because of the negative outcomes (often death) observed in a large number of patients whose immune response becomes hyperactive in response to SARS-CoV-2, leading to severe outcomes such as ARDS and sepsis [@doi:10.1056/NEJMra2026131].
Additionally, there is a well-established link between diet and inflammation [@doi:10.1177/0884533610385703], potentially mediated in part by the microbiome [@doi:10.1096/fj.201802477R].
Thus, the idea that dietary modifications or supplementation could be used to modify the inflammatory response is tied to a broader view of how diet and the immune system are interconnected.
The supplements and nutraceuticals discussed here therefore lie in sharp contrast to other alleged nutraceutical or dietary supplements that have attracted during the pandemic, such as colloidal silver [@url:https://www.nccih.nih.gov/health/colloidal-silver], which have no known nutritional function and can be harmful.
Importantly, while little clinical evidence is available about the effects of any supplements against COVID-19, the risks associated with those discussed above are likely to be low, and in some cases, they can be obtained from dietary sources alone.

There are various other products and molecules that have garnered scientific interest and could merit further investigation.
These include polyphenols, lipid extracts, and tomato-based nutraceuticals, all of which have been suggested for the potential prevention of cardiovascular complications of COVID-19 such as thrombosis [@doi:10.3389/fnut.2020.583080; @doi:10.3390/nu12061718].
Melatonin is another supplement that has been identified as a potential antiviral agent against SARS-CoV-2 using computational methods [@doi:10.1038/s41421-020-0153-3], and it has also been highlighted as a potential therapeutic agent for COVID-19 due to its documented antioxidant, anti-apoptotic, immunomodulatory, and anti-inflammatory effects [@doi:10.3389/fnut.2020.583080; @doi:10.3390/antiox10010047; @doi:10.3390/nu13010047].
Notably, melatonin, vitamin D and zinc have attracted public attention because they were included in the treatment plan of the former President of the United States upon his hospitalization due to COVID-19 [@doi:10.1126/science.abf0974].
These are just some of the many substances and supplements that are currently under investigation but as of yet lack evidence to support their use for the prevention or treatment of COVID-19.
While there is plenty of skepticism put forward by physicians and scientists surrounding the use of supplements, these statements have not stopped consumers from purchasing these products, with one study reporting that online searches for dietary supplements in Poland began trending with the start of the pandemic [@doi:10.3390/nu13010054].
Additionally, supplement usage increased between the first and second wave of the pandemic.
Participants reported various reasons for their use of supplements, including to improve immunity (60%), to improve overall health (57%), and to fill nutrient gaps in their diet (53%).
Other efforts to collect large datasets regarding such behavior have also sought to explore a possible association between vitamin or supplement consumption and COVID-19.
An observational analysis of survey responses from 327,720 users of the COVID Symptom Study App found that the consumption of n-3 PUFA supplements, probiotics, multivitamins, and vitamin D was associated with a lower risk of SARS-CoV-2 infection in women but not men after adjusting for potential confounders [@doi:10.1101/2020.11.27.20239087].
According to the authors, the sexual dimorphism observed may in part be because supplements may better support females due to known differences between the male and female immune systems, or it could be due to behavioral and health consciousness differences between the sexes [@doi:10.1101/2020.11.27.20239087].
Certainly, randomized controlled trials are required to investigate these findings further.

Finally, it is known that a patient's nutritional status affects health outcomes in various infectious diseases [@doi:10.3390/nu12041181], and COVID-19 is no different [@doi:10.1016/j.clnu.2020.03.022; @doi:10.1093/advances/nmaa125; @doi:10.1016/j.clnesp.2020.09.214].
Some of the main risk factors for severe COVID-19, which also happen to be linked to poor nutritional status, include obesity, hypertension, cardiovascular diseases, type II diabetes mellitus, and indeed age-related malnutrition [@doi:10.1093/advances/nmaa125; @doi:10.3390/nu12051466; @doi:10.1002/jpen.1954].
Although not the main focus of this review, it is important to consider the nutritional challenges associated with severe COVID-19 patients.
Hospitalized COVID-19 patients tend to report an unusually high loss of appetite preceding admission, some suffer diarrhea and gastrointestinal symptoms that result in significantly lower food intake, and patients with poorer nutritional status were more likely to have worse outcomes and require nutrition therapy [@doi:10.1002/jpen.1953].
Dysphagia also seems to be a significant problem in pediatric patients that suffered multisystem inflammatory syndrome [@doi:10.1016/j.eclinm.2020.100527] and rehabilitating COVID-19 patients, potentially contributing to poor nutritional status [@doi:10.1038/s41430-020-0664-x].
Almost two-thirds of discharged COVID-19 ICU patients exhibit significant weight loss, of which 26% had weight loss greater than 10% [@doi:10.1016/j.clnesp.2020.09.214].
As investigated in this review, hospitalized patients also tend to exhibit vitamin D deficiency or insufficiency, which may be associated with greater disease severity [@doi:10.1080/10408398.2020.1841090].
Therefore, further research is required to determine how dietary supplements and nutraceuticals may contribute to the treatment of severely ill and rehabilitating patients, who often rely on enteral nutrition.

### Conclusions

Despite all the potential benefits of nutraceutical and dietary supplement interventions presented, currently there is a paucity of clinical evidence to support their use for the prevention or mitigation of COVID-19 infection.
Nevertheless, optimal nutritional status can prime an individual’s immune system to protect against the effects of acute respiratory viral infections by supporting normal maintenance of the immune system [@doi:10.3390/nu12041181; @doi:10.3390/nu12051466].
Nutritional strategies can also play a role in the treatment of hospitalized patients, as malnutrition is a risk to COVID-19 patients [@doi:10.1038/s41430-020-0664-x].
Overall, supplementation of vitamin C, vitamin D, and zinc may be an effective method of ensuring their adequate intake to maintain optimal immune function, which may also convey beneficial effects against viral infections due to their immunomodulatory effects.
Individuals should pay attention to their nutritional status, particularly their intake of vitamin D, considering that vitamin D deficiency is widespread.
The prevailing evidence seems to indicate an association between vitamin D deficiency with COVID-19 incidence and, potentially, severity [@doi:10.1111/joim.13149].
As a result, some international authorities have advised the general public, particularly those at high risk of infection, to consider vitamin D supplementation.
However, further well-controlled clinical trials are required to confirm these observations.

Many supplements and nutraceuticals designed for various ailments that are available in the United States and beyond are not strictly regulated [@doi:10.1080/10408398.2019.1592107].
Consequently, there can be safety and efficacy concerns associated with many of these products.
Often, the vulnerable members of society can be exploited in this regard and, unfortunately, the COVID-19 pandemic has proven no different.
As mentioned above, the FDA has issued warnings to several companies for advertising falsified claims in relation to the preventative and therapeutic capabilities of their products against COVID-19 [@url:https://www.fda.gov/news-events/press-announcements/coronavirus-update-fda-and-ftc-warn-seven-companies-selling-fraudulent-products-claim-treat-or].
Further intensive investigation is required to establish the effects of these nutraceuticals, if any, against COVID-19.
Until more effective therapeutics are established, the most effective mitigation strategies consist of encouraging standard public health practices such as regular hand washing with soap, wearing a face mask, and covering a cough with your elbow [@url:https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/prevention.html], along with following social distancing measures, "stay at home" guidelines, expansive testing, and contact tracing [@doi:10.1101/2020.03.27.20045815; @doi:10.3201/eid2608.201093].
Indeed, in light of this review, it would also be pertinent to adopt a healthy diet and lifestyle following national guidelines in order to maintain optimal immune health.
Because of the broad public appeal of dietary supplements and nutraceuticals, it is important to evaluate the evidence regarding the use of such products.
We will continue to update this review as more findings become available.


## Social Factors Influencing COVID-19 Exposure and Outcomes

### Social Factors Influencing COVID-19 Outcomes

In addition to understanding the fundamental biology of the SARS-CoV-2 virus and COVID-19, it is critical to consider how the broader environment can influence both COVID-19 outcomes and efforts to develop and implement treatments for the disease.
The evidence clearly indicates that social environmental factors are critical determinants of individuals' and communities' risks related to COVID-19.
There are distinct components to COVID-19 susceptibility, and an individual's risk can be elevated at one or all stages from exposure to recovery/mortality: an individual may be more likely to be exposed to the virus, more likely to get infected once exposed, more likely to have serious complications once infected, and be less likely to receive adequate care once they are seriously ill.
The fact that differences in survival between Black and white patients were no longer significant after controlling for comorbidities and socioeconomic status (type of insurance, neighborhood deprivation score, and hospital where treatment was received) in addition to sex and age [@doi:10.1001/jamanetworkopen.2020.18039] underscores the relevance of social factors to understanding mortality differences between racial and ethnic groups.
Moreover, the Black patients were younger and more likely to be female than white patients, yet still had a higher mortality rate without correction for the other variables [@doi:10.1001/jamanetworkopen.2020.18039].
Here, we outline a few systemic reasons that may exacerbate the COVID-19 pandemic in communities of color.

### Factors Observed to be Associated with Susceptibility

As COVID-19 has spread into communities around the globe, it has become clear that the risks associated with this disease are not equally shared by all individuals or all communities.
Significant disparities in outcomes have led to interest in the demographic, biomedical, and social factors that influence COVID-19 severity.
Untangling the factors influencing COVID-19 susceptibility is a complex undertaking.
Among patients who are admitted to the hospital, outcomes have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].
However, hospitalization rates vary by location [@doi:10.15585/mmwr.mm6915e3].
This variation may be influenced by demographic (e.g., average age in the area), medical (e.g., the prevalence of comorbid conditions such as diabetes), and social (e.g., income or healthcare availability) factors that vary geographically.
Additionally, some of the same factors may influence an individual's probability of exposure to SARS-CoV-2, their risk of developing a more serious case of COVID-19 that would require hospitalization, and their access to medical support.
As a result, quantifying or comparing susceptibility among individuals, communities, or other groups requires consideration of a number of complex phenomena that intersect across many disciplines of research.
In this section, the term "risk factor" is used to refer to variables that are statistically associated with more severe COVID-19 outcomes.
Some are intrinsic characteristics that have been observed to carry an association with variation in outcomes, whereas others may be more functionally linked to the pathophysiology of COVID-19.

#### Patient Traits Associated with Increased Risk

Two traits that have been consistently associated with more severe COVID-19 outcomes are male sex and advanced age (typically defined as 60 or older, with the greatest risk among those 85 and older [@url:https://www.cdc.gov/coronavirus/2019-ncov/need-extra-precautions/older-adults.html]).
In the United States, males and older individuals diagnosed with COVID-19 were found to be more likely to require hospitalization [@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1].
A retrospective study of hospitalized Chinese patients [@doi:10/ggnxb3] found that a higher probability of mortality was associated with older age, and world-wide, population age structure has been found to be an important variable for explaining differences in outbreak severity [@doi:10.1073/pnas.2004911117].
The CFR for adults over 80 has been estimated upwards of 14% or even 20% [@doi:10.1111/jgs.16472].
Male sex has also been identified as a risk factor for severe COVID-19 outcomes, including death [@doi:10.1136/bmj.m1985; @doi:10.1186/s13293-020-00304-9; @url:https://globalhealth5050.org/the-sex-gender-and-covid-19-project/].
Early reports from China and Europe indicated that even though the case rates were similar across males and females, males were at elevated risk for hospital admission, ICU admission, and death [@doi:10.1186/s13293-020-00304-9], although data from some US states indicates more cases among females, potentially due to gender representation in care-taking professions [@doi:10.1371/journal.ppat.1008570].
In older age groups (e.g., age 60 and older), comparable absolute numbers of male and female cases actually suggests a higher rate of occurrence in males, due to increased skew in the sex ratio [@doi:10.1186/s13293-020-00304-9].
Current estimates based on worldwide data suggest that, compared to females, males may be 30% more likely to be hospitalized, 80% more likely to be admitted to the ICU, and 40% more likely to die as a result of COVID-19 [@url:https://globalhealth5050.org/the-sex-gender-and-covid-19-project/].
There also may be a compounding effect of advanced age and male sex, with differences time to recovery worst for males over 60 years old relative to female members of their age cohort [@doi:10.1371/journal.ppat.1008520].

Both of these risk factors can be approached through the lens of biology.
The biological basis for greater susceptibility with age is likely linked to the prevalence of extenuating health conditions such as heart failure or diabetes [@doi:10.1111/jgs.16472].
Several hypotheses have been proposed to account for differences in severity between males and females.
For example, some evidence suggests that female sex hormones may be protective [@doi:10.1186/s13293-020-00304-9; @doi:10.1371/journal.ppat.1008570].
ACE2 expression in the kidneys of male mice was observed to be twice as high as that of females, and a regulatory effect of estradiol on ACE2 expression was demonstrated by removing the gonads and then supplementing with estradiol [@doi:10.1186/2042-6410-1-6; @doi:10.1371/journal.ppat.1008570].
Other work in mice has shown an inverse association between mortality due to SARS-CoV-1 and estradiol, suggesting a protective role for the sex hormone [@doi:10.1371/journal.ppat.1008570].
Similarly, evidence suggests that similar patterns might be found in other tissues.
A preliminary analysis identified higher levels of ACE2 expression in the myocardium of male patients with aortic valve stenosis showed  than female patients, although this pattern was not found in controls [@doi:10.1186/s13293-020-00304-9].
Additionally, research has indicated that females respond to lower doses than males of heart medications that act on the Renin angiotensin aldosterone system (RAAS) pathway, which is shared with ACE2 [@doi:10.1186/s13293-020-00304-9].
Additionally, several components of the immune response, including the inflammatory response, may differ in intensity and timing between males and females [@doi:10.1371/journal.ppat.1008570; @doi:10.1186/2042-6410-1-6].
This hypothesis is supported by some preliminary evidence showing that female patients who recovered from severe COVID-19 had higher antibody titers than males [@doi:10.1371/journal.ppat.1008570].
Sex steroids can also bind to immune cell receptors to influence cytokine production [@doi:10.1186/s13293-020-00304-9].
Additionally, social factors may influence risks related to both age and sex: for example, older adults are more likely to live in care facilities, which have been a source for a large number of outbreaks [@doi:10.1093/qjmed/hcaa136], and gender roles may also influence exposure and/or susceptibility due to differences in care-taking and/or risky behaviors (e.g., caring for elder relatives and smoking, respectively) [@doi:10.1186/s13293-020-00304-9] among men and women (however, it should be noted that both transgender men and women are suspected to be at heightened risk [@url:https://escholarship.org/uc/item/55t297mc].)

#### Comorbid Health Conditions

A number of pre-existing or comorbid conditions have repeatedly been identified as risk factors for more severe COVID-19 outcomes.
Several underlying health conditions were identified at high prevalence among hospitalized patients, including obesity, diabetes, hypertension, lung disease, and cardiovascular disease [@doi:10.15585/mmwr.mm6915e3].
Higher Sequential Organ Failure Assessment (SOFA) scores have been associated with a higher probability of mortality [@doi:10/ggnxb3], and comorbid conditions such as cardiovascular and lung disease as well as obesity were also associated with an increased risk of hospitalization and death, even when correcting for age and sex [@doi:10.1136/bmj.m1985].
Diabetes may increase the risk of lengthy hospitalization [@doi:10.1016/j.cmet.2020.04.021] or of death [@doi:10.1016/j.cmet.2020.04.021;@doi:10.1007/s00592-020-01546-0].
[@doi:10.1111/dom.14057] and [@doi:10.1152/ajpendo.00124.2020] discuss possible ways in which COVID-19 and diabetes may interact.
Obesity also appears to be associated with higher risk of severe outcomes from SARS-CoV-2 [@doi:10.1016/j.metabol.2020.154262; @doi:10.1101/2020.04.23.20076042].
Obesity is considered an underlying risk factor for other health problems, and the mechanism for its contributions to COVID-19 hospitalization or mortality is not yet clear [@doi:10.1016/j.medj.2020.06.005].
Dementia and cancer were also associated with the risk of death in an analysis of a large number (more than 20,000) COVID-19 patients in the United Kingdom [@doi:10.1136/bmj.m1985].
It should be noted that comorbid conditions are inextricably tied to age, as conditions tend to be accumulated over time, but that the prevalence of individual comorbidities or of population health overall can vary regionally [@doi:10.1073/pnas.2008760117].
Several comorbidities that are highly prevalent in older adults, such as COPD, hypertension, cardiovascular disease, and diabetes, have been associated with CFRs upwards of 8% compared to an estimate of 1.4% in people without comorbidities [@doi:10.1111/jgs.16472; @url:https://www.who.int/publications/i/item/report-of-the-who-china-joint-mission-on-coronavirus-disease-2019-(covid-19)]. <!-- will this url work for Manubot?-->
Therefore, both age and health are important considerations when predicting the impact of COVID-19 on a population [@doi:10.1073/pnas.2008760117].
However, other associations may exist, such as patients with sepsis having higher SOFA scores -- in fact, SOFA was developed for the assessment of organ failure in the context of sepsis, and the acronym originally stood for Sepsis-Related Organ Failure Assessment [@doi:10.1007/BF01709751; @doi:10.1001/jama.2016.0287].
Additionally, certain conditions are likely to be more prevalent under or exacerbated by social conditions, especially poverty, as is discussed further below.

#### Ancestry

A number of studies have suggested associations between individuals' racial and ethnic backgrounds and their COVID-19 risk.
In particular, Black Americans are consistently identified as carrying a higher burden of COVID-19 than white Americans [@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1], with differences in the rates of kidney complications from COVID-19 particularly pronounced [@doi:10.1016/j.kint.2020.05.006].
Statistics from a number of cities indicate significant discrepancies between the proportion of COVID-19 cases and deaths in Black Americans relative to their representation in the general population [@doi:10/ggv494].
In addition to Black Americans, disproportionate harm and mortality from COVID-19 has also been noted in Latino/Hispanic Americans and in Native American and Alaskan Native communities, including the Navajo nation [@doi:10.1001/jama.2020.8598; @doi:10.1136/bmj.m1483; @doi:10.1111/jrh.12451; https://www.nytimes.com/2020/04/09/us/coronavirus-navajo-nation.html?searchResultPosition=10; @url:https://www.nytimes.com/interactive/2020/07/05/us/coronavirus-latinos-african-americans-cdc-data.html; @doi:10.1093/tbm/ibaa055; @url:https://journalhosting.ucalgary.ca/index.php/jisd/article/view/70753/54416].
In Brazil, indigenous communities likewise carry an increased burden of COVID-19 [@doi:10.1126/science.abc0073].
In the United Kingdom, nonwhite ethnicity (principally Black or South Asian) was one of several factors found to be associated with a higher risk of death from COVID-19 [@doi:10.1038/s41586-020-2521-4].

From a genetic standpoint, it is highly unlikely that ancestry itself predisposes individuals to contracting COVID-19 or to experiencing severe COVID-19 outcomes.
Examining human genetic diversity indicates variation over a geographic continuum, and that most human genetic variation is associated with the African continent [@doi:10.1038/ng1438].
African-Americans are also a more genetically diverse group relative to European-Americans, with a large number of rare alleles and a much smaller fraction of common alleles identified in African-Americans [@doi:10.1146/annurev.genom.9.081307.164258].
Therefore, the idea that African ancestry (at the continent level) might convey some sort of genetic risk for severe COVID-19 contrasts with what is known about worldwide human genetic diversity [@doi:10.1126/science.abd4842].
The possibility for genetic variants that confer some risk or some protection remains possible, but has not been widely explored, especially at a global level.
Research in Beijing of a small number (n=80) hospitalized COVID-19 patients revealed an association between severe COVID-19 outcomes and homozygosity for an allele in the interferon-induced transmembrane protein 3 (IFITM3) gene, which was selected as a candidate because it was previously found to be associated with influenza outcomes in Chinese patients [@doi:10.1093/infdis/jiaa224].
Genetic factors may also play a role in the risk of respiratory failure for COVID-19 [@doi:10.1056/NEJMoa2020283; @doi:10.1093/gerona/glaa131; @doi:10.1101/2020.06.16.155101].
However, genetic variants associated with outcomes within ancestral groups are far less surprising than genetic variants explaining outcomes between groups.
Alleles in _ACE2_ and _TMPRSS2_ have been identified that vary in frequency among ancestral groups [@doi:10.1186/s12916-020-01673-z], but whether these variants are associated with COVID-19 susceptibility has not been explored.

Instead, examining patterns of COVID-19 susceptibility on a global scale that suggest that social factors are of primary importance in predicting mortality.
Reports from several sub-Saharan African countries have indicated that the effects of the COVID-19 pandemic have been less severe than expected based on the outbreaks in China and Italy.
In Kenya, for example, estimates of national prevalence based on testing blood donors for SARS-CoV-2 antibodies were consistent with 5% of Kenyan adults having recovered from COVID-19 [@doi:10.1101/2020.07.27.20162693].
This high seroprevalence of antibodies lies in sharp contrast to the low number of COVID-19 fatalities in Kenya, which at the time was 71 out of 2093 known cases [@doi:10.1101/2020.07.27.20162693].
Likewise, a serosurvey of health care workers in Blantyre City, Malawi reported an adjusted antibody prevalence of 12.3%, suggesting that the virus had been circulating more widely than thought and that the death rate was up eight times lower than models had predicted [@doi:10.1101/2020.07.30.20164970].
While several possible hypotheses for the apparent reduced impact of COVID-19 on the African continent are being explored, such as young demographics in many places [@doi:10.1126/science.369.6505.756], these reports present a stark contrast to the severity of COVID-19 in Americans and Europeans of African descent.
Additionally, ethnic minorities in the United Kingdom also tend to be younger than white British living in the same areas, yet the burden of COVID-19 is still more serious for minorities, especially people of Black Caribbean ancestry, both in absolute numbers and when controlling for age and location [@url:https://www.ifs.org.uk/inequality/chapter/are-some-ethnic-groups-more-vulnerable-to-covid-19-than-others/].
Furthermore, the groups in the United States and United Kingdom that have been identified as carrying elevated COVID-19 burden, namely Black American, indigenous American, and Black and South Asian British, are quite distinct in their position on the human ancestral tree.
What is shared across these groups is instead a history of disenfranchisement under colonialism and ongoing systematic racism.
A large analysis of over 11,000 COVID-19 patients hospitalized in 92 hospitals across U.S. states revealed that Black patients were younger, more often female, more likely to be on Medicaid, more likely to have comorbidities, and came from neighborhoods identified as more economically deprived than white patients [@doi:10.1001/jamanetworkopen.2020.18039].
This study reported that when these factors were accounted for, the differences in mortality between Black and white patients were no longer significant.
Thus, the current evidence suggests that the apparent correlations between ancestry and health outcomes must be examined in the appropriate social context.

### Environmental Influences on Susceptibility

#### Exposure to COVID-19

Social distancing has emerged as one of the main social policies used to manage the COVID-19 epidemic in many countries.
Many governments issued stay-at-home orders, especially in the initial months of the crisis.
However, data clearly indicates that these orders impacted different socioeconomic groups differently.
In U.S. counties with and without stay-at-home orders, smartphone tracking indicated a significant decrease in the general population's mobility in April relative to February through March of 2020 (-52.3% and -60.8%, respectively) [@doi:10.1101/2020.05.03.20084624].
A linear relationship was observed between counties' reduction in mobility and their wealth and health, as measured by access to health care, food security, income, space, and other factors [@doi:10.1101/2020.05.03.20084624].
Counties with greater reductions in mobility were also found to have much lower child poverty and household crowding and to be more racially segregated, and to have fewer youth and more elderly residents [@doi:10.1101/2020.05.03.20084624].
Similar associations between wealth and decreased mobility were observed in cellphone GPS data from Colombia, Indonesia, and Mexico collected between January and May 2020 [@arXiv:2006.15195], as well as in a very large data set from several US cities [@doi:10.1038/s41586-020-2923-3].
These disparities in mobility are likely to be related to the role that essential workers have played during the pandemic.
Essential workers are disproportionately likely to be female, people of color, immigrants, and to have an income below 200% of the poverty line [@url:https://mronline.org/wp-content/uploads/2020/06/2020-04-Frontline-Workers.pdf].
Black Americans in particular are over-represented among front-line workers and in professions where social distancing is infeasible [@doi:10.1002/ajim.23145].
Health care work in particular presents an increased risk of exposure to SARS-CoV-2 [@doi:10.1002/ajim.23145; @doi:10.1371/journal.pone.0232452; @url:https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/causesofdeath/bulletins/coronaviruscovid19relateddeathsbyoccupationenglandandwales/deathsregistereduptoandincluding20april2020; @url:https://www.ons.gov.uk/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/articles/whichoccupationshavethehighestpotentialexposuretothecoronaviruscovid19/2020-05-11; @url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/892085/disparities_review.pdf].
In the United Kingdom, (South) Asians are more likely than their white counterparts to be medical professionals [@url:https://www.ifs.org.uk/inequality/chapter/are-some-ethnic-groups-more-vulnerable-to-covid-19-than-others/], although BAME medical professionals are still disproportionately represented in the proportion of National Health Service staff deaths [@url:https://www.hsj.co.uk/exclusive-deaths-of-nhs-staff-from-covid-19-analysed/7027471.article].
Similar trends have been reported for nurses, especially nurses of color, in the United States [@url:https://act.nationalnursesunited.org/page/-/files/graphics/0920_Covid19_SinsOfOmission_Data_Report.pdf].
Furthermore, beyond the risks associated with work itself, use of public transportation may also impact COVID-19 risk [@url:http://www.nber.org/papers/w27407].
The socioeconomic and racial/ethnic gaps in who is working on the front lines of the pandemic make it clear that socioeconomic privilege is likely to decrease the probability of exposure to SARS-CoV-2.

Increased risk of exposure can also arise outside the workplace.
Nursing homes and skilled nursing facilities received attention early on as high-risk locations for COVID-19 outbreaks [@doi:10.1001/jama.2020.11642].
Prisons and detention centers also confer a high risk of exposure or infection [@doi:10.1001/jamainternmed.2020.1856; @doi:10.1001/jama.2020.12528].
Populations in care facilities are largely older adults, and in the United States, incarcerated people are more likely to be male and persons of color, especially Black [@url:https://www.issuelab.org/resources/695/695.pdf].
Additionally, multi-generational households are less common among non-Hispanic white Americans than people of other racial and ethnic backgrounds [@doi:10.1111/j.1751-9020.2010.00306.x], increasing the risk of exposure for more susceptible family members.
Analysis suggests that household crowding may also be associated with increased risk of COVID-19 exposure [@doi:10.1101/2020.05.03.20084624], and household crowding is associated with poverty [@doi:10.1037/12057-014].
Forms of economic insecurity like housing insecurity, which is associated with poverty and more pronounced in communities subjected to racism [@doi:10.7916/D8WH2W9T; @doi:10.1093/socpro/spv025], would be likely to increase household crowding and other possible sources of exposure.
As a result, facets of systemic inequality such as mass incarceration of Black Americans and poverty are likely to increase the risk of exposure outside of the workplace.

#### Severity of COVID-19 Following Exposure

Following exposure to SARS-CoV-2, the likelihood that an individual develops COVID-19 and the severity of the disease presentation can be influenced by a number of social factors.
As discussed above, a number of patient characteristics are associated with the likelihood of severe COVID-19 symptoms.
In some cases, these trends run counter to those expected given rates of exposure: for example, although women are more likely to be exposed, men are more likely to be diagnosed with, hospitalized from, or die from COVID-19 [@doi:10.1371/journal.ppat.1008570].
In the case of comorbid conditions and racial/ethnic demographics, however, social factors are highly likely to modulate or at least influence the apparent association between these traits and the increased risk from COVID-19.
In particular, the comorbidities and racial/ethnic correlates of severe COVID-19 outcomes suggest that poverty confers additional risk for COVID-19.

In order to explore the relationship between poverty and COVID-19 outcomes, it is necessary to consider how poverty impacts biology.
In particular, we focus on the United States and the United Kingdom.
Comorbidities that increase risk for COVID-19, including obesity, type II diabetes, hypertension, and cardiovascular disease, are known to be intercorrelated [@doi:10/bpp37d].
Metabolic conditions related to heightened inflammation, like obesity, type II diabetes, and hypertension, are more strongly associated with negative COVID-19 outcomes than other comorbid conditions, such as chronic heart disease [@doi:10.2337/dbi19-0023].
As discussed above, dysregulated inflammation characteristic of cytokine release syndrome is one of the greatest concerns for COVID-19-related death.
Therefore, it is possible that chronic inflammation characteristic of these metabolic conditions predisposes patients to COVID-19-related death [@doi:10.2337/dbi19-0023].
The association between these diseases and severe COVID-19 outcomes is a concern from a health equity perspective because poverty exposes people to "obesogenic" conditions [@doi:10/gddg84] and is therefore unsurprisingly associated with higher incidence of obesity and associated disorders [@doi:10.5604/12321966.1120608].
Furthermore, cell phone GPS data suggests that lower socioeconomic status may also be associated with decreased access to healthy food choices during the COVID-19 pandemic [@doi:10.1002/oby.22940; @doi:10.1002/oby.22993], suggesting that health-related risk factors for COVID-19 may be exacerbated as the pandemic continues [@doi:10.1016/j.pcad.2020.07.002].
Chronic inflammation is a known outcome of chronic stress (e.g., [@doi:10.1067/mai.2000.110163; @doi:10.1037/0278-6133.21.6.531; @doi:10.1037/a0025536; @doi:10.1111/j.1467-8721.2006.00450.x]).
Therefore, the chronic stress of poverty is likely to influence health broadly (as summarized in [@doi:10.1038/scientificamerican1205-92]) and especially during the stress of the ongoing pandemic.

A preprint [@doi:10.1101/2020.04.05.20054502] provided observational evidence that geographical areas in the United States that suffer from worse air pollution by fine particulate matter have also suffered more COVID-19 deaths per capita, after adjusting for demographic covariates.
Although lack of individual-level exposure data and the impossibility of randomization make it difficult to elucidate the exact causal mechanism, this finding would be consistent with similar findings for all-cause mortality (e.g., [@doi:10.1073/pnas.1803222115]).
Exposure to air pollution is associated with both poverty (e.g., [@doi:10.3390/ijerph15061114]) and chronic inflammation [@doi:10.1016/j.neubiorev.2018.06.002].
Other outcomes of environmental racism, such as the proximity of abandoned uranium mines to Navajo land, can also cause respiratory illnesses and other health issues [@url:https://journalhosting.ucalgary.ca/index.php/jisd/article/view/70753/54416].
Similarly, preliminary findings indicate that nutritional status (e.g., vitamin D deficiency [@doi:10.1210/clinem/dgaa733]) may be associated with COVID-19 outcomes, and reduced access to grocery stores and fresh food often co-occurs with environmental racism [@url:https://journalhosting.ucalgary.ca/index.php/jisd/article/view/70753/54416; @doi:10.1056/NEJMp2021264].
Taken together, the evidence suggests that low-income workers who face greater exposure to SARS-CoV-2 due to their home or work conditions are also more likely to face environmental and social stressors associated with increased inflammation, and therefore with increased risk from COVID-19.
In particular, structural racism can play an important role on disease severity after SARS-CoV-2 exposure, due to consequences of racism which include an increased likelihood of poverty and its associated food and housing instability.
COVID-19 can thus be considered a "syndemic", or a synergistic interaction between several epidemics [@doi:10.1002/ajhb.23482].
As a result, it is not surprising that people from minoritized backgrounds and/or with certain pre-existing conditions are more likely to suffer severe effects of COVID-19, but these "risk factors" are likely to be causally linked to poverty [@doi:10.1001/jama.2020.26443].

#### Access to Treatment

Finally, COVID-19 outcomes can be influenced by access to healthcare.
Receiving care for COVID-19 can, but does not always, include receiving a positive test for the SARS-CoV-2 virus.
For example, it is common to see treatment guidelines for suspected cases regardless of whether the presence of SARS-CoV-2 has been confirmed (e.g., [@doi:10.1016/j.ajem.2020.03.036]).
Whether and where a patient is diagnosed can depend on their access to testing, which can vary both between and within countries.
In the United States, it is not always clear whether an individual will have access to free testing [@url:https://www.npr.org/sections/health-shots/2020/06/19/880543755/insurers-may-only-pay-for-coronavirus-tests-when-theyre-medically-necessary; @url:https://www.commonwealthfund.org/blog/2020/private-health-insurance-coverage-covid-19-public-health-emergency].
The concern has been raised that more economic privilege is likely to correspond to increased access to testing, at least within the United States [@doi:10.1016/j.jaad.2020.04.046].
This is supported by the fact that African Americans seem to be more likely to be diagnosed in the hospital, while individuals from other groups were more likely to have been diagnosed in ambulatory settings in the community [@doi:10.1377/hlthaff.2020.00598].
Any delays in treatment are a cause for concern [@doi:10.1016/j.jaad.2020.04.046], which could potentially be increased by an inability to acquire testing because in the United States, insurance coverage for care received can depend on a positive test [@url:https://www.hrsa.gov/coviduninsuredclaim/frequently-asked-questions].

Another important question is whether patients with moderate to severe cases are able to access hospital facilities and treatments, to the extent that they have been identified.
Early findings from China as of February 2020 suggested the COVID-19 mortality rate to be much lower in the most developed regions of the country [@doi:10/ggqscd], although reported mortality is generally an estimate of CFR, which is dependent on rates of testing.
Efforts to make treatment accessible for all confirmed and suspected cases of COVID-19 in China are credited with expanding care to people with fewer economic resources [@doi:10.1038/s41591-020-0823-6].
In the United States, access to healthcare varies widely, with certain sectors of the workforce less likely to have health insurance; many essential workers in transportation, food service, and other frontline fields are among those likely to be uninsured or underinsured [@doi:10.1016/j.jaad.2020.04.046].
As of 2018, Hispanic Americans of all races were much less likely to have health insurance than people from non-Hispanic backgrounds [@url:https://www.census.gov/content/dam/Census/library/publications/2019/demo/p60-267.pdf].
Therefore, access to diagnostics and care prior to the development of severe COVID-19 is likely to vary depending on socioeconomic and social factors, many of which overlap with the risks of exposure and of developing more severe COVID-19 symptoms.
This discrepancy ties into concerns about broad infrastructural challenges imposed by COVID-19.
A major concern in many countries has been the saturation of healthcare systems due to the volume of COVID-19 hospitalizations (e.g., [@doi:10.1073/pnas.2004064117]).
Similarly, there have been shortages of supplies such as ventilators that are critical to the survival of many COVID-19 patients, leading to extensive ethical discussions about how to allocate limited resources among patients [@doi:10.19044/esj.2020.v16n21p24; @doi:10.7249/PEA228-1; @arxiv:2008.00374; @doi:10.1016/j.surg.2020.04.044].
Although it is generally considered unethical to consider demographic factors such as age, sex, race, or ethnicity while making such decisions, and ideally this information would not be shared with triage teams tasked with allocating limited resources among patients [@doi:10.1080/15265161.2020.1764137], there are substantial concerns about implicit and explicit biases against older adults [@doi:10.1111/jgs.16539], premature infants [@doi:10.1111/apa.15334], and people with disabilities or comorbidities [@doi:10.1080/15265161.2020.1764138; @doi:10.2105/AJPH.2020.305837; @doi:10.1080/15265161.2020.1764137].
Because of the greater burden of chronic disease in populations subjected to systemic racism, algorithms intended to be blind to race and ethnicity could, in fact, reinforce systemic inequalities caused by structural racism [@doi:10.1080/15265161.2020.1777347; @doi:10.1016/j.jnma.2020.05.010; @doi:10.1056/NEJMp2023616].
Because of this inequality, it has been argued that groups facing health disparities should be prioritized by these algorithms [@doi:10.1080/15265161.2020.1795538].
This approach would carry its own ethical concerns, including the fact that many resources that need to be distributed do not have well-established risks and benefits [@doi:10.1080/15265161.2020.1795538].

As the pandemic has progressed, it has become clear that ICU beds and ventilators are not the only limited resources that needs to be allocated, and, in fact, the survival rate for patients who receive mechanical ventilation is lower than these discussions would suggest [@doi:10.1136/medethics-2020-106460].
Allocation of interventions that may reduce suffering, including palliative care, has become critically important [@doi:10.1136/medethics-2020-106460; @doi:10.1080/15265161.2020.1788663].
The ambiguities surrounding the risks and benefits associated with therapeutics that have been approved under emergency use authorizations also present ethical concerns related to the distribution of resources [@doi:10.1080/15265161.2020.1795538].
For example, remdesivir, discussed above, is currently available for the treatment of COVID-19 under compassionate use guidelines and through expanded access programs, and in many cases has been donated to hospitals by Gilead [@doi:10.1016/j.mayocp.2020.06.016; @doi:10.1080/15265161.2020.1795529].
Regulations guiding the distribution of drugs in situations like these typically do not address how to determine which patients receive them [@doi:10.1080/15265161.2020.1795529].
Prioritizing marginalized groups for treatment with a drug like remdesivir would also be unethical because it would entail disproportionately exposing these groups to a therapeutic that may or not be beneficial [@doi:10.1080/15265161.2020.1795538].
On the other hand, given that the drug is one of the most promising treatments available for many patients, using a framework that tacitly feeds into structural biases would also be unethical.
At present, the report prepared for the Director of the CDC by Ethics Subcommittee of the CDC fails to address the complexity of this ethical question given the state of structural racism in the United States, instead stating that "prioritizing individuals according to their chances for short-term survival also avoids ethically irrelevant considerations, such as race or socioeconomic status" [@url:https://www.cdc.gov/about/advisory/pdf/VentDocument_Release.pdf].
In many cases, experimental therapeutics are made available only through participation in clinical trials [@doi:10.1080/15265161.2020.1795541].
However, given the history of medical trials abusing minority communities, especially Black Americans, there is a history of unequal representation in clinical trial enrollment [@doi:10.1080/15265161.2020.1795541].
As a result, the standard practice of requiring enrollment in a clinical trial in order to receive experimental treatment may also reinforce patterns established by systemic racism.
<!--Risk of comorbid health conditions associated with more severe outcomes may be influenced by long-term damage caused by chronic stress related to traumatic social experiences [@doi:10.1007/s12170-013-0338-5], perhaps mediated by cardiovascular risk factors [@doi:10.1101/2020.05.10.20097253], although the effects of chronic stress have not yet been researched in the specific case of COVID-19 disparities.-->

#### Access to and Representation in Clinical Trials

Experimental treatments are often made available to patients primarily or even exclusively through clinical trials.
The advantage of this approach is that clinical trials are designed to collect rigorous data about the effects of a treatment on patients.
The disadvantage is that access to clinical trials is not equal among all people who suffer from a disease.
Two important considerations that can impact an individual's access to clinical trials are geography and social perceptions of clinical trials.
For the first, the geographic distribution of trial recruitment efforts are typically bounded and can vary widely among difference locations, and for the second, the social context of medical interactions can impact strategies for and the success of outreach to different communities.
Differential access to clinical trials raises concerns because it introduces biases that can influence scientific and medical research on therapeutics and prophylactics broadly.
Concerns about bias in clinical trials need to address both trial recruitment and operation.
In the present crisis, such biases are particularly salient because COVID-19 is a disease of global concern.
Treatment is needed by people all over the world, and clinical research that characterizes treatment outcomes in a variety of populations is critically important.

Global representation in clinical trials is important to ensuring that experimental treatments are available equally to COVID-19 patients who may need them.
The advantage to a patient of participation in a clinical trial is that they may receive an experimental treatment they would not have been able to access otherwise.
The potential downsides of participation include that the efficacy and side effects of such treatments are often poorly characterized and that patients who enroll in clinical trials will in some cases run the risk of being assigned to a placebo condition where they do not receive the treatment but miss out on opportunities to receive other treatments.
The benefits and burdens of clinical trials therefore need to be weighed carefully to ensure that they don't reinforce existing health disparities.
The WHO Director‐General Tedros Adhanom Ghebreyesus stated his condemnation of utilizing low and middle income countries as test subjects for clinical trials, yet having highly developed countries as the majority of clinical trial representation is also not the answer [@doi:10.1002/eahr.500055].
Figure @fig:ebm-map showcases two choropleths detailing COVID-19 clinical trial recruitment by country.
China, the United States, and France are among the countries with the most clinical trial recruiting for trials with single-country enrollment.
Many countries have little to no clinical trial recruiting, with the continents of Africa and South America much less represented than Asia, Europe, and North America.
Trials that recruit across multiple countries do appear to broaden geographic representation, but these trials seem to be heavily dominated by the United States and European Union.

![
**Geographic distribution of COVID-19 clinical trials.**
The density of clinical trials is reported at the country level.
As of December 31, 2020, there are 6,987 trials in the University of Oxford Evidence-Based Medicine Data Lab's COVID-19 TrialsTracker [@doi:10.5281/zenodo.3732709], of which 3,962 are interventional.
The top figure demonstrates the density of interventional trials recruiting only from a singular country, while the bottom shows the distribution of recruitment for interventional trials that involve more than one country.
](https://github.com/greenelab/covid19-review/raw/0e36a517bc88e54459c2b6ed3a4803e621267045/ebmdatalab/ebmdatalab-map.png "COVID-19 clinical trials"){#fig:ebm-map secno=1}

A few different concerns arise from this skewed geographic representation in clinical trial recruitment.
First, treatments such as remdesivir that are promising but primarily available to clinical trial participants are unlikely to be accessible by people in many countries.
Second, it raises the concern that the findings of clinical trials will be based on participants from many of the wealthiest countries, which may lead to ambiguity in whether the findings can be extrapolated to COVID-19 patients elsewhere.
Especially with the global nature of COVID-19, equitable access to therapeutics and vaccines has been a concern at the forefront of many discussions about policy (e.g., [@doi:10/ggq7mf], yet data like that shown in Figure @fig:ebm-map demonstrates that accessibility is likely to be a significant issue.
Another concern with the heterogeneous international distribution of clinical trials is that the governments of countries leading these clinical trials might prioritize their own populations once vaccines are developed, causing unequal health outcomes [@doi:10.1001/jama.2020.6641].
Additionally, even within a single state in the United States (Maryland), geography was found to influence the likelihood of being recruited into or enrolled in a clinical trial, with patients in under-served rural areas less likely to enroll [@doi:10.1016/j.cdp.2005.12.001].
Thus, geography both on the global and local levels may influence when treatments and vaccines are available and who is able to access them.
Efforts such as the African Union's efforts to coordinate and promote vaccine development [@doi:10/fgzk] are therefore critical to promoting equity in the COVID-19 response.


Even when patients are located within the geographic recruitment area of clinical trials, however, there can still be demographic inequalities in enrollment.
When efforts are made to ensure equal opportunity to participate in clinical trials, there is no significant difference in participation among racial/ethnic groups [@doi:10.1002/cncr.28483].
However, within the United States, real clinical trial recruitment numbers have indicated for many years that racial minorities, especially African-Americans, tend to be under-represented (e.g., [@doi:10.1001/jama.291.22.2720; @doi:10.1245/s10434-007-9500-y; @doi:10.1089/jwh.2010.2469; @doi:10.1007/s11926-018-0728-2]).
This trend is especially concerning given the disproportionate impact of COVID-19 on African-Americans.
Early evidence suggests that the proportion of Black, Latinx, and Native American participants in clinical trials for drugs such as remdesivir is much lower than the representation of these groups among COVID-19 patients [@doi:10.1056/NEJMp2021971].

One proposed explanation for differences among racial and ethnic groups in clinical trial enrollment refers to different experiences in healthcare settings.
While some plausible reasons for the disparity in communication between physicians and patients could be a lack of awareness and education, mistrust in healthcare professionals, and a lack of health insurance [@doi:10.1002/cncr.28483], a major concern is that patients from certain racial and ethnic groups are marginalized even while seeking healthcare.
In the United States, many patients experience "othering" from physicians and other medical professionals due to their race or other external characteristics such as gender (e.g., [@doi:10.1207/S15327027HC1602_7]).
Many studies have sought to characterize implicit biases in healthcare providers and whether they affect their perceptions or treatment of patients.
A systematic review that examined 37 such studies reported that most (31) identified racial and/or ethnic biases in healthcare providers in many different roles, although the evidence about whether these biases translated to different attitudes towards patients was mixed [@doi:10.1016/j.socscimed.2017.05.009], with similar findings reported by a second systematic review [@doi:10.1111/acem.13214].
However, data about real-world patient outcomes are very limited, with most studies relying on clinical vignette-based exercises [@doi:10.1016/j.socscimed.2017.05.009], and other analyses suggest that physician implicit bias could impact the patient's perception of the negativity/positivity of the interaction regardless of the physician's explicit behavior towards the patient [@doi:10.1016/j.jesp.2009.11.004].
Because racism is a common factor in both, negative patient experiences with medical professionals are likely to compound other issues of systemic inequality, such as a lack of access to adequate care, a lack of insurance, or increased exposure to SARS-CoV-2 [@doi:10.1542/peds.2020-003657].
Furthermore, the experience of being othered is not only expected to impact patients' trust in and comfort with their provider, but also may directly impact whether or not the patient is offered the opportunity to participate in a clinical trial at all.
Some studies suggest communication between physicians and patients impacts whether or not a physician offers a patient participation in a clinical trial.
For example, researchers utilized a linguistic analysis to assess mean word count of phrases related to clinical trial enrollment, such as voluntary participation, clinical trial, etc. [@doi:10.1002/cncr.28483].
The data indicated that the mean word count of the entire visit was 1.5 times more for white patients in comparison to Black patients.
In addition, the greatest disparity between white and Black patients' experience was the discussion of risks, with over 2 times as many risk-related words spoken with white patients than Black patients [@doi:10.1002/cncr.28483].
The trends observed for other clinical trials raise the concern that COVID-19 clinical trial information may not be discussed as thoroughly or as often with Black patients compared to white patients.

 These discrepancies are especially concerning given that many COVID-19 treatments are being or are considered being made available to patients prior to FDA approval through Emergency Use Authorizations.
In the past, African-Americans have been over-represented relative to national demographics in use of the FDA's Exception From Informed Consent (EFIC) pathway [@doi:10.1377/hlthaff.2018.0501].
Through this pathway, people who are incapacitated can receive an experimental treatment even if they are not able to consent and there is not sufficient time to seek approval from an authorized representative.
This pathway presents concerns, however, when it is considered in the context of a long history of systematic abuses in medical experimentation where informed consent was not obtained from people of color, such as the Tuskegee syphilis experiments [@doi:10.1067/mem.2003.17].
While the goal of EFIC approval is to provide treatment to patients who urgently need it, the combination of the ongoing legacy of racism in medicine renders this trend concerning.
With COVID-19, efforts to prioritize people who suffer from systemic racism are often designed with the goal of righting some of these inequalities (e.g., [@url:https://www.businessinsider.com/cdc-official-considering-giving-covid-19-vaccine-most-vulnerable-first-2020-10]), but particular attention to informed consent will be imperative in ensuring these trials are ethical given that the benefits and risks of emerging treatments are still poorly characterized.
Making a substantial effort to run inclusive clinical trials is also important because of the possibility that racism could impact how a patient responds to a treatment.
For example, as discussed above, dexamethasone has been identified as a promising treatment for patients experiencing cytokine release syndrome, but the mechanism of action is tied to the stress response.
A study from 2005 reported that Black asthma patients showed reduced responsiveness to dexamethasone in comparison to white patients and suggested Black patients might therefore require higher doses of the drug [@doi:10.1378/chest.127.2.571].
In the context of chronic stress caused by systemic racism, this result is not surprising: chronic stress is associated with dysregulated production of glucocorticoids [@doi:10.1146/annurev.physiol.67.040403.120816] and glucocorticoid receptor resistance [@doi:10.1073/pnas.1118355109].
However, it underscores the critical need for treatment guidelines to take into account differences in life experience, which would be facilitated by the recruitment of patients from a wide range of backgrounds.
Attention to the social aspects of clinical trial enrollment must therefore be an essential component of the medical research community's response to COVID-19.

<!--Table for future version: ### Indirect Effects of the Pandemic
Besides the direct harms caused by infection, many populations are in indirect risk of serious harm due to the social and economic effects of the pandemic and of the efforts needed to fight it.
These might include individuals with substance use disorder [@doi:10.7326/M20-1141], victims of natural disasters [@doi:10.1038/s41558-020-0804-2], and victims of human trafficking [@doi:10.1016/j.eclinm.2020.100409].
The pandemic could also delay the fight against other major infectious diseases, such as HIV, malaria and tuberculosis, potentially leading to a further increase of mortality [@doi:10.25561/78670].
Although they are beyond the scope of this paper, further research is needed in order to prevent these harms.
[There are already some policy recommendations in literature that we cite - we should probably direct folks there. I think we probably want a stronger synthesis and closing than this] -->

### Conclusions and Future Directions

As the COVID-19 pandemic evolves, the scientific community's response will be critical for identifying potential pharmacological and biotechnological developments that may aid in combating the virus and the disease it causes.
However, this global crisis highlights the importance of mounting a response based on collaboration among a wide variety of disciplines.
Understanding the basic science of the virus and its pathogenesis is imperative for identifying and envisioning possible diagnostic and therapeutic approaches; understanding how social factors can influence outcomes and shape implementation of a response is critical to disseminating any scientific advancements.
Summarizing such a complex and ever-changing topic presents a number of challenges.
This review represents the effort of over 50 contributors <!-- TO DO: check with final number -->to distill and interpret the available information.
However, this text represents a dynamic and evolving document, and we welcome continued contributions from all researchers who have insights into how these topics intersect.
A multidisciplinary perspective is critical to understanding this evolving crisis, and in this review we seek to use open science tools to coordinate a response among a variety of researchers.
We intend to publish additional updates as the situation evolves.

<!-- Additional info on inflammation removed from pathogenesis manuscript, can use here if useful
Inflammation is one of the most visible components of the immune response, as it is responsible for the hallmarks of injury, such as pain, heat, and swelling [@doi:10.18632/oncotarget.23208].
The inflammatory response has received particular attention for its role in both a healthy response to infection and a pathogenic one.
In response to injury or to signaling by pattern recognition receptors indicating the detection of a molecular pattern associated with a pathogen or foreign body, the immune system stimulates leukocytes that travel to the site of the threat, where they then produce cytokines [@doi:10.18632/oncotarget.23208].
Elevated levels of inflammation over the long-term are associated with many chronic health conditions, including type 2 diabetes, dementia and Alzheimer's, and arthritis [@doi:10.1038/s41591-019-0675-0].
Some notable pro-inflammatory cytokines include the interleukins IL-1&beta; and IL-6 and tumor necrosis factor &alpha; (TNF-&alpha;) [@doi:10.1097/AIA.0b013e318034194e].
A number of interleukins and interferons play anti-inflammatory roles, and receptors or receptor antagonists for inflammatory cytokines are also important for regulating inflammation [@doi:10.1097/AIA.0b013e318034194e].
IL-10 is an anti-inflammatory cytokine of particular note because it regulates the expression of TNF-&alpha;, IL-1, and IL-6 [@doi:10.1097/AIA.0b013e318034194e].

Symptoms of a disease can be caused by a pathogen, but they can also be caused by the immune system's reaction to the pathogen.
Cytokines are a diverse group of small proteins that play an important role in intercellular signaling [@doi:10.1128/MMBR.05015-11].
They can be both pro- and anti-inflammatory, meaning they can either stimulate or inhibit the production of additional cytokines [@doi:10.1128/MMBR.05015-11; @doi:10.1097/AIA.0b013e318034194e], with anti-inflammatory cytokines playing an immunoregulatory role complementary to the cascading effect of pro-inflammatory cytokines [@doi:10.1128/MMBR.05015-11; @doi:10.1097/AIA.0b013e318034194e].
When the pro- and anti-inflammatory responses are both commensurate with the threat posed, the immune system drives a shift back to homeostasis [@doi:10.1111/imr.12671].
-->


## An Open-Publishing Response to the COVID-19 Infodemic

### ABSTRACT

The COVID-19 pandemic catalyzed the rapid dissemination of papers and preprints investigating the disease and its associated virus, SARS-CoV-2.
The multifaceted nature of COVID-19 demands a multidisciplinary approach, but the urgency of the crisis combined with the need for social distancing measures present unique challenges to collaborative science.
We applied a massive online open publishing approach to this problem using Manubot.
Through GitHub, collaborators summarized and critiqued COVID-19 literature, creating a review manuscript.
Manubot automatically compiled citation information for referenced preprints, journal publications, websites, and clinical trials.
Continuous integration workflows retrieved up-to-date data from online sources nightly, regenerating some of the manuscript's figures and statistics.
Manubot rendered the manuscript into PDF, HTML, LaTeX, and DOCX outputs, immediately updating the version available online upon the integration of new content.
Through this effort, we organized over 50 scientists from a range of backgrounds who evaluated over 1,500 sources and developed seven literature reviews.
While many efforts from the computational community have focused on mining COVID-19 literature, our project illustrates the power of open publishing to organize both technical and non-technical scientists to aggregate and disseminate information in response to an evolving crisis.

### KEYWORDS  {.unnumbered}

COVID-19, living document, open publishing, open-source, data integration, manubot

### INTRODUCTION

Coronavirus Disease 2019 (COVID-19) caused a worldwide public health crisis that has reshaped many aspects of society.
The scientific community has, in turn, devoted significant attention and resources towards COVID-19 and the associated virus, SARS-CoV-2, resulting in the release of data and publications at a rate and scale never previously seen for a single topic.
Over 20,000 articles about COVID-19 were released in the first four months of the pandemic [@doi:10.1053/j.ackd.2020.08.003], causing an "infodemic" [@doi:10.1053/j.ackd.2020.08.003; @doi:10/ggpx67].
The COVID-19 Open Research Dataset (CORD-19) [@arxiv:2004.10706], which was developed in part with the goal of training machine learning algorithms on COVID-19-related text, illustrates the growth of related scholarly literature (Figure @fig:cord19-growth).
This resource was developed by querying several sources for terms related to SARS-CoV-2 and COVID-19, as well as the coronaviruses SARS-CoV-1 and MERS-CoV and their associated diseases [@arxiv:2004.10706].
CORD-19 contained 768929 manuscripts as of 2021-09-20.
Additional curation by CoronaCentral [@doi:10.1073/pnas.2100766118] has produced, at present, a set of over 180,000 publications particularly relevant to COVID-19 and closely related viruses.
Despite many advances in understanding the virus and the disease, there are also downsides to the availability of so much information.
"Excessive publication" has been recognized as a concern for over forty years [@doi:10/d3bmnv] and has been discussed with respect to the COVID-19 literature [@url:https://blogs.sciencemag.org/pipeline/archives/2021/07/19/too-many-papers].
Any effort to synthesize, summarize, and contextualize COVID-19 research will face a vast corpus of potentially relevant material.

![
**Growth of the CORD-19 dataset.**
The number of articles has proliferated, with both traditional and preprint manuscripts in the corpus.
The first release (March 16, 2020) contained 28,000 documents [@arxiv:2004.10706].
As of 2021-09-20, this had increased to 768929 articles.
Of these, 31360 are preprints from _arXiv_, _medRxiv_, and _bioRxiv_.
](https://github.com/greenelab/covid19-review/raw/a8610c481b21e769ef64336a134217b82eacec65/CORD-19/cord19-growth.png "CORD-19 dataset growth"){#fig:cord19-growth secno=1}

Information was released rapidly by both traditional publishers and preprint servers, and many papers faced subsequent scrutiny.
The number of COVID-19 papers retracted may be higher, and potentially much higher, than is typical, although a thorough investigation of this question requires more time to elapse [@doi:10.1080/08989621.2020.1782203; @doi:10.1080/08989621.2020.1793675].
Many papers and preprints are also associated with corrections or expressions of concern^[<https://asapbio.org/preprints-and-covid-19> as well as <https://retractionwatch.com/retracted-coronavirus-covid-19-papers>] [@doi:10.1080/08989621.2020.1793675].
Preprints are released prior to peer review, but some traditional publishing venues have fast-tracked COVID-19 papers through peer review, leading to questions about whether they are held to typical standards [@doi:10.1111/bioe.12772].
Therefore, evaluating the COVID-19 literature requires not only digesting available information but also monitoring subsequent changes.

Because of the fast-moving nature of the topic, many efforts to summarize and synthesize the COVID-19 literature have been undertaken. 
These efforts include newsletters^[<https://depts.washington.edu/pandemicalliance/covid-19-literature-report/latest-reports>] [@doi:10.1080/10872981.2020.1770562], web portals^[<https://outbreaksci.prereview.org>] [@doi:10.1126/science.abc7839] or the now-defunct http://covidpreprints.com^[<https://asapbio.org/preprints-and-covid-19>], comments on preprint servers^[<https://disqus.com/by/sinaiimmunologyreviewproject>] [@doi:10.1038/s41577-020-0319-0], and even a journal^[<https://rapidreviewscovid19.mitpress.mit.edu>].
However, the explosive rate of publication presents challenges for such efforts, many of which are no longer active.
Similarly, many literature reviews have been written on the available COVID-19 literature [@doi:10.1016/j.molmed.2020.02.008; @doi:10.1126/scitranslmed.abc1931; @doi:10.1001/jama.2020.6019; @doi:10.1038/d41591-020-00026-w; @doi:10.1001/jama.2020.12839], but static reviews quickly become outdated as new research is released or existing research is retracted or superseded.
One example is a review of topics in COVID-19 research including vaccine development [@doi:10.1001/jama.2020.12839].
This review was published on July 10, 2020, four days before Moderna released the surprisingly promising results of their phase 1 trial [@doi:10.1056/NEJMoa2022483] that changed expectations surrounding vaccines.
Therefore, the COVID-19 publishing climate presented a challenge where curation of the literature by a diverse group of experts in a format that could respond quickly to high-volume, high-velocity information was desirable.

We therefore sought to develop a platform for scientific discussion and collaboration around COVID-19 by adapting open publishing infrastructure to accommodate the scale of COVID-19 publishing.
Recent advances in open publishing have created an infrastructure that facilitates distributed, version-controlled collaboration on manuscripts [@doi:10.1371/journal.pcbi.1007128].
Manubot [@doi:10.1371/journal.pcbi.1007128] is a collaborative framework developed to adapt open-source software development techniques and version control for manuscript writing.
With Manubot, manuscripts are managed and maintained using GitHub, a popular, online version control interface.
We selected Manubot because it offers several advantages over comparable collaborative writing platforms such as Authorea, Overleaf, Google Docs, Word Online, or wikis [@doi:10.1371/journal.pcbi.1007128].
Citation-by-identifier ensures consistent reference metadata standards that would be difficult to maintain manually in a manuscript with dozens of authors and over 1,500 citations.
Manubot's pull request-based contribution model balances the goals of making the project open to everyone and maintaining scientific accuracy.
All contributions are reviewed, discussed, and formally approved on GitHub before text updates appear in the public-facing manuscript^[<https://greenelab.github.io/covid19-review>].
Continuous integration (CI) seamlessly combines author-produced text and figures with automatically generated and updated statistics and figures derived from external data sources and the manuscript's own content.
In addition, the authors who initially launched this project included Manubot developers who had prior successes using Manubot for massively open and traditional manuscripts.

Collaboration via massively open online papers has been identified as a strategy for promoting inclusion and interdisciplinary thought [@doi:10.5334/kula.63].
However, the Manubot workflow can be intimidating to contributors who are not well-versed in git [@doi:10.5334/kula.63].
The synthesis and discussion of the emerging literature by biomedical scientists and clinicians is imperative to a robust interpretation of COVID-19 research.
Such efforts in biology often rely on What You See Is What You Get tools such as Google Docs, despite the significant limitations of these platforms in the face of excessive publication.
We recognized that the problem of synthesizing the COVID-19 literature lent itself well to the Manubot platform, but that the potential technical expertise required to work with Manubot presented a barrier to domain experts.

Here, we describe the adaptation of Manubot to facilitate collaboration in the extreme case of the COVID-19 infodemic, with the objective of developing a centralized platform for summarizing and synthesizing a massive amount of preprints, news stories, journal publications, and data.
Unlike prior collaborations built on Manubot, most contributors to the COVID-19 collaborative literature review came from biology or medicine.
The members of the COVID-19 Review Consortium consolidated information about the virus in the context of related viruses and to synthesize rapidly emerging literature.
Manubot provided the infrastructure to manage contributions from the community and create a living, scholarly document integrating data from multiple sources.
Its back-end allowed biomedical scientists to sort and distill informative content out of the overwhelming flood of information [@doi:10.1038/s42254-020-0175-7] in order to provide a resource that would be useful to the broader scientific community.
This case study demonstrates the value of open collaborative writing tools such as Manubot to emerging challenges.
Because it is open source software, we were able to adapt and customize Manubot to flexibly meet the needs of COVID-19 review.
Recording the evolution of information over time and assembling a resource that auto-updated in response to the evolving crisis revealed the particular value that Manubot holds for managing rapid changes in scientific thought.

### METHODS

#### Contributor Recruitment and Roles

First, it was necessary to establish Manubot as a platform accessible to researchers with limited experience working version control, given that this is not typically emphasized in biology and medicine [@doi:10.1109/SE4Science.2017.11; @doi:10.1177/2515245918754826; @doi:10.1186/1751-0473-8-7].
Contributors were recruited primarily by word of mouth and on Twitter, and we also collaborated with existing efforts to train early-career researchers.
We invited potential collaborators to contribute a short introduction on a GitHub issue in order to collect information about participants and provide an introduction to working with GitHub issues.
Interested participants were encouraged to contribute in several ways.
One option was to catalog articles of interest as issues.
We developed a standardized set of questions for contributors to consider when evaluating an article following a framework often used for assessing medical literature.
This approach emphasizes examining the methods used, assignment (whether the study was observational or randomized), assessment, results, interpretation, and how well the study extrapolates [@doi:10.5014/ajot.60.4.367].
Contributors were also invited to contribute or edit text using GitHub's pull request system.
These contributions were not strictly defined and could range from minor corrections to punctuation and grammar to large-scale additions of text.
Finally, a small number of contributors (the authors of this paper) contributed technical expertise, either through the development of standardized approaches to the evaluation of papers based on the MAARIE Framework [@isbn:9780781774260], the writing of code to generate manuscript figures, or the addition of features to Manubot.
All of these additions were also submitted as pull requests, either to the COVID-19 review repository or to an external repository, as appropriate.

Each pull request was reviewed and approved by at least one other contributor before being merged into the main branch.
We tagged potential reviewers based on the introductions they had contributed in order to encourage participation.
Authorship was determined based on the Contributor Roles Taxonomy^[<https://casrai.org/credit>].
Due to the permeability of ideas among different sections, contributors to a specific manuscript were recognized with masthead authorship, while all contributors to the project were recognized with consortium authorship on all papers.
Emphasizing the use of issues and pull requests was designed to encourage authors with and without git experience to discuss papers and provide feedback (both formal and informal) on proposed text additions or changes.
We also used the Gitter chat platform^[<https://www.gitter.im>] to promote informal questions and sharing of information among collaborators.

#### Utilization and Expansion of Manubot

Applying Manubot's existing capabilities allowed us to confront several challenges common in large-scale collaborations, such as maintaining a record of contributions that allowed us to allocate credit appropriately or to contact the original author if questions arose.
Additionally, an up-to-date version of the content was available at all times online in HTML^[<https://greenelab.github.io/covid19-review>] or PDF format^[<https://greenelab.github.io/covid19-review/manuscript.pdf>].
This approach also allowed us to minimize the demand on authors to curate and sync bibliographic resources.
Manubot provides the functionality to create a bibliography using digital object identifiers (DOIs), website URLs, or other identifiers such as PubMed identifiers and arXiv IDs.
The author can insert a citation in-line using a format such as `[@doi:10.1371/journal.pcbi.1007128]`.
Manubot then obtains reference metadata, exports the citations as Citation Style Language JSON Data Items, and renders the bibliographic information needed to generate the references section [@doi:10.1371/journal.pcbi.1007128].
This approach allows multiple authors to work on a piece of text without needing to make manual adjustments to the reference lists.

Due to the needs of this project, several new features were implemented in Manubot.
Because of the ever-evolving nature of the COVID-19 crisis, figures and statistics in the text quickly became outdated.
To address this concern, Manubot and GitHub's CI features were used to create figures that integrated online data sources and to dynamically update information, such as the current number of active COVID-19 clinical trials [@individual-pharmaceuticals], within the text of the manuscripts (Figure @fig:manubot-workflow).
GitHub Actions runs a nightly workflow to update these external data and regenerate the statistics and figures for the manuscript.
The workflow uses the GitHub API to detect and save the latest commit of the external data sources that are GitHub repositories^[Vaccines: <https://github.com/owid/covid-19-data>; Clinical Trials: <https://github.com/ebmdatalab/covid_trials_tracker-covid>; Cases and Deaths: <https://github.com/CSSEGISandData/COVID-19>].
It then downloads versioned data from that snapshot of the external repositories and runs bash and Python scripts to calculate the desired statistics and produce the summary figures using Matplotlib [@doi:10.1109/MCSE.2007.55].
The statistics are stored in JSON files that are accessed by Manubot to populate the values of placeholder template variables dynamically every time the manuscript is built.
For instance, the template variable `{{ebm_trials_results}}` in the manuscript is replaced by the actual number of clinical trials with results, 98.
The template variables also include versioned URLs to the dynamically updated figures.
The JSON files and figures are stored in the `external-resources` branch of the GitHub repository, providing versioned storage.
The GitHub Actions workflow automatically adds and commits the new JSON files and figures to the `external-resources` branch every time it runs, and Manubot uses the latest version of these resources when it builds the manuscript.
The GitHub Actions  workflow file is available online^[<https://github.com/greenelab/covid19-review/blob/master/.github/workflows/update-external-resources.yaml>], as are the scripts^[<https://github.com/greenelab/covid19-review/tree/external-resources>].
The Python package versions are also available^[<https://github.com/greenelab/covid19-review/blob/external-resources/environment.yml>].<!-- To Do: These files are archived with [Zenodo?](...). -->

![
**COVID-19 review GitHub repository organization and workflows.**
Manubot uses CI to combine author-contributed content with automatically updated information from outside sources.
A nightly workflow updates figures and statistics derived from external resources.
Authors write text and add figures to the `master` branch (starred) via GitHub pull requests.
Manubot generates updated manuscript outputs for each new git commit, integrating the static text and figures with the dynamic statistics and figures and automatically-extracted citation information.
GitHub Pages hosts the latest HTML and PDF versions of the manuscript along with permanent links to prior versions.
](images/covid-19-review-workflow-horizontal.png){#fig:manubot-workflow secno=1}

Another issue identified was the need for standardized citation to clinical trials.
Other researchers identified the same need^[<https://forums.zotero.org/discussion/74933/import-from-clinical-trials-registry> and <https://forums.zotero.org/discussion/77721/add-reference-from-clinical-trials-org>].
Trials that are registered with clinicaltrials.gov receive a unique clinical trial identifier, or "NCT ID."
Because clinical trials are registered long before results are published, referencing clinical trial identifiers was a priority.
Manubot uses the Zotero translation server^[<https://www.zotero.org> and <https://github.com/zotero/translation-server>] to extract citation metadata for some types of citations.
However, Zotero did not support clinical trial identifiers and could not extract relevant metadata from their URLs.
In order to pull clinical trial metadata associated into Manubot, we added Zotero support for these identifiers.
To achieve this, we query clinicaltrials.gov to retrieve XML metadata associated with each identifier using JavaScript^[<https://github.com/zotero/translators/pull/2153>].
This extension enables citing a trial as `@clinicaltrials:NCT04280705` instead of the URL. 
Then, when Manubot requests clinical trial metadata from the Zotero translation server, the response includes the trial sponsors, responsible investigators, title, and summary.
Manubot now supports directly citing hundreds of registered Compact Uniform Resource Identifiers^[<https://identifiers.org>], beyond just the `clinicaltrials` identifier.

Because of the large number of citations used in this manuscript and the fast-moving nature of COVID-19 research, keeping track of retractions, corrections, and notices of concern also became a challenge.
We implemented a new Manubot plugin to support "smart citations" in the HTML build of manuscripts.
The plugin uses the scite [@doi:10.1101/2021.03.15.435418] service to display a badge below any citation with a DOI.
The badge contains a set of icons and numbers that indicate how many times that source has been mentioned, supported, or disputed and whether there have been any important editorial notices.
We were thus able to identify references that needed to be reevaluated by an expert.
This addition was invaluable given the nature of the project, where we were disseminating rapidly evolving information of great consequence from over 1,500 different sources.
The badges also allow readers to ascertain a rough approximation of the reliability of cited sources at a glance.

Because most collaborators were writing and editing text through the GitHub website rather than in a local text editor, we also needed to add spell-checking functionalities to Manubot.
We integrated an existing Pandoc^[<https://pandoc.org>] spell-check extension with AppVeyor CI to automatically post spelling errors as comments in a GitHub pull request.
The comment reported both unique misspelled tokens and all locations where the token was detected.
Project maintainers managed a custom dictionary to allow over 1,500 scientific and technical terms that were not common English words.
Spell-checking also helped standardize the writing style across dozens of authors by detecting features such as British versus American English spellings.
The actual spell-checking was implemented using GNU Aspell^[<http://aspell.net>] and the Pandoc spellcheck filter^[<https://github.com/pandoc/lua-filters/tree/master/spellcheck>].
The filter enables checking only the manuscript text, ignoring URLs and formatting.

Manubot can render a manuscript in several formats that serve different purposes.
Prior to this project, Manubot could use Pandoc to convert the markdown-formatted manuscript to HTML, PDF, and DOCX formats.
We expanded this functionality to export individual sections of the manuscript as separate DOCX files while still rendering the complete manuscript in HTML and PDF formats.
This development was necessary because the manuscript grew so large that it needed to be split into seven separate papers for journal submission while still maintaining shared GitHub discussion across topics.
When exporting an individual section, Manubot customizes the manuscript title, authors, and author contributions to pertain to that specific section.
In addition, we expanded the export formats to include partial LaTeX support via Pandoc.
Pandoc converts the markdown content for an individual section to TeX and the Citation Style Language JSON, which contains reference metadata generated by Manubot, to BibTeX.
We customized a LaTeX template and reformatted the Manubot metadata, such as authors and their affiliations, for the LaTeX template.
The exported TeX file requires manual refinement but contains all manuscript content and most of the formatting.
Because LaTeX is required for manuscript submission in many fields, automating most of the process of converting markdown to a submission-friendly format expands Manubot's potential user base.
Manubot users can write in the simple markdown format, render the manuscript in continuously-updated PDF or interactive HTML formats, and export the manuscript in DOCX or TeX and BibTeX for submission to traditional publishers, taking full advantage of Pandoc's powerful document conversion capabilities and Manubot's automation.

### RESULTS

#### Recruitment and Manuscript Development

![
**Project growth over time.**
The number of authors, word count, and number of references have all grown dramatically from when the project began on March 20, 2020.
As of September 10, 2021, there were 52 authors (including consortia), 1676 references, and 138213 words.
The spike in word count during summer 2020 was caused by erroneous duplication and subsequent removal of a large appendix.
](https://github.com/greenelab/covid19-review/raw/2c853716f1745468c8180fd6da351de2bd8f1c00/analyze-ms-stats/manuscript_stats.png "Project authors, references, and words over time"){#fig:projectstats secno=1}

<!--To Do: make the figure URL dynamic and add commas to large numbers-->

Coverage by _Nature Toolbox_ [@doi:10.1038/d41586-020-00916-6] and an associated tweet^[<https://twitter.com/j_perkel/status/1245454628235309057>] about the project on April 1, 2020 attracted the interest of the scientific community (Figure @fig:projectstats).
Because the GitHub issues and comment systems are similar to other common web commenting systems, authors learned these tools quickly.
The Gitter chat also presented a low barrier to entry.
The manuscript continued to grow throughout the first year and a half of the project in both word count and the number of references (Figure @fig:projectstats).
Though only a fraction of potential contributors contributed to the text included in the manuscripts (Figure @fig:projectstats), many contributors remained engaged over the long term (Figure @fig:projectdots).
Additionally, new contributors continued to join even into the second year of the project.

![
**User contributions to the manuscript text over time.**
The dot size indicates the number of words added or edited each month since March 2020.
The figure does not depict other types of author contributions such as literature summaries, pull request review, visualization, or software.
](https://github.com/greenelab/covid19-review/raw/e12aa314707d5a66f20e1b2179d4b29b3efef0c0/contrib-viz/covid19-review-contribution-dot.png "Project authors, references, and words over time"){#fig:projectdots secno=1}

<!-- To Do: use latest version of the figure after adding it to workflow -->

In order to make the project more accessible, we developed resources explaining how to use GitHub's web interface to develop and edit text for Manubot assuming no prior experience working with version control.
These tutorials explained how to open an issue, open a pull request, and review a pull request^[CONTRIBUTING.md and INSTRUCTIONS.md within the
repository].
Additionally, the framework for evaluating literature was converted into issue templates to simplify the review of new articles.
Articles were classified as _diagnostic_, _therapeutic_, or _other_, with an associated template developed to guide the review of papers and preprints in each category.
A total of 285 new paper issues had been opened as of September 13, 2021.<!--To Do: automate (currently have open and closed issues but not paper issues) -->

The manuscripts produced by the consortium (excluding this one) will be submitted to _mSystems_ as part of a special issue that provides support for continuous updates as more information becomes available.
One has been published and two are available as preprints.
This approach allows for a version of record to be maintained alongside the most recent version, which is always available through GitHub.
These manuscripts cover a wide range of topics including the fundamental biology of SARS-CoV-2 (pathogenesis [@individual-pathogenesis] and evolution), biomedical advances in responding to the virus and COVID-19 (pharmaceuticals [@individual-pharmaceuticals], nutraceuticals [@individual-nutraceuticals], vaccines, and diagnostic technologies), and biological and social factors influencing disease transmission and outcomes.
To date, 50 authors are associated with the consortium (Figure @fig:projectstats).

More formal recruitment efforts to integrate with existing projects providing support for undergraduate students during COVID-19 were also successful.
We incorporated summaries written by the students, post-docs, and faculty of the Immunology Institute at the Mount Sinai School of Medicine^[<https://github.com/ismms-himc/covid-19_sinai_reviews>] [@doi:10.1038/s41577-020-0319-0].
Additionally, two of the consortium authors were undergraduate students recruited through the American Physician Scientist Association's Virtual Summer Research Program.
Thus, the consortium was successful in providing a venue for researchers across all career stages to continue investigating and publishing at a time when many biomedical researchers were unable to access their laboratory facilities.

#### Integrating Data

We integrated data into the manuscripts from several sources (Figure @fig:manubot-workflow).
Worldwide cases and deaths were tracked by the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University^[<https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series>].
The clinical trials statistics and figure were generated based on data from the University of Oxford Evidence-Based Medicine Data Lab's COVID-19 TrialsTracker [@doi:10.5281/zenodo.3732709].
Information about vaccine distribution was extracted from Our World In Data^[<https://github.com/owid/covid-19-data>] [@owidcoronavirus].
(Figure @fig:cord19-growth) integrates data from the CORD-19 dataset [@arxiv:2004.10706].

Manubot's bibliographic management capabilities were critical because the amount of relevant literature published far outstripped what we had anticipated at the beginning of the project.
As of September 10, 2021, there were 1676 references (Figure @fig:projectstats).
The scite plugin provided a way to visually inspect the reference list to identify possible references of concern.
This and the other new features required for the COVID-19 project are now included in Manubot's rootstock, which is the template GitHub repository for creating a new manuscript.
Using CI, Manubot now checks that the manuscript was built correctly, runs spell-checking, and cross-references the manuscripts cited in this review.
In addition, Manubot rootstock now supports citing clinical trial identifiers such as `clinicaltrials:NCT04292899` [@clinicaltrials:NCT04292899].

### DISCUSSION

The current project was based in the GitHub repository `greenelab/covid19-review` using Manubot [@doi:10.1371/journal.pcbi.1007128] to continuously generate the manuscript.
The Manubot framework facilitated a massive collaborative review on an urgent topic.
We demonstrated the utility of Manubot to a project where many contributors lacked expertise or even experience working with version control.
This effort has produced not only seven literature reviews on topics relevant to the COVID-19 pandemic, but has also generated cyberinfrastructure for training novice users in GitHub.
We also extended the functionalities of Manubot to provide more of the benefits of What You See Is What You Get platforms such as Google Docs (Table @tbl:manubot-addons).
Open publishing thus allowed us to harness the domain expertise of a large group of non-technical users to respond to the flood of COVID-19 publications.

Several existing and new features in Manubot aid in responding to the challenges posed by the infodemic.
Manuscripts are written in markdown and can be rendered in several formats providing different advantages to users.
For example, beyond building just a PDF, Manubot also renders the manuscript in HTML, DOCX, and now, LaTeX (in a more limited capacity).
The interactive HTML manuscript format offers several advantages over a static PDF to harmonize available resources and address specific problems related to COVID-19.
The integration of scite into the HTML build makes references more manageable by visually indicating whether their results are contested or whether they have been corrected or retracted.
Cross-referencing different pieces of the manuscript, such as cited preprints with reviews stored in an appendix, is another interactive option presented by HTML.
The DOCX format was preferred by most non-technical users for reviewing the final version of the manuscript and was useful for creating submissions to a biological journal.
Additionally, because of the heavy emphasis on Word processing in biology, Manubot's ability to generate DOCX outputs was expanded to allow users to generate DOCX files containing only a section of the manuscript.
In our case, where the full project is nearly 150,000 words, this allows individual pieces to be shared more easily.
Finally, the preliminary addition of LaTeX output is useful for researchers from computational fields who submit papers in TeX format and removes the step of reformatting markdown prior to submission. 

| Type | Description |
|------|-----------------|
| CI | Regularly download external data sources, generate new figures and statistics, and read them when Manubot builds the latest manuscript |
| CI | Post spell-checking reports as pull request comments |
| Citations  | Zotero extension to report more relevant clinical trial metadata from <https://clinicaltrials.gov> |
| Citations  | Cite any Compact Uniform Resource Identifier, such as `clinicaltrials` or `ncbigene` |
| Citations  | scite badges to track retractions, corrections, and notices of concern |
| Outputs  | Improved support for Pandoc's LaTeX output |
| Outputs  | Build complete manuscript alongside individual sections as standalone documents |

Table: Manubot extensions for the COVID-19 review.
{#tbl:manubot-addons}

The COVID-19 Review Consortium provided a platform for researchers to engage in scientific investigation early in the pandemic when many biological scientists were unable to access their research spaces.
In turn, by seeking to adapt Manubot to allow for broader participation, we made a number of improvements that are expected to increase its appeal to researchers from all backgrounds.
Manubot provided a way for contributors from a variety of backgrounds, including early-career researchers, to join a massive collaborative project while demonstrating their individual contributions to the larger work and gaining experience with version control.
The licensing and infrastructure also provide the basis for individuals to adapt from this project to create their own snapshots of the COVID-19 literature that derive from, but are not wholly identical to, the primary versions of these reviews.
This project suggests that massive online open publishing efforts can indeed advance scholarship through inclusion [@doi:10.5334/kula.63], including during the extreme challenges presented by the COVID-19 pandemic.

Some challenges did arise in efforts to include an academically diverse set of authors.
The barriers to entry posed by git and GitHub likely still reduced participation from individuals who might have otherwise been interested.
Using pull requests as a tool for writing text is also unfamiliar to many or most scientists, and the review process can be slow, which might cause interested contributors to lose interest.
Additionally, the pull request model may limit people from providing general feedback on the manuscript or a section of the manuscript, unless there is an open pull request.
As a result, some feedback came through email or comments on the DOCX outputs that were then translated into issues or pull requests by the project managers.
Given that our approach hinged on these version control tools, it is likely that our group of contributors was biased towards those who were interested in or experienced with computational tools.
The trajectory of the pandemic itself also likely influenced participation: engagement waned over the course of the pandemic as labs opened back up and researchers were able to return to their work, and we recruited very few senior clinicians to the project, which is unsurprising given the load on medical professionals during this time.
Engagement that waxes and wanes is, however, typical when writing massively open online papers [@doi:10.5334/kula.63].
Adding features such as spell-check did improve usability, and additional features such as automatically checking the formatting of citations could further improve the usability of this tool.
In the future, a formal study of participation could allow for quantification of these biases and improved efforts to foster inclusion.

Additional limitations are challenges associated with massively open online papers in general.
With such a large amount of text, it is not possible to keep all sections of the manuscript up to date at all times.
Readers are not able to distinguish when each section was updated.
Even GitHub's blame functionality does not distinguish minor changes from substantive updates to the text.
While much of the data and statistics update automatically, the text itself required updating by human experts.
This asynchronicity could potentially introduce incompatibility between the figures and the surrounding text.
Similarly, in line with the collaboration-related challenges of the project, some authors returned to update their text, while others did not.
As a result, the lead authors of each paper often spent several weeks prior to journal submission updating the text to reflect new developments in each area.
In the future, it may be possible to streamline this process through integration with a tool such as CoronaCentral [@doi:10.1073/pnas.2100766118] to automatically identify relevant, high-impact papers that need to be included, although expertise would still be required to incorporate them.
Another challenge involves tracking preprints as they are reviewed or critiqued, revised, and potentially published.
While updating the content of the manuscript would likely fall to human contributors, automatic detection of published versions of preprints [@doi:10.1101/2021.03.04.433874] could be integrated in the future.
These challenges are exacerbated by the scale of the infodemic, but developing solutions would benefit future projects tracking more typical trends in publication.
Similarly, outputting machine readable summaries of key information in the COVID-19 review manuscripts could reduce their contribution to the infodemic.
As it stands, the integration of Compact Uniform Resource Identifier does make a step in this direction.
Formal identifiers could be used to extract relationships among clinical trials, genes, publications, and other entities.
Thus, the experience of using Manubot for a massive project has laid the foundation for future additions to enhance user experience and inclusivity.

### CONCLUSION 

With the worldwide scientific community uniting during 2020 and 2021 to investigate COVID-19 from a wide range of perspectives, findings from many disciplines are relevant on a rapid timescale to a broad scientific audience.
As many other efforts have described, the publishing rate of formal manuscripts and preprints about COVID-19 has been unprecedented [@doi:10.1053/j.ackd.2020.08.003], and efforts to review the body of COVID-19 literature are faced with an ever-expanding corpus to evaluate.
In the case of the seven manuscripts produced by the COVID-19 Review Consortium, Manubot allows for continuous updating of the manuscripts as the pandemic enters its second year and the landscape shifts with the emergence of promising therapeutics and vaccines [@individual-pharmaceuticals].
These manuscripts pull data from external sources and update information and visualizations daily using CI.
By off-loading some updates to computational pipelines, domain experts can focus on the broader implications of new information as it emerges.
Centralizing, summarizing, and critiquing data and literature broadly relevant to COVID-19 can expedite the interdisciplinary scientific process that is currently happening at an advanced pace.
As of September 13, 2021, 2886 commits have been made to the manuscript across 575 merged pull requests.
The efforts of the COVID-19 Review Consortium illustrate the value of including open source tools, including those focused on open publishing, in these efforts.
By facilitating the versioning of text, such platforms also allow for documentation of the evolution of thought in an evolving area and formal analysis of a collaborative project.
This application of version control holds the potential to improve scientific publishing in a range of disciplines, including those outside of traditional computational fields.
While Manubot is a technologically complex tool, this project demonstrates that it can be applied to a variety of projects.
Future work can address remaining limitations and continue to advance Manubot as an inclusive tool for open publishing projects.




## Additional Items {.page_break_before} 

### Competing Interests

|Author|Competing Interests|Last Reviewed|
|---|---|---|
|Halie M. Rando|None|2021-01-20|
|Casey S. Greene|None|2021-01-20|
|Michael P. Robson|None|2020-11-12|
|Simina M. Boca|Currently an employee at AstraZeneca, Gaithersburg, MD, USA, may own stock or stock options. Work initially conducted at Georgetown University Medical Center with writing, reviewing, and editing continued while working at AstraZeneca.|2021-07-01|
|Nils Wellhausen|None|2020-11-03|
|Ronan Lordan|None|2020-11-03|
|Christian Brueffer|Employee and shareholder of SAGA Diagnostics AB.|2020-11-11|
|Sandipan Ray|None|2020-11-11|
|Lucy D\'Agostino McGowan|Received consulting fees from Acelity and Sanofi in the past five years|2020-11-10|
|Anthony Gitter|Filed a patent application with the Wisconsin Alumni Research Foundation related to classifying activated T cells|2020-11-10|
|Anna Ada Dattoli|None|2020-03-26|
|Ryan Velazquez|None|2020-11-10|
|John P. Barton|None|2020-11-11|
|Jeffrey M. Field|None|2020-11-12|
|Bharath Ramsundar|None|2020-11-11|
|Adam L. MacLean|None|2021-02-23|
|Alexandra J. Lee|None|2020-11-09|
|Immunology Institute of the Icahn School of Medicine|None|2020-04-07|
|Fengling Hu|None|2020-04-08|
|Nafisa M. Jadavji|None|2020-11-11|
|Elizabeth Sell|None|2020-11-11|
|Vincent Rubinetti|None|2021-04-29|
|Jinhui Wang|None|2021-01-21|
|Diane N. Rafizadeh|None|2020-11-11|
|Ashwin N. Skelly|None|2020-11-11|
|Marouen Ben Guebila|None|2021-08-02|
|Likhitha Kolla|None|2020-11-16|
|David Manheim|None|2020-11-11|
|Soumita Ghosh|None|2020-11-09|
|James Brian Byrd|Funded by FastGrants to conduct a COVID-19-related clinical trial|2020-11-12|
|YoSon Park|Now employed by Pfizer (subsequent to contributions to this project)|2020-01-22|
|Vikas Bansal|None|2021-01-25|
|Stephen Capone|None|2020-11-11|
|John J. Dziak|None|2020-11-11|
|Yuchen Sun|None|2020-11-11|
|Yanjun Qi|None|2020-07-09|
|Lamonica Shinholster|None|2020-11-11|
|Temitayo Lukan|None|2020-11-10|
|Sergey Knyazev|None|2020-11-11|
|Dimitri Perrin|None|2020-11-11|
|Serghei Mangul|None|2020-11-11|
|Shikta Das|None|2020-08-13|
|Gregory L Szeto|None|2020-11-16|
|Tiago Lubiana|None|2020-11-11|
|David Mai|None|2021-01-08|
|COVID-19 Review Consortium|None|2021-01-16|
|Rishi Raj Goel|None|2021-01-20|
|Joel D Boerckel|None|2021-03-26|
|Amruta Naik|None|2021-04-05|
|Yusha Sun|None|2021-04-10|
|Daniel S. Himmelstein|None|2021-04-30|
|Jeremy P. Kamil|TBD|2021-04-30|

### Author Contributions

|Author|Contributions|
|---|---|
|Halie M. Rando|A, D, E, Project Administration, Software, Visualization, Writing - Original Draft, Writing - Review & Editing|
|Casey S. Greene|Conceptualization, Project Administration, Software, Supervision, Writing - Review & Editing|
|Michael P. Robson|Methodology, Software, Supervision|
|Simina M. Boca|Methodology, Project Administration, Writing - Review & Editing|
|Nils Wellhausen|Project Administration, Visualization, Writing - Original Draft, Writing - Review & Editing|
|Ronan Lordan|Conceptualization, Project Administration, Writing - Original Draft, Writing - Review & Editing|
|Christian Brueffer|Project Administration, Writing - Original Draft, Writing - Review & Editing|
|Sandipan Ray|Writing - Original Draft, Writing - Review & Editing|
|Lucy D\'Agostino McGowan|Methodology, Writing - Original Draft, Writing - Review & Editing|
|Anthony Gitter|Methodology, Project Administration, Software, Visualization, Writing - Original Draft, Writing - Review & Editing|
|Anna Ada Dattoli|Writing - Original Draft|
|Ryan Velazquez|Methodology, Software, Writing - Review & Editing|
|John P. Barton|Writing - Original Draft, Writing - Review & Editing|
|Jeffrey M. Field|Writing - Original Draft, Writing - Review & Editing|
|Bharath Ramsundar|Writing - Review & Editing|
|Adam L. MacLean|Writing - Original Draft, Writing - Review & Editing|
|Alexandra J. Lee|Writing - Original Draft, Writing - Review & Editing|
|Immunology Institute of the Icahn School of Medicine|Data Curation|
|Fengling Hu|Writing - Original Draft, Writing - Review & Editing|
|Nafisa M. Jadavji|Supervision, Writing - Original Draft, Writing - Review & Editing|
|Elizabeth Sell|Writing - Original Draft, Writing - Review & Editing|
|Vincent Rubinetti|Software, Visualization, Writing - Original Draft|
|Jinhui Wang|Writing - Original Draft, Writing - Review & Editing|
|Diane N. Rafizadeh|Project Administration, Writing - Original Draft, Writing - Review & Editing|
|Ashwin N. Skelly|Writing - Original Draft, Writing - Review & Editing|
|Marouen Ben Guebila|Writing - Original Draft, Writing - Review & Editing|
|Likhitha Kolla|Writing - Original Draft|
|David Manheim|Investigation, Writing - Original Draft|
|Soumita Ghosh|Writing - Original Draft|
|James Brian Byrd|Writing - Original Draft, Writing - Review & Editing|
|YoSon Park|Writing - Original Draft, Writing - Review & Editing|
|Vikas Bansal|Writing - Original Draft, Writing - Review & Editing|
|Stephen Capone|Writing - Original Draft, Writing - Review & Editing|
|John J. Dziak|Writing - Original Draft, Writing - Review & Editing|
|Yuchen Sun|Visualization|
|Yanjun Qi|Visualization|
|Lamonica Shinholster|Writing - Original Draft|
|Temitayo Lukan|Investigation, Writing - Original Draft|
|Sergey Knyazev|Writing - Original Draft, Writing - Review & Editing|
|Dimitri Perrin|Writing - Original Draft, Writing - Review & Editing|
|Serghei Mangul|Writing - Review & Editing|
|Shikta Das|Writing - Review & Editing|
|Gregory L Szeto|Writing - Review & Editing|
|Tiago Lubiana|Writing - Review & Editing|
|David Mai|Writing - Original Draft, Writing - Review & Editing|
|COVID-19 Review Consortium|Project Administration|
|Rishi Raj Goel|Writing - Original Draft, Writing - Review & Editing|
|Joel D Boerckel|Writing - Review & Editing|
|Amruta Naik|**MISSING**|
|Yusha Sun|Writing - Review & Editing|
|Daniel S. Himmelstein|Software|
|Jeremy P. Kamil|Writing - Review & Editing|

### Acknowledgements

We thank Nick DeVito for assistance with the Evidence-Based Medicine Data Lab COVID-19 TrialsTracker data.
We thank Yael Evelyn Marshall who contributed writing (original draft) as well as reviewing and editing of pieces of the text but who did not formally approve the manuscript, as well as Ronnie Russell, who contributed text to and helped develop the structure of the manuscript early in the writing process and Matthias Fax who helped with writing and editing text related to diagnostics.
We are also very grateful to James Fraser for suggestions about successes and limitations in the area of computational screening for drug repurposing.
We are grateful to the following contributors for reviewing pieces of the text: Nadia Danilova, James Eberwine and Ipsita Krishnan.
<!-- To Do: Add  Patryk Orzechowski to acknowledgements for vaccines-->


### References <!-- $data-collapsed="true" --> {.page_break_before}

<!-- 
  This style exists in the accordion plugin already, but it will be loaded after 
  this section, and thus the browser will start to render the intensive refs div 
  (despite it explicitly being set to collapsed). Duplicate here to hopefully 
  *immediately* hide the references div before the browser tries to render.
-->
<style>
  *:not(h2)[data-collapsed="true"] {
    display: none;
  }
</style>
<!-- Explicitly insert bibliography here -->
<div id="refs" data-collapsed="true"></div>

<!-- Define citation tags below -->
[@tag:Park2020_distancing]: doi:10.3201/eid2611.201099

<!-- Individual sections that have been published as preprints or journal manuscripts -->
[@individual-pathogenesis]: arxiv:2102.01521
[@individual-nutraceuticals]: https://pubmed.ncbi.nlm.nih.gov/33947804/
[@individual-pharmaceuticals]: arxiv:2103.02723
[@individual-methods]: arxiv:2109.08633


## Appendix A {.page_break_before}

This appendix contains reviews produced by the Immunology Institute of the Icahn School of Medicine


<!-- Appendix source: https://github.com/ismms-himc/covid-19_sinai_reviews/tree/05d387bf10b829dad6bedb4bf313686750b78cc9 -->

### Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody

Tian et al. *Emerg Microbes Infect 2020* [@doi:10.1101/2020.01.28.923011]

#### Keywords

- Monoclonal antibody
- Cross-reactivity
- receptor binding domain

#### Summary

Considering the relatively high identity of the receptor binding domain
(RBD) of the spike proteins from 2019-nCoV and SARS-CoV (73%), this
study aims to assess the cross-reactivity of several anti-SARS-CoV
monoclonal antibodies with 2019-nCoV. The results showed that the
SARS-CoV-specific antibody CR3022 can potently bind 2019-nCoV RBD.

#### Main Findings

The structure of the 2019-nCoV spike RBD and its conformation in complex
with the receptor angiotensin-converting enzyme (ACE2) was modeled *in
silico* and compared with the SARS-CoV RBD structure. The models
predicted very similar RBD-ACE2 interactions for both viruses. The
binding capacity of representative SARS-CoV-RBD specific monoclonal
antibodies (m396, CR3014, and CR3022) to recombinant 2019-nCoV RBD was
then investigated by ELISA and their binding kinetics studied using
biolayer interferometry. The analysis showed that only CR3022 was able
to bind 2019-nCoV RBD with high affinity (KD of 6.3 nM), however it did
not interfere with ACE2 binding. Antibodies m396 and CR3014, which
target the ACE2 binding site of SARS-CoV failed to bind 2019-nCoV spike
protein.

#### Limitations

The 2019-nCoV RBD largely differ from the SARS-CoV at the C-terminus
residues, which drastically impact the cross-reactivity of antibodies
described for other B beta-coronaviruses, including SARS-CoV. This study
claims that CR3022 antibody could be a potential candidate for therapy.
However, none of the antibodies assayed in this work showed
cross-reactivity with the ACE2 binding site of 2019-nCoV, essential for
the replication of this virus. Furthermore, neutralization assays with
2019-nCoV virus or pseudovirus were not performed. Although the use of
neutralizing antibodies is an interesting approach, these results
suggest that it is critical the development of novel monoclonal
antibodies able to specifically bind 2019-nCoV spike protein.

#### Credit

*Review by D.L.O as part of a project by students, postdocs and faculty
at the Immunology Institute of the Icahn School of Medicine, Mount
Sinai.*


### Integrative Bioinformatics Analysis Provides Insight into the Molecular Mechanisms of 2019-nCoV

He et al. *medRxiv* [@doi:10.1101/2020.02.03.20020206]

#### Keywords

- ACE2
- lungs
- smoking
- COPD
- asthma
- SARS-Cov
- IL-1
- IL-10
- IL-6
- IL-8

#### Main Findings

The authors used bioinformatics tools to identify features
of ACE2 expression in the lungs of different patent groups: healthy,
smokers, patients with chronic airway disease (i.e., COPD) or asthma.
They used gene expression data publicly available from GEO that included
lung tissues, bronchoalveolar lavage, bronchial epithelial cells, small
airway epithelial cells, or SARS-Cov infected cells.

The authors describe no significant differences in ACE2 expression in
lung tissues of Healthy, COPD, and Asthma groups (p=0.85); or in BAL of
Healthy and COPD (p=0.48); or in epithelial brushings of Healthy and
Mild/Moderate/Severe Asthma (p=0.99). ACE2 was higher in the small
airway epithelium of long-term smokers vs non-smokers (p&lt;0.001).
Consistently, there was a trend of higher ACE2 expression in the
bronchial airway epithelial cells 24h post-acute smoking exposure
(p=0.073). Increasing ACE2 expression at 24h and 48h compared to 12h
post SARS-Cov infection (p=0.026; n=3 at each time point) was also
detected.

15 lung samples’ data from healthy participants were separated into high
and low ACE2 expression groups. “High” ACE2 expression was associated
with the following GO pathways: innate and adaptive immune responses, B
cell mediated immunity, cytokine secretion, and IL-1, IL-10, IL-6, IL-8
cytokines. The authors speculate that a high basal ACE2 expression will
increase susceptibility to SARS-CoV infection.

In 3 samples SARS-Cov infection was associated with IL-1, IL-10 and IL-6
cytokine production (GO pathways) at 24h. And later, at 48h, with T-cell
activation and T-cell cytokine production. It is unclear whether those
changes were statistically significant.

The authors describe a time course quantification of immune infiltrates
in epithelial cells infected with SARS-Cov infection. They state that in
healthy donors ACE2 expression did not correlate with the immune cell
infiltration. However, in SARS-Cov samples, at 48h they found that ACE2
correlated with neutrophils, NK-, Th17-, Th2-, Th1- cells, and DCs.
Again, while authors claim significance, the corresponding correlation
coefficients and p-values are not presented in the text or figures. In
addition, the source of the data for this analysis is not clear.

Using network analysis, proteins SRC, FN1, MAPK3, LYN, MBP, NLRC4, NLRP1
and PRKCD were found to be central (Hub proteins) in the regulating
network of cytokine secretion after coronavirus infection. Authors
conclude this indicates that these molecules were critically important
in ACE2-induced inflammatory response. Additionally, authors speculate
that the increased expression of ACE2 affected RPS3 and SRC, which were
the two hub genes involved in viral replication and inflammatory
response.

#### Limitations

The methods section is very limited and does not
describe any of the statistical analyses; and description of the
construction of the regulatory protein networks is also limited. For the
findings in Figures 2 authors claim significance, which is not supported
by p-values or coefficients. For the sample selection, would be useful
if sample sizes and some of the patients’ demographics (e.g. age) were
described.

For the analysis of high vs low ACE2 expression in healthy subjects, it
is not clear what was the cut off for ‘high’ expression and how it was
determined. Additionally, further laboratory studies are warranted to
confirm that high ACE2 gene expression would have high correlation with
the amount of ACE2 protein on cell surface. For the GO pathway analysis
significance was set at p&lt;0.05, but not adjusted for multiple
comparisons.

There were no samples with SARS-CoV-2 infection. While SARS-Cov and
SARS-CoV-2 both use ACE2 to enter the host cells, the analysis only
included data on SARS-Cov and any conclusions about SARS-CoV2 are
limited.

Upon checking GSE accession numbers of the datasets references, two
might not be cited correctly: GSE37758 ("A spergillus niger: Control
(fructose) vs. steam-exploded sugarcane induction (SEB)"" was used in
this paper as “lung tissue” data) and GSE14700 ("Steroid Pretreatment
of Organ Donors to Prevent Postischemic Renal Allograft Failure: A
Randomized, Controlled Trial" – was used as SARS-Cov infection data).

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Diarrhea may be underestimated: a missing link in 2019 novel coronavirus

Liang et al. *medRxiv* [@doi:10.1101/2020.02.03.20020289]

#### Keywords

- SARS-CoV-2
- diarrhea
- ACE2
- scRNA-seq

#### Main Findings

This study examined the incidence of diarrhea in
patients infected with SARS-CoV-2 across three recently published
cohorts and found that there are statistically significant differences
by Fisher’s exact test. They report that this could be due to subjective
diagnosis criterion for diarrhea or from patients first seeking medical
care from gastroenterologist. In order to minimize nosocomial infections
arising from unsuspected patients with diarrhea and gain comprehensive
understanding of transmission routes for this viral pathogen, they
compared the transcriptional levels of ACE2 of various human tissues
from NCBI public database as well as in small intestine tissue from
CD57BL/6 mice using single cell sequencing. They show that ACE2
expression is not only increased in the human small intestine, but
demonstrate a particular increase in mice enterocytes positioned on the
surface of the intestinal lining exposed to viral pathogens. Given that
ACE2 is the viral receptor for SARS-CoV-2 and also reported to regulate
diarrhea, their data suggests the small intestine as a potential
transmission route and diarrhea as a potentially underestimated symptom
in COVID19 patients that must be carefully monitored. Interestingly,
however, they show that ACE2 expression level is not elevated in human
lung tissue.

#### Limitations

Although this study demonstrates a
statistical difference in the incidence of diarrhea across three
separate COVID19 patient cohorts, their conclusions are limited by a
small sample size. Specifically, the p-value computed by Fisher’s exact
test is based on a single patient cohort of only six cases of which 33%
are reported to have diarrhea, while the remaining two larger cohorts
with 41 and 99 cases report 3% and 2% diarrhea incidence, respectively.
Despite showing significance, they would need to acquire larger sample
sizes and cohorts to minimize random variability and draw meaningful
conclusions. Furthermore, they do not address why ACE2 expression level
is not elevated in human lung tissue despite it being a major
established route of transmission for SARS-CoV-2. It could be helpful to
validate this result by looking at ACE2 expression in mouse lung tissue.
Finally, although this study is descriptive and shows elevated ACE2
expression in small intestinal epithelial cells, it does not establish a
mechanistic link to SARS-CoV-2 infection of the host. Overall, their
claim that infected patients exhibiting diarrhea pose an increased risk
to hospital staff needs to be further substantiated.

#### Significance

This study provides a possible transmission route and a
potentially underappreciated clinical symptom for SARS-CoV-2 for better
clinical management and control of COVID19.

#### Credit

*Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Specific ACE2 Expression in Cholangiocytes May Cause Liver Damage After 2019-nCoV Infection

Chai et al. *bioRxiv* [@doi:10.1101/2020.02.03.931766]

#### Keywords

- ACE2
- Cholangiocytes
- COVID-associated Liver Damage

#### Summary

Using both publicly available scRNA-seq dataset of liver
samples from colorectal patients and scRNA-sequencing of four liver
samples from healthy volunteers, the authors show that ACE2 is
significantly enriched in the majority of cholangiocytes (59.7 %) but
not in hepatocytes (2.6%).

#### Main Findings

Using bioinformatics approaches of
RNASeq analysis, this study reveals that ACE2 dominates in
cholangiocytes and is present at very low levels in hepatocytes.

#### Limitations

The
study does not provide mechanistic insights into how SARS-CoV-2 can
infect and replicate in cholangiocytes and the types of intrinsic
anti-viral responses induced by cholangiocytes when infected. In
addition, because the study relies on the assumption that SARS-CoV-2
infects cells only through ACE2, it cannot discount the possibility that
the virus can infect hepatocytes through mechanisms other than
ACE2-mediated entry. Furthermore, because the scRNA-seq analysis were
performed on healthy liver samples, one cannot draw any definitive
conclusions about gene expression states (including ACE2 expression in
liver cell types) in system-wide inflammatory contexts.

#### Significance

This article with other studies on liver
damage in COVID patients suggests that liver damage observed in COVID
patients is more due to inflammatory cytokines than direct infection of
the liver. Even if cholangiocytes are infectable by SARS-CoV-2 (which
was demonstrated by human liver ductal organoid study
([@doi:10.1101/2020.03.16.990317]),
published clinical data show no significant increase in bile duct injury
related indexes (i.e. alkaline phosphatase, gamma-glutamyl
transpeptidase and total bilirubin). In sum, it underscores the
importance of future studies characterizing cellular responses of
extra-pulmonary organs in the context of COVID or at least in viral lung
infections..

#### Credit

*Summary generated by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### ACE2 expression by colonic epithelial cells is associated with viral infection, immunity and energy metabolism

Wang et al. *medRxiv*. [@doi:10.1101/2020.02.05.20020545]

#### Keywords

- single cell RNA seq
- ACE2 expression
- human colonic biopsy

#### Main Findings

Colonic enterocytes primarily express ACE2. Cellular pathways associated with ACE2 expression include innate immune signaling, HLA up regulation, energy metabolism and apoptotic signaling.  

#### Limitations

This is a study of colonic biopsies taken from 17 children with and without IBD and analyzed using scRNAseq to look at ACE2 expression and identify gene families correlated with ACE2 expression. The authors find ACE2 expression to be primarily in colonocytes.  It is not clear why both healthy and IBD patients were combined for the analysis. Biopsies were all of children so extrapolation to adults is limited. The majority of genes found to be negatively correlated with ACE2 expression include immunoglobulin genes (IGs). IG expression will almost certainly be low in colonocytes irrespective of ACE2 expression.  

#### Significance

This study performs a retrospective analysis of ACE2 expression using an RNAseq dataset from intestinal biopsies of children with and without IBD. The implications for the CoV-19 epidemic are modest, but do provide support that ACE2 expression is specific to colonocytes in the intestines.  The ontological pathway analysis provides some limited insights into gene expression associated with ACE2.

#### Credit

*Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### The Pathogenicity of 2019 Novel Coronavirus in hACE2 Transgenic Mice

Bao et al. *bioRxiv* [@doi:10.1101/2020.02.07.939389]

#### Keywords

- Covid-19 mouse model
- hACE2 mice
- 2019-nCoV model
- ACE2
- 2019-nCoV 

#### Main Findings

Using a transgenic human Angiotensin-converting enzyme 2
(hACE2) mouse that has previously been shown susceptible to infection by
SARS-CoV, Bao et al. create a model of pandemic 2019-nCoV strain
coronavirus. The model includes interstitial hyperplasia in lung tissue,
moderate inflammation in bronchioles and blood vessels, and histology
consistent with viral pneumonia at 3 days post infection. Wildtype did
not experience these symptoms. In addition, viral antigen and hACE2
receptor were found to co-localize the lung by immunofluorescence 3-10
days post infection only in the hACE2 infected mice.

#### Limitations

The characterization of the infection remains incomplete,
as well as lacking characterization of the immune response other than
the presence of a single antiviral antibody. Though they claim to
fulfill Koch’s postulates, they only isolate the virus and re-infect
Vero cells, rather than naive mice.

#### Significance

This paper establishes a murine model for 2019-nCoV
infection with symptoms consistent with viral pneumonia. Though not
fully characterized, this model allows in vivo analysis of viral entry
and pathology that is important for the development of vaccines and
antiviral therapeutics.

#### Credit

*Review by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty
at the Immunology Institute of the Icahn School of Medicine, Mount
Sinai.*


### Caution on Kidney Dysfunctions of 2019-nCoV Patients

Li et al. *medRxiv*. [@doi:10.1101/2020.02.08.20021212]

#### Keywords

CoVID-19, 2019-nCoV, SARS-CoV-2, kidney, clinical, creatinine,
proteinuria, albuminuria, CT

#### Main Findings

-   Retrospective study of 59 patients assayed key function indicators
    of the kidney–including urine protein, blood urea nitrogen (BUN),
    plasma creatinine (Cre), and renal CT scan data.

-   Found that 34% of patients developed massive albuminuria on the
    first day of admission, and 63% developed proteinuria during their
    stay in hospital; and 19% of patients had high plasma creatinine,
    especially the terminal cases.

-   CT analyses of 27 patients showed all patients to have abnormal
    kidney damage; indicate that inflammation and edema of the renal
    parenchyma very common.

#### Limitations

-   No analysis of immunity-dependent damage and cytokines in
    blood/plasma/urine. Will be worth correlating disease progression
    with cytokine production, immune activity and kidney function.

-   Extrapolating to earlier SARS-CoV studies provides the only
    rationale for viral-damage in kidney and resultant pathologic immune
    response *(understandable for this clinical study)*.

#### Significance

-   Multiple lines of evidence along this study’s finding point to the
    idea that renal impairment/injury is a key risk factor in 2019-nCoV
    patients similar to what has been reported for SARS-CoV[@doi:10.1111/j.1523-1755.2005.67130.x]; this
    may be one of the major causes of virally-induced damage and
    contribute to multiorgan failure.

-   ACE2 expression in kidney proximal tubule epithelia and bladder
    epithelia [@doi:10.1101/2020.02.08.939892] support
    these clinical findings.

-   Study argues for closely monitoring kidney function, and applying
    potential interventions including continuous renal replacement
    therapies (CRRT) for protecting kidney functions as early as
    possible, particularly for those with rising plasma creatinine.

#### Credit

*Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Profiling the immune vulnerability landscape of the 2019 Novel Coronavirus

Zhu et al. *bioRxiv* [@doi:10.1101/2020.02.08.939553]

#### Keywords

- epitope prediction
- vaccine development.

#### Main Findings

This study harnesses bioinformatic profiling to predict the potential of
COV2 viral proteins to be presented on MHC I and II and to form linear
B-cell epitopes. These estimates suggest a T-cell antigenic profile
distinct from SARS-CoV or MERS-CoV, identify focused regions of the
virus with a high density of predicted epitopes, and provide preliminary
evidence for adaptive immune pressure in the genetic evolution of the
virus.

#### Limitations

While the study performs a comprehensive analysis of potential epitopes
within the virus genome, the analysis relies solely on bioinformatic
prediction to examine MHC binding affinity and B-cell epitope potential
and does not capture the immunogenicity or recognition of
these epitopes. Future experimental validation in data from patients
infected with SARS-CoV-2 will be important to validate and refine these
findings. Thus some of the potential conclusions stated, including viral
evolution toward lower immunogenicity or a dominant role for CD4+
T-cells rather than CD8+ T-cells in viral clearance, require further
valiadtion.

#### Significance

These findings may help direct peptide vaccine design toward relevant
epitopes and provide intriguing evidence of viral evolution in response
to immune pressure.

#### Credit

*Summary generated as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Single-cell Analysis of ACE2 Expression in Human Kidneys and Bladders Reveals a Potential Route of 2019-nCoV Infection

Lin et al. *bioRxiv* [@doi:10.1101/2020.02.08.939892]

#### Keywords

- CoVID-19
- 2019-nCoV
- SARS-CoV-2
- ACE2
- scRNAseq
- kidney
- bladder
- public dataset

#### Main Findings

-   To investigate the possible cause of kidney damage in 2019-nCoV
    patients, authors used published kidney and bladder cell atlas data
    (GSE131685, GSE108097; 3 healthy donors each) as well as an
    unpublished kidney single-cell RNA-Seq data (in-house from 2
    transplant donors) to evaluate ACE2 gene expressions in all cell
    types of healthy kidneys and bladders.

-   They find enriched expression of ACE2 transcript in all subtypes of
    proximal tubule cells of kidney, with 5%-15% of both straight and
    convoluted proximal tubule cells expressing ACE2.

-   They also find detectable levels of ACE2 in bladder epithelial
    cells, noting expression from around 1.5% of cells in the outer
    layer umbrella cells of the bladder epithelium and decreasing in the
    basal cells.

-   Importantly endothelial or immune cells in kidney/bladder do not
    express ACE2.

#### Limitations

-   This study primarily characterizes ACE2 expression (amongst other
    genes) from a small healthy-donor dataset, and will benefit from
    supporting data in (expired) patient samples to show functional
    viral damage. ACE2 transcript does not necessarily translate to
    viral permissiveness in kidney/bladder epithelia or cytokine
    release.

-   This study focuses on only healthy tissue; it will be useful to
    analyze kidney/bladder epithelial ACE2 expression under inflammatory
    conditions or in patients with underlying kidney conditions.

-   Given what is known about protease TMPRSS2 expression during
    SARS-CoV-2 infection, ACE2+TMPRSS2+ double-positive cell
    identification would be useful in these datasets.

#### Significance

-   ACE2 protein is spatially restricted to brush border of proximal
    tubules and in bladder umbrella cells [@doi:10.1002/path.1570], such cells in direct
    contact with viral particles are likely to be highly sensitive to
    viral-induced damage.

-   SARS-CoV and MERS-CoV have been shown to be detected in urine of
    patients and associate with higher mortality [@doi:10.1111/j.1523-1755.2005.67130.x; @doi:10.7326/m13-2486], thus worth
    understanding kidney damage and resultant immune response in
    SARS-CoV-2 as well.

-   This study argues for a potential mode of viral infectivity and
    resultant inflammatory responses in these tissue in addition to
    reported infectivity in the lung and digestive system, which is
    supported by clinical data showing acute and early kidney
    complications in 2019-nCoV patients [@doi:10.1101/2020.02.08.20021212].

-   Clinically, thus very important to track urinary CoVID-19 shedding
    as well as study acute kidney injury-related co-morbidities.

#### Credit

*Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Neutrophil-to-Lymphocyte Ratio Predicts Severe Illness Patients with 2019 Novel Coronavirus in the Early Stage

Liu et al. *medRxiv* [@doi:10.1101/2020.02.10.20021584]

#### Keywords

- severe disease
- pneumonia
- lymphocytes
- neutrophils

#### Main Findings

This study aimed to find prognostic biomarkers of COVID-19
pneumonia severity. Sixty-one (61) patients with COVID-19 treated in
January at a hospital in Beijing, China were included. On average,
patients were seen within 5 days from illness onset. Samples were
collected on admission; and then patients were monitored for the
development of severe illness with a median follow-up of 10 days\].

Patients were grouped as “mild” (N=44) or “moderate/severe” (N=17)
according to symptoms on admission and compared for different
clinical/laboratory features. “Moderate/severe” patients were
significantly older (median of 56 years old, compared to 41 years old).
Whereas comorbidies rates were largely similar between the groups,
except for hypertension, which was more frequent in the severe group (p=
0.056). ‘Severe’ patients had higher counts of neutrophils, and serum
glucose levels; but lower lymphocyte counts, sodium and serum chlorine
levels. The ratio of neutrophils to lymphocytes (NLR) was also higher
for the ‘severe’ group. ‘Severe’ patients had a higher rate of bacterial
infections (and antibiotic treatment) and received more intensive
respiratory support and treatment.

26 clinical/laboratory variables were used to select NLR and age as the
best predictors of the severe disease. Predictive cutoffs for a severe
illness as NLR ≥ 3.13 or age ≥ 50 years.

#### Limitations

Identification of early biomarkers is important for
making clinical decisions, but large sample size and validation cohorts
are necessary to confirm findings. It is worth noting that patients
classified as “mild” showed pneumonia by imaging and fever, and in
accordance with current classifications this would be consistent with
“moderate” cases. Hence it would be more appropriate to refer to the
groups as “moderate” vs “severe/critical”. Furthermore, there are
several limitations that could impact the interpretation of the results:
e.g. classification of patients was based on symptoms presented on
admission and not based on disease progression, small sample size,
especially the number of ‘severe’ cases (with no deaths among these
patients). Given the small sample size, the proposed NLR and age cut
offs might not hold for a slightly different set of patients. For
example, in a study of &gt;400 patients, ‘non-severe’ and ‘severe’ NLR
were 3.2 and 5.5, respectively [@doi:10.1093/cid/ciaa248].

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Characteristics of lymphocyte subsets and cytokines in peripheral blood of 123 hospitalized patients with 2019 novel coronavirus pneumonia (NCP)

Wan et. al. *medRxiv* [@doi:10.1101/2020.02.10.20021832]

#### Keywords

- Cytokines
- lymphocyte subsets
- CD8 + T
- B cells
- NK cells,
- PBMCs
- IL-6
- IL-10

#### Main Findings

The authors analyzed lymphocyte subsets and cytokines of 102 patients
with mild disease and 21 with severe disease. CD8+T cells and CD4+T
cells were significantly reduced in both cohort. particularly in severe
patients. The cytokines IL6 and IL10 were significantly elevated in
severe patients as compared to mild. No significant differences were
observed in frequency of B cells and NK cells.

The authors argue that the measurement of T cell frequencies and
cytokine levels of IL6 and IL10 can be used to predict progression of
disease from Mild to severe Cov-2 infection.

#### Limitations

The study demonstrates in a limited cohort
similar associations to several other reported studies. The authors
didn’t compare the changes in lymphocyte and cytokine with healthy
individual (Covid-19 Negative) rather used an internal standard value.
The recently preprint in LANCET shows The degree of lymphopenia and a
pro-inflammatory cytokine storm is higher in severe COVID-19 patients
than in mild cases, and is associated with the disease severity [@doi:10.2139/ssrn.3539682].

#### Significance

This translational data identifies key cytokines and lymphopenia
associated with disease severity although mechanism and key cellular
players are still unknown. Higher level IL-6 production in severe
patient suggests potential role of Tocilizumab (anti-IL6R) biologic
although clinical trial will be necessary.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Epidemiological and Clinical Characteristics of 17 Hospitalized Patients with 2019 Novel Coronavirus Infections Outside Wuhan, China

Li et al. *medRxiv* [@doi:10.1101/2020.02.11.20022053]

#### Keywords

- epidemiology
- clinical characteristics

#### Major Findings

These authors looked at 17 hospitalized patients with COVID-19
confirmed by RT-PCR in Dazhou, Sichuan. Patients were admitted between
January 22 and February 10 and the final data were collected on February
11. Of the 17 patients, 12 remained hospitalized while 5 were discharged
after meeting national standards. The authors observed no differences
based on the sex of the patients but found that the discharged patients
were younger in age (p = 0.026) and had higher lymphocyte counts (p =
0.005) and monocyte counts (p = 0.019) upon admission.

#### Limitations

This study is limited in the sample size of the study and the last
data collection point was only one day after some of the patients were
admitted.

#### Significance

These findings have been somewhat supported by subsequent studies
that show that older age and an immunocompromised state are more likely
to result in a more severe clinical course with COVID-19. However, other
studies have been published that report on larger numbers of cases.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### ACE2 Expression in Kidney and Testis May Cause Kidney and Testis Damage After 2019-nCoV Infection

[@doi:10.1101/2020.02.12.20022418]

#### Keywords

- CoVID-19
- 2019-nCoV
- SARS-CoV-2
- kidney
- testis
- ACE2
- scRNAseq

#### Main Findings

-   Study used online datasets (scRNAseq GSE131685, scRNAseq GSE107585,
    Human Protein Atlas, GTEx portal, CCLE) to analyze ACE2 expression
    in different human organs.

-   Study re-analyzed three clinical datasets (n=6, n=99, and n=41) to
    show 3\~10% of 2019-nCoV patients present with abnormal renal
    function.

-   results indicate ACE2 highly expressed in renal tubular cells,
    Leydig cells and seminiferous ductal cells of testis.

#### Limitations

-   Very preliminary transcript/protein dataset analysis in healthy
    cohorts; does not necessarily translate to actual viral tropism and
    permissiveness.

-   Clinically, would be important to determine with larger longitudinal
    dataset if SARS-CoV-2 infection changes sperm quality or testicular
    inflammation.

-   Similarly, would be important to determine if simultaneous HBV or
    syphilis infection and orchitis impacts SARS-CoV-2 severity.

-   Examination and follow-up of renal function and viral orchitis/sperm
    quality of CoVID-19 patients not done in this preliminary study.

#### Significance

-   Kidney ACE2 result supports other concurrent sequencing studies
    [@doi:10.1101/2020.02.08.939892] and clinical reports
    of abnormal renal function or even kidney damage in patients
    infected with 2019-nCoV
    [@doi:10.1101/2020.02.08.20021212].

-   High ACE2 expression in testis suggests potential tropism of the
    virus to testicular tissues and indicates potential risks for male
    fertility. Viral orchitis reported for SARS-CoV previously \[1\],
    but no clear evidence so far of infertility in SARS, MERS or
    CoVID-19 patients.

#### Credit

*Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Aberrant pathogenic GM-CSF+ T cells and inflammatory CD14+CD16+ monocytes in severe pulmonary syndrome patients of a new coronavirus

[@doi:10.1101/2020.02.12.945576]

#### Keywords
- immunopathology
- Th1
- inflammatory monocytes
- GM-CSF
- IFN-γ
- IL-6

#### Main Findings

The authors of this study sought to characterize the immune
mechanism causing severe pulmonary disease and mortality in 2019-nCoV
(COVID-19) patients. Peripheral blood was collected from hospitalized
ICU (n=12) and non-ICU (n=21) patients with confirmed 2019-nCoV and from
healthy controls (n=10) in The First Affiliated Hospital of University
of Science and Technology China (Hefei, Anhui). Immune analysis was
conducted by flow cytometry. 2019-nCoV patients had decreased
lymphocyte, monocyte, and CD4 T cell counts compared to healthy
controls. ICU patients had fewer lymphocytes than non-ICU patients. CD4
T cells of 2019-nCoV patients expressed higher levels of activation
markers (OX40, CD69, CD38, CD44) and exhaustion markers (PD-1 and Tim3)
than those of healthy controls. CD4 cells of ICU patients expressed
significantly higher levels of OX40, PD-1, and Tim3 than those of
non-ICU patients. 2019-nCoV patients had higher percentages of CD4 T
cells co-expressing GM-CSF and IL-6 compared to healthy controls, while
ICU patients had a markedly higher percentage of GM-CSF+ IFN-γ+ CD4 T
cells than non-ICU patients. The CD4 T cells of nCoV patients and
healthy controls showed no differences in TNF-α secretion.

The CD8 T cells of 2019-nCoV patients also showed higher expression of
activation markers CD69, CD38, and CD44, as well as exhaustion markers
PD-1 and Tim3, compared to healthy controls. CD8 T cells of ICU patients
expressed higher levels of GM-CSF than those of non-ICU patients and
healthy controls. No IL-6 or TNF-α was found in the CD8 T cells of any
group. There were no differences in numbers of NK cells or B cells in
2019-nCoV patients and healthy controls, nor was there any GM-CSF or
IL-6 secretion from these cells in either group.

Percentages of CD14+ CD16+ GM-CSF+ and CD14+ CD16+ IL-6+ inflammatory
monocytes were significantly increased in nCoV patients compared to
healthy controls; in particular, patients in the ICU had greater
percentages of CD14+ CD16+ IL-6+ monocytes than non-ICU patients. The
authors suggest that in 2019-nCoV patients, pathogenic Th1 cells produce
GM-CSF, recruiting CD14+ CD16+ inflammatory monocytes that secrete high
levels of IL-6. These may enter pulmonary circulation and damage lung
tissue while initiating the cytokine storm that causes mortality in
severe cases. This is consistent with the cytokine storm seen in similar
coronaviruses, as IL-6, IFN-γ, and GM-CSF are key inflammatory mediators
seen in patients with SARS-CoV-1 and MERS-CoV.

#### Limitations

Though the results of this study open
questions for further investigation, this is an early study on a small
cohort of patients, and as such there are a number of limitations. The
study included only 12 ICU patients and 21 non-ICU patients, and ideally
would be repeated with a much larger patient cohort. Though the authors
make claims about differences in lymphocyte and monocyte counts between
patients and healthy controls, they did not report baseline laboratory
findings for the control group. Additionally, severity of disease was
classified based on whether or not patients were in the ICU. It would be
interesting to contextualize the authors’ immunological findings with
more specific metrics of disease severity or time course. Noting
mortality, time from disease onset, pre-existing conditions, or severity
of lung pathology in post-mortem tissue samples would paint a fuller
picture of how to assess risk level and the relationship between
severity of disease and immunopathology. Another limitation is the
selection of cytokines and immune markers for analysis, as the selection
criteria were based on the cell subsets and cytokine storm typically
seen in SARS-CoV-1 and MERS-CoV patients. Unbiased cytokine screens and
immune profiling may reveal novel therapeutic targets that were not
included in this study.

#### Significance

This study identifies potential therapeutic targets
that could prevent acute respiratory disease syndrome (ARDS) and
mortality in patients most severely affected by COVID-19. The authors
propose testing monoclonal antibodies against IL6-R or GM-CSF to block
recruitment of inflammatory monocytes and the subsequent cytokine storm
in these patients.

#### Credit

*Review by Gabrielle Lubitz as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Clinical Characteristics of 2019 Novel Infected Coronavirus Pneumonia：A Systemic Review and Meta-analysis

Qian et al. *medRxiv*.
[@doi:10.1101/2020.02.14.20021535]

#### Keywords
- White Blood Cells
- Lymphocytes
- Neutrophils

#### Main Findings

The authors performed a meta analysis of literature on clinical,
laboratory and radiologic characteristics of patients presenting with
pneumonia related to SARSCoV2 infection, published up to Feb 6 2020.
They found that symptoms that were mostly consistent among studies were
sore throat, headache, diarrhea and rhinorrhea. Fever, cough, malaise
and muscle pain were highly variable across studies. Leukopenia (mostly
lymphocytopenia) and increased white blood cells were highly variable
across studies. They identified three most common patterns seen on CT
scan, but there was high variability across studies. Consistently across
the studies examined, the authors found that about 75% of patients need
supplemental oxygen therapy, about 23% mechanical ventilation and about
5% extracorporeal membrane oxygenation (ECMO). The authors calculated a
staggering pooled mortality incidence of 78% for these patients.

#### Limitations

The authors mention that the total number of studies included in this
meta analysis is nine, however they also mentioned that only three
studies reported individual patient data. It is overall unclear how many
patients in total were included in their analysis. This is mostly
relevant as they reported an incredibly high mortality (78%) and mention
an absolute number of deaths of 26 cases overall. It is not clear from
their report how the mortality rate was calculated.

The data is based on reports from China and mostly from the Wuhan area,
which somewhat limits the overall generalizability and applicability of
these results.

#### Significance

This meta analysis offers some important data for clinicians to refer to
when dealing with patients with COVID-19 and specifically with
pneumonia. It is very helpful to set expectations about the course of
the disease.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Longitudinal characteristics of lymphocyte responses and cytokine profiles in the peripheral blood of SARS-CoV-2 infected patients

Liu et al. *medRxiv* [@doi:10.1101/2020.02.16.20023671]

#### Keywords

- Lymphopenia
- Neutrophil to CD8 T cell ratio (N8R)
- inflammatory cytokines

#### Main Findings

Liu et al. enrolled a cohort of 40 patients from Wuhan
including 27 mild cases and 13 severe cases of COVID-19. They performed
a 16-day kinetic analysis of peripheral blood from time of disease
onset. Patients in the severe group were older (medium age of 59.7,
compared to 48.7 in mild group) and more likely to have hypertension as
a co-morbidity. Lymphopenia was observed in 44.4% of the mild patients
and 84.6% of the severe patients. Lymphopenia was due to low T cell
count, specially CD8 T cells. Severe patients showed higher neutrophil
counts and an increase of cytokines in the serum (IL2, IL6, IL10 and
IFNγ). The authors measured several other clinical laboratory parameters
were also higher in severe cases compared to mild, but concluded that
neutrophil to CD8 T cell ratio (N8R) as the best prognostic factor to
identify the severe cases compared to other receiver operating
characteristic (ROC).

#### Limitations

This was a small cohort (N=40), and two of the patients
initially included in the severe group (N=13) passed away and were
excluded from the analysis due to lack of longitudinal data. However, it
would be most important to be able to identify patients with severe
disease with higher odds of dying. It seems that the different time
points analyzed relate to hospital admission, which the authors describe
as disease onset. The time between first symptoms and first data points
is not described. It would have been important to analyze how the
different measured parameters change according to health condition, and
not just time (but that would require a larger cohort). The predictive
value of N8R compared to the more commonly used NLR needs to be assessed
in other independent and larger cohorts. Lastly, it is important to note
that pneumonia was detected in patients included in the “mild” group,
but according to the Chinese Clinical Guidance for COVID-19 Pneumonia
Diagnosis and Treatment (7th edition) this group should be considered
“moderate”.

#### Significance

Lymphopenia and cytokine storm have been described to be
detrimental in many other infections including SARS-CoV1 and MERS-CoV.
However, it was necessary to confirm that this dramatic immune response
was also observed in the SARS-CoV2 infected patients. These results and
further validation of the N8R ratio as a predictor of disease severity
will contribute for the management of COVID19 patients and potential
development of therapies.

#### Credit

*Review by Pauline Hamon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Clinical and immunologic features in severe and moderate forms of Coronavirus Disease 2019

Chen et al. *medRxiv* [@doi:10.1101/2020.02.16.20023903]

#### Keywords

- severe disease
- lymphocytes
- cytokines
- IFNy
- CD4 Tcells
- HLA-DR CD8
- Tcells

#### Main Findings

This study retrospectively evaluated clinical, laboratory,
hematological, biochemical and immunologic data from 21 subjects
admitted to the hospital in Wuhan, China (late December/January) with
confirmed SARS-CoV-2 infection. The aim of the study was to compare
‘severe’ (n=11, \~64 years old) and ‘moderate’ (n=10, \~51 years old)
COVID-19 cases. Disease severity was defined by patients’ blood oxygen
level and respiratory output. They were classified as ‘severe’ if SpO2
93% or respiratory rates 30 per min.

In terms of the clinical laboratory measures, ‘severe’ patients had
higher CRP and ferritin, alanine and aspartate aminotransferases, and
lactate dehydrogenase but lower albumin concentrations.

The authors then compared plasma cytokine levels (ELISA) and immune cell
populations (PBMCs, Flow Cytometry). ‘Severe’ cases had higher levels of
IL-2R, IL-10, TNFa, and IL-6 (marginally significant). For the immune
cell counts, ‘severe’ group had higher neutrophils, HLA-DR+ CD8 T cells
and total B cells; and lower total lymphocytes, CD4 and CD8 T cells
(except for HLA-DR+), CD45RA Tregs, and IFNy-expressing CD4 T cells. No
significant differences were observed for IL-8, counts of NK cells,
CD45+RO Tregs, IFNy-expressing CD8 T and NK cells.

#### Limitations

Several potential limitations should be noted: 1) Blood samples were
collected 2 days post hospital admission and no data on viral loads were
available; 2) Most patients were administered medications (e.g.
corticosteroids), which could have affected lymphocyte counts.
Medications are briefly mentioned in the text of the manuscript; authors
should include medications as part of Table 1. 3) ‘Severe’ cases were
significantly older and 4/11 ‘severe’ patients died within 20 days.
Authors should consider a sensitivity analysis of biomarkers with the
adjustment for patients’ age.

#### Significance

Although the sample size was small, this paper presented a broad range
of clinical, biochemical, and immunologic data on patients with
COVID-19. One of the main findings is that SARS-CoV-2 may affect T
lymphocytes, primarily CD4+ T cells, resulting in decreased IFNy
production. Potentially, diminished T lymphocytes and elevated cytokines
can serve as biomarkers of severity of COVID-19.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### SARS-CoV-2 and SARS-CoV Spike-RBD Structure and Receptor Binding Comparison and Potential Implications on Neutralizing Antibody and Vaccine Development

Sun et al. *bioRxiv* [@doi:10.1101/2020.02.16.951723]

#### Keywords

- SARS-CoV
- SARS-CoV-2
- ACE2
- Spike (S) protein
- receptor binding domain (RBD)
- receptor binding motif (RBM)
- neutralizing antibody

#### Main Findings

This study compared the structure of SARS-CoV and SARS-CoV-2 Spike (S)
protein receptor binding domain (RBD) and interactions with ACE2 using
computational modeling, and interrogated cross-reactivity and
cross-neutralization of SARS-CoV-2 by antibodies against SARS-CoV. While
SARS-CoV and SARS-CoV-2 have over 70 % sequence homology and share the
same human receptor ACE2, the receptor binding motif (RBM) is only 50%
homologous.

Computational prediction of the SARS-CoV-2 and ACE2 interactions based
on the previous crystal structure data of SARS-CoV, and measurement of
binding affinities against human ACE2 using recombinant SARS-CoV and
SARS-CoV-2 S1 peptides, demonstrated similar binding of the two S1
peptides to ACE2, explaining the similar transmissibility of SARS-CoV
and SARS-CoV-2 and consistent with previous data (Wall et al Cell
2020).

The neutralization activity of SARS-CoV-specific rabbit polyclonal
antibodies were about two-order of magnitude less efficient to
neutralize SARS-CoV-2 than SARS-CoV, and four potently neutralizing
monoclonal antibodies against SARS-CoV had poor binding and neutralizing
activity against SARS-CoV-2. In contrast, 3 poor SARS-CoV-binding
monoclonal antibodies show some efficiency to bind and neutralize
SARS-CoV-2. The results suggest that that antibodies to more conserved
regions outside the RBM motif might possess better cross-protective
neutralizing activities between two strains.

#### Limitations

It would have been helpful to show the epitopes recognized by the
monoclonal antibodies tested on both SARS-CoV, SARS-CoV-2 to be able to
make predictions for induction of broadly neutralizing antibodies. The
data on monoclonal antibody competition with ACE2 for binding to
SARS-CoV RBD should have also included binding on SARS-CoV2, especially
for the three monoclonal antibodies that showed neutralization activity
for SARS-CoV2. Because of the less homology in RBM sequences between
viruses, it still may be possible that these antibodies would recognize
the ACE2 RBD in SARS-CoV-2.

#### Significance

It is noteworthy that immunization to mice and rabbit with SARS-CoV S1
or RBD protein could induce monoclonal antibodies to cross-bind and
cross-neutralize SARS-CoV-2 even if they are not ACE2-blocking. If these
types of antibodies could be found in human survivors or in the
asymptomatic populations as well, it might suggest that exposure to
previous Coronavirus strains could have induced cross-neutralizing
antibodies and resulted in the protection from severe symptoms in some
cases of SARS-CoV2.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Protection of Rhesus Macaque from SARS-Coronavirus challenge by recombinant adenovirus vaccine

Chen et al. *bioRxiv* [@doi:10.1101/2020.02.17.951939]

#### Keywords

- SARS-CoV-1
- rhesus macaque
- recombinant adenovirus vaccine

#### Main Findings

Rhesus macaques were immunized intramuscularly twice (week 0 and
week 4) with SV8000 carrying the information to express a S1-orf8
fusion protein and the N protein from the BJ01 strain of SARS-CoV-1.
By week 8, immunized animals had signs of immunological protection
(IgG and neutralization titers) against SARS-CoV-1 and were
protected against challenge with the PUMC-1 strain, with fewer
detectable symptoms of respiratory distress, lower viral load,
shorter periods of viral persistence, and less pathology in the
lungs compared to non-immunized animals.

#### Limitations

The authors should write clearer descriptions of the methods used in
this article. They do not describe how the IgG titers or
neutralization titers were determined. There are some issues with
the presentation of data, for example, in Figure 1a, y-axis should
not be Vmax; forming cells and 1d would benefit from showing error
bars. Furthermore, although I inferred that the animals were
challenged at week 8, the authors did not explicitly detail when the
animals were challenged. The authors should explain the design of
their vaccine, including the choice of antigens and vector. The
authors also do not include a description of the ethical use of
animals in their study.

#### Significance

The authors describe a vaccine for SARS-CoV-1 with no discussion of
possible implications for the current SARS-CoV-2 pandemic. Could a
similar vaccine be designed to protect against SARS-CoV-2 and would
the concerns regarding emerging viral mutations that the authors
describe as a limitation for SARS-CoV-1 also be true in the context
of SARS-CoV-2?

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Reduction and Functional Exhaustion of T cells in Patients with Coronavirus Disease 2019 (COVID-19)

[@doi:10.1101/2020.02.18.20024364]

#### Keywords

- T cell exhaustion
- T cell lymphopenia
- IL-6
- IL-10
- TNF-α

#### Main Findings

Based on a retrospective study of 522 COVID patients and 40
healthy controls from two hospitals in Wuhan, China, authors show both
age-dependent and clinical severity-dependent decrease in T cell numbers
with elderly patients and patients who are in ICU-care showing the most
dramatic decrease in T cell counts. Cytokine profiling of COVID patients
reveal that TNF-α, IL-6 and IL-10 are increased in infected patients
with patients in the ICU showing the highest levels. Interestingly,
these three cytokine levels were inversely correlated with T cell counts
and such inverse relationship was preserved throughout the disease
progression. Surface staining of exhaustion markers (PD-1 and Tim-3) and
flow cytometry of stained peripheral blood of 14 patients and 3 healthy
volunteers demonstrate that T cells of COVID patients have increased
expression of PD-1 with patients in ICU having the highest number of
CD8^+^PD-1^+^ cells than their counterparts in non-ICU groups.

#### Limitations

Compared to the number of patients, number of control (n=
40) is small and is not controlled for age. Additional data linking
inflammatory cytokines and the quality of the adaptive response
including humoral and antigen specific T cell response is much needed. T
cell exhaustion study relies on marker-dependent labeling of T cell
functionality of a very limited sample size (n=17)—a
functional/mechanistic study of these T cells from PBMCs would have
bolstered their claims.

#### Significance

Limited but contains interesting
implications. It is already known in literature that in the context of
acute respiratory viral infections CD8 T cells exhibit exhaustion-like
phenotypes which further underscores the importance of mechanistic
studies that can elucidate how COVID infection leads to lymphopenia and
T cell exhaustion-like phenotype.

However, as authors have noted, the data does point to an interesting
question: How these inflammatory cytokines (TNF-α, IL-6 and IL-10)
correlate with or affect effective viral immunity and what types of
cells produce these cytokines? Answering that question will help us
refine our targets for immune-modulatory therapies especially in
patients suffering from cytokine storms.

#### Credit

*This review by Chang Moon was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Clinical Characteristics of 25 death cases infected with COVID-19 pneumonia: a retrospective review of medical records in a single medical center, Wuhan, China

[@doi:10.1101/2020.02.19.20025239]

#### Keywords

- COVID-19
- pneumonia
- hypertension
- diabetes
- biomarker
- neutrophilia
- lymphopenia

#### Main Findings

Most common chronic conditions among 25 patients that died from
COVID-19 related respiratory failure were hypertension (64%) and
diabetes (40%). Disease progression was marked by progressive organ
failure, starting first with lung dysfunction, then heart (e.g.
increased cTnI and pro-BNP), followed by kidney (e.g. increased BUN,
Cr), and liver (e.g. ALT, AST). 72% of patients had neutrophilia and 88%
also had lymphopenia. General markers of inflammation were also
increased (e.g. PCT, D-Dimer, CRP, LDH, and SAA).

#### Limitations

The limitations of this study include small sample size and
lack of measurements for some tests for several patients. This study
would also have been stronger with comparison of the same measurements
to patients suffering from less severe disease to further validate and
correlate proposed biomarkers with disease severity.

#### Significance

This study identifies chronic conditions (i.e. hypertension
and diabetes) that strongly correlates with disease severity. In
addition to general markers of inflammation, the authors also identify
concomitant neutrophilia and lymphopenia among their cohort of patients.
This is a potentially interesting immunological finding because we would
typically expect increased lymphocytes during a viral infection.
Neutrophilia may also be contributing to cytokine storm. In addition,
PCT was elevated in 90.5% of patients, suggesting a role for sepsis or
secondary bacterial infection in COVID-19 related respiratory failure.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### SARS-CoV-2 infection does not significantly cause acute renal injury: an analysis of 116 hospitalized patients with COVID-19 in a single hospital, Wuhan, China

[@doi:10.1101/2020.02.19.20025288]

#### Keywords

- CoVID-19
- 2019-nCoV
- SARS-CoV-2
- kidney
- clinical
- longitudinal

#### Main Findings

-   Clinical data from 116 hospitalized CoVID-19 patients analyzed over
    4 weeks for correlation with renal injury. Comorbidities included
    chronic renal failure (CRF) in 5 patients (4.3%).

-   10.8% of patients with no prior kidney disease showed elevations in
    blood urea or creatinine, and 7.2% of patients with no prior kidney
    disease showed albuminuria.

-   Patients with pre-existing CRF underwent continuous renal
    replacement therapy (CRRT) alongside CoVID-19 treatment. Renal
    functions remained stable in these patients.

-   All 5 patients with CRF survived CoVID-19 therapy without
    progression to ARDS or worsening of CRF.

#### Limitations

-   Renal injury biomarkers in patients with incipient kidney
    abnormalities not tabulated separately, making overall data hard to
    interpret. It will be critical to separately examine kidney function
    (BUN, urine creatinine and eGFR) in patients that developed any
    kidney abnormalities (7.2\~10.8% of cohort).

-   No information on type of CoVID-19 therapy used across cohort; will
    be useful to correlate how treatment modality influences kidney
    function (and other parameters).

-   Invokes previous clinical-correlation studies that indicate low
    instances of kidney damage[@doi:10.1001/jama.2020.1585; @doi:10.1101/2020.02.06.20020974], but those studies did not track
    longitudinal urine samples for acute renal injury markers and viral
    shedding.

-   CRRT in patients with CRF is standard therapy irrespective of
    CoVID-19 status; it will be important to compare clinical parameters
    of these patients (n=5) with virus-naïve CRF patients (none in this
    study) to make any meaningful conclusions.

#### Significance

-   This study argues that renal impairment is uncommon in CoVID-19 and
    not associated with high mortaility, in stark contrast with a
    concurrent study [@doi:10.1101/2020.02.08.20021212].
    If supported by further studies, this argues kidney impairment is
    secondary to cytokine storm/inflammation-induced organ failure, and
    not due to direct viral replication.

-   Will be important to comprehensively characterize large-datasets of
    CoVID-19 patients to conclude if kidney function actively disrupted
    due to viral infection.


#### Credit

*Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Potential T-cell and B-Cell Epitopes of 2019-nCoV

[@doi:10.1101/2020.02.19.955484]

#### Keywords

- COVID-19
- vaccine
- epitopes
- spike protein
- MHC-I
- MHC-II
- neutralizing antibodies
- ACE2

#### Main Findings

The authors use 2 neural network algorithms, NetMHCpan4 and
MARIA, to identify regions within the COVID-19 genome that are
presentable by HLA. They identify 405 viral epitopes that are
presentable on MHC-I and MHC-II and validate using known epitopes from
SARS-CoV. To determine whether immune surveillance drives viral
mutations to evade MHC presentation, the authors analyzed 68 viral
genomes from 4 continents. They identified 93 point mutations that
occurred preferentially in regions predicted to be presented by MHC-I
(p=0.02) suggesting viral evolution to evade CD8 T-cell mediated
killing. 2 nonsense mutations were also identified that resulted in loss
of presentation of an associated antigen (FGDSVEEVL) predicted to be
good antigen for presentation across multiple HLA alleles.

To identify potential sites of neutralizing antibody binding, the
authors used homology modeling to the SARS-CoV’s spike protein (S
protein) to determine the putative structure of the CoV2 spike protein.
They used Discotope2 to identify antibody binding sites on the protein
surface in both the down and up conformations of the S protein. The
authors validate this approach by first identifying antibody binding
site in SARS-CoV S protein. In both the down and up conformation of the
CoV2 S protein, the authors identified a potential antibody binding site
on the S protein receptor binding domain (RBD) of the ACE2 receptor
(residues 440-460, 494-506). While RBDs in both SARS-CoV and CoV2 spike
proteins may be important for antibody binding, the authors note that
SARS-CoV has larger attack surfaces than CoV2. These results were later
validated on published crystal structures of the CoV2 S protein RBD and
human ACE2. Furthermore, analysis of 68 viral genomes did not identify
any mutations in this potential antibody binding site in CoV2.

Finally, the authors compile a list of potential peptide vaccine
candidates across the viral genome that can be presented by multiple HLA
alleles. Several of the peptides showed homology to SARS-CoV T-cell and
B-cell epitopes.

#### Limitations

While the authors used computational methods of validation,
primarily through multiple comparisons to published SARS-CoV structures
and epitopes, future work should include experimental validation of
putative T-cell and B-cell epitopes.

#### Significance

The authors identified potential T-cell and B-cell epitopes
that may be good candidates for peptide based vaccines against CoV2.
They also made interesting observations in comparing SARS-CoV and CoV2
potential antibody binding sites, noting that SARS-CoV had larger attack
surfaces for potential neutralizing antibody binding. One of the
highlights of this paper was the authors’ mutation analysis of 68 viral
genomes from 4 continents. This analysis not only validated their
computational method for identifying T-cell epitopes, but showed that
immune surveillance likely drives viral mutation in MHC-I binding
peptides. The smaller attack surface may point to potential mechanisms
of immune evasion by CoV2. However, absence of mutations in the RBD of
CoV2 and the small number of mutations in peptides presentable to T
cells suggests that vaccines against multiple epitopes could still
elicit robust immunity against CoV2.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Structure, Function, and Antigenicity of the SARSCoV-2 Spike Glycoprotein

Walls et al. *bioRxiv*. [@doi:10.1101/2020.02.19.956581] now [@doi:10.1016/j.cell.2020.02.058]

#### Keywords
- binding affinity
- antigenicity
- neutralizing antibody

#### Main Findings

The authors highlight a human angiotensin-converting enzyme 2
(hACE2), as a potential receptor used by the current Severe Acute
respiratory syndrome coronavirus-2 (SARS-CoV-2) as a host factor that
allows the virus target human cells. This virus-host interaction
facilitates the infection of human cells with a high affinity comparable
with SARS-CoV. The authors propose this mechanism as a probable
explanation of the efficient transmission of SARS-CoV-2 between humans.
Besides, Walls and colleagues described SARS-CoV-2 S glycoprotein S by
Cryo-EM along with neutralizing polyclonal response against SAR-CoV-2 S
from mice immunized with SAR-CoV and blocking SAR-CoV-2 S-mediated entry
into VeroE6 infected cells.**

#### Limitations

The SARS-CoV-2 depends on the cell factors ACE2 and TMPRSS2, this
last, according to a recent manuscript by Markus Hoffman et al., *Cell*,
2020. The authors used green monkey (VeroE6) and hamster (BHK) cell
lines in the experiments to drive its conclusions to humans; however, it
is well known the caucasian colon adenocarcinoma human cell line
(CaCo-2), highly express the hACE2 receptor as the TMPRSS2 protease as
well. In humans, ACE2 protein is highly expressed in the
gastrointestinal tract, which again, makes the CaCo-2 cell line suitable
for the following SARS-CoV-2 studies.


#### Significance

The results propose a functional receptor used by SARS-CoV-2 to
infect humans worldwide and defining two distinct conformations of spike
(S) glycoprotein by cryogenic electron microscopy (Cryo-EM). This study
might help establish a precedent for initial drug design and treatment
of the current global human coronavirus epidemic.

#### Credit
*Review by postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Breadth of concomitant immune responses underpinning viral clearance and patient recovery in a non-severe case of COVID-19

Thevarajan et al. *medRxiv* [@doi:10.1101/2020.02.20.20025841]

#### Keywords

- IgG
- IgM
- TfH cells
- NK cells
- SNP

#### Main Findings

The authors characterized the immune response in peripheral blood of a
47-year old COVID-19 patient.

SARS-CoV2 was detected in nasopharyngeal swab, sputum and faeces
samples, but not in urine, rectal swab, whole blood or throat swab. 7
days after symptom onset, the nasopharyngeal swab test turned negative,
at day 10 the radiography infiltrates were cleared and at day 13 the
patient became asymptomatic.

Immunofluorescence staining shows from day 7 the presence of
**COVID-19-binding IgG and IgM** antibodies in plasma, that increase
until day 20.

Flow cytometry on whole blood reveals a plasmablast peak at day 8, a
gradual increase in T follicular helper cells, stable HLA-DR^+^ NK
frequencies and decreased monocyte frequencies compared to healthy
counterparts. The expression of CD38 and HLA-DR peaked on T cells at D9
and was associated with higher production of cytotoxic mediators by
CD8^+^ T cells.

IL-6 and IL-8 were undetectable in plasma.

The authors further highlight the presence of the **IFITM3
SNP-rs12252-C/C variant** in this patient, which is associated with
higher susceptibility to influenza virus.

#### Limitations

These results need to be confirmed in additional patients.

COVID-19 patients have increased infiltration of macrophages in their
lungs [@doi:10.1101/2020.02.23.20026690]. Monitoring monocyte proportions in blood earlier in the
disease might help to evaluate their eventual migration to the lungs.

The stable concentration of HLA-DR^+^ NK cells in blood from day 7 is
not sufficient to rule out NK cell activation upon SARS-CoV2 infection.
In response to influenza A virus, NK cells express higher levels of
activation markers CD69 and CD38, proliferate better and display higher
cytotoxicity [@doi:10.3389/fimmu.2019.01116]. Assessing these parameters in COVID-19 patients is
required to better understand NK cell role in clearing this infection.

Neutralization potential of the COVID-19-binding IgG and IgM antibodies
should be assessed in future studies.

This patient was able to clear the virus, while presenting a SNP
associated with severe outcome following influenza infection. The
association between this SNP and outcome upon SARS-CoV2 infection should
be further investigated.

#### Significance

This study is among the first to describe the appearance of
COVID-19-binding IgG and IgM antibodies upon infection. The emergence of
new serological assays might contribute to monitor more precisely the
seroconversion kinetics of COVID-19 patients [@doi:10.1101/2020.03.17.20037713]. Further association
studies between IFITM3 SNP-rs12252-C/C variant and clinical data might
help to refine the COVID-19 outcome prediction tools.

#### Credit

*Review by Bérengère Salomé as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### The landscape of lung bronchoalveolar immune cells in COVID-19 revealed by single-cell RNA sequencing

Liao et al. *medRxiv* [@doi:10.1101/2020.02.23.20026690]

#### Keywords

- COVID-19
- SARS-CoV-2
- Broncho-alveolar lavage
- macrophages
- NK cells
- T cells
- cytokine storm
- scRNAseq

#### Main Findings

The authors performed single-cell RNA sequencing (scRNAseq) on
bronchoalveolar lavage fluid (BAL) from 6 COVID-19 patients (n=3 mild
cases, n=3 severe cases). Data was compared to previously generated
scRNAseq data from healthy donor lung tissue (n=8).

Clustering analysis of the 6 patients revealed distinct immune cell
organization between mild and severe disease. Specifically, they found
that transcriptional clusters annotated as tissue resident alveolar
macrophages were strongly reduced while monocytes-derived FCN1^+^SPP1^+^
inflammatory macrophages dominated the BAL of patients with severe
COVID19 diseases. They show that inflammatory macrophages upregulated
interferon-signaling genes, monocytes recruiting chemokines including
CCL2, CCL3, CCL4 as well as IL-6, TNF, IL-8 and profibrotic cytokine
TGF-β, while alveolar macrophages expressed lipid metabolism genes, such
as PPARG.

The lymphoid compartment was overall enriched in lungs from patients.
Clonally expanded CD8 T cells were enriched in mild cases suggesting
that CD8 T cells contribute to viral clearance as in Flu infection,
whereas proliferating T cells were enriched in severe cases.

SARS-CoV-2 viral transcripts were detected in severe patients, but
considered here as ambient contaminations.

#### Limitations

These results are based on samples from 6 patients and should therefore
be confirmed in the future in additional patients. Longitudinal
monitoring of BAL during disease progression or resolution would have
been most useful.

The mechanisms underlying the skewing of the macrophage compartment in
patients towards inflammatory macrophages should be investigated in
future studies.

Deeper characterization of the lymphoid subsets is required. The
composition of the “proliferating” cluster and how these cells differ
from conventional T cell clusters should be assessed. NK and CD8 T cell
transcriptomic profile, in particular the expression of cytotoxic
mediator and immune checkpoint transcripts, should be compared between
healthy and diseased lesions.

#### Significance

COVID-19 induces a robust inflammatory cytokine storm in patients that
contributes to severe lung tissue damage and ARDS [@doi:10/ggjfnn]. Accumulation of
monocyte-derived inflammatory macrophages at the expense of Alveolar
macrophages known to play an anti-inflammatory role following
respiratory viral infection, in part through the PPARγ pathway [@doi:10.3389/fimmu.2018.01777; @doi:10.1128/JVI.00030-19] are
likely contributing to lung tissue injuries. These data suggest that
reduction of monocyte accumulation in the lung tissues could help
modulate COVID-19-induced inflammation. Further analysis of lymphoid
subsets is required to understand the contribution of adaptive immunity
to disease outcome.

#### Credit

*Review by Bérengère Salomé and Assaf Magen as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Can routine laboratory tests discriminate 2019 novel coronavirus infected pneumonia from other community-acquired pneumonia?

Pan et al. *medRxiv* [@doi:10.1101/2020.02.25.20024711]

#### Keywords

- Routine laboratory testing

#### Main Findings

In an attempt to use standard laboratory testing for the discrimination
between “Novel Coronavirus Infected Pneumonia” (NCIP) and a usual
community acquired pneumonia (CAP), the authors compared laboratory
testing results of 84 NCIP patients with those of a historical group of
316 CAP patients from 2018 naturally COVID-19 negative. The authors
describe significantly lower white blood- as well as red blood- and
platelet counts in NCIP patients. When analyzing differential blood
counts, lower absolute counts were measured in all subsets of NCIP
patients. With regard to clinical chemistry parameters, they found
increased AST and bilirubin in NCIP patients as compared to CAP
patients.

#### Limitations

The authors claim to describe a simple method to rapidly assess a
pre-test probability for NCIP. However, the study has substantial
weakpoints. The deviation in clinical laboratory values in NCIP patients
described here can usually be observed in severely ill patients. The
authors do not comment on how severely ill the patients tested here were
in comparison to the historical control. Thus, the conclusion that the
tests discriminate between CAP and NCIP lacks justification.

#### Significance

The article strives to compare initial laboratory testing results in
patients with COVID-19 pneumonia as compared to patients with a usual
community acquired pneumonia. The implications of this study for the
current clinical situation seem restricted due to a lack in clinical
information and the use of a control group that might not be
appropriate.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Correlation Analysis Between Disease Severity and Inflammation-related Parameters in Patients with COVID-19 Pneumonia

[@doi:10.1101/2020.02.25.20025643]

#### Keywords
- cytokine
- COVID-19 pneumonia
- severity
- disease progression

#### Main Findings

This study is a cross-sectional analysis of 100 patients with
COVID-19 pneumonia, divided into mild (n = 34), severe (n = 34), and
critical (n = 32) disease status based on clinical definitions.

The criteria used to define disease severity are as follows:

1.  *Severe* – any of the following: respiratory distress or respiratory
    rate ≥ 30 respirations/minute; oxygen saturation ≤ 93% at rest;
    oxygen partial pressure (PaO2)/oxygen concentration (FiO2) in
    arterial blood ≤ 300mmHg, progression of disease on imaging
    to &gt;50% lung involvement in the short term.

2.  *Critical* – any of the following: respiratory failure that requires
    mechanical ventilation; shock; other organ failure that requires
    treatment in the ICU.

3.  Patients with pneumonia who test positive for COVID-19 who do not
    have the symptoms delineated above are considered *mild*.

Peripheral blood inflammatory markers were correlated to disease status.
Disease severity was significantly associated with levels of IL-2R,
IL-6, IL-8, IL-10, TNF-α, CRP, ferroprotein, and procalcitonin. Total
WBC count, lymphocyte count, neutrophil count, and eosinophil count were
also significantly correlated with disease status. Since this is a
retrospective, cross-sectional study of clinical laboratory values,
these data may be extrapolated for clinical decision making, but without
studies of underlying cellular causes of these changes this study does
not contribute to a deeper understanding of SARS-CoV-2 interactions with
the immune system.

It is also notable that the mean age of patients in the mild group was
significantly different from the mean ages of patients designated as
severe or critical (p &lt; 0.001). The mean patient age was not
significantly different between the severe and critical groups. However,
IL-6, IL-8, procalcitonin (Table 2), CRP, ferroprotein (Figure 3A, 3B),
WBC count, and neutrophil count (Figure 4A, 4B) were all significantly
elevated in the critical group compared to severe. These data suggest
underlying differences in COVID-19 progression that is unrelated to age.

#### Significance

Given the inflammatory profile outlined in this study,
patients who have mild or severe COVID-19 pneumonia, who *also* have any
elevations in the inflammatory biomarkers listed above, should be
closely monitored for potential progression to critical status.

#### Credit

*This review by JJF was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### An Effective CTL Peptide Vaccine for Ebola Zaire Based on Survivors' CD8+ Targeting of a Particular Nucleocapsid Protein Epitope with Potential Implications for COVID-19 Vaccine Design

Herst et al. *bioRxiv* [@doi:10.1101/2020.02.25.963546]

#### Keywords

- Peptide vaccine
- Ebolavirus
- nucleocapsid
- epitope
- vaccine design
- microsphere

#### Main Findings

Vaccination of mice with a single dose of a 9-amino-acid peptide NP44-52
located in a conserved region of ebolavirus (EBOV) nucleocapsid protein
(NP) confers CD8+ T-cell-mediated immunity against mouse adapted EBOV
(maEBOV). Bioinformatic analyses predict multiple conserved CD8+ T cell
epitopes in the SARS-CoV-2 NP, suggesting that a similar approach may be
feasible for vaccine design against SARS-CoV-2.

The authors focus on a site within a 20-peptide region of EBOV NP which
was commonly targeted by CD8+ T cells in a group of EBOV survivors
carrying the HLA-A\*30:01:01 allele. To justify the testing of specific
vaccine epitopes in a mouse challenge setting, the authors cite known
examples of human pathogen-derived peptide antigens that are also
recognized by C57BL/6 mice, as well as existing data surrounding known
mouse immunogenicity of peptides related to this EBOV NP region. Testing
3 distinct 9mer peptides over an 11 amino-acid window and comparing to
vaccination with the 11mer with a T-cell reactivity readout demonstrated
that optimizing peptide length and position for immunogenicity may be
crucial, likely due to suboptimal peptide processing and MHC-class-I
loading.

Vaccines for maEBOV challenge studies were constructed by packaging
NP44-52 in d,l poly(lactic-co-glycolic) acid microspheres. CpG was also
packaged within the microspheres, while Monophosphoryl Lipid A (a TLR4
ligand) was added to the injectate solution. A second peptide consisting
of a predicted MHC-II epitope from the EBOV VG19 protein was added using
a separate population of microspheres, and the formulation was injected
by intraperitoneal administration. The vaccine was protective against a
range of maEBOV doses up to at least 10,000 PFU. Survival was
anticorrelated with levels of IL6, MCP-1 (CCL2), IL9, and GM-CSF, which
recapitulated trends seen in human EBOV infection.

While HLA-A\*30:01:01 is only present in a minority of humans, the
authors state that MHC binding algorithms predict NP44-52 to be a strong
binder of a set of more common HLA-A\*02 alleles. The authors predict
that a peptide vaccine based on the proposed formulation could elicit
responses in up to 50% of people in Sudan or 30% of people in North
America.

SARS-CoV-2 NP, meanwhile, has conserved regions which may provide
peptide-vaccine candidates. Scanning the SARS-CoV-2 NP sequence for
HLA-binding 9mers identified 53 peptides with predicted binding affinity
&lt; 500nM, including peptides that are predicted to bind to HLA-class-I
alleles of 97% of humans, 7 of which have previously been tested
*in-vitro*.

The results support previously appreciated correlations between certain
cytokines and disease severity, specifically IL6 which relates to
multiple trial therapies. Prediction of HLA-class-I binding of
SARS-CoV-2 NP peptides suggests the plausibility of a peptide vaccine
targeting conserved regions of SARS-CoV-2 NP although further validation
in previously infected patient samples will be essential.

#### Credit

*Review by Andrew M. Leader as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Epitope-based peptide vaccines predicted against novel coronavirus disease caused by SARS-CoV-2

Li et al. *bioRxiv*. [@doi:10.1101/2020.02.25.965434]

#### Keywords
- SARS-CoV-2
- immune-informatics
- vaccine design
- T cell epitope
- B cell epitope

#### Main Findings

This study employs a series of bioinformatic
pipelines to identify T and B cell epitopes on spike (S) protein of
SARS-CoV-2 and assess their properties for vaccine potential. To
identify B cell epitopes, they assessed structural accessibility,
hydrophilicity, and beta-turn and flexibility which are all factors that
promote their targeting by antibodies. To identify T cell epitopes, they
filtered for peptides with high antigenicity score and capacity to bind
3 or more MHC alleles. Using the protein digest server, they also
demonstrated that their identified T and B cell epitopes are stable,
having multiple non-digesting enzymes per epitope. Epitopes were also
determined to be non-allergenic and non-toxin as assessed by Allergen FP
1.0 and ToxinPred, respectively. For T cell epitopes, they assessed the
strength of epitope-HLA interaction via PepSite. Overall, they predict
four B cell and eleven T cell epitopes (two MHC I and nine MHC II
binding) to pass stringent computational thresholds as candidates for
vaccine development. Furthermore, they performed sequence alignment
between all identified SARS-CoV-2 S protein mutations and predicted
epitopes, and showed that the epitopes are conserved across 134 isolates
from 38 locations worldwide. However, they report that these conserved
epitopes may soon become obsolete given the known mutation rate of
related SARS-CoV is estimated to be 4x10-4/site/year, underscoring the
urgency of anti-viral vaccine development.

#### Limitations

While spike (S) protein may have a
critical role in viral entry into host cells and their epitope
prediction criterion were comprehensive, this study did not examine
other candidate SARS-CoV-2 proteins. This point is particularly
important given that a single epitope may not be sufficient to induce
robust immune memory, and recent approaches involve multi-epitope
vaccine design. Furthermore, their study only included a direct
implementation of various published methods, but did not validate
individual bioinformatic tools with controls to demonstrate robustness.
Finally, it is critical that these predicted epitopes are experimentally
validated before any conclusions can be drawn about their potential as
vaccine candidates or their clinical efficacy.

#### Significance

This study provides a computational framework to
rapidly identify epitopes that may serve as potential vaccine candidates
for treating SARS-CoV-2.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### The definition and risks of Cytokine Release Syndrome-Like in 11 COVID-19-Infected Pneumonia critically ill patients: Disease Characteristics and Retrospective Analysis

Wang Jr. et al. *medRxiv*.
[@doi:10.1101/2020.02.26.20026989]

#### Keywords
- Cytokine release syndrome (CRS)
- biomarkers
- ARDS
- IL-6
- lymphopenia

#### Main Findings

This study describes the occurrence of a cytokine release syndrome-like
(CRSL) toxicity in ICU patients with COVID-19 pneumonia. The median time
from first symptom to acute respiratory distress syndrome (ARDS) was 10
days. All patients had decreased CD3, CD4 and CD8 cells, and a
significant increase of serum IL-6. Furthermore, 91% had decreased NK
cells. The changes in IL-6 levels preceded those in CD4 and CD8 cell
counts. All of these parameters correlated with the area of pulmonary
inflammation in CT scan images. Mechanical ventilation increased the
numbers of CD4 and CD8 cells, while decreasing the levels of IL-6, and
improving the immunological parameters.

#### Limitations

The number of patients included in this retrospective single center
study is small (n=11), and the follow-up period very short (25 days).
Eight of the eleven patients were described as having CRSL, and were
treated by intubation (7) or ECMO (2). Nine patients were still in the
intensive care unit at the time of publication of this article, so their
disease outcome is unknown.

#### Significance

The authors define a cytokine release syndrome-like toxicity in patients
with COVID-19 with clinical radiological and immunological criteria: 1)
decrease of circulating CD4, CD8 and NK cells; 2) substantial increase
of IL-6 in peripheral blood; 3) continuous fever; 4) organ and tissue
damage. This event seems to occur very often in critically ill patients
with COVID-19 pneumonia. Interestingly, the increase of IL-6 in the
peripheral blood preceded other laboratory alterations, thus, IL-6 might
be an early biomarker for the severity of COVID-19 pneumonia. The
manuscript will require considerable editing for organization and
clarity.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Clinical characteristics of 36 non-survivors with COVID-19 in Wuhan, China

Huang et al. *medRxiv*. [@doi:10.1101/2020.02.27.20029009]

#### Keywords

- Clinical Characteristics
- Non Survivors
- retrospective study

#### Main Findings

This is a simple study reporting clinical
characteristics of patients who did not survive COVID-19. All patients
(mean age=69.22 years) had acute respiratory distress syndrome (ARDS)
and their median time from onset to ARDS was 11 days. The median time
from onset to death was 17 days. Most patients were older male (70%
male) with co-morbidities and only 11 % were smokers. 75% patients
showed bilateral pneumonia. Many patients had chronic diseases,
including hypertension (58.33%). cardiovascular disease (22.22%) and
diabetes (19.44%). Typical clinical feature measured in these patients
includes lymphopenia and elevated markers of inflammation.

#### Limitations

As noted by the authors, the conclusions of this study are
very limited because this is single-centered study focusing on a small
cohort of patients who did not survive. Many clinical parameters
observed by the authors (such* as increase levels of serum CRP, PCT,
IL-6) have also been described in other COVID19 patients who survived
the infection

#### Significance

This study is essentially descriptive and may be useful for
clinical teams monitoring COVID19 patients.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Risk Factors Related to Hepatic Injury in Patients with Corona Virus Disease 2019

[@doi:10.1101/2020.02.28.20028514]

#### Keywords

- COVID-related Hepatic Injury

#### Main Findings

Based on a retrospective study of 85 hospitalized COVID
patients in a Beijing hospital, authors showed that patients with
elevated ALT levels (n = 33) were characterized by significantly higher
levels of lactic acid and CRP as well as lymphopenia and hypoalbuminemia
compared to their counterparts with normal ALT levels. Proportion of
severe and critical patients in the ALT elevation group was
significantly higher than that of normal ALT group. Multivariate
logistic regression performed on clinical factors related to ALT
elevation showed that CRP $\geq$ 20mg/L and low lymphocyte count
(&lt;1.1\*10\^9 cells/L) were independently related to ALT elevation—a
finding that led the authors to suggest cytokine storm as a major
mechanism of liver damage.

#### Limitations

The article’s most attractive claim that liver damage seen in COVID
patients is caused by cytokine storm (rather than direct infection of
the liver) hinges solely on their multivariate regression analysis.
Without further mechanistic studies a) demonstrating how high levels of
inflammatory cytokines can induce liver damage and b) contrasting types
of liver damage incurred by direct infection of the liver vs.
system-wide elevation of inflammatory cytokines, their claim remains
thin. It is also worth noting that six of their elevated ALT group
(n=33) had a history of liver disease (i.e. HBV infection, alcoholic
liver disease, fatty liver) which can confound their effort to pin down
the cause of hepatic injury to COVID.

#### Significance

Limited. This article confirms a rich body
of literature describing liver damage and lymphopenia in COVID patients.

#### Credit

*Review by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Detectable serum SARS-CoV-2 viral load (RNAaemia) is closely associated with drastically elevated interleukin 6 (IL-6) level in critically ill COVID-19 patients

[@doi:10.1101/2020.02.29.20029520]

#### Keywords

- ARDS
- interleukin-6 (IL-6)
- procalcitonin (PCT)
- pro-inflammatory cytokines
- SARS-CoV-2 RNAaemia

#### Main Findings

48 adult patients diagnosed with Covid19 according to Chinese guidelines
for Covid19 diagnosis and treatment version 6 were included in this
study. Patients were further sub-divided into three groups based on
clinical symptoms and disease severity: (1) mild, positive Covid19 qPCR
with no or mild clinical symptoms (fever; respiratory; radiological
abnormalities); (2) severe, at least one of the following: shortness of
breath/respiratory rate &gt;30/min, oxygen saturation SaO~2~&lt;93%,
Horowitz index paO2/FiO~2~ &lt; 300 mmHg (indicating moderate pulmonary
damage); and (3) critically ill, at least one additional complicating
factor: respiratory failure with need for mechanical ventilation;
systemic shock; multi-organ failure and transfer to ICU. Serum samples
and throat-swaps were collected from all 48 patients enrolled.
SARS-CoV-2 RNA was assessed by qPCR with positive results being defined
as Ct values &lt; 40, and serum interleukin-6 (IL-6) was quantified
using a commercially available detection kit. Briefly, patient
characteristics in this study confirm previous reports suggesting that
higher age and comorbidities are significant risk factors of clinical
severity. Of note, 5 out of 48 of patients (10.41%), all in the
critically ill category, were found to have detectable serum SARS-CoV-2
RNA levels, so-called RNAaemia. Moreover, serum IL-6 levels in these
patients were found to be substantially higher and this correlated with
the presence of detectable SARS-CoV-2 RNA levels. The authors
hypothesize that viral RNA might be released from acutely damages
tissues in moribund patients during the course of Covid19 and that
RNaemia along with IL-6 could potentially be used as a prognostic
marker.

#### Limitations

While this group’s report generally confirms some of the major findings
of a more extensive study, published in early February 2020, [@doi:10/ggjfnn],
there are limitations that should be taken into account. First, the
number of patients enrolled is relatively small; second, interpretation
of these data would benefit from inclusion of information about study
specifics as well as providing relevant data on the clinical course of
these patients other than the fact that some were admitted to ICU (i.e.
demographics on how many patients needed respiratory support, dialysis,
APACHE Ii/III or other standard ICU scores as robust prognostic markers
for mortality etc). It also remains unclear at which time point the
serum samples were taken, i.e. whether at admission, when the diagnosis
was made or during the course of the hospital stay (and potentially
after onset of therapy, which could have affected both IL-6 and RNA
levels). The methods section lacks important information on the qPCR
protocol employed, including primers and cycling conditions used. From a
technical point of view, Ct values &gt;35 seem somewhat non-specific
(although Ct &lt;40 was defined as the CDC cutoff as well) indicating
that serum RNA levels are probably very low, therefore stressing the
need for highly specific primers and high qPCR efficiency. In addition,
the statistical tests used (t-tests, according to the methods section)
do not seem appropriate as the organ-specific data such as BUN and
troponin T values seem to be not normally distributed across groups (n=
5 RNAaemia+ vs. n= 43 RNAaemia-). Given the range of standard deviations
and the differences in patient sample size, it is difficult to believe
that these data are statistically significantly different.

#### Significance

This study is very rudimentary and lacks a lot of relevant clinical
details. However, it corroborates some previously published observations
regarding RNAemia and IL-6 by another group. Generally, regarding future
studies, it would be important to address the question of IL-6 and other
inflammatory cytokine dynamics in relation to Covid19 disease kinetics
(high levels of IL-6, IL-8 and plasma leukotriene were shown to have
prognostic value at the onset of ARDS ; serum IL-2 and IL-15 have been
associated with mortality; reviewed by Chen W & Ware L, Clin Transl Med.
2015 [@doi:10.1186/s40169-015-0065-2]).

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Lymphopenia predicts disease severity of COVID-19: a descriptive and predictive study

[@doi:10.1101/2020.03.01.20029074]

#### Keywords

- Lymphopenia

#### Main Findings
Based on a retrospective study of 162 COVID patients from a
local hospital in Wuhan, China, the authors show an inverse correlation
between lymphocyte % (LYM%) of patients and their disease severity. The
authors have also tracked LYM% of 70 cases (15 deaths; 15 severe; 40
moderate) throughout the disease progression with fatal cases showing no
recovery of lymphocytes ( &lt;5%) even after 17-19 days post-onset. The
temporal data of LYM % in COVID patients was used to construct a
Time-Lymphocyte% model which is used to categorize and predict patients’
disease severity and progression. The model was validated using 92
hospitalized cases and kappa statistic test was used to assess agreement
between predicted disease severity and the assigned clinical severity (k
= 0.49).

#### Limitations

Time-Lymphocyte % Model (TLM) that authors have proposed as
a predictive model for clinical severity is very simple in its
construction and derives from correlative data of 162 patients. In order
for the model to be of use, it needs validation using a far more robust
data set and possibly a mechanistic study on how COVID leads to
lymphopenia in the first place. In addition, it should be noted that no
statistical test assessing significance of LYM % values between disease
severities was performed.

#### Significance

This article is of limited significance as
it simply reports similar descriptions of COVID patients made in
previous literature that severe cases are characterized by lymphopenia.

#### Credit

*Review by Chang Moon as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### The potential role of IL-6 in monitoring severe case of coronavirus disease 2019

Liu et al. *medRxiv*. [@doi:10.1101/2020.03.01.20029769]

#### Keywords
- Cytokine Release Syndrome
- lymphocytopenia
- IL-6
- CRP
- COVID19
- pneumonia

#### Main Findings

Study on blood biomarkers on 80 COVID19 patients (69 severe
and 11 non-severe). Patients with severe symptoms at admission
(baseline) showed obvious lymphocytopenia and significantly increased
interleukin-6 (IL-6) and CRP, which was positively correlated with
symptoms severity. IL-6 at baseline positively correlates with CRP, LDH,
ferritin and D-Dimer abundance in blood.

Longitudinal analysis of 30 patients (before and after treatment) showed
significant reduction of IL-6 in remission cases.

#### Limitations

Limited sample size at baseline, especially for the
non-severe leads to question on representativeness. The longitudinal
study method is not described in detail and suffers from
non-standardized treatment. Limited panel of pro-inflammatory cytokine
was analyzed. Patients with severe disease show a wide range of altered
blood composition and biomarkers of inflammation, as well as differences
in disease course (53.6% were cured, about 10% developed acute
respiratory distress syndrome). The authors comment on associations
between IL-6 levels and outcomes, but these were not statistically
significant (maybe due to the number of patients, non-standardized
treatments, etc.) and data is not shown. Prognostic biomarkers could
have been better explored. Study lacks multivariate analysis.

#### Significance

IL-6 could be used as a pharmacodynamic
marker of disease severity. Cytokine Release Syndrome (CRS) is a
well-known side effect for CAR-T cancer therapy and there are several
effective drugs to manage CRS. Drugs used to manage CRS could be tested
to treat the most severe cases of COVID19.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Clinical and Laboratory Profiles of 75 Hospitalized Patients with Novel Coronavirus Disease 2019 in Hefei, China

Zhao et al. *medRxiv*. [@doi:10.1101/2020.03.01.20029785]

#### Keywords

- Routine laboratory testing

#### Main Findings

The authors of this study provide a comprehensive analysis of clinical
laboratory assessments in 75 patients (median age 47 year old)
hospitalized for Corona virus infection in China measuring differential
blood counts including T-cell subsets (CD4, CD8), coagulation function,
basic blood chemistry, of infection-related biomarkers including CRP,
Procalcitonin (PCT) (Precursor of calcitonin that increases during
bacterial infection or tissue injury), IL-6 and erythrocyte
sedimentation rate as well as clinical parameters. Among the most common
hematological changes they found increased neutrophils, reduced CD4 and
CD8 lymphocytes, increased LDH, CRP and PCT

When looking at patients with elevated IL-6, the authors describe
significantly reduced CD4 and CD8 lymphocyte counts and elevated CRP and
PCT levels were significantly increased in infected patients suggesting
that increased IL-6 may correlate well with disease severity in COVID-19
infections

#### Limitations

The authors performed an early assessment of clinical standard
parameters in patients infected with COVID-19. Overall, the number of
cases (75) is rather low and the snapshot approach does not inform about
dynamics and thus potential relevance in the assessment of treatment
options in this group of patients.

#### Significance

The article summarizes provides a good summary of some of the common
changes in immune cells inflammatory cytokines in patients with a
COVID-19 infection and. Understanding how these changes can help predict
severity of disease and guide therapy including IL-6 cytokine receptor
blockade using Tocilizumab or Sarilumab will be important to explore.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Exuberant elevation of IP-10, MCP-3 and IL-1ra during SARS-CoV-2 infection is associated with disease severity and fatal outcome

Yang et al. *medRxiv* [@doi:10.1101/2020.03.02.20029975]

#### Keywords

- cytokine
- IP-10
- MCP-3
- IL-1Ra
- lymphocyte
- neutrophil
- stratification
- disease severity
- viral load
- lung function
- complications
- clinical data

#### Summary

Plasma cytokine analysis (48 cytokines) was
performed on COVID-19 patient plasma samples, who were sub-stratified as
severe (N=34), moderate (N=19), and compared to healthy controls (N=8).
Patients were monitored for up to 24 days after illness onset: viral
load (qRT-PCR), cytokine (multiplex on subset of patients), lab tests,
and epidemiological/clinical characteristics of patients were reported.



#### Main Findings

-   Many elevated cytokines with COVID-19 onset compared to healthy
    controls (IFNy, IL-1Ra, IL-2Ra, IL-6, IL-10, IL-18, HGF, MCP-3, MIG,
    M-CSF, G-CSF, MIG-1a, and IP-10).

-   IP-10, IL-1Ra, and MCP-3 (esp. together) were associated with
    disease severity and fatal outcome.

-   IP-10 was correlated to patient viral load (r=0.3006, p=0.0075).

-   IP-10, IL-1Ra, and MCP-3 were correlated to loss of lung function
    (PaO~2~/FaO~2~ (arterial/atmospheric O2) and Murray Score (lung
    injury) with MCP-3 being the most correlated (r=0.4104 p&lt;0.0001
    and r=0.5107 p&lt;0.0001 respectively).

-   Viral load (Lower Ct Value from qRT-PCR) was associated with
    upregulated IP-10 only (not IL-1Ra or MCP-3) and was mildly
    correlated with decreased lung function: PaO~2~/FaO~2~
    (arterial/atmospheric O2) and Murray Score (lung injury).

-   Lymphopenia (decreased CD4 and CD8 T cells) and increased neutrophil
    correlated w/ severe patients.

-   Complications were associated with COVID severity (ARDS, hepatic
    insufficiency, renal insufficiency).

#### Limitations

Collection time of clinical data and lab results
not reported directly (likely 4 days (2,6) after illness onset), making
it very difficult to determine if cytokines were predictive of patient
outcome or reflective of patient compensatory immune response (likely
the latter). Small N for cytokine analysis (N=2 fatal and N=5
severe/critical, and N=7 moderate or discharged). Viral treatment
strategy not clearly outlined.

#### Expanded Results

*NOTE*: Moderate COVID-19 was classified by fever, respiratory
manifestations, and radiological findings consistent with pneumonia
while severe patients had one or more of the following: 1) respiratory
distraction, resting O2 saturation, or 3) arterial PaO2/FiO2 *&lt;* 300
mmHg.

Cytokine Results (Human Cytokine Screening Panel, Bio-Rad):

-   **Significant elevation of cytokines observed in COVID patients
    compared to healthy controls: IFNy, IL-1Ra, IL-2Ra, IL-6, IL-10,
    IL-18, HGF, MCP-3, MIG, M-CSF, G-CSF, MIG-1a, and IP-10.**

-   Severity was correlated **with increase in measured IP-10, MCP-3,
    and IL-Ra** as measure by area under the curve analysis during
    sample timecourse (2-24 days after illness onset).

-   IL-1Ra incr. significant 0-7 days after onset, MCP-3 signif.
    upregulated throughout observation timecourse, and IP-10 increased
    and upregulated throughout (trending downwards over time).

-   **The three cytokines together (IP-10, IL-1Ra, and MCP-3 AUC) served
    as the best predictors of disease deterioration and fatal outcome.**

-   No significant differences between moderate/severe observed between
    groups in IL-2Ra, IL-6, IL-10, IL-18, CTACK, G-CSF, HGF, M-CSF,
    MIP-1a, MIG, and IFNy at any timepoints.

-   **Viral load (Lower Ct Value from qRT-PCR) was associated with
    upregulated IP-10 only (not IL-1Ra or MCP-3) and was highly
    correlated with decreased lung function: PaO~2~/FaO~2~
    (arterial/atmospheric O2) and Murray Score (lung injury).**

-   **Antibodies against these cytokines (esp. anti-IP-10) may serve as
    a potential treatment for amelioration of COVID-19 (and associated
    ARDS).**

Lab results:

-   **Decreased lymphocytes (%) in all patients – lymphopenia corr. w/
    severe patients**

    -   **Decreased CD4 and CD8 T cells** – no monocyte or
        eosino/basophil % measured

-   **Increased neutrophils (%)**

-   Increased BUN (mmol/L) – other kidney markers, liver markers, and
    LDH were not significantly different between groups and were not
    compared to healthy controls.

Clinical features (between moderate vs. severe patient groups):

-   **Complications were associated with severity (ARDS, hepatic
    insufficiency, renal insufficiency).**

-   Coexisting conditions between groups were not significantly
    different (chronic heart/lung/renal/liver disease, diabetes, or
    cancer) and patient time courses (onset to admission and onset to
    viral tx) also not significantly different – 4 days (2, 6) on
    average for admission and 4 (3,7) for antiviral.

-   Increased corticosteroids and mechanical/ invasive mechanical
    ventilation in severe patients.

-   Increased median age in severe group (Median (Range = 63.5 (42-74)
    vs. 51 (22-78)) and patients *&gt;* 60 yrs had higher ratio of
    severe patients as compared patients 16-59 yrs.

-   Higher incidence of fever is severe patients (91.2 vs. 68.4%),
    myalgia (57.7 vs. 48.1%), and chill (17.6% vs. 0%).

-   No differences in cough, headache, nausea/vomiting, or diarrhea.

#### Significance

Outline of pathological time course (implicating innate
immunity esp.) and identification key cytokines associated with disease
severity and prognosis (+ comorbidities). Anti-IP-10 as a possible
therapeutic intervention (ex: Eldelumab).

#### Credit

*Review by Natalie Vaninov as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Antibody responses to SARS-CoV-2 in patients of novel coronavirus disease 2019

Zhao Jr. et al. *medRxiv*. [@doi:10.1101/2020.03.02.20030189]

#### Keywords
- SARS-CoV-2 IgG
- seroconversion rate
- total Ab
- Ig
- IgM

#### Main Findings

This study examined antibody responses in
the blood of COVID-19 patients during the early SARS CoV2 outbreak in
China. Total 535 plasma samples were collected from 173 patients (51.4%
female) and were tested for seroconversion rate using ELISA. Authors
also compared the sensitivity of RNA and antibody tests over the course
of the disease . The key findings are:

-   Among 173 patients, the seroconversion rates for total antibody
    (Ab), IgM and IgG were 93.1% (161/173), 82.7% (143/173) and 64.7%
    (112/173), respectively.

-   The seroconversion sequentially appeared for Ab, IgM and then IgG,
    with a median time of 11, 12 and 14 days, respectively. Overall, the
    seroconversion of Ab was significantly quicker than that of IgM (p =
    0.012) and IgG (p &lt; 0.001). Comparisons of seroconversion rates
    between critical and non-critical patients did not reveal any
    significant differences.

-   RNA tests had higher sensitivity in early phase and within 7 days of
    disease onset than antibody assays (66.7% Vs 38.3% respectively).

-   The sensitivity of the Ab assays was higher 8 days after disease
    onset, reached 90% at day 13 and 100% at later time points (15-39
    days). In contrast, RNA was only detectable in 45.5% of samples at
    days 15-39.

-   In patients with undetectable RNA in nasal samples collected during
    day 1-3, day 4-7, day 8-14 and day 15-39 since disease onset, 28.6%
    (2/7), 53.6% (15/28), 98.2% (56/57) and 100% (30/30) had detectable
    total Ab titers respectively Combining RNA and antibody tests
    significantly raised the sensitivity for detecting COVID-19 patients
    in different stages of the disease (p &lt; 0.001).

-   There was a strong positive correlation between clinical severity
    and antibody titer 2-weeks after illness onset.

-   Dynamic profiling of viral RNA and antibodies in representative
    COVID-19 patients (n=9) since onset of disease revealed that
    antibodies may not be sufficient to clear the virus. It should be
    noted that increases in of antibody titers were not always
    accompanied by RNA clearance.

#### Limitations

Because different types of ELISA assays were used for
determining antibody concentrations at different time points after
disease onset, sequential seroconversion of total Ab, IgM and IgG may
not represent actual temporal differences but rather differences in the
affinities of the assays used. Also, due to the lack of blood samples
collected from patients in the later stage of illness, how long the
antibodies could last remain unknown. For investigative dynamics of
antibodies, more samples were required.

#### Significance

Total and IgG antibody titers could be used to understand
the epidemiology of SARS CoV-2 infection and to assist in determining
the level of humoral immune response in patients.

The findings provide strong clinical evidence for routine serological
and RNA testing in the diagnosis and clinical management of COVID-19
patients. The understanding of antibody responses and their half-life
during and after SARS CoV2 infection is important and warrants further
investigations.

#### Credit

*This review was undertaken by Zafar Mahmood and edited by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Restoration of leukomonocyte counts is associated with viral clearance in COVID-19 hospitalized patients

Chen et al. *medRxiv* [@doi:10.1101/2020.03.03.20030437]

#### Keywords

- COVID-19
- T cell
- B cell
- NK cell
- IL-6
- pro-calcitonin
- cytokine storm

#### Main Findings

The authors collected data on 25 COVID-19 patients (n=11 men, n=14
women) using standard laboratory tests and flow cytometry. All patients
were treated with antibiotics. Twenty-four of the 25 patients were also
treated with anti-viral Umefinovir and 14 of the patients were treated
with corticosteroids. 14 patients became negative for the virus after
8-14 days of treatment. The same treatment course was extended to 15-23
days for patients who were still positive for the virus at day 14.

The authors found a negative association between age and resolution of
infection. Patients with hypertension, diabetes, malignancy or chronic
liver disease were all unable to clear the virus at day 14, though not
statistically significant.

Elevated procalcitonin and a trend for increased IL-6 were also found in
peripheral blood prior to the treatment.

A trend for lower NK cell, T cell and B cell counts in patients was also
reported. B cell, CD4 and CD8 T cell counts were only increased upon
treatment in patients who cleared the virus. NK cell frequencies
remained unchanged after treatment in all the patients.

#### Limitations

73% of the patients who remained positive for SARS-CoV2 after the 1^st^
treatment, and 43% of all patients who cleared the virus were treated
with corticosteroids. Corticosteroids have strong effects on the immune
compartment in blood [@doi:10.1038/srep23002]. The authors should have accounted for
corticosteroid treatment when considering changes in T, NK and B cell
frequencies.

Assessing if IL-6 concentrations were back to baseline levels following
treatment would have provided insights into the COVID-19 cytokine storm
biology. Patients with higher baseline levels of IL-6 have been reported
to have lower CD8 and CD4 T cell frequencies [@doi:10.1101/2020.03.01.20029785]. Correlating IL-6 with
cell counts before and after treatment would thus have also been of
interest. The report of the laboratory measures in table 2 is incomplete
and should include the frequencies of patients with increased/decreased
levels for each parameter.

Correction is needed for the 1^st^ paragraph of the discussion as data
does not support NK cell restoration upon treatment in patients who
cleared the virus. NK cells remain unchanged after the 1^st^ treatment
course and only seem to increase in 2 out of 6 donors after the 2^nd^
treatment course in those patients.

#### Significance

Previous reports suggest an association between disease severity and
elevated IL-6 or pro-calcitonin concentrations in COVID-19
patients [@doi:10.1101/2020.02.29.20029520; @doi:10.1016/j.cca.2020.03.004]. IL-6 receptor blockade is also being administered to
patients enrolled in clinical trials (NCT04317092). This report thus
contributes to highlight elevated concentrations of these analytes in
COVID-19 patients. Mechanisms underlying the association between viral
clearance and restoration of the T cell and B cell frequencies suggests
viral-driven immune dysregulation, which needs to be investigated in
further studies.

#### Credit

*Review by Bérengère Salomé as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Clinical findings in critically ill patients infected with SARS-CoV-2 in Guangdong Province, China: a multi-center, retrospective, observational study


Xu et al. *medRxiv*. [@doi:10.1101/2020.03.03.20030668]

#### Keywords

- clinical outcomes
- prognosis
- critically ill patients
- ICU
- lymphopenia
- LDH

#### Main Findings:

This work analyses laboratory and clinical data from 45 patients treated
in the in ICU in a single province in China. Overall, 44% of the
patients were intubated within 3 days of ICU admission with only 1
death.

Lymphopenia was noted in 91% of patient with an inverse correlation with
LDH.

Lymphocyte levels are negatively correlated with Sequential Organ
Failure Assessment (SOFA) score (clinical score, the higher the more
critical state), LDH levels are positively correlated to SOFA score.
Overall, older patients (&gt;60yo), with high SOFA score, high LDH
levels and low lymphocytes levels at ICU admission are at higher risk of
intubation.

Of note, convalescent plasma was administered to 6 patients but due to
limited sample size no conclusion can be made.

#### Limitations

While the study offers important insights into disease course and
clinical lab correlates of outcome, the cohort is relatively small and
is likely skewed towards a less-severe population compared to other ICU
reports given the outcomes observed. Analysis of laboratory values and
predictors of outcomes in larger cohorts will be important to make
triage and treatment decisions. As with many retrospective analyses,
pre-infection data is limited and thus it is not possible to understand
whether lymphopenia was secondary to underlying comorbidities or
infection.

Well-designed studies are necessary to evaluate the effect of
convalescent plasma administration.

#### Significance

This clinical data enables the identification of at-risk
patients and gives guidance for research for treatment options. Indeed,
further work is needed to better understand the causes of the
lymphopenia and its correlation with outcome.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Immune Multi-epitope vaccine design using an immunoinformatics approach for 2019 novel coronavirus in China (SARS-CoV-2)

[@doi:10.1101/2020.03.03.962332]

#### Keywords

- Vaccine
- in silico
- B cell epitopes
- T cell epitopes

#### Main Findings

Using in silico bioinformatic tools, this study identified putative
antigenic B-cell epitopes and HLA restricted T-cell epitopes from the
spike, envelope and membrane proteins of SARS-CoV-2, based on the genome
sequence available on the NCBI database. T cell epitopes were selected
based on predicted affinity for the more common HLA-I alleles in the
Chinese population. Subsequently, the authors designed vaccine peptides
by bridging selected B-cell epitopes and adjacent T-cell epitopes.
Vaccine peptides containing only T-cell epitopes were also generated.

From 61 predicted B-cell epitopes, only 19 were exposed on the surface
of the virion and had a high antigenicity score. A total of 499 T-cell
epitopes were predicted. Based on the 19 B-cell epitopes and their 121
adjacent T-cell epitopes, 17 candidate vaccine peptides were designed.
Additionally, another 102 vaccine peptides containing T-cell epitopes
only were generated. Based on the epitope counts and HLA score, 13 of
those were selected. Thus, a total of 30 peptide vaccine candidates were
designed.

#### Limitations

While this study provides candidates for the development of vaccines
against SARS-CoV-2, in vitro and in vivo trials are required to validate
the immunogenicity of the selected B and T cell epitopes. This could be
done using serum and cells from CoV-2-exposed individuals, and in
preclinical studies. The implication of this study for the current
epidemic are thus limited. Nevertheless, further research on this field
is greatly needed.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Clinical Features of Patients Infected with the 2019 Novel Coronavirus (COVID-19) in Shanghai, China

Cao et al. *medRxiv* [@doi:10.1101/2020.03.04.20030395]

#### Keywords

- Disease severity
- clinical features
- laboratory abnormalities

#### Main Findings

This single-center cohort study analyzes the clinical and
laboratory features of 198 patients with confirmed COVID-19 infection in
Shanghai, China and correlated these parameters with clinical disease
severity, including subsequent intensive care unit (ICU) admission. 19
cases (9.5%) required ICU admission after developing respiratory failure
or organ dysfunction. Age, male sex, underlying cardiovascular disease,
and high symptom severity (high fever, dyspnea) were all significantly
correlated with ICU admission. Additionally, ICU admission was more
common in patients who presented with lymphopenia and elevated
neutrophil counts, among other laboratory abnormalities. Flow cytometric
analysis revealed that patients admitted to the ICU had significantly
reduced circulating CD3+ T cell, CD4+ T cell, CD8+ T cell, and CD45+
leukocyte populations compared to the cohort of patients not requiring
ICU admission.

#### Limitations

The limitations of this study include the relatively
small sample size and lack of longitudinal testing. The authors also did
not assess whether respiratory comorbidity – such as asthma or chronic
obstructive lung disease – in addition to immunosuppression affected ICU
admission likelihood.

#### Significance

COVID-19 has already sickened thousands across the globe,
though the severity of these infections is markedly diverse, ranging
from mild symptoms to respiratory failure requiring maximal
intervention. Understanding what clinical, laboratory, and immunologic
factors predict the clinical course of COVID-19 infection permits
frontline providers to distribute limited medical resources more
effectively.

#### Credit

*Review by Andrew Charap as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine at
Mount Sinai.*


### Serological detection of 2019-nCoV respond to the epidemic: A useful complement to nucleic acid testing

Zhang et al. *medRxiv*. [@doi:10.1101/2020.03.04.20030916]

#### Keywords

- Immunoassay
- serum IgM and IgG
- specific antibodies

#### Main Finding

This study showed that both anti-2019-nCov IgM and IgG were detected by
automated chemiluminescent immunoassay in the patients who had been
already confirmed as positive by nucleic acid detection, while single
positivity of IgM or IgG were detected in a very few cases in the other
population including 225 non-COVID-19 cases. In addition to the increase
of anti-2019-nCov IgM 7-12 days after morbidity, the increase of IgG was
detected in three patients with COVID-19 within a very short of time
(0-1 day).

#### Limitations

The limitation of this study is only
3 confirmed COVID-19 cases were included, so that the relationship
between anti-2019-nCov antibodies and disease progression might not be
clearly defined. Another limitation is that they did not show the course
of 2019-nCov specific antibodies in the cases with positive for COVID-19
but without clinical symptoms.

#### Significance

The detection of anti-2019-nCov antibodies can be an alternative method
to diagnose and treat COVID-19 more comprehensively by distinguish non
COVID-19 patients. It may be helpful to understand the course of
individual cases with COVID-19 to predict the prognosis if more cases
will be evaluated.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Human Kidney is a Target for Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection

[@doi:10.1101/2020.03.04.20031120]

#### Keywords

- Kidney/Renal Failure
- Macrophage Infiltration
- Complement Activation

#### Main Finding

Analyzing the eGFR (effective glomerular flow rate) of 85 Covid-19
patients and characterizing tissue damage and viral presence in
post-mortem kidney samples from 6 Covid-19 patients, the authors
conclude that significant damage occurs to the kidney, following
Covid-19 infection. This is in contrast to the SARS infection from the
2003 outbreak. They determine this damage to be more prevalent in
patients older than 60 years old, as determined by analysis of eGFR. H&E
and IHC analysis in 6 Covid-19 patients revealed that damage was in the
tubules, not the glomeruli of the kidneys and suggested that macrophage
accumulation and C5b-9 deposition are key to this process. 

#### Limitations

Severe limitations include that the H&E and IHC samples were performed
on post-mortem samples of unknown age, thus we cannot assess how/if age
correlates with kidney damage, upon Covid-19 infection. Additionally,
eGFR was the only *in-vivo* measurement. Blood urea nitrogen and
proteinuria are amongst other measurements that could have been obtained
from patient records. An immune panel of the blood was not performed to
assess immune system activation. Additionally, patients are only from
one hospital. 

#### Significance

This report makes clear that kidney damage is prevalent
in Covid-19 patients and should be accounted for. 

#### Credit

*Review by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine at Mount Sinai.*


### COVID-19 early warning score: a multi-parameter screening tool to identify highly suspected patients

Song et al. *medRxiv*. [@doi:10.1101/2020.03.05.20031906]

#### Keywords

- retrospective
- electronic health records
- blood counts
- diagnostic
- prognostic
- modeling

#### Main Findings

The aim of this study was to identify diagnostic or prognostic criteria
which could identify patients with COVID-19 and predict patients who
would go on to develop severe respiratory disease. The authors use EMR
data from individuals taking a COVID-19 test at Zhejiang hospital, China
in late January/Early February. A large number of clinical parameters
were different between individuals with COVID-19 and also between
‘severe’ and ‘non-severe’ infections and the authors combine these into
a multivariate linear model to derive a weighted score, presumably
intended for clinical use.

#### Limitations

Unfortunately, the paper is lacking a lot of crucial information, making
it impossible to determine the importance or relevance of the findings.
Most importantly, the timings of the clinical measurements are not
described relative to the disease course, so it is unclear if the
differences between ‘severe’ and ‘non-severe’ infections are occurring
before progression to severe disease (which would make them useful
prognostic markers), or after (which would not).

#### Significance

This paper is one of many retrospective studies coming from hospitals in
China studying individuals with COVID-19. Because of the sparse
description of the study design, this paper offers little new
information. However, studies like this could be very valuable and we
would strongly encourage the authors to revise this manuscript to
include more information about the timeline of clinical measurements in
relation to disease onset and more details of patient outcomes.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### LY6E impairs coronavirus fusion and confers immune control of viral disease

Pfaender et al. *bioRxiv*. [@doi:10.1101/2020.03.05.979260]

#### Keywords

- interferon-stimulated genes
- antiviral interferons
- human coronaviruses (CoV)
- murine hepatitis virus (MHV)

#### Main Findings

Screening a cDNA library of &gt;350 human interferon-stimulated genes
for antiviral activity against endemic human coronavirus HCoV-229E
(associated with the common cold), Pfaender S & Mar K *et al.* identify
lymphocyte antigen 6 complex, locus E (Ly6E) as an inhibitor of cellular
infection of Huh7 cells, a human hepatoma cell line susceptible to
HCoV-229E and other coronaviruses. In a series of consecutive *in vitro*
experiments including both stable Ly6E overexpression and
CRISPR-Cas9-mediated knockout the authors further demonstrate that Ly6E
reduces cellular infection by various other coronaviruses including
human SARS-CoV and SARS-CoV-2 as well as murine CoV mouse hepatitis
virus (MHV). Their experiments suggest that this effect is dependent on
Ly6E inhibition of CoV strain-specific spike protein-mediated membrane
fusion required for viral cell entry.

To address the function of Ly6E *in vivo*, hematopoietic stem
cell-specific Ly6E knock-out mice were generated by breeding Ly6E^fl/fl^
mice (referred to as functional wild-type mice) with transgenic
*Vav-iCre* mice (offspring referred to as Ly6E HSC ko mice); wild-type
and Ly6E HSC ko mice of both sexes were infected intraperitoneally with
varying doses of the natural murine coronavirus MHV, generally causing a
wide range of diseases in mice including hepatitis, enteritis and
encephalomyelitis. Briefly, compared to wild-type controls, mice lacking
hematopoietic cell-expressed Ly6E were found to present with a more
severe disease phenotype as based on serum ALT levels (prognostic of
liver damage), liver histopathology, and viral titers in the spleen.
Moreover, bulk RNAseq analysis of infected liver and spleen tissues
indicated changes in gene expression pathways related to tissue damage
and antiviral immune responses as well as a reduction of genes
associated with type I IFN response and inflammation. Finally, the
authors report substantial differences in the numbers of hepatic and
splenic APC subsets between wild-type and knockout mice following MHV
infection and show that Ly6E-deficient B cells and to a lesser extent
also DCs are particularly susceptible to MHV infection *in vitro*.

#### Limitations

Experiments and data in this study are presented in an overall logical
and coherent fashion; however, some observations and the conclusions
drawn are problematic and should be further addressed & discussed by the
authors. Methodological & formal limitations include relatively low
replicate numbers as well as missing technical replicates for some *in
vitro* experiments (*cf*. Fig. legend 1; Fig. legend 2e); the omission
of “outliers” in Fig. legend 2 without an apparent rationale as to why
this approach was chosen; the lack of detection of actual Ly6E protein
levels in Ly6E HSC ko or wild-type mice; and most importantly, missing
information on RNAseq data collection & analysis in the method section
and throughout the paper. A more relevant concern though is that the
interpretation of the experimental data presented and the language used
tend to overrate and at times overgeneralize findings: for example,
while the authors demonstrate statistically significant, Ly6E-mediated
reduction of coronavirus titers in stable cells lines *in vitro*, it
remains unclear whether a viral titer reduction by one log decade would
be of actual biological relevance in face of high viral titers *in
vivo*. After high-dose intraperitoneal MHV infection *in vivo*, early
viral titers in Ly6E HSC knockout vs. wt mice only showed an elevation
in the spleen (\~1.5 log decades) but not liver of the ko mice (other
tissue not evaluated), and while ko mice presented with only modestly
increased liver pathology, both male and female ko mice exhibited
significantly higher mortality. Thus, the manuscript tile statement that
“Ly6E … confers immune control of viral disease” is supported by only
limited *in vivo* data, and gain-of-function experiments (eg. Ly6E
overexpression) were not performed. Of additional note here, tissue
tropism and virulence differ greatly among various MHV strains and
isolates whereas dose, route of infection, age, genetic background and
sex of the mice used may additionally affect disease outcome and
phenotype (*cf*. Taguchi F & Hirai-Yuki A,
<https://doi.org/10.3389/fmicb.2012.00068>; Kanolkhar A et al,
[https://jvi.asm.org/content/
83/18/9258](https://jvi.asm.org/content/%2083/18/9258)). Observations
attributed to hematopoietic stem cell-specific Ly6E deletion could
therefore be influenced by the different genetic backgrounds of floxed
and cre mice used, and although it appears that littermates wt and ko
littermates were used in the experiments, the potentially decisive
impact of strain differences should at least have been discussed. Along
these lines, it should also be taken into account that the majority of
human coronaviruses cause respiratory symptoms, which follow a different
clinical course engaging other primary cellular mediators than the
hepatotropic murine MHV disease studied here. It therefore remains
highly speculative how the findings reported in this study will
translate to human disease and it would therefore be important to test
other routes of MHV infection and doses that have been described to
produce a more comparable phenotype to human coronavirus disease (*cf.*
Kanolkhar A et al, [https://jvi.asm.org/content/
83/18/9258](https://jvi.asm.org/content/%2083/18/9258)). Another
important shortcoming of this study is the lack of any information on
functional deficits or changes in Ly6E-deficient immune cells and how
this might relate to the phenotype observed. Overall, the *in vitro*
experiments are more convincing than the *in vivo* studies which appear
somewhat limited.

#### Significance

Despite some shortcomings, the experiments performed in this study
suggest a novel and somewhat unexpected role of Ly6E in the protection
against coronaviruses across species. These findings are of relevance
and should be further explored in ongoing research on potential
coronavirus therapies. Yet an important caveat pertains to the authors’
suggestion that “therapeutic mimicking of Ly6E action” may constitute a
first line of defense against novel coronaviruses since their own prior
work demonstrated that Ly6E can enhance rather than curtail infection
with influenza A and other viruses.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### A preliminary study on serological assay for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in 238 admitted hospital patients

Liu et al. *medRxiv*. [@doi:10.1101/2020.03.06.20031856]

#### Keywords
- diagnosis
- serological assay
- ELISA
- RT-PCR

#### Main Findings

While RT-PCR is being used currently to routinely diagnose infection
with SARS-CoV-2, there are significant limitations to the use of a
nucleic acid test that lead to a high false-negative rate. This
article describes ELISAs that can measure IgM and IgG antibodies
against the N protein of SARS-CoV-2 to test samples from 238
patients (153 positive by RT-PCR and 85 negative by RT-PCR) at
different times after symptom onset. The positivity rate of the IgM
and/or IgG ELISAs was greater than that of the RT-PCR (81.5%
compared to 64.3%) with similar positive rates in the confirmed and
suspected cases (83% and 78.8%, respectively), suggesting that many
of the suspected but RT-PCR-negative cases were also infected. The
authors also found that the ELISAs have higher positive rates later
after symptom onset while RT-PCR is more effective as a diagnostic
test early during the infection.

#### Limitations

I cannot identify any limitations to this study.

#### Significance

The authors make a strong case for using a combination of ELISA and
RT-PCR for diagnosis of infection with SARS-CoV-2, especially
considering the dynamics of positivity rates of RT-PCR and ELISA.
Fewer false-negative diagnoses would improve infection control and
patient management.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Monoclonal antibodies for the S2 subunit of spike of SARS-CoV cross-react with the newly-emerged SARS-CoV-2

[@doi:10.1101/2020.03.06.980037]

#### Keywords

- Coronavirus Disease 2019 (COVID-19)
- SARS-CoV-2
- Spike protein
- Cross- reactive antibodies

#### Main Findings

Whole genome sequencing-based comparisons of the 2003
Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and the 2019
SARS-CoV-2 revealed conserved receptor binding domain (RBD) and host
cell receptor, angiotensin-converting enzyme 2 (ACE2). In line with
this, the authors tested cross-reactivity of murine monoclonal
antibodies (mAbs) previously generated against the SARS-CoV spike (S)
glycoprotein involved in viral entry. One of the screened mAb, 1A9, was
able to bind and cross-neutralize multiple strains of SARS-CoV, as well
as, detect the S protein in SARS-CoV-2-infected cells. mAb 1A9 was
generated using an immunogenic fragment in the S2 subunit of SARS-CoV
and binds through a novel epitope within the S2 subunit at amino acids
1111-1130. It is important to note that CD8+ T lymphocyte epitopes
overlap with these residues, suggesting that S2 subunit could be
involved in inducing both, humoral and cell-mediated immunity.

#### Limitations

The authors used previously generated mouse mAbs
against the S protein in SARS-CoV expressed in mammalian cell line.
Future experimental validation using COVID-19 patient samples is needed
to validate these findings. In addition, the results of these studies
are predominantly based on in vitro experiments and so, evaluating the
effects of the mAb 1A9 in an animal model infected with this virus will
help us better understand the host immune responses in COVID-19 and
potential therapeutic vaccines.

#### Significance

This study identified mAbs that recognize the new
coronavirus, SARS-Cov-2. These cross-reactive mAbs will help in
developing diagnostic assays for COVID-19.

#### Credit

*This review was undertaken by Tamar Plitt and Katherine Lindblad as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Mortality of COVID-19 is Associated with Cellular Immune Function Compared to Immune Function in Chinese Han Population

Zeng et al. *medRxiv*. [@doi:10.1101/2020.03.08.20031229]

#### Keywords
- WBC
- peripheral blood
- CD4
- CD8 T cells

#### Main Findings

Retrospective study of the clinical characteristics of 752
patients infected with COVID-19 at Chinese PLA General Hospital, Peking
Union Medical College Hospital, and affiliated hospitals at Shanghai
University of medicine & Health Sciences. This study is the first one
that compares PB from healthy controls from the same regions in Shanghai
and Beijing, and infected COVID-19 patients to standardize a reference
range of WBCs of people at high risk.

#### Limitations

Lower levels of leukocyte counts -B cells, CD4 and CD8 T
cells- correlated with mortality (WBCs are significantly lower in severe
or critical UCI patients vs mild ones). Based on 14,117 normal controls
in Chinese Han population (ranging in age from 18-86) it is recommended
that reference ranges of people at high risk of COVID-19 infection are
CD3+ lymphocytes below 900 cells/mm3, CD4+ lymphocytes below 500
cells/mm3, and CD8+ lymphocytes below 300 cells/mm3. Importantly, this
study also reported that the levels of D-dimer, C-reactive protein and
IL-6 were elevated in COVID-19 pts., indicating clot formation, severe
inflammation and cytokine storm.

#### Significance

This study sets a threshold to identify patients at risk
by analyzing their levels of leukocytes, which is an easy and fast
approach to stratify individuals that require hospitalization. Although
the study is limited (only counts of WBC are analyzed and not its
profile) the data is solid and statistically robust to correlate levels
of lymphopenia with mortality.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Retrospective Analysis of Clinical Features in 101 Death Cases with COVID-19

Chen et al. *medRxiv*. [@doi:10.1101/2020.03.09.20033068]

#### Keywords

- death biomarkers
- cardiac damage
- Troponin
- Blood type
- respiratory failure
- hypertension

#### Main Findings

This is a retrospective study involving 101 death cases with COVID-19 in
Wuhan Jinyintan Hospital. The aim was to describe clinical,
epidemiological and laboratory features of fatal cases in order to
identify the possible primary mortality causes related to COVID-19.

Among 101 death cases, 56.44% were confirmed by RT-PCR and 43.6% by
clinical diagnostics. Males dominated the number of deaths and the
average age was 65.46 years. All patients died of respiratory failure
and multiple organs failure, except one (acute coronary syndrome). The
predominant comorbidities were hypertension (42.57%) and diabetes
(22.77%). 25.74% of the patients presented more than two underlying
diseases. 82% of patients presented myocardial enzymes abnormalities at
admission and further increase in myocardial damage indicators with
disease progression: patients with elevated Troponin I progressed faster
to death. Alterations in coagulation were also detected. Indicators of
liver and kidney damage increased 48 hours before death. The authors
studied the deceased patients’ blood type and presented the following
results: type A (44.44%), type B (29.29%), type AB (8.08%) and type O
(18.19%), which is inconsistent with the distribution in Han population
in Wuhan.

Clinical analysis showed that the most common symptom was fever (91.9%),
followed by cough and dyspnea. The medium time from onset of symptoms to
acute respiratory distress syndrome (ARDS) development was 12 days.
Unlike SARS, only 2 patients with COVID-19 had diarrhea. 98% presented
abnormal lung imaging at admission and most had double-lung
abnormalities. Related to the laboratorial findings some inflammatory
indicators gradually increased during the disease progression, such as
IL-6 secretion in the circulation, procalcitonin (PCT) and C-reactive
protein (CRP), while platelets numbers decreased. The authors also
reported an initial lymphopenia that was followed by an increase in the
lymphocytes numbers. Neutrophil count increased with disease
progression.

The patients received different treatments such as antiviral drugs
(60.40%), glucocorticoids, thymosin and immunoglobulins. All patients
received antibiotic treatment and some received antifungal drugs. All
patients received oxygen therapy (invasive or non-invasive ones).

#### Limitations

This study involves just fatal patients, lacking comparisons with other
groups of patients e.g. patients that recovered from COVID-19. The
authors didn’t discuss the different approaches used for treatments and
how these may affect the several parameters measured. The possible
relationship between the increase of inflammatory indicators and
morbidities of COVID-19 are not discussed.

#### Significance

This study has the largest cohort of fatal cases reported so far. The
authors show that COVID-19 causes fatal respiratory distress syndrome
and multiple organ failure. This study highlights prevalent myocardial
damage and indicates that cardiac function of COVID-19 patients should
be carefully monitored. The data suggest that Troponin I should be
further investigated as an early indicator of patients with high risk of
accelerated health deterioration. Secondary bacterial and fungal
infections were frequent in critically ill patients and these need to be
carefully monitored in severe COVID-19 patients. Differences in blood
type distribution were observed, suggesting that type A is detrimental
while type O is protective – but further studies are needed to confirm
these findings and elucidate if blood type influences infection or
disease severity. Several inflammatory indicators (neutrophils, PCT, CRP
and IL-6, D-dimer) increased according to disease severity and should be
assessed as biomarkers and to better understand the biology of
progression to severe disease.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Relationship between the ABO Blood Group and the COVID-19 Susceptibility

Zhao et al. *medRxiv*. [@doi:10.1101/2020.03.11.20031096]

#### Keywords

- ABO blood group
- COVID-19 susceptibility

#### Main Findings

These authors compared the ABO blood group of 2,173 patients with
RT-PCR-confirmed COVID-19 from hospitals in Wuhan and Shenzhen with the
ABO blood group distribution in unaffected people in the same cities
from previous studies (2015 and 2010 for Wuhan and Shenzhen,
respectively). They found that people with blood group A are
statistically over-represented in the number of those infected and who
succumb to death while those with blood group O are statistically
underrepresented with no influence of age or sex.

#### Limitations

This study compares patients with COVID-19 to the general population
but relies on data published 5 and 10 years ago for the control. The
mechanisms that the authors propose may underlie the differences they
observed require further study.

#### Significance

Risk stratification based on blood group may be beneficial for
patients and also healthcare workers in infection control. Additionally,
investigating the mechanism behind these findings could lead to better
developing prophylactic and therapeutic targets for COVID-19.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### The inhaled corticosteroid ciclesonide blocks coronavirus RNA replication by targeting viral NSP15

Matsuyama et al. *bioRxiv* [@doi:10.1101/2020.03.11.987016]

#### Keywords

- Corticosteroids
- ciclesonide
- mometasone
- NSP15
- MERS-CoV

#### Main Findings

This study reconsiders the use of inhaled corticosteroids in the
treatment of pneumonia by coronavirus. Corticosteroids were associated
with increased mortality for SARS in 2003 and for MERS in 2013, probably
due to that fact that systemic corticosteroids suppress the innate
immune system, resulting in increased viral replication. However, some
steroid compounds might block coronavirus replication. The authors
screened steroids from a chemical library and assessed the viral growth
suppression and drug cytotoxicity. Ciclesonide demonstrated low
cytotoxicity and potent suppression of MERS-CoV viral growth. The
commonly used systemic steroids cortisone, prednisolone and
dexamethasone did not suppress viral growth, nor did the commonly used
inhaled steroid fluticasone. To identify the drug target of virus
replication, the authors conducted 11 consecutive MERS-CoV passages in
the presence of ciclesonide or mometasone, and they could generate a
mutant virus that developed resistance to ciclesonide, but not to
mometasone. Afterwards, they performed next-generation sequencing and
identified an amino acid substitution in nonstructural protein 15
(NSP15) as the predicted mechanism for viral resistance to ciclesonide.
The authors were able to successfully generate a recombinant virus
carrying that amino acid substitution, which overcome the antiviral
effect of ciclesonide, suggesting that ciclosenide interacts with NSP15.
The mutant virus was inhibited by mometasone, suggesting that the
antiviral target of mometasone is different from that of ciclesonide.
Lastly, the effects of ciclesonide and mometason on suppressing the
replication of SARS-CoV-2 were evaluated. Both compounds were found to
suppress viral replication with a similar efficacy to lopinavir.

#### Limitations

Most of the experiments, including the identification of the mutation in
NSP15 were conducted with MERS-CoV. This is not the closest related
virus to SARS-CoV-2, as that would be SARS-CoV. Thus, to repeat the
initial experiments with SARS-CoV, or preferably SARS-CoV-2, is
essential. The manuscript should address this and, therefore, it will
require considerable editing for organization and clarity. Also, in
terms of cell immunogenic epitopes, while SARS-CoV-2 spike protein
contains several predicted B and T cell immunogenic epitopes that are
shared with other coronaviruses, some studies have shown critical
differences between MERS-CoV, SARS-CoV and SARS-CoV-2. A main criticism
is that the authors only used VeroE6/TMPRSS2 cells to gauge the direct
cytotoxic effects of viral replication. To evaluate this in other cell
lines, including human airway epithelial cells, is crucial, as the
infectivity of coronavirus strains greatly varies in different cell
lines,

#### Significance

Nevertheless, these findings encourage evaluating ciclesonide and
mometasone as better options for patients with COVID-19 in need of
inhaled steroids, especially as an alternative to other corticosteroids
that have been shown to increase viral replication in vitro. This should
be evaluated in future clinical studies.

#### Credit

*This review was undertaken by Alvaro Moreira, MD as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### A human monoclonal antibody blocking SARS-CoV-2 infection **

Wang et al. *bioRxiv*. [@doi:10.1101/2020.03.11.987958]

#### Keywords

- Monoclonal antibodies
- SARS-CoV2
- cross-neutralization
- potential treatment
- spike receptor

#### Main Findings

The authors reported a human monoclonal
antibody that neutralizes SARS-CoV-2 and SARS-Cov which belong to same
family of corona viruses. For identifying mAbs, supernatants of a
collection of 51 hybridomas raised against the spike protein of SARS-CoV
(SARS-S) were screened by ELISA for cross-reactivity against the spike
protein of SARS-CoN2 (SARS2-S). Hybridomas were derived from immunized
transgenic H2L2 mice (chimeric for fully human VH-VL and rat constant
region). Four SARS-S hybridomas displayed cross-reactivity with SARS2-S,
one of which (47D11) exhibited cross-neutralizing activity for SARS-S
and SARS2-S pseudotyped VSV infection. A recombinant, fully human IgG1
isotype antibody was generated and used for further characterization.

The humanized 47D11 antibody inhibited infection of VeroE6 cells with
SARS-CoV and SARS-CoV-2 with IC50 values of 0.19 and 0.57 μg/ml
respectively. 47D11 mAb bound a conserved epitope on the spike receptor
binding domain (RBD) explaining its ability to cross-neutralize SARS-CoV
and SARS-CoV-2. 47D11 was shown to target the S1B RBD of SARS-S and
SARS2-S with similar affinities. Interestingly, binding of 47D11 to
SARS-S1B and SARS2-S1B did not interfere with S1B binding to ACE2
receptor-expressing cells assayed by flow cytometry.

#### Limitations

These results show that the human 47D11 antibody
neutralizes SARS-CoV and SARS-Cov2 infectivity via an as yet unknown
mechanism that is different from receptor binding interference.
Alternative mechanisms were proposed but these as yet remain to be
tested in the context of SARS-CoV2. From a therapeutic standpoint and in
the absence of in vivo data, it is unclear whether the 47D11 ab can
alter the course of infection in an infected host through virus
clearance or protect an uninfected host that is exposed to the virus.
There is a precedent for the latter possibility as it relates to
SARS-CoV that was cited by the authors and could turn out to be true for
SARS-CoV2.

#### Significance

This study enabled the identification of novel neutralizing
antibody against COV-that could potentially be used as first line of
treatment in the near future to reduce the viral load and adverse
effects in infected patients. In addition, neutralizing antibodies such
as 47D11 represent promising reagents for developing
antigen-antibody-based detection test kits and assays.

#### Credit

*This review was edited by K. Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


**Heat inactivation of serum interferes with the immunoanalysis of
antibodies to SARS-CoV-2**

Heat inactivation, immunochromatography, diagnosis, serum antibodies,
IgM, IgG

*Summary*

The use of heat inactivation to neutralize pathogens in serum samples
collected from suspected COVID-19 patients reduces the sensitivity of a
fluorescent immunochromatographic assay to detect anti-SARS-CoV-2 IgM
and IgG.

*Major findings*

Coronaviruses can be killed by heat inactivation, and this is an
important safety precaution in laboratory manipulation of clinical
samples. However, the effect of this step on downstream
SARS-CoV-2-specific serum antibody assays has not been examined. The
authors tested the effect of heat inactivation (56 deg C for 30 minutes)
versus no heat inactivation on a fluorescence immunochromatography
assay. Heat inactivation reduced all IgM measurements by an average of
54% and most IgG measurements (22/36 samples, average reduction of 50%),
consistent with the lower thermal stability of IgM than that of IgG.
Heat inactivation caused a subset of IgM but not IgG readings to fall
below a specified positivity threshold.

*Limitations*

Limitations included the use of only one type of assay for testing heat
inactivated vs non-inactivated sera, and the use of the same baseline
for heat inactivated and non-inactivated sera. The results indicate that
heat inactivation affects the quantification of SARS-CoV-2-antibody
response, specially IgM, but still allows to distinguish positive
specific IgG. Therefore, the effect of heat inactivation should be
studied when designing assays that quantitatively associate
immunoglobulin levels (especially IgM) to immune state.

*Review by Andrew M. Leader as part of a project by students, postdocs
and faculty at the Immunology Institute of the Icahn school of medicine,
Mount Sinai.*


### Immune phenotyping based on neutrophil-to-lymphocyte ratio and IgG predicts disease severity and outcome for patients with COVID-19

Zhang et al. *medRxiv* [@doi:10.1101/2020.03.12.20035048]

#### Keywords

- Biomarkers
- cytokines
- IgG
- immune cells

#### Main Findings

In a cohort of 222 patients, anti-SARS-CoV-2 IgM and IgG levels were
analyzed during acute and convalescent phases (up to day 35) and
correlated to the diseases’ severity. The same was done with
neutrophil-to-lymphocyte ratio. High IgG levels and high
neutrophil-to-lymphocyte ratio in convalescence were both independently
associated to the severity of the disease. The simultaneous occurrence
of both of these laboratory findings correlated even stronger to the
diseases’ severity.

Severe cases with high neutrophil-to-lymphocyte ratios had clearly
higher levels of IL-6. The authors propose that a robust IgG response
leads to immune-mediated tissue damage, thus explaining the worse
outcome in patients with overexuberant antibody response.

#### Limitations

A main criticism is that the criteria for stratifying patients in severe
vs. non-severe are not described. The only reference related to this is
the difference between the percentage of patients who needed mechanical
ventilation, which was greater in patients with both high IgG levels and
high neutrophil-to-lymphocyte ratio. No patient with both low IgG levels
and low neutrophil-to-lymphocyte ratio was treated with mechanical
ventilation.

The proposed correlation of severity with IL-2 and IL-10 levels is not
very strong.

Furthermore, although mostly ignored in the paper’s discussion, one of
the most interesting findings is that an early increase in
anti-SARS-CoV-2 IgM levels also seems to correlate with severe disease.
However, as only median values are shown for antibody kinetics curves,
the extent of variation in acute phase cannot be assessed.

#### Significance

Anti-SARS-CoV-2 IgG levels and with neutrophil-to-lymphocyte ratio
predict severity of COVID-19 independently of each other. An additive
predictive value of both variables is noticeable. Importantly, an
early-on increase in anti-SARS-CoV-2 IgM levels also seem to predict
outcome.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Reinfection could not occur in SARS-2 CoV-2 infected rhesus macaques

Bao et al. *bioRxiv* [@doi:10.1101/2020.03.13.990226]

#### Keywords

- SARS-CoV-2
- viral load
- reinfection
- relapse
- non-human primate model

#### Main Findings

This study addresses the issue or acquired immunity after a primary
COVID-19 infection in rhesus monkeys. Four Chinese rhesus macaques were
intratracheally infected with SARS-CoV-2 and two out of the four were
re-infected at 28 days post initial infection (dpi) with the same viral
dose after confirming the recovery by the absence of clinical symptoms,
radiological abnormalities and viral detection (2 negative RT-PCR
tests). While the initial infection led the viral loads in nasal and
pharyngeal swabs that reach approximately 6.5 log~10~ RNA copies/ml at 3
dpi in all four monkeys, viral loads in the swabs tested negative after
reinfection in the two reinfected monkeys. In addition, the necropsies
from a monkey (M1) at 7 days after primary infection, and another monkey
(M3) at 5 days post reinfection, revealed the histopathological damages
and viral replication in the examined tissues from M1, while no viral
replication as well as no histological damages were detected in the
tissues from M3. Furthermore, sera from three monkeys at 21 and 28 dpi
exhibited neutralizing activity against SARS-CoV-2 in vitro, suggesting
the production of protective neutralizing antibodies in these monkeys.
Overall, this study indicates that primary infection with SARS-CoV-2 may
protect from subsequent exposure to the same virus.

#### Limitations

In human, virus has been detected by nasopharyngeal swabs until 9 to 15
days after the onset of symptoms. In the infected monkeys in this study,
virus were detected from day 1 after the infection, declining to
undetectable level by day 15 post infection. It may suggest that there
is a faster viral clearance mechanism in monkeys, therefore the
conclusions of reinfection protection for humans need to be carefully
considered. In addition, only two monkeys were re-infected in this study
and the clinical signs of these monkeys were not similar: M3 did not
show weight loss and M4 showed relatively higher fever on the day of
infection and the day of re-challenge.

#### Significance

This study showed clear viral clearance and no indications of relapse or
viremia after a secondary infection with SARS-CoV-2 in a Chinese rhesus
macaque model. These results support the idea that patients with full
recovery (two negative RT-PCR results) may also be protected from
secondary SARS-CoV-2 infection. Recovered patients may be able to
reintegrate to normal public life and provide protective serum perhaps
even if having had a mild infection. The results are also encouraging
for successful vaccine development against SARS-CoV-2.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV

[@doi:10.1101/2020.03.13.991570]

#### Keywords

- neutralizing antibody
- cross-reactivity

#### Main Findings

Given the sequence similarity of the surface spike glycoprotein (S) of
SARS-CoV-2 and SAR-CoV, Yuan et al. (2020) propose that neutralizing
antibodies isolated from convalescent SARS-CoV patients may offer
insight into cross-reactive antibodies targeting SARS-CoV-2. In
particular, they find that the receptor-binding domain (RBD) of
SARS-CoV-2 S protein shares 86% sequence similarity with the RBD of
SARS-CoV S protein that binds to the CR3022 neutralizing antibody.
CR3022 also displays increased affinity for the “up” conformation of the
SARS-CoV-2 S protein compared to the “down” conformation as it does for
the SARS-CoV S protein. Therefore, the authors propose that this
cross-reactive antibody may confer some degree of protection *in vivo*
even if it fails to neutralize *in vitro*.

#### Limitations
Although the authors offer a logical rationale for
identifying cross-reactive neutralizing antibodies derived from
SARS-CoV, their study using only CR3022 failed to demonstrate whether
this approach will be successful. After all, CR3022 failed to neutralize
*in vitro* despite the binding affinity to a similar epitope on
SARS-CoV-2. They would benefit from testing more candidates and using an
*in vivo* model to demonstrate their claim that protection may be
possible in the absence neutralization if combinations are used *in
vivo.*

#### Significance

The ability to make use of previously characterized
neutralizing antibodies for conserved epitopes can expedite drug design
and treatment options.

#### Credit

*This review was undertaken by Dan Fu Ruan, Evan Cody and Venu Pothula as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Highly accurate and sensitive diagnostic detection of SARS-CoV-2 by digital PCR

Dong et al. *medRxiv* [@doi:10.1101/2020.03.14.20036129]

#### Keywords

- Diagnosis
- digital PCR

#### Main Findings

The authors present a digital PCR (dPCR) diagnostic test for
SARS-CoV-2 infection. In 103 individuals that were confirmed in a
follow-up to be infected, the standard qPCR test had a positivity
rate of 28.2% while the dPCR test detected 87.4% of the infections
by detecting an additional 61 positive cases. The authors also
tested samples from close contacts (early in infection stage) and
convalescing individuals (late in infection stage) and were able to
detect SARS-CoV-2 nucleic acid in many more samples using dPCR
compared to qPCR.

#### Limitations

I did not detect limitations.

#### Significance

The authors make a strong case for the need for a highly sensitive
and accurate confirmatory method for diagnosing COVID-19 during this
outbreak and present a potential addition to the diagnostic arsenal.
They propose a dPCR test that they present has a dramatically lower
false negative rate than the standard RT-qPCR tests and can be
especially beneficial in people with low viral load, whether they
are in the earlier or later stages of infection.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### SARS-CoV-2 invades host cells via a novel route: CD147-spike protein

Wang et al. *bioRxiv* [@doi:10.1101/2020.03.14.988345]

#### Keywords

- spike protein
- viral entry
- CD147
- SARS-CoV-2

#### Main Findings

The authors propose a novel mechanism of SARS-CoV-2 viral entry through
the interaction of the viral spike protein (SP) and the immunoglobulin
superfamily protein CD147 (also known as Basigin). Using an in-house
developed humanized antibody against CD147 (maplazumab), they show that
blocking CD147 decreases viral replication in Vero E6 cells. Using
surface plasmon resonance (SPR), ELISA, and Co-IP assays, they show that
the spike protein of SARS-CoV-2 directly interacts with CD147. Lastly,
they utilize immune-election microscopy to show spike protein and CD147
localize to viral inclusion bodies of Vero E6 cells.

#### Limitations

The authors claim that an anti-CD147 antibody (Meplazumab) inhibits
SARS-CoV-2 replication by testing cell growth and viral load in cells
infected with SARS-CoV-2, however there are key pieces of this
experiment that are missing. First, the authors fail to use a
non-specific antibody control. Second, the authors claim that viral
replication is inhibited, and that they test this by qPCR, however this
data is **not shown.** To further prove specificity, the authors should
introduce CD147 to non-susceptible cells and show that they become
permissive.

The authors claim that there is a direct interaction between CD147 and
SP through SPR, ELISA, and Co-IP, and this data seems generally
convincing. The electron microscopy provides further correlative
evidence that SARS-CoV-2 may interact with CD147 as they are both found
in the same viral inclusion body. A quantification of this data would
make the findings more robust.

Finally, the data in this paper lacks replicates, error bars, and
statistics to show that the data are reproducible and statistically
significant.

#### Significance

It has been shown in various studies that SARS-CoV-2 binds to the cell
surface protein ACE2 for cell entry, yet ACE2 is highly expressed in
heart, kidney, and intestinal cells, raising the concern that blocking
ACE2 would result in harmful side effects [@doi:10.1093/ndt/gfu302] CD147 on the other hand
is highly expressed in various tumor types, inflamed tissues, and
pathogen infected cells, suggesting that the inhibition of CD147 would
not result in major side effects [@doi:10.1016/j.yexmp.2018.05.001; @pmid:20551248] The research in this paper has
resulted in an ongoing clinical trail in China to test the safety and
efficacy of anti-CD147 Meplazumab to treat COVID-19. (ClinicalTrails.gov
identifier NCT04275245).

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Blood single cell immune profiling reveals that interferon-MAPK pathway mediated adaptive immune response for COVID-19

Huang et al. *medRxiv* [@doi:10.1101/2020.03.15.20033472]

#### Keywords

- COVID-19
- SARS-CoV-2
- PBMC
- single cell
- MAPK

#### Main Findings

The authors performed single-cell RNA sequencing (scRNAseq) of
peripheral blood mononuclear cells isolated from whole blood samples of
COVID-19 patients (n=10). Data was compared to scRNAseq of samples
collected from patients with influenza A (n=1), acute pharyngitis (n=1),
and cerebral infarction (n=1), as well as, three healthy controls.
COVID-19 patients were categorized into those with moderate (n=6),
severe (n=1), critical (n=1), and cured (n=2) disease. Analysis across
all COVID-19 disease levels revealed 56 different cellular subtypes,
among 17 immune cell types; comparisons between each category to the
normal controls revealed **increased proportions of CD1c^+^ dendritic
cells, CD8^+^ CTLs, and plasmacytoid dendritic cells and a decrease in
proportions of B cells and CD4^+^ T cells**.

**TCR sequencing revealed that greater clonality is associated with
milder COVID-19 disease**; BCR sequencing revealed that COVID-19
patients have circulating antibodies against known viral antigens,
including EBV, HIV, influenza A, and other RNA viruses. This may suggest
that the immune response to SARS-CoV-2 infection elicits production of
antibodies against known RNA viruses.

Excluding enriched pathways shared by COVID-19 patients and patients
with other conditions (influenza A, acute pharyngitis, and cerebral
infarction), the authors identified the **interferon-MAPK signaling
pathway as a major response to SARS-CoV-2 infection**. The authors
performed quantitative real-time reverse transcriptase polymerase chain
reaction (RT-PCR) for interferon-MAPK signaling genes: *IRF27, BST2,*
and *FOS*. These samples were collected from a separate cohort of
COVID-19 patients (critical, n=3; severe, n=3; moderate, n=19; mild,
n=3; and cured, n=10; and healthy controls, n=5). Notably, consistent
with the original scRNAseq data, *FOS* showed up-regulation in COVID-19
patients and down-regulation in cured patients. **The authors propose
that *FOS* may be a candidate marker gene for curative COVID-19
disease**.

#### Limitations

The sample size of this study is limited. To further delineate
differences in the immune profile of peripheral blood of COVID-19
patients, a greater sample size is needed, and longitudinal samples are
needed, as well. A better understanding of the immunological
interactions in cured patients, for example, would require a profile
before and after improvement.

Moreover, the conclusions drawn from this scRNAseq study point to
potential autoimmunity and immune deficiency to distinguish different
severities of COVID-19 disease. However, this requires an expanded
number of samples and a more robust organization of specific immune cell
subtypes that can be compared across different patients. Importantly,
this criterion is likely needed to ensure greater specificity in
identifying markers for COVID-19 infection and subsequent immune
response.

#### Significance

At the single-cell level, COVID-19 disease has been characterized in the
lung, but a greater understanding of systemic immunological responses is
furthered in this study. Type I interferon is an important signaling
molecule for the anti-viral response. The identification of the
interferon-MAPK signaling pathway and the differential expression of
MAPK regulators between patients of differing COVID-19 severity and
compared to cured patients may underscore the importance of either
immune deficiency or autoimmunity in COVID-19 disease.

#### Credit

*This review was undertaken by Matthew D. Park as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Cross-reactive antibody response between SARS-CoV-2 and SARS-CoV infection.

Lv et al. *bioRxiv* [@doi:10.1101/2020.03.15.993097]

#### Keywords

SARS-CoV-2, SARS-CoV, spike protein, RBD, cross-reactivity,
cross-neutralization, antibody, human patients, mouse

#### Main Findings

The authors explore the antigenic differences between SARS-CoV-2 and
SARS-CoV by analyzing plasma samples from SARS-CoV-2 (n = 15) and
SARS-CoV (n = 7) patients. Cross-reactivity in antibody binding to the
spike protein between SARS-CoV-2 and SARS-CoV was found to be common,
mostly targeting non-RBD regions in plasma from SARS-CoV-2 patients.
Only one SARS-CoV-2 plasma sample was able to cross-neutralize SARS-CoV,
with low neutralization activity. No cross-neutralization response was
detected in plasma from SARS-CoV patients.

To further investigate the cross-reactivity of antibody responses to
SARS-CoV-2 and SARS-CoV, the authors analyzed the antibody response of
plasma collected from mice infected or immunized with SARS-CoV-2 or
SARS-CoV (n = 5 or 6 per group). Plasma from mice immunized with
SARS-CoV-2 displayed cross-reactive responses to SARS-CoV S ectodomain
and, to a lesser extent, SARS-CoV RBD. Similarly, plasma from mice
immunized with SARS-CoV displayed cross-reactive responses to SARS-CoV-2
S ectodomain. Cross-neutralization activity was not detected in any of
the mouse plasma samples.

#### Limitations

The size of each patient cohort is insufficient to accurately determine
the frequency of cross-reactivity and cross-neutralization in the
current SARS-CoV-2 pandemic. Recruitment of additional patients from a
larger range of geographical regions and time points would also enable
exploration into the effect of the genetic diversity and evolution of
the SARS-CoV-2 virus on cross-reactivity. This work would also benefit
from the mapping of specific epitopes for each sample. Future studies
may determine whether the non-neutralizing antibody responses can confer
*in vitro* protection or lead to antibody-dependent disease enhancement.

#### Significance

The cross-reactive antibody responses to S protein in the majority of
SARS-CoV-2 patients is an important consideration for development of
serological assays and vaccine development during the current outbreak.
The limited extent of cross-neutralization demonstrated in this study
indicates that vaccinating to cross-reactive conserved epitopes may have
limited efficacy, presenting a key concern for the development of a more
universal coronavirus vaccine to address the global health risk of novel
coronavirus outbreaks.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### The feasibility of convalescent plasma therapy in severe COVID-19 patients: a pilot study

Duan et al. *medRxiv* [@doi:10.1101/2020.03.16.20036145]

#### Keywords

- COVID-19
- SARS-CoV-2
- convalescent plasma
- treatment outcome
- pilot
- therapy
- transfusion

#### Main Findings

This is the first report to date of convalescent plasma therapy as a
therapeutic against COVID-19 disease. This is a feasibility pilot study.
The authors report the administration and clinical benefit of 200 mL of
convalescent plasma (CP) (1:640 titer) derived from recently cured
donors (CP selected among 40 donors based on high neutralizing titer and
ABO compatibility) to 10 severe COVID-19 patients with confirmed
viremia. The primary endpoint was the safety of CP transfusion. The
secondary endpoint were clinical signs of improvement based on symptoms
and laboratory parameters.

The authors reported use of methylene blue photochemistry to inactivate
any potential residual virus in the plasma samples, without compromising
neutralizing antibodies, and no virus was detected before transfusion.

The authors report the following:

-   No adverse events were observed in all patients, except 1 patient
    who exhibited transient facial red spotting.

-   All patients showed significant improvement in or complete
    disappearance of clinical symptoms, including fever, cough,
    shortness of breath, and chest pain after 3 days of CP therapy.

-   Reduction of pulmonary lesions revealed by chest CT.

-   Elevation of lymphocyte counts in patients with lymphocytopenia.

-   Increase in SaO~2~ in all patients, indicative of recuperating lung
    function.

-   Resolution of SARS-CoV-2 viremia in 7 patients and increase in
    neutralizing antibody titers in 5 patients. Persistence of
    neutralizing antibody levels in 4 patients.

#### Limitations

It is important to note that most recipients had high neutralization
titers of antibodies before plasma transfusion and even without
transfusion it would be expected to see an increase in neutralizing
antibodies over time. In addition to the small sample set number (n=10),
there are additional limitations to this pilot study:

1.  All patients received concurrent therapy, in addition to the CP
    transfusion. Therefore, it is unclear whether a combinatorial or
    synergistic effect between these standards of care and CP
    transfusion contributed to the clearance of viremia and improvement
    of symptoms in these COVID-19 patients.

2.  The kinetics of viral clearance was not investigated, with respect
    to the administration of CP transfusion. So, the definitive impact
    of CP transfusion on immune dynamics and subsequent viral load is
    not well defined.

3.  Comparison with a small historical control group is not ideal.

#### Significance

For the first time, a pilot study provides promising results involving
the use of convalescent plasma from cured COVID-19 patients to treat
others with more severe disease. The authors report that the
administration of a single, high-dose of neutralizing antibodies is
safe. In addition, there were encouraging results with regards to the
reduction of viral load and improvement of clinical outcomes. It is,
therefore, necessary to expand this type of study with more
participants, in order to determine optimal dose and treatment kinetics.
It is important to note that CP has been studied to treat H1N1
influenza, SARS-CoV-1, and MERS-CoV, although it has not been proven to
be effective in treating these infections.

#### Credit

*Review by Matthew D. Park and revised by Alice O. Kamphorst and Maria A. Curotto de Lafaille as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Hydroxychloroquine and azithromycin as a treatment of COVID-19: results of an open label non-randomized clinical trial

[@doi:10.1101/2020.03.16.20037135]

#### Keywords

- hydroxychloroquine
- clearance
- viral load
- clinical trial

#### Main Findings

This study was a single-arm, open label clinical trial with
600 mg hydroxychloroquine (HCQ) in the treatment arm (n = 20). Patients
who refused participation or patients from another center not treated
with HCQ were included as negative controls (n = 16). Among the patients
in the treatment arm, 6 received concomitant azithromycin to prevent
superimposed bacterial infection. The primary endpoint was respiratory
viral loads on day 6 post enrollment, measured by nasopharyngeal swab
followed by real-time reverse transcription-PCR.

HCQ alone was able to significantly reduce viral loads by day 6 (n =
8/14, 57.1% complete clearance, p &lt; 0.001); azithromycin appears to
be synergistic with HCQ, as 6/6 patients receiving combined treatment
had complete viral clearance (p &lt; 0.001).

#### Limitations

Despite what is outlined above, this study has a number
of limitations that must be considered. First, there were originally n =
26 patients in the treatment arm, with 6 lost to follow up for the
following reasons: 3 transferred to ICU, 1 discharge, 1
self-discontinued treatment d/t side effects, and 1 patient expired.
Total length of clinical follow up was 14 days, but the data beyond day
6 post-inclusion are not shown.

Strikingly, in supplementary table 1, results of the real-time RT-PCR
are listed for the control and treatment arms from D0 – D6. However, the
data are not reported in a standard way, with a mix of broadly positive
or negative result delineation with Ct (cycle threshold) values, the
standard output of real time PCR. It is impossible to compare what is
defined as a positive value between the patients in the control and
treatment arms without a standardized threshold for a positive test.
Further, the starting viral loads reported at D0 in the groups receiving
HCQ or HCQ + azithromycin were significantly different (ct of 25.3 vs
26.8 respectively), which could explain in part the differences observed
in the response to treatment between 2 groups. Finally, patients in the
control arm from outside the primary medical center in this study
(Marseille) did not actually have samples tested by PCR daily. Instead,
positive test results from every other day were extrapolated to mean
positive results on the day before and after testing as well (Table 2,
footnote ^a^).

Taken together, the results of this study suggest that HCQ represents a
promising treatment avenue for COVID-19 patients. However, the limited
size of the trial, and the way in which the results were reported does
not allow for other medical centers to extrapolate a positive or
negative result in the treatment of their own patients with HCQ +/-
azithromycin. Further larger randomized clinical trials will be required
to ascertain the efficacy of HCQ +/- azithromycin in the treatment of
COVID-19.

#### Significance

Chloroquine is thought to inhibit viral infection,
including SARS-Cov-2, by increasing pH within endosomes and lysosomes,
altering the biochemical conditions required for viral fusion [@doi:10.1038/s41422-020-0282-0; @doi:10.1016/j.imlet.2013.07.004].
However, chloroquine also has immuno-modulatory effects that I think may
play a role. Chloroquine has been shown to increase CTLA-4 expression at
the cell surface by decreasing its degradation in the endo-lysosome
pathway; AP-1 traffics the cytoplasmic tail of CTLA-4 to lysosomes, but
in conditions of increased pH, the protein machinery required for
degradation is less functional [@doi:10.1126/science.aaa1663]. As such, more CTLA-4 remains in
endosomes and is trafficked back to the cell surface. It is possible
that this may also contribute to patient recovery via reduction of
cytokine storm, in addition to the direct anti-viral effects of HCQ.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Recapitulation of SARS-CoV-2 Infection and Cholangiocyte Damage with Human Liver Organoids

[@doi:10.1101/2020.03.16.990317]

#### Keywords

- CoVID-19
- 2019-nCoV
- SARS-CoV-2
- liver
- organoids
- Cholangiocyte

#### Main Findings

-   Used human liver ductal organoids to determine ACE2+ cholangiocytes
    in healthy liver (2.92% of all cells) are infectable and support
    SARS-CoV-2 viral replication.

-   Plaque-purified SARS-CoV-2 viral infection disrupted organoid
    barrier and bile transporting functions of cholangiocytes through
    dysregulation of genes involved in tight junction formation (CLDN1)
    and bile acid transportation (ASBT and CFTR).

#### Limitations:

-   Unclear if liver damage observed in patients due to direct
    cholangiocyte infection or due to secondary immune/cytokine effects.
    This study argues for direct damage as it lacks immune contexture;
    but further studies needed with autopsy samples or organoid-immune
    cell co-culture to conclude strongly.

-   Would be important to measure cholangiocyte-intrinsic anti-viral
    response and alarmins secreted upon infection, and furthermore study
    tropism of various immune cells to conditioned media from organoids
    infected with SARS-CoV-2.

-   Does not address how cirrhotic liver or
    alcohol/smoking/obesity-associated liver organoids respond to
    SARS-CoV-2 infectivity and replication, worth pursuing to
    experimentally address clinical data indicating co-morbidities.

#### Significance


-   Useful model to rapidly study drug activity against SARS-CoV-2
    infection in liver, while monitoring baseline liver damage.

-   Liver abnormality observed in &gt;50% of CoVID-19 patients; the
    results from this study could explain the bile acid accumulation and
    consequent liver damage observed.

#### Credit

*Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### The sequence of human ACE2 is suboptimal for binding the S spike protein of SARS coronavirus 2

[@doi:10.1101/2020.03.16.994236]

#### Keywords

- Coronavirus Disease 2019 (COVID-19)
- SARS-CoV-2
- Spike protein S
- ACE2

#### Main Findings

Severe Acute Respiratory Syndrome Coronavirus 2
(SARS-CoV-2) infects cells through S spike glycoprotein binding
angiotensin-converting enzyme (ACE2) on host cells. S protein can bind
both membrane-bound ACE2 and soluble ACE2 (sACE2), which can serve as a
decoy that neutralizes infection. Recombinant sACE2 is now being tested
in clinical trials for COVID-19. To determine if a therapeutic sACE2
with higher affinity for S protein could be designed, authors generated
a library containing every amino acid substitution possible at the 117
sites spanning the binding interface with S protein. The ACE2 library
was expressed in human Expi293F cells and cells were incubated with
medium containing the receptor binding domain (RBD) of SARS-CoV-2 fused
to GFP. Cells with high or low affinity mutant ACE2 receptor compared to
affinity of wild type ACE2 for the RBD were FACS sorted and transcripts
from these sorted populations were deep sequenced. Deep mutagenesis
identified numerous mutations in ACE2 that enhance RBD binding. This
work serves to identify putative high affinity ACE2 therapeutics for the
treatment of CoV-2.

#### Limitations

The authors generated a large library of mutated
ACE2, expressed them in human Expi293F cells, and performed deep
mutagenesis to identify enhanced binders for the RBD of SARS-CoV-2 S
protein. While these data serve as a useful resource, the ability of the
high affinity ACE2 mutants identified to serve as therapeutics needs
further validation in terms of conformational stability when purified as
well as efficacy/safety both *in vitro* and *in vivo*. Additionally,
authors mentioned fusing the therapeutic ACE2 to Fc receptors to elicit
beneficial host immune responses, which would need further design and
validation.

#### Significance

This study identified structural ACE2 mutants that
have potential to serve as therapeutics in the treatment of SARS-CoV-2
upon further testing and validation.

#### Credit

*This review was undertaken by Katherine Lindblad and Tamar Plitt as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


***Title***:  A serological assay to detect SARS-Cov-2 seroconversion in
humans

***Immunology keywords***:  specific serological assay - ELISA -
seroconversion - antibody titers

*Note: the authors of this review work in the same institution as the
authors of the study*

***Main findings***:  

Production of recombinant whole Spike (S) protein and the smaller
Receptor Binding Domain (RBD) based on the sequence of Wuhan-Hu-1
SARS-CoV-2 isolate. The S protein was modified to allow trimerization
and increase stability. The authors compared the antibody reactivity of
59 banked human serum samples (non-exposed) and 3 serum samples from
confirmed SARS-CoV-2 infected patients. All Covid-19 patient sera
reacted to the S protein and RBD domain compared to the control sera.

The authors also characterized the antibody isotypes from the Covid-19
patients, and observed stronger IgG3 response than IgG1. IgM and IgA
responses were also prevalent.

***Limitations of the study***:  

The authors analyzed a total of 59 control human serum samples, and
samples from only three different patients to test for reactivity
against the RBD domain and full-length spike protein. It will be
important to follow up with a larger number of patient samples to
confirm the data obtained. Furthermore, it would be interesting to
assess people at different age groups and determine whether unexposed
control kids have a higher “background”.

Applications of the assay described in this study in diagnosis are
limited, since antibody response should start to be detectable only one
to two weeks after infection. Future studies will be required to assess
how long after infection this assay allow to detect anti-CoV2
antibodies. Finally, while likely, the association of seroconversion
with protective immunity against SARS-Cov-2 infection still needs to be
fully established.

***Relevance***:

This study has strong implications in the research against SARS-Cov-2.
First, it is now possible to perform serosurveys and determine who has
been infected, allowing a more accurate estimate of infection prevalence
and death rate. Second, if it is confirmed that re-infection does not
happen (or is rare), this assay can be used as a tool to screen
healthcare workers and prioritize immune ones to work with infected
patients. Third, potential convalescent plasma donors can now be
screened to help treating currently infected patients. Of note, this
assay does not involve live virus handling. experimentally, this is an
advantage as the assay does not require the precautions required by
manipulation of live virus. Finally, the recombinant proteins described
in this study represent new tools that can be used for further
applications, including vaccine development.


### COMPARATIVE PATHOGENESIS OF COVID-19, MERS AND SARS IN A NON-HUMAN PRIMATE MODEL

[@doi:10.1101/2020.03.17.995639]

#### Keywords

- SARS-CoV2
- cynomolgus macaque
- SARS-CoV

#### Main Findings

This work assesses SARS-CoV-2 infection in young adult and aged
cynomolgus macaques. 4 macaques per age group were infected with
low-passage clinical sample of SARS-CoV-2 by intranasal and
intratracheal administration. Viral presence was assessed in nose,
throat and rectum through RT-PCR and viral culture. SARS-CoV-2
replication was confirmed in the respiratory track (including nasal
samples), and it was also detected in ileum. Viral nucleocapsid
detection by IHC showed infection of type I and II pneumocytes and
epithelia. Virus was found to peak between 2 and 4 days after
administration and reached higher levels in aged vs. young animals. The
early peak is consistent with data in patients and contrasts to SARS-CoV
replication. SARS-CoV-2 reached levels below detection between 8 and 21
days after inoculation and macaques established antibody immunity
against the virus by day 14. There were histopathological alteration in
lung, but no overt clinical signs. At day 4 post inoculation of
SARS-CoV-2, two of four animals presented foci of pulmonary
consolidation, with limited areas of alveolar edema and pneumonia, as
well as immune cell infiltration. In sum, cynomolgus macaques are
permissive to SARS- CoV-2 and develop lung pathology (less severe than
SARC-CoV, but more severe than MERS-CoV).

#### Limitations

Even though cynomolgus macaques were permissive to SARS-CoV-2
replication, it is unclear if the viral load reaches levels comparable
to humans and there wasn’t overt clinical pathology.

#### Significance

The development of platforms in which to carry out relevant
experimentation on SARS-CoV-2 pathophysiology is of great urgency.
Cynomolgus macaques offer an environment in which viral replication can
happen, albeit in a limited way and without translating into clinically
relevant symptoms. Other groups are contributing to SARS-CoV2 literature
using this animal model [@doi:10.1101/2020.03.13.990226], potentially showing protection against
reinfection in cured macaques. Therefore, this platform could be used to
examine SARS-CoV2 pathophysiology while studies in other animal models
are also underway [@doi:10.1128/JVI.02012-06; @doi:10.1101/2020.02.07.939389].

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Investigating the Impact of Asymptomatic Carriers on COVID-19 Transmission

[@doi:10.1101/2020.03.18.20037994]

#### Keywords

- Coronavirus Disease 2019 (COVID-19)
- SARS-CoV-2
- asymptomatic carriers
- mathematical model
- transmission

#### Main Findings

Multiple studies reported the same level of infectiousness
between symptomatic and asymptomatic carriers of SARS-CoV-2. Given that
asymptomatic and undocumented carriers escape public health surveillance
systems, a better mathematical model of transmission is needed to
determine a more accurate estimate of the basic reproductive number
(R~0~) of the virus to assess the contagiousness of virus. The authors
developed a SEYAR dynamical model for transmission of the new
coronavirus that takes into account asymptomatic and undocumented
carriers. The model was validated using data reported from thirteen
countries during the first three weeks of community transmission. While
current studies estimate R~0~ to be around 3, this model indicates that
the value could range between 5.5 to 25.4.

#### Limitations

The SEYAR model realistically depicts
transmission of the virus only during the initial stages of the disease.
More data is necessary to better fit the model with current trends. In
addition, multiple factors (e.g. behavioral patterns, surveillance
capabilities, environmental and socioeconomic factors) affect
transmission of the virus and so, these factors must be taken into
consideration when estimating the R~0~.

#### Significance

Public health authorities use the basic reproductive
number to determine the severity of disease. An accurate estimate of
R~0~ will inform intervention strategies. This model can be applied to
different locations to assess the potential impact of COVID-19.

#### Credit

*This review was undertaken by Tamar Plitt and Katherine Lindblad as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Antibody responses to SARS-CoV-2 in COVID-19 patients: the perspective application of serological tests in clinical practice

Long et al. *medRxiv* [@doi:10.1101/2020.03.18.20038018]

#### Keywords

- Serum antibodies
- IgM
- IgG
- immunoassay
- diagnosis
- seroconversion

#### Main Findings

This study investigated the profile of the acute antibody response
against SARS-CoV-2 and provided proposals for serologic tests in
clinical practice. Magnetic Chemiluminescence Enzyme Immunoassay was
used to evaluate IgM and IgG seroconversion in 285 hospital admitted
patients who tested positive for SARS-CoV-2 by RT-PCR and in 52 COVID-19
suspected patients that tested negative by RT-PCR. A follow up study
with 63 patients was performed to investigate longitudinal effects. In
addition, IgG and IgM titers were evaluated in a cohort of close
contacts (164 persons) of an infected couple.

The median day of seroconversion for both IgG and IgM was 13 days after
symptom onset. Patients varied in the order of IgM/ IgG seroconversion
and there was no apparent correlation of order with age, severity, or
hospitalization time. This led the authors to conclude that for
diagnosis IgM and IgG should be detected simultaneously at the early
phase of infection.

IgG titers, but not IgM titers were higher in severe patients compared
to non-severe patients after controlling for days post-symptom onset.
Importantly, 12% of COVID-19 patients (RT-PCR confirmed) did not meet
the WHO serological diagnosis criterion of either seroconversion or
*&gt;* 4-fold increase in IgG titer in sequential samples. This suggests
the current serological criteria may be too stringent for COVID-19
diagnosis.

Of note, 4 patients from a group of 52 suspects (negative RT-PCR test)
had anti-SARS-Cov-2 IgM and IgG. Similarly, 4.3% (7/162) of "close
contacts" who had negative RT-PCR tests were positive for IgG and/or
IgM. This highlights the usefulness of a serological assay to identify
asymptomatic infections and/or infections that are missed by RT-PCR.

#### Limitations

This group’s report generally confirms the findings of others that have
evaluated the acute antibody response to SARS-Cov-2. However, these data
would benefit from inclusion of data on whether the participants had a
documented history of viral infection. Moreover, serum samples that were
collected prior to SARS-Cov-2 outbreak from patients with other viral
infections would serve as a useful negative control for their assay.
Methodological limitations include that only one serum sample per case
was tested as well as the heat inactivation of serum samples prior to
testing. It has previously been reported that heat inactivation
interferes with the level of antibodies to SARS-Cov-2 and their protocol
may have resulted in diminished quantification of IgM, specifically [@doi:10.1101/2020.03.12.20034231].

#### Significance

Understanding the features of the antibody responses against SARS-CoV is
useful in the development of a serological test for the diagnosis of
COVID-19. This paper addresses the need for additional screening methods
that can detect the presence of infection despite lower viral titers.
Detecting the production of antibodies, especially IgM, which are
produced rapidly after infection can be combined with PCR to enhance
detection sensitivity and accuracy and map the full spread of infection
in communities, Moreover, serologic assays would be useful to screen
health care workers in order to identify those with immunity to care for
patients with COVID19.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### SARS-CoV-2 specific antibody responses in COVID-19 patients

[@doi:10.1101/2020.03.18.20038059]

#### Keywords
- immunoassay
- antibody specificity
- serology
- cross-reactivity
  
#### Main findings

Antibodies specific to SARS-CoV-2 S protein, the S1 subunit and the RBD (receptor-binding domain) were detected in all SARS-CoV-2 patient sera by 13 to 21 days post onset of disease.  Antibodies specific to SARS-CoV N protein (90% similarity to SARS-CoV-2) were able to neutralize SARS-CoV-2 by PRNT (plaque reduction neutralizing test).  SARS-CoV-2 serum cross-reacted with SARS-CoV S and S1 proteins, and to a lower extent with MERS-CoV S protein, but not with the MERS-CoV S1 protein, consistent with an analysis of genetic similarity.  No reactivity to SARS-CoV-2 antigens was observed in serum from patients with ubiquitous human CoV infections (common cold) or to non-CoV viral respiratory infections.  
 
#### Limitations

Authors describe development of a serological ELISA based assay for the detection of neutralizing antibodies towards regions of the spike and nucleocapsid domains of the SARS-CoV-2 virus.  Serum samples were obtained from PCR-confirmed COVID-19 patients.  Negative control samples include a cohort of patients with confirmed recent exposure to non-CoV infections (i.e. adenovirus, bocavirus, enterovirus, influenza, RSV, CMV, EBV) as well as a cohort of patients with confirmed infections with ubiquitous human CoV infections known to cause the common cold. The study also included serum from patients with previous MERS-CoV and SARS-CoV zoonotic infections. This impressive patient cohort allowed the authors to determine the sensitivity and specificity of the development of their in-house ELISA assay. Of note, seroconversion was observed as early as 13 days following COVID-19 onset but the authors were not clear how disease onset was determined.  
 
#### Significance

Validated serological tests are urgently needed to map the full spread of SARS-CoV-2 in the population and to determine the kinetics of the antibody response to SARS-CoV-2.  Furthermore, clinical trials are ongoing using plasma from patients who have recovered from SARS-CoV-2 as a therapeutic option.  An assay such as the one described in this study could be used to screen for strong antibody responses in recovered patients.  Furthermore, the assay could be used to screen health care workers for antibody responses to SARS-CoV-2 as personal protective equipment continues to dwindle.  The challenge going forward will be to standardize and scale-up the various in-house ELISA’s being developed in independent laboratories across the world.  
 
 


### A brief review of antiviral drugs evaluated in registered clinical trials for COVID-19

Belhadi et al. [@doi:10.1101/2020.03.18.20038190]

#### Keywords

- Clinical trials
- COVID-19
- SARS CoV-2
- 2019-nCoV
- SARS Cov-2
- Hcov-19
- novel coronal virus
- new corona virus
- antiviral drugs

#### Main Findings

Summary of clinical trials registered as
of March7, 2020 from U.S, Chinese, Korean, Iranian and European
registries. Out of the 353 studies identified, 115 were selected for
data extraction. 80% of the trials were randomized with parallel
assignment and the median number of planned inclusions was 63 (IRQ,
36-120). Most frequent therapies in the trials included; 1) antiviral
drugs \[lopinavir/ritonavir (n-15); umifenovir (n=9); favipiravir (n=7);
redmesivir (n=5)\]; 2) anti-malaria drugs \[chloroquine (n-11);
hydroxychloroquine (n=7)}; immunosuppressant drugs \[methylprednisolone
(n=5)\]; and stem cell therapies (n=23). Medians of the total number of
planned inclusions per trial for these therapies were also included.
Stem cells and lopunavir/ritonavir were the most frequently evaluated
candidate therapies (23 and 15 trials respectively), whereas remdesivir
was only tested in 5 trials but these trials had the highest median
number of planned inclusions per trial (400, IQR 394-453). Most of the
agents used in the different trials were chosen based on preclinical
assessments of antiviral activity against SARS CoV and MERS Cov corona
viruses.

The primary outcomes of the studies were clinical (66%); virological
(23%); radiological (8%); or immunological (3%). The trials were
classified as those that included patients with severe disease only;
trials that included patients with moderate disease; and trials that
included patients with severe or moderate disease.

#### Limitations

The trials evaluated provided incomplete information:
23% of these were phase IV trials but the bulk of the trials (54%) did
not describe the phase of the study. Only 52% of the trials (n=60)
reported treatment dose and only 34% (n=39) reported the duration. A lot
of the trials included a small number of patients and the trials are
still ongoing, therefore no insight was provided on the outcome of the
trials.

#### Significance

Nonetheless, this review serves as framework for
identifying COVID-19 related trials, which can be expanded upon as new
trials begin at an accelerated rate as the disease spreads around the
world.

#### Credit

*This review was undertaken by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### ACE-2 Expression in the Small Airway Epithelia of Smokers and COPD Patients: Implications for COVID-19

Leung et al. *medRxiv*. [@doi:10.1101/2020.03.18.20038455]

#### Keywords

- chronic obstructive pulmonary disease
- COPD
- smokingE-2
- risk factors

#### Main Findings

In bronchial epithelial samples from 3 different cohorts of
individuals, ACE-2 gene expression was found to be significantly
increased in both COPD patients and smokers relative to healthy
controls. Across all test subjects, ACE-2 gene expression was also
highly correlated with decreased forced expiratory volume in 1 second
(FEV1), which may explain the increased COVID-19 disease severity in
COPD patients. Former smokers were also found to show decreased ACE2
expression relative to current smokers and had no significant difference
when compared to non-smokers.

#### Limitations

While the upregulation of ACE-2 is an interesting
hypothesis for COVID-19 disease severity in COPD patients, this study
leaves many more unanswered questions than it addresses. Further studies
are required to show whether the specific cell type isolated in these
studies is relevant to the pathophysiology of COVID-19. Furthermore,
there is no attempt to show whether that increased ACE-2 expression
contributes to greater disease severity. Does the increased ACE-2
expression lead to greater infectivity with SARS-CoV-2? There is no
mechanistic explanation for why ACE-2 levels are increased in COPD
patients. The authors could also have considered the impact of
co-morbidities and interventions such as corticosteroids or
bronchodilators on ACE-2 expression. Finally, given the extensive
sequencing performed, the authors could have conducted significantly
more in-depth analyses into gene signature differences.

#### Significance

This study attempts to address an
important clinical finding that both smokers and COPD patients show
increased mortality from COVID-19. The novel finding that ACE-2
expression is induced in smokers and COPD patients suggests not only a
mechanism for the clinical observation, but also highlights the
potential benefit of smoking cessation in reducing the risk of severe
COVID-19 disease.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Dynamic profile of severe or critical COVID-19 cases

[@doi:10.1101/2020.03.18.20038513]

#### Keywords
- Coronavirus Disease 2019 (COVID-19)
- SARS-CoV-2
- progressive lymphopenia (PLD)
- T-lymphocytes
- clinical data
- co-infection
- influenza A

#### Main Findings

Authors evaluate clinical correlates of 10 patients (6 male
and 4 female) hospitalized for severe Acute Respiratory Syndrome
Coronavirus 2 (SARS-CoV-2). All patients required oxygen support and
received broad spectrum antibiotics and 6 patients received anti-viral
drugs. Additionally, 40% of patients were co-infected with influenza A.
All 10 patients developed lymphopenia, two of which developed
progressive lymphopenia (PLD) and died. Peripheral blood (PB)
lymphocytes were analyzed – low CD4 and CD8 counts were noted in most
patients, though CD4:CD8 ratio remained normal.

#### Limitations

The authors evaluated a small cohort of severe
SARS-CoV-2 cases and found an association between T cell lymphopenia and
adverse outcomes. However, this is an extremely small and diverse cohort
(40% of patients were co-infected with influenza A). These findings need
to be validated in a larger cohort. Additionally, the value of this data
would be greatly increased by adding individual data points for each
patient as well as by adding error bars to each of the figures.

#### Significance

This study provides a collection of clinical data and
tracks evolution of T lymphocyte in 10 patients hospitalized for
SARS-CoV-2, of which 4 patients were co-infected with influenza A.

#### Credit

*This review was undertaken by Katherine Lindblad and Tamar Plitt as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Association between Clinical, Laboratory and CT Characteristics and RT-PCR Results in the Follow-up of COVID-19 patients

Fu et al. *medRxiv*. [@doi:10.1101/2020.03.19.20038315]

#### Keywords

- COVID-19
- clinical
- lymphocyte
- CRP
- LDH
- HSST TNT
- PCR test
- readmission
- CT
- GGO
- disease progression

#### Study description

Data analyzed from 52 COVID-19 patients admitted
and then discharged with COVID-19. Clinical, laboratory, and
radiological data were longitudinally recorded with illness timecourse
(PCR + to PCR-) and 7 patients (13.5%) were readmitted with a follow up
positive test (PCR+) within two weeks of discharge.

#### Main Findings

-   At admission:

    -   The majority of patients had increased CRP at admission (63.5%).

    -   LDH, and HSST TNT were significantly increased at admission.

    -   Radiographic signs via chest CT showed increased involvement in
        lower lobes: right lower lobe (47 cases, 90.4%), left lower lobe
        (37 cases, 71.2%).

    -   GGO (90.4%), interlobular septal thickening (42.3%), vascular
        enlargement (42.3%), and reticulation (11.5%) were most commonly
        observed.

-   After negative PCR test (discharge):

    -   CRP levels decreased lymphocyte counts (\#/L) increased
        significantly (CD3+, CD3+/8+ and CD3+/4+) after negative PCR.

    -   Consolidation and mixed GGO observed in longitudinal CT imaging
        w different extents of inflammatory exudation in lungs, with
        overall tendency for improvement (except 2/7 patients that were
        readmitted after discharge with re-positive test) after negative
        PCR.

-   Seven patients repeated positive RT-PCR test and were readmitted to
    the hospital (9 to 17 day after initial discharge).

    -   Follow up CT necessary to monitor improvement during recovery
        and patients with lesion progression should be given more
        attention.

    -   Dynamic CT in addition to negative test essential in clinical
        diagnosis due to nasal swab PCR sampling bias (false-negatives).

    -   Increase in CRP occurred in 2 readmitted patients (and decr. in
        lymphocytes in one patient), but was not correlated with new
        lesions or disease progression vs. improvement (very low N).

    -   Patients readmitted attributed to false-negative PCR vs.
        re-exposure.

#### Limitations

Patients sampled in this study were generally
younger (65.4% &lt; 50 yrs) and less critically ill/all discharged.
Small number of recovered patients (N=18). Time of follow up was
relatively short.. Limited clinical information available about patients
with re-positive test (except CRP and lymph tracking).

#### Extended Results

*NOTE*: Patients sampled in this study were generally younger (65.4%
&lt; 50 yrs) and less critically ill/all discharged. After two
consecutive negative PCR tests, patients were discharged.

Clinical Results at Admission

-   Median interval disease onset to admission (5 days, IQR: 3-7)

-   Most common symptoms included fever, fatigue, dry cough, and
    expectoration.

-   Fifteen patients had reduced lymphocyte counts (28.8%).

-   No change in WBC or Neutrophil counts.

-   **The majority of patients had increased CRP at admission (63.5%).**

-   **LDH, and HSST TNT were significantly increased at admission.**

-   Fibrinogen was trending high though not significant.

-   No major changes in liver function observed.

-   **Radiographic signs via chest CT showed increased involvement in
    lower lobes: right lower lobe (47 cases, 90.4%), left lower lobe (37
    cases, 71.2%).**

-   **GGO (90.4%), interlobular septal thickening (42.3%), vascular
    enlargement (42.3%)**, and reticulation (11.5%) were most commonly
    observed.

Change in Clinical Results following Negative Test

-   **CRP levels decreased after negative PCR.**

-   **Lymphocyte counts (\#/L) increased significantly (CD3+, CD3+/8+
    and CD3+/4+).**

-   No significant change to CD4/8 ratio.

-   LDH, HSST TNT, and Fibronegin remained high throughout, though range
    observed decreased over time.

-   **Consolidation and mixed GGO observed in longitudinal CT imaging.**

-   **Patients showed different extents of inflammatory exudation in
    lungs, with overall tendency for improvement (except 2/7 patients
    that were readmitted after discharge with re-positive test).**

Patients Readmitted with PCR+ test

-   **Seven patients repeated positive RT-PCR test and were readmitted
    to the hospital (9 to 17 day after initial discharge).**

-   Improvement during readmission in 4 patients and observation of
    segmental progression CT in 2 patients (2/18 or 11.1% - re-positive
    9 and 10 days post-discharge).

-   Two patients showed new GGO, while others improved greatly.

-   **Follow up CT necessary to monitor improvement during recovery and
    patients with lesion progression should be given more attention.**

-   **Dynamic CT in addition to negative test essential in clinical
    diagnosis due to nasal swab PCR sampling bias (false-negatives).**

-   **Increase in CRP occurred in 2 readmitted patients (and decr. in
    lymphocytes in one patient), but was not correlated with new lesions
    or disease progression vs. improvement (very low N).**

#### Significance

Study tracked key clinical features associated with
disease progression, recovery, and determinants of clinical
diagnosis/management of COVID-19 patients.

#### Credit

*This review was undertaken by Natalie Vaninov as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 and multiple 2 endemic, epidemic and bat coronavirus

Sheahan et al. *bioRxiv*. [@doi:10.1101/2020.03.19.997890]

#### Keywords
- Treatment
- Antiviral
- Broad spectrum antiviral
- ribonucleoside analog β-D-N4 30 hydroxycytidine (NHC)
- Remdesivir

#### Main Findings

β-D-N4 30 –hydroxycytidine (NHC, EIDD-1931) is an
orally bioavailable ribonucleoside with antiviral activity against
various RNA viruses including Ebola, Influenza and CoV. NHC activity
introduce*d*~~s~~ mutations in the viral (but not cellular) RNA in a
dose dependent *manner* that directly correlated with a decrease in
viral titers. Authors show that NHC inhibited multiple genetically
distinct Bat-CoV viruses in human primary epithelial cells *without
affecting* *cell viability even at high concentrations (100* *µM)*.
Prophylactic oral administration of NHC in C57BL/6 mice reduce lung
titers of SARS-CoV and prevented weight loss and hemorrhage. Therapeutic
administration of NHC in C57BL/6 mice 12 hours post infected with
SARS-CoV reduced acute lung injury, viral titer, and lung hemorrhage.
The degree of clinical benefit was dependent on the time of treatment
initiation post infection. The authors also demonstrate that NHC reduces
MERS-CoV infection titers, pathogenesis, and viral RNA in prophylactic
and therapeutic settings.

#### Limitations

Most of the experiments were conducted using MERS-CoV, and
SARS-CoV and a few experiments were conducted using other strains of CoV
as opposed to SARS-CoV-2. The authors note the core residues that make
up the RNA interaction sites (which constitutes the NHC interaction
sites) are highly conserved among CoV and because of this conservation
their understanding is that NHC can inhibit a broad-spectrum of CoV
including SARS-CoV-2.

The increased viral mutation rates associated with NHC activity may have
adverse effects if mutations cause the virus to become drug resistant,
more infectious or speed-up immune evasion. *In addition, the temporal
diminishing effectiveness of NHC on clinical outcome when NHC was used
therapeutically is concerning. However, the longer window (7-10 days)
for clinical disease onset in human patients from the time of infection
compared to that of mice (24-48 hours), may associate with increased NHC
effectiveness in the clinic.*

#### Significance

Prophylactic or therapeutic oral administration of NHC
reduces lung titers and prevents acute lung failure in C57B\\6 mice
infected with CoV. Given its *broad-spectrum antiviral activity, NHC
could turn out to be a useful drug for treating current, emerging and
future corona virus outbreaks.*
 
#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Identification of antiviral drug candidates against SARS-CoV-2 from FDA-approved drugs

Sangeun Jeon et al. [@doi:10.1101/2020.03.20.999730]

#### Keywords

- COVID-19
- SARS CoV-2
- antiviral drugs
- niclosamide
- ciclesonide

#### Main Findings

A panel of \~3,000 FDA- and IND-approved
antiviral drugs were previously screened for inhibitory efficacy against
SARS CoV, a coronavirus related to the novel coronavirus SARS CoV-2
(79.5%) homology. 35 of these drugs along with another 15 (suggested by
infectious disease specialists) were tested in vitro for their ability
to inhibit SARS CoV-2 infectivity of Vero cells while preserving cell
viability. The infected cells were scored by immunofluorescence analysis
using an antibody against the N protein of SARS CoV-2. Chloroquine,
lopinavir and remdesivir were used as reference drugs.

Twenty four out of 50 drugs exhibited antiviral activity with IC50
values ranging from 0.1-10µM. Among these, two stood ou: 1) the-anti
helminthic drug niclosamide which exhibited potent antiviral activity
against SARS CoV-2 (IC50=0.28 µM). The broad-spectrum antiviral effect
of niclosamide against SARS and MERS-CoV have been previously documented
and recent evidence suggests that in may inhibit autophagy and reduce
MERS C0V replication. 2) Ciclesonide, a corticosteroid used to treat
asthma and allergic rhinitis, also exhibited antiviral efficacy but with
a lower IC50 (4.33µM) compared to niclosamide. The antiviral effects of
ciclesonide were directed against NSP15, a viral riboendonuclease which
is the molecular target of this drug.

#### Limitations

The drugs were tested against SARS CoV-2 infectivity in
vitro only, therefore preclinical studies in animals and clinical trials
in patients will be need for validation of these drugs as therapeutic
agents for COVID-19. In addition, niclosamide exhibits low adsorption
pharmatokinetically which could be alleviated with further development
of drug formulation to increase effective delivery of this drug to
target tissues. Nonetheless, niclosamide and ciclesonide represent
promising therapeutic agents against SARS CoV-2 given that other
compounds tested in the same study including favipiravir (currently used
in clinical trials) and atazanavir (predicted as the most potent
antiviral drug by AI-inference modeling) did not exhibit antiviral
activity in the current study.

#### Credit

*This review was undertaken by K Alexandropoulos as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Respiratory disease and virus shedding in rhesus macaques inoculated with SARS-CoV-2

Munster et al. *bioRxiv*. [@doi:10.1101/2020.03.21.001628]

#### Keywords

animal model, pulmonary infiltrates, dynamic
of antibody response, cytokine

#### Main Findings

Inoculation of 8 Resus macaques with SARS-Cov-2 , which
all showed clinical signs of infection (respiratory pattern, reduced
appetite, weight loss, elevated body temperature) resulting in moderate,
transient disease. Four animals were euthanized at 3dpi, the 4 others at
21 dpi. Study of viral loads in different organs showed that nose swab
and throat swabs were the most sensitive, with bronchio-alveolar lavage.
Interstitial pneumonia was visible in radiographies and at the
histological scale too. Clinically, the macaques had similar symptoms as
described in human patients with moderate disease.

Viral shedding was consistently detected in nose swabs and throat swabs
immediately after infection but less consistent thereafter which could
reflect virus administration route (intranasal, oral). Bronchoalveolar
lavages performed as a measure of virus replication in the lower
respiratory tract on animals maintained for 21 days, contained high
viral loads in 1 and 3dpi. The majority of the animals exhibited
pulmonary edema and mild to moderate interstitial pneumonia on terminal
bronchioles. In addition to the lung, viral RNA could also be detected
throughout the respiratory track where viral replication mainly
occurred.

Immunologic responses included leukocytosis, neutrophilia, monocytosis
and lymphopenia in the majority of the animals at 1dpi. Lymphocytes and
monocytes re-normalized at 2dpi. Neutrophils declined after 3dpi and
through 10dpi after which they started to recover. After infection,
serum analysis revealed significant increases in **IL1ra, IL6, IL10,
IL15, MCP-1, MIP-1b, but quick normalization** (3dpi). **Antibody
response started around 7dpi, and the antibody titers stayed elevated
until 21dpi** (day of animal euthanasia).

#### Limitations

The macaques were inoculated via a combination
of intratracheal, intranasal, ocular and oral routes, which might not
reproduce how humans get infected. Maybe this can lead to different
dynamics in the host immune response. Also, the authors noted that the
seroconversion was not directly followed by a decline in viral loads, as
observed in covid19 patients.

#### Significance

This work confirms that rhesus
macaques can be a good model to study Covid-19, as it has been shown by
other groups [@doi:10.1101/2020.03.13.990226; @doi:10.1101/2020.03.13.990036; @doi:10.1101/2020.03.17.995639]. While these experiments recapitulate
moderate COVID-19 in humans, the mode of inoculation via a combination
of intratracheal, intranasal, ocular and oral routes, might not
reproduce how humans get infected and may lead to different dynamics in
the host immune response. For example, the authors noted that the
seroconversion was not directly followed by a decline in viral loads, as
observed in COVID-19 patients. Therefore, it will be interesting to
follow their antibody titers longer and further assess the
possibility/effect of reinfection in these macaques. It is essential to
be able to understand the dynamic of the disease and associated immune
responses, and to work on vaccine development and antiviral drug
testing.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### ACE2 Expression is Increased in the Lungs of Patients with Comorbidities Associated with Severe COVID-19

[@doi:10.1101/2020.03.21.20040261]

#### Keywords

- CoVID-19
- 2019-nCoV
- SARS-CoV-2
- lung
- comorbidities
- histone
- epigenetics

#### Main Findings

-   Transcriptomic analysis using systems-level meta-analysis and
    network analysis of existing literature to determine ACE2 regulation
    in patients who have frequent COVID-19 comorbidities \[eg-
    cardiovascular diseases, familial pulmonary hypertension, cancer\].

-   Enrichment analyses indicated pathways associated with inflammation,
    metabolism, macrophage autophagy, and ER stress.

-   ACE2 higher in adenocarcinoma compared to adjacent normal lung; ACE2
    higher in COPD patients compared to normal.

-   Co-expression analysis identified genes important to viral entry
    such as RAB1A, ADAM10, HMGBs, and TLR3 to be associated with ACE2 in
    diseased lungs.

-   ACE2 expression could be potentially regulated by enzymes that
    modify histones, including HAT1, HDAC2, and KDM5B.

#### Limitations:

-   Not actual CoVID-19 patients with co-morbidities, so interpretations
    in this study need to be confirmed by analyzing upcoming
    transcriptomics from CoVID-19 patients having co-morbidity metadata.

-   As mentioned by authors, study does not look at diabetes and
    autoimmunity as risk factors in CoVID-19 patients due to lack of
    data; would be useful to extend such analyses to those datasets when
    available.

-   Co-expression analysis is perfunctory and needs
    validation-experiments especially in CoVID-19 lung samples to mean
    anything.

-   Epigenomic analyses are intriguing but incomplete, as existence of
    histone marks does not necessarily mean occupancy. Would be
    pertinent to check cell-line data (CCLE) or actual CoVID-19 patient
    samples to confirm ACE2 epigenetic control.

#### Significance

-   Study implies vulnerable populations have ACE2 upregulation that
    could promote CoVID-19 severity. Shows important data-mining
    strategy to find gene-networks associated with ACE2 upregulation in
    co-morbid patients.

-   Several of the genes co-upregulated with ACE2 in diseased lung might
    play an important role in CoVID-19 and can be preliminary targets
    for therapeutics

-   If in silico findings hold true, epigenetic control of ACE2
    expression could be a new target for CoVID-19 therapy with
    strategies such as KDM5 demethylases.

#### Credit

*Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Meplazumab treats COVID-19 pneumonia: an open-labelled, concurrent controlled add-on clinical trial

Bian et al. *medRxiv*. [@doi:10.1101/2020.03.21.20040691]

#### Keywords

- Meplazumab
- CD147
- humanized antibody
- clinical trial

#### Main Findings

This work is based on previous work by the same group
that demonstrated that SARS-CoV-2can also enter host cells via CD147
(also called Basigin, part of the immunoglobulin superfamily, is
expressed by many cell types) consistent with their previous work with
SARS-CoV-1. ^1^ A prospective clinical trial was conducted with 17
patients receiving Meplazumab, a humanized anti-CD147 antibody, in
addition to all other treatments. 11 patients were included as a control
group (non-randomized).

They observed a faster overall improvement rate in the Meplazumab group
(e.g. at day 14 47% vs 17% improvement rate) compared to the control
patients. Also, virological clearance was more rapid with median of 3
days in the Meplazumab group vs 13 days in control group. In laboratory
values, a faster normalization of lymphocyte counts in the Meplazumab
group was observed, but no clear difference was observed for CRP levels.

#### Limitations

While the results from the study are encouraging, this
study was non-randomized, open-label and on a small number of patients,
all from the same hospital. It offers evidence to perform a larger scale
study. Selection bias as well as differences between treatment groups
(e.g. age 51yo vs 64yo) may have contributed to results. The authors
mention that there was no toxic effect to Meplazumab injection but more
patient and longer-term studies are necessary to assess this.

#### Significance

These results seem promising as for now there are
limited treatments for Covid-19 patients, but a larger cohort of patient
is needed. CD147 has already been described to facilitate HIV [@doi:10.1073/pnas.111583198],
measles virus [@doi:10.1128/JVI.02168-09], and malaria [@doi:10.1038/nature10606] entry into host cells. This group was
the first to describe the CD147-spike route of SARS-Cov-2 entry in host
cells [@doi:10.1101/2020.03.14.988345] p147. Indeed, they had previously shown in 2005 that SARS-Cov
could enter host cells via this transmembrane protein [@doi:10.1086/427811]. Further
biological understanding of how SARS-CoV-2 can enter host cells and how
this integrates with ACE2R route of entry is needed. Also, the specific
cellular targets of the anti-CD147 antibody need to be assessed, as this
protein can be expressed by many cell types and has been shown to
involved in leukocytes aggregation [@doi:10.1371/journal.pone.0215557]. Lastly, Meplazumab is not a
commercially-available drug and requires significant health resources to
generate and administer which might prevent rapid development and use.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Potent human neutralizing antibodies elicited 1 by SARS-CoV-2 infection

Ju et al. *bioRxiv*. [@doi:10.1101/2020.03.21.990770]

#### Keywords

- monoclonal antibodies
- neutralization
- antibody cross-reactivity
- Receptor Binding Domain

#### Main Findings

In this study the authors report the affinity, cross reactivity (with
SARS-CoV and MERS-CoV virus) and viral neutralization capacity of 206
monoclonal antibodies engineered from isolated IgG memory B cells of
patients suffering from SARS-CoV-2 infection in Wuhan, China. All
patients but one recovered from disease. Interestingly, the patient that
did not recover had less SARS-CoV-2 specific B cells circulating
compared to other patients.

Plasma from all patients reacted to trimeric Spike proteins from
SARS-CoV-2, SARS-CoV and MERS-CoV but no HIV BG505 trimer. Furthermore,
plasma from patients recognized the receptor binding domain (RBD) from
SARS-CoV-2 but had little to no cross-reactivity against the RBD of
related viruses SARS-CoV and MERS-CoV, suggesting significant
differences between the RBDs of the different viruses. Negligible levels
of cross-neutralization using pseudoviruses bearing Spike proteins of
SARS-CoV-2, SARS-CoV or MERS-CoV, were observed, corroborating the ELISA
cross-reactivity assays on the RBDs.

SARS-CoV-2 RBD specific B cells constituted 0.005-0.065% of the total B
cell population and 0.023-0.329% of the memory subpopulation. SARS-CoV
specific IgG memory B cells were single cell sorted to sequence the
antibody genes that were subsequently expressed as recombinant IgG1
antibodies. From this library, 206 antibodies with different binding
capacities were obtained. No discernible patterns of VH usage were found
in the 206 antibodies suggesting immunologically distinct responses to
the infection. Nevertheless, most high-binding antibodies were derived
by clonal expansion. Further analyses in one of the patient derived
clones, showed that the antibodies from three different timepoints did
not group together in phylogenetic analysis, suggesting selection during
early infection.

Using surface plasmon resonance (SPR) 13 antibodies were found to have
10^-8^ tp 10^-9^ dissociation constants (Kd). Of the 13 antibodies, two
showed 98-99% blocking of SARS-CoV-2 RBD-ACE2 receptor binding in
competition assays. Thus, low Kd values alone did not predict ACE2
competing capacities. Consistent with competition assays the two
antibodies that show high ACE2 blocking (P2C-2F6 and P2C-1F11) were the
most capable of neutralizing pseudoviruses bearing SARS-CoV-2 spike
protein (IC~50~ of 0.06 and 0.03 µg/mL, respectively). Finally, using
SPR the neutralizing antibodies were found to recognize both overlapping
and distinct epitopes of the RBD of SARS-CoV-2.

#### Limitations

1.  Relatively low number of patients

    a.  No significant conclusion can be drawn about the possible
        > correlation between humoral response and disease severity

2.  *In vitro* Cytopathic Effect Assay (CPE) for neutralization activity

    a.  Huh7 cells were used in neutralization assays with
        > pseudoviruses, and they may not entirely mimic what happens in
        > the upper respiratory tract

    b.  CPE assay is not quantitative

3.  Duplicated panel in Figure 4C reported (has been fixed in version 2)

#### Significance

This paper offers an explanation as to why
previously isolated antibodies against SARS-CoV do not effectively block
SARS-CoV-2. Also, it offers important insight into the development of
humoral responses at various time points during the first weeks of the
disease in small but clinically diverse group of patients. Furthermore,
it provides valuable information and well characterized antibody
candidates for the development of a recombinant antibody treatment for
SARS-CoV-2. Nevertheless, it also shows that plasmapheresis might have
variability in its effectiveness, depending on the donor’s antibody
repertoire at the time of donation.

#### Credit

*Review by Jovani Catalan-Dibene as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Characterisation of the transcriptome and proteome of SARS-CoV-2 using direct RNA sequencing and tandem mass spectrometry reveals evidence for a cell passage induced in-frame deletion in the spike glycoprotein that removes the furin-like cleavage site.

Davidson et al. [@doi:10.1101/2020.03.22.002204]

#### Keywords

- Transcription
- RNA-seq
- proteomics
- mass spec
- furin cleavage site
- mutation
- pathogenicity

#### Main Findings

The authors performed long read RNA sequencing using an Oxford Nanopore
MinION as well as tandem mass spec (MS) on Vero cells (a cell line
derived from kidney cells of the African green monkey that is deficient
in interferon) infected with SARS-CoV-2.

The authors found that passage of the virus in Vero cells gave rise to a
spontaneous 9 amino acid deletion (679-NSPRRARSV-687 to I) in the spike
(S) protein. The deleted sequence overlaps a predicted furin cleavage
site at the S1 / S2 domain boundary that is present in SARS-CoV-2 but
not SARS-CoV or the closely related bat coronavirus RaTG13, which are
cleaved at S1 / S2 by other proteases [@doi:10.1126/science.abb2507]. Furin cleavage sites at
similar positions in other viruses have been linked to increased
pathogenicity and greater cell tropism [@doi:10.1128/JVI.00797-10]. Loss of this site in
SARS-CoV-2 has also already been shown to increase viral entry into Vero
but not BHK cells (which are also interferon deficient) [@doi:10.1016/j.cell.2020.02.058]. The
authors therefore make an important contribution in demonstrating that
passage in Vero cells may lead to spontaneous loss of a key
pathogenicity-conferring element in SARS-CoV-2.

#### Limitations

As the authors note, a similar study posted earlier by Kim et al., which
also passaged SARS-CoV-2 in Vero cells, did not identify any loss in the
S protein furin cleavage site [@doi:10.1101/2020.03.12.988865]. It therefore remains to be
determined how likely it is that this mutation spontaneously arises. A
quantitative investigation using multiple experimental replicas to
understand the spontaneous viral mutation rate at this site and
elsewhere would be informative. Also, the mechanistic basis for the
higher viral fitness conferred by loss of the furin cleavage site in
Vero cells – but, evidently, not in vivo in humans, as this site is
maintained in all currently sequenced circulating isolates - remains to
be understood.

Due to the high base-call error rate of MinION sequencing, the authors’
bioinformatic pipeline required aligning transcripts to a reference to
correct sequencing artifacts. This presumably made it difficult or
impossible to identify other kinds of mutations, such as single
nucleotide substitutions, which may occur even more frequently than the
deletions identified in this work. Pairing long read sequencing with
higher-accuracy short-read sequencing may be one approach to overcome
this issue.

#### Significance

As the authors suggest, animal studies using live virus challenge may
need to periodically verify the genomic integrity of the virus, or
potentially risk unknowingly using a likely less-pathogenic variant of
the virus.

More broadly, the results emphasize the complexity and plasticity of the
SARS-CoV-2 viral transcriptome and proteome. For example, the authors
found multiple versions of transcripts encoding the nucleocapsid (N)
protein, each with different small internal deletions, some of which
were verified for translation by MS. A number of peptides arising from
translation of unexpected rearrangements of transcripts were also
detected. Additionally, the authors identified phosphorylation of a
number of viral proteins (N, M, ORF 3a, nsp3, nsp9, nsp12 and S). For
any cases where these have functional consequences, targeting the
kinases responsible could be an avenue for drug development.
Understanding the functional consequences of the mutations, transcript
variations, and post translational modifications identified in this
study will be important future work.

#### Credit

*This review was undertaken by Tim O’Donnell, Maria Kuksin as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### A SARS-CoV-2-Human Protein-Protein Interaction Map Reveals Drug Targets and Potential Drug- Repurposing

Gordon et al. *bioRxiv* [@doi:10.1101/2020.03.22.002386]

#### Keywords

- protein-protein interactions
- mass spectrometry
- drug targets

#### Main Findings

Gordon et al cloned, tagged and expressed 26 of the 29
SARS-CoV-2 proteins individually in HEK293T cells and used mass
spectrometry to identify protein-protein interactions. They identified
332 viral-host protein-protein interactions. Furthermore, they used
these interactions to identify 66 existing drugs known to target host
proteins or host pathways (eg SARS-CoV-2 N and Orf8 proteins interact
with proteins regulated by the mTOR pathway, so mTOR inhibitors
Silmitasertib and Rapamycin are possible drug candidates).

#### Limitations

The main limitation of the study stems from
the reductionist model: overexpression of plasmids encoding individual
viral proteins in HEK293T cells. This precludes any interactions between
the viral proteins, or the combined effects of multiple proteins on the
host, as they are expressed individually. Moreover, HEK293T cells come
from primary embryonic kidney and therefore might not reflect how
SARS-CoV-2 interacts with its primary target, the lung. However, the
authors found that the proteins found to interact with viral proteins in
their experiments are enriched in lung tissue compared to HEK293Ts.

#### Significance

The authors provide a “SARS-CoV-2 interaction map,”
which may provide potential hypotheses as to how the virus interacts
with the host. Further, they identified existing drugs that could
disrupt these host-viral interactions and curb SARS-CoV-2 infection.
Although these interactions have not been validated, this paper acts as
a valuable resource.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### First Clinical Study Using HCV Protease Inhibitor Danoprevir to Treat Naïve and Experienced COVID-19 Patients

Chen et al. *medRxiv*.
[@doi:10.1101/2020.03.22.20034041]

#### Keywords
- Clinical study
- HCV protease inhibitor
- Danoprevir
- Ritonavir
- Covid19 treatment

#### Main Findings

The authors treated 11 Covid-19 patients with Danoprevir, a
commercialized HCV protease inhibitor [@doi:10.1128/AAC.00699-08]^(p4)^, boosted by ritonavir [@doi:10.14218/JCTH.2019.00033], a
CYP3A4 inhibitor (which enhances the plasma concentration and
bioavailabilty of Danoprevir). Two patients had never received
anti-viral therapy before (=naïve), whereas nine patients were on
Lopinavir/Ritonavir treatment before switching to Danoprevir/Ritonavir
(=experienced). The age ranged from 18 to 66yo.

Naïve patients that received Danoprevir/Ritonavir treatment had a
decreased hospitalization time. Patients treated with
Lopinavir/Ritonavir did not have a negative PCR test, while after
switching to Danoprevir/Ritonavir treatment, the first negative PCR test
occurred at a median of two days.

#### Limitations

The results of the study are very hard to interpret as there is no
control group not receiving Danoprevir/Ritonavir treatment. This was
especially true in naïve patients who seemed to have more mild symptoms
before the start of the study and were younger (18 and 44yo) compared to
the experienced patients (18 to 66yo). The possibility that the patients
would have recovered without Danoprevir/Ritonavir treatment cannot be
excluded.

#### Significance

The authors of the study treated patients with Danoprevir, with the
rational to that this is an approved and well tolerated drug for HCV
patients [@doi:10.14218/JCTH.2019.00033], and that it could also target the protease from SARS-CoV-2
(essential for viral replication and transcription). Indeed, homology
modelling data indicated that HCV protease inhibitors have the highest
binding affinity to Sars-Cov2 protease among other approved drugs [@doi:10.1101/2020.02.05.936013].

While this study shows that the combination of Danoprevir and Ritonavir
might be beneficial for Covid-19 patients, additional clinical trials
with more patients and with better methodology (randomization and
control group) are needed to make further conclusions.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Efficacy of hydroxychloroquine in patients with COVID-19: results of a randomized clinical trial

[@doi:10.1101/2020.03.22.20040758]

#### Keywords

- hydroxychloroquine

#### Study Description

This is a randomized clinical trial of hydroxychloroquine (HCQ) efficacy
in the treatment of COVID-19. From February 4 – February 28, 2020 142
COVID-19 positive patients were admitted to Renmin Hospital of Wuhan
University. 62 patients met inclusion criteria and were enrolled in a
double blind, randomized control trial, with 31 patients in each arm.

Inclusion criteria:

1.  Age ≥ 18 years

2.  Positive diagnosis COVID-19 by detection of SARS-CoV-2 by RT-PCR

3.  Diagnosis of pneumonia on chest CT

4.  Mild respiratory illness, defined by SaO~2~/SPO~2~ ratio &gt; 93% or
    PaO~2~/FIO~2~ ratio &gt; 300 mmHg in hospital room conditions (Note:
    relevant clinical references described below.)

    a.  Hypoxia is defined as an SpO2 of 85-94%; severe hypoxia &lt;
        85%.

    b.  The PaO~2~/FIO~2~ (ratio of arterial oxygen tension to fraction
        of inspired oxygen) is used to classify the severity of acute
        respiratory distress syndrome (ARDS). Mild ARDS has a
        PaO~2~/FIO~2~ of 200-300 mmHg, moderate is 100-200, and severe
        &lt; 100.

5.  Willing to receive a random assignment to any designated treatment
    group; not participating in another study at the same time

Exclusion criteria:

1.  Severe or critical respiratory illness (not explicitly defined,
    presumed to be respiratory function worse than outlined in inclusion
    criteria); or participation in trial does not meet patient’s maximum
    benefit or safe follow up criteria

2.  Retinopathy or other retinal diseases

3.  Conduction block or other arrhythmias

4.  Severe liver disease, defined by Child-Pugh score ≥ C or AST &gt;
    twice the upper limit

5.  Pregnant or breastfeeding

6.  Severe renal failure, defined by eGFR ≤ 30 mL/min/1.73m^2^, or on
    dialysis

7.  Potential transfer to another hospital within 72h of enrollment

8.  Received any trial treatment for COVID-19 within 30 days before the
    current study

All patients received the standard of care: oxygen therapy, antiviral
agents, antibacterial agents, and immunoglobulin, with or without
corticosteroids. Patients in the HCQ treatment group received additional
oral HCQ 400 mg/day, given as 200 mg 2x/day. HCQ was administered from
days 1-5 of the trial. The primary endpoint was 5 days post enrollment
or a severe adverse reaction to HCQ. The primary outcome evaluated was
time to clinical recovery (TTCR), defined as return to normal body
temperature and cough cessation for &gt; 72h. Chest CT were imaged on
days 0 and 6 of the trial for both groups; body temperature and patient
reports of cough were collected 3x/day from day 0 – 6. The mean age and
sex distribution between the HCQ and control arms were comparable.

#### Main Findings

There were 2 patients showing mild secondary effects of HCQ treatment.
More importantly, while 4 patients in the control group progressed to
severe disease, none progressed in the treatment group.

TTCR was significantly decreased in the HCQ treatment arm; recovery from
fever was shortened by one day (3.2 days control vs. 2.2 days HCQ, p =
0.0008); time to cessation of cough was similarly reduced (3.1 days
control vs. 2.0 days HCQ, p = 0.0016).

Overall, it appears that HCQ treatment of patients with mild COVID-19
has a modest effect on clinical recovery (symptom relief on average 1
day earlier) but may be more potent in reducing the progression from
mild to severe disease.

#### Limitations

This study is limited in its inclusion of only patients with mild
disease, and exclusion of those on any treatment other than the standard
of care. It would also have been important to include the laboratory
values of positive RT-PCR detection of SARS-CoV-2 to compare the
baseline and evolution of the patients’ viral load.

#### Limitations

Despite its limitations, the study design has good rigor as a double
blind RCT and consistent symptom checks on each day of the trail. Now
that the FDA has approved HCQ for treatment of COVID-19 in the USA, this
study supports the efficacy of HCQ use early in treatment of patients
showing mild symptoms, to improve time to clinical recovery, and
possibly reduce disease progression. However, most of the current
applications of HCQ have been in patients with severe disease and for
compassionate use, which are out of the scope of the findings presented
in this trial. Several additional clinical trials to examine
[hydroxychloroquine](https://twitter.com/hashtag/hydroxychloroquine?src=hashtag_click)
are now undergoing; their results will be critical to further validate
these findings.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


**Structure-based modeling of SARS-CoV-2 peptide/HLA-A02 antigens**

<https://doi.org/10.1101/2020.03.23.004176>

*Immunology keywords*:

CoVID-19, 2019-nCoV, SARS-CoV-2, comparative, homology, peptide,
modeling, simulation, HLA-A, antigen

*Summary of Findings*:

-   The authors utilize homology modeling to identify peptides from the
    SARS-CoV-2 proteome that potentially bind HLA-A\*02:01.

-   They utilize high-resolution X-ray structures of peptide/MHC
    complexes on Protein Data Bank, substitute homologous peptides with
    SARS-CoV-2 peptides, and calculate MHC/SARS-CoV-2 peptide Rosetta
    binding energy.

-   They select MHC/SARS-CoV-2 complex models with highest binding
    energy for further study and publish models in an online database
    (https://rosettamhc.chemistry.ucsc.edu).

*Limitations*:

-   The authors only utilize computational methods and predicted
    SARS-CoV-2 peptides must be validated experimentally for
    immunogenicity and clinical response.

-   Due to computational burden and limited availability of high
    resolution X-ray structures on PDB, authors only simulate 9-mer and
    10-mer peptide binding to HLA-A\*02:01.

-   Since the authors compare select existing X-ray structures as a
    starting point, backbone conformations that deviate significantly
    between test and template peptides are not captured. Furthermore,
    Rosetta modeling protocols do not capture all possible structures
    and binding energy scoring does not fully recapitulate fundamental
    forces.^1,2^

*Importance/Relevance:*

-   The authors identify and publish high-scoring SARS-CoV-2 peptides
    that may direct a targeted, experimental validation approach toward
    a COVID-19 vaccine.

-   The authors utilize Rosetta simulation to further filter results
    from NetMHCpan 4.0, supporting machine learning prediction with
    structural analysis.

-   The authors develop RosettaMHC, a computationally efficient method
    of leveraging existing X-ray structures for identification of
    immunogenic peptides.

*References:*

1.  Bender, B. J., Cisneros, A., 3rd, Duran, A. M., Finn, J. A., Fu, D.,
    Lokits, A. D., . . . Moretti, R. (2016). Protocols for Molecular
    Modeling with Rosetta3 and RosettaScripts. Biochemistry, 55(34),
    4748-4763. doi:10.1021/acs.biochem.6b00444

2.  Alford, R. F., Leaver-Fay, A., Jeliazkov, J. R., O'Meara, M. J.,
    DiMaio, F. P., Park, H., . . . Gray, J. J. (2017). The Rosetta
    All-Atom Energy Function for Macromolecular Modeling and Design. J
    Chem Theory Comput, 13(6), 3031-3048. doi:10.1021/acs.jctc.7b00125

*Review by Jonathan Chung as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn school of medicine,
Mount Sinai.*


### Serology characteristics of SARS-CoV-2 infection since the exposure and post symptoms onset

Lou et al. *medRxiv*. [@doi:10.1101/2020.03.23.20041707]

#### Keywords

- Seroconversion rate
- Total Antibody
- Ab
- IgG and IgM
- antibody

#### Main Findings

Currently, the diagnosis of SARS-CoV-2 infection
entirely depends on the detection of viral RNA using polymerase chain
reaction (PCR) assays. False negative results are common, particularly
when the samples are collected from upper respiratory. Serological
detection may be useful as an additional testing strategy. In this study
the authors reported that a typical acute antibody response was induced
during the SARS-CoV-2 infection, which was discuss earlier^1^. The
seroconversion rate for Ab, IgM and IgG in COVID-19 patients was 98.8%
(79/80), 93.8% (75/80) and 93.8% (75/80), respectively. The first
detectible serology marker was total antibody followed by IgM and IgG,
with a median seroconversion time of 15, 18 and 20 days-post exposure
(d.p.e) or 9, 10- and 12-days post-onset (d.p.o). Seroconversion was
first detected at day 7d.p.e in 98.9% of the patients. Interestingly
they found that viral load declined as antibody levels increased. This
was in contrast to a previous study [@doi:10.1101/2020.03.02.20030189], showing that increased antibody
titers did not always correlate with RNA clearance (low number of
patient sample).

#### Limitations

Current knowledge of the antibody response to SAR-CoV-2
infection and its mechanism is not yet well elucidated. Similar to the
RNA test, the absence of antibody titers in the early stage of illness
could not exclude the possibility of infection. A diagnostic test, which
is the aim of the authors, would not be useful at the early time points
of infection but it could be used to screen asymptomatic patients or
patients with mild disease at later times after exposure.

#### Significance

Understanding the antibody responses against SARS-CoV2 is
useful in the development of a serological test for the diagnosis of
COVID-19. This manuscript discussed acute antibody responses which can
be deducted in plasma for diagnostic as well as prognostic purposes.
Thus, patient-derived plasma with known antibody titers may be used
therapeutically for treating COVID-19 patients with severe illness.
[[]{#OLE_LINK4 .anchor}]{#OLE_LINK3 .anchor}

#### Credit

*This review was undertaken and edited by Konstantina A as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### SARS-CoV-2 launches a unique transcriptional signature from in vitro, ex vivo, and in vivo systems

Blanco-Melo et al. *bioRxiv*. [@doi:10.1101/2020.03.24.004655]

#### Keywords

- host cellular response
- host-pathogen interaction
- type I interferon
- type III interferon
- inflammation
- RNA-seq
- comparative analysis

#### Main Findings

Given the high mortality rate of SARS-CoV-2 relative to
other respiratory viruses such seasonal IAV and RSV, there may be
underlying host-pathogen interactions specific to SARS-CoV-2 that
predispose to a worse clinical outcome. Using *in vivo*, *ex vivo*, and
*in vitro* systems, the authors profiled host cell transcriptional
responses to SARS-CoV-2 and to other common respiratory viruses
(seasonal IAV and RSV). SARS-CoV-2 infection *in vitro* led to an
induction of type I interferon response signaling and the upregulation
of cytokine/chemokines transcripts. In comparison with IAV and RSV
infection, SARS-CoV-2 *in vitro* appears to uniquely induce less type I
and type III interferon expression and higher levels of two cytokines
previously implicated in respiratory inflammation. Lastly, *in vivo*
data from ferrets showed a reduced induction of cytokines and chemokines
by SARS-CoV-2 infection relative to IAV infection.

#### Limitations

While these results are promising, there are several
key weaknesses of this paper. 1) As the authors point out, there is an
undetectable level of SARS-CoV-2 putative receptor (ACE2) and protease
(TMPRSS2) expression in the lung epithelial cell line used for the *in
vitro* studies. This raises the important question of whether viral
replication actually occurs in any of the models used, which may explain
the lack of interferon production observed *in vitro* in SARS-CoV-2
treated cells. Further studies characterizing viral titers across
timepoints are needed. 2) Furthermore, these studies only characterize
the host response at a single dose and timepoint per virus, and it is
unclear why these doses/timepoints were chosen. This leaves open the
possibility that the observed differences between viruses could be due
to differences in dose, timing, host response, or a combination of all
of these. 3) It is unclear whether ferrets are productively infected,
which cell types are infected, and the extent/timing of the clinical
course of infection. Moreover, the *in vitro* and *in vivo* data do not
strongly correlate and the reasons for this are unclear.

#### Significance

This paper describes potentially unique
transcriptional signatures of host cells exposed to SARS-CoV-2. If
validated, these findings may help explain clinical outcomes and could
be targeted in future therapeutic interventions.

#### Potential Conflicts of Interest Disclosure

The reviewers are also researchers at the Icahn School
of Medicine at Mount Sinai.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### A New Predictor of Disease Severity in Patients with COVID-19 in Wuhan, China

Zhou et al. *bioRxiv*. [@doi:10.1101/2020.03.24.20042119]

#### Keywords

- disease severity
- clinical data
- Neutrophils/lymphocytes ratio
- CRP
- D-dimer

#### Main Findings

377 hospitalized patients were divided into two groups: severe and
non-severe pneumonia. The laboratory results of their first day of
admission were retrospectively analyzed to identify predictors of
disease severity.

After adjusting for confounding factors from chronic comorbidities (such
as high blood pressure, type 2 diabetes, coronary heart disease, and
chronic obstructive pulmonary disease), the independent risk factors
identified for severe pneumonia were **age**, the **ratio of
neutrophil/lymphocytes counts**, **CRP** and **D-dimer** levels.

To further increase the specificity and sensibility of these markers,
they showed that their multiplication **\[(Neutrophil/lymphocyte count)
\* CRP \* D-dimer\]** was a better predictor of disease severity, with
higher sensitivity (95.7%) and specificity (63.3%), with a cutoff value
of 2.68.

#### Limitations

This study included 377 hospitalized patients. Among them, 45.6%
patients tested positive for SARS-Cov-2 nucleic acid test results, and
others were included in the study based on clinically diagnosis even if
the molecular diagnosis was negative. Thus, additional studies are
needed to verify this on a larger number of covid-19 certified patients
and the cutoff value might be adjusted. Also, all the patients that did
not have the clinical characteristics of severe pneumonia were included
in the non-severe pneumonia group, but usually patients are also divided
into moderate and mild disease.

Also, studying different subset of lymphocytes could lead to a more
specific predictor. Another study showed that the neutrophils to CD8+ T
cells ratio was a strong predictor of disease severity [@doi:10.1101/2020.02.16.20023671]. Another more
precise study showed that the percentage of helper T cells and
regulatory T cells decrease but the percentage of naïve helper T cells
increases in severe cases [@doi:10.1093/cid/ciaa248]. Taking these subpopulations into account
might make the predictor more powerful.

Other studies also noted an inverse correlation between disease severity
and LDH [@doi:10.1101/2020.03.03.20030668] or IL6 [@doi:10.1101/2020.03.08.20031229] levels, but the authors here do not discuss LDH nor
IL6 levels, although this could help to strengthen the predictor.

The study is based on the results obtained on the first day of
admission, studying the dynamic of the changes in patients might also be
interesting to better predict disease severity.

#### Significance

This study confirms that the neutrophil to lymphocyte ratio can be a
predictor of disease severity as shown by many others [@doi:10.1093/cid/ciaa248; @doi:10.1001/jama.2020.1585; @doi:10.1101/2020.02.10.20021584]. The
novelty here is that they show that a combination with other markers can
enhance the specificity and sensibility of the predictor, although the
study could be improved by taking into account sub-populations of
lymphocytes and more biological factors from patients such as LDH and
IL6.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Metabolic disturbances and inflammatory dysfunction predict severity of coronavirus disease 2019 (COVID-19): a retrospective study

Shuke Nie et al. *medRxiv*. [@doi:10.1101/2020.03.24.20042283]

#### Keywords

- metabolism
- fasting blood glucose
- serum total protein
- albumin
- blood lipid
- HDL-C
- APOA1
- lymphocytopenia
- IL-6
- CRP
- severity prediction of COVID19

#### Main Findings

Retrospective Study on 97 COVID-19 hospitalized patients
(25 severe and 72 non-severe) analyzing clinical and laboratory
parameter to predict transition from mild to severe disease based on
more accessible indicators (such as fasting blood glucose, serum protein
or blood lipid) than inflammatory indicators. In accordance with other
studies, age and hypertension were risk factors for disease severity,
and lymphopenia and increased IL-6 was observed in severe patients. The
authors show that fasting blood glucose (FBG) was altered and patients
with severe disease were often hyperglycemic. Data presented support
that hypoproteinaemia, hypoalbuminemia, and reduction in
high-densitylipoprotein (HDL-C) and ApoA1 were associated with disease
severity.

#### Limitations

In this study non-severe patients were divided in two groups based on
average course of the disease: mild group1 (14 days, n=28) and mild
group 2 (30 days, n=44). However mild patients with a longer disease
course did not show an intermediate phenotype (between mild patients
with shorter disease course and severe patients), hence it is unclear
whether this was a useful and how it impacted the analysis. Furthermore,
the non-exclusion of co-morbidity factors in the analysis may bias the
results (e.g. diabetic patients and glucose tests) It is not clear at
what point in time the laboratory parameters are sampled. In table 3, it
would have been interesting to explore a multivariate multiple
regression. The correlation lacks of positive control to assess the
specificity of the correlation to the disease vs. correlation in any
inflammatory case. The dynamic study assessing the predictability of the
laboratory parameter is limited to 2 patients. Hence there are several
associations with disease severity, but larger studies are necessary to
test the independent predictive value of these potential biomarkers.

#### Significance

As hospital are getting overwhelmed a set of
easily accessible laboratory indicators (such as serum total protein)
would potentially provide a triage methodology between potentially
severe cases and mild ones. This paper also opens the question regarding
metabolic deregulation and COVID-19 severity.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Viral Kinetics and Antibody Responses in Patients with COVID-19

[@doi:10.1101/2020.03.24.20042382]

#### Keywords

- CoVID-19
- 2019-nCoV
- SARS-CoV-2
- IgG
- IgM
- clinical
- kinetics
- antibodies

#### Main Findings

-   Prospective cohort of 67 patients, clinical specimens taken and
    follow-up conducted.

-   Viral shedding, serum IgM, IgG antibody against NP evaluated and
    correlated to disease severity and clinical outcome

-   Viral RNA levels peaked at 1 week from febrile/cough symptom onset
    in sputum, nasal swabs, and stool samples. Shedding ranged from
    12-19 days (median ranges) and was longer in severe patients.

-   IgM and IgG titers stratified patients into three archetypes as
    ‘strong vs weak vs non-responders’. Strong responders (with higher
    IgM/IgG titers) were significantly higher in severe patients.

#### Limitations

Specific for immune monitoring.

-   Not clear if stool RNA captured from live infection in
    intestine/liver or from swallowed sputum. Transmission electron
    microscopy (TEM) carried out on sputum samples as proof of concept,
    but not stools. TEM unreasonable for actual clinical diagnosis.

-   Several patients had co-morbidities (such as pulmonary and liver
    disease) that were not accounted for when tracking antibody
    responses. Viral kinetics and IgM/IgG titers in subsets of patients
    with underlying conditions/undergoing certain medication would be
    informative.

#### Significance

-   Three archetypes of antibody response to SARS-CoV-2 with different
    disease progression and kinetics is useful to stratify patients, and
    for future serological tests.

-   Strong spike-IgG levels often correlate with lymphopenia and
    CoVID-19 disease severity [@doi:10.1101/2020.03.20.20039495], similar to macaque
    studies in SARS [@doi:10.1172/jci.insight.123158]. It would be critical to see if anti-NP or
    anti-Spike IgG antibodies for SARS-CoV-2 also elicit similar
    detrimental effects before clinical use.

#### Credit

*Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### COVID-19 infection induces readily detectable morphological and inflammation-related phenotypic changes in peripheral blood monocytes, the severity of which correlate with patient outcome

[@doi:10.1101/2020.03.24.20042655]

#### Keywords

- Monocytes
- FSC-high
- PBMC
- ACE2
- inflammatory cytokines

#### Main Findings

This study is based on flow cytometry immunophenotyping of
PBMCs from 28 patients diagnosed positive for SARS-Cov2 (COVID19). The
authors identify a population of abnormally large (FSC-hi) monocytes,
present in COVID19 patients, but absent in PBMCs of healthy volunteers
(n=16) or patients with different infections (AIDS, malaria, TB). This
FSC-hi monocytic population contains classical, intermediate and
non-classical (monocytes (based on CD14 and CD16 expression) that
produce inflammatory cytokines (IL-6, TNF and IL-10). The authors
suggest an association of FSC-hi monocytes with poor outcome and
correlate a high percentage of FSC-low monocytes, or higher ratio of
FSC-low/hi monocytes, with faster hospital discharge.

#### Limitations

While identification of the monocytic population based
on FSC is rather robust, the characterization of these cells remains
weak. A comprehensive comparison of FSC-hi monocytes with FSC-low
monocytes from patients and healthy controls would be of high value. It
is unclear if percentages in blood are among CD45+ cells. Furthermore,
it would have been important to include absolute numbers of different
monocytic populations (in table 1 there are not enough samples and it is
unclear what the authors show).

The authors show expression of the ACE2 receptor on the surface of the
monocytes, and highlight these cells as potential targets of SARS-Cov2.
However, appropriate controls are needed. CD16 has high affinity to
rabbit IgG and it is unclear whether the authors considered unspecific
binding of rabbit anti-ACE2 to Fc receptors. Gene expression of ACE-2 on
monocytes needs to be assessed. Furthermore, it would be important to
confirm infection of monocytes by presence of viral proteins or viral
particles by microscopy.

Considering the predictive role of FSC-hi monocytes on the development
of the disease and its severity, some data expected at this level are
neither present nor addressed. Although the cohort is small, it does
include 3 ICU patients. What about their ratio of FSC-low vs FSC-hi
monocytes in comparison to other patients? Was this apparent early in
the disease course? Does this population of FSC-hi monocytes differ
between ICU patients and others in terms of frequency, phenotype or
cytokine secretion?

In general, figures need to revised to make the data clear. For example,
in Fig. 5, according to the legend it seems that patients with FSC-high
monocytes are discharged faster from the hospital. However according to
description in the text, patients were grouped in high or low levels of
FSC-low monocytes.

#### Significance

Despite the limitations of this study, the discovery of
a FSC-high monocyte population in COVID-19 patients is of great
interest. With similar implication, a the recent study by Zhou et al. [@doi:10.1101/2020.02.12.945576]
identified a connection between an inflammatory CD14+CD16+ monocyte
population and pulmonary immunopathology leading to deleterious clinical
manifestations and even acute mortality after SARS-CoV-2 infections.
Although the presence of these monocytes in the lungs has yet to be
demonstrated, such results support the importance of monocytes in the
critical inflammation observed in some COVID19 patients.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Correlation between universal BCG vaccination policy and reduced morbidity and mortality for COVID-19: an epidemiological study

Miller et al. *medRxiv*. [@doi:10.1101/2020.03.24.20042937]

#### Keywords

- BCG vaccine
- epidemiology
- vaccination policy

#### Main Findings

The authors compared middle and high income countries that never had a
universal BCG vaccination policy (Italy, Lebanon, Nederland, Belgium)
and countries with a current policy (low income countries were excluded
from the analysis as their number of cases and deaths might be
underreported for the moment). **Countries that never implement BCG
vaccination have a higher mortality rate than countries which have a BCG
vaccination policy** (16.38 deaths per million people vs 0.78). Next,
**the authors show that an earlier start of vaccination correlates with
a lower number of deaths per million inhabitants**. They interpret this
as the vaccine protecting a larger fraction of elderly people, which are
usually more affected by COVID-19. Moreover, higher number of COVID-19
**cases** were presented in countries that never implemented a universal
BCG vaccination policy.

#### Limitations

While this study aims to test an intriguing hypothesis unfortunately,
the data is not sufficient at this time to accurately make any
determinations. Several caveats must be noted including: not all
countries are in the same stage of the pandemic, the number of
cases/deaths is still changing very rapidly in a lot of countries and
thus the association may only reflect exposure to the virus. This
analysis would need to be re-evaluated when all the countries are passed
the pandemic and more accurate numbers are available. Additionally, very
few middle and high-income countries ever implemented universal BCG
vaccination, which can be a source of bias (5 countries, vs 55 that have
a BCG vaccine policy). Effective screening and social isolation policies
also varied considerable across the countries tested and may reflect
another important confounder. The authors could consider analyzing the
Case Fatality Rate (CFR, % of patients with COVID-19 that die), to more
correct for exposure although testing availability will still bias this
result. Variability in mortality within countries or cities with
variable vaccination and similar exposure could also be appropriate
although confounders will still be present.

#### Significance

BCG vaccine is a live attenuated strain derived from *Mycobacterium
bovis* and used for a vaccine for tuberculosis (TB). This vaccine has
been proven to be efficient in preventing childhood meningitis TB, but
doesn’t prevent adult TB as efficiently. For this reason, several
countries are now only recommending this vaccine for at-risk population
only.

**This study shows that there is a correlation between BCG vaccination
policy and reduced mortality for Covid-19**. Indeed, BCG vaccine has
been shown to protect against several viruses and enhance innate
immunity [@doi:10.1016/j.cmi.2019.04.020], which could explain why it could protect against SARS-CoV-2
infection, but the exact mechanism is still unknown. **Moreover, the
efficiency of adult/older people vaccination and protection against
Covid-19 still needs to be assessed.** Regarding this, Australian
researchers are starting a clinical trial of BCG vaccine for healthcare
workers [@url:https://www.mcri.edu.au/BRACE], to assess if it can protect them against Covid-19.

#### Credit

*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### Non-neural expression of SARS-CoV-2 entry genes in the olfactory epithelium

[@doi:10.1101/2020.03.25.009084]

#### Keywords

- CoVID-19
- 2019-nCoV
- SARS-CoV-2
- anosmia
- olfaction
- scRNAseq


#### Main Findings

-   Study analyzed bulk and scRNAseq data of olfactory cell types from
    publicly-available mouse, nonhuman primate and human datasets.

-   show that *ACE2* and *TMPRSS2* (genes involved in SARS-CoV-2 entry)
    are expressed in olfactory epithelial (OE) cells, basal stem cells
    and respiratory epithelium (RE), but not sensory neurons.

-   Comparison of human RE and OE datasets (Deprez et al. 2019; Durante
    et al. 2020) revealed that *ACE2* and *TMPRSS2* expression in OE
    sustentacular cells was similar to expression in the remainder of
    the non-nasal respiratory tract.

#### Limitations

-   Transcript data alone from healthy respiratory/olfactory cells is
    not sufficient to confirm infectivity of nasal passage, or to
    indicate damage to epithelia.

-   No mechanism defined for anosmia; it is not clear if epithelial
    injury leads to reduced sensitivity or increased inflammation and
    altered immune contexture drives neural/epithelial dysfunction. Will
    be critical to test this in CoVID-19 patient samples or mouse
    models.

#### Significance

-   Study provides possible rationale for anosmia observed in several
    CoVID-19 patients.

-   Raises possibility that nasal respiratory goblet, ciliated cells,
    and olfactory epithelia may serve as a viral reservoir after initial
    SARS-CoV-2 infection.

-   Human olfactory sensory neurons express several other molecules
    important to CoV (not CoV-19) entry such as *FURIN, ST6GAL1,
    ST3GAL4*; this suggests wider mechanism of neuronal infectivity in
    other coronaviruses.

#### Credit

*Review by Samarth Hegde as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.*


**Title: **

**SARS-CoV-2 proteome microarray for mapping COVID-19 antibody
interactions at amino acid resolution**

**Immunology keywords:** SARS-CoV-2, COVID-19, high throughput, peptide
microarray, antibody epitope screening

**The main finding of the article: **

This study screened the viral protein epitopes recognized by antibodies
in the serum of 10 COVID-19 patients using a new SARS-CoV-2 proteome
peptide microarray. The peptide library was constructed with 966 linear
peptides, each 15 amino acids long with a 5 amino acid overlap, based on
the protein sequences encoded by the genome of the Wuhan-Hu-1 strain.

To investigate crossreactivity between SARS-CoV-1 and SARS-CoV-2, they
tested rabbit monoclonal and polyclonal antibodies against SARS-CoV-1
nucleocapsid (N) in the microarray. Antibodies against SARS-CoV-1 N
displayed binding to the SARS-CoV-2 nucleocapsid (N) peptides.
Polyclonal antibodies showed some crossreactivity to other epitopes from
membrane (M), spike (S), ORF1ab and ORF8. This suggests that previous
exposure to SARS-CoV-1 may induced antibodies recognizing both viruses.

Screening of IgM and IgG antibodies from 10 COVID-19 patients showed
that many antibodies targeted peptides on M, N, S, Orf1ab, Orf3a, Orf7a,
and Orf8 from SARS-CoV-2, while immunodominant epitopes with antibodies
in more than 80 % COVID-19 patients were present in N, S and Orf3. It is
shown that the receptor binding domain (RBD) resides on S protein and
RBD is important for SARS-CoV-2 to enter the host cells via ACE2. Among
six epitopes on S protein, structural analysis predicted that three
epitopes were located at the surface and three epitopes were located
inside of the protein. Furthermore, some IgM antibodies from 1 patient
and IgG antibodies from 2 patients bound to the same epitope (residue
456-460, FRKSN) which resided within the RBD, and structural analysis
determined that this epitope was located in the region of the RBD loop
that engages with ACE2.

**Critical analysis of the study:**

In addition to the limitations mentioned in the manuscript, it would
have been informative to do the analysis over the course of the disease.
The pattern of antibody recognition, especially on S protein, and the
course of antibodies of different isotypes recognizing the same peptide
might correlate to the clinical course in these patients. It would alos
have been informative to analyze the presence of cross-reactive
antibodies from pateints previously exposed to SARS-CoV-1.

**The importance and implications for the current epidemics:**

This study identified linear immunodominant epitopes on SARS-CoV-2,
Wuhan-Hu-1 strain. This is a valuable information to design vaccines
that will elicit desirable immune responses.


**The Novel Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)
Directly Decimates Human Spleens and Lymph Nodes**

Review by Matthew D. Park

Revised by Miriam Merad

Keywords: COVID-19, SARS-CoV-2, spleen, lymph node, ACE2, macrophage

*Main findings*

It has been previously reported that COVID-19 patients exhibit severe
lymphocytopenia, but the mechanism through which this depletion occurs
has not been described. In order to characterize the cause and process
of lymphocyte depletion in COVID-19 patients, the authors performed
gross anatomical and *in situ* immune-histochemical analyses of spleens
and lymph nodes (hilar and subscapular) obtained from post-mortem
autopsies of 6 patients with confirmed positive viremia and 3 healthy
controls (deceased due to vehicle accidents).

Primary gross observations noted significant splenic and LN atrophy,
hemorrhaging, and necrosis with congestion of interstitial blood vessels
and large accumulation of mononuclear cells and massive lymphocyte
death. They found that CD68^+^ CD169^+^ cells in the spleens, hilar and
subscapular LN, and capillaries of these secondary lymphoid organs
expressed the ACE2 receptor and stain positive for the SARS-CoV-2
nucleoprotein (NP) antigen, while CD3^+^ T cells and B220^+^ B cells
lacked both the ACE2 receptor and SARS-CoV-2 NP antigen. ACE2^+^ NP^+^
CD169^+^ macrophages were positioned in the splenic marginal zone (MZ)
and in the marginal sinuses of LN, which suggests that these macrophages
were positioned to encounter invading pathogens first and may contribute
to virus dissemination.

Since SARS-CoV-2 does not directly infect lymphocytes, the authors
hypothesized that the NP^+^ CD169^+^ macrophages are responsible for
persistent activation of lymphocytes via Fas::FasL interactions that
would mediate activation-induced cell death (AICD). Indeed, the
expression of Fas was significantly higher in virus-infected tissue than
that of healthy controls, and TUNEL staining showed significant
lymphocytic apoptosis. Since pro-inflammatory cytokines like IL-6 and
TNF-α can also engage cellular apoptosis and necrosis, the authors
interrogated the cytokine expression of the secondary lymphoid organs
from COVID-19 patients; IL-6, not TNF-α, was elevated in virus-infected
splenic and lymph node tissues, compared to those of healthy controls,
and immunofluorescent staining showed that IL-6 is primarily produced by
the infected macrophages. *In vitro* infection of THP1 cells with
SARS-CoV-2 spike protein resulted in selectively increased *Il6*
expression, as opposed to *Il1b* and *Tnfa* transcription. Collectively,
the authors concluded that a combination of Fas up-regulation and IL-6
production by NP^+^ CD169^+^ macrophages induce AICD in lymphocytes in
secondary lymphoid organs, resulting in lymphocytopenia.

In summary, this study reports that CD169^+^ macrophages in the splenic
MZ, subscapular LN, and the lining capillaries of the secondary lymphoid
tissues express ACE2 and are susceptible to SARS-CoV-2 infection. The
findings point to the potential role of these macrophages in viral
dissemination, immunopathology of these secondary lymphoid organs,
hyperinflammation and lymphopenia.

*Limitations*

*Technical*

A notable technical limitation is the small number of samples (n=6);
moreover, the analysis of these samples using multiplexed
immunohistochemistry and immunofluorescence do not necessarily provide
the depth of unbiased interrogation needed to better identify the cell
types involved.

*Biological*

The available literature and ongoing unpublished studies, including
single-cell experiments of spleen and LN from organ donors, do not
indicate that ACE2 is expressed by macrophages; however, it remains
possible that ACE2 expression may be triggered by type I IFN in COVID-19
patients. Importantly, the SARS-CoV-2 NP staining of the macrophages
does not necessarily reflect direct infection of these macrophages;
instead, positive staining only indicates that these macrophages carry
SARS-CoV-2 NP as antigen cargo, which may have been phagocytosed. Direct
viral culture of macrophages isolated from the secondary lymphoid organs
with SARS-CoV-2 is required to confirm the potential for direct
infection of macrophages by SARS-CoV-2. Additionally, it is important to
note that the low to negligible viremia reported in COVID-19 patients
to-date does not favor a dissemination route via the blood, as suggested
by this study, which would be necessary to explain the presence of
virally infected cells in the spleen.

*Relevance*

Excess inflammation in response to SARS-CoV-2 infection is characterized
by cytokine storm in many COVID-19 patients. The contribution of this
pathology to the overall fatality rate due to COVID-19, not even
necessarily directly due to SARS-CoV-2 infection, is significant. A
better understanding of the full effect and source of some of these
major cytokines, like IL-6, as well as the deficient immune responses,
like lymphocytopenia, is urgently needed. In this study, the authors
report severe tissue damage in spleens and lymph nodes of COVID-19
patients and identify the role that CD169^+^ macrophages may play in the
hyperinflammation and lymphocytopenia that are both characteristic of
the disease. It may, therefore, be important to note the effects that
IL-6 inhibitors like Tocilizumab and Sarilumab may specifically have on
splenic and LN function. It is important to note that similar
observations of severe splenic and LN necrosis and inflammation in
patients infected with SARS-CoV-1 further support the potential
importance and relevance of this study.


### Cigarette smoke triggers the expansion of a subpopulation of respiratory epithelial cells that express the SARS-CoV-2 receptor ACE2

[@doi:10.1101/2020.03.28.013672]

#### Keywords

- CoVID-19
- 2019-nCoV
- SARS-CoV-2
- respiration
- cigarette
- ACE2
- lung

#### Main Findings

-   Study uses scRNAseq, bulk seq data and air-liquid interface culture
    experiments to show that cigarette smoke causes a dose-dependent
    upregulation of ACE2 in mouse and human lungs (transplantation,
    tumor resection, or IPF datasets).

-   ACE2 was not up-regulated in patients with asthma or
    lung-sarcoidosis or in mouse models of cystic fibrosis or carcinogen
    exposure.

-   Cathepsin B (alternate protease involved in viral entry) is
    increased in smoke-exposed mouse or human lungs.

-   Smoke triggers a protective expansion of mucus-secreting MUC5AC+
    goblet and SCGB1A1+ club cells; ACE2 presence in these cells is
    increased upon smoke exposure.

#### Limitations:

-   Long-term smokers usually have several co-morbidities including
    immune dysfunction, which can affect interpretation of CoV-2
    susceptibility in these datasets. Ideally, analyses can control for
    major co-morbidities across smokers and non-smokers (immune
    suppression, cardiovascular disease and atherosclerosis).

-   Hyperplasia of ACE2+ goblet cells upon smoking needs to be separated
    from ACE2 upregulation in existing goblet cells.

-   ACE2 expression increase alone does not confirm increased viral
    entry into goblet cells; future studies with air-liquid interface
    cultures testing CoV-2 infectivity in *ex vivo* epithelial cells
    from human epithelial lines, ex vivo samples or hACE2 mice will be
    very informative.

#### Significance

-   This study may partially explain why smokers are more likely to
    develop severe SARS-CoV-2 infections. Also, the reversibility of
    ACE2 expression upon smoking cessation suggests that quitting
    smoking could lessen CoV-2 susceptibility.

-   Absence of ACE2 upregulation in other lung inflammation pathologies
    implies CoV-2 susceptibility might be smoking-specific, and not
    fibrosis-specific.

-   Another preprint showed ACE2 expression increases in lung of
    patients with CoV-2 co-morbidities such as hypertension [@doi:10.1101/2020.03.21.20040261]; these studies
    collectively paint a better picture of CoV-2 susceptibility before
    actual experiments can be carried out.

#### Credit

*Review by Samarth Hegde as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*


### The comparative superiority of IgM-IgG antibody test to real-time reverse transcriptase PCR detection for SARS-CoV-2 infection diagnosis

Liu et al. *medRxiv*.
[@doi:10.1101/2020.03.28.20045765]

#### Keywords

- IgM/IgG antibody test
- nucleic acid test
- SARS-CoV-2 detection

#### Main Findings

The study compares IgM and IgG antibody testing to RT-PCR detection of
SARS-CoV-2 infection. 133 patients diagnosed with SARS-CoV-2 in Renmin
Hospital (Wuhan University, China) were analyzed. The positive ratio was
78.95% (105/133) in IgM antibody test (SARS-CoV-2 antibody detection kit
from YHLO Biotech) and 68.42% (91/133) in RT-PCR (SARS-CoV-2 ORF1ab/N
qPCR detection kit). There were no differences in the sensitivity of
SARS-CO-V2 diagnosis in patients grouped according to disease severity.
For example, IgG responses were detected in 93.18% of moderate cases,
100% of severe cases and 97.3% of critical cases. In sum, positive
ratios were higher in antibody testing compared to RT-PCR detection,
demonstrating a higher detection sensitivity of IgM-IgG testing for
patients hospitalized with COVID-19 symptoms.

#### Limitations

This analysis only included one-time point of 133 hospitalized patients,
and the time from symptom onset was not described. There was no
discussion about specificity of the tests and no healthy controls were
included. It would be important to perform similar studies with more
patients, including younger age groups and patients with mild symptoms
as well as asymptomatic individuals. It is critical to determine how
early after infection/symptom onset antibodies can be detected and the
duration of this immune response.

#### Significance

The IgM-IgG combined testing is important to improve clinical
sensitivity and diagnose COVID-19 patients. The combined antibody test
shows higher sensitivity than individual IgM and IgG tests or nucleic
acid-based methods, at least in patients hospitalized with symptoms.

#### Credit

*Review by Erica Dalla as part of a project by students, postdocs and
faculty at the Immunology Institute of the Icahn School of Medicine,
Mount Sinai.*


**Title:** Lectin-like Intestinal Defensin Inhibits 2019-nCoV Spike
binding to ACE2

**Immunology keywords:** defensins, spike protein, intestinal Paneth
cells, ACE2 binding

**Main Findings:**

Human ACE2 was previously identified as the host receptor for
SARS-CoV-2. Despite ACE2 being expressed in both lung alveolar
epithelial cells and small intestine enterocytes, respiratory problems
are the most common symptom after viral infection while intestinal
symptoms are much less frequent. Thus, the authors here investigate the
biology behind the observed protection of the intestinal epithelium from
SARS-CoV-2. Human defensin 5 (HD5), produced by Paneth cells in the
small intestine, was shown to interact with human ACE2, with a binding
affinity of 39.3 nM by biolayer interferometry (BLI). A blocking
experiment using different doses of HD5 coating ACE2 showed that HD5
lowered viral spike protein S1 binding to ACE2. Further, a molecular
dynamic simulation demonstrated a strong intermolecular interaction
between HD5 and the ACE2 ligand binding domain. To test HD5 inhibitory
effect on S1 binding to ACE2, human intestinal epithelium Caco-2 cells
were preincubated with HD5. Preincubation strongly reduced adherence of
S1 to surface of cells. HD5 was effective at a concentration as low as
10 µg/mL, comparable to the concentration found in the intestinal fluid.

**Limitations:**

The study focuses exclusively on intestinal cells. However, HD5 could
have been tested to block ACE2-S1 binding in human lung epithelial cells
as a potential treatment strategy. It would be useful to know whether
HD5 could also prevent viral entry in lung cells.

**Relevance:**

This work provides the first understanding of the different efficiency
of viral entry and infection among ACE2-expressing cells and tissues.
Specifically, the authors show that human defensin 5 produced in the
small intestine is able to block binding between S1 and ACE2 necessary
for viral entry into cells. The study provides a plausible explanation
on why few patients show intestinal symptoms and suggests that patients
with intestinal disease that decrease defensins’ production may be more
susceptible to SARS-CoV-2. It also indicates that HD5 could be used as a
molecule to be exogenously administered to patients to prevent viral
infection in lung epithelial cells.


**Title: **

**Susceptibility of ferrets, cats, dogs and different domestic animals
to SARS-coronavirus-2**

**Immunology keywords:** SARS-CoV-2, ferret, cat, laboratory animal,
domestic animals

**The main finding of the article: **

This study evaluated the susceptibility of different model laboratory
animals (ferrets), as well as companion (cats and dogs), and domestic
animals (pigs, chickens and ducks) to SARS-CoV-2. They tested infection
with two SARS-CoV2 isolates, one from an environmental sample collected
in the Huanan Seafood Market in Wuhan (F13-E) and the other from a human
patient in Wuhan (CTan-H).

*Ferrets* were inoculated with either of the two viruses by intranasal
route with 10^5^ pfu, and the viral replication was evaluated. Two
ferrets from each group were euthanized on day 4 post infection (p.i.).
AT day 4 p.i., viral RNA and infectious viruses were detected only in
upper respiratory tract (nasal turbinate, upper palate, tonisls, but not
in the trachea, lungs or other tissues. Viral RNA and virus titer in the
remaining ferrets were monitored in nasal washes and rectal swabs on
days 2, 4, 6, 8 and 10 p.i. Viral RNA and infectious viruses were
detected in nasal washes until day 8 p.i. One ferret in each group
developed fever and loss of appetite on days 10 and 12 p.i., however,
viral RNA was practically undetactable. These two ferrets showed severe
lymphoplasmacytic perivasculitis and vasculitis in the lungs and lower
antibody titers compare to other 4 ferrets.

*Cats*. Five *s*ubadult 8-month-old domestic cats were inoculated with
CTan-h virus and three uninfected cats were placed in a cage adjacent to
each of the infected cats to monitor respiratory droplet transmission.
Viral RNA was detected in the upper respiratory organs from all infected
cats and in one out of three exposed cats. All infected (inoculated and
exposed) cats developed elevated antibodies against SARS-CoV2. Viral
replication studies with juvenile cats (70-100 days) revealed massive
lesions in the nasal and tracheal mucosa epithelium and lungs of two
inoculated cats which died or were euthanized on day 3 p.i., and
infection in one out of three exposed cats. These results indicated
SARS-CoV2 could replicate in cats, that juvenile cats were more
susceptible that adults, and theat SARS-CoV2 could be transmit via
respiratory droplets between cats.

*Dogs and others.* Five 3-month-old beagle dogs were inoculated and
housed with two uninoculated beagles in a room. Two virus inoculated
dogs seroconverted, but others including two contact dogs were all
seronegative for SARS-CoV2 and infectious virus was not detected in any
swabs collected. Viral RNA was not detected in swabs from pigs,
chickens, and ducks inoculated or contacted. These results indicated
that dogs, pigs, chickens, and ducks might have low or no susceptibility
to SARS-CoV2.

**Critical analysis of the study:**

This manuscript describes the viral replication and clinical symptoms of
SARS-CoV2 infection in ferrets, and the SARS-CoV2 infection and
transmission in cats. Clinical and pathological analysis was not
performed in cats, therefore the correlation of virus titer with
symptoms severity in the adult and juvenile cats could not be
determined.

**The importance and implications for the current epidemics:**

SARS-CoV-2 transmission to tigers, cats and dogs has been previously
reported. It should be noted that this manuscript did not evaluate the
transmission from cats to human. Nevertheless, it clearly showed higher
susceptibility of ferrets and domestic cats to SARS-CoV-2. This data
strongly indicates the need for surveillance of possible infection and
transmission of SARS-CoV-2 by domestic cats.


### Virus-host interactome and proteomic survey of PMBCs from COVID-19 patients reveal potential virulence factors influencing SARS-CoV-2 pathogenesis

Li et al. *bioRxiv*.
[@doi:10.1101/2020.03.31.019216]

#### Keywords
- PBMC
- virulence factors
– interaction network
– nsp9
- nsp10
– NKRF

#### Main findings

The authors identified **intra-viral protein-protein interactions**
(PPI) with two different approaches: genome wide yeast-two hybrid (Y2H)
and co-immunoprecipitation (co-IP). A total of 58 distinct PPI were
characterized. A screen of **viral-host PPI** was also established by
overexpressing all the SARS-CoV-2 genes with a Flag epitope into HEK293
cells and purifying each protein complex. Interacting host proteins were
then identified by liquid chromatography and tandem mass spectrometry.
251 cellular proteins were identified, such as subunits of ATPase, 40S
ribosomal proteins, T complex proteins and proteasome related proteins,
for a total of 631 viral-host PPI. Several interactions suggesting
protein-mediated modulation of the immune response were identified,
highlighting the multiple ways SARS-CoV-2 might reprogram infected
cells.

Subsequently, the authors compared global proteome profiles of PBMCs
from healthy donors (n=6) with PBMC from COVID-19 patients with mild
(n=22) or severe (n=13) symptoms. 220 proteins were found to be
differentially expressed between *healthy donors and mild COVID-19
patients*, and a pathway analysis showed **a general activation of the
innate immune response.** 553 proteins were differentially expressed
between the PBMC of *mild and severe COVID-19 patients*, most of them
(95%) being downregulated in severe patients*.* Functional pathway
analysis indicated a defect of T cell activation and function in severe
COVID-19. There was also evidence suggesting reduced antibody secretion
by B cells. Together, these results suggest a **functional decline of
adaptive immunity**. A FACS analysis of PBMC from severe patients
indicated higher levels of IL6 and IL8 but not IL17 compared to mild
patients.

Finally, the authors focused on NKRF, an endogenous repressor of IL8/IL6
synthesis that was previously identified as interacting with SARS-Cov-2
nsp9,10,12,13 and 15. Individually expressed nsp9 and nsp10 (but not
nsp12, nsp13, nsp15) induced both IL6 and IL8 in lung epithelial A459
cells, indicating that nsp9 and nsp10 may be directly involved in the
induction of these pro-inflammatory cytokines. The authors finally argue
that nsp9 and nsp10 represent potential drug targets to prevent
over-production of IL6 and IL8 in infected cells, and reducing the
over-activation of neutrophils.

#### Limitations

First, the authors seem to have forgotten to include the extended data
in the manuscript, and their proteomic data does not seem to be publicly
available for the moment, which limits greatly our analysis of their
results.

While this work provides important data on host and viral PPI, only 19
interactions were identified by Y2H system but 52 with co-IP. The
authors do not comment about what could lead to such differences between
the two techniques and they don’t specify whether they detected the same
interactions using the two techniques.

Moreover, the PBMC protein quantification was performed comparing bulk
PBMC. Consequently, protein differences likely reflect differences in
cell populations rather than cell-intrinsic differences in protein
expression. While this analysis is still interesting, a similar
experiment performed on pre-sorted specific cell populations would allow
measuring proteome dynamics at a higher resolution.

Finally, the authors did not discussed their results in regards to
another SARS-CoV-2 interactome of host-viral PPI that had been published
previously^1^. This study reported 332 host-virus PPI, but no
interaction of viral proteins with NKRF was found. Some interactions
were found in both studies (eg. N and G3BP1, Orf6 and RAE1). However,
the time point used to lyse the cells were different (40h previously vs
72h here), which could explain some of the differences.

#### Relevance

The identification of many interactions between intra-viral and
host-virus PPI provides an overview of host protein and pathways that
are modulated by SARS-CoV-2, which can lead to the identification of
potential targets for drug development.

In the model proposed by the authors, nsp9 and nsp10 from SARS-Cov-2
induce an over-expression of IL6 and IL8 by lung epithelial cells, which
recruits neutrophils and could lead to an excess in lung infiltration.
Nsp9 has been shown to be essential for viral replication for
SARS-Cov-1^2^, and shares a 97% homology with nsp9 from SARS-Cov-2^3^.
Further, nsp9 crystal structure was recently solved^3^, which can help
to develop drug inhibitors if this protein is further confirmed as being
important for the virulence of SARS-Cov-2.

1\. Gordon DE, Jang GM, Bouhaddou M, et al. A SARS-CoV-2-Human
Protein-Protein Interaction Map Reveals Drug Targets and Potential
Drug-Repurposing. *bioRxiv*. March 2020:2020.03.22.002386.
doi:10.1101/2020.03.22.002386

2\. Miknis ZJ, Donaldson EF, Umland TC, Rimmer RA, Baric RS, Schultz LW.
Severe acute respiratory syndrome coronavirus nsp9 dimerization is
essential for efficient viral growth. *J Virol*. 2009;83(7):3007-3018.
doi:10.1128/JVI.01505-08

3\. Littler DR, Gully BS, Colson RN, Rossjohn J. *Crystal Structure of
the SARS-CoV-2 Non-Structural Protein 9, Nsp9*. Molecular Biology; 2020.
doi:10.1101/2020.03.28.013920


***Title:*** Prediction and Evolution of B Cell Epitopes of Surface
Protein in SARS-CoV-2

***Keywords:*** SARS-CoV-2; Epitopes; Bioinformatics; Evolution

***Summary/Main findings: ***

Lon et al. used a bioinformatic analysis of the published SARS-CoV-2
genomes in order to identify conserved linear and conformational B cell
epitopes found on the spike (S), envelope (E), and membrane (M)
proteins. The characterization of the surface proteins in this study
began with an assessment of the peptide sequences in order to identify
hydrophilicity indices and protein instability indices using the
Port-Param tool in ExPASy. All three surface proteins were calculated to
have an instability score under 40 indicating that they were stable.
Linear epitopes were identified on the basis of surface probability and
antigenicity, excluding regions of glycosylation. Using BepiPred 2.0
(with a cutoff value of 0.35) and ABCpred (with a cutoff value of 0.51),
4 linear B cell epitopes were predicted for the S protein, 1 epitope for
the E protein, and 1 epitope for the M protein. For structural analysis,
SARS-CoV assemblies published in the Protein Data Bank (PDB) acting as
scaffolds for the SARS-CoV-2 S and E amino acid sequences were used for
input into the SWISS-MODEL server in order to generate three-dimensional
structural models for the assessment of conformational epitopes. Using
Ellipro (cutoff value of 0.063) and SEPPA (cutoff value of 0.5), 1
conformational epitope was identified for the S protein and 1 epitope
was identified for the E protein, both of which are accessible on the
surface of the virus. Finally, the Consurf Server was used to assess the
conservation of these epitopes. All epitopes were conserved across the
published SARS-CoV-2 genomes and one epitope of the spike protein was
predicted to be the most stable across coronavirus phylogeny.

***Critical Analysis/Limitations:***

While this study provides a preliminary identification of potential
linear and conformational B cell epitopes, the translational value of
the epitopes described still needs extensive experimental validation to
ascertain whether these elicit a humoral immune response. The
conformational epitope analyses are also limited by the fact that they
are based off of predicted 3D structure from homology comparisons and
not direct crystal structures of the proteins themselves. Additionally,
since there was not a published M protein with a high homology to
SARS-CoV-2, no conformational epitopes were assessed for this protein.
Finally, while evolutionary conservation is an important consideration
in understanding the biology of the virus, conservation does not
necessarily imply that these sites neutralize the virus or aid in
non-neutralizing *in vivo* protection.

***Relevance/Implications:***

With further experimental validation that confirms that these epitopes
induce effective antibody responses to the virus, the epitopes described
can be used for the development of treatments and vaccines as well as
better characterize the viral structure to more deeply understand
pathogenesis.




## Appendix B {.page_break_before}
Contributors were asked to complete this template to summarize and evaluate new papers related to diagnostics.


<!--Hi there! Please use the template below as a guide for what information about this paper to include. It's ok to leave fields blank so that other contributors can fill them in later, or to add fields that you think are important.-->

Title: Please edit the title to add the name of the paper after the colon

Please paste a link to the paper or a citation here:

Link:

What is the paper's [Manubot-style citation](https://github.com/greenelab/covid19-review/blob/master/USAGE.md#citations)?
<!--Leave the citation blank if you are unsure.-->

Citation:

Please list some keywords (3-10) that help identify the relevance of this paper to COVID-19

* keyword 1 (replace me, copy and paste more than three if needed)
* keyword 2 (replace me, copy and paste more than three if needed)
* keyword 3 (replace me, copy and paste more than three if needed)

Please note the publication / review status
<!--Replace one of the empty brackets below with [x].-->

- [ ] Pre-print
- [ ] New Peer-Reviewed Paper
- [ ] Peer-Reviewed Paper Pre-2020

Which areas of expertise are particularly relevant to the paper?
<!--Replace the empty brackets below with [x].-->

- [ ] virology
- [ ] epidemiology
- [ ] biostatistics
- [ ] immunology
- [ ] pharmacology

<!-- The following questions are the things that we'll want to answer about each paper reporting a diagnostic. If you are ready to answer some of them now, please create the issue, then answer them in a comment. Otherwise, just create the issue so that a future reviewer can address them in the comments.-->

Questions to answer about each paper:

Please provide 1-2 sentences introducing the study and its main findings



Study question(s) being investigated:

What type of testing scenario is being considered?

Is it a screening test (used for individuals with no symptoms), diagnostic test (used for individuals with symptoms), or definitive test (used for individuals who have had previous positive test results on diagnostic or screening tests)?

Study population:

What is the model system (e.g., human study, animal model, cell line study)?

What is the sample size?

What is the "pre-test" probability of disease in the study population (i.e., what is the anticipated prevalence of the disease?)

For human studies, the following are related to the pre-test probability:

What countries/regions are considered?

What is the age range, gender, other relevant characteristics?

What is the setting of the study (e.g., random sample of school children, retirement communities, etc.)?

What other specific inclusion-exclusion criteria are considered?

Reference test:

What reference test is considered as a "gold standard" comparator for the test under investigation?

Test assignment:

How are the new and reference tests assigned?

Examples of assignment could include: Recruited individuals have initially undergone neither the new nor the reference test; individuals tested as positive or negative by the reference test undergo the new test; individuals who have undertaken the new test are assessed by the standard test.

Are there any other relevant details about the study design?

Depending on how individuals are chosen, the test may be biasing towards more sick or less sick individuals or very clear-cut positive/negative cases.
Any factors that would influence this bias should be included here.

Test conduct:

How were tests performed?

Describe technical details of assays used, when measurements were taken and by whom, etc. for both the new and standard tests.

Test Assessment

Describe how individuals are classified as positive or negative, e.g. if a threshold is used.

Is there evidence that the test is precise/reproducible when repeated more than once?

Are measurements complete?

For example: Do some participants undergo just one test (the new or the reference test)?
Are there individuals with inconclusive results?

Results summary:

What are the estimated sensitivity, specificity, positive predictive value (PPV), and negative predicted value (NPV)?

Note that the PPV and NPV represent "post-test" probabilities of disease and are generally more meaningful than sensitivity and specificity.
Sometimes the post-test odds will be given instead.

What are the confidence bounds around these intervals?

Interpretation of results for study population:

How good is the test at ruling in or ruling out a disease based on the post-test probabilities?

Are there identified side affects of the test?

Is patient adherence to the test likely to be an issue?

Extrapolation of conclusions to other groups of individuals

How well is the test likely to work in populations with different pretest odds?

For example, if the prevalence is lower, then the PPV will also be lower, but the NPV will be higher.

How costly is the test?

How difficult is it to perform the test in different settings?

Could the test be combined with other existing tests?

Summary of reliability

1-2 sentences on concluding remarks, including summary of strengths, weaknesses, limitations.

Progress

_Check off the components as they are completed. If the component is not applicable, check the box as well._

<!--Check off the components that you have completed. It is fine if you have not completed all of them, we hope others will add on in the comments. Switch to "Preview" mode to click the box(es), or else replace the empty brackets below with [x].-->

- [ ] 1-2 sentences introducing the study and its main findings
- [ ] Describe testing scenario 
- [ ] Describe model system 
- [ ] Sample size 
- [ ] Describe prevalnce of disease 
- [ ] Describe countries/regions are considered 
- [ ] Describe age range, gender, other relevant characteristics 
- [ ] Describe setting of the study 
- [ ] Describe other specific inclusion-exclusion criteria 
- [ ] Describe "gold standard" 
- [ ] Describe how  the new and reference tests assigned 
- [ ] Describe other relevant details about the study design 
- [ ] Describe how the tests were performed 
- [ ] Describe how individuals are classified as positive or negative 
- [ ] Describe if test is precise/reproducible 
- [ ] Describe whether measurements are complete 
- [ ] What are the estimated sensitivity, specificity, positive predictive value (PPV), and negative predicted value (NPV)? 
- [ ] What are the confidence bounds around these intervals? 
- [ ] Describe post-test probabilities 
- [ ] Describe side affects of the test 
- [ ] Describe patient adherence 
- [ ] Describe how it will extrapolate 
- [ ] How costly is the test? 
- [ ] How difficult is it to perform the test in different settings? 
- [ ] Could the test be combined with other existing tests?
- [ ] Summary of reliability

## Appendix C {.page_break_before}
Contributors were asked to complete this template to summarize and evaluate new papers related to therapeutics.


<!--Hi there! Please use the template below as a guide for what information about this paper to include. It's ok to leave fields blank so that other contributors can fill them in later, or to add fields that you think are important.-->

Title: Please edit the title to add the name of the paper after the colon

Please paste a link to the paper or a citation here:

Link:

What is the paper's [Manubot-style citation](https://github.com/greenelab/covid19-review/blob/master/USAGE.md#citations)?
<!--Leave the citation blank if you are unsure.-->

Citation:

Please list some keywords (3-10) that help identify the relevance of this paper to COVID-19

* keyword 1 (replace me, copy and paste more than three if needed)
* keyword 2 (replace me, copy and paste more than three if needed)
* keyword 3 (replace me, copy and paste more than three if needed)

Please note the publication / review status
<!--Replace one of the empty brackets below with [x].-->

- [ ] Pre-print
- [ ] New Peer-Reviewed Paper
- [ ] Peer-Reviewed Paper Pre-2020

Which areas of expertise are particularly relevant to the paper? 
<!--Replace the empty brackets below with [x].-->

- [ ] virology
- [ ] epidemiology
- [ ] biostatistics
- [ ] immunology
- [ ] pharmacology

<!-- The following questions are the things that we'll want to answer about each paper reporting a diagnostic. If you are ready to answer some of them now, please create the issue, then answer them in a comment. Otherwise, just create the issue so that a future reviewer can address them in the comments.-->

Questions to answer about each paper:

Please provide 1-2 sentences introducing the study and its main findings


Study question(s) being investigated:

How many/what drugs/combinations are being considered?


What are the main hypotheses being tested?


Study population:

What is the model system (e.g., human study, animal model, cell line study)?


What is the sample size? If multiple groups are considered, give sample size for each group (including controls).

* number treated with treatment A
* number treated with treatment B

For human studies:

What countries/regions are considered?

What is the age range, gender, other relevant characteristics?

What is the setting of the study (random sample of school children, inpatient, outpatient, etc)?

What other specific inclusion-exclusion criteria are considered?

For example, do the investigators exclude patients with diagnosed neoplasms or patients over/under a certain age?


Treatment assignment:

How are treatments assigned?

For example, is it an interventional or an observational study?

Is the study randomized?

A study can be interventional but not randomized (e.g., a phase I or II clinical trial is interventional but often not randomized).

Provide other relevant details about the design.

This includes possible treatment stratification (e.g., within litters for animal studies, within hospitals for human studies), possible confounding variables (e.g., having a large age range of individuals), possible risks of bias and how they are addressed (e.g., is there masking in a clinical trial? how are individuals chosen in an observational study?).


Outcome Assessment:

Describe the outcome that is assessed and whether it is appropriate.

For example: Is the outcome assessed by a clinician or is it self-reported?
Is the outcome based on viral load or a functional measurement (e.g., respiratory function, discharge from hospital)?
What method is used to measure the outcome?
How long after a treatment is the outcome measured?

Are outcome measurements complete?

For example, are there individuals lost to follow up?

Are outcome measurements subject to various kinds of bias?

For example, a lack of masking in randomized clinical trials.


Statistical Methods Assessment:

What methods are used for inference?

For example, logistic regression, nonparametric methods.

Are the methods appropriate for the study?

For example, are clustered data treated independently or are clusters adjusted for, such as different hospitals or litters?

Are adjustments made for possible confounders?

For example, adjustment for age, sex, or comorbidities.


Results Summary:

What is the estimated association?

For example, is it an estimated odds ratio, a median difference in detected cases, etc?

What measures of confidence or statistical significance are provided?

For example, confidence intervals, p-values, and/or Bayes factors.


Interpretation of results for study population:

Can we make a causal interpretation for the individuals in the study of drug -> outcome, such as "taking drug A improves likelihood of survival twofold over taking drug B."

For example, with a well-performed animal study or randomized trial it is often possible to infer causality.
If is an observational study, does it match up with some of the Bradford Hill criteria? https://www.edwardtufte.com/tufte/hill https://en.wikipedia.org/wiki/Bradford_Hill_criteria

Are there identified side effects or interactions with other drugs?

For example, is the treatment known to cause liver damage or to not be prescribed for individuals with certain comorbities?

Are there specific subgroups with different findings?

For example, do individuals with a specific baseline seem to do particularly well? Be particularly cautious with respect to multiple testing here.


Extrapolation of conclusions to other groups of individuals not specifically included in the study:

If the study is an animal study, which animal and how relevant is that model?

Is the model system appropriate? Is there evidence from past use that it's highly-relevant to therapeutic design in this context?

If it is a human study, what characteristics of the study population may support/limit extrapolation?

* Can results extrapolate easily to other similar groups? (e.g., same country, similar age groups)
* What would happen if conditions are extended in terms of dose or duration?
* Can results be extrapolated to other populations or in very different settings? (e.g., different age group, primary care setting vs emergency department etc)


Summary of reliability

1-2 sentences on concluding remarks, including summary of strengths, weaknesses, limitations.

Progress

_Check off the components as they are completed. If the component is not applicable, check the box as well._

<!--Check off the components that you have completed. It is fine if you have not completed all of them, we hope others will add on in the comments. Switch to "Preview" mode to click the box(es), or else replace the empty brackets below with [x].-->

- [ ] 1-2 sentences introducing the study and its main findings
- [ ] Describe How many/what drugs/combinations are being considered 
- [ ] Describe the model system
- [ ] What is the sample size? 
- [ ] What countries/regions are considered 
- [ ] What is the age range, gender, other relevant characteristics 
- [ ] Describe study setting 
- [ ] Describe  other specific inclusion-exclusion criteria 
- [ ] Describe how treatments are assigned
- [ ] Describe randomization (or not) and other relavent details about the design
- [ ] Describe the outcome that is assessed and whether it is appropriate.
- [ ] Describe whether the outcome measurements are complete
- [ ] Are outcome measurements subject to various kinds of bias?
- [ ] Describe methods used for inference
- [ ] Describe whether the methods are appropriate for the study
- [ ] Are adjustments made for possible confounders?
- [ ] Describe the estimated association
- [ ] What measures of confidence or statistical significance are provided?
- [ ] Describe whether a causal interpretation can be made
- [ ] Are there identified side effects or interactions with other drugs?
- [ ] Are there specific subgroups with different findings?
- [ ] If the study is an animal study, which animal and how relevant is that model?
- [ ] If it is a human study, what characteristics of the study population may support/limit extrapolation?
- [ ] Summary of reliability

## Appendix D {.page_break_before}
Contributors were asked to complete this template to summarize and evaluate new papers related to topics besides therapeutics and diagnostics.


<!--Hi there! Please use the template below as a guide for what information about this paper to include. It's ok to leave fields blank so that other contributors can fill them in later, or to add fields that you think are important.-->

Title: Please edit the title to add the name of the paper after the colon.

General Information
Please paste a link to the paper or a citation here:

Link:

What is the paper's [Manubot-style citation](https://github.com/greenelab/covid19-review/blob/master/USAGE.md#citations)?
<!--Leave the citation blank if you are unsure.-->

Citation:

Is this paper primarily relevant to Background or Pathogenesis? 
<!--Replace the empty brackets below with [x].
If it's primarily relevant to Diagnostics or Therapeutics, please go back and use the associated template-->

- [ ] Background
- [ ] Pathogenesis
- [ ] Methods

Please list some keywords (3-10) that help identify the relevance of this paper to COVID-19

* keyword 1 (replace me, copy and paste more than three if needed)
* keyword 2 (replace me, copy and paste more than three if needed)
* keyword 3 (replace me, copy and paste more than three if needed)

Please note the publication / review status
<!--Replace one of the empty brackets below with [x].-->

- [ ] Pre-print
- [ ] New Peer-Reviewed Paper
- [ ] Peer-Reviewed Paper Pre-2020

Which areas of expertise are particularly relevant to the paper?
<!--Replace the empty brackets below with [x].-->

- [ ] virology
- [ ] epidemiology
- [ ] biostatistics
- [ ] immunology
- [ ] pharmacology
- [ ] other:

<!-- STOP
Please fill out as much of the above as you feel able, then:
   (1) Select all the text below and cut it onto your clipboard
   (2) Submit this issue using the green button below
   (3) If you would like to submit a summary of the paper, paste the cut text from your clipboard to a new comment below and fill in the summary information
-->
Summary

Suggested questions to answer about each paper:
- What did they analyze?
- What methods did they use?
- Does this paper study COVID-19, SARS-CoV-2, or a related disease and/or virus?
- What is the main finding (or a few main takeaways)?
- What does this paper tell us about the background and/or diagnostics/therapeutics for COVID-19 / SARS-CoV-2?
- Do you have any concerns about methodology or the interpretation of these results beyond this analysis?

Any comments or notes?
