## Introduction {.page_break_before}

On January 21, 2020, the World Health Organization (WHO) released its first report concerning what is now known as the Coronavirus Disease 2019 (COVID-19) [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf].
This infectious disease came to international attention on December 31, 2019 following an announcement by national officials in China describing 44 cases of a respiratory infection of unknown cause.
The first known cases were located in Wuhan City within the Hubei province of China, but the disease spread rapidly throughout China and subsequently around the world.
At the time of the WHO's first situation report [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf], 282 confirmed cases had been identified.
Most of these cases were in China, but 1-2 exported cases had also been identified in each of several neighboring countries (Thailand, Japan, and the Republic of Korea).
One week later, 4,593 confirmed cases had been identified, spanning not only Asia, but also Australia, North America, and Europe [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200128-sitrep-8-ncov-cleared.pdf].
On March 11, 2020, WHO formally classified the situation as a pandemic [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200311-sitrep-51-covid-19.pdf].
On April 4, 2020, the WHO reported that the global number of confirmed cases had surpassed one million [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200404-sitrep-75-covid-19.pdf].

**Global COVID-19 deaths since January 22, 2020.**
{{csse_deaths}} COVID-19 deaths had been reported worldwide as of {{csse_date_pretty}} (Figure @fig:csse-deaths).

![
**Cumulative global COVID-19 deaths since January 22, 2020.**
Data are from the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University [@https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series].
]({{csse_deaths_figure}} "Global COVID-19 deaths"){#fig:csse-deaths secno=1}

As international attention remains focused on the ongoing public health crisis, the scientific community has responded by mobilizing resources and turning much of its attention to the virus and disease.
This rapid influx of information is disseminated in both traditional publishing mechanisms and through preprint servers, which provide a venue for scientists to release findings without undergoing the formal publication process.
While having information available is valuable to efforts to understand and combat COVID-19, many contributions come from researchers across a wide range of fields who have varying degrees of experience working on coronaviruses and other highly relevant topics.
The volume of information available, much of which has not gone through rigorous peer review, presents a significant challenge to individual efforts to keep abreast of the state of COVID-19 research [@doi:10.1038/s42254-020-0175-7].
However, research on these topics is proceeding so rapidly that any static review is likely to quickly become dated.
Our goal as a community was to consolidate information about the virus in the context of related viruses and to synthesize rapidly emerging literature centered on the diagnosis and treatment of COVID-19.
We used an open publishing framework, Manubot [@doi:10.1371/journal.pcbi.1007128], to manage hundreds of contributions from the community to create a living, scholarly document.
We designed software to generate figures like Figure @fig:csse-deaths that automatically update using external data sources.
Our primary goal is to sort and distill informative content out of the overwhelming flood of information [@doi:10.1038/s42254-020-0175-7] and help the broader scientific community become more conversant on this critical subject.
Thus, our approach has been to develop a real-time, collaborative effort that welcomes submissions from scientists worldwide into this ongoing effort.
This document represents the first snapshot, which aims to reflect the state of the field as of August, 2020.
We plan to refine and expand this document until technologies to mitigate the pandemic are widely available.

### Interdisciplinary Context

Collaboration across several broad areas of research is critical, as different areas provide different information and context necessary to understanding the virus and disease.
This review provides a biological perspective on the SARS-CoV-2 virus and efforts to develop diagnostic, prophylactic, and therapeutic responses to COVID-19.
We provide only brief summaries of two other important perspectives on this pandemic: epidemiology and public health.
Research in these areas often seeks to anticipate, model, and prevent outbreaks of infectious disease or to understand and manage human behavior relevant to health and disease.
Their insights are critical to mounting a global response to the pandemic.
Epidemiological analyses have investigated patterns of transmission within and between communities, the symptoms associated with and the duration of infection and/or contagiousness, and how the virus propagates, among other characteristics 
[@url:https://www.cdc.gov/coronavirus/2019-ncov/cases-updates/about-epidemiology/index.html].
Epidemiology also has a close relationship to public policy by providing model-based insights into how preventative measures and public response can shift outcomes [@url:https://covid19.healthdata.org/united-states-of-america].
Public health addresses social and human factors influencing individuals' exposure and susceptibility to pathogens, such as resource availability, inequality, human behavior, and accurate information.
Strategies to manage the current epidemic have included the promotion of hand hygiene, social distancing, and personal protective equipment such as masks to mitigate spread and containment approaches such as test, trace, and isolate, which depends on widespread testing, contact tracing, and quarantining.
An effective public health response involves response coordination, disease surveillance, intervention monitoring, risk communication, and health education (including the containment of ‚Äúinfodemics‚Äù of false information) [@isbn:978-92-4-156553-0].
Epidemiology and public health intersect with the topics addressed in this manuscript because they both inform and benefit from relevant biotechnological developments.
For example, the development of accurate and fast diagnostic testing is relevant to test, trace, and isolate strategies for containment, and public education will be critical to deploying a vaccine once one is developed.
The present analysis focuses less on human and social factors and more on the basic biology of infection, diagnosis, and recovery, but these areas are inextricable in understanding and responding to the COVID-19 pandemic.

### Initial Characterization

The first genome sequence of the virus was released on January 3, 2020 and then used to determine that the cluster of pneumonia cases seen in Wuhan were caused by a novel coronavirus [@doi:10.46234/ccdcw2020.017].
Multiple research groups have drafted the genome sequence of SARS-CoV-2 based on sequencing of clinical samples collected from the lower respiratory tract, namely bronchoalveolar lavage fluid (BALF) and the upper respiratory tract, in the form of throat swabs [@doi:10.1038/s41586-020-2008-3; @doi:10.1038/s41586-020-2012-7; @doi:10/ggjr43].
Analysis of the SARS-CoV-2 genome revealed significant sequence homology with two coronaviruses known to infect humans, with about 79% identity to SARS-CoV and 50% to MERS-CoV [@doi:10/ggjr43].
However, the highest degree of similarity was observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21) [@doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7], with identity between SARS-CoV-2 and RATG13 as high as 96.2% [@doi:10.1038/s41586-020-2012-7; @doi:10.1093/nsr/nwaa036].
This evidence therefore suggests the SARS-CoV-2 virus may be the result of zoonotic transfer of a virus from bats to humans.
Nevertheless, some fragments between SARS-CoV-2 and RATG13 have a difference of up to 17%, suggesting a complex natural selection process during zoonotic transfer.
While the _S_ region is highly similar to that of viruses found in pangolins [@doi:10.1101/2020.02.19.950253], there is no consensus about the origin of _S_ in SARS-CoV-2, as it could potentially be the result either of recombination or coevolution [@doi:10.1093/nsr/nwaa036; @doi:10.1126/sciadv.abb9153].
Though the intermediate host serving as the source for the zoonotic introduction of SARS-CoV-2 to human populations has not yet been identified, the SARS-CoV-2 virus has been placed within the coronavirus phylogeny through comparative genomic analyses.
enomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-2 is unlikely to have originated in a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi:10.1038/s41591-020-0820-9].
While the position of the SARS-CoV-2 virus within the coronavirus phylogeny has been largely resolved, the functional consequences of molecular variation between this virus and other viruses, such as its bat and pangolin sister taxa or SARS-CoV, remain unknown [@doi:10.1038/s41591-020-0820-9, @doi:10.1038/s41586-020-2012-7].
Fortunately, the basic genome structure of coronaviruses is highly conserved, and insight into the mechanisms the virus uses to enter cells, replicate, and spread is likely to be available from prior research in coronaviruses.

### Coronaviruses and Humans Hosts

Coronaviruses have long been known to infect animals and have been the subject of veterinary medical investigations and vaccine development efforts due to their effect on the health of companion and agricultural animals [@doi:10/ckfn8b].
Most coronaviruses show little to no transmission in humans.
However, today it is thought that approximately one-third of common cold infections are caused by four HCoV (HCoV-229E, HCoV-NL63, HCoV-OC43 and HCoV-HKU1) [@doi:10.1038/s41579-018-0118-9;@doi:10.3390/diseases4030026].
The first coronavirus species infecting humans (human coronavirus or HCoV) were identified in the 1960s: HCoV-229E in 1965 [@doi:10.3181/00379727-121-30734] and HCoV-OC43 in 1967 [@doi:10.1073/pnas.57.4.933].
Both of these viruses cause cold-like symptoms [@doi:10.1186/1743-422X-1-7; @pmid:20700397].
Two additional HCoV were subsequently identified [@doi:10.1038/nm1024; @doi:10.1128/JVI.79.2.884-895.2005].
In 2003, HCoV-NL63 [@doi:10.1038/nm1024] was identified in a 7-month-old infant and then identified in clinical specimens collected from seven additional patients, five of whom were infants (younger than 1 year) and the remainder of whom were adults.
CoV-HKU1 was identified in samples collected from a 71-year-old pneumonia patient in 2004 and identified in samples collected from a second adult patient [@doi:10.1128/JVI.79.2.884-895.2005].
These viruses are associated with respiratory diseases of varying severity, ranging from common cold to severe pneumonia, with severe symptoms mostly observed in immunocompromised individuals [@doi:10.1086/377612].
In addition to these relatively mild HCoV, however, highly pathogenic human coronaviruses have been identified, including the severe acute respiratory syndrome coronavirus 1 (SARS-CoV or SARS-CoV-1) and the Middle East respiratory syndrome coronavirus (MERS-CoV) although both infections were largely confined to specific geographic regions [@doi:10.1038/nrmicro.2016.81; @doi:10.1038/s41579-018-0118-9; @doi:10.1016/bs.aivir.2018.01.001].

At the time that SARS-CoV emerged in the early 2000s, no HCoV had been identified in almost 40 years [@doi:10.1038/nrmicro.2016.81].
The first case of SARS was reported in November 2002 in the Guangdong Province of China, and over the following month, the disease spread more widely within China and then into several countries across multiple continents [@doi:10.1093/ajcp/aqaa029; @doi:10.1038/nrmicro.2016.81].
Unlike previously identified HCoV, SARS was much more severe, with an estimated death rate of 9.5% [@doi:10.1093/ajcp/aqaa029].
It was also highly contagious via droplet transmission, with a basic reproduction number (R~0~) of 4 (i.e., each person infected was estimated to infect four other people) [@doi:10.1093/ajcp/aqaa029].
However, the identity of the virus behind the infection remained unknown until April of 2003, when the SARS-CoV virus was identified through a worldwide scientific effort spearheaded by the WHO [@doi:10.1038/nrmicro.2016.81].
SARS-CoV belonged to a distinct lineage from the two other HCoV known at the time [@doi:10.1093/ajcp/aqaa029].
By July 2003, the SARS outbreak was officially determined to be under control, with the success credited to successful infection management practices [@doi:10.1038/nrmicro.2016.81].
A decade later, a second outbreak of severe respiratory illness associated with a coronavirus emerged, this time in the Arabian Peninsula.
This disease, known as Middle East respiratory syndrome (MERS), was linked to another novel coronavirus, MERS-CoV.
The fatality rate associated with MERS is much higher than that of SARS, at almost 35%, but the disease is much less easily transmitted, with an R~0~ of 1 [@doi:10.1093/ajcp/aqaa029].
Although MERS is still circulating, its low reproduction number has allowed for its spread to be contained [@doi:10.1093/ajcp/aqaa029].
The COVID-19 pandemic is thus associated with the seventh HCoV to be identified and the fifth since the turn of the millennium, though additional HCoVs in circulation may remain undetected.

SARS-CoV and MERS-CoV were ultimately managed largely through infection management practices (e.g., mask wearing) and properties of the virus itself (i.e., low rate of transmission), respectively [@doi:10.1038/nrmicro.2016.81; @doi:10.1093/ajcp/aqaa029].
Vaccines were not used to control either virus, although vaccine development was undertaken for SARS-CoV [@doi:10.3390/v11010059].
In general, care for SARS and MERS patients focuses on supportive care and symptom management [@doi:10.1093/ajcp/aqaa029].
Clinical treatments for SARS and MERS developed during the outbreaks generally do not have strong evidence supporting their use.
Common treatments included Ribavirin, an antiviral, often in combination with corticosteroids or sometimes interferon (IFN) medications, which would both be expected to have immunomodulatory effects [@doi:10.1038/nrmicro.2016.81].
However, retrospective and _in vitro_ analyses have reported inconclusive results of these treatments on SARS and the SARS-CoV virus, respectively [@doi:10.1038/nrmicro.2016.81].
IFNs and Ribavirin have shown promise in _in vitro_ analyses of MERS, but their clinical effectiveness remains unknown [@doi:10.1038/nrmicro.2016.81].
Therefore, only limited strategy for the pharmaceutical management of COVID-19 can be adopted from previous severe hCoV infections. 
Research in response to prior outbreaks of hCoV-borne infections, such as SARS and MERS, have provided a strong foundation for hypotheses about the pathogenesis of SARS-CoV-2 as well as potential diagnostic and therapeutic approaches.

#### Human Immune Response to Viral Threats

Understanding the fundamental organization of the human immune response to viral threats is critical to understanding the varied response to SARS-CoV-2.
The human immune system utilizes a variety of innate and adaptive responses to protect against the pathogens it encounters.
The innate immune system consists of barriers, such as the skin, mucous secretions, neutrophils, macrophages, and dendritic cells.
It also includes cell-surface receptors that can recognize the molecular patterns of pathogens.
The adaptive immune system utilizes antigen-specific receptors that are expressed on B and T lymphocytes.
These components of the immune system typically act together; the innate response acts first, and the adaptive response begins to act several days after initial infection following the clonal expansion of T and B cells [@doi:10.1016/j.jaci.2005.09.034].
After a virus enters into a host cell, its antigen is presented by major histocompatibility complex 1 (MHC 1) molecules and is then recognized by cytotoxic T lymphocytes.

There are several ways in which the SARS-CoV virus can evade the human immune response.
One way is through CD163+ and CD68+ macrophage cells, which are crucial for the establishment of SARS in the body [@doi:10.1038/mi.2015.127].
These cells most likely serve as viral reservoirs that help shield SARS-CoV from the innate immune response.
According to a study on the viral dissemination of SARS-CoV in Chinese macaques, viral RNA could be detected in some monocytes throughout the process of differentiation into dendritic cells [@doi:10.1038/mi.2015.127].
This lack of active viral replication allows SARS-CoV to escape innate immunity because reduced levels of detectable viral RNA allow the virus to avoid both natural killer (NK) cells and Toll-like receptors [@doi:10.1038/mi.2015.127].
Another strategy that SARS-CoV utilizes to increase its virulence is the suppression of host gene expression by nonstructural protein 1 (Nsp1).
Nsp1 accomplishes this by stalling mRNA translation and inducing endonucleolytic cleavage and mRNA degradation [@doi:10.1126/science.abc8665].
These evasion mechanisms, in turn, can lead to systemic infection.

SARS-CoV also evades the immune response by interfering with type I IFN induction signaling, which is a mechanism that leads to cellular resistance to viral infections.
SARS-CoV employs methods such as ubiquitination and degradation of RNA sensor adaptor molecules MAVS and TRAF3/6 [@doi:10.12932/AP-200220-0772].
MERS-CoV also utilizes these mechanisms, as well as histone modification.
Also, MERS-CoV downregulates antigen presentation via MHC class I and MHC class II, which leads to a reduction in T cell activation [@doi:10.12932/AP-200220-0772].

Based on the aforementioned mechanisms for both SARS-CoV and MERS-CoV, SARS-CoV-2 most likely utilizes similar mechanisms for immune evasion.
In addition, SARS-CoV-2 also uses a protein encoded from open reading frame 8 (ORF8) that interferes with the expression of MHC-I molecules, which reduces the elimination of infected cells by cytotoxic T lymphocytes [@doi:10.1101/2020.05.24.111823].

##### Cytokine Storm

One of the main immune responses that contribute to the onset of Acute Respiratory Distress Syndrome (ARDS) in COVID-19 patients is the cytokine storm, which causes an extreme inflammatory response due to a release of pro-inflammatory cytokines and chemokines by immune effector cells.
In addition to respiratory distress, this mechanism leads to organ failure and death in severe COVID-19 cases [@doi:10.1016/j.jpha.2020.03.001].
<!-- As I am cleaning up the text, I'll move the cytokine storm text here, as right now it's very decentralized and one of our goals is to merge everything together in the intro!-->
IL-6 promotes the differentiation of activated B cells into immunoglobulin-producing plasma cells [@pmid:9052836] and acts as a growth factor for hybridoma and myeloma cells [@pmid:8431556; @pmid:2104241].
In addition, IL-6 also induces the differentiation of na√Øve CD4+ T cells into effector T-cell subsets [@doi:10.1158/2326-6066.CIR-14-0022].
In this way interleukins regulate both the pro- and anti-inflammatory responses.
<!-- Note: make sure these terms are defined!-->
In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system, ensuring homeostasis.
The introduction of the S-protein from SARS-CoV to mouse macrophages was found to increase production of IL-6 and TNF-Œ± [@doi:10.1016/j.virusres.2007.02.007], and deceased SARS-CoV patients were found to have intermediate levels of IL-6, IL-1ùõΩ, and TNF-Œ± expressed in a number of ACE2-expressing cell types sampled from the lung and bronchial tissues during autopsy [@doi:10.1002/path.2067].
However, other reports found the severe respiratory condition ARDS to be associated with elevated concentrations of IL-6 in BALF, but that concentrations of Tumor Necrosis Factor Œ± (TNF-Œ±) and IL-1ùõΩ decreased with the onset of ARDS [@doi:10.1164/ajrccm.164.10.2104013].
<!--
Just to clarify, is the point here that SARS patients have intermediate levels of IL-6 but other respiratory conditions, like ARDS have higher levels of IL-6?
As a non-immunologist, my high level understanding of what is written is: IL-6 plays a role in regulating the immune response to infection. And different IL-6 levels are found depending on the disease. This IL-6 cytokine causes a large immune response that damages cells and is fatal. - @ajlee21 
-->
These cytokines enhance the pro-inflammatory reaction by increase acute-phase signaling, trafficking of immune cells to the site of primary infection, epithelial cell activation, and secondary cytokine production.
The acute phase response to infection results in the heavy damage of the endothelium of blood vessels, which disrupts the balance between the pro- and anti-inflammatory response [@doi:10.1164/ajrccm.164.10.2104013].
Thus, the holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death in ARDS and Severe Acute Respiratory Syndrome (SARS) [@doi:10.1007/s00281-017-0629-x], also caused by the new coronavirus.
<!-- Suggestion: define distinction between Cytokine Release Syndrome and Cytokine Storm Syndrome-->

### COVID-19: Infection and Presentation

#### General Immunological Background

Until more information becomes available about the pathogenesis of SARS-CoV-2 specifically, insight can be gained by examining how the human body responds to viral pathogens generally, how infection from related viruses proceeds, and how this information intersects with what is known about COVID-19.

##### hCOV and the Immune Response

Research in other coronaviruses that affect humans provides some indication of how SARS-CoV-2 infection proceeds in spite of the human immune response.
While the first line of defense for the immune system is physical barriers such as the skin and mucus, viruses such as SARS-CoV that infect the epithelium are known to be adept at bypassing these defenses [@doi:10.1038/mi.2015.127]. 
Additionally, SARS-CoV is able to mask its dsRNA from detection by the immune system.  
dsRNA is produced during the viral replication process and is a pathogen-associated molecular pattern (PAMP) that would typically initiate a response from the innate immune system [@doi:10.3389/fimmu.2018.00829].
_In vitro_ analysis of nidoviruses including SARS-CoV suggests that these viruses can induce the development of double-membrane vesicles that protect the dsRNA signature from being detected by the host immune system [@doi:10.1128/jvi.02501-05].
This protective envelope can therefore insulate these coronaviruses from the innate immune response's detection mechanism [@doi:10.1016/j.jpha.2020.03.001].
Coronaviruses such as SARS-CoV are also able to evade the humoral immune response through other mechanisms, such as inhibiting certain cytokine pathways or down-regulating antigen presentation by the cells [@doi:10.1128/jvi.02501-05].

Research on other coronaviruses provides some evidence of how the immune response may proceed in the presence of SARS-CoV-2.
Two of the most common immunoglobulin classes are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most common antibodies found in the blood.
When an antibody binds to an antigen, it forms a complex that can then be cleared through several possible processes.
Following SARS infection, IgM antibodies were first detected in most patients during the second week following infection and remained detectable until the twelfth week, while IgG antibodies were first detected in the second or third week and remained at close to their peak concentration through the thirteen weeks of the study [@doi:10.1056/NEJM200307313490520].
A two-year longitudinal study following convalesced SARS patients with a mean age of 29 found that IgG antibodies were detectable in all 56 patients surveyed for at least 16 months, and remained detectable in all but 4 (11.8%) of patients through the full two-year study period [@doi:10.1086/500469].
Understanding the immune response to viral exposure and the mechanisms that allow viruses to evade these mechanisms will be critical for understanding the therapeutic interventions that are being considered for COVID-19.
The humoral response to SARS-CoV-2 is important because this component of the immune response works to reduce the severity of infection in the short term and is an important mechanism for protection in the event of reexposure.
Entry and replication are described above; in brief, during entry into a host cell, a virion must first bind to a receptor on the host cell surface, and in enveloped viruses such as coronaviruses, the viral membrane then fuses to the cellular membrame.
During the spread phase, the viral infection begins to target specific tissues or organs and can also pass form the host into the environment in a process known as viral shedding [@doi:10.3390/v4061011].
In order to understand the pathogenesis of SARS-CoV-2 and related coronaviruses, it is necessary to understand how the human body responds to viral pathogens generally.
The response to an antigen such as a virus comprises two components: the innate and adaptive immune responses.
Through innate immunity, also known as humoral or antibody-mediated immunity, the body can eliminate pathogens in circulation that have not yet entered cells.
Adaptive immunity, or the cell-mediated response, is the process by which infected cells are destroyed.
The antibody-mediated response works to prevent or reduce entry into cells, while the cell-mediated response works to stop the pathogen's replication and spread.
Thus, pathogens infects hosts and hosts have immune defenses and responses; furthermore, pathogens have developed ways to evade the immune system (as well as, in general, various ways to manipulate the host).
Other general complexities to consider include that disease symptoms can be due to a pathogen or due to the reaction of the immune system to the pathogen and that the host immune system can defend against or fight infection but it can also sometimes cause too much damage to the host [@doi: 978-0815332183; @doi: 9780071283663].

##### SARS-CoV-2 and the Immune Response

The first line of defense for the immune system is physical barriers such as the skin and mucus, but viruses such as SARS-CoV that infect the epithelium are able to bypass these defenses [@doi:10.1038/mi.2015.127].
Once a pathogen enters circulation, the innate immune response uses several mechanisms to try to eliminate it.
Several types of leukocytes (white blood cells) are professional phagocytic cells that can detect and engulf a particle, enveloping it in a phagosome.
The conditions within the phagosome are then modulated to destroy the pathogen through fusion of the phagosome with a lysosome or granule to form a phagolysosome.
This allows the cell to break the pathogen into its component peptides. 
Monocytes, which include both dendritic cells and macrophages, are types of phagocytic white blood cells that are also professional antigen-presenting cells, which meaning that they can then present the resulting viral peptides on the cell surface via Major Histocompatability Complex II (MHC-II) proteins.
However, research in related viruses suggests that SARS-CoV-2 may possess some mechanisms that facilitate evasion of the antibody-mediated response.
_In vitro_ analysis of nidoviruses including SARS-CoV suggests that these viruses can induce the development of double-membrane vesicles, preventing the dsRNA signature from being detected by the host immune system [@doi:10.1128/jvi.02501-05].
This protective envelope can therefore insulate these coronaviruses from the innate immune response's detection mechanism [@doi:10.1016/j.jpha.2020.03.001].
Presentation of the antigen is important because this step allows it to be detected by B-lymphocytes, also called B-cells, which are a type of white blood cell with immunoglobulins bound to the membrane.
Individual B-cells have unique membrane-bound immunoglobulins (i.e. membrane-bound antibodies).
A B-cell is able to bind to an antigen only if its immunoglobulin is a match.
For example, toll-like receptor 3 (TLR3) can detect double-stranded RNA (dsRNA) present during viral replication, allowing it to recognize a viral threat.
The binding of a pattern recognition receptor (PRR), such as a TLR, to the antigen allows the B-cell to engulf it the virus via endocytocis.
Once the antigen has been processed and presented on the cell surface, helper T-cells can bind to the MHC-II/antigen complex.
This second binding step catalyzes the release of cytokines as part of the inflammatory response, as well as the proliferation of active B-cells (plasma cells) capable of producing the antibodies that respond to the antigen.
Coronaviruses such as SARS-CoV are also able to evade the humoral immune response through other mechanisms, such as inhibiting certain cytokine pathways or down-regulating antigen presentation by the cells [@doi:10.1128/jvi.02501-05].
Additionally, the cell will produce memory B-cells that can retain information about how to respond to future immune challenges by this pathogen.
Understanding the humoral response to SARS-CoV-2 is important because this component of the immune response works to reduce the severity of infection in the short term and is also the mechanism through which the body can respond to reexposure.

Research on other coronaviruses provides some evidence of how the immune response may proceed in the presence of SARS-CoV-2.
Antibodies, also known as immunoglobulins, belong to five classes that describe which antigens they bind to.
Two of the most common immunoglobulin classes are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most common antibodies found in the blood.
When an antibody binds to an antigen, it forms a complex that can then be cleared through several possible processes.
Following SARS infection, IgM antibodies were first detected in most patients during the second week following infection and remained detectable until the twelfth week, while IgG antibodies were first detected in the second or third week and remained at close to their peak concentration through the thirteen weeks of the study [@doi:10.1056/NEJM200307313490520].
A two-year longitudinal study following convalesced SARS patients with a mean age of 29 found that IgG antibodies were detectable in all 56 patients surveyed for at least 16 months, and remained detectable in all but 4 (11.8%) of patients through the full two-year study period [@doi:10.1086/500469].
Understanding the immune response to viral exposure and the mechanisms that allow viruses to evade these mechanisms will be critical for understanding the therapeutic interventions that are being considered for COVID-19.
The humoral response to SARS-CoV-2 is important because this component of the immune response works to reduce the severity of infection in the short term and is an important mechanism for protection in the event of reexposure.

##### The Adaptive Immune Response

##### The Inflammatory Response and the Cytokine Storm

#### Clinical Presentation of COVID-19

A great diversity of symptom profiles has been observed for COVID-19, although a large study from Wuhan, China suggests fever and cough as the two most common symptoms on admission [@doi:10.1056/NEJMoa2002032].
One early retrospective study in China described the clinical presentations of patients infected with SARS-CoV-2 as including lower respiratory tract infection with fever, dry cough, and dyspnea [@doi:10/ggnxb3].
This study [@doi:10/ggnxb3] noted that upper respiratory tract symptoms were less common, which suggests that the virus targets cells located in the lower respiratory tract.
However, data from the New York City region [@doi:10.1001/jama.2020.6775; @doi:10.1056/NEJMc2010419] showed variable rates of fever as a presenting symptom, suggesting that symptoms may not be consistent across samples.
These differences are present when comparing both between institutions in similar locations and between different regions experiencing COVID-19 outbreaks, leading to conflicting reports of the frequency of fever as a presenting symptom for patients upon hospital admission.
For example, even within New York City, one study [@doi:10.1001/jama.2020.6775] identified low oxygen saturation (<90% without the use of supplemental oxygen or ventilation support) in a significant percentage of patients on presentation, while another study [@doi:10.1056/NEJMc2010419] reported cough, fever, and dyspnea as the most common presenting symptoms.
The variability of both which symptoms present and their severity makes it difficult for public health agencies to provide clear recommendations for citizens regarding what symptoms indicate SARS-CoV-2 infection and should prompt isolation.

### Role of the COVID-19 Review

![**Summary of the relationships among topics covered in this review.**](images/N000-overview.png){#fig:overview secno=1}

<!-- Rough draft, but basically: distill signal from noise to help accelerate the discovery process-->
Several review articles on aspects of COVID-19 have already been published.
These have included reviews on the disease epidemiology [@doi:10.1016/j.molmed.2020.02.008], immunological response [@doi:10.1016/j.immuni.2020.05.002], diagnostics [@doi:10.1126/scitranslmed.abc1931], and pharmacological treatments [@doi:10.1016/j.immuni.2020.05.002; @doi:10.1001/jama.2020.6019].
Others [@doi:10.1038/d41591-020-00026-w] and [@doi:10.1001/jama.2020.12839] provide narrative reviews of progress on some important ongoing COVID-19 research questions.
With the worldwide scientific community uniting during 2020 to investigate SARS-CoV-2 and COVID-19 from a wide range of perspectives, findings from many disciplines are relevant on a rapid timescale to a broad scientific audience.
Additionally, many findings are published as preprints, which are available prior to going through the peer review process.
As a result, centralizing, summarizing, and critiquing new literature broadly relevant to COVID-19 can help to expedite the interdisciplinary scientific process that is currently happening at an advanced pace.
We are particularly interested in providing background to the development of diagnostic, prophylactic, and therapeutic approaches to COVID-19.
Two major concerns within diagnosis include the detection of current infections in individuals with and without symptoms, and the detection of past exposure without an active infection.
In the latter category, identifying whether individuals can develop or have developed sustained immunity is also a major consideration.
The development of high-throughput, affordable methods for detecting active infections and sustained immunity will be critical to understanding and controlling the disease.
The identification of interventions that can mitigate the effect of the virus on exposed and infected individuals is a significant research priority.
Some possible approaches include the identification of existing pharmaceuticals that reduce the severity of infection, either by reducing the virus' virulence (e.g., antivirals) or managing the most severe symptoms of infection.
Due to the long timeline for the development of novel pharmaceuticals, in most cases, research surrounding possible pharmaceutical interventions focuses on the identification and investigation of existing compounds whose mechanisms may be relevant to COVID-19.
Other foci of current research include the identification of antibodies produced by survivors of COVID-19 and the development of vaccines.
Understanding the mechanisms describing host-virus interactions between humans and SARS-CoV-2 is thus critical to identifying candidate therapeutics.
An overview of the topics covered is visualized in Figure (Figure @fig:overview).
Thus, in this review, we seek to consolidate information about efforts to develop strategies for diagnosis and therapeutics as new information is released by the scientific community.
We include information from both traditional peer-reviewed scientific literature and from preprints, which typically have not undergone peer review but have been critically evaluated by the scientists involved in this effort.
The goal of this manuscript is to present preliminary findings within the broader context of COVID-19 research and to identify the broad interpretations of new research, as well as limitations to interpretability.
