## Pathogenesis

The current COVID-19 pandemic, caused by the SARS-CoV-2 virus, represents an acute global health crisis where symptoms can range from mild to severe or fatal [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html], can affect a variety of organs and systems, and includes outcomes such as acute respiratory distress and acute lung injury, among other complications.
Viral pathogenesis is typically broken down into three major components: entry, replication, and spread [@isbn:0-9631172-1-1 {chap. 45}].
However, in order to draw a more complete picture of pathogenesis, it is also necessary to examine how infection manifests clinically, identify systems-level interactions between the virus and the human body, and consider the possible effects of variation or evolutionary change on pathogenesis and virulence.
Though hCoV are not common and the SARS-CoV-2 virus appears to have emerged only recently, the rapid release of the genomic sequence of the virus in January 2020 provided early opportunities for comparative genomic analysis of the virus compared to its close phylogenetic relatives.
Because many mechanisms of pathogenicity are conserved among coronaviruses, this phylogenetic analysis provided an important basis for forming hypotheses about how the virus interacts with hosts.
Additionally, worldwide sequencing of viral samples has provided some preliminary insights into possible mechanisms of adaptation in the virus, and omics-based analysis of patient samples has elucidated some of the biological changes the virus induces in its human hosts.
In this way, both biomedicine and genomics are important pieces of the puzzle of SARS-CoV-2 presentation and pathogenesis.

### Fundamental Viral Pathogenesis

As discussed above, the virus clusters with known coronaviruses (order *Nidovirales*, family *Coronaviridae*, subfamily *Orthocoronavirinae*).
Coronaviruses are large viruses that can be identified by their distinctive "crown-like" shape.
Their spherical virions are made from lipid envelopes ranging from 100 to 160 nanometers (nm) in which peplomers of two to three spike (S) glycoproteins are anchored, creating the crown [@isbn:978-0781760607; @doi:10/dvvg2h].
These spikes, which are critical to both viral pathogenesis and the host immune response, have been visualized using cryo-electron microscopy [@doi:10.1126/science.abb2507].
Because they induce the human immune response, they are the target of many proposed therapeutic agents.
Phylogenetic analysis of the coronaviruses reveals four major subclades, each corresponding to a genus: the alpha, beta, delta and gamma coronaviruses.
Among them, alpha and beta coronaviruses infect mammalian species, gamma coronaviruses infect avian species, and delta coronaviruses infect both mammalian and avian species [@doi:10.1146/annurev-virology-110615-042301].
Phylogenetic analysis of a polymerase chain reaction (PCR) amplicon fragment from five patients along with the full genomic sequence revealed SARS-CoV-2 to be a novel betacoronavirus belonging to the B lineage, also known as sarbecovirus, which also includes the coronavirus SARS-CoV that causes SARS in humans [@doi:10/ggjs7j].
Because viral structure and mechanisms of pathogenicity are highly conserved within the order, this phylogenetic analysis provides a basis for forming hypotheses about how the virus interacts with hosts, including which tissues, organs, and systems would be most susceptible to SARS-CoV-2 infection.

#### Genomic Structure of Coronaviruses

Genome structure is highly conserved among coronaviruses, meaning that the relationship between the SARS-CoV-2 genome and its pathogenesis can be inferred from prior research in related viral species.
The genomes of viruses in the *Nidovirales* order share several fundamental characteristics.
They are non-segmented, which means the viral genome is contained in a single capsid, and are enveloped, which means that the genome and capsid are encased by a lipid bilayer.
Coronaviruses have large positive-sense RNA (ssRNA+) genomes ranging from 27 kilobases (Kb) to 32 Kb in length [@doi:10.1007/978-1-4939-2438-7_1; @doi:10/ggjr43].
The SARS-CoV-2 genome lies in the middle of this range at 29,903 bp [@doi:10/ggjr43].
Genome organization is highly conserved within the order [@doi:10.1007/978-1-4939-2438-7_1].
There are three major regions: one containing the replicase gene and one containing the genes encoding structural proteins [@doi:10.1007/978-1-4939-2438-7_1].
The replicase gene comprises about two-thirds of the genome of coronaviruses and consists of two open reading frames that are translated with ribosomal frameshifting [@doi:10.1007/978-1-4939-2438-7_1].
This polypeptide is then translated into 16 non-structural proteins (nsp1-16, except in Gammacoronaviruses, where nsp1 is absent) that form replication machinery used to synthesize viral RNA [@doi:10.1002/jmv.25681].
The remaining third of the genome encodes structural proteins, including the spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins.
Additional accessory genes are sometimes present between these two regions, depending on the species or strain. <!-- TODO: Add a figure showing the structure of the genome.-->
Much attention has been focused on the S protein, which is a critical structure involved in cell entry.

#### Pathogenic Mechanisms of Coronaviruses

While, like most viruses, it is possible that SARS-CoV and SARS-CoV-2 can enter cells through endocytosis, coronaviruses are able to target cells for entry through fusion with the plasma membrane [@doi:10.1038/cr.2008.15; @doi:10.1146/annurev-biochem-060208-104626].
This process is conserved among coronaviruses and is closely associated with the content of their genomes.
Cell entry proceeds in three steps: binding, cleavage, and fusion.
First, the viral spike protein binds to a host cell via a recognized receptor.
Coronaviruses can bind to a range of host receptors [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009], with binding conserved only at the genus level [@doi:10.1146/annurev-virology-110615-042301].
Viruses in the betacoronavirus genus, to which SARS-CoV-2 belongs, are known to bind to CEACAM1, Neu 5,9 Ac2, and Angiotensin-Converting Enzyme 2 (ACE2) [@doi:10.3390/v4061011].
SARS-CoV-2 has a high affinity for the human ACE2 receptor, which is expressed in the vascular epithelium, other epithelial cells, and cardiovascular and renal tissues [@doi:10.1007/s00134-020-05985-9; @doi:10.1007/978-0-387-33012-9_85].
The binding process is guided by the molecular structure of the spike protein, which is structured in three segments: an ectodomain, a transmembrane anchor, and an intracellular tail [@doi:10.1128/JVI.02615-14].
The ectodomain forms the crown-like structures on the viral membrane and contains two subdomains known as the S1 and S2 subunits [@doi:10.1038/nrmicro2090].
The S1 (N-terminal) domain forms the head of the crown and contains the receptor binding motif (RBM), and the S2 (C-terminal) domain forms the stalk that supports the head [@doi:10.1038/nrmicro2090].
The S1 subunit guides the binding of the virus to a host cell receptor, and the S2 subunit guides the fusion process [@doi:10.1128/JVI.02615-14].

After the binding of the S1 subunit to a receptor, the spike protein is often cleaved at the S1-S2 boundary by a host protease [@doi:10.1016/j.cell.2020.02.058; @doi:10/cf8chz; @doi:10.1016/j.antiviral.2020.104792].
Similar to SARS-CoV, SARS-CoV-2 exhibits redundancy in which host proteases can cleave the S protein [@doi:10.1016/j.cell.2020.02.052].
Specifically, both transmembrane protease serine protease-2 (TMPRSS2) and cathepsins B/L have been shown to mediate SARS-CoV-2 S protein proteolytic priming, and small molecule inhibition of these enzymes fully inhibited viral entry _in vitro_ [@doi:10.1016/j.cell.2020.02.052; @doi:10.1038/s41467-020-15562-9].
Proteolytic priming prepares the S protein for fusion [@doi:10/cf8chz; @doi:10.1016/j.antiviral.2020.104792].
The two subunits remain bound by van der Waals forces, with the S1 subunit stabilizing the S2 subunit during the membrane fusion process [@doi:10.1016/j.cell.2020.02.058].
Electron microscopy suggests that in some coronaviruses, including SARS-CoV and MERS-CoV, a six-helix bundle separates the two subunits in the postfusion conformation, and the unusual length of this bundle facilitates membrane fusion through the release of additional energy [@doi:10.1146/annurev-virology-110615-042301].
Cleavage at a second site within S2 by these same proteases activates _S_ for fusion by inducing conformational changes [@doi:10.1016/j.cell.2020.02.058].
The viral membrane can then fuse with the endosomal membrane to release the viral genome into the host cytoplasm.
Once the virus enters a host cell, the replicase gene is translated and assembled into the viral replicase complex.
This complex then synthesizes the double-stranded RNA (dsRNA) genome from the genomic ssRNA(+).
The dsRNA genome is transcribed and replicated to create viral mRNAs and new ssRNA(+) genomes [@doi:10.1007/978-1-4939-2438-7_1; @url:https://viralzone.expasy.org/30?outline=all_by_species].
From there, the virus can spread in other cells.
In this way, the genome of SARS-CoV-2 provides some insight into the pathogenic behavior of the virus.

#### Immune Evasion Strategies

Research in other coronaviruses that affect humans provides some indication of how SARS-CoV-2 infection proceeds in spite of the human immune response.
By infecting the epithelium, viruses such as SARS-CoV are known to bypass the physical barriers such as skin and mucus that comprise the immune systems' first line of defense [@doi:10.1038/mi.2015.127].
Once the virus infiltrates host cells, it is adept at evading detection.
CD163+ and CD68+ macrophage cells especially are crucial for the establishment of SARS-CoV in the body [@doi:10.1038/mi.2015.127].
These cells most likely serve as viral reservoirs that help shield SARS-CoV from the innate immune response.
According to a study on the viral dissemination of SARS-CoV in Chinese macaques, viral RNA could be detected in some monocytes throughout the process of differentiation into dendritic cells [@doi:10.1038/mi.2015.127].
This lack of active viral replication allows SARS-CoV to escape innate immunity because reduced levels of detectable viral RNA allow the virus to avoid both natural killer (NK) cells and Toll-like receptors [@doi:10.1038/mi.2015.127].
Even during replication, SARS-CoV is able to mask its dsRNA from detection by the immune system.
Although dsRNA is a pathogen-associated molecular pattern (PAMP) that would typically initiate a response from the innate immune system [@doi:10.3389/fimmu.2018.00829], _in vitro_ analysis of nidoviruses including SARS-CoV suggests that these viruses can induce the development of double-membrane vesicles that protect the dsRNA signature from being detected by the host immune system [@doi:10.1128/jvi.02501-05].
This protective envelope can therefore insulate these coronaviruses from the innate immune response's detection mechanism [@doi:10.1016/j.jpha.2020.03.001].

HCoV are also known to interfere with the host immune response, rather than just evade it.
For example, the virulence of SARS-CoV is increased by nonstructural protein 1 (Nsp1), which can suppress host gene expression.
Nsp1 accomplishes this by stalling mRNA translation and inducing endonucleolytic cleavage and mRNA degradation [@doi:10.1126/science.abc8665].
SARS-CoV also evades the immune response by interfering with type I IFN induction signaling, which is a mechanism that leads to cellular resistance to viral infections.
SARS-CoV employs methods such as ubiquitination and degradation of RNA sensor adaptor molecules MAVS and TRAF3/6 [@doi:10.12932/AP-200220-0772].
Also, MERS-CoV downregulates antigen presentation via MHC class I and MHC class II, which leads to a reduction in T cell activation [@doi:10.12932/AP-200220-0772].
These evasion mechanisms, in turn, can lead to systemic infection.
Coronaviruses such as SARS-CoV are also able to evade the humoral immune response through other mechanisms, such as inhibiting certain cytokine pathways or down-regulating antigen presentation by the cells [@doi:10.1128/jvi.02501-05].

#### Host Cell Susceptibility

ACE2 and TMPRSS2 have been identified as a primary receptor and a critical protease, respectively, facilitating the entry of SARS-CoV/CoV-2 into a target cell [@doi:10.1038/nature02145; @doi:10.1126/science.abb2507; @doi:10.1128/JVI.01542-10; @doi:10.1016/j.cell.2020.02.052; @doi:10.1128/JVI.02232-10].
This finding has led to a hypothesized role for ACE2 and TMPRSS2 expression in determining which cells, tissues, and organs are most likely to be infected by SARS-CoV-2.
The ACE2 receptor is expressed in numerous organs, such as the heart, kidney, and intestine, but it is most prominently expressed in alveolar epithelial cells; this pattern of expression is expected to contribute to the virus' association with lung pathology [@doi:10.1101/2020.02.24.963348; @doi:10.1007/s00134-020-05985-9].
Clinical investigations of COVID-19 patients have detected SARS-CoV-2 transcripts in BALF (93% of specimens), sputum (72%), nasal swabs (63%), fibrobronchoscopy brush biopsies (46%), pharyngeal swabs (32%), feces (29%) and blood (1%) [@doi:10.1001/jama.2020.3786].
Two studies reported that SARS-CoV-2 could not be detected in the urine specimens [@doi:10.1001/jama.2020.3786; @doi:10.1001/jama.2020.3204]; however, a third study identified four urine samples (out of 58) that were positive for SARS-CoV-2 nucleic acids [@doi:10.1097/CM9.0000000000000774].
Although respiratory failure remains the leading cause of death for COVID-19 patients [@doi:10.1111/his.14134], SARS-CoV-2 infection can damage many other organ systems including the heart [@doi:10.1001/jamacardio.2020.0950], kidneys [@doi:10.1038/s41585-020-0319-7; @doi:10.1186/s13054-020-02872-z], liver [@doi:10/ggpx6s], and gastrointestinal tract [@doi:10.1053/j.gastro.2020.02.055; @doi:10.1111/1751-2980.12851].
As it becomes clear that SARS-CoV-2 infection can damage multiple organs, the scientific community is pursuing multiple avenues of investigation in order to build a consensus about how the virus affects the human body.

### Clinical Presentation of COVID-19

Reports have described diverse symptom profiles associated with COVID-19.
A large study from Wuhan, China suggested that fever and cough are the two most common symptoms at hospital admission [@doi:10.1056/NEJMoa2002032], and another early retrospective study in China described the clinical presentations of patients infected with SARS-CoV-2 as including lower respiratory tract infection with fever, dry cough, and dyspnea [@doi:10/ggnxb3].
This study [@doi:10/ggnxb3] noted that upper respiratory tract symptoms were less common, which suggested that the virus was targeting cells located in the lower respiratory tract.
A later study reported radiographic findings such as ground-glass opacity and bilateral patchy shadowing in the lungs of many hospitalized patients, and most COVID-19 patients had lymphocytopenia, meaning they had low levels of lymphocytes (a type of white blood cell) [@doi:10.1056/NEJMoa2002032].
However, data from the New York City region [@doi:10.1001/jama.2020.6775; @doi:10.1056/NEJMc2010419] showed variable rates of fever as a presenting symptom, suggesting that symptoms may not be consistent across samples.
These differences persist when comparing both between institutions in similar locations and between different regions experiencing COVID-19 outbreaks, leading to conflicting reports of the frequency of fever at hospital admission.
For example, even within New York City, one study [@doi:10.1001/jama.2020.6775] identified low oxygen saturation (<90% without the use of supplemental oxygen or ventilation support) in a significant percentage of patients upon presentation, while another study [@doi:10.1056/NEJMc2010419] reported cough, fever, and dyspnea as the most common presenting symptoms.
Patients may also experience loss of smell, myalgias (muscle aches), fatigue, or headache.
Radiographic findings such as ground-glass opacity and bilateral patchy shadowing in the lungs of many hospitalized patients have also been reported, and most COVID-19 patients had lymphocytopenia, meaning they had low levels of lymphocytes (a type of white blood cell) [@doi:10.1056/NEJMoa2002032].
The variability of both which symptoms present and their severity makes it difficult for public health agencies to provide clear recommendations for citizens regarding what symptoms indicate SARS-CoV-2 infection and should prompt isolation.
For example, evidence suggests that gastrointestinal symptoms presentation does occur [@doi:10.15585/mmwr.mm6928a2], and the CDC has updated its list of symptoms suggestive of COVID-19 to include nausea and vomiting, as well congestion and runny nose [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html].
A recent preprint using data from an app-based survey of 500,000 individuals in the US found that among those tested for SARS-CoV-2, a loss of taste or smell, fever, and a cough were significant predictors of a positive test result [@doi:10.1101/2020.06.09.20126813].
It is important to note that in this study, the predictive value of symptoms may be underestimated if they are not specific to COVID-19 because the outcome measured was a positive, as opposed to a negative, COVID-19 test result.
At the time the surveys were conducted, due to limits in US testing infrastructure, respondents typically needed to have some symptoms known to be specific to COVID-19 in order to qualify for testing in the first place.
Widespread testing of asymptomatic individuals may therefore provide additional insight into the range of symptoms associated with COVID-19.

COVID-19 can affect diverse body systems in addition to causing respiratory problems [@doi:10.1038/s41591-020-0968-3].
For example, COVID-19 can lead to acute kidney injury, especially in patients with severe respiratory symptoms or certain preexisting conditions [@doi:10.1016/j.kint.2020.05.006].
It can also cause neurological complications [@doi:10.1016/j.bbi.2020.03.031; @doi:10/d259], potentially including stroke, seizures or meningitis [@doi:10.1002/jmv.26000; @doi:10.1002/ana.25807].
COVID-19 has also been associated with an increased incidence of large vessel stroke, particularly in patients under the age of 40 [@doi:10.1056/NEJMc2009787], and other thrombotic events including pulmonary embolism and deep vein thrombosis [@doi:10.1016/j.thromres.2020.04.013].
The mechanism behind these complications has been suggested to be related to coagulopathy, with reports indicating the presence of antiphospholipid antibodies [@doi:10.1056/NEJMc2007575] and elevated levels of d-dimer and fibrinogen degradation products (FDP) in deceased patients [@doi:10.1111/jth.14768].
Other viral infections have been associated with coagulation defects and changes to the coagulation cascade; notably SARS was also found to lead to disseminated intravascular coagulation (DIC) and was associated with both pulmonary embolism and deep vein thrombosis [@doi:10.1002/jmv.23354].
The mechanism behind these insults has been suggested to be related to inflammation-induced increases in the von Willebrand factor (VWF) clotting protein, leading to a pro-coagulative state [@doi:10.1002/jmv.23354].
Abnormal clotting (thromboinflammation or coagulopathy) has been increasingly discussed recently as a possible key mechanism in many cases of severe COVID-19, and may be associated with the high d-dimer levels often observed in severe cases [@doi:10/ggvd74; @doi:10.1186/s13054-020-03060-9; @doi:10.1007/s11239-020-02134-3].
This excessive clotting in lung capillaries has been suggested to be related to a dysregulated activation of the complement system, part of the innate immune system [@doi:10.1172/jci.insight.140711; @doi:10.1172/JCI140183].

#### Cytokine Release Syndrome

Symptoms of a disease can be caused by a pathogen, but they can also be caused by the immune system's reaction to the pathogen.
A dysregulated immune response can cause significant damage to the host [@isbn:9780815332183; @isbn:9780071283663; @doi:10.1016/j.chom.2020.05.009].
The inflammatory response has received particular attention for its role in both a healthy response to infection and a pathogenic one.
Inflammation is one of the most visible components of the immune response, as it is responsible for hallmarks of injury such as pain, heat, and swelling [@doi:10.18632/oncotarget.23208].
In response to injury or to signaling by pattern recognition receptors (PRRs) indicating the detection of a molecular pattern associated with a pathogen or foreign body, the immune system stimulates leukocytes that travel to the site of the threat, where they then produce cytokines [@doi:10.18632/oncotarget.23208].
Cytokines are a diverse group of small proteins that play an important role in intercellular signaling [@doi:10.1128/MMBR.05015-11].
Cytokines can be both pro- and anti-inflammatory, which means they can either stimulate or inhibit the production of additional cytokines [@doi:10.1128/MMBR.05015-11; @doi:10.1097/AIA.0b013e318034194e].
Some notable pro-inflammatory cytokines include the interleukins IL-1&beta; and IL-6 and tumor necrosis factor &alpha; (TNF-&alpha;) [@doi:10.1097/AIA.0b013e318034194e].
Anti-inflammatory cytokines play an immunoregulatory role complementary to the cascading effect of pro-inflammatory cytokines [@doi: 10.1128/MMBR.05015-11; @doi:10.1097/AIA.0b013e318034194e].
A number of interleukins and interferons play anti-inflammatory roles, and receptors or receptor antagonists for inflammatory cytokines are also important for regulating inflammation [@doi:10.1097/AIA.0b013e318034194e].
IL-10 is an anti-inflammatory cytokines of particular note because it regulates the expression of TNF-&alpha;, IL-1, and IL-6 [@doi:10.1097/AIA.0b013e318034194e].
When the pro- and anti-inflammatory responses are both commensurate with the threat posed, the immune system drives a shift back to homeostasis [@doi:10.1111/imr.12671].
However, when the responses are disproportionate, the cytokine response can become dysregulated.
Too low of an inflammatory response will not eliminate the immune threat [@doi:10.1111/imr.12671].
In contrast, if the response is dysregulated towards excessive pro-inflammatory cytokine activity, inflammation can cascade [@doi:10.1159/000087104] and cause cell damage, among other problems [@doi:10.18632/oncotarget.23208].
Elevated levels of inflammation over the long-term are associated with many chronic health conditions, including type 2 diabetes, dementia and Alzheimer's, and arthritis, among others [@doi:10.1038/s41591-019-0675-0].
On a shorter timescale, dysregulated systemic inflammation can cause sepsis, which can lead to multi-organ failure and death [@doi:10.1128/MMBR.05015-11; @pmid:22142805].

Cytokines have been investigated for their role in the immune response to lung infections long before the COVID-19 pandemic.
Dysregulation of the inflammatory response, including elevated levels of pro-inflammatory cytokines, is found in patients with ARDS, which is a severe condition that can arise from pneumonia, SARS, and COVID-19 [@doi:10.1159/000087104].
One study of patients with and at risk for ARDS (specifically, those who were intubated for medical ventilation) found that shortly after the onset of ARDS, anti-inflammatory cytokine concentration in BALF increased relative to the concentration of pro-inflammatory cytokines [@doi:10.1164/ajrccm.164.10.2104013].
However, patients with severe ARDS were excluded from this study.
The results suggest that an increase in pro-inflammatory cytokines such as IL-6 may signal the onset of ARDS, but recovery depends on an increased anti-inflammatory response [@doi:10.1164/ajrccm.164.10.2104013].
Acute phase response to an infection can also cause damage to the capillary endothelium, allowing leaks that disrupt the balance between pro-inflammatory cytokines and their regulators [@doi:10.1164/ajrccm.164.10.2104013].
Hyperactivity of the pro-inflammatory response due to lung infection is commonly associated with acute lung injury and more rarely with the more severe manifestation, ARDS [@doi:10.1128/MMBR.05015-11].
The heightened inflammatory response in the lungs can also serve as a source for systemic inflammation, or sepsis, which can lead to multi-organ failure [@doi:10.1128/MMBR.05015-11].
The shift from local to systemic inflammation is a phenomenon often referred to broadly as a cytokine storm [@doi:10.1128/MMBR.05015-11] or, more precisely, as cytokine release syndrome (CRS) [@doi:10.1186/s40425-018-0343-9].
Sepsis is a known possible complication of pneumonia, and in an analysis of over 1400 US pneumonia patients, IL-6, tumor necrosis factor (TNF), and IL-1&beta; were found to be elevated at intake in patients who developed severe sepsis and/or ultimately deceased [@doi:10.1001/archinte.167.15.1655].
IL-6 and TNF are pro-inflammatory cytokines, while IL-1&beta; is anti-inflammatory [@doi:10.1001/archinte.167.15.1655].
However, this study reported that unbalanced pro-/anti-inflammatory cytokine profiles were rare, although they measured only the three cytokines listed above.
Prior work therefore made it clear that pulmonary infection and injury were associated with systemic inflammation through sepsis.
While IL-6 is a biomarker sometimes used to assess cytokine storm activity in sepsis [@doi:10.1128/MMBR.05015-11], the relationship between cytokine profiles and the risks associated with sepsis may be more complex.
IL-6 is a pleiotropic cytokine that plays an integral role in both the inflammatory and anti-inflammatory responses and is associated with both healthy and pathological responses to viral threat [@doi:10.3389/fmicb.2019.01057].

The inflammatory response was identified early on as a potential driver of COVID-19 outcomes due to existing research in SARS and emerging research in COVID-19.
In addition to the known role of cytokines in ARDS and lung infection more broadly, immunohistological analysis at autopsy of patients who deceased from SARS revealed that ACE2-expressing cells that were infected by SARS-CoV showed elevated expression of IL-6, IL-1&beta;, and TNF-&alpha; [@doi:10.1002/path.2067].
Similarly, the introduction of the S-protein from SARS-CoV to mouse macrophages was found to increase production of IL-6 and TNF-&alpha; [@doi:10.1016/j.virusres.2007.02.007].
For SARS-CoV-2 infection leading to COVID-19, early reports described a cytokine storm-like response in patients with particularly severe infections [@doi:10.1101/2020.02.24.963348; @doi:10/ggnzmc; @doi:10.1016/j.immuni.2020.06.017].
Among patients hospitalized with COVID-19 in Wuhan, China, 112 out of 191 (59%) developed sepsis, including all 54 of the non-survivors [@doi:10/ggnxb3].
However, the argument has been made that while the cytokine levels observed in COVID-19 patients fall outside of the normal range, they are not be as high as typically found in patients with ARDS [@doi:10.1001/jamainternmed.2020.3313].
Regardless, inflammation has received significant interest both in regards to the pathology of COVID-19 as well as potential avenues for treatment, as the similarities between the cytokine storm and the pathophysiology of COVID-19 has led to the suggestion that a number of immunomodulatory pharmaceutical interventions could hold therapeutic value for the treatment of COVID-19 [@doi:10.1016/j.jaut.2020.102452].

#### Pediatric Presentation

The presentation of COVID-19 infection can vary greatly among pediatric patients and in some cases manifests in distinct ways from COVID-19 in adults.
A review examined symptoms reported in 17 studies of children infected with COVID-19 during the early months of the COVID-19 epidemic in China and one study from Singapore [@doi:10.1001/jamapediatrics.2020.1467].
Of the more than a thousand cases described, the most commonly reports were for mild symptoms such as fever, dry cough, fatigue, nasal congestion and/or runny nose, while three children were reported to be asymptomatic.
Severe lower respiratory infection was described in only one of the pediatric cases reviewed.
Gastrointestinal symptoms such as vomiting or diarrhea were occasionally reported.
Radiologic findings were not always reported in the case studies reviewed, but when they were mentioned they included bronchial thickening, ground-glass opacities, and/or inflammatory lesions [@doi:10.1001/jamapediatrics.2020.1467].
Neurological symptoms have also been reported [@doi:10.1001/jamaneurol.2020.2687].

These analyses indicate that most pediatric cases of COVID-19 are not severe.
However, serious complications and, in rare cases, deaths have occurred [@doi:10.1016/j.eclinm.2020.100433].
Of particular interest, children have occasionally experienced a serious inflammatory syndrome, multisystem inflammatory syndrome in children (MIS-C), following COVID-19 infection.
This syndrome is similar in some respects to Kawasaki disease or to Kawasaki disease shock syndrome [@doi:10.1093/jpids/piaa069; @doi:10.1001/jama.2020.10369; @doi:10.1016/j.ajem.2020.05.117] and is thought to be a distinct clinical manifestation of SARS-CoV-2 due to its distinct cytokine profile and the presence of burr cells in peripheral blood smears [@doi:10.1172/JCI140970; @doi:10.1016/j.cell.2020.09.016].
MIS-C has been associated with heart failure in some cases [@doi:10.1161/CIRCULATIONAHA.120.048360].
One case study [@doi:10/gg4sd6] described an adult who appeared to show symptoms similar to MIS-C after exposure to COVID-19, but cautioned against broad conclusions; a second possible adult case has also been reported [@doi:10.1016/j.ajem.2020.06.053].
The presentation of SARS-CoV-2 infection is therefore likely to be largely distinct between adult and pediatric populations.

### Systems-Level Effects

Systems biology provides a cross-disciplinary analytical paradigm through which the host response to an infection can be analyzed.
This field integrates the "omics" (genomics, transcriptomics, proteomics, metabolomics, etc.) with bioinformatics and other computational strategies.
These cutting-edge research approaches hold enormous potential for the study of the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004].
Over the last decade, systems biology approaches have been used widely to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].
Omics-based studies have also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].
Though the complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not yet fully understood, “omics” technologies offer the opportunity to discovery-drive analysis of biological changes associated with SARS-CoV-2 infection.
For example, previous studies suggest that infection by coronaviruses such as SARS-CoV and MERS-CoV and other viruses is associated with the upregulation of ACE2.
In several preliminary assays and an analysis of previous microarray data, ACE2 expression was reported to be significantly upregulated following infection of human embryonic kidney cells and human airway epithelial cells [@doi:10.1101/2020.02.24.963348].
This study also reported that direct stimulation with inflammatory cytokines such as type I interferons (e.g., IFN&beta;) resulted in the upregulation of ACE2 in human bronchial epithelial cells, with treated groups showing four-fold higher ACE2 expression than control groups at 18 hours post-treatment [@doi:10.1101/2020.02.24.963348].
Whether SARS-CoV-2 facilitates the positive regulation of its own transmission between host cells is still unclear, the host immune response itself likely plays a key role in mediating infection-associated pathologies.
A systems-biology approach allows for analyses such as these to identify possible phenotypic and endophenotypic responses to SARS-CoV-2 infection and to develop new hypotheses about how pathogenesis proceeds.

#### Transcriptomics

In addition to the study described above, two other studies have profiled expression following SARS-CoV-2 infection using human cell lines.
The first study [@doi:10.1016/j.cell.2020.04.026] compared transcriptional responses to SARS-CoV-2 and to other respiratory viruses, including MERS-CoV, SARS-CoV, human parainfluenza virus 3 (HPIV3), respiratory syncytial virus (RSV), and influenza A virus (IAV).
They analyzed the responses of three human cell lines: A549 (adenocarcinomic human alveolar basal epithelial cells), Calu-3 (human airway epithelial cells derived from human bronchial submucosal glands), and MRC-5 (human fetal lung fibroblast cells).
As the viral receptor ACE2 has low expression in A549 cells, they supplemented the A549 cells with adenovirus (AdV)-based vectors expressing either mCherry (a fluorescent protein used as a control) or ACE2 (A549-ACE2).
The authors also measured host transcriptional responses to SARS-CoV-2 in primary normal human bronchial epithelial cells (HBEC or NHBE cells), nasal washes from an animal model (ferret), and lung samples from two COVID-19 patients.
Differential expression (DE) analysis was then carried out to compare infected cells with control cells that underwent only a mock treatment.
In the hosts where SARS-CoV-2 was able to replicate efficiently, DE analysis revealed that the transcriptional response was significantly different from the response to all of the other viruses tested.
A unique proinflammatory cytokine signature associated with SARS-CoV-2 was present under in cells exposed to both high and low doses of the virus, with the cytokines IL-6 and IL1RA uniquely elevated in response to SARS-CoV-2 relative to other viruses.
However, the A549-ACE2 cells showed significant IFN-I or IFN-III expression when exposed to high, but not low, doses of SARS-CoV-2.
This finding suggests that IFN induction is dependent on the extent of exposure.
Similarly, in cells from the NHBE line, ferrets, and COVID-19 patients, chemokine signaling was significantly enriched, but there was no significant induction of IFN-I or IFN-III.
Together, these results suggest that SARS-CoV-2 induces a limited antiviral state with low IFN-I or IFN-III expression and a moderate IFN-stimulated gene response, in contrast to other viruses.
However, in ACE2-expressing A549 cells, this state could be overcome by using a 10-fold increase in SARS-CoV-2 exposure.
This finding suggests that the SARS-CoV-2 antagonist is insufficient for large doses of the virus [@doi:10.1016/j.cell.2020.04.026].
This hypothesis was further supported by a recent study [@doi:10.1101/2020.05.11.088179] that showed that the SARS-CoV-2 _ORF3b_ gene suppresses IFNB1 promoter activity (IFN-I induction) more efficiently than the SARS-CoV_ORF3b_ gene.
Taken together, these findings suggest that a unique cytokine profile is associated with the response to the SARS-CoV-2 virus, and that this response differs depending on the extent of exposure.

Another study [@doi:10.1101/2020.05.05.079194] analyzed cells' transcriptional response to SARS-CoV-2 and SARS-CoV over time.
They characterized the response of three human cell lines, H1299 (human non-small cell lung carcinoma cell line), Calu-3, and Caco-2 (human epithelial colorectal adenocarcinoma cell line), at 4 to 36 hpi.
Using poly(A) bulk RNA-seq, the authors found negligible susceptibility of H1299 cells (< 0.08 viral read percentage of total reads) compared to Caco-2 and Calu-3 cells (>10% of viral reads).
This finding suggests that the risk of infection varies among cell types, and that cell type could influence which hosts are more or less susceptible.
Based on visual inspection (microscopy images) and transcriptional profiling, the authors also showed distinct responses among the host cell lines evaluated.
In contrast to Caco-2, Calu-3 cells infected with SARS-CoV-2 showed signs of impaired growth and cell death at 24 hpi, as well as moderate IFN induction with a strong up-regulation of IFN-stimulated genes.
Interestingly, the results were similar to those reported in Calu-3 cells exposed to much higher levels of SARS-CoV-2 [@doi:10.1016/j.cell.2020.04.026], as described above.
This finding suggests that IFN induction in Calu-3 cells is not dependent on the level of exposure, in contrast to A549-ACE2 cells.
The discrepancy could be explained by the observations that Calu-3 cells are highly susceptible to SARS-CoV-2 and show rapid viral replication [@doi:10.1038/s41467-020-15562-9], whereas A549 cells are incompatible with SARS-CoV-2 infection [@doi:10.1101/2020.03.02.972935].
This discrepancy raises the concern that _in vitro_ models may vary in their similarity to the human response, underscoring the importance of follow-up studies in additional models.

#### Proteomics

No comprehensive proteomic analyses of the pathogen or of patients suffering from its infection have yet been reported.
One study investigated proteomics associated with _in vitro_ infection using Caco-2 cells infected with SARS-CoV-2 [@doi:10.1038/s41586-020-2332-7].
This study reported that SARS-CoV-2 induced alterations in multiple vital physiological pathways, including translation, splicing, carbon metabolism and nucleic acid metabolism in the host cells.
Another area of interest is whether SARS-CoV-2 is likely to induce similar changes to other hCoV.
For example, because of the high level of sequence homology between SARS-CoV-2 and SARS-CoV, it has been hypothesized that sera from convalescent SARS-CoV patients might show some efficacy in cross-neutralizing SARS-CoV-2-S-driven entry [@doi:10.1016/j.cell.2020.02.052].
However, despite the high level of sequence homology, certain protein structures might be immunologically distinct, which would be likely to prohibit effective cross-neutralization across different SARS species [@doi:10.1101/2020.03.21.990770].
Consequently, proteomic analyses of SARS-CoV might also provide some essential information regarding the new pathogen [@doi:10.1073/pnas.0407992101; @doi:10.1016/j.bbrc.2004.02.074].
Considering the paucity of omics-level big data sets for SARS-CoV-2 currently available, existing data hubs that contain information for other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), and The Virus Pathogen Resource (ViPR) will serve as useful resources for computational and bioinformatics research on SARS-CoV-2.
Using such databases, the systems-level reconstruction of the protein-protein interaction (PPI) will enable the generation of hypotheses about the mechanism of action of SARS-CoV-2 and suggest potential drug targets.
In an initial study [@doi:10.1101/2020.03.22.002386], 26 of the 29 SARS-CoV-2 proteins were cloned and expressed in HEK293T kidney cells, allowing for the identification of 332 high-confidence human proteins interacting with them.
Notably, this study suggested that SARS-CoV-2 interacts with innate immunity pathways.
Ranking pathogens by the similarity between their interactomes and that of SARS-CoV-2 suggested *West Nile Virus*, *Mycobacterium tuberculosis*, and *human papillomavirus* as the top three hits.
Therefore, given the lung symptoms associated with COVID-19, the *Mycobacterium tuberculosis* host-pathogen interactome in particular might provide new insights to the mechanism of SARS-CoV-2 infection.
Additionally, it was suggested that the envelope protein (E) could disrupt host bromodomain-containing proteins, i.e., BRD2 and BRD4, that bind to histones, and the spike protein could likely intervene in viral fusion by modulating the GOLGA7-ZDHHC5 acyl-transferase complex to increase palmitoylation, which is a post-translational modification that affects how proteins interact with membranes [@doi:10.3389/fimmu.2017.02003].
Another study [@doi:10.1101/2020.03.31.019216] used patient-derived peripheral blood mononuclear cells (PBMCs) to identify 251 host proteins targeted by SARS-CoV-2.
This study also reported that more than 200 host proteins were disrupted following infection.
In particular, a network analysis showed that non-structural proteins 9 and 10 (nsp9 and nsp10) interacted with NF-Kappa-B-Repressing Factor (NKRF), which encodes a transcriptional repressor that mediates repression of genes responsive to Nuclear Factor kappa-light-chain-enhancer of activated B-cells (NF-kB).
These genes are important to pro-, and potentially also anti-, inflammatory signaling [@doi:10.1101/cshperspect.a001651].
This finding could explain the exacerbation of the immune response that shapes the pathology and the high cytokine levels characteristic of COVID-19, possibly due to the chemotaxis of neutrophils mediated by IL-8 and IL-6.
Finally, it was suggested [@doi:10.1101/2020.04.09.033522] that the E protein of both SARS-CoV and SARS-CoV-2 has a conserved Bcl-2 Homology 3 (BH3)-like motif, which could inhibit anti-apoptosis proteins, e.g., BCL2, and trigger the apoptosis of T cells.
Several compounds are known to disrupt the host-pathogen protein interactome, largely through the inhibition of host proteins.
Therefore, preliminary research in the proteomics of SARS-CoV-2 infection suggests that drugs modulating the protein-level interactions between virus and host might be worth investigating.

### Viral Evolution and Virulence

Like that of SARS-CoV, the entry of SARS-CoV-2 into host cells is mediated by interactions between the viral spike glycoprotein (S) and human angiotensin-converting enzyme 2 (ACE2) [@doi:10.1126/science.1116480; @doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2762; @doi:10.1038/s41586-020-2179-y; @doi:10.1101/2020.02.19.956235; @doi:10.1016/j.cell.2020.02.052; @doi:10.1016/j.cell.2020.03.045; @doi:10.1128/jvi.00127-20].
Differences in how the S proteins of the two viruses interact with the human ACE2 (hACE2) receptor could also partially account for the increased transmissibility of SARS-CoV-2.
Recent studies have reported conflicting binding constants for the S-hACE2 interaction, though they have agreed that the SARS-CoV-2 S protein binds with equal, if not greater, affinity to the SARS-CoV S protein does [@doi:10.1016/j.cell.2020.02.058; @doi:10.1101/2020.02.19.956235; @doi:10.1126/science.abb2507].
The C-terminal domain of the SARS-CoV-2 S protein in particular was identified as the key region of the virus that interacts with hACE2, and the crystal structure of the C-terminal domain of the SARS-CoV-2 S protein in complex with hACE2 reveals stronger interaction and a higher affinity for receptor binding than that of SARS-CoV [@doi:10.1016/j.cell.2020.03.045].
Among the 14 key binding residues identified in the SARS-CoV S protein, 8 are conserved in SARS-CoV-2, and the remaining 6 are semi-conservatively substituted, potentially explaining variation in binding affinity [@doi:10.1016/j.cell.2020.02.058; @doi:10.1101/2020.02.19.956235].
Recent crystal structures have shown that the receptor-binding domain (RBD) of the SARS-CoV-2 S protein, like that of other coronaviruses, undergoes stochastic hinge-like movement that flips it from a “closed” conformation, in which key binding residues are hidden at the interface between protomers, to an “open” one [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507].
Because the RBD plays such a critical role in viral entry, blocking its interaction with ACE2 represents a promising therapeutic approach.
Nevertheless, despite the high structural homology between SARS-CoV-2 RBD and that of SARS-CoV, monoclonal antibodies targeting SARS-CoV-RBD failed to bind to SARS-CoV-2-RBD [@doi:10.1126/science.abb2507].
Promisingly, though, sera from convalescent SARS patients inhibited SARS-CoV-2 viral entry _in vitro_, albeit with lower efficiency than it inhibited SARS-CoV [@doi:10.1016/j.cell.2020.02.052].

Comparative genomic analysis reveals that several regions of the coronavirus genome are likely critical to virulence.
The S1 domain of the spike protein, which contains the RBM, evolves more rapidly than _S_'s S2 domain [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009].
However, even within the S1 domain, some regions are more conserved than others, with the receptors in S1's N-terminal domain (S1-NTD) evolving more rapidly than those in its C-terminal domain (S1-CTD) [@doi:10.1016/j.jmb.2020.04.009].
Both S1-NTD and S1-CTD are involved in receptor binding and can function as RBDs to bind proteins and sugars [@doi:10.3390/v4061011], but RBDs in the S1-NTD typically bind to sugars, while those in the S1-CTD recognize protein receptors [@doi:10.1146/annurev-virology-110615-042301].
Viral receptors show higher affinity with protein receptors than sugar receptors [@doi:10.1146/annurev-virology-110615-042301], which suggests that positive selection on or relaxed conservation of the S1-NTD might reduce the risk of a deleterious mutation that would prevent binding.
The SARS-CoV-2 S protein also contains a an RRAR furin recognition site at the S1/S2 junction [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507], setting it apart from both bat coronavirus RaTG13, with which it shares 96% genome sequence identity, and SARS-CoV [@doi:10.1038/s41586-020-2012-7].
Such furin cleavage sites are commonly found in highly virulent influenza viruses, and as such may contribute to the heightened pathogenicity of SARS-CoV-2 [@doi:10/bvgh5b; @doi:10.1006/viro.1999.9716].
Effective cell entry is a critical component to pathogenesis and therefore an important process to understand when examining possible therapeutics.

Evolution in SARS-CoV-2 has also been observed over a short timescale.
After zoonotic transfer, SARS-CoV-2 continued evolving in the human population [@doi:10.1186/s12967-020-02344-6].
The SARS-CoV-2 mutation rate is moderate compared to other RNA viruses [@doi:10.1016/j.meegid.2020.104351], which likely restricts the pace of evolution in SARS-CoV-2.
Nevertheless, genomic analyses have yielded statistical evidence of ongoing evolution.
There are two known variants of the spike protein that differ by a single amino acid at position 614 (G614 and D614), and there is evidence that G614 had become more prevalent than D614 by June 2020 [@doi:10.1016/j.cell.2020.06.043].
While there is a hypothesis that this genomic change increased the SARS-CoV-2 infectivity and virulence, this hypothesis has not yet been tested due to a lack of data [@doi:10.1016/j.cell.2020.06.040].
Another study [@doi:10.1016/j.meegid.2020.104351] identified 198 recurrent mutations in a dataset of 7666 curated sequences.
This pattern of convergent evolution at some sites could indicate that certain mutations confer an adaptive advantage.
While it is evident that SARS-CoV-2 exhibits moderate potential for ongoing and future evolution, the relationship between mutations and pathogenicity is not yet known.
Additional data is needed in order to understand patterns of evolutionary change and whether they are likely to affect virulence.
