## Evolutionary Perspectives on SARS-CoV-2

### Abstract

### Importance

### Introduction

The emergence of what is now known to be the pathogen SARS-CoV-2 has dramatically reshaped modern life for the past two years.
The genomic revolution provided the tools needed to understand the virus in ways that were not feasible during previous pandemics.
For example, the first genome sequence of the pathogen was released on January 3, 2020, providing valuable information about the pathogen within a month and a half of the first known cases.
As the pandemic has unfolded, evolutionary questions and methods of investigation have framed the scientific approach to understanding the virus.
These questions have evolved along with the pandemic.
Thus far, four major evolutionary questions have emerged.<!--The Who does it affect question will be one we have to be careful to avoid overlap with pathogenesis paper? Also is there a fifth question of what is next? We have never deeply sequenced a virus like this before... viral surveillance that has never been achieved. Implications for preventing future pandemics?-->
The first was "what is it?", the second "where did it come from?", the third "whom does it affect?" and the fourth "how is it changing?"
Evolutionary biology provides a framework through which these questions can be evaluated and explored.

### Question 1: What Is It?

What is now known as SARS-CoV-2 emerged in November 2019 as an unknown pathogen causing a cluster of pneumonia cases in Wuhan, China.
The initial genome sequence, which was released in early January 2020, revealed the pathogen to be a novel coronavirus [@doi:10.46234/ccdcw2020.017].
Coronaviruses have long been known to infect animals and have been the subject of veterinary medical investigations and vaccine development efforts due to their effect on the health of companion and agricultural animals [@doi:10/ckfn8b].
Although most coronaviruses show little transmission in humans, several human coronaviruses (HCoV) have been identified since the 1960s.
Therefore, in the early days of the pandemic, many strategies to understand or manage the emergent viral threat focused on contextualizing it amongst better-studied coronaviruses.

Many people have previously been infected by an HCoV.
Approximately one-third of common cold infections are thought to be caused by four seasonal HCoV: _Human coronavirus 229E_ (HCoV-229E), _Human coronavirus NL63_ (HCoV-NL63), _Human coronavirus OC43_ (HCoV-OC43), and _Human coronavirus HKU1_ (HCoV-HKU1) [@doi:10.1038/s41579-018-0118-9; @doi:10.3390/diseases4030026; @doi:10.1016/j.jcv.2018.01.019].
The first HCoV were identified in the 1960s: HCoV-229E in 1965 [@doi:10.3181/00379727-121-30734] and HCoV-OC43 in 1967 [@doi:10.1073/pnas.57.4.933].
Both of these viruses typically cause cold-like symptoms, including upper and lower respiratory infections [@doi:10.1186/1743-422X-1-7; @pmid:20700397; @doi:10.1186/s40779-020-00279-z], but they have also been associated with gastrointestinal symptoms [@doi:10.1111/j.1440-1754.2007.01246.x].
Two additional HCoV were subsequently identified [@doi:10.1038/nm1024; @doi:10.1128/JVI.79.2.884-895.2005].
In 2003, HCoV-NL63 [@doi:10.1038/nm1024] was first identified in a 7-month-old infant and then in clinical specimens collected from seven additional patients, five of whom were infants younger than 1 year old and the remainder of whom were adults.
CoV-HKU1 was identified in samples collected from a 71-year-old pneumonia patient in 2004 and then found in samples collected from a second adult patient [@doi:10.1128/JVI.79.2.884-895.2005].
These viruses are associated with respiratory diseases of varying severity, ranging from common cold to severe pneumonia, with severe symptoms mostly observed in immunocompromised individuals [@doi:10.1086/377612], and also have gastrointestinal involvement in some cases [@doi:10.1111/j.1440-1754.2007.01246.x].

In addition to these relatively mild HCoV, however, highly pathogenic human coronaviruses have been identified, including _Severe acute respiratory syndrome-related coronavirus_ (SARS-CoV or SARS-CoV-1) and _Middle East respiratory syndrome-related coronavirus_ (MERS-CoV) [@doi:10.1038/nrmicro.2016.81; @doi:10.1038/s41579-018-0118-9; @doi:10.1016/bs.aivir.2018.01.001].
At the time that SARS-CoV-1 emerged in the early 2000s, no HCoV had been identified in almost 40 years [@doi:10.1038/nrmicro.2016.81].
The first case of SARS was reported in November 2002 in the Guangdong Province of China, and over the following month, the disease spread more widely within China and then into several countries across multiple continents [@doi:10.1093/ajcp/aqaa029; @doi:10.1038/nrmicro.2016.81].
Unlike previously identified HCoV, SARS was much more severe, with an estimated death rate of 9.5% [@doi:10.1093/ajcp/aqaa029].
It was also highly contagious via droplet transmission, with a basic reproduction number (R~0~) of 4 (i.e., each person infected was estimated to infect four other people) [@doi:10.1093/ajcp/aqaa029].

However, the identity of the virus behind the infection remained unknown until April of 2003, when the SARS-CoV-1 virus was identified through a worldwide scientific effort spearheaded by the WHO [@doi:10.1038/nrmicro.2016.81].
SARS-CoV-1 belonged to a distinct lineage from the two other HCoV known at the time [@doi:10.1093/ajcp/aqaa029].
By July 2003, the SARS outbreak was officially determined to be under control, with the success credited to infection management practices [@doi:10.1038/nrmicro.2016.81].
A decade later, a second outbreak of severe respiratory illness associated with a coronavirus emerged, this time in the Arabian Peninsula.
This disease, known as Middle East respiratory syndrome (MERS), was linked to another novel coronavirus, MERS-CoV.
The fatality rate associated with MERS is much higher than that of SARS, at almost 35%, but the disease is much less easily transmitted, with an R~0~ of 1 [@doi:10.1093/ajcp/aqaa029].
Although MERS is still circulating, its low reproduction number has allowed for its spread to be contained [@doi:10.1093/ajcp/aqaa029].
The COVID-19 pandemic is thus associated with the seventh HCoV to be identified and the fifth since the turn of the millennium, though additional HCoVs may be in circulation but remain undetected (e.g., [@doi:10.1093/cid/ciab456]).

Following the release of the SARS-CoV-2 genome sequence, multiple research groups sequenced the genomes of SARS-CoV-2 specimens identified in clinical samples.
These samples were primarily collected from patients' lower respiratory tract, namely bronchoalveolar lavage fluid (BALF), and the upper respiratory tract, in the form of throat and nasopharyngeal swabs [@doi:10.1038/s41586-020-2008-3; @doi:10.1038/s41586-020-2012-7; @doi:10/ggjr43].
Integration of these sequences allowed for a more complete picture of the viral genome.
Analysis of the viral genome revealed significant sequence homology with two known HCoV: the novel coronavirus shared about 79% sequence identity with SARS-CoV-1 and 50% with MERS-CoV [@doi:10/ggjr43].
Therefore, this early phylogenetic analysis of the novel coronavirus allow its similarity to other, known viruses to be established. 
SARS-CoV-1 and MERS-CoV were ultimately managed largely through infection management practices (e.g., mask wearing) and properties of the virus itself (i.e., low rate of transmission), respectively [@doi:10.1038/nrmicro.2016.81; @doi:10.1093/ajcp/aqaa029].
Research in response to prior outbreaks of HCoV-borne infections, such as SARS and MERS, provided a strong foundation for hypotheses about the pathogenesis of SARS-CoV-2 as well as potential diagnostic and therapeutic approaches, as we review elsewhere [@individual-pathogenesis; @individual-pharmaceuticals; @individual-vaccines].
Therefore, this phylogenetic information was valuable for gaining an understanding of the pathogen and identifying strategies to manage it.

### Coronaviruses and Animal Hosts

Coronaviruses have long been known to infect animals and have been the subject of veterinary medical investigations and vaccine development efforts due to their effect on the health of companion and agricultural animals [@doi:10/ckfn8b].
<!-- TO DO: Expand on the relationship between coronaviruses and animals -- which taxa are most often affected? how severe are veterinary infections? how has vaccine development worked for these diseases?--> 

<!-- TO DO: Be care to check overlap between this section and 20.pharmaceuticals.md. Trim to address just the important information about human-coronavirus interactions that feeds into an evolutionary perspective of how these host/virus interactions evolve across taxa and over time -->
Most coronaviruses show little to no transmission in humans.
However, it is thought that approximately one-third of common cold infections are caused by four seasonal human coronaviruses (HCoV): _Human coronavirus 229E_ (HCoV-229E), _Human coronavirus NL63_ (HCoV-NL63), _Human coronavirus OC43_ (HCoV-OC43), and _Human coronavirus HKU1_ (HCoV-HKU1) [@doi:10.1038/s41579-018-0118-9; @doi:10.3390/diseases4030026; @doi:10.1016/j.jcv.2018.01.019].
The first HCoV were identified in the 1960s: HCoV-229E in 1965 [@doi:10.3181/00379727-121-30734] and HCoV-OC43 in 1967 [@doi:10.1073/pnas.57.4.933].
Both of these viruses typically cause cold-like symptoms, including upper and lower respiratory infections [@doi:10.1186/1743-422X-1-7; @pmid:20700397; @doi:10.1186/s40779-020-00279-z], but they have also been associated with gastrointestinal symptoms [@doi:10.1111/j.1440-1754.2007.01246.x].
Two additional HCoV were subsequently identified [@doi:10.1038/nm1024; @doi:10.1128/JVI.79.2.884-895.2005].
In 2003, HCoV-NL63 [@doi:10.1038/nm1024] was first identified in a 7-month-old infant and then in clinical specimens collected from seven additional patients, five of whom were infants younger than 1 year old and the remainder of whom were adults.
CoV-HKU1 was identified in samples collected from a 71-year-old pneumonia patient in 2004 and then found in samples collected from a second adult patient [@doi:10.1128/JVI.79.2.884-895.2005].
These viruses are associated with respiratory diseases of varying severity, ranging from common cold to severe pneumonia, with severe symptoms mostly observed in immunocompromised individuals [@doi:10.1086/377612], and also have gastrointestinal involvement in some cases [@doi:10.1111/j.1440-1754.2007.01246.x].
In addition to these relatively mild HCoV, however, highly pathogenic human coronaviruses have been identified, including _Severe acute respiratory syndrome-related coronavirus_ (SARS-CoV or SARS-CoV-1) and _Middle East respiratory syndrome-related coronavirus_ (MERS-CoV) [@doi:10.1038/nrmicro.2016.81; @doi:10.1038/s41579-018-0118-9; @doi:10.1016/bs.aivir.2018.01.001].

At the time that SARS-CoV-1 emerged in the early 2000s, no HCoV had been identified in almost 40 years [@doi:10.1038/nrmicro.2016.81].
The first case of SARS was reported in November 2002 in the Guangdong Province of China, and over the following month, the disease spread more widely within China and then into several countries across multiple continents [@doi:10.1093/ajcp/aqaa029; @doi:10.1038/nrmicro.2016.81].
Unlike previously identified HCoV, SARS was much more severe, with an estimated death rate of 9.5% [@doi:10.1093/ajcp/aqaa029].
It was also highly contagious via droplet transmission, with a basic reproduction number (R~0~) of 4 (i.e., each person infected was estimated to infect four other people) [@doi:10.1093/ajcp/aqaa029].
However, the identity of the virus behind the infection remained unknown until April of 2003, when the SARS-CoV-1 virus was identified through a worldwide scientific effort spearheaded by the WHO [@doi:10.1038/nrmicro.2016.81].
SARS-CoV-1 belonged to a distinct lineage from the two other HCoV known at the time [@doi:10.1093/ajcp/aqaa029].
By July 2003, the SARS outbreak was officially determined to be under control, with the success credited to infection management practices [@doi:10.1038/nrmicro.2016.81].
A decade later, a second outbreak of severe respiratory illness associated with a coronavirus emerged, this time in the Arabian Peninsula.
This disease, known as Middle East respiratory syndrome (MERS), was linked to another novel coronavirus, MERS-CoV.
The fatality rate associated with MERS is much higher than that of SARS, at almost 35%, but the disease is much less easily transmitted, with an R~0~ of 1 [@doi:10.1093/ajcp/aqaa029].
Although MERS is still circulating, its low reproduction number has allowed for its spread to be contained [@doi:10.1093/ajcp/aqaa029].
The COVID-19 pandemic is thus associated with the seventh HCoV to be identified and the fifth since the turn of the millennium, though additional HCoVs may be in circulation but remain undetected.  
Possible evidence for a rare eighth HCoV was later reported by [@doi:10.1093/cid/ciab456].

SARS-CoV-1 and MERS-CoV were ultimately managed largely through infection management practices (e.g., mask wearing) and properties of the virus itself (i.e., low rate of transmission), respectively [@doi:10.1038/nrmicro.2016.81; @doi:10.1093/ajcp/aqaa029].
Vaccines were not used to control either outbreak, although vaccine development programs were established for SARS-CoV-1 [@doi:10.3390/v11010059].
In general, care for SARS and MERS patients focuses on supportive care and symptom management [@doi:10.1093/ajcp/aqaa029].
Clinical treatments for SARS and MERS developed during the outbreaks generally do not have strong evidence supporting their use.
Common treatments included Ribavirin, an antiviral, often in combination with corticosteroids or sometimes interferon (IFN) medications, which would both be expected to have immunomodulatory effects [@doi:10.1038/nrmicro.2016.81].
However, retrospective and _in vitro_ analyses have reported inconclusive results of these treatments on SARS and the SARS-CoV-1 virus, respectively [@doi:10.1038/nrmicro.2016.81].
IFNs and Ribavirin have shown promise in _in vitro_ analyses of MERS, but their clinical effectiveness remains unknown [@doi:10.1038/nrmicro.2016.81].
Therefore, only limited strategies can be adopted for the pharmaceutical management of COVID-19 from previous severe HCoV infections.
Research in response to prior outbreaks of HCoV-borne infections, such as SARS and MERS, have, however, provided a strong foundation for hypotheses about the pathogenesis of SARS-CoV-2 as well as potential diagnostic and therapeutic approaches.

### Question 2: Where Did It Come From?

Despite the high degree of similarity to SARS-CoV-1, even greater sequence identity was observed between SARS-CoV-2 and zoonotic coronaviruses.
A 2001 literature review estimated that 61% of human pathogens have a zoonotic origin [@doi:10.1098/rstb.2001.0888].
A zoonotic disease, or zoonosis, arises when a pathogen can both a) infect and b) cause a disease in humans [@doi:10.1016/j.pt.2016.04.007].
As a result, the risk of zoonotic disease increases when there is substantial interaction between humans and wildlife [@doi:10.1016/j.pt.2016.04.007].
Many factors can influence this human/wildlife interface and therefore the risk of zoonotic transmission events [@doi:10.1016/j.pt.2016.04.007; @doi:10.3201/eid1112.040789].

In the SARS epidemic, SARS-CoV-1 was also thought to have emerged in a live animal market.
A survey of a market in Shenzen, China revealed that individuals from two carnivore species, namely several masked palm civets (_Paguma larvata_) and one racoon dog (_Nyctereutes procyonoides_), were likely carriers of SARS-CoV-1, despite presenting as healthy [@doi:10.1126/science.1087139].
However, further analysis suggested that these species might be only intermediate hosts who were exposed in the market setting [@doi:10.7150/ijbs.45472].
A closely related virus was identified in Chinese horseshoe bats (_Rhinolophus sinicus_), but the sequence identity was only 88% with SARS-CoV-1 [@doi:10.1073/pnas.0506735102].
Therefore, the species of origin for SARS-CoV-1 remains unresolved.

In the case of SARS-CoV-2, early interest for the emergence of the pathogen turned to live-animal markets in Wuhan [@doi:10.1038/d41586-020-00548-w; @url:https://www.thestandard.com.hk/sections-news-print/215457/Ill-husband-and-wife--add-to-Wuhan-riddle], where it would later emerge that many animals were sold suffering from poor health and hygiene [@doi:10.1038/s41598-021-91470-2].
A large percentage of early patients had visited the Huanan seafood market in Wuhan, and next-generation sequencing of samples collected from nine patients, eight of whom had visited the market, revealed extremely high sequence identity (99.98%), indicative of rapid spread [@doi:10/ggjr43].
The sequence of the viral pathogen collected from these patients was also compared to known zoonotic pathogens.
In particular, genomic research quickly highlighted significant similarity (about 88% sequence identity) between SARS-CoV-2 and bat-derived SARS-like coronaviruses, namely bat-SL-CoVZC45 and bat-SL-CoVZXC21 [@doi:10/ggjr43].
Other analyses have reported even greater similarity between SARS-CoV-2 and the bat coronavirus BatCoV-RaTG13, with shared sequence identity as high as 96.2% [@doi:10.1038/s41586-020-2012-7; @doi:10.1093/nsr/nwaa036].
Bats are well-established as a disease reservoir, including for RNA viruses [@doi:10.1016/j.tim.2014.12.004; @doi:10.1016/j.virusres.2015.05.006; @doi:10.1128/CMR.00017-06].
This evidence therefore suggested that the virus may have emerged as a result of zoonotic transfer of a virus from bats to humans, with the wildlife trade considered a potential source of exposure.

Nevertheless, some fragments of the genome differ between SARS-CoV-2 and RATG13 by up to 17%, suggesting a complex natural selection process.
Additionally, SARS-CoV-2 is closely related (91.02%) to a novel coronaviruses identified in Malayan pangolins (_Manis javanica_) infected with a respiratory disease in October 2019 [@doi:10.1016/j.cub.2020.03.022].
Although the genome-wide sequence identity was lower between SARS-CoV-2 and this pangolin virus than BatCoV-RaTG13, its particularly high similarity in the receptor binding domain (RBD) of the spike (_S_) gene with SARS-CoV-2 drew further attention [@doi:10.1038/s41586-020-2169-0; @doi:10.1016/j.cub.2020.03.022].
The SARS-CoV-2 RBD differs from the pangolin coronavirus RBD by only one amino acid change [@doi:10.1016/j.cub.2020.03.022], and the sequence identity between the regions is 97.4% [@doi:10.1038/s41586-020-2169-0].
Pangolins were therefore identified as a potential intermediate host of SARS-CoV-2 between bats and humans.

However, data collected from May 2017 to November 2019 by a research team interested in tick-borne illnesses identified no bats or pangolins sold at these markets leading up to the emergence of COVID-19 [@doi:10.1038/s41598-021-91470-2].
Additionally, endemic bat species are typically in hibernation at the time that SARS-CoV-2 emerged [@doi:10/ggjr43].
Therefore, it is possible that animals associated with these markets were infected by bats, but it is not clear whether the disease emerged in a different location and/or whether it is associated with a different species.
There were 38 species observed at the market in the 2.5 years leading up to the emergence of SARS-CoV-2, indiciating significant diversity in the animals with which humans were interacting [@doi:10.1038/s41598-021-91470-2].
As with SARS-CoV-1, the species of origin for SARS-CoV-2 therefore remains unresolved.

Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-2 is unlikely to have originated in a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi:10.1038/s41591-020-0820-9].
However, potentially due to public misunderstanding about recombination and complex evolutionary processes like coevolution, the similarity to pangolin _S_ has resulted in popular conspiracy theories that the virus did not arise naturally.
The similarity of _S_ to that of pangolin viruses could arise from either recombination or coevolution [@doi:10.1093/nsr/nwaa036; @doi:10.1126/sciadv.abb9153], rather than requiring human intervention.
Such suspicions may also have been fueled, in part, by the lack of well-characterized bat coronaviruses which means that SARS-CoV-2 is still relatively derived from documented coronaviruses surveyed in bats [@doi:10.1016/j.tree.2020.12.002].
This ambiguity therefore serves to highlight the value of monitoring zoonotic diseases, as more robust reference panels against which to compare SARS-CoV-2 would be valuable in pinpointing its origin.
More importantly, it would allow researchers to have a better understanding of and to communite concerns about potential emerging viral threats.

### Question 3: Who is Affected?

#### Genetic Contributors to Susceptibility

Throughout the pandemic, many hypotheses have been raised about factors that might influence individuals' susceptibility to COVID-19 or to severe disease .
Many risk factors, such as underlying health conditions, are related to the body's inflammatory response, as we review elsewhere [@individual-inequality].
Here, we focus narrowly on genetic bases of differences in susceptibility or outcomes.
Historically, the identification of genetic risk factors for a disease typically utilized a candidate gene approach, where a gene of interest was evaluated to identify variants that showed an association with the outcome of interest.
While economical in terms of sequencing, this approach is prone to spurious results when applied to complex traits [@doi:10.1016/j.biopsych.2006.11.010].
Today, in the age of next-generation sequencing (NGS), alternative approaches have emerged.
NGS makes it possible to conduct genome-wide scans where a large number of single-nucleotide polymorphisms (SNPs) or variants are evaluated to identify regions of the genome associated with variation in a phenotype.
Genome-wide association studies (GWAS) in particular are a popular approach that employs this strategy.
During COVID-19, both of these paradigms have been applied to the problem of identifying genetic correlates of disease severity.

##### Candidate-Gene Approaches

Many candidate genes have been investigated throughout the pandemic.
Here, we review three examples of candidate gene studies in COVID-19.
First, an early study (published in April 2020) investigated a known variant in interferon-induced transmembrane protein 3 (_IFITM3_) among hospitalized patients in Beijing [@doi:10.1093/infdis/jiaa224].
This gene and variant were selected because of a prior candidate gene study by some of the same authors that found an association with influenza severity among Chinese patients during the 2009 influenza A H1N1/09 pandemic [@doi:10.1038/ncomms2433].
Here, they evaluated a small number (n=80) hospitalized COVID-19 patients to determine whether homozygosity for the previously identified risk allele was associated with mild versus severe disease [@doi:10.1093/infdis/jiaa224].
They stated that they found an association between homozygosity for the SNP of interest and the severity of COVID-19.<!--Note: in-line stats differ from those in the associated table-->
A follow-up study demonstrated worldwide variation in the frequency of these SNPs [@doi:10.1007/s00251-020-01188-0], and subsequent studies claimed to support this result by comparing the frequency of the SNP in different groups to the COVID-19 case fatality rate in those groups; they examined SNPs in several candidate genes and identified an association with another SNP in _IFITM3_ [@doi:10.3390/genes12010042].
However, in the original study, the population-level frequency of the risk allele was consistent with its frequency in the mild population [@doi:10.1038/ncomms2433].
A similar analysis examined both SNPs in Britons of different ancestral backgrounds and also reported a correlation [@doi:10.7717/peerj.10402].
While this gene has been investigated for functions potentially relevant to COVID-19 pathogenesis by other groups as well (e.g., [@doi:10.15252/embj.2020106501; @doi:10.1080/22221751.2020.1787797], a follow-up analysis in Germany evaluated the effect of in 239 cases and 252 controls and reported non-significant effects [@doi:10.7717/peerj.10402].
The narrative surrounding _IFITM3_ therefore reflects a broad methodological critique about candidate gene studies, where results often fail to replicate [@doi:10.1038/nrg796].
The region associated with this gene was not identified in the large-scale GWAS conducted by the COVID-19 Host Genetics Initiative (COVID-19 HGI) [@doi:10.1038/s41586-021-03767-x], which is described in more detail below.

A second source of genetic variability that was hypothesized to have an effect on COVID-19 outcomes were human leukocyte antigens (HLA), or the major histocompatibility complex (MHC).
Both MHC classes I and II play a critical role in both the innate and adaptive immune system because they are a pivotal component of antigen presentation.
HLA classes I and II are also the most polymorphic loci in the human genome [@doi:10.1111/j.1399-0039.2012.01881.x].
Additionally, because HLA polymorphisms are associated with geographic ancestry, study location and participant background offers important context [@doi:10.1371/journal.pone.0097282].
Given the important role of the HLA complex in the immune response and the standing variation in the human population, HLA variation has been investigated for potential associations with COVID-19 outcomes.

Several approaches have been taken to evaluate a potential role of HLA in COVID-19.
_In silico_ analysis suggested one particular HLA locus that could affect binding of SARS-CoV-2 peptides to MHC class II [@doi:10.1016/j.ijid.2020.07.016].
Other studies evaluated outcomes using retrospective cohort analyses.
An analysis of 95 South Asian COVID-19 patients found that HLA genotype was not significant in differentiating case severity when the necessary statistical corrections were applied [@doi:10.1002/jmv.27003].<!--risk:HLA-B*51, protect: HLA-B*35-->
Another study in a European population (n =147) did identify HLA alleles associated with severity [@doi:10.1111/tan.14284].
In St. Louis, MO (USA), another study enrolled 234 COVID-19 cases, who were genotyped for HLA alleles and compared to a control population of 20,000 individuals from the National Marrow Donor Program [@doi:10.1111/tan.14387].
They compared cases and controls on the basis of four "race/ethnic" populations and reported alleles showing a statistical association within each group [@doi:10.1111/tan.14387].
However, because of this stratification, two of the demographic categories had less than ten cases.
Across all of these studies, there was minimal overlap in the risk alleles identified, and the small sample sizes raise concerns about the possibility for spurious hits.
The hypervariability of this region means that statistical power will necessarily be reduced, with much higher recruitment needed than for studies of biallelic loci.
A much larger analysis of 72,912 Israelis, 8.8% of whom tested positive for COVID-19, found no association between HLA genotype and infection or hospitalization [@doi:10.1007/s10875-021-01071-x].
Therefore, while MHC is functionally important to the immune response to COVID-19, it is not clear whether HLA genotypes are predictive of COVID-19 severity, and certainly such studies face exacerbated versions of the typical challenges of candidate gene studies.
Because of the challenges associated with analyzing such a variable region, it was excluded from the large-scale COVID-19 HGI GWAS analysis [@doi:10.1038/s41586-021-03767-x].

Finally, significant attention has been paid to the question of whether ABO blood type is associated with COVID-19 outcomes.<!--This paper is interesting but I'm not totally sure how to incorporate it 10.1371/journal.pone.0251535-->
ABO blood type has been found to modulate susceptibility to other pathogens [@doi:10.1001/jama.2020.16516].
While ABO blood type is a genetic trait, it is more easily evaluated than the genetic regions discussed above because of the simple relationship between genetic variants and phenotype.
The possibility for an association between blood type and COVID-19 infection was raised early in the pandemic in a preprint that reported associations in 2,173 patients in Wuhan and Shenzhen, China [@doi:10.1101/2020.03.11.20031096].
The protective effect of O and increased risk associated with A blood types that they reported was subsequently investigated by many studies that returned varied results (e.g., [@doi:10.1002/jmv.27444; @doi:10.3389/fcimb.2020.00404; @doi:10.1182/bloodadvances.2020002623; @doi:10.1001/jamanetworkopen.2021.7429]; see [@doi:10.1007/s00277-021-04489-w] for a literature review).
Observations of higher and lower risk, respectively, of SARS-CoV-2 infection with A and O blood types was supported by a meta-analysis [@doi:10.1016/j.meegid.2020.104485].
While the support for the association was independent of a mechanism, a possible relationship between ACE activity and blood type has been proposed [@doi:10.1177/2047487320922370] as has an effect on carbohydrate-carbohydrate interactions relevant to ACE2 binding [@doi:10.1016/j.mehy.2020.110155].
This is the only candidate gene described that has received additional support from GWAS, as is discussed below.

The COVID-19 literature related to candidate gene investigations demonstrates relatively low inter-study consistency in findings.
In particular, sample size is a major challenge in designing these studies.
However, for many traits, the relationships between genes and phenotypes are complex, and selecting which variants to sequence is not always straightforward.
As a result, in the age of next-generation sequencing, discovery-driven studies have emerged as an alternative approach.

##### Genome-Wide Association Studies

Genome-wide association studies (GWAS) offers a discovery-driven approach that provides a different perspective than candidate gene studies.
Instead of selecting a gene or variant _a priori_, in GWAS, a large number of SNPs (usually several million) are evaluated at once to identify those most likely to vary in correlation with a trait of interest.
Because of the large number of statistical tests, statistical power and multiple hypothesis testing are both very important considerations in executing GWAS, which have also struggled with issues related to replicability [@doi:10.1016/j.tig.2018.03.005].
In cases such as COVID-19 where outcomes can differ among ancestry groups (likely for non-genetic reasons, as reviewed in [@individual-inequality]), it is especially important that GWAS samples be selected with attention paid to ancestry, as incorrect or misleading associations can otherwise be identified with neutral markers indicative of ancestry itself [@doi:10.1038/nrg2813].

Over the past two years, many GWAS have been undertaken with the aim of identifying variants associated with COVID-19 outcomes.
In some cases, the results have been consistent with hypothesized genetic correlates of susceptibility to COVID-19.
One study conducted a GWAS on a total of 435 COVID-19 patients from four countries and identified another HLA allele to be associated with an increased risk of intubation [@doi:10.1016/j.eclinm.2021.101099].<!--HLA-C*04:01-->
Other GWAS have identified an association with the ABO blood group locus.
One conducted a case/control GWAS in two populations, Italians and Spaniards, with 1980 cases and 2205 controls.
They reported two loci that met the genome-wide significance threshold, one on chromosome 3 and one on chromosome 9 [@doi:10.1056/NEJMoa2020283].
The hit on chromosome 9 fell on the ABO locus and the alleles identified suggested a protective association with blood group O and a risk association with blood group A [@doi:10.1056/NEJMoa2020283].

As the pandemic has progressed, large-scale efforts have been assembled to conduct GWAS on massive scales.
In March 2020, COVID-19 HGI was established as a world-wide consortium that combines data to conduct meta-analyses [@doi:10.1038/s41431-020-0636-6].
One year later, COVID-19 HGI released a meta-analysis of data from 46 studies, comprising 49,562 cases and 1,770,206 controls [@doi:10.1038/s41586-021-03767-x].
They identified 13 loci, seven of which were significant at the genome-wide level when considering all data available, that were associated with one or more phenotypes related to COVID-19 infection or severity.
Notably, strong signals were identified for both of the loci suggested by previous medium-scale GWAS in association with COVID-19 infection [@doi:10.1056/NEJMoa2020283].
Additionally, several other loci could be mapped onto hypotheses about genetic contributors to immune function, lung function and disease.
This world-wide GWAS study made an effort towards strategic incorporation of genetic information from different ancestral groups.
Interestingly, the risk variant on chromosome 3 is likely to be inherited from Neanderthal introgression, meaning it is likely to be more prevalent in certain populations, especially non-African populations [@doi:10.1038/s41586-020-2818-3; @doi:10.1093/molbev/mst260].
The potential functional relationship between this region of the genome and COVID-19 is unknown, but phenome-wide association study has suggested blood cell traits as a potential trait regulated by this region [@doi:10.1093/gerona/glab035].

Identifying genetic variants associated with a complex disease is always complicated.
In COVID-19 studies, the results of candidate gene analyses have in general been difficult to replicate.
However, large-scale collaboration on GWAS has made it possible to detect at least two loci that do appear to replicate across studies and potentially even across ancestral backgrounds.

### Evolution of the SARS-CoV-2 Virus

#### Emergence of SARS-CoV-2

<!-- TO DO: Which zoonotic viruses have been identified as likely relatives of SARS-CoV-2? How do we know this viral species evolved naturally (as opposed to the conspiracy theories? What differs between SARS-CoV-2 and its close relatives?-->

#### Evolution of SARS-CoV-2 Variants

Evolution in SARS-CoV-2 has also been observed over a short timescale.
After zoonotic transfer, SARS-CoV-2 continued evolving in the human population [@doi:10.1186/s12967-020-02344-6].
The SARS-CoV-2 mutation rate is moderate compared to other RNA viruses [@doi:10.1016/j.meegid.2020.104351], which likely restricts the pace of evolution in SARS-CoV-2.
Nevertheless, genomic analyses have yielded statistical evidence of ongoing evolution.
Initially, two known variants of the spike protein emerged that differed by a single amino acid at position 614 (G614 and D614), and there is evidence that G614 had become more prevalent than D614 by June 2020 [@doi:10.1016/j.cell.2020.06.043].
While there is a hypothesis that this genomic change increased the SARS-CoV-2 infectivity and virulence, this hypothesis has not yet been tested due to a lack of data [@doi:10.1016/j.cell.2020.06.040].
Another study [@doi:10.1016/j.meegid.2020.104351] identified 198 recurrent mutations in a dataset of 7,666 curated sequences, all of which defined non-synonymous protein-level modifications.
This pattern of convergent evolution at some sites could indicate that certain mutations confer an adaptive advantage.
While it is evident that SARS-CoV-2 exhibits moderate potential for ongoing and future evolution, the relationship between mutations and pathogenicity is not yet known.
Additional data is needed in order to understand patterns of evolutionary change and the mechanisms by which they might affect virulence.

Several factors could promote the evolution of SARS-CoV-2, including host immunodeficiency and transient exposure to antibodies directed against SARS-CoV-2 proteins. 
A single case study of SARS-CoV-2 infection in an immunocompromised female with chronic lymphocytic leukemia and hypogammaglobulinemia [@doi:10.1016/j.cell.2020.10.049] suggested that an accelerated evolution of the virus could occur in conditions of immunodeficiency. 
A first administration of convalescent plasma did not clear the virus, and an ensuing increase in the genomic diversity in the samples was observed, suggesting an accelerated evolution due to selection pressure. 
A second administration of convalescent plasma cleared the virus from the host 105 days after the initial diagnosis. 
However, throughout the duration of infection, the patient was asymptomatic but contagious.
A second single case study in a 45-year old male with antiphospholipid syndrome [@doi:10.1056/NEJMc2031364] confirmed the earlier results, providing evidence of persistent COVID-19 symptoms in an immunocompromised patient for 154 days following diagnosis, ultimately leading to the death of patient. 
The treatments administered included remdesivir and the Regeneron anti-spike protein antibody cocktail. 
Genomic analyses of the patient's nasopharyngeal swabs confirmed an accelerated evolution of the virus through mutations in the spike gene and the receptor-binding domain. 
In summary, these two case studies suggested an accelerated evolution and persistent shedding of the virus in conditions of immunodeficiency. 
In particular, the first case highlighted the role of convalescent plasma in creating escape variants. 
In fact, one study [@doi:10.1101/2020.12.28.424451] exposed the SARS-CoV-2 virus to convalescent plasma _in vitro_ repeatedly to see how much plasma was required to neutralize the virus.
The results of the first six exposures were similar, but they reported that after the seventh exposure (on day 45), the amount of plasma required began to increase.
In analyzing the viral variants present, they found that this viral escape was promoted by the sudden accumulation of mutations, especially in the receptor-binding domain (RBD) and N-terminal domain (NTD), that quickly rose in frequency.
By the thirteenth exposure (day 85), the virus had evolved three mutations and could no longer be neutralized by the plasma used, even though the plasma was comprised of polyclonal serum that targeted a variety of epitopes.
Taken together, these observations suggest that evolutionary analyses of SARS-CoV-2 can provide crucial information about the conditions that promote resistance in SARS-CoV-2 and the kinetics of how resistance develops, information which will be important for understanding the implications of how vaccine regimens are designed and whether/when next-generation vaccines will be needed.

When variants occur, they can rise in frequency by chance or through an adaptive process that confers a competitive advantage to the virus.
Variants that had the D614G mutation in the spike glycoprotein seemed to spread faster. <!-- TO DO: add more history of this variant?-->
However, it has been suggested that the mutation rose in frequency due to early chance events rather than by adaptive events [@doi:10.1001/jama.2020.27124].
Another mutation, Y453F, that occurred in the receptor binding domain of _S_, was first detected in mink; however, the transmission to humans has been established.
In mink, this mutation conferred an advantage by increasing the affinity towards ACE2 [@doi:10.1126/science.abe5901].
Similarly, N501Y mutation induces an increased affinity towards human ACE2 and has been involved in the dominance of B.1.1.7 by outcompeting other variants [@doi:10.1101/2020.12.24.20248822].
Therefore, genomic surveillance is essential to prevent the emergence of super-spreaders [@doi:10.1126/science.abf2946].

Emerging methods are being applied to this problem in an effort to understand which mutations are most likely to be of significant concern.
Novel machine learning methods were developed to predict the mutations in the sequence that promote viral escape.
While they preserve the pathogenicity of the virus, escape mutations change the virus's sequence to evade detection by the immune system. 
By using tools from natural language processing (NLP), viral escape was modeled as an NLP problem [@doi:10.1126/science.abd7331] where a modification makes a sentence grammatically correct but semantically different.
Therefore, language models of viruses could predict mutations that change the presentation of the virus to the immune system but preserve its infectivity.

#### Variants of concern and variants under surveillance

Viral replication naturally leads to the occurrence of mutations, and thus to genetic variation [@doi:10.1038/s41564-020-0690-4].
However, due to an intrinsic RNA proof-reading process in the SARS-CoV-2 virus, the pace of evolution of SARS-CoV-2 is moderate in comparison to other viruses [@doi:10.1186/1471-2148-4-21].
The declaration of the first SARS-CoV-2 variant of concern (VOC) B.1.1.7 in December 2020 has attracted significant media attention.
While the B.1.1.7 lineage garnered attention in November 2020, two genomes of the lineage were detected as early as September 20th, 2020 from routine genomic data sampled in Kent (U.K.) by the COVID-19 Genomics UK Consortium (COG-UK). 
The following day, a second B.1.1.7 genome was reported in greater London [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf; @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563; @doi:10.1101/2020.12.30.20249034; @doi:10.1001/jama.2020.27124]
Since then, B.1.1.7 has spread across the UK and internationally, and it has now been detected in at least 62 countries [@url:https://cov-lineages.org/global_report_B.1.1.7.html], despite several countries imposing travel restrictions on travelers from the UK. 
Of the twenty-three mutations that define B.1.1.7 from the original strain isolated in Wuhan (lineage A), fourteen are lineage-specific and three appear to be biologically consequential mutations associated with the spike protein, namely N501Y, P681H, and 69-70del [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf;  @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563].
The latter is a 6-bp deletion that leads to the loss of two amino acids and has consequences for immune recognition; it may, in conjunction with N501Y, be responsible for the increased transmissibility of the B.1.1.7 VOC due to changes in the RBD that increase binding affinity with ACE2 [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf; @doi:10.1101/2020.12.14.422555]. 
B.1.1.7 has increased transmissibility by up to 56%, leading to an R~0~ of approximately 1.4.
Additionally, this VOC has been shown to be associated with increased disease severity and increased mortality [@doi:10.1038/s41586-021-03426-1]. 
Other variants also express the 69-70del mutation [@url:https://virological.org/t/identification-of-a-novel-sars-cov-2-spike-69-70-deletion-lineage-circulating-in-the-united-states/577; @doi:10.1101/2020.12.24.20248814], and public health officials in the United States and the UK have been able to use RT-PCR-based assays (ThermoFisher TaqPath COVID-19 assay) to identify sequences with this deletion because it occurs where the qPCR probe binds [@doi:10.1101/2020.12.30.20249034]. 
In the UK, B.1.1.7 is present in more than 97% of diagnostic tests that return negative for S-gene targets and positive for the other targets; thus, the frequency of S-gene target failure can be used as a proxy for the detection of B.1.1.7 [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf; @url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/950823/Variant_of_Concern_VOC_202012_01_Technical_Briefing_3_-_England.pdf]._
The FDA has highlighted that the performance of three diagnostic tests may be affected by the B.1.1.7 lineage because it could cause false negative tests [@url:https://www.fda.gov/medical-devices/letters-health-care-providers/genetic-variants-sars-cov-2-may-lead-false-negative-results-molecular-tests-detection-sars-cov-2].

While B.1.1.7 is currently the main VOC, other genetic variants also currently designated as VOCs have been detected, including B.1.351 and P.1, both of which emerged independently [@url:https://www.cdc.gov/coronavirus/2019-ncov/more/science-and-research/scientific-brief-emerging-variants.html; @url:https://www.cdc.gov/coronavirus/2019-ncov/cases-updates/variant-surveillance/variant-info.html].
B.1.351 was first detected in October 2020 in South Africa, was later detected in the EU on December 28th, 2020 and has now spread to at least 26 countries [@url:https://www.gov.za/speeches/minister-zweli-mkhize-confirms-8-725-more-cases-coronavirus-covid-19-18-dec-2020-0000; @url:https://cov-lineages.org/global_report_B.1.351.html; @url:https://virological.org/t/tracking-the-international-spread-of-sars-cov-2-lineages-b-1-1-7-and-b-1-351-501y-v2/592]. 
B.1.351 contains several mutations at the RBD including K417N, E484K, and N501Y.
While the biological significance of these mutations are still under investigation, it does appear that this lineage may be associated with increased transmissibility [@doi:10.1101/2020.12.21.20248640] due to the N501Y mutation [@doi:10.1101/2020.12.14.422555; @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563].
Additionally, an analysis of a pseudovirus expressing the 501Y.V2 spike protein (B.1.351) showed that this variant demonstrates increased resistance to neutralization by convalescent plasma, even though total binding activity remained mostly intact [@doi:10.1038/s41591-021-01285-x].
Further, using a live virus neutralization assay (LVNA), it was shown that 501Y.V2 (B.1.351) is poorly neutralized by convalescent plasma obtained from individuals who responded to non-501Y.V2 variants [@doi:10.1038/s41586-021-03471-w]. 
However, 501Y.V2 infection-elicited plasma was able to cross-neutralize earlier non-501Y.V2 variants, suggesting that vaccines targeting VOCs may be effective against other mutant lineages [@doi:10.1038/s41586-021-03471-w].

The P.1 variant is a sublineage of the B.1.1.28 lineage that was first detected in Japan in samples obtained from four travelers from Brazil during a screening at a Tokyo airport on January 10, 2021 [@url:https://www.ecdc.europa.eu/sites/default/files/documents/COVID-19-risk-related-to-spread-of-new-SARS-CoV-2-variants-EU-EEA-first-update.pdf].
Shortly thereafter, it was established that there was a concentration of cases of the P.1 variant in Manaus, Brazil.
In a small number of samples (n=31) sequenced in Manaus, 42% were identified as the P.1 variant as early as mid-December, but the variant seemed to be absent in genome surveillance testing prior to December [@url:https://virological.org/t/genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-manaus-preliminary-findings/586].
To date, at least eight countries have detected the P.1 lineage [@url:https://cov-lineages.org/global_report_P.1.html].
While the majority of P.1 cases detected internationally have been linked to travel originating from Brazil, the UK has also reported evidence of community transmission detected via routine community sequencing [@url:https://cov-lineages.org/global_report_P.1.html; @url:https://www.reuters.com/article/us-health-coronavirus-britain-variants/uk-detects-77-cases-of-south-african-covid-variant-nine-of-brazilian-idUSKBN29T07P].  
P.1 has eight lineage-specific mutations along with three concerning spike protein mutations in the RBD, including K417T, E484K, and N501Y [@doi:10.1101/2020.12.21.20248640].
 
There have been multiple different SARS-CoV-2 lineages detected that have mostly been of no more clinical concern than the original devastating lineage originating in Wuhan [@url:https://cov-lineages.org/lineages.html].
However, the spotlight has been cast on other variants of unknown clinical relevance due to the increase of cases observed that have been associated with B.1.1.7 in particular.  
Although early in its ascendency, B.1.427/429 are SARS-CoV-2 variants that was detected in California, USA and also known as CAL.20C [@doi:10.1101/2021.01.18.21249786].
It was first detected in July 2020 but was not detected again until October 2020. 
In December 2020, B.1.427/429 accounted for ~24% of the total cases in Southern California and ~36% of total cases in the Los Angeles area. 
B.1.427/429 have now been detected in several U.S. states and at least 38 countries worldwide [@url:https://www.gisaid.org/hcov19-variants; @doi:10.1101/2021.01.18.21249786].
This variant is characterized by five key lineage-specific mutations (ORF1a: I4205V, ORF1b:D1183Y, S: S13I;W152C;L452R).
The latter spike mutation, L452R, is found in an area of the RBD known to resist monoclonal antibodies to the spike protein [@doi:10.1016/j.cell.2020.07.012], and it is hypothesized that this mutation may resist polyclonal sera in convalescent patients or in individuals post-vaccination [@doi:10.1101/2020.12.31.425021; @doi:10.1101/2021.01.18.21249786].
B.1.427/429 are now designated VOCs [@url:https://www.cdc.gov/coronavirus/2019-ncov/cases-updates/variant-surveillance/variant-info.html]; however, further research is still required to determine the implications of the mutations encoded in this genetic variant.  
Another notable variant has recently been discovered in 35 patients in a Bavarian hospital in Germany; however, the sequencing data has not been published to date and it remains to be determined whether this variant is of any further concern [@url:https://www.br.de/nachrichten/bayern/neue-coronavirus-mutation-35-faelle-in-garmisch-partenkirchen,SMQ1V6u].

There are several shared mutations and deletions between the three lineages, P.1, B.1.1.7, and B.1.315 and indeed other variants of SARS-CoV-2 that are under investigation [@url:https://virological.org/t/genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-manaus-preliminary-findings/586].
For example, N501Y, which appears to have occurred independently in each of the three lineages.  
E484K is present in both B.1.351 and P.1 [@url:https://cov-lineages.org/global_report.html].
The mutations N501Y and E484K are found in the RBD within the receptor-binding motif responsible for forming an interface with the ACE2 receptor, which seems to be consequential for ACE2 binding affinity [@doi:10.1016/j.cell.2020.08.012].
Indeed, N501Y is associated with increased virulence and infectivity in mouse models [@doi:10.1126/science.abc4730].
E484K has also been associated with evasion from neutralizing antibodies [@doi:10.1016/j.chom.2020.11.007; @doi:10.1101/2020.12.28.424451; @doi:10.1101/2020.12.31.425021].
The del69-70 (del:11288:9) is also shared between P.1 and B.1.1.7 and happens to be a common deletion found in the N terminal mutation of the spike protein. 
This deletion has also been associated with several RBD mutations [@doi:10.1101/2020.11.19.389916; @doi:10.1101/2020.12.14.422555; @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563].
There is concern that mutations in the spike protein of variants may lead to clinical consequences for transmissibility, disease severity, re-infection, therapeutics, and vaccinations [@doi:10.1126/science.abg4312; @url:https://virological.org/t/sars-cov-2-reinfection-by-the-new-variant-of-concern-voc-p-1-in-amazonas-brazil/596; @doi:10.1101/2020.12.28.424451; @doi:10.1101/2020.12.31.425021; @doi:10.1038/d41586-021-00121-z; @doi:10.1126/science.abf9302; @doi:10.1126/science.371.6527.329]. 

Vaccine producers are working to determine whether the vaccines are still effective against the novel genetic variants.
Moderna recently published data for their mRNA-1273 vaccine that showed no significant impact of neutralization against the B.1.1.7 variant upon vaccination in humans and non-human primates. 
On the other hand, Moderna reported a reduced but significant neutralization against the B.1.351 variant upon vaccination [@doi:10.1101/2021.01.25.427948].
Indeed, PfizerBioNTech reported that sera from twenty participants vaccinated with the BNT162b COVID-19 vaccine in previous clinical trials [@doi:10.1056/NEJMoa2034577; @doi:10.1056/NEJMoa2027906] elicited equivalent neutralizing titers against isogenic Y501 SARS-CoV-2 on an N501Y genetic background _in vitro_ [@doi:10.1101/2021.01.07.425740].
Another study has reported that the plasma neutralizing activity against SARS-CoV-2 variants encoding the combination of K417N:E484K:N501Y or E484K or N501Y was variably and significantly reduced in the sera of twenty participants who received either the PfizerBioNTech BNT162b (n = 6) vaccine or the Modernas mRNA-1273 vaccine (n =14) [@doi:10.1101/2021.01.15.426911]. 
In a study focusing on serum samples from a combination of convalescent individuals, those who obtained the mRNA-1273 vaccine, and those who obtained Novavax, in comparison to the D614G variant, the B.1.419 variant was 2-3 times less sensitive to neutralization while the B.1.351 variant was 9-14 times less sensitive [@doi:10.1056/NEJMc2103740].
Indeed, the E484K substitution seen in the P.1 and B.1.315 variants of the B.1.1.7 lineage are broadly reported to substantially reduce the efficacy of mRNA-based vaccines [@doi:10/f53t; @doi:10.1038/s41586-021-03412-7; @doi:10.1056/NEJMc2103740]. 
For now, the consensus appears to be that the FDA-approved vaccines still seem to be generally effective against the genetic variants of SARS-CoV-2 and their accompanying mutations, albeit with a lower neutralizing capacity [@doi:10.1073/pnas.2008281117; @doi:10.1101/2021.01.15.426911; @doi:10.1101/2021.01.07.425740; @doi:10.1101/2021.01.25.427948], though select VOCs may present challenges. 
Further research is required to discern the clinical, prophylactic, and therapeutic consequences of these genetic SARS-CoV-2 variants as the pandemic evolves. 

### Conclusions

As of October 2020 the SARS-CoV-2 virus remains a serious worldwide threat.
The scientific community has responded by rapidly collecting and disseminating information about the SARS-CoV-2 virus and the associated illness, COVID-19.
The rapid identification of the genomic sequence of the virus allowed for early contextualization of SARS-CoV-2 among other known respiratory viruses.
The pathogen is a coronavirus that is closely related to SARS-CoV-1, which caused the SARS pandemics of the early 2000s.
Knowing the phylogenetic context and genomic sequence of the virus then allowed for rapid insights into its structure and pathogenesis.
