## Diagnostics

Identifying individuals who have contracted the infection is crucial to slow down the global pandemic.
Given the high transmissibility of SARS-CoV-2, the development of reliable assays to detect SARS-CoV-2 infection even in asymptomatic carriers is vitally important.
For instance, the deployment of wide-scale diagnostic testing followed by the isolation of infected people has been a key factor in South Korea's successful strategy for controlling the spread of the virus.
Following the first release of the genetic sequence of the virus by Chinese officials on January 10 2020, the first test was released about 13 days later [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
The assays available to date use the following approaches to identify the active virus in patient samples.
A range of diagnostic approaches from a methodological standpoint are being or could possibly be developed.

### Molecular Tests

Molecular tests are used to identify distinct genomic subsequences of a viral molecule in a sample.
This first requires getting a patient sample from somewhere the virus is expected to be present in an infected individual.
Common sources for a sample used in a molecular test include nasopharyngeal cavity samples, including throat wash and saliva [@doi:10/ggp4qx], or stool samples [@doi:10.1002/jmv.25742].
Given a sample from a patient, molecular tests involve a number of steps to analyze a sample and produce results.
These steps include sample pre-processing, followed by library preparation, and then sequencing itself [doi:10.1016/j.biotechadv.2020.107537].
When testing for RNA viruses like SARS-CoV-2, pre-processing is needed in order to create DNA, which is easier to process and sequence, from the initial RNA sample.
The DNA can then be manipulated and copied to produce quantities sufficient for the test, called amplification.
Library preparation is the process of adding reagents that cut the sequences into fragments appropriate for use in a PCR machine [doi:10.1016/j.biotechadv.2020.107537].
In some cases, library preparation can involve other modifications of the sample, such as adding "barcoding" to identify a particular sample in the sequence data.
Sequential pattern matching is then used to identify unique subsequences of the virus that identify it in specific. If sufficient subsequences are found, the test is considered positive.

#### RT-PCR

RT-PCR tests use reverse transcriptase to translate the viral RNA into DNA during pre-processing and then measure the rate of amplification during PCR compared to a standard to determine presence of a target.
There are different reagents used for library preparation that are specific to identifying one or more target sections with PCR [@doi:10.1021/acsnano.0c02624].
The Drosten Lab, from Germany, was the first lab to establish and validate a diagnostic test to detect SARS-CoV-2.
This test uses real-time reverse transcription PCR (RT-PCR) [@doi:10.2807/1560-7917.ES.2020.25.3.2000045] to detect several regions of the viral genome: the orf1b of the RNA dependent RNA polymerase (RdRP), the Envelope protein gene (E), and the Nucleocapsid protein gene (N).
In collaboration with several other labs in Europe and in China, the researchers confirmed the specificity of this test with respect to other coronaviruses against specimens from 297 patients infected with a broad range of respiratory agents.
Specifically this tests utilizes two probes against RdRP of which one is specific to SARS-CoV-2 [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
Importantly, this assay did not give any false positive results.


#### qRT-PCR

Chinese researchers developed a quantitative real-time reverse transcription PCR (qRT-PCR) test to identify two regions of the viral genome, orf1b and N region [@doi:10.1093/clinchem/hvaa029].
Specifically, this assay was tested on samples coming from two COVID-19 patients, including a panel of positive and negative controls consisting of RNA extracted from several cultured viruses.
The assay uses the N gene to screen patients, while the orf1b region is used to confirm the infection [@doi:10.1093/clinchem/hvaa029].
In this case the test was designed to detect sarbecoviruses, of which SARS-CoV-2 is one.
Considering that no other sarbecoviruses are currently known to infect humans, a positive test indicates that the patient is infected with SARS-CoV-2.
However, this test is not able to discriminate the genetics of viruses within the sarbecovirus clade.


#### Pooled and Automated PCR Testing

Due to limited supplies and the need for more tests, several labs have found ways to enable pooled or similar tests.
The first such result came from a group at the Technion, which found they could pool up to 32 samples [@doi:10.1101/2020.03.26.20039438].
This was followed by larger-scale pooling utilizing slightly different methods at the University of Minnesota [@doi:10.1101/2020.04.02.022186].
Although these approaches are also PCR based, they allow for more rapid scaling and higher efficiency for testing than the initial PCR-based methods developed.


##### CRISPR-based detection

Two American companies, Mammoth Biosciences and Sherlock Biosciences, adapted their CRISPR-based detection technology [@doi:10.1016/j.cell.2014.05.010] for COVID-19 diagnostics to increase testing throughput and accessibility [@doi:10.1101/2020.03.06.20032334].
Their methodology involves purification of RNA extracted from patient specimens, amplification of extracted RNAs by loop-mediated amplification, a rapid, isothermal nucleic acid amplification technique, and application of their CRISPR-Cas12-based technology.
In the assay designed by Mammoth Biosciences, guide RNAs were designed to recognize portions of sequences corresponding to the SARS-CoV-2 genome, specifically the N, E and RdRP regions.
In the presence of SARS-CoV-2 genetic material, sequence recognition by the guide RNAs results in double-stranded DNA cleavage by Cas12, as well as cleavage of a single-stranded DNA molecular beacon.
The cleavage of this molecular beacon acts as a colorimetric reporter that is subsequently read out in a lateral flow assay and indicates the positive presence of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection [@doi:10.1101/2020.03.06.20032334].
This assay has been reported to have a sensitivity as high as detection of 70-300 copies of the target RNA/µl, requires simple, accessible equipment, and can output an easily-interpretable result in approximately 30 minutes [@doi:10.1126/science.abb8400].
Initial testing with patient samples (n = 23) demonstrated a positive predictive value of 100% and a negative predictive value of 91.7%, which is highly competitive with the CDC's current testing standard using qRT-PCR. 
This test seeks to develop a practical solution for rapid, low-barrier testing in areas that are at greater risk of infection, such as airports and local community hospitals.

#### Limitation of Molecular Tests

Tests that identify SARS-CoV-2 using nucleic-acid-based technologies will identify only individuals with current infections and are not appropriate for identifying individuals who have recovered from a previous infection.
Within this category, different types of tests have different limitations.
For example, PCR-based test can be highly sensitive, but in high-throughput settings they can show several problems:

1. False-negative responses, which can present a significant problem to large-scale testing.
   To reduce occurrence of false negatives, correct execution of the analysis is crucial [@doi:10.1038/d41587-020-00002-2].
2. Uncertainty surrounding the SARS-CoV-2 viral shedding kinetics, which could affect the result of a test depending on when it was taken [@doi:10.1038/d41587-020-00002-2].
3. Type of specimen, as it is not clear which clinical samples are best to detect the virus [@doi:10.1038/d41587-020-00002-2].
4. Expensive machinery, which might be present in major hospitals and/or diagnostic centers but is often not available to smaller facilities [@doi:10.1126/science.abb8400].
5. Timing of the test, which might take up to 4 days to give results [@doi:10.1126/science.abb8400].
6. The availability of supplies for testing, including swabs and testing media, has been limited [@doi:10.1128/JCM.00512-20].
7. Because the guide RNA can recognize other interspersed sequences on the patient’s genome, false positives and a loss of specificity can occur
False positives are a concern with this technique, as the guide RNA can recognize other interspersed sequences on the patient’s genome, resulting in a loss of specificity.
Similarly, in tests that use CRISPR, false positives can occur due to the specificity of the technique, as the guide RNA can recognise other interspersed sequences on the patient’s genome.
As noted above, false negatives are a significant concern for several reason.
Clinical experience has led to particular caution because negative molecular tests do not necessarily mean a patient is virus-free [doi:10.1128/JCM.00297-20]

### Serological Tests

Although diagnostic tests based on the detection of the genetic material can be quite sensitive, they cannot provide information about the extent of the disease over time.
Most importantly, they would not work on a patient who has fully recovered from the virus at the time of sample collection.
In this context, immunity tests are significantly more informative.
Additionally, they can help scientists to understand why the disease has a different course among patients, as well as what strategy might work to manage the spread of the infection.
Furthermore, serological tests hold significant interest at present because they can provide information relevant to advancing economic recovery and allowing reopenings.
For instance, people that have developed antibodies can plausibly return to work prior to the others, based on (still-unproven) protective immunity [@doi:10.1101/2020.04.14.20065771], and if extensive enough, herd immunity which will prevent further diffusion of the virus.

#### Current Approaches

Several countries are now focused on implementing antibody tests, and in the United States, the FDA recently approved a serological test by Cellex for use under emergency conditions [@url:https://www.fda.gov/media/136625/download].
Specifically, the Cellex qSARS-CoV-2 IgG/IgM Rapid Test is a chromatographic immunoassay designed to qualitatively detect IgM and IgG antibodies against SARS-CoV-2 in the plasma of patients (blood sample) suspected to have developed the infection [@url:https://www.fda.gov/media/136625/download].
Such tests allow for the progress of the viral disease to be understood, as IgM are the first antibodies produced by the body and indicate that the infection is active.
Once the body has responded to the infection, IgG are produced and gradually replace IgM, indicating that the body has developed immunogenic memory [@doi:10.1002/jmv.25820].
The test cassette contains a pad of SARS-CoV-2 antigens and a nitrocellulose strip with lines for each of IgG and IgM, as well as a control (goat IgG) [@url:https://www.fda.gov/media/136625/download].
In a specimen that contains antibodies against the SARS-CoV-2 antigen, the antibodies will bind to the strip and be captured by the IgM and/or IgG line(s), resulting in a change of color [@url:https://www.fda.gov/media/136625/download].
With this particular assay results can be read within 15-20 minutes [@url:https://www.fda.gov/media/136625/download].
Other research groups, such as the Krammer lab of the Icahn School of Medicine at Mount Sinai proposed an ELISA test that detects IgG and IgM that react against the receptor binding domain (RBD) of the spike proteins (S) of the virus [@doi:10.1101/2020.03.17.20037713].
The authors are now working to get the assay into clinical use [@url:https://www.livescience.com/coronavirus-tests-available.html].

#### Limitations of Sierological Tests

Importantly, false-positives can occur due to the cross-reactivity with other antibodies according to the clinical condition of the patient [@url:https://www.fda.gov/media/136625/download].
Therefore, this test should be used in combination with RNA detection tests [@url:https://www.fda.gov/media/136625/download].
Due to the long incubation times and delayed immune responses of infected patients, serological tests are insufficiently sensitive for a diagnosis in the early stages of an infection.
The limitation on timing makes serological tests vfar less useful for enabling test-and-trace strategies.

### Possible Alternatives to Current Practices for Identifying Active Cases

Clinical symptoms are too similar to other types of pneumonia to be sufficient as a sole diagnostics criterion.
In addition, as noted above, identifying asymptomatic cases is critical.
Clinical symptom-based tests are only able to identify symptomatic cases, not presymptomatic or asymptomatic cases.
They are still important for clinical practice and for reducing the need for other tests.

X-ray diagnostics have been reported to have high sensitivity but low specificity in some studies [@doi:10.1148/radiol.2020200642].
Other studies have shown that specificity varies between radiologists [@doi:10.1148/radiol.2020200823], though the sensitivity reported here was lower than that published in the previous paper.
However, preliminary machine-learning results have shown far higher sensitivity and specificity from analyzing chest X-rays than was possible with clinical examination [@doi:10.1007/s13246-020-00865-4].
X-ray tests with machine learning can potentially detect asymptomatic or presymptomatic infections that show lung manifestations.
This approach would still not recognize entirely asymptomatic cases.
Given the above, the widespread use of X-ray tests on otherwise healthy adults is likely inadvisable.

### Challenges to Diagnostic Approaches

#### Limitations to Implementation of Large-Scale Testing
More information to follow.

#### Strategies and Considerations for Determining Whom to Test

Currently, Coronavirus tests are limited to people that are in danger of serious illness [@url:https://www.cdc.gov/coronavirus/2019-nCoV/hcp/clinical-criteria.html].
Specifically, the individuals at risk include:

- people with severe symptoms
- people showing mild symptoms that have been in contact with a person who has developed the infection
- people with underlying health conditions


However, this method of testing administration does not detect a high proportion of infections, and does not allow for test-and-trace methods to be used.
Individuals that are asymptomatic (potentially able to infect others) and individuals that are able to recover at home are therefore often unaware of their status.
For instance, a recent study from the Imperial College estimates that in Italy the true number of infections is around 5.9 million against the 70,000 detected as of March 28th [@doi:10.25561/77731].
