<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Halie M. Rando" />
  <meta name="author" content="Casey S. Greene" />
  <meta name="author" content="Michael P. Robson" />
  <meta name="author" content="Simina M. Boca" />
  <meta name="author" content="Nils Wellhausen" />
  <meta name="author" content="Ronan Lordan" />
  <meta name="author" content="Christian Brueffer" />
  <meta name="author" content="Sadipan Ray" />
  <meta name="author" content="Lucy D'Agostino McGowan" />
  <meta name="author" content="Anthony Gitter" />
  <meta name="author" content="Anna Ada Dattoli" />
  <meta name="dcterms.date" content="2020-04-02" />
  <meta name="keywords" content="covid-19, coronavirus, pandemic, viral infection" />
  <title>SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/master/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta name="citation_title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta property="og:title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta property="twitter:title" content="SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics" />
  <meta name="dc.date" content="2020-04-02" />
  <meta name="citation_publication_date" content="2020-04-02" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Halie M. Rando" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-7688-1770" />
  <meta name="twitter:creator" content="@tamefoxtime" />
  <meta name="citation_author" content="Casey S. Greene" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-8713-9213" />
  <meta name="twitter:creator" content="@GreeneScientist" />
  <meta name="citation_author" content="Michael P. Robson" />
  <meta name="citation_author_institution" content="Department of Computing Sciences, Villanova University, Villanova, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-4859-0033" />
  <meta name="citation_author" content="Simina M. Boca" />
  <meta name="citation_author_institution" content="Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, District of Columbia, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-1400-3398" />
  <meta name="citation_author" content="Nils Wellhausen" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-8955-7582" />
  <meta name="citation_author" content="Ronan Lordan" />
  <meta name="citation_author_institution" content="Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104-5158, USA" />
  <meta name="citation_author_orcid" content="0000-0001-9668-3368" />
  <meta name="twitter:creator" content="@el_ronan" />
  <meta name="citation_author" content="Christian Brueffer" />
  <meta name="citation_author_institution" content="Department of Clinical Sciences, Lund University, Lund, Sweden" />
  <meta name="citation_author_orcid" content="0000-0002-3826-0989" />
  <meta name="twitter:creator" content="@cbrueffer" />
  <meta name="citation_author" content="Sadipan Ray" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology &amp; Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_institution" content="Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0002-9960-5768" />
  <meta name="citation_author" content="Lucy D\&#39;Agostino McGowan" />
  <meta name="citation_author_institution" content="Department of Mathematics and Statistics, Wake Forest University, Winston-Salem, North Carolina, United States of America" />
  <meta name="citation_author_orcid" content="0000-0001-7297-9359" />
  <meta name="twitter:creator" content="@LucyStats" />
  <meta name="citation_author" content="Anthony Gitter" />
  <meta name="citation_author_institution" content="Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin, United States of America" />
  <meta name="citation_author_institution" content="Morgridge Institute for Research, Madison, Wisconsin, United States of America" />
  <meta name="citation_author_orcid" content="0000-0002-5324-9833" />
  <meta name="twitter:creator" content="@anthonygitter" />
  <meta name="citation_author" content="Anna Ada Dattoli" />
  <meta name="citation_author_institution" content="-Department of Systems Pharmacology &amp; Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA" />
  <meta name="citation_author_orcid" content="0000-0003-1462-831X" />
  <meta name="twitter:creator" content="@aadattoli" />
  <link rel="canonical" href="https://greenelab.github.io/covid19-review/" />
  <meta property="og:url" content="https://greenelab.github.io/covid19-review/" />
  <meta property="twitter:url" content="https://greenelab.github.io/covid19-review/" />
  <meta name="citation_fulltext_html_url" content="https://greenelab.github.io/covid19-review/" />
  <meta name="citation_pdf_url" content="https://greenelab.github.io/covid19-review/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://greenelab.github.io/covid19-review/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://greenelab.github.io/covid19-review/v/b9c4ebddbb3c4c335aefe4bb2a9923a94e23bf9a/" />
  <meta name="manubot_html_url_versioned" content="https://greenelab.github.io/covid19-review/v/b9c4ebddbb3c4c335aefe4bb2a9923a94e23bf9a/" />
  <meta name="manubot_pdf_url_versioned" content="https://greenelab.github.io/covid19-review/v/b9c4ebddbb3c4c335aefe4bb2a9923a94e23bf9a/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
</head>
<body>
<header id="title-block-header">
<h1 class="title">SARS-CoV-2 and COVID-19: An Evolving Review of Diagnostics and Therapeutics</h1>
</header>
<p><small><em>
This manuscript
(<a href="https://greenelab.github.io/covid19-review/v/b9c4ebddbb3c4c335aefe4bb2a9923a94e23bf9a/">permalink</a>)
was automatically generated
from <a href="https://github.com/greenelab/covid19-review/tree/b9c4ebddbb3c4c335aefe4bb2a9923a94e23bf9a">greenelab/covid19-review@b9c4ebd</a>
on April 2, 2020.
</em></small></p>
<!-- include the Font Awesome library, per: https://fontawesome.com/start -->
<p><link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css">
<span class="banner lightred"><span class="fas fa-info-circle fa-lg"></span> <strong>This in progress manuscript is not intended for the general public.</strong><br>
This is a review paper that is authored by scientists for an audience of scientists to discuss research that is in progress.
If you are interested in guidelines on testing, therapies, or other issues related to your health, you should not use this document.
Instead, you should collect information from your local health department, the <a href="https://www.cdc.gov/coronavirus/2019-ncov/index.html">CDC‚Äôs guidance</a>, or your own government.</span></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>Halie M. Rando</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7688-1770">0000-0001-7688-1770</a>
¬∑ <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/rando2">rando2</a>
¬∑ <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/tamefoxtime">tamefoxtime</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
¬∑ Funded by the Gordon and Betty Moore Foundation (GBMF 4552)
</small></p></li>
<li><p><strong>Casey S. Greene</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8713-9213">0000-0001-8713-9213</a>
¬∑ <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/cgreene">cgreene</a>
¬∑ <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/GreeneScientist">GreeneScientist</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America; Childhood Cancer Data Lab, Alex‚Äôs Lemonade Stand Foundation, Philadelphia, Pennsylvania, United States of America
¬∑ Funded by the Gordon and Betty Moore Foundation (GBMF 4552)
</small></p></li>
<li><p><strong>Michael P. Robson</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-4859-0033">0000-0002-4859-0033</a>
¬∑ <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/mprobson">mprobson</a><br>
<small>
Department of Computing Sciences, Villanova University, Villanova, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Simina M. Boca</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1400-3398">0000-0002-1400-3398</a>
¬∑ <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/SiminaB">SiminaB</a><br>
<small>
Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, District of Columbia, United States of America
</small></p></li>
<li><p><strong>Nils Wellhausen</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8955-7582">0000-0001-8955-7582</a>
¬∑ <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/nilswellhausen">nilswellhausen</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
</small></p></li>
<li><p><strong>Ronan Lordan</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-9668-3368">0000-0001-9668-3368</a>
¬∑ <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/RLordan">RLordan</a>
¬∑ <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/el_ronan">el_ronan</a><br>
<small>
Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104-5158, USA
</small></p></li>
<li><p><strong>Christian Brueffer</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-3826-0989">0000-0002-3826-0989</a>
¬∑ <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/cbrueffer">cbrueffer</a>
¬∑ <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/cbrueffer">cbrueffer</a><br>
<small>
Department of Clinical Sciences, Lund University, Lund, Sweden
</small></p></li>
<li><p><strong>Sadipan Ray</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9960-5768">0000-0002-9960-5768</a>
¬∑ <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/rays1987">rays1987</a><br>
<small>
Department of Systems Pharmacology &amp; Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA; Institute for Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
</small></p></li>
<li><p><strong>Lucy D'Agostino McGowan</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7297-9359">0000-0001-7297-9359</a>
¬∑ <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/LucyMcGowan">LucyMcGowan</a>
¬∑ <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/LucyStats">LucyStats</a><br>
<small>
Department of Mathematics and Statistics, Wake Forest University, Winston-Salem, North Carolina, United States of America
</small></p></li>
<li><p><strong>Anthony Gitter</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-5324-9833">0000-0002-5324-9833</a>
¬∑ <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/agitter">agitter</a>
¬∑ <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/anthonygitter">anthonygitter</a><br>
<small>
Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin, United States of America; Morgridge Institute for Research, Madison, Wisconsin, United States of America
</small></p></li>
<li><p><strong>Anna Ada Dattoli</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1462-831X">0000-0003-1462-831X</a>
¬∑ <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/aadattoli">aadattoli</a>
¬∑ <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/aadattoli">aadattoli</a><br>
<small>
-Department of Systems Pharmacology &amp; Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA
</small></p></li>
</ul>
<h2 class="page_break_before" id="abstract">Abstract</h2>
<p>Since late 2019, Coronavirus disease 2019 (COVID-19) has spread around the world, resulting in the declaration of a pandemic by the World Health Organization (WHO).
This infectious disease is caused by the newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2).
Research on the virus SARS-CoV-2 and the disease it causes is emerging rapidly through global scientific efforts.
The development of diagnostics, treatments, and vaccines will be critical to mitigating the impact of the virus.
Here we present a collaborative effort to organize and consolidate the rapidly emerging scientific literature related to SARS-CoV-2.
We present information about the virus in the context of what is known about related viruses and synthesize studies emerging about the diagnosis and treatment of COVID-19 alongside literature about related illnesses.
A broad scientific effort to understand this pandemic and related viruses and diseases will be foundational to efforts to predict possible interventions.
This text is an evolving and collaborative document that seeks to incorporate the ever-expanding body of information related to SARS-CoV-2 and COVID-19.</p>
<h2 id="where-to-contribute">Where to Contribute</h2>
<p>Introduce Yourself (GitHub Issue) <a href="https://github.com/greenelab/covid19-review/issues/17">https://github.com/greenelab/covid19-review/issues/17</a></p>
<p>Community Chat (Gitter Room) <a href="https://gitter.im/covid19-review/community">https://gitter.im/covid19-review/community</a></p>
<p>More Info (GitHub Readme) <a href="https://github.com/greenelab/covid19-review#sars-cov-2-and-covid-19-an-evolving-review-of-diagnostics-and-therapeutics">https://github.com/greenelab/covid19-review#sars-cov-2-and-covid-19-an-evolving-review-of-diagnostics-and-therapeutics</a></p>
<h2 class="page_break_before" id="introduction">Introduction</h2>
<h3 id="general-background">General Background</h3>
<p>On January 21, 2020, the World Health Organization (WHO) released its first report concerning what is now known as the Coronavirus disease 2019 (COVID-19) <span class="citation" data-cites="Vb5KFWnO">[<a href="#ref-Vb5KFWnO" role="doc-biblioref">1</a>]</span>.
This infectious disease came to international attention on December 31, 2019 following an announcement by national officials in China about 44 cases of a respiratory infection of unknown cause.
The first known cases were located in Wuhan City within the Hubei province of China, but the disease spread rapidly beyond Wuhan within China and subsequently around the world.
At the time of the first situation report <span class="citation" data-cites="Vb5KFWnO">[<a href="#ref-Vb5KFWnO" role="doc-biblioref">1</a>]</span>, 282 confirmed cases had been identified, primarily in China, but also 1-2 exported cases had been identified in several neighboring countries (Thailand, Japan, and the Republic of Korea).
One week later, 4593 confirmed cases had been identified, spanning not only Asia, but also Australia, North America, and Europe <span class="citation" data-cites="fLTiwToY">[<a href="#ref-fLTiwToY" role="doc-biblioref">2</a>]</span>.
On March 11, 2020, WHO formally classified the situation as a pandemic <span class="citation" data-cites="x9975iBK">[<a href="#ref-x9975iBK" role="doc-biblioref">3</a>]</span>.
By WHO Situation Report 61, released on March 20, 2020, 266,073 confirmed cases had been reported worldwide, with cases on every continent except Antarctica <span class="citation" data-cites="11JitNwZt">[<a href="#ref-11JitNwZt" role="doc-biblioref">4</a>]</span>.
At this time, over 11,000 deaths had been reported worldwide.</p>
<p>[Note: Maybe add a graph here, update as new reports come out.]</p>
<p>COVID-19 is caused by the newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2).
SARS-CoV-2 is a coronavirus, a family of RNA viruses known to cause respiratory and intestinal infections in humans and other species.
Infectious diseases of global concern have previously been associated with coronaviruses, including Severe Acute Respiratory Syndrome (SARS-CoV) and Middle East respiratory syndrome (MERS-CoV) <span class="citation" data-cites="G5NJrE75 UBTUyMGV">[<a href="#ref-G5NJrE75" role="doc-biblioref">5</a>,<a href="#ref-UBTUyMGV" role="doc-biblioref">6</a>]</span>; however, neither of these reached pandemic status, owing to proper containment procedures (SARS) or intrinsic limitations in virus transmission (MERS).
Additionally, there are four endemic human coronaviruses that rarely progress beyond the mild symptoms associated with the common cold <span class="citation" data-cites="UBTUyMGV">[<a href="#ref-UBTUyMGV" role="doc-biblioref">6</a>]</span>.
The precise identity of SARS-CoV-2 virus was unknown until approximately January 12, 2020, when Chinese officials released its genetic sequence to aid in worldwide efforts to diagnose the disease <span class="citation" data-cites="Vb5KFWnO">[<a href="#ref-Vb5KFWnO" role="doc-biblioref">1</a>]</span>.
As researchers worldwide work to characterize SARS-CoV-2 and COVID-19, information about the transmission and life cycle of the virus as well as the diagnosis and treatment of the disease is emerging rapidly.
In this review, we seek to consolidate information about the virus in the context of related viruses and to synthesize what is known about the diagnosis and treatment of COVID-19 and related diseases.
This is a real-time, collaborative effort that welcomes submissions from scientists worldwide.</p>
<h3 id="coronaviruses-what-are-they-and-what-do-we-know-about-sars-cov-19">Coronaviruses: What are they, and what do we know about SARS-CoV-19?</h3>
<p>Coronaviruses are RNA viruses that‚Ä¶
[Summarize relevant mechanisms for cell entry &amp; address evidence for/against ACE2 being important]</p>
<p>The origin of the SARS-CoV-19 virus is not yet fully understood.
Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-19 is unlikely to have originated from a laboratory ‚Äì either purposely engineered and released, or escaped ‚Äì and instead evolved naturally in an animal host <span class="citation" data-cites="NEIoHDjq">[<a href="#ref-NEIoHDjq" role="doc-biblioref">7</a>]</span>.
Among known coronaviruses, SARS-CoV-19 has the closest overall sequence similarity to RaTG13 (~96%) found in a <em>Rhinolophus affinis</em> bat <span class="citation" data-cites="VSkK7CeP">[<a href="#ref-VSkK7CeP" role="doc-biblioref">8</a>]</span>, while the receptor binding domain (RBD) is highly similar to that of viruses found in pangolins <span class="citation" data-cites="eQIU1Nbs">[<a href="#ref-eQIU1Nbs" role="doc-biblioref">9</a>]</span>.
This suggests that SARS-CoV-19 may have originated in viral reservoirs of similar hosts, however current evidence cannot discriminate an origin of the virus before or after zoonotic transfer to humans <span class="citation" data-cites="NEIoHDjq">[<a href="#ref-NEIoHDjq" role="doc-biblioref">7</a>]</span>.</p>
<h3 id="mechanisms-of-coronavirus-driven-disease-in-humans">Mechanisms of Coronavirus-driven Disease in Humans</h3>
<p>Coronaviruses are known to cause respiratory illnesses in humans through the following possible mechanisms‚Ä¶</p>
<h3 id="presentation-of-covid-19">Presentation of COVID-19</h3>
<p>Information is rapidly becoming available about the wide range of symptoms that can be associated with COVID-19 as well as the range of symptom severity, onset from exposure, and possible risk or protective factors‚Ä¶</p>
<h3 id="vaccines-for-viruses-strategies-for-and-challenges-to-development">Vaccines for Viruses: Strategies for and challenges to development</h3>
<p>What information is needed to develop a vaccine? How have vaccines for other viruses such as H1N1 been developed?</p>
<h3 id="diagnostics-and-therapeutics-for-viruses">Diagnostics and Therapeutics for Viruses</h3>
<p>Two major concerns within diagnosis include the detection of current infections in individuals with and without symptoms, and the detection of past exposure without an active infection.
In the latter category, identifying whether individuals can develop or have developed sustained immunity is also a major consideration.</p>
<p>Within therapeutics, some possible efforts include efforts to identify strategies for the management of symptoms as well as the development of antivirals‚Ä¶</p>
<p>In this review, we seek to consolidate information about efforts to develop strategies for diagnosis and therapeutics as new information is released by the scientific community.</p>
<h2 id="pathogenesis">Pathogenesis</h2>
<h3 id="mechanism-of-host-infection-by-sars-cov-2">Mechanism of Host Infection by SARS-CoV-2</h3>
<p>This section would also be great for the introduction of zoonotic diseases which has been shown to be the origin of SARS-CoV2.</p>
<h4 id="primary-transmission-and-viral-entry">Primary Transmission and Viral Entry</h4>
<p>[How does SARS-CoV-2 enter human cells?]
[What cells are primary infection sites for SARS-CoV-2?]
[What structural aspects allow for viral entry]?</p>
<h4 id="viral-replication-and-spreading">Viral Replication and Spreading</h4>
<p>[Renamed section with transmission dynamics below]
[Basic introduction into replication cycle]
[What are the routes of transmission]</p>
<h4 id="reproduction-number-and-dynamics-of-transmission">Reproduction Number and Dynamics of Transmission</h4>
<p>Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
The basic reproduction number, R<sub>0</sub>, is the expected number of new infections caused by one infected person, assuming no time dependence and a wholly susceptible population <span class="citation" data-cites="16CD4voW1">[<a href="#ref-16CD4voW1" role="doc-biblioref">10</a>]</span>.
The effective reproduction number, R<sub>t</sub>, describes how the reproduction number may change over time, and is used to quantify deviations in R from R<sub>0</sub>, for example as some fraction of the population becomes infected, or as interventions are put into place.
R<sub>0</sub> and R<sub>t</sub> can be estimated directly from epidemiological data or inferred using mathematical modeling.
Modeling approaches are typically based upon a classic epidemiological model structure: the susceptible-infected-recovered (SIR) model and its extensions <span class="citation" data-cites="lwH4gcts">[<a href="#ref-lwH4gcts" role="doc-biblioref">11</a>]</span>.</p>
<p>R<sub>0</sub> for COVID-19 is estimated to lie in the range R<sub>0</sub>=1.4-6.5 <span class="citation" data-cites="1E0r4uZy9 LHtuVmaq mHYmt0mv">[<a href="#ref-1E0r4uZy9" role="doc-biblioref">12</a>,<a href="#ref-LHtuVmaq" role="doc-biblioref">13</a>,<a href="#ref-mHYmt0mv" role="doc-biblioref">14</a>]</span>; estimates vary considerably depending on the data and the methods used.
Most estimates currently derive from populations in Asia, since outbreaks in Europe and North America are more recent.
Data-derived estimates (i.e.¬†those that do not incorporate SIR-type models into their analysis) typically predict lower values of R<sub>0</sub>.
For data-derived estimates, in one study of international cases, the predicted value is R<sub>0</sub>=1.7 <span class="citation" data-cites="vITui6ac">[<a href="#ref-vITui6ac" role="doc-biblioref">15</a>]</span>, in China (both Hubei province and nationwide), the value is predicted to lie in the range R<sub>0</sub>=2.0-3.6 <span class="citation" data-cites="nxM0rP5R WLc2UMgQ 1E0r4uZy9">[<a href="#ref-1E0r4uZy9" role="doc-biblioref">12</a>,<a href="#ref-nxM0rP5R" role="doc-biblioref">16</a>,<a href="#ref-WLc2UMgQ" role="doc-biblioref">17</a>]</span>, and on a cruise ship where an outbreak occurred, predicted R<sub>0</sub>=2.28 <span class="citation" data-cites="10bBqMHH7">[<a href="#ref-10bBqMHH7" role="doc-biblioref">18</a>]</span>.
SIR model-derived estimates of R<sub>0</sub> range from 2.0 - 6.5 in China <span class="citation" data-cites="ITh0Anof uTI5OuZt Yj8Xh4Wz ZzrrVDoE">[<a href="#ref-ITh0Anof" role="doc-biblioref">19</a>,<a href="#ref-uTI5OuZt" role="doc-biblioref">20</a>,<a href="#ref-Yj8Xh4Wz" role="doc-biblioref">21</a>,<a href="#ref-ZzrrVDoE" role="doc-biblioref">22</a>]</span> to R<sub>0</sub>=4.8 in France <span class="citation" data-cites="itj26agd">[<a href="#ref-itj26agd" role="doc-biblioref">23</a>]</span>.
Using the same model as for the French population, this study estimated R<sub>0</sub>=2.6 in South Korea <span class="citation" data-cites="itj26agd">[<a href="#ref-itj26agd" role="doc-biblioref">23</a>]</span>, which is consistent with other studies <span class="citation" data-cites="YfOcGRRa">[<a href="#ref-YfOcGRRa" role="doc-biblioref">24</a>]</span>.
From a meta-analysis of studies estimating R<sub>0</sub>, <span class="citation" data-cites="LHtuVmaq">[<a href="#ref-LHtuVmaq" role="doc-biblioref">13</a>]</span> predict the median as R<sub>0</sub>=2.79.</p>
<p>Inference of the effective reproduction number can provide insight into how populations respond to an infection, and the effectiveness of interventions.
In China, R<sub>t</sub> was predicted to lie in the range 1.6-2.6 in Jan 2020, before travel restrictions <span class="citation" data-cites="tiRfUgvs">[<a href="#ref-tiRfUgvs" role="doc-biblioref">25</a>]</span>.
R<sub>t</sub> decreased from 2.35 one week before travel restrictions were imposed (Jan 23, 2020), to 1.05 one week after.
Using their model, the authors also estimate the probability of new outbreaks occurring: the probability of a single individual exporting virus causing a large outbreak is 17-25% assuming MERS-like or SARS-like transmission, and the probability of a large outbreak occurring after ‚â•4 infections exist at a new location is greater than 50%.
An independent study came to similar conclusions: in a two-week period before Jan 23 finding R<sub>t</sub>=2.38, and decreasing to R<sub>t</sub> = 1.34 (using data from Jan 24 to Feb 3) or R<sub>t</sub>=0.98 (using data from Jan 24 to Feb 8) <span class="citation" data-cites="mHYmt0mv">[<a href="#ref-mHYmt0mv" role="doc-biblioref">14</a>]</span>.
In South Korea, R<sub>t</sub> was inferred for Feb-Mar 2020 in two cities: Daegu (the center of the outbreak), and Seoul <span class="citation" data-cites="YfOcGRRa">[<a href="#ref-YfOcGRRa" role="doc-biblioref">24</a>]</span>.
Metro data was also analyzed to estimate the effects of social distancing measures.
R<sub>t</sub> decreased in Daegu from around 3 to &lt;1 over the period that social distancing measures were introduced.
In Seoul, R<sub>t</sub> decreased slightly, but remained close to 1 (and larger than R<sub>t</sub> in Daegu).
This highlights that social distancing measures appeared to work to contain the infection in Daegu, but that in Seoul, R<sub>t</sub> remains above 1, thus secondary outbreaks are possible.
It also shows the importance of region-specific analysis: the large decline in case load nationwide is mainly due to the Daegu region, and could hide persistence of the epidemic in other regions, such as Seoul and Gyeonggi-do.
Similarly in Iran, estimates of R<sub>t</sub> declined from 4.86 in the first week to 2.1 by the fourth week after the first cases in Iran were reported <span class="citation" data-cites="TH4ymqvJ">[<a href="#ref-TH4ymqvJ" role="doc-biblioref">26</a>]</span>. The authors attributed this decline to the effects of self-quarantine and government reductions in working time.</p>
<h3 id="immune-response-to-sars-cov-2">Immune Response to SARS-CoV-2</h3>
<p>[Cellular responses to SARS-CoV-2 infection]
[What is causing neutropenia and lymphopenia observed in COVID-19 patients]
[Antibody production against SARS-CoV-2 by patient who recovered vs patient who did not recover]
[Cytokines and other soluble factors contribution to immune response]</p>
<h3 id="systems-level-approaches-for-understanding-sars-cov-2-pathogenesis">Systems level approaches for understanding SARS-CoV-2 pathogenesis</h3>
<p>Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies.
These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases <span class="citation" data-cites="5MhhHQO0">[<a href="#ref-5MhhHQO0" role="doc-biblioref">27</a>]</span>.
Over the last decade, systems biology approaches have been used widely to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases <span class="citation" data-cites="8esIIWt9">[<a href="#ref-8esIIWt9" role="doc-biblioref">28</a>]</span>.
Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections <span class="citation" data-cites="8vUHj9cm">[<a href="#ref-8vUHj9cm" role="doc-biblioref">29</a>]</span>.</p>
<p>The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet.
A significant breakthrough in SARS-CoV-2 research was achieved through the successful full-length genome sequencing of the pathogen <span class="citation" data-cites="vHq0hOWQ 1Fqilxaum VSkK7CeP">[<a href="#ref-VSkK7CeP" role="doc-biblioref">8</a>,<a href="#ref-vHq0hOWQ" role="doc-biblioref">30</a>,<a href="#ref-1Fqilxaum" role="doc-biblioref">31</a>]</span>.
Multiple research groups have drafted the genome sequence of SARS-CoV-2 based on sequencing of clinical samples collected from bronchoalveolar lavage fluid (BALF) <span class="citation" data-cites="vHq0hOWQ VSkK7CeP">[<a href="#ref-VSkK7CeP" role="doc-biblioref">8</a>,<a href="#ref-vHq0hOWQ" role="doc-biblioref">30</a>]</span> or from BALF, throat swabs, or isolates of the virus cultured from BALF <span class="citation" data-cites="1Fqilxaum">[<a href="#ref-1Fqilxaum" role="doc-biblioref">31</a>]</span>.
Importantly, SARS-CoV-2 has significant sequence homology with SARS-CoV (about 79%) and also to some extent with MERS-CoV (about 50%) <span class="citation" data-cites="1Fqilxaum">[<a href="#ref-1Fqilxaum" role="doc-biblioref">31</a>]</span>.
However, a higher level of similarity (about 90%) has been observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21), indicating a possible origin in bats <span class="citation" data-cites="1Fqilxaum VSkK7CeP">[<a href="#ref-VSkK7CeP" role="doc-biblioref">8</a>,<a href="#ref-1Fqilxaum" role="doc-biblioref">31</a>]</span>.</p>
<p>The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection.
Availability of the genome sequence of SARS-CoV-2 enhances the potential for subsequent proteome-level studies to provide further mechanistic insights into the virus‚Äô complex pathogenesis.
Of note, the cryo-electron microscopy structure of the SARS-CoV-2 spike (S) glycoprotein, which plays an important role in the early steps of viral infection, was reported very recently <span class="citation" data-cites="CqQVkaqj">[<a href="#ref-CqQVkaqj" role="doc-biblioref">32</a>]</span>.
Even though no comprehensive proteomic analysis of the pathogen or of patients suffering from its infection has yet been reported, one forthcoming study has demonstrated SARS-CoV-2 infected host cell proteomics using human Caco‚Äì2 cells as an infection model <span class="citation" data-cites="CtMA3hTi">[<a href="#ref-CtMA3hTi" role="doc-biblioref">33</a>]</span>.
The authors observed SARS-CoV-2 induced alterations in multiple vital physiological pathways, including translation, splicing, carbon metabolism and nucleic acid metabolism in the host cells.</p>
<p>There is a high level of sequence homology between SARS-CoV-2 and SARS-CoV, and sera from convalescent SARS-CoV patients can effectively cross-neutralize SARS-CoV-2-S-driven entry <span class="citation" data-cites="15l3di3Wj">[<a href="#ref-15l3di3Wj" role="doc-biblioref">34</a>]</span>.
Consequently, earlier proteome-level studies on SARS-CoV can also provide some essential information regarding the new pathogen <span class="citation" data-cites="GpnngtWK JLTf2Fwb">[<a href="#ref-GpnngtWK" role="doc-biblioref">35</a>,<a href="#ref-JLTf2Fwb" role="doc-biblioref">36</a>]</span>.
Considering the paucity of omics-level big data sets for SARS-CoV-2 up until now, existing data hubs that contain information for other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), and The Virus Pathogen Resource (ViPR) will serve as useful resources for computational and bioinformatics research on SARS-CoV-2.</p>
<h2 id="diagnostics">Diagnostics</h2>
<h3 id="current-strategies-for-diagnosing-covid-19-and-similar-viral-infections">Current Strategies for Diagnosing COVID-19 and Similar Viral Infections</h3>
<p>Given the heterogeneity of symptom presentation across patients with COVID-19, the development of standardized protocols for testing samples for SARS-CoV-2 is urgent.
Following the release of the genetic sequence of the virus by Chinese officials on January 12, 2020, the first tests for detecting the virus were released on XX, 2020.
These tests used the following approach to identify the active virus in patient samples‚Ä¶
However, many countries have struggled to acquire the tests required to keep pace with the epidemic.
[Why is it so difficult to scale up testing? What are some of the considerations?]</p>
<h3 id="possible-alternatives-to-current-practices-for-identifying-active-cases">Possible Alternatives to Current Practices for Identifying Active Cases</h3>
<p>[Are there other approaches that have worked for diagnosing other viruses at a rapid pace in large numbers of people?]
[What are some approaches people are currently testing for detecting live viruses, especially SARS-CoV-2?]</p>
<h3 id="detection-of-past-exposure-andor-sustained-immunity">Detection of Past Exposure and/or Sustained Immunity</h3>
<p>[What are approaches that allow us to detect past exposure for other viruses?]
[What efforts are underway to develop similar approaches for SARS-CoV-2?]
[What is sustained immunity and what are the indicators?]</p>
<h3 id="limitations-to-implementation-of-large-scale-testing">Limitations to Implementation of Large-Scale Testing</h3>
<p>[Right now, reagent supply is an issue. Are there others concerns that are likely to emerge?]</p>
<h3 id="strategies-and-considerations-for-determining-whom-to-test">Strategies and Considerations for Determining Whom to Test</h3>
<p>[If it‚Äôs not possible to test everyone, what strategies exist for selecting who to test?]
[Are these strategies likely to change over time? Presumably there are different stages of managing spread vs mitigating severity once it‚Äôs already at high prevalence?]</p>
<h2 id="therapeutics">Therapeutics</h2>
<p>Given the rapid predicted spread of the disease, the development of therapeutics will be critical to mitigating its effect on health and the mortality rate.
Typically, therapeutics can take a few forms.
First, the treatment and reduction of symptoms can result in the reduction of the severity and risk associated with an active infection.
Second, the development of antiviral drugs can drive a reduced recovery time for patients by inhibiting the development of the virus once an individual is infected.
Finally, vaccines present a strategy for bolstering the immune response of the populus broadly to the virus, resulting in a lower rate of infection.
All three of these strategies have been valuable elements of responses to other viruses, including coronaviruses, and are being investigated by researchers at present.
Additionally, there have been suggestions within the scientific community that nutraceutical or dietary supplement interventions may prime an individual‚Äôs immune system to prevent or lessen the impact of RNA virus infections <span class="citation" data-cites="eyBAUUCl GT7yDBjv">[<a href="#ref-eyBAUUCl" role="doc-biblioref">37</a>,<a href="#ref-GT7yDBjv" role="doc-biblioref">38</a>]</span>.
In the following sections, we critically appraise the literature surrounding the repurposing of existing treatments and development of novel therapeutics for the prevention, mitigation, and treatment of coronavirus infections.</p>
<h3 id="treatment-of-symptoms">Treatment of Symptoms</h3>
<p>The clinical picture of SARS-CoV-2 infection differs dramatically between individuals.
Some are asymptomatic.
Others experience mild COVID-19 symptoms, such as cough, sore throat and fever, while the most severe cases of COVID-19 include severe complications including pneumonia and Acute Respiratory Distress Syndrome (ARDS), which can lead to respiratory failure and death <span class="citation" data-cites="vEFSIErd">[<a href="#ref-vEFSIErd" role="doc-biblioref">39</a>]</span>.
Vaccines are one avenue to mitigate harm from viral pathogens, but in the case of a rapidly growing pandemic the longer timeframe of vaccine development and distribution means that there can be a key for treatments that palliate symptoms to avoid the most severe outcomes from infection.</p>
<h4 id="tocilizumab">Tocilizumab</h4>
<p>A recent study carried out on a sample of 191 adult COVID-19 in-patients at two Wuhan hospitals found that blood samples taken at admission contained significantly higher concentrations of interleukin-6 (IL-6) in patients who ultimately deceased compared to those who survived; average concentrations of IL-6 remained higher in the deceased group than the surviving group throughout hospitalization <span class="citation" data-cites="10THxyeCg">[<a href="#ref-10THxyeCg" role="doc-biblioref">40</a>]</span>.
This suggests that these individuals may be experiencing a ‚Äúcytokine storm‚Äù, which refers to an excessive inflammatory response.
IL-6 plays a key role in this response <span class="citation" data-cites="bZMKqj6e">[<a href="#ref-bZMKqj6e" role="doc-biblioref">41</a>]</span>.
IL-6 is a pro-inflammatory cytokine belonging to the family of interleukins, which are immune system regulators that are primarily responsible for immune cell differentiation.
Specifically, IL-6 promotes the differentiation of activated B cells into immunoglobulin-producing plasma cells <span class="citation" data-cites="1AVsj9DAP">[<a href="#ref-1AVsj9DAP" role="doc-biblioref">42</a>]</span> and acts as a growth factor for hybridoma and myeloma cells <span class="citation" data-cites="J7evEkJG glBmWODR">[<a href="#ref-J7evEkJG" role="doc-biblioref">43</a>,<a href="#ref-glBmWODR" role="doc-biblioref">44</a>]</span>.
In addition, IL-6 also induces the differentiation of na√Øve CD4+ T cells into effector T-cell subsets <span class="citation" data-cites="eW4MC6Do">[<a href="#ref-eW4MC6Do" role="doc-biblioref">45</a>]</span>.
In this way interleukins regulate both the pro- and anti-inflammatory responses.
In this context, the observation of elevated IL-6 in patients who died may reflect an over-production of proinflammatory interleukins.</p>
<p>In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system, ensuring homeostasis.
The introduction of the S-protein from SARS-CoV to mouse macrophages was found to increase production of IL-6 and TNF-Œ± <span class="citation" data-cites="QkTGQUcj">[<a href="#ref-QkTGQUcj" role="doc-biblioref">46</a>]</span>, and deceased SARS-CoV patients were found to have intermediate levels of IL-6, IL-1ùõΩ, and TNF-Œ± expressed in a number of ACE2-expressing cell types sampled from the lung and bronchial tissues during autopsy <span class="citation" data-cites="NNFGje9g">[<a href="#ref-NNFGje9g" role="doc-biblioref">47</a>]</span>.
However, other reports found the severe respiratory condition ARDS to be associated with elevated concentrations of IL-6 in BALF, but that concentrations of Tumor Necrosis Factor Œ± (TNF-Œ±) and IL-1ùõΩ decreased with the onset of ARDS <span class="citation" data-cites="f61jsRKY">[<a href="#ref-f61jsRKY" role="doc-biblioref">48</a>]</span>.
These cytokines enhance the pro-inflammatory reaction by increase acute-phase signaling, trafficking of immune cells to the site of primary infection, epithelial cell activation, and secondary cytokine production.
The acute phase response to infection results in the heavily damage of the endotelium of blood vessels, which disrupts the balance between pro and anti-inflammatory response <span class="citation" data-cites="f61jsRKY">[<a href="#ref-f61jsRKY" role="doc-biblioref">48</a>]</span>.
Thus, the holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death as per Acute Distress Respiratory Syndrome (ADRS) or Severe Acute Respiratory Syndrome (SARS) <span class="citation" data-cites="suIFBW">[<a href="#ref-suIFBW" role="doc-biblioref">49</a>]</span>, also caused by the new coronavirus.
Recently Chinese and Italian doctors have found that the the Tocilizumab (actemra, by Roche), a drug commonly used in the rheumatoid arthritis, may palliate the most severe symptoms associated with COVID-19.</p>
<h5 id="anticipated-mechanism">Anticipated Mechanism</h5>
<p>Human IL-6 is a glycoprotein of 26 kDa and it consists of 184 amino acids containing 2 potential N-glycosylation sites and four cysteine residues. IL-6 binds to its receptor either in the insoluble (IL-6R) and soluble (sIL-6R) form.
The receptor specificity determines the type of signaling. Specifically, the binding of IL-6 to the cell membrane receptor IL-6R gives rise to the ‚Äúclassical transduction of the signaling‚Äù, while to binding to sIL-6R generate the so called ‚Äútrans-signaling‚Äù <span class="citation" data-cites="1BxkPZYnj 1EEloBjzn">[<a href="#ref-1BxkPZYnj" role="doc-biblioref">50</a>,<a href="#ref-1EEloBjzn" role="doc-biblioref">51</a>]</span>.
IL-6 signaling occurs through 3 independent pathways: the Janus-activated kinase (JAK)-STAT3 pathway, the Ras/Mitogen-Activated Protein Kinases (MAPK) pathway and the Phosphoinositol-3 Kinase (PI3K)/Akt pathway <span class="citation" data-cites="VXDsRwHC">[<a href="#ref-VXDsRwHC" role="doc-biblioref">52</a>]</span>.
The ultimate result of the IL-6 cascade is to direct transcriptional activity of various promoters of pro-inflammatory cytokines, such as IL-1 and TFN, including IL-6 own regulation through the activity of NF-Œ∫B <span class="citation" data-cites="VXDsRwHC">[<a href="#ref-VXDsRwHC" role="doc-biblioref">52</a>]</span>.
Particularly, IL-6 synthesis is tightly regulated both transcriptionally and post-transcriptionally . In this context, it has been shown that viral proteins can enhance transcription of the IL-6 gene, via strengthening the DNA-binding activity between several transciptional factors and IL-6 gene-cis-regulatory elements <span class="citation" data-cites="1H8o8TFtM">[<a href="#ref-1H8o8TFtM" role="doc-biblioref">53</a>]</span>.
Tocilizumab is a humanised monoclonal antibody that binds both to the insoluble and soluble receptor of IL-6, de facto inhibiting the IL-6 immune cascade.</p>
<h5 id="current-evidence">Current Evidence</h5>
<p>Chinese doctors have started a trial enrolling 188 patients of which 14 with severe lung disease have shown clear sings of improvements, according to their results <span class="citation" data-cites="JgXCBFEo">[<a href="#ref-JgXCBFEo" role="doc-biblioref">54</a>]</span>.
Also, The AIFA (the Italian Drug Agency) approved the start of a new trial on March 19 recruiting patients at the initial stage of the infection <span class="citation" data-cites="123fXWICW">[<a href="#ref-123fXWICW" role="doc-biblioref">55</a>]</span>.
Together with these independent trials, Roche, also in collaboration with the FDA, will start a randomised, double-blind, placebo-controlled phase III trial early April.
The trial will enroll 330 patients globally, which will be followed for 60 days upon use of the drug via injection to analyse its efficiency/safety (Biopharma-reporter.com).
However, previous studies on RA showed that in patients treated with TCZ the rate of incident infections in clinical practice patients was higher than the one observed during clinical trial <span class="citation" data-cites="TgV8gb30">[<a href="#ref-TgV8gb30" role="doc-biblioref">56</a>]</span>.
Also, RA patients with chronic hepatitis B (HB) infection showed high risk of HB virus reactivation upon TCZ administration in combination with other RA drugs <span class="citation" data-cites="IJzmtJOr">[<a href="#ref-IJzmtJOr" role="doc-biblioref">57</a>]</span>.
These last findings highlight the need to search for a balance between impairing a harmful immune response, such as the one generated by the cytokine storm, and preventing the worsening of the clinical picture of the patients by potential new viral infections.
This aspect is probably crucial to be investigated further in the trials that are about to start.
Perhaps, the TCZ treatment would best suit patients with severely compromised lungs due to the Covid-19 infection and are therefore at greater risk of death, in order to stop the uncontrolled immune response before it‚Äôs too late.</p>
<h5 id="summary">Summary</h5>
<p>Summarize the state of the symptom management approach.</p>
<h3 id="small-molecule-drugs-for-targeting-sars-cov-2">Small Molecule Drugs for Targeting SARS-CoV-2</h3>
<p>The replication cycle of a virus within an epithelial host cell includes 6 basic steps which can be shortly summarised as follow: i) attachment of the virus to the host cell; ii) penetration by endocytosis; iii) uncoating, classically defined as the release of viral contents into the host cell; iv) biosynthesis, during which the viral genetic material enters the nucleus where it gets replicated; v) assembly, where viral proteins are translated and new viral particles are assembled; vi) release, when the new viruses are released into the extracellular environment <span class="citation" data-cites="14gthSuhm">[<a href="#ref-14gthSuhm" role="doc-biblioref">58</a>]</span>.
Antiviral drugs do not kill the virus, rather they inhibit its amplification by impairing one of these steps.
Nowadays, many of these drugs act during the biosynthesis step in order to inhibit the virus‚Äô genetic material replication.
Importantly, SARS-CoV-2 is an RNA virus.
Differently from a DNA virus, which can use the host enzymes to propagate itself, RNA viruses depends on their own polymerase, the RNA-dependent RNA polymerase (RdRP), in order to be replicated <span class="citation" data-cites="tn6j6IZ0 Sh1FwWps">[<a href="#ref-tn6j6IZ0" role="doc-biblioref">59</a>,<a href="#ref-Sh1FwWps" role="doc-biblioref">60</a>]</span>.</p>
<h4 id="nucleotide-analogs">Nucleotide Analogs</h4>
<p>Why one might use nucleotide analogs.</p>
<h5 id="avigan">Avigan</h5>
<p>Avigan (Flavipiravir, from Toyama chemical Fujifilm) is a drug which has been found effective to block viral amplification in the Influenza virus infection.</p>
<h6 id="anticipated-mechanism-1">Anticipated Mechanism</h6>
<p>Specifically, Avigan is a nucleoside precursor efficiently recognised as guanosine and adenosine analogue by the virus polymerase (RNA-dependent RNA polymerase).
While a single incorporation does not influence RNA transcription, multiple events of incorporation lead to the arrest of RNA synthesis <span class="citation" data-cites="ukZRKllk">[<a href="#ref-ukZRKllk" role="doc-biblioref">61</a>]</span>.
Importantly It was already shown that Avigan is able to give 100% coverage against the Ebola virus in mice <span class="citation" data-cites="gY3NNmLM">[<a href="#ref-gY3NNmLM" role="doc-biblioref">62</a>]</span>.
Furthermore, there are evidences that drug might also work against Corona virus infection.
For instance, a recent study showed its effectiveness compared to other antivirals such as lopinavir and ritonavir <span class="citation" data-cites="13Oz7yRPx">[<a href="#ref-13Oz7yRPx" role="doc-biblioref">63</a>]</span>.
The drug was tested on a sample of 80 patients (35 experimental sample, 45 control group) and increased the speed of recovery (measured as viral clearance from the patient by RT-PCR) of about 4 days, compared to the control sample treated with the other anti-virals, such as Lopinavir and Ritonavir.
Also lung body scan seem to look better in about 91% of the patients analysed <span class="citation" data-cites="13Oz7yRPx">[<a href="#ref-13Oz7yRPx" role="doc-biblioref">63</a>]</span>.
However, the size of the sample is too small to give a powerful statistics, as well as the choice of the patients did not take into consideration important factors such as previous clinical conditions, sex, while there was no age categorisation.</p>
<h6 id="current-evidence-1">Current Evidence</h6>
<p>A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.</p>
<h6 id="summary-1">Summary</h6>
<p>Summarize the state of the antiviral approach.</p>
<h5 id="remdesivir">Remdesivir</h5>
<p>Remdesivir (GS-5734) was developed by Gilead Sciences to treat Ebola.
It does not have any FDA-approved use.
Although a clinical trial in the Democratic Republic of Congo found some evidence of effectiveness against ebola, two antibody preparations were found to be more effective, and Remdesivir was not pursued <span class="citation" data-cites="gnfviuGp">[<a href="#ref-gnfviuGp" role="doc-biblioref">64</a>]</span>.</p>
<h6 id="anticipated-mechanism-2">Anticipated Mechanism</h6>
<p>Remdesivir is metabolized to GS-441524, an adenosine analog that inhibits a broad range of polymerases and then evades exonuclease repair causing chain termination <span class="citation" data-cites="YaxKcqq8 lIko9Imd IhdcvbzC">[<a href="#ref-YaxKcqq8" role="doc-biblioref">65</a>,<a href="#ref-lIko9Imd" role="doc-biblioref">66</a>,<a href="#ref-IhdcvbzC" role="doc-biblioref">67</a>]</span>.
Although it was developed against Ebola, it also inhibits the MERS-CoV and SARS-CoV polymerase and inhibits coronavirus replication in cell culture assays with submicromolar IC50s <span class="citation" data-cites="1H0NPTMw9">[<a href="#ref-1H0NPTMw9" role="doc-biblioref">68</a>]</span>.
It also inhibits SARS-CoV-2, showing synergy with chloroquine <span class="citation" data-cites="IhdcvbzC">[<a href="#ref-IhdcvbzC" role="doc-biblioref">67</a>]</span>.</p>
<h6 id="current-evidence-2">Current Evidence</h6>
<p>In addition to the previous work showing Remdesivir to be an effective treatment for viral pathogens such as SARS-CoV and MERS-CoV in cultured cells and animal models, a recent study found that administration of Remdesivir to non-human primate models resulted in 100% protection against infection by the Ebola virus.
Remdesivir has also been reported to inhibit SARS-CoV-2 infection in a human cell line sensitive to the virus <span class="citation" data-cites="IhdcvbzC">[<a href="#ref-IhdcvbzC" role="doc-biblioref">67</a>]</span>.</p>
<p>The effectiveness of Remdesivir for treating patients with COVID-19 is currently under investigation.
Remdesivir has been used on some COVID-19 patients under compassionate use guidelines <span class="citation" data-cites="D4HwvRMd zoLViMNg">[<a href="#ref-D4HwvRMd" role="doc-biblioref">69</a>,<a href="#ref-zoLViMNg" role="doc-biblioref">70</a>]</span>.
All were in late stages of COVID-19 infection, and these reports are inconclusive about the drug‚Äôs efficacy.
Remdesivir recently entered controlled clinical trials, and as of March 2020, there are six clinical trials underway to treat COVID-19 patients at both early and late stages of infection and in combinations with other drugs <span class="citation" data-cites="doi 8a4WEEFh 7LMnOrzx 19tnPvv1p 1q4PmqTm DvAlA1ad 1CJoAQiUA">[<span class="citeproc-not-found" data-reference-id="doi"><strong>???</strong></span> 10.1038/s41422-020-0282-0,<a href="#ref-8a4WEEFh" role="doc-biblioref">72</a>,<a href="#ref-7LMnOrzx" role="doc-biblioref">73</a>,<a href="#ref-19tnPvv1p" role="doc-biblioref">74</a>,<a href="#ref-1q4PmqTm" role="doc-biblioref">75</a>,<a href="#ref-DvAlA1ad" role="doc-biblioref">76</a>,<a href="#ref-1CJoAQiUA" role="doc-biblioref">77</a>]</span>.</p>
<h6 id="summary-2">Summary</h6>
<p>Remdesivir is a major drug candidate since it attacks the virus with high potency and known mechanism.
Moreover, one of the most successful therapies for viral diseases is to target the viral replication machinery, which are typically virally encoded polymerases.
Small molecule drugs targeting viral polymerases are the backbones of treatments for other viral diseases including HIV and Herpes.
Note that the HIV and Herpes polymerases are a reverse transcriptase and a DNA polymerase respectively, whereas SARS-CoV-2 encodes an RNA dependent RNA polymerase, so most of the commonly used polymerase inhibitors are not likely to be active against SARS-CoV-2.
In clinical use, polymerase inhibitors show short term benefits for HIV patients but for long term benefits they must be part of combination regimens.
They are typically combined with protease inhibitors, integrase inhibitors and even other polymerase inhibitors.</p>
<h6 id="current-evidence-3">Current Evidence</h6>
<p>A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.</p>
<h6 id="summary-3">Summary</h6>
<p>Summarize the state of the antiviral approach.</p>
<h4 id="protease-inhibitors">Protease Inhibitors</h4>
<p>Why it may be useful</p>
<h5 id="protease-inhibitor-1">Protease Inhibitor 1</h5>
<h6 id="anticipated-mechanism-3">Anticipated Mechanism</h6>
<h6 id="current-evidence-4">Current Evidence</h6>
<p>A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.</p>
<h6 id="summary-4">Summary</h6>
<p>Summarize the state of the antiviral approach.</p>
<h4 id="molecules-targeting-the-viral-envelope">Molecules Targeting the Viral Envelope</h4>
<p>Why it may be useful</p>
<h5 id="viral-envelope-targeter-1">Viral Envelope Targeter 1</h5>
<h6 id="anticipated-mechanism-4">Anticipated Mechanism</h6>
<h6 id="current-evidence-5">Current Evidence</h6>
<p>A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.</p>
<h6 id="summary-5">Summary</h6>
<p>Summarize the state of the antiviral approach.</p>
<h3 id="drugs-targeting-host-proteins">Drugs Targeting Host Proteins</h3>
<p>Brief background on the therapeutic.</p>
<h4 id="drug-or-drug-class-1">Drug (or drug class) 1</h4>
<h5 id="viral-entry-receptors">Viral Entry Receptors</h5>
<p>Why it may be useful</p>
<h6 id="current-evidence-6">Current Evidence</h6>
<p>A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.</p>
<h6 id="summary-6">Summary</h6>
<p>Summarize the state of the antiviral approach.</p>
<h3 id="broad-spectrum-pharmaceuticals">Broad-Spectrum Pharmaceuticals</h3>
<h4 id="hydroxychloroquine">Hydroxychloroquine</h4>
<h5 id="potential-mechanisms">Potential Mechanisms</h5>
<h5 id="current-evidence-7">Current Evidence</h5>
<h5 id="summary-7">Summary</h5>
<h4 id="nutraceuticals">Nutraceuticals</h4>
<h3 id="biological-drugs-for-covid-19">Biological Drugs for COVID-19</h3>
<h4 id="neutralizing-antibodies">Neutralizing Antibodies</h4>
<p>Monoclonal antibodies (mAbs) have revolutionized the way we treat human diseases.
As a result, they have become some of the best-selling drugs in the pharmaceutical market in recent years <span class="citation" data-cites="doi">[<span class="citeproc-not-found" data-reference-id="doi"><strong>???</strong></span> 10.1186/s12929-019-0592-z]</span>.
There are currently 79 FDA approved mAbs on the market including antibodies for viral infections (e.g.¬†Ibalizumab for HIV and Palivizumab for RSV) <span class="citation" data-cites="1AKi0FYUB yumBaJ6U">[<a href="#ref-1AKi0FYUB" role="doc-biblioref">78</a>,<a href="#ref-yumBaJ6U" role="doc-biblioref">79</a>]</span>.
Although vaccines remain the most important way to prevent viral infections, their development process is long and they fail to provide immediate prophylactic protection or treat ongoing infections <span class="citation" data-cites="181QWa7HL">[<a href="#ref-181QWa7HL" role="doc-biblioref">80</a>]</span>.
For that reason, neutralizing antibodies have emerged to address these shortcomings.
Virus-specific neutralizing antibodies commonly target viral surface glycoproteins or host structures, thereby inhibiting viral entry through receptor binding intereference <span class="citation" data-cites="dUnB3gD6 cwbHiM1o">[<a href="#ref-dUnB3gD6" role="doc-biblioref">81</a>,<a href="#ref-cwbHiM1o" role="doc-biblioref">82</a>]</span>.
This section discusses current efforts in developing neutralizing antibodies against SARS-CoV-2 and how expertise gained from previous approaches for MERS-CoV and SARS-CoV may benefit antibody development.</p>
<h5 id="spike-s-neutralizing-antibody">Spike (S) Neutralizing Antibody</h5>
<p>During the first SARS epidemic in 2002, nAbs were found in SARS-CoV infected patients <span class="citation" data-cites="doi10.1111/j.1469-0691.2004.01009.x KULxo48U">[<span class="citeproc-not-found" data-reference-id="doi10.1111/j.1469-0691.2004.01009.x"><strong>???</strong></span>,<a href="#ref-KULxo48U" role="doc-biblioref">83</a>]</span>.
Several studies following up on these findings identified various S glycoprotein epitopes as the major targets of neutralizing antibodies against SARS-CoV <span class="citation" data-cites="MTcTiS7i">[<a href="#ref-MTcTiS7i" role="doc-biblioref">84</a>]</span>.
The passive transfer of immune serum containing nAbs from SARS-CoV-infected mice resulted in protection of na√Øve mice from viral lower respiratory tract infection upon intranasal challenge <span class="citation" data-cites="rXzkS3gA">[<a href="#ref-rXzkS3gA" role="doc-biblioref">85</a>]</span>.
Similarly, a meta-analysis suggested that administration of plasma from recovered SARS-CoV patients reduced mortality upon SARS-CoV infection <span class="citation" data-cites="RRx6YovP">[<a href="#ref-RRx6YovP" role="doc-biblioref">86</a>]</span>.</p>
<p>Similar results have been observed for MERS-CoV infections, which emerged as the second coronavirus-related epidemic.
Neutralizing antibodies have been identified against various epitopes of the RBD of the S glycoprotein [<span class="citation" data-cites="Ou3P9pGV"><a href="#ref-Ou3P9pGV" role="doc-biblioref">87</a></span>; doi:10.1128/JVI.00912-14].</p>
<h6 id="spike-s-neutralizing-antibody-anticipated-mechanisms">Spike (S) Neutralizing Antibody Anticipated Mechanisms</h6>
<p>Coronaviruses use trimeric spike (S) glycoproteins on their surface to bind to host cell receptors, such as ACE2, allowing for cell entry [<span class="citation" data-cites="15l3di3Wj"><a href="#ref-15l3di3Wj" role="doc-biblioref">34</a></span>; doi:10.1016/j.cell.2020.02.058].
Each S glycoprotein protomer is comprised of an S1 domain, also called the receptor binding domain (RBD), and an S2 domain.
The S1 domain binds to host cell receptors while the S2 domain facilitates the fusion between the viral envelope and host cell membranes <span class="citation" data-cites="MTcTiS7i">[<a href="#ref-MTcTiS7i" role="doc-biblioref">84</a>]</span>.
Although targeting of the host cell receptor ACE2 shows efficacy in inhibiting SARS-CoV-2 infection <span class="citation" data-cites="2QTH37Xi">[<a href="#ref-2QTH37Xi" role="doc-biblioref">88</a>]</span>, given the physiological relevance of ACE2 <span class="citation" data-cites="1GLr0EJU">[<a href="#ref-1GLr0EJU" role="doc-biblioref">89</a>]</span>, it would be favorable to target virus-specific structures rather than host receptors.
This forms the rationale of developing neutralizing antibodies against the S glycoprotein, disrupting its interaction with ACE2 and other receptors and thereby inhibiting viral entry.</p>
<h6 id="spike-s-neutralizing-antibody-current-evidence">Spike (S) Neutralizing Antibody Current Evidence</h6>
<p>The first human neutralizing antibody against SARS-CoV-2 targeting the trimeric spike (S) glycoproteins has been developed using hybridoma technology, <span class="citation" data-cites="XcuzhxrJ">[<a href="#ref-XcuzhxrJ" role="doc-biblioref">90</a>]</span>, where antibody-producing B-cells developed by mice can be inserted into myeloma cells to produce a hybrid cell line (the hybridoma) that is grown in culture.
The 47D11 clone was able to cross-neutralize SARS-CoV and SARS-CoV2 by a mechanism that is different from receptor binding interference.
The exact mechanism of how this clone neutralizes SARS-CoV-2 and inhibits infection in vitro remains unknown, but a potential mechanism might be antibody induced destabilization of the membrane prefusion structure <span class="citation" data-cites="XcuzhxrJ Ftbm1M9p">[<a href="#ref-XcuzhxrJ" role="doc-biblioref">90</a>,<a href="#ref-Ftbm1M9p" role="doc-biblioref">91</a>]</span>.
The ability of this antibody to prevent infection at a feasible dose needs to be validated in vivo, especially since in vitro neutralization effects have been shown to not be reflective of in vivo efficacy <span class="citation" data-cites="O1whWg6Q">[<a href="#ref-O1whWg6Q" role="doc-biblioref">92</a>]</span>.
Only a week later, a different group successfully isolated multiple nAbs targeting the RBD of the S glycoprotein from blood samples taken from COVID-19 patients in China <span class="citation" data-cites="GhJYjnft">[<a href="#ref-GhJYjnft" role="doc-biblioref">93</a>]</span>.
Interestingly, the patient isolated antibodies did not cross-react with RBD‚Äôs from SARS-CoV and MERS-CoV, although cross-reactivity to the trimeric spike proteins of SARS-CoV and MERS-CoV was observed.
This suggests that the RBDs between the three coronavirus species are immunologically distinct and that the isolated nAbs targeting the RBD of SARS-CoV-2 are species specific.
While this specificity is desirable, it also raises the question of whether these antibodies are more susceptible to viral escape mechanisms.
Viral escape is a common resistance mechanism to nAbs therapy due to selective pressure from neutralizing antibodies <span class="citation" data-cites="pUnzB8wV 14motyCOm">[<a href="#ref-pUnzB8wV" role="doc-biblioref">94</a>,<a href="#ref-14motyCOm" role="doc-biblioref">95</a>]</span>.
For HIV, broadly neutralizing antibodies (bnAbs) targeting the CD4 binding site (CD4bs) show greater neutralization breadth than monoclonal antibodies, which target only specific HIV strains <span class="citation" data-cites="oGHFF5Cm">[<a href="#ref-oGHFF5Cm" role="doc-biblioref">96</a>]</span>.
For MERS-CoV, a combination of multiple neutralizing antibodies targeting different antigenic sites prevented neutralization escape <span class="citation" data-cites="hvjXMm9H">[<a href="#ref-hvjXMm9H" role="doc-biblioref">97</a>]</span>.
It was found that the different antibody isolates did not target the same epitopes, suggesting that using them in combination might produce a synergistic effect that prevents viral escape <span class="citation" data-cites="GhJYjnft">[<a href="#ref-GhJYjnft" role="doc-biblioref">93</a>]</span>.
It was also demonstrated that binding affinity of the antibodies does not reflect their capability to compete with ACE2 binding.
Furthermore, no conclusions about correlations between the severity of disease and the ability to produce neutralizing antibodies can be drawn at this point.
Rather, higher neutralizing antibody titers were more frequently found in patients with severe disease.
Correspondingly, higher levels of anti-spike IgG were observed in patients that deceased from infection compared to patient that recovered <span class="citation" data-cites="tC5vJmwj">[<a href="#ref-tC5vJmwj" role="doc-biblioref">98</a>]</span>.</p>
<h6 id="spike-s-neutralizing-antibody-summary">Spike (S) Neutralizing Antibody Summary</h6>
<p>Results from the SARS-CoV and MERS-CoV epidemics can provide valuable lessons for the design of neutralizing antibodies for the current outbreak.
The findings for SARS-CoV and MERS can aid in identifying which structures constitute suitable targets for nAbs, despite the fact that the RBD appears to be distinct between the three coronavirus species.
These studies also suggest that a combination of nAbs targeting distinct antigens might be necessary to provide protection <span class="citation" data-cites="hvjXMm9H">[<a href="#ref-hvjXMm9H" role="doc-biblioref">97</a>]</span>.
The biggest challenge remains identifying antibodies that not only bind to their target, but also prove to be beneficial for disease management.
On that note, a recently published study indicates that anti-spike antibodies could make the disease worse rather than eliminating the virus <span class="citation" data-cites="tC5vJmwj">[<a href="#ref-tC5vJmwj" role="doc-biblioref">98</a>]</span>.
These findings underscores our current lack of understanding the full immune response to SARS-CoV-2.</p>
<h5 id="anticipated-mechanism-5">Anticipated Mechanism</h5>
<p>Why it may be useful</p>
<h5 id="current-evidence-8">Current Evidence</h5>
<p>A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.</p>
<h5 id="summary-8">Summary</h5>
<p>Summarize the state of the neutralizing antibody approach.</p>
<h4 id="interferons">Interferons</h4>
<p>Interferons (IFNs) are a family of cytokines crucial to activate the first (innate) immune system response against viral infections.
Interferons are classified into three categories based on their receptor specificity: type I, II and III <span class="citation" data-cites="bZMKqj6e">[<a href="#ref-bZMKqj6e" role="doc-biblioref">41</a>]</span>.
Specifically, IFNs I (IFN-ùõº and ùõΩ) and II (IFN-ùõæ ) induce the expression of antiviral proteins which bring the viral RNA to degradation <span class="citation" data-cites="LtFtUo2P">[<a href="#ref-LtFtUo2P" role="doc-biblioref">99</a>]</span>.
Among these IFNs, IFN- ùõΩ was already found to strongly inhibit the replication of other corona viruses, such as SARS-Cov, in cell culture, while IFN-ùõº and ùõæ were shown to be less effective in this context <span class="citation" data-cites="LtFtUo2P">[<a href="#ref-LtFtUo2P" role="doc-biblioref">99</a>]</span>.
There are evidences that patients with higher susceptibility to develop Acute respiratory distress syndrome (ARDS) show indeed deficiency of IFN- ùõΩ.
For instance, upon other Corona viruses infection IFN- ùõΩ expression and synthesis is impaired, so that the virus can in fact escape the innate immune response <span class="citation" data-cites="Nfu9kiae">[<a href="#ref-Nfu9kiae" role="doc-biblioref">100</a>]</span>.</p>
<p>On March 18 2020 Synairgen plc has received approval to start a phase II trial for SNG001, an IFN- ùõΩ-1a formulation to be delivered to lungs via inhalation.
SNG001 was already shown to be effective reducing viral load in swine flu in vivo model, as well as it has been shown to be effective in the protection from other Corona virus infection in vitro (Synairgen plc, press release).</p>
<h5 id="anticipated-mechanism-6">Anticipated Mechanism</h5>
<p>Why it may be useful</p>
<h5 id="current-evidence-9">Current Evidence</h5>
<p>A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.</p>
<h5 id="summary-9">Summary</h5>
<p>Summarize the state of interferons.</p>
<h3 id="vaccines">Vaccines</h3>
<p>Vaccines, widely recognized as one of the most significant advances in human health during the 20th century, can be used to bolser both individual and herd immunity to a virus by promoting the development of antibodies without infection.
[Are vaccines available for other coronaviruses or related viral illnesses?]
[What are some of the challenges to developing a vaccine? What needs to be taken into account about how the virus works?]
[Are there any challenges or opportunities unique to coronaviruses and/or SARS-CoV-2?]
[What are some approaches being tested or considered?]</p>
<h4 id="dna-vaccines">DNA Vaccines</h4>
<p>Brief background on the therapeutic.</p>
<h5 id="anticipated-mechanism-7">Anticipated Mechanism</h5>
<p>Why it may be useful</p>
<h5 id="current-evidence-10">Current Evidence</h5>
<p>A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.</p>
<h5 id="summary-10">Summary</h5>
<p>Summarize the state of the vaccine approach.</p>
<h4 id="rna-vaccines">RNA Vaccines</h4>
<p>Brief background on the therapeutic.</p>
<h5 id="anticipated-mechanism-8">Anticipated Mechanism</h5>
<p>Why it may be useful</p>
<h5 id="current-evidence-11">Current Evidence</h5>
<p>A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.</p>
<h5 id="summary-11">Summary</h5>
<p>Summarize the state of the vaccine approach.</p>
<h4 id="viral-particle-vaccines">Viral Particle Vaccines</h4>
<p>Brief background on the therapeutic.</p>
<h5 id="anticipated-mechanism-9">Anticipated Mechanism</h5>
<p>Why it may be useful</p>
<h5 id="current-evidence-12">Current Evidence</h5>
<p>A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.</p>
<h5 id="summary-12">Summary</h5>
<p>Summarize the state of the vaccine approach.</p>
<h4 id="oligonucleotide-therapies">Oligonucleotide Therapies</h4>
<p>Background</p>
<h5 id="anticipated-mechanism-10">Anticipated Mechanism</h5>
<p>Why it may be useful</p>
<h5 id="current-evidence-13">Current Evidence</h5>
<p>A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.</p>
<h5 id="summary-13">Summary</h5>
<p>Summarize the state of the neutralizing antibody approach.</p>
<h2 id="methods">Methods</h2>
<h3 id="article-selection-and-evaluation">Article Selection and Evaluation</h3>
<p>The authors solicited relevant articles to be submitted via <a href="https://github.com/greenelab/covid19-review">GitHub</a> for review.
Articles were classified as <em>diagnostic</em>, <em>therapeutic</em>, or <em>other</em>.
Following a framework often used for assessing medical literature, the review consisted of examining the methods used in the article, the assignment (whether the study was observational or randomized), the assessment, the results, the interpretation, and how well the study extrapolates <span class="citation" data-cites="17OQtAY4l">[<a href="#ref-17OQtAY4l" role="doc-biblioref">101</a>]</span>.</p>
<h4 id="diagnostic-papers">Diagnostic Papers</h4>
<h5 id="methods-1">Methods</h5>
<p>Reviewers began by describing the study question(s) being investigated by the article.
They then described the study population, the sample size, the prevalence of the disease in the study population, if in human subjects, the countries / regions considered, the demographics of participants, the setting, and any remaining inclusion / exclusion criteria considered.
They then described the reference test or ‚Äúgold standard,‚Äù if one was utilized.</p>
<h5 id="assignment">Assignment</h5>
<p>Reviewers described how the new and reference tests were assigned and any further details about the study design, for example whether the diagnostic test was biased towards sicker or healthier individuals or very clear-cut positive/negative cases.</p>
<h5 id="assessment">Assessment</h5>
<p>Reviewers described how the test was performed.
For example, if provided, reviewers described the technical details of the assays used, when measurements were taken and by whom for both the standard and reference diagnostic tests.
They then described how individuals were classified as positive or negative and whether there was evidence that the test results were precise or reproducible when repeated more than once.
Reviewers described whether there was any missing data, whether some participants underwent only one test, or whether there were individuals with inconclusive results.</p>
<h5 id="results">Results</h5>
<p>Reviewers reported the estimated sensitivity, specificity, positive predictive value (PPV), and negative predicted value (NPV), as well as the confidence bounds around these measures, if provided.</p>
<h5 id="interpretation">Interpretation</h5>
<p>Reviewers reported how well the test ruled in or rules out disease based on the population, if there were identified side effects, and patient adherence.</p>
<h5 id="extrapolation">Extrapolation</h5>
<p>Reviewers described how well this test will extrapolate outside the measured population.</p>
<h4 id="therapeutic-papers">Therapeutic Papers</h4>
<h5 id="methods-2">Methods</h5>
<p>Reviewers began by describing the study question(s) being investigated by the article.
They then described the study population, the sample size, the prevalence of the disease in the study population, if in human subjects, the countries / regions considered, the demographics of participants, the setting, and any remaining inclusion / exclusion criteria considered.</p>
<h5 id="assignment-1">Assignment</h5>
<p>Reviewers described how the treatment is assigned, whether it was an interventional or observational study, whether randomization took place, etc.</p>
<h5 id="assessment-1">Assessment</h5>
<h6 id="outcome-assessment">Outcome Assessment</h6>
<p>Reviewers described the outcome that was assessed and evaluated whether it was appropriate given the underlying study question.
They described whether there was any missing data, for example whether there were individuals lost to follow up.
They then describe whether there were any potential sources of bias, for example lack of blinding in a randomized controlled trial.</p>
<h6 id="statistical-methods-assessment">Statistical Methods Assessment</h6>
<p>Reviewers describe which statistical methods were used for inference and whether the methods were appropriate for the study.
They then described whether adjustments were made for possible confounders.</p>
<h5 id="results-1">Results</h5>
<p>Reviewers described the estimated association between the treatment and outcome.
They described measures of confidence or statistical significance, if provided.</p>
<h5 id="interpretation-1">Interpretation</h5>
<p>Reviewers described whether a causal claim could be made.
They described whether any side effects or interactions with other drugs were identified, as well as any subgroup findings.</p>
<h5 id="extrapolation-1">Extrapolation</h5>
<p>Reviewers describe how the study may extrapolate to a different species or population.</p>
<h3 id="collaborative-writing">Collaborative Writing</h3>
<p>Crowd-sourced writing with Manubot <span class="citation" data-cites="YuJbg3zO">[<a href="#ref-YuJbg3zO" role="doc-biblioref">102</a>]</span>.</p>
<h2 class="page_break_before" id="additional-items">Additional Items</h2>
<h3 id="competing-interests">Competing Interests</h3>
<table>
<thead>
<tr class="header">
<th>Author</th>
<th>Competing Interests</th>
<th>Last Reviewed</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Halie M. Rando</td>
<td>None</td>
<td>2020-03-22</td>
</tr>
<tr class="even">
<td>Casey S. Greene</td>
<td>None</td>
<td>2020-03-22</td>
</tr>
<tr class="odd">
<td>Michael P. Robson</td>
<td>None</td>
<td>2020-03-23</td>
</tr>
<tr class="even">
<td>Simina M. Boca</td>
<td>None</td>
<td>2020-03-23</td>
</tr>
<tr class="odd">
<td>Nils Wellhausen</td>
<td>None</td>
<td>2020-03-22</td>
</tr>
<tr class="even">
<td>Ronan Lordan</td>
<td>None</td>
<td>2020-03-25</td>
</tr>
<tr class="odd">
<td>Christian Brueffer</td>
<td>None</td>
<td>2020-03-25</td>
</tr>
<tr class="even">
<td>Sadipan Ray</td>
<td>None</td>
<td>2020-03-25</td>
</tr>
<tr class="odd">
<td>Lucy D'Agostino McGowan</td>
<td>None</td>
<td>2020-03-26</td>
</tr>
<tr class="even">
<td>Anthony Gitter</td>
<td>None</td>
<td>2020-03-26</td>
</tr>
<tr class="odd">
<td>Anna Ada Dattoli</td>
<td>None</td>
<td>2020-03-26</td>
</tr>
</tbody>
</table>
<h3 id="author-contributions">Author Contributions</h3>
<table>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<thead>
<tr class="header">
<th>Author</th>
<th>Contributions</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Halie M. Rando</td>
<td>Project Administration, Writing - Original Draft, Writing - Review &amp; Editing, Methodology</td>
</tr>
<tr class="even">
<td>Casey S. Greene</td>
<td>Conceptualization, Software</td>
</tr>
<tr class="odd">
<td>Michael P. Robson</td>
<td>Software</td>
</tr>
<tr class="even">
<td>Simina M. Boca</td>
<td>Methodology</td>
</tr>
<tr class="odd">
<td>Nils Wellhausen</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="even">
<td>Ronan Lordan</td>
<td>Writing - Original Draft, Writing - Review &amp; Editing</td>
</tr>
<tr class="odd">
<td>Christian Brueffer</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="even">
<td>Sadipan Ray</td>
<td>Writing - Original Draft</td>
</tr>
<tr class="odd">
<td>Lucy D'Agostino McGowan</td>
<td>Methodology, Writing - Original Draft</td>
</tr>
<tr class="even">
<td>Anthony Gitter</td>
<td>Methodology</td>
</tr>
<tr class="odd">
<td>Anna Ada Dattoli</td>
<td>Writing- original draft</td>
</tr>
</tbody>
</table>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-Vb5KFWnO">
<p>1. <br />
Cramer<br />
(2020-01-27) <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf</a></p>
</div>
<div id="ref-fLTiwToY">
<p>2. <br />
Ikejezie, Mr. Juniorcaius (WDC)<br />
(2020-01-28) <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200128-sitrep-8-ncov-cleared.pdf">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200128-sitrep-8-ncov-cleared.pdf</a></p>
</div>
<div id="ref-x9975iBK">
<p>3. <br />
Ikejezie, Mr. Juniorcaius (WDC)<br />
(2020-03-11) <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200311-sitrep-51-covid-19.pdf">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200311-sitrep-51-covid-19.pdf</a></p>
</div>
<div id="ref-11JitNwZt">
<p>4. <br />
Ikejezie, Mr. Juniorcaius (WDC)<br />
(2020-03-21) <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200321-sitrep-61-covid-19.pdf">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200321-sitrep-61-covid-19.pdf</a></p>
</div>
<div id="ref-G5NJrE75">
<p>5. <strong>SARS and MERS: recent insights into emerging coronaviruses</strong><br />
Emmie de Wit, Neeltje van Doremalen, Darryl Falzarano, Vincent J. Munster<br />
<em>Nature Reviews Microbiology</em> (2016-06-27) <a href="https://doi.org/f8v5cv">https://doi.org/f8v5cv</a><br />
DOI: <a href="https://doi.org/10.1038/nrmicro.2016.81">10.1038/nrmicro.2016.81</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27344959">27344959</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097822">PMC7097822</a></p>
</div>
<div id="ref-UBTUyMGV">
<p>6. <strong>Origin and evolution of pathogenic coronaviruses</strong><br />
Jie Cui, Fang Li, Zheng-Li Shi<br />
<em>Nature Reviews Microbiology</em> (2018-12-10) <a href="https://doi.org/ggh4vb">https://doi.org/ggh4vb</a><br />
DOI: <a href="https://doi.org/10.1038/s41579-018-0118-9">10.1038/s41579-018-0118-9</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30531947">30531947</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097006">PMC7097006</a></p>
</div>
<div id="ref-NEIoHDjq">
<p>7. <strong>The proximal origin of SARS-CoV-2</strong><br />
Kristian G. Andersen, Andrew Rambaut, W. Ian Lipkin, Edward C. Holmes, Robert F. Garry<br />
<em>Nature Medicine</em> (2020-03-17) <a href="https://doi.org/ggn4dn">https://doi.org/ggn4dn</a><br />
DOI: <a href="https://doi.org/10.1038/s41591-020-0820-9">10.1038/s41591-020-0820-9</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095063">PMC7095063</a></p>
</div>
<div id="ref-VSkK7CeP">
<p>8. <strong>A pneumonia outbreak associated with a new coronavirus of probable bat origin</strong><br />
Peng Zhou, Xing-Lou Yang, Xian-Guang Wang, Ben Hu, Lei Zhang, Wei Zhang, Hao-Rui Si, Yan Zhu, Bei Li, Chao-Lin Huang, ‚Ä¶ Zheng-Li Shi<br />
<em>Nature</em> (2020-02-03) <a href="https://doi.org/ggj5cg">https://doi.org/ggj5cg</a><br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2012-7">10.1038/s41586-020-2012-7</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32015507">32015507</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095418">PMC7095418</a></p>
</div>
<div id="ref-eQIU1Nbs">
<p>9. <strong>Pangolin homology associated with 2019-nCoV</strong><br />
Tao Zhang, Qunfu Wu, Zhigang Zhang<br />
<em>bioRxiv</em> (2020-02-20) <a href="https://doi.org/ggpvpt">https://doi.org/ggpvpt</a><br />
DOI: <a href="https://doi.org/10.1101/2020.02.19.950253">10.1101/2020.02.19.950253</a></p>
</div>
<div id="ref-16CD4voW1">
<p>10. <strong>The concept of R <sub>o</sub> in epidemic theory</strong><br />
J. A. P. Heesterbeek, K. Dietz<br />
<em>Statistica Neerlandica</em> (1996-03) <a href="https://doi.org/d29ch4">https://doi.org/d29ch4</a><br />
DOI: <a href="https://doi.org/10.1111/j.1467-9574.1996.tb01482.x">10.1111/j.1467-9574.1996.tb01482.x</a></p>
</div>
<div id="ref-lwH4gcts">
<p>11. <strong>Population Dynamics of Immune Responses to Persistent Viruses</strong><br />
M. A. Nowak, C. R. M. Bangham<br />
<em>Science</em> (1996-04-05) <a href="https://doi.org/fqgprd">https://doi.org/fqgprd</a><br />
DOI: <a href="https://doi.org/10.1126/science.272.5258.74">10.1126/science.272.5258.74</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8600540">8600540</a></p>
</div>
<div id="ref-1E0r4uZy9">
<p>12. <strong>Nowcasting and forecasting the potential domestic and international spread of the 2019-nCoV outbreak originating in Wuhan, China: a modelling study</strong><br />
Joseph T Wu, Kathy Leung, Gabriel M Leung<br />
<em>The Lancet</em> (2020-01) <a href="https://doi.org/ggjvr7">https://doi.org/ggjvr7</a><br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30260-9">10.1016/s0140-6736(20)30260-9</a></p>
</div>
<div id="ref-LHtuVmaq">
<p>13. <strong>The reproductive number of COVID-19 is higher compared to SARS coronavirus</strong><br />
Ying Liu, Albert A Gayle, Annelies Wilder-Smith, Joacim Rockl√∂v<br />
<em>Journal of Travel Medicine</em> (2020-02-13) <a href="https://doi.org/ggnntv">https://doi.org/ggnntv</a><br />
DOI: <a href="https://doi.org/10.1093/jtm/taaa021">10.1093/jtm/taaa021</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32052846">32052846</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7074654">PMC7074654</a></p>
</div>
<div id="ref-mHYmt0mv">
<p>14. <strong>Substantial undocumented infection facilitates the rapid dissemination of novel coronavirus (SARS-CoV2)</strong><br />
Ruiyun Li, Sen Pei, Bin Chen, Yimeng Song, Tao Zhang, Wan Yang, Jeffrey Shaman<br />
<em>Science</em> (2020-03-16) <a href="https://doi.org/ggn6c2">https://doi.org/ggn6c2</a><br />
DOI: <a href="https://doi.org/10.1126/science.abb3221">10.1126/science.abb3221</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32179701">32179701</a></p>
</div>
<div id="ref-vITui6ac">
<p>15. <strong>Epidemiological parameters of coronavirus disease 2019: a pooled analysis of publicly reported individual data of 1155 cases from seven countries</strong><br />
Shujuan Ma, Jiayue Zhang, Minyan Zeng, Qingping Yun, Wei Guo, Yixiang Zheng, Shi Zhao, Maggie H Wang, Zuyao Yang<br />
<em>medRxiv</em> (2020-03-24) <a href="https://doi.org/ggqhzz">https://doi.org/ggqhzz</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.21.20040329">10.1101/2020.03.21.20040329</a></p>
</div>
<div id="ref-nxM0rP5R">
<p>16. <strong>Early Transmissibility Assessment of a Novel Coronavirus in Wuhan, China</strong><br />
Maimuna Majumder, Kenneth D. Mandl<br />
<em>SSRN Electronic Journal</em> (2020) <a href="https://doi.org/ggqhz3">https://doi.org/ggqhz3</a><br />
DOI: <a href="https://doi.org/10.2139/ssrn.3524675">10.2139/ssrn.3524675</a></p>
</div>
<div id="ref-WLc2UMgQ">
<p>17. <strong>Time-varying transmission dynamics of Novel Coronavirus Pneumonia in China</strong><br />
Tao Liu, Jianxiong Hu, Jianpeng Xiao, Guanhao He, Min Kang, Zuhua Rong, Lifeng Lin, Haojie Zhong, Qiong Huang, Aiping Deng, ‚Ä¶ Wenjun Ma<br />
<em>bioRxiv</em> (2020-02-13) <a href="https://doi.org/dkx9">https://doi.org/dkx9</a><br />
DOI: <a href="https://doi.org/10.1101/2020.01.25.919787">10.1101/2020.01.25.919787</a></p>
</div>
<div id="ref-10bBqMHH7">
<p>18. <strong>Estimation of the reproductive number of novel coronavirus (COVID-19) and the probable outbreak size on the Diamond Princess cruise ship: A data-driven analysis</strong><br />
Sheng Zhang, MengYuan Diao, Wenbo Yu, Lei Pei, Zhaofen Lin, Dechang Chen<br />
<em>International Journal of Infectious Diseases</em> (2020-04) <a href="https://doi.org/ggpx56">https://doi.org/ggpx56</a><br />
DOI: <a href="https://doi.org/10.1016/j.ijid.2020.02.033">10.1016/j.ijid.2020.02.033</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32097725">32097725</a></p>
</div>
<div id="ref-ITh0Anof">
<p>19. <strong>Estimation of the Transmission Risk of the 2019-nCoV and Its Implication for Public Health Interventions</strong><br />
Tang, Wang, Li, Bragazzi, Tang, Xiao, Wu<br />
<em>Journal of Clinical Medicine</em> (2020-02-07) <a href="https://doi.org/ggmkf4">https://doi.org/ggmkf4</a><br />
DOI: <a href="https://doi.org/10.3390/jcm9020462">10.3390/jcm9020462</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32046137">32046137</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7074281">PMC7074281</a></p>
</div>
<div id="ref-uTI5OuZt">
<p>20. <a href="https://doi.org/10.1101/2020.01.27.20018952">https://doi.org/10.1101/2020.01.27.20018952</a></p>
</div>
<div id="ref-Yj8Xh4Wz">
<p>21. <strong>Modelling the epidemic trend of the 2019 novel coronavirus outbreak in China</strong><br />
Mingwang Shen, Zhihang Peng, Yanni Xiao, Lei Zhang<br />
<em>bioRxiv</em> (2020-01-25) <a href="https://doi.org/ggqhzw">https://doi.org/ggqhzw</a><br />
DOI: <a href="https://doi.org/10.1101/2020.01.23.916726">10.1101/2020.01.23.916726</a></p>
</div>
<div id="ref-ZzrrVDoE">
<p>22. <strong>Novel coronavirus 2019-nCoV: early estimation of epidemiological parameters and epidemic predictions</strong><br />
Jonathan M Read, Jessica RE Bridgen, Derek AT Cummings, Antonia Ho, Chris P Jewell<br />
<em>medRxiv</em> (2020-01-28) <a href="https://doi.org/dkzb">https://doi.org/dkzb</a><br />
DOI: <a href="https://doi.org/10.1101/2020.01.23.20018549">10.1101/2020.01.23.20018549</a></p>
</div>
<div id="ref-itj26agd">
<p>23. <strong>Mechanistic-statistical SIR modelling for early estimation of the actual number of cases and mortality rate from COVID-19</strong><br />
Lionel Roques, Etienne Klein, Julien Papaix, Samuel Soubeyrand<br />
<em>medRxiv</em> (2020-03-24) <a href="https://doi.org/ggqhz2">https://doi.org/ggqhz2</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.22.20040915">10.1101/2020.03.22.20040915</a></p>
</div>
<div id="ref-YfOcGRRa">
<p>24. <strong>Potential roles of social distancing in mitigating the spread of coronavirus disease 2019 (COVID-19) in South Korea</strong><br />
Sang Woo Park, Kaiyuan Sun, C√©cile Viboud, Bryan T Grenfell, Jonathan Dushoff<br />
<em>GitHub</em> (2020) <a href="https://github.com/parksw3/Korea-analysis/blob/master/v1/korea.pdf">https://github.com/parksw3/Korea-analysis/blob/master/v1/korea.pdf</a></p>
</div>
<div id="ref-tiRfUgvs">
<p>25. <strong>Early dynamics of transmission and control of COVID-19: a mathematical modelling study</strong><br />
Adam J Kucharski, Timothy W Russell, Charlie Diamond, Yang Liu, John Edmunds, Sebastian Funk, Rosalind M Eggo, Fiona Sun, Mark Jit, James D Munday, ‚Ä¶ Stefan Flasche<br />
<em>The Lancet Infectious Diseases</em> (2020-03) <a href="https://doi.org/ggptcf">https://doi.org/ggptcf</a><br />
DOI: <a href="https://doi.org/10.1016/s1473-3099(20)30144-4">10.1016/s1473-3099(20)30144-4</a></p>
</div>
<div id="ref-TH4ymqvJ">
<p>26. <strong>Estimating the reproduction number of COVID-19 in Iran using epidemic modeling</strong><br />
Ebrahim Sahafizadeh, Samaneh Sartoli<br />
<em>medRxiv</em> (2020-03-29) <a href="https://doi.org/ggqhzx">https://doi.org/ggqhzx</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.20.20038422">10.1101/2020.03.20.20038422</a></p>
</div>
<div id="ref-5MhhHQO0">
<p>27. <strong>Systems Approaches to Biology and Disease Enable Translational Systems Medicine</strong><br />
Leroy Hood, Qiang Tian<br />
<em>Genomics, Proteomics &amp; Bioinformatics</em> (2012-08) <a href="https://doi.org/f4f599">https://doi.org/f4f599</a><br />
DOI: <a href="https://doi.org/10.1016/j.gpb.2012.08.004">10.1016/j.gpb.2012.08.004</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23084773">23084773</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3844613">PMC3844613</a></p>
</div>
<div id="ref-8esIIWt9">
<p>28. <strong>A systems approach to infectious disease</strong><br />
Manon Eckhardt, Judd F. Hultquist, Robyn M. Kaake, Ruth H√ºttenhain, Nevan J. Krogan<br />
<em>Nature Reviews Genetics</em> (2020-02-14) <a href="https://doi.org/ggnv63">https://doi.org/ggnv63</a><br />
DOI: <a href="https://doi.org/10.1038/s41576-020-0212-5">10.1038/s41576-020-0212-5</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32060427">32060427</a></p>
</div>
<div id="ref-8vUHj9cm">
<p>29. <strong>Differential expression of serum/plasma proteins in various infectious diseases: Specific or nonspecific signatures</strong><br />
Sandipan Ray, Sandip K. Patel, Vipin Kumar, Jagruti Damahe, Sanjeeva Srivastava<br />
<em>PROTEOMICS - Clinical Applications</em> (2014-02) <a href="https://doi.org/f2px3h">https://doi.org/f2px3h</a><br />
DOI: <a href="https://doi.org/10.1002/prca.201300074">10.1002/prca.201300074</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24293340">24293340</a></p>
</div>
<div id="ref-vHq0hOWQ">
<p>30. <strong>A new coronavirus associated with human respiratory disease in China</strong><br />
Fan Wu, Su Zhao, Bin Yu, Yan-Mei Chen, Wen Wang, Zhi-Gang Song, Yi Hu, Zhao-Wu Tao, Jun-Hua Tian, Yuan-Yuan Pei, ‚Ä¶ Yong-Zhen Zhang<br />
<em>Nature</em> (2020-02-03) <a href="https://doi.org/dk2w">https://doi.org/dk2w</a><br />
DOI: <a href="https://doi.org/10.1038/s41586-020-2008-3">10.1038/s41586-020-2008-3</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32015508">32015508</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094943">PMC7094943</a></p>
</div>
<div id="ref-1Fqilxaum">
<p>31. <strong>Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</strong><br />
Roujian Lu, Xiang Zhao, Juan Li, Peihua Niu, Bo Yang, Honglong Wu, Wenling Wang, Hao Song, Baoying Huang, Na Zhu, ‚Ä¶ Wenjie Tan<br />
<em>The Lancet</em> (2020-02) <a href="https://doi.org/ggjr43">https://doi.org/ggjr43</a><br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30251-8">10.1016/s0140-6736(20)30251-8</a></p>
</div>
<div id="ref-CqQVkaqj">
<p>32. <strong>Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</strong><br />
Daniel Wrapp, Nianshuang Wang, Kizzmekia S. Corbett, Jory A. Goldsmith, Ching-Lin Hsieh, Olubukola Abiona, Barney S. Graham, Jason S. McLellan<br />
<em>Science</em> (2020-02-19) <a href="https://doi.org/ggmtk2">https://doi.org/ggmtk2</a><br />
DOI: <a href="https://doi.org/10.1126/science.abb2507">10.1126/science.abb2507</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32075877">32075877</a></p>
</div>
<div id="ref-CtMA3hTi">
<p>33. <strong>SARS-CoV-2 infected host cell proteomics reveal potential therapy targets</strong><br />
Denisa Bojkova, Kevin Klann, Benjamin Koch, Marek Widera, David Krause, Sandra Ciesek, Jindrich Cinatl, Christian M√ºnch<br />
(2020-03-11) <a href="https://doi.org/ggn4ds">https://doi.org/ggn4ds</a><br />
DOI: <a href="https://doi.org/10.21203/rs.3.rs-17218/v1">10.21203/rs.3.rs-17218/v1</a></p>
</div>
<div id="ref-15l3di3Wj">
<p>34. <strong>SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor</strong><br />
Markus Hoffmann, Hannah Kleine-Weber, Simon Schroeder, Nadine Kr√ºger, Tanja Herrler, Sandra Erichsen, Tobias S. Schiergens, Georg Herrler, Nai-Huei Wu, Andreas Nitsche, ‚Ä¶ Stefan P√∂hlmann<br />
<em>Cell</em> (2020-03) <a href="https://doi.org/ggnq74">https://doi.org/ggnq74</a><br />
DOI: <a href="https://doi.org/10.1016/j.cell.2020.02.052">10.1016/j.cell.2020.02.052</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32142651">32142651</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7102627">PMC7102627</a></p>
</div>
<div id="ref-GpnngtWK">
<p>35. <strong>Plasma proteome of severe acute respiratory syndrome analyzed by two-dimensional gel electrophoresis and mass spectrometry</strong><br />
J.-H. Chen, Y.-W. Chang, C.-W. Yao, T.-S. Chiueh, S.-C. Huang, K.-Y. Chien, A. Chen, F.-Y. Chang, C.-H. Wong, Y.-J. Chen<br />
<em>Proceedings of the National Academy of Sciences</em> (2004-11-30) <a href="https://doi.org/dtv8sx">https://doi.org/dtv8sx</a><br />
DOI: <a href="https://doi.org/10.1073/pnas.0407992101">10.1073/pnas.0407992101</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15572443">15572443</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC535397">PMC535397</a></p>
</div>
<div id="ref-JLTf2Fwb">
<p>36. <strong>Analysis of multimerization of the SARS coronavirus nucleocapsid protein</strong><br />
Runtao He, Frederick Dobie, Melissa Ballantine, Andrew Leeson, Yan Li, Nathalie Bastien, Todd Cutts, Anton Andonov, Jingxin Cao, Timothy F. Booth, ‚Ä¶ Xuguang Li<br />
<em>Biochemical and Biophysical Research Communications</em> (2004-04) <a href="https://doi.org/dbfwr9">https://doi.org/dbfwr9</a><br />
DOI: <a href="https://doi.org/10.1016/j.bbrc.2004.02.074">10.1016/j.bbrc.2004.02.074</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15020242">15020242</a></p>
</div>
<div id="ref-eyBAUUCl">
<p>37. <strong>Nutraceuticals have potential for boosting the type 1 interferon response to RNA viruses including influenza and coronavirus</strong><br />
Mark F. McCarty, James J. DiNicolantonio<br />
<em>Progress in Cardiovascular Diseases</em> (2020-02) <a href="https://doi.org/ggpwx2">https://doi.org/ggpwx2</a><br />
DOI: <a href="https://doi.org/10.1016/j.pcad.2020.02.007">10.1016/j.pcad.2020.02.007</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32061635">32061635</a></p>
</div>
<div id="ref-GT7yDBjv">
<p>38. <strong>Reducing mortality from 2019-nCoV: host-directed therapies should be an option</strong><br />
Alimuddin Zumla, David S Hui, Esam I Azhar, Ziad A Memish, Markus Maeurer<br />
<em>The Lancet</em> (2020-02) <a href="https://doi.org/ggkd3b">https://doi.org/ggkd3b</a><br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30305-6">10.1016/s0140-6736(20)30305-6</a></p>
</div>
<div id="ref-vEFSIErd">
<p>39. <strong>Coronavirus Disease 2019 (COVID-19)</strong><br />
CDC<br />
<em>Centers for Disease Control and Prevention</em> (2020-02-11) <a href="https://www.cdc.gov/coronavirus/2019-ncov/hcp/clinical-guidance-management-patients.html">https://www.cdc.gov/coronavirus/2019-ncov/hcp/clinical-guidance-management-patients.html</a></p>
</div>
<div id="ref-10THxyeCg">
<p>40. <strong>Clinical course and risk factors for mortality of adult inpatients with COVID-19 in Wuhan, China: a retrospective cohort study</strong><br />
Fei Zhou, Ting Yu, Ronghui Du, Guohui Fan, Ying Liu, Zhibo Liu, Jie Xiang, Yeming Wang, Bin Song, Xiaoying Gu, ‚Ä¶ Bin Cao<br />
<em>The Lancet</em> (2020-03) <a href="https://doi.org/ggnxb3">https://doi.org/ggnxb3</a><br />
DOI: <a href="https://doi.org/10.1016/s0140-6736(20)30566-3">10.1016/s0140-6736(20)30566-3</a></p>
</div>
<div id="ref-bZMKqj6e">
<p>41. <strong>Into the Eye of the Cytokine Storm</strong><br />
J. R. Tisoncik, M. J. Korth, C. P. Simmons, J. Farrar, T. R. Martin, M. G. Katze<br />
<em>Microbiology and Molecular Biology Reviews</em> (2012-03-05) <a href="https://doi.org/f4n9h2">https://doi.org/f4n9h2</a><br />
DOI: <a href="https://doi.org/10.1128/mmbr.05015-11">10.1128/mmbr.05015-11</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22390970">22390970</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3294426">PMC3294426</a></p>
</div>
<div id="ref-1AVsj9DAP">
<p>42. <strong>Induction of cell cycle arrest and B cell terminal differentiation by CDK inhibitor p18(INK4c) and IL-6.</strong><br />
L Morse, D Chen, D Franklin, Y Xiong, S Chen-Kiang<br />
<em>Immunity</em> (1997-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/9052836">https://www.ncbi.nlm.nih.gov/pubmed/9052836</a><br />
DOI: <a href="https://doi.org/10.1016/s1074-7613(00)80241-1">10.1016/s1074-7613(00)80241-1</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/9052836">9052836</a></p>
</div>
<div id="ref-J7evEkJG">
<p>43. <strong>The effects of hybridoma growth factor in conditioned media upon the growth, cloning, and antibody production of heterohybridoma cell lines.</strong><br />
Y Zhu, B Jin, C Sun, C Huang, X Liu<br />
<em>Human antibodies and hybridomas</em> (1993-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/8431556">https://www.ncbi.nlm.nih.gov/pubmed/8431556</a><br />
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8431556">8431556</a></p>
</div>
<div id="ref-glBmWODR">
<p>44. <strong>Interleukin-6 is the central tumor growth factor in vitro and in vivo in multiple myeloma.</strong><br />
B Klein, XG Zhang, M Jourdan, JM Boiron, M Portier, ZY Lu, J Wijdenes, J Brochier, R Bataille<br />
<em>European cytokine network</em> <a href="https://www.ncbi.nlm.nih.gov/pubmed/2104241">https://www.ncbi.nlm.nih.gov/pubmed/2104241</a><br />
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/2104241">2104241</a></p>
</div>
<div id="ref-eW4MC6Do">
<p>45. <strong>The Biology and Medical Implications of Interleukin-6</strong><br />
T. Tanaka, T. Kishimoto<br />
<em>Cancer Immunology Research</em> (2014-04-03) <a href="https://doi.org/f56dzb">https://doi.org/f56dzb</a><br />
DOI: <a href="https://doi.org/10.1158/2326-6066.cir-14-0022">10.1158/2326-6066.cir-14-0022</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24764575">24764575</a></p>
</div>
<div id="ref-QkTGQUcj">
<p>46. <strong>Up-regulation of IL-6 and TNF-Œ± induced by SARS-coronavirus spike protein in murine macrophages via NF-Œ∫B pathway</strong><br />
Wei Wang, Linbai Ye, Li Ye, Baozong Li, Bo Gao, Yingchun Zeng, Lingbao Kong, Xiaonan Fang, Hong Zheng, Zhenghui Wu, Yinglong She<br />
<em>Virus Research</em> (2007-09) <a href="https://doi.org/bm7m55">https://doi.org/bm7m55</a><br />
DOI: <a href="https://doi.org/10.1016/j.virusres.2007.02.007">10.1016/j.virusres.2007.02.007</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17532082">17532082</a></p>
</div>
<div id="ref-NNFGje9g">
<p>47. <strong>Expression of elevated levels of pro-inflammatory cytokines in SARS-CoV-infected ACE2 <sup>+</sup> cells in SARS patients: relation to the acute lung injury and pathogenesis of SARS</strong><br />
L He, Y Ding, Q Zhang, X Che, Y He, H Shen, H Wang, Z Li, L Zhao, J Geng, ‚Ä¶ S Jiang<br />
<em>The Journal of Pathology</em> (2006-11) <a href="https://doi.org/bwb8ns">https://doi.org/bwb8ns</a><br />
DOI: <a href="https://doi.org/10.1002/path.2067">10.1002/path.2067</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17031779">17031779</a></p>
</div>
<div id="ref-f61jsRKY">
<p>48. <strong>Cytokine Balance in the Lungs of Patients with Acute Respiratory Distress Syndrome</strong><br />
WILLIAM‚ÄÇY. PARK, RICHARD‚ÄÇB. GOODMAN, KENNETH‚ÄÇP. STEINBERG, JOHN‚ÄÇT. RUZINSKI, FRANK RADELLA, DAVID‚ÄÇR. PARK, JEROME PUGIN, SHAWN‚ÄÇJ. SKERRETT, LEONARD‚ÄÇD. HUDSON, THOMAS‚ÄÇR. MARTIN<br />
<em>American Journal of Respiratory and Critical Care Medicine</em> (2001-11-15) <a href="https://doi.org/ggqfq7">https://doi.org/ggqfq7</a><br />
DOI: <a href="https://doi.org/10.1164/ajrccm.164.10.2104013">10.1164/ajrccm.164.10.2104013</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11734443">11734443</a></p>
</div>
<div id="ref-suIFBW">
<p>49. <strong>Pathogenic human coronavirus infections: causes and consequences of cytokine storm and immunopathology</strong><br />
Rudragouda Channappanavar, Stanley Perlman<br />
<em>Seminars in Immunopathology</em> (2017-05-02) <a href="https://doi.org/ggqf2w">https://doi.org/ggqf2w</a><br />
DOI: <a href="https://doi.org/10.1007/s00281-017-0629-x">10.1007/s00281-017-0629-x</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28466096">28466096</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7079893">PMC7079893</a></p>
</div>
<div id="ref-1BxkPZYnj">
<p>50. <strong>Plasticity and cross-talk of Interleukin 6-type cytokines</strong><br />
Christoph Garbers, Heike M. Hermanns, Fred Schaper, Gerhard M√ºller-Newen, Joachim Gr√∂tzinger, Stefan Rose-John, J√ºrgen Scheller<br />
<em>Cytokine &amp; Growth Factor Reviews</em> (2012-06) <a href="https://doi.org/f3z743">https://doi.org/f3z743</a><br />
DOI: <a href="https://doi.org/10.1016/j.cytogfr.2012.04.001">10.1016/j.cytogfr.2012.04.001</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22595692">22595692</a></p>
</div>
<div id="ref-1EEloBjzn">
<p>51. <strong>Soluble receptors for cytokines and growth factors: generation and biological function</strong><br />
S Rose-John, PC Heinrich<br />
<em>Biochemical Journal</em> (1994-06-01) <a href="https://doi.org/ggqmgd">https://doi.org/ggqmgd</a><br />
DOI: <a href="https://doi.org/10.1042/bj3000281">10.1042/bj3000281</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8002928">8002928</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1138158">PMC1138158</a></p>
</div>
<div id="ref-VXDsRwHC">
<p>52. <strong>Hall of Fame among Pro-inflammatory Cytokines: Interleukin-6 Gene and Its Transcriptional Regulation Mechanisms</strong><br />
Yang Luo, Song Guo Zheng<br />
<em>Frontiers in Immunology</em> (2016-12-19) <a href="https://doi.org/ggqmgv">https://doi.org/ggqmgv</a><br />
DOI: <a href="https://doi.org/10.3389/fimmu.2016.00604">10.3389/fimmu.2016.00604</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28066415">28066415</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5165036">PMC5165036</a></p>
</div>
<div id="ref-1H8o8TFtM">
<p>53. <strong>Interleukin-6; pathogenesis and treatment of autoimmune inflammatory diseases</strong><br />
Toshio Tanaka, Masashi Narazaki, Kazuya Masuda, Tadamitsu Kishimoto<br />
<em>Inflammation and Regeneration</em> (2013) <a href="https://doi.org/ggqmgt">https://doi.org/ggqmgt</a><br />
DOI: <a href="https://doi.org/10.2492/inflammregen.33.054">10.2492/inflammregen.33.054</a></p>
</div>
<div id="ref-JgXCBFEo">
<p>54. <strong>Genentech Launches Phase III Trial of Actemra as Coronavirus Treatment</strong><br />
GEN - Genetic Engineering and Biotechnology News<br />
(2020-03-19) <a href="https://www.genengnews.com/virology/coronavirus/genentech-launches-phase-iii-trial-of-actemra-as-coronavirus-treatment/">https://www.genengnews.com/virology/coronavirus/genentech-launches-phase-iii-trial-of-actemra-as-coronavirus-treatment/</a></p>
</div>
<div id="ref-123fXWICW">
<p>55. <strong>COVID-19 ‚Äì Italy launches an independent trial on tocilizumab</strong><br />
COVID-19 ‚Äì Italy launches an independent trial on tocilizumab | Univadis<br />
<a href="https://www.univadis.co.uk/viewarticle/covid-19-italy-launches-an-independent-trial-on-tocilizumab-715741">https://www.univadis.co.uk/viewarticle/covid-19-italy-launches-an-independent-trial-on-tocilizumab-715741</a></p>
</div>
<div id="ref-TgV8gb30">
<p>56. <strong>Risk of infections in rheumatoid arthritis patients treated with tocilizumab</strong><br />
Veronika R. Lang, Matthias Englbrecht, J√ºrgen Rech, Hubert N√ºsslein, Karin Manger, Florian Schuch, Hans-Peter Tony, Martin Fleck, Bernhard Manger, Georg Schett, Jochen Zwerina<br />
<em>Rheumatology</em> (2012-05) <a href="https://doi.org/d3b3rh">https://doi.org/d3b3rh</a><br />
DOI: <a href="https://doi.org/10.1093/rheumatology/ker223">10.1093/rheumatology/ker223</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21865281">21865281</a></p>
</div>
<div id="ref-IJzmtJOr">
<p>57. <strong>Short-course tocilizumab increases risk of hepatitis B virus reactivation in patients with rheumatoid arthritis: a prospective clinical observation</strong><br />
Le-Feng Chen, Ying-Qian Mo, Jun Jing, Jian-Da Ma, Dong-Hui Zheng, Lie Dai<br />
<em>International Journal of Rheumatic Diseases</em> (2017-07) <a href="https://doi.org/f9pbc5">https://doi.org/f9pbc5</a><br />
DOI: <a href="https://doi.org/10.1111/1756-185x.13010">10.1111/1756-185x.13010</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28160426">28160426</a></p>
</div>
<div id="ref-14gthSuhm">
<p>58. <strong>Introduction to modern virology</strong><br />
N. J. Dimmock, A. J. Easton, K. N. Leppard<br />
<em>Blackwell Pub</em> (2007)<br />
ISBN: <a href="https://worldcat.org/isbn/9781405136457">9781405136457</a></p>
</div>
<div id="ref-tn6j6IZ0">
<p>59. <strong>Coronaviruses</strong><br />
Methods in Molecular Biology<br />
(2015) <a href="https://doi.org/ggqfqx">https://doi.org/ggqfqx</a><br />
DOI: <a href="https://doi.org/10.1007/978-1-4939-2438-7">10.1007/978-1-4939-2438-7</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25870870">25870870</a></p>
</div>
<div id="ref-Sh1FwWps">
<p>60. <strong>The potential chemical structure of anti‚ÄêSARS‚ÄêCoV‚Äê2 RNA‚Äêdependent RNA polymerase</strong><br />
Jrhau Lung, Yu‚ÄêShih Lin, Yao‚ÄêHsu Yang, Yu‚ÄêLun Chou, Li‚ÄêHsin Shu, Yu‚ÄêChing Cheng, Hung Te Liu, Ching‚ÄêYuan Wu<br />
<em>Journal of Medical Virology</em> (2020-03-13) <a href="https://doi.org/ggp6fm">https://doi.org/ggp6fm</a><br />
DOI: <a href="https://doi.org/10.1002/jmv.25761">10.1002/jmv.25761</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32167173">32167173</a></p>
</div>
<div id="ref-ukZRKllk">
<p>61. <strong>The Ambiguous Base-Pairing and High Substrate Efficiency of T-705 (Favipiravir) Ribofuranosyl 5‚Ä≤-Triphosphate towards Influenza A Virus Polymerase</strong><br />
Zhinan Jin, Lucas K. Smith, Vivek K. Rajwanshi, Baek Kim, Jerome Deval<br />
<em>PLoS ONE</em> (2013-07-10) <a href="https://doi.org/f5br92">https://doi.org/f5br92</a><br />
DOI: <a href="https://doi.org/10.1371/journal.pone.0068347">10.1371/journal.pone.0068347</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23874596">23874596</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3707847">PMC3707847</a></p>
</div>
<div id="ref-gY3NNmLM">
<p>62. <strong>Successful treatment of advanced Ebola virus infection with T-705 (favipiravir) in a small animal model</strong><br />
Lisa Oestereich, Anja L√ºdtke, Stephanie Wurr, Toni Rieger, C√©sar Mu√±oz-Fontela, Stephan G√ºnther<br />
<em>Antiviral Research</em> (2014-05) <a href="https://doi.org/f524m3">https://doi.org/f524m3</a><br />
DOI: <a href="https://doi.org/10.1016/j.antiviral.2014.02.014">10.1016/j.antiviral.2014.02.014</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24583123">24583123</a></p>
</div>
<div id="ref-13Oz7yRPx">
<p>63. <strong>WITHDRAWN: Experimental Treatment with Favipiravir for COVID-19: An Open-Label Control Study</strong><br />
Qingxian Cai, Minghui Yang, Dongjing Liu, Jun Chen, Dan Shu, Junxia Xia, Xuejiao Liao, Yuanbo Gu, Qiue Cai, Yang Yang, ‚Ä¶ Lei Liu<br />
<em>Engineering</em> (2020-03) <a href="https://doi.org/ggpprd">https://doi.org/ggpprd</a><br />
DOI: <a href="https://doi.org/10.1016/j.eng.2020.03.007">10.1016/j.eng.2020.03.007</a></p>
</div>
<div id="ref-gnfviuGp">
<p>64. <strong>A Randomized, Controlled Trial of Ebola Virus Disease Therapeutics</strong><br />
Sabue Mulangu, Lori E. Dodd, Richard T. Davey, Olivier Tshiani Mbaya, Michael Proschan, Daniel Mukadi, Mariano Lusakibanza Manzo, Didier Nzolo, Antoine Tshomba Oloma, Augustin Ibanda, ‚Ä¶ the PALM Writing Group<br />
<em>New England Journal of Medicine</em> (2019-12-12) <a href="https://doi.org/ggqmx4">https://doi.org/ggqmx4</a><br />
DOI: <a href="https://doi.org/10.1056/nejmoa1910993">10.1056/nejmoa1910993</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31774950">31774950</a></p>
</div>
<div id="ref-YaxKcqq8">
<p>65. <strong>The antiviral compound remdesivir potently inhibits RNA-dependent RNA polymerase from Middle East respiratory syndrome coronavirus</strong><br />
Calvin J Gordon, Egor P Tchesnokov, Joy Y. Feng, Danielle P Porter, Matthias Gotte<br />
<em>Journal of Biological Chemistry</em> (2020-02-24) <a href="https://doi.org/ggqm6x">https://doi.org/ggqm6x</a><br />
DOI: <a href="https://doi.org/10.1074/jbc.ac120.013056">10.1074/jbc.ac120.013056</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32094225">32094225</a></p>
</div>
<div id="ref-lIko9Imd">
<p>66. <strong>Coronavirus Susceptibility to the Antiviral Remdesivir (GS-5734) Is Mediated by the Viral Polymerase and the Proofreading Exoribonuclease</strong><br />
Maria L. Agostini, Erica L. Andres, Amy C. Sims, Rachel L. Graham, Timothy P. Sheahan, Xiaotao Lu, Everett Clinton Smith, James Brett Case, Joy Y. Feng, Robert Jordan, ‚Ä¶ Mark R. Denison<br />
<em>mBio</em> (2018-03-06) <a href="https://doi.org/gc45v6">https://doi.org/gc45v6</a><br />
DOI: <a href="https://doi.org/10.1128/mbio.00221-18">10.1128/mbio.00221-18</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29511076">29511076</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5844999">PMC5844999</a></p>
</div>
<div id="ref-IhdcvbzC">
<p>67. <strong>Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro</strong><br />
Manli Wang, Ruiyuan Cao, Leike Zhang, Xinglou Yang, Jia Liu, Mingyue Xu, Zhengli Shi, Zhihong Hu, Wu Zhong, Gengfu Xiao<br />
<em>Cell Research</em> (2020-02-04) <a href="https://doi.org/ggkbsg">https://doi.org/ggkbsg</a><br />
DOI: <a href="https://doi.org/10.1038/s41422-020-0282-0">10.1038/s41422-020-0282-0</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32020029">32020029</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054408">PMC7054408</a></p>
</div>
<div id="ref-1H0NPTMw9">
<p>68. <strong>Broad-spectrum antiviral GS-5734 inhibits both epidemic and zoonotic coronaviruses</strong><br />
Timothy P. Sheahan, Amy C. Sims, Rachel L. Graham, Vineet D. Menachery, Lisa E. Gralinski, James B. Case, Sarah R. Leist, Krzysztof Pyrc, Joy Y. Feng, Iva Trantcheva, ‚Ä¶ Ralph S. Baric<br />
<em>Science Translational Medicine</em> (2017-06-28) <a href="https://doi.org/gc3grb">https://doi.org/gc3grb</a><br />
DOI: <a href="https://doi.org/10.1126/scitranslmed.aal3653">10.1126/scitranslmed.aal3653</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28659436">28659436</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5567817">PMC5567817</a></p>
</div>
<div id="ref-D4HwvRMd">
<p>69. <strong>Did an experimental drug help a U.S. coronavirus patient?</strong><br />
Jon Cohen<br />
<em>Science</em> (2020-03-13) <a href="https://doi.org/ggqm62">https://doi.org/ggqm62</a><br />
DOI: <a href="https://doi.org/10.1126/science.abb7243">10.1126/science.abb7243</a></p>
</div>
<div id="ref-zoLViMNg">
<p>70. <strong>First 12 patients with coronavirus disease 2019 (COVID-19) in the United States</strong><br />
Stephanie A. Kujawski, Karen K Wong, Jennifer P. Collins, Lauren Epstein, Marie E. Killerby, Claire M. Midgley, Glen R. Abedi, N. Seema Ahmed, Olivia Almendares, Francisco N. Alvarez, ‚Ä¶ Jing Zhang<br />
<em>medRxiv</em> (2020-03-12) <a href="https://doi.org/ggqm6z">https://doi.org/ggqm6z</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.09.20032896">10.1101/2020.03.09.20032896</a></p>
</div>
<div id="ref-gleq16qf">
<p>71. <strong>First Case of 2019 Novel Coronavirus in the United States</strong><br />
Michelle L. Holshue, Chas DeBolt, Scott Lindquist, Kathy H. Lofy, John Wiesman, Hollianne Bruce, Christopher Spitters, Keith Ericson, Sara Wilkerson, Ahmet Tural, ‚Ä¶ Satish K. Pillai<br />
<em>New England Journal of Medicine</em> (2020-01-31) <a href="https://doi.org/ggjvr6">https://doi.org/ggjvr6</a><br />
DOI: <a href="https://doi.org/10.1056/nejmoa2001191">10.1056/nejmoa2001191</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32004427">32004427</a></p>
</div>
<div id="ref-8a4WEEFh">
<p>72. <strong>Study to Evaluate the Safety and Antiviral Activity of Remdesivir (GS-5734‚Ñ¢) in Participants With Moderate Coronavirus Disease (COVID-19) Compared to Standard of Care Treatment - Full Text View - ClinicalTrials.gov</strong><a href="https://clinicaltrials.gov/ct2/show/NCT04292730">https://clinicaltrials.gov/ct2/show/NCT04292730</a></p>
</div>
<div id="ref-7LMnOrzx">
<p>73. <strong>Study to Evaluate the Safety and Antiviral Activity of Remdesivir (GS-5734‚Ñ¢) in Participants With Severe Coronavirus Disease (COVID-19) - Full Text View - ClinicalTrials.gov</strong><a href="https://clinicaltrials.gov/ct2/show/NCT04292899">https://clinicaltrials.gov/ct2/show/NCT04292899</a></p>
</div>
<div id="ref-19tnPvv1p">
<p>74. <strong>Adaptive COVID-19 Treatment Trial (ACTT) - Full Text View - ClinicalTrials.gov</strong><a href="https://clinicaltrials.gov/ct2/show/NCT04280705">https://clinicaltrials.gov/ct2/show/NCT04280705</a></p>
</div>
<div id="ref-1q4PmqTm">
<p>75. <a href="https://www.clinicaltrialsregister.eu/ctr-search/trial/2020-000936-23/FR">https://www.clinicaltrialsregister.eu/ctr-search/trial/2020-000936-23/FR</a></p>
</div>
<div id="ref-DvAlA1ad">
<p>76. <strong>Mild/Moderate 2019-nCoV Remdesivir RCT - Full Text View - ClinicalTrials.gov</strong><a href="https://clinicaltrials.gov/ct2/show/NCT04252664">https://clinicaltrials.gov/ct2/show/NCT04252664</a></p>
</div>
<div id="ref-1CJoAQiUA">
<p>77. <strong>Severe 2019-nCoV Remdesivir RCT - Full Text View - ClinicalTrials.gov</strong><a href="https://clinicaltrials.gov/ct2/show/NCT04257656">https://clinicaltrials.gov/ct2/show/NCT04257656</a></p>
</div>
<div id="ref-1AKi0FYUB">
<p>78. <strong>Development of therapeutic antibodies for the treatment of diseases</strong><br />
Ruei-Min Lu, Yu-Chyi Hwang, I-Ju Liu, Chi-Chiu Lee, Han-Zen Tsai, Hsin-Jung Li, Han-Chung Wu<br />
<em>Journal of Biomedical Science</em> (2020-01-02) <a href="https://doi.org/ggqbpx">https://doi.org/ggqbpx</a><br />
DOI: <a href="https://doi.org/10.1186/s12929-019-0592-z">10.1186/s12929-019-0592-z</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31894001">31894001</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6939334">PMC6939334</a></p>
</div>
<div id="ref-yumBaJ6U">
<p>79. <strong>Broadly Neutralizing Antiviral Antibodies</strong><br />
Davide Corti, Antonio Lanzavecchia<br />
<em>Annual Review of Immunology</em> (2013-03-21) <a href="https://doi.org/gf25g8">https://doi.org/gf25g8</a><br />
DOI: <a href="https://doi.org/10.1146/annurev-immunol-032712-095916">10.1146/annurev-immunol-032712-095916</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23330954">23330954</a></p>
</div>
<div id="ref-181QWa7HL">
<p>80. <strong>Neutralizing Monoclonal Antibodies as Promising Therapeutics against Middle East Respiratory Syndrome Coronavirus Infection</strong><br />
Hui-Ju Han, Jian-Wei Liu, Hao Yu, Xue-Jie Yu<br />
<em>Viruses</em> (2018-11-30) <a href="https://doi.org/ggp87v">https://doi.org/ggp87v</a><br />
DOI: <a href="https://doi.org/10.3390/v10120680">10.3390/v10120680</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30513619">30513619</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6315345">PMC6315345</a></p>
</div>
<div id="ref-dUnB3gD6">
<p>81. <strong>Ibalizumab Targeting CD4 Receptors, An Emerging Molecule in HIV Therapy</strong><br />
Simona A. Iacob, Diana G. Iacob<br />
<em>Frontiers in Microbiology</em> (2017-11-27) <a href="https://doi.org/gcn3kh">https://doi.org/gcn3kh</a><br />
DOI: <a href="https://doi.org/10.3389/fmicb.2017.02323">10.3389/fmicb.2017.02323</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29230203">29230203</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5711820">PMC5711820</a></p>
</div>
<div id="ref-cwbHiM1o">
<p>82. <strong>Product review on the monoclonal antibody palivizumab for prevention of respiratory syncytial virus infection</strong><br />
Bernhard Resch<br />
<em>Human Vaccines &amp; Immunotherapeutics</em> (2017-06-12) <a href="https://doi.org/ggqbps">https://doi.org/ggqbps</a><br />
DOI: <a href="https://doi.org/10.1080/21645515.2017.1337614">10.1080/21645515.2017.1337614</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28605249">28605249</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5612471">PMC5612471</a></p>
</div>
<div id="ref-KULxo48U">
<p>83. <strong>Neutralizing Antibodies in Patients with Severe Acute Respiratory Syndrome‚ÄìAssociated Coronavirus Infection</strong><br />
Yuchun Nie, Guangwen Wang, Xuanling Shi, Hong Zhang, Yan Qiu, Zhongping He, Wei Wang, Gewei Lian, Xiaolei Yin, Liying Du, ‚Ä¶ Mingxiao Ding<br />
<em>The Journal of Infectious Diseases</em> (2004-09-15) <a href="https://doi.org/cgqj5b">https://doi.org/cgqj5b</a><br />
DOI: <a href="https://doi.org/10.1086/423286">10.1086/423286</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15319862">15319862</a></p>
</div>
<div id="ref-MTcTiS7i">
<p>84. <strong>Potent human monoclonal antibodies against SARS CoV, Nipah and Hendra viruses</strong><br />
Ponraj Prabakaran, Zhongyu Zhu, Xiaodong Xiao, Arya Biragyn, Antony S Dimitrov, Christopher C Broder, Dimiter S Dimitrov<br />
<em>Expert Opinion on Biological Therapy</em> (2009-04-08) <a href="https://doi.org/b88kw8">https://doi.org/b88kw8</a><br />
DOI: <a href="https://doi.org/10.1517/14712590902763755">10.1517/14712590902763755</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19216624">19216624</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705284">PMC2705284</a></p>
</div>
<div id="ref-rXzkS3gA">
<p>85. <strong>Prior Infection and Passive Transfer of Neutralizing Antibody Prevent Replication of Severe Acute Respiratory Syndrome Coronavirus in the Respiratory Tract of Mice</strong><br />
K. Subbarao, J. McAuliffe, L. Vogel, G. Fahle, S. Fischer, K. Tatti, M. Packard, W.-J. Shieh, S. Zaki, B. Murphy<br />
<em>Journal of Virology</em> (2004-03-11) <a href="https://doi.org/b8wr7c">https://doi.org/b8wr7c</a><br />
DOI: <a href="https://doi.org/10.1128/jvi.78.7.3572-3577.2004">10.1128/jvi.78.7.3572-3577.2004</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15016880">15016880</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC371090">PMC371090</a></p>
</div>
<div id="ref-RRx6YovP">
<p>86. <strong>The Effectiveness of Convalescent Plasma and Hyperimmune Immunoglobulin for the Treatment of Severe Acute Respiratory Infections of Viral Etiology: A Systematic Review and Exploratory Meta-analysis</strong><br />
John Mair-Jenkins, Maria Saavedra-Campos, J. Kenneth Baillie, Paul Cleary, Fu-Meng Khaw, Wei Shen Lim, Sophia Makki, Kevin D. Rooney, Jonathan S. Nguyen-Van-Tam, Charles R. Beck, Convalescent Plasma Study Group<br />
<em>Journal of Infectious Diseases</em> (2015-01-01) <a href="https://doi.org/f632n7">https://doi.org/f632n7</a><br />
DOI: <a href="https://doi.org/10.1093/infdis/jiu396">10.1093/infdis/jiu396</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25030060">25030060</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4264590">PMC4264590</a></p>
</div>
<div id="ref-Ou3P9pGV">
<p>87. <strong>Identification of human neutralizing antibodies against MERS-CoV and their role in virus adaptive evolution</strong><br />
X.-C. Tang, S. S. Agnihothram, Y. Jiao, J. Stanhope, R. L. Graham, E. C. Peterson, Y. Avnir, A. S. C. Tallarico, J. Sheehan, Q. Zhu, ‚Ä¶ W. A. Marasco<br />
<em>Proceedings of the National Academy of Sciences</em> (2014-04-28) <a href="https://doi.org/smr">https://doi.org/smr</a><br />
DOI: <a href="https://doi.org/10.1073/pnas.1402074111">10.1073/pnas.1402074111</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24778221">24778221</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4024880">PMC4024880</a></p>
</div>
<div id="ref-2QTH37Xi">
<p>88. <strong>Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</strong><br />
Wenhui Li, Michael J. Moore, Natalya Vasilieva, Jianhua Sui, Swee Kee Wong, Michael A. Berne, Mohan Somasundaran, John L. Sullivan, Katherine Luzuriaga, Thomas C. Greenough, ‚Ä¶ Michael Farzan<br />
<em>Nature</em> (2003-11) <a href="https://doi.org/bqvpjh">https://doi.org/bqvpjh</a><br />
DOI: <a href="https://doi.org/10.1038/nature02145">10.1038/nature02145</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14647384">14647384</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095016">PMC7095016</a></p>
</div>
<div id="ref-1GLr0EJU">
<p>89. <strong>The Role of ACE2 in Cardiovascular Physiology</strong><br />
Gavin Y. Oudit, Michael A. Crackower, Peter H. Backx, Josef M. Penninger<br />
<em>Trends in Cardiovascular Medicine</em> (2003-04) <a href="https://doi.org/bsbp49">https://doi.org/bsbp49</a><br />
DOI: <a href="https://doi.org/10.1016/s1050-1738(02)00233-5">10.1016/s1050-1738(02)00233-5</a></p>
</div>
<div id="ref-XcuzhxrJ">
<p>90. <strong>A human monoclonal antibody blocking SARS-CoV-2 infection</strong><br />
Chunyan Wang, Wentao Li, Dubravka Drabek, Nisreen M. A. Okba, Rien van Haperen, Albert D. M. E. Osterhaus, Frank J. M. van Kuppeveld, Bart L. Haagmans, Frank Grosveld, Berend-Jan Bosch<br />
<em>bioRxiv</em> (2020-03-12) <a href="https://doi.org/ggnw4t">https://doi.org/ggnw4t</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.11.987958">10.1101/2020.03.11.987958</a></p>
</div>
<div id="ref-Ftbm1M9p">
<p>91. <strong>Unexpected Receptor Functional Mimicry Elucidates Activation of Coronavirus Fusion</strong><br />
Alexandra C. Walls, Xiaoli Xiong, Young-Jun Park, M. Alejandra Tortorici, Joost Snijder, Joel Quispe, Elisabetta Cameroni, Robin Gopal, Mian Dai, Antonio Lanzavecchia, ‚Ä¶ David Veesler<br />
<em>Cell</em> (2019-02) <a href="https://doi.org/gft3jg">https://doi.org/gft3jg</a><br />
DOI: <a href="https://doi.org/10.1016/j.cell.2018.12.028">10.1016/j.cell.2018.12.028</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30712865">30712865</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6751136">PMC6751136</a></p>
</div>
<div id="ref-O1whWg6Q">
<p>92. <strong><em>In Vitro</em> Neutralization Is Not Predictive of Prophylactic Efficacy of Broadly Neutralizing Monoclonal Antibodies CR6261 and CR9114 against Lethal H2 Influenza Virus Challenge in Mice</strong><br />
Troy C. Sutton, Elaine W. Lamirande, Kevin W. Bock, Ian N. Moore, Wouter Koudstaal, Muniza Rehman, Gerrit Jan Weverling, Jaap Goudsmit, Kanta Subbarao<br />
<em>Journal of Virology</em> (2017-12-15) <a href="https://doi.org/ggqbpt">https://doi.org/ggqbpt</a><br />
DOI: <a href="https://doi.org/10.1128/jvi.01603-17">10.1128/jvi.01603-17</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29046448">29046448</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5709608">PMC5709608</a></p>
</div>
<div id="ref-GhJYjnft">
<p>93. <strong>Potent human neutralizing antibodies elicited by SARS-CoV-2 infection</strong><br />
Bin Ju, Qi Zhang, Xiangyang Ge, Ruoke Wang, Jiazhen Yu, Sisi Shan, Bing Zhou, Shuo Song, Xian Tang, Jinfang Yu, ‚Ä¶ Linqi Zhang<br />
<em>bioRxiv</em> (2020-03-26) <a href="https://doi.org/ggp7t4">https://doi.org/ggp7t4</a><br />
DOI: <a href="https://doi.org/10.1101/2020.03.21.990770">10.1101/2020.03.21.990770</a></p>
</div>
<div id="ref-pUnzB8wV">
<p>94. <strong>Human neutralizing antibodies against MERS coronavirus: implications for future immunotherapy</strong><br />
Xian-Chun Tang, Wayne A Marasco<br />
<em>Immunotherapy</em> (2015-07) <a href="https://doi.org/ggqbpz">https://doi.org/ggqbpz</a><br />
DOI: <a href="https://doi.org/10.2217/imt.15.33">10.2217/imt.15.33</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26098703">26098703</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5068219">PMC5068219</a></p>
</div>
<div id="ref-14motyCOm">
<p>95. <strong>A Potent and Broad Neutralizing Antibody Recognizes and Penetrates the HIV Glycan Shield</strong><br />
R. Pejchal, K. J. Doores, L. M. Walker, R. Khayat, P.-S. Huang, S.-K. Wang, R. L. Stanfield, J.-P. Julien, A. Ramos, M. Crispin, ‚Ä¶ I. A. Wilson<br />
<em>Science</em> (2011-10-13) <a href="https://doi.org/bzqv8c">https://doi.org/bzqv8c</a><br />
DOI: <a href="https://doi.org/10.1126/science.1213256">10.1126/science.1213256</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21998254">21998254</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3280215">PMC3280215</a></p>
</div>
<div id="ref-oGHFF5Cm">
<p>96. <strong>Broadly Neutralizing Antibodies against HIV: Back to Blood</strong><br />
Amir Dashti, Anthony L. DeVico, George K. Lewis, Mohammad M. Sajadi<br />
<em>Trends in Molecular Medicine</em> (2019-03) <a href="https://doi.org/ggqbpr">https://doi.org/ggqbpr</a><br />
DOI: <a href="https://doi.org/10.1016/j.molmed.2019.01.007">10.1016/j.molmed.2019.01.007</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30792120">30792120</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6401214">PMC6401214</a></p>
</div>
<div id="ref-hvjXMm9H">
<p>97. <strong>Importance of Neutralizing Monoclonal Antibodies Targeting Multiple Antigenic Sites on the Middle East Respiratory Syndrome Coronavirus Spike Glycoprotein To Avoid Neutralization Escape</strong><br />
Lingshu Wang, Wei Shi, James D. Chappell, M. Gordon Joyce, Yi Zhang, Masaru Kanekiyo, Michelle M. Becker, Neeltje van Doremalen, Robert Fischer, Nianshuang Wang, ‚Ä¶ Barney S. Graham<br />
<em>Journal of Virology</em> (2018-04-27) <a href="https://doi.org/ggqbpv">https://doi.org/ggqbpv</a><br />
DOI: <a href="https://doi.org/10.1128/jvi.02002-17">10.1128/jvi.02002-17</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29514901">29514901</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5923077">PMC5923077</a></p>
</div>
<div id="ref-tC5vJmwj">
<p>98. <strong>Anti‚Äìspike IgG causes severe acute lung injury by skewing macrophage responses during acute SARS-CoV infection</strong><br />
Li Liu, Qiang Wei, Qingqing Lin, Jun Fang, Haibo Wang, Hauyee Kwok, Hangying Tang, Kenji Nishiura, Jie Peng, Zhiwu Tan, ‚Ä¶ Zhiwei Chen<br />
<em>JCI Insight</em> (2019-02-21) <a href="https://doi.org/ggqbpw">https://doi.org/ggqbpw</a><br />
DOI: <a href="https://doi.org/10.1172/jci.insight.123158">10.1172/jci.insight.123158</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30830861">30830861</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6478436">PMC6478436</a></p>
</div>
<div id="ref-LtFtUo2P">
<p>99. <strong>The antiviral effect of interferon-beta against SARS-Coronavirus is not mediated by MxA protein</strong><br />
Martin Spiegel, Andreas Pichlmair, Elke M√ºhlberger, Otto Haller, Friedemann Weber<br />
<em>Journal of Clinical Virology</em> (2004-07) <a href="https://doi.org/cmc3ds">https://doi.org/cmc3ds</a><br />
DOI: <a href="https://doi.org/10.1016/j.jcv.2003.11.013">10.1016/j.jcv.2003.11.013</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15135736">15135736</a></p>
</div>
<div id="ref-Nfu9kiae">
<p>100. <strong>Coronavirus virulence genes with main focus on SARS-CoV envelope gene</strong><br />
Marta L. DeDiego, Jose L. Nieto-Torres, Jose M. Jimenez-Guarde√±o, Jose A. Regla-Nava, Carlos Casta√±o-Rodriguez, Raul Fernandez-Delgado, Fernando Usera, Luis Enjuanes<br />
<em>Virus Research</em> (2014-12) <a href="https://doi.org/f6wm24">https://doi.org/f6wm24</a><br />
DOI: <a href="https://doi.org/10.1016/j.virusres.2014.07.024">10.1016/j.virusres.2014.07.024</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25093995">25093995</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4261026">PMC4261026</a></p>
</div>
<div id="ref-17OQtAY4l">
<p>101. <strong>Using the MAARIE Framework To Read the Research Literature</strong><br />
M. Corcoran<br />
<em>American Journal of Occupational Therapy</em> (2006-07-01) <a href="https://doi.org/bqh97x">https://doi.org/bqh97x</a><br />
DOI: <a href="https://doi.org/10.5014/ajot.60.4.367">10.5014/ajot.60.4.367</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16915865">16915865</a></p>
</div>
<div id="ref-YuJbg3zO">
<p>102. <strong>Open collaborative writing with Manubot</strong><br />
Daniel S. Himmelstein, Vincent Rubinetti, David R. Slochower, Dongbo Hu, Venkat S. Malladi, Casey S. Greene, Anthony Gitter<br />
<em>PLOS Computational Biology</em> (2019-06-24) <a href="https://doi.org/c7np">https://doi.org/c7np</a><br />
DOI: <a href="https://doi.org/10.1371/journal.pcbi.1007128">10.1371/journal.pcbi.1007128</a> ¬∑ PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31233491">31233491</a> ¬∑ PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6611653">PMC6611653</a></p>
</div>
</div>
<!-- Define citation tags below -->
<!-- default theme -->

<style>
    /* import google fonts */
    @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
    @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

    /* -------------------------------------------------- */
    /* global */
    /* -------------------------------------------------- */

    /* all elements */
    * {
        /* force sans-serif font unless specified otherwise */
        font-family: "Open Sans", "Helvetica", sans-serif;

        /* prevent text inflation on some mobile browsers */
        -webkit-text-size-adjust: none !important;
        -moz-text-size-adjust: none !important;
        -o-text-size-adjust: none !important;
        text-size-adjust: none !important;
    }

    @media only screen {
        /* "page" element */
        body {
            position: relative;
            box-sizing: border-box;
            font-size: 12pt;
            line-height: 1.5;
            max-width: 8.5in;
            margin: 20px auto;
            padding: 40px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* "page" element */
        body {
            padding: 20px;
            margin: 0;
            border-radius: 0;
            border: none;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
            background: none;
        }
    }

    /* -------------------------------------------------- */
    /* headings */
    /* -------------------------------------------------- */

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
        margin: 20px 0;
        padding: 0;
        font-weight: bold;
    }

    /* biggest heading */
    h1 {
        margin: 40px 0;
        text-align: center;
    }

    /* second biggest heading */
    h2 {
        margin-top: 30px;
        padding-bottom: 5px;
        border-bottom: solid 1px #bdbdbd;
    }

    /* -------------------------------------------------- */
    /* manuscript header */
    /* -------------------------------------------------- */

    /* manuscript title */
    header > h1 {
        margin: 0;
    }

    /* manuscript title caption text (ie "automatically generated on") */
    header + p {
        text-align: center;
        margin-top: 10px;
    }

    /* -------------------------------------------------- */
    /* text elements */
    /* -------------------------------------------------- */

    /* links */
    a {
        color: #2196f3;
        overflow-wrap: break-word;
    }

    /* normal links (not empty, not button link, not syntax highlighting link) */
    a:not(:empty):not(.button):not(.sourceLine) {
        padding-left: 1px;
        padding-right: 1px;
    }

    /* superscripts and subscripts */
    sub,
    sup {
        /* prevent from affecting line height */
        line-height: 0;
    }

    /* unordered and ordered lists*/
    ul,
    ol {
        padding-left: 20px;
    }

    /* class for styling text semibold */
    .semibold {
        font-weight: 600;
    }

    /* class for styling elements horizontally left aligned */
    .left {
        display: block;
        text-align: left;
        margin-left: auto;
        margin-right: 0;
        justify-content: left;
    }

    /* class for styling elements horizontally centered */
    .center {
        display: block;
        text-align: center;
        margin-left: auto;
        margin-right: auto;
        justify-content: center;
    }

    /* class for styling elements horizontally right aligned */
    .right {
        display: block;
        text-align: right;
        margin-left: 0;
        margin-right: auto;
        justify-content: right;
    }

    /* -------------------------------------------------- */
    /* section elements */
    /* -------------------------------------------------- */

    /* horizontal divider line */
    hr {
        border: none;
        height: 1px;
        background: #bdbdbd;
    }

    /* paragraphs, horizontal dividers, figures, tables, code */
    p,
    hr,
    figure,
    table,
    pre {
        /* treat all as "paragraphs", with consistent vertical margins */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* figures */
    /* -------------------------------------------------- */

    /* figure */
    figure {
        max-width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure caption */
    figcaption {
        padding: 0;
        padding-top: 10px;
    }

    /* figure image element */
    figure img {
        max-width: 100%;
        display: block;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure auto-number */
    img + figcaption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* tables */
    /* -------------------------------------------------- */

    /* table */
    table {
        border-collapse: collapse;
        border-spacing: 0;
        width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* table cells */
    th,
    td {
        border: solid 1px #bdbdbd;
        padding: 10px;
        /* squash table if too wide for page by forcing line breaks */
        overflow-wrap: break-word;
        word-break: break-word;
    }

    /* header row and even rows */
    th,
    tr:nth-child(2n) {
        background-color: #fafafa;
    }

    /* odd rows */
    tr:nth-child(2n + 1) {
        background-color: #ffffff;
    }

    /* table caption */
    caption {
        text-align: left;
        padding: 0;
        padding-bottom: 10px;
    }

    /* table auto-number */
    table > caption > span:first-of-type,
    div.table_wrapper > table > caption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* code */
    /* -------------------------------------------------- */

    /* multi-line code block */
    pre {
        padding: 10px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
        break-inside: avoid;
        text-align: left;
    }

    /* inline code, ie code within normal text */
    :not(pre) > code {
        padding: 0 4px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
    }

    /* code text */
    /* apply all children, to reach syntax highlighting sub-elements */
    code,
    code * {
        /* force monospace font */
        font-family: "Source Code Pro", "Courier New", monospace;
    }

    /* -------------------------------------------------- */
    /* quotes */
    /* -------------------------------------------------- */

    /* quoted text */
    blockquote {
        margin: 0;
        padding: 0;
        border-left: 4px solid #bdbdbd;
        padding-left: 16px;
        break-inside: avoid;
    }

    /* -------------------------------------------------- */
    /* banners */
    /* -------------------------------------------------- */

    /* info banners */
    .banner {
        box-sizing: border-box;
        display: block;
        position: relative;
        width: 100%;
        margin-top: 20px;
        margin-bottom: 20px;
        padding: 20px;
        text-align: center;
    }

    /* paragraph in banner */
    .banner > p {
        margin: 0;
    }

    /* -------------------------------------------------- */
    /* highlight colors */
    /* -------------------------------------------------- */

    .white {
        background: #ffffff;
    }
    .lightgrey {
        background: #eeeeee;
    }
    .grey {
        background: #757575;
    }
    .darkgrey {
        background: #424242;
    }
    .black {
        background: #000000;
    }
    .lightred {
        background: #ffcdd2;
    }
    .lightyellow {
        background: #ffecb3;
    }
    .lightgreen {
        background: #dcedc8;
    }
    .lightblue {
        background: #e3f2fd;
    }
    .lightpurple {
        background: #f3e5f5;
    }
    .red {
        background: #f44336;
    }
    .orange {
        background: #ff9800;
    }
    .yellow {
        background: #ffeb3b;
    }
    .green {
        background: #4caf50;
    }
    .blue {
        background: #2196f3;
    }
    .purple {
        background: #9c27b0;
    }
    .white,
    .lightgrey,
    .lightred,
    .lightyellow,
    .lightgreen,
    .lightblue,
    .lightpurple,
    .orange,
    .yellow,
    .white a,
    .lightgrey a,
    .lightred a,
    .lightyellow a,
    .lightgreen a,
    .lightblue a,
    .lightpurple a,
    .orange a,
    .yellow a {
        color: #000000;
    }
    .grey,
    .darkgrey,
    .black,
    .red,
    .green,
    .blue,
    .purple,
    .grey a,
    .darkgrey a,
    .black a,
    .red a,
    .green a,
    .blue a,
    .purple a {
        color: #ffffff;
    }

    /* -------------------------------------------------- */
    /* buttons */
    /* -------------------------------------------------- */

    /* class for styling links like buttons */
    .button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        margin: 5px;
        padding: 10px 20px;
        font-size: 0.75em;
        font-weight: 600;
        text-transform: uppercase;
        text-decoration: none;
        letter-spacing: 1px;
        background: none;
        color: #2196f3;
        border: solid 1px #bdbdbd;
        border-radius: 5px;
    }

    /* buttons when hovered */
    .button:hover:not([disabled]),
    .icon_button:hover:not([disabled]) {
        cursor: pointer;
        background: #f5f5f5;
    }

    /* buttons when disabled */
    .button[disabled],
    .icon_button[disabled] {
        opacity: 0.35;
        pointer-events: none;
    }

    /* class for styling buttons containg only single icon */
    .icon_button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        text-decoration: none;
        margin: 0;
        padding: 0;
        background: none;
        border-radius: 5px;
        border: none;
        width: 20px;
        height: 20px;
        min-width: 20px;
        min-height: 20px;
    }

    /* icon button inner svg image */
    .icon_button > svg {
        height: 16px;
    }

    /* -------------------------------------------------- */
    /* icons */
    /* -------------------------------------------------- */

    /* class for styling icons inline with text */
    .inline_icon {
        height: 1em;
        position: relative;
        top: 0.125em;
    }

    /* -------------------------------------------------- */
    /* print control */
    /* -------------------------------------------------- */

    @media print {
        @page {
            /* suggested printing margin */
            margin: 0.5in;
        }

        /* document and "page" elements */
        html, body {
            margin: 0;
            padding: 0;
            width: 100%;
            height: 100%;
        }

        /* "page" element */
        body {
            font-size: 11pt !important;
            line-height: 1.35;
        }

        /* all headings */
        h1,
        h2,
        h3,
        h4,
        h5,
        h6 {
            margin: 15px 0;
        }

        /* heading 1 */
        h1 {
            font-size: 1.75em;
        }

        /* heading 2 */
        h2 {
            font-size: 1.25em;
            margin-top: 0;
        }

        /* heading 3 */
        h3 {
            font-size: 1.10em;
        }

        /* figures and tables */
        figure, table {
            font-size: 0.85em;
        }

        /* table cells */
        th,
        td {
            padding: 5px;
        }

        /* shrink font awesome icons */
        i.fas,
        i.fab,
        i.far,
        i.fal {
            transform: scale(0.85);
        }

        /* decrease banner margins */
        .banner {
            margin-top: 15px;
            margin-bottom: 15px;
            padding: 15px;
        }

        /* class for centering an element vertically on its own page */
        .page_center {
            margin: auto;
            width: 100%;
            height: 100%;
            display: flex;
            align-items: center;
            vertical-align: middle;
            break-before: page;
            break-after: page;
        }

        /* always insert a page break before the element */
        .page_break_before {
            break-before: page;
        }

        /* always insert a page break after the element */
        .page_break_after {
            break-after: page;
        }

        /* avoid page break before the element */
        .page_break_before_avoid {
            break-before: avoid;
        }

        /* avoid page break after the element */
        .page_break_after_avoid {
            break-after: avoid;
        }

        /* avoid page break inside the element */
        .page_break_inside_avoid {
            break-inside: avoid;
        }
    }

    /* -------------------------------------------------- */
    /* override pandoc css quirks */
    /* -------------------------------------------------- */

    .sourceCode {
        /* prevent unsightly overflow in wide code blocks */
        overflow: auto !important;
    }

    div.sourceCode {
        /* prevent background fill on top-most code block  container */
        background: none !important;
    }

    .sourceCode * {
        /* force consistent line spacing */
        line-height: 1.5 !important;
    }

    div.sourceCode {
        /* style code block margins same as <pre> element */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* tablenos */
    /* -------------------------------------------------- */

    /* tablenos wrapper */
    .tablenos {
        /* show scrollbar on tables if necessary to prevent overflow */
        width: 100%;
        margin: 20px 0;
    }

    .tablenos > table {
        /* move margins from table to table_wrapper to allow margin collapsing */
        margin: 0;
    }

    @media only screen {
        /* tablenos wrapper */
        .tablenos {
            /* show scrollbar on tables if necessary to prevent overflow */
            overflow-x: auto !important;
        }

        .tablenos th,
        .tablenos td {
            overflow-wrap: unset !important;
            word-break: unset !important;
        }

        /* table in wrapper */
        .tablenos table,
        .tablenos table * {
            /* don't break table words */
            overflow-wrap: normal !important;
        }
    }

    /* -------------------------------------------------- */
    /* mathjax */
    /* -------------------------------------------------- */

    /* mathjax containers */
    .math.display > span:not(.MathJax_Preview) {
        /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
        display: flex !important;
        overflow-x: auto !important;
        overflow-y: hidden !important;
        justify-content: center;
        align-items: center;
        margin: 0 !important;
    }

    /* right click menu */
    .MathJax_Menu {
        border-radius: 5px !important;
        border: solid 1px #bdbdbd !important;
        box-shadow: none !important;
    }

    /* equation auto-number */
    span[id^="eq:"] > span.math.display + span {
        font-weight: 600;
    }

    /* equation */
    span[id^="eq:"] > span.math.display > span {
        /* nudge to make room for equation auto-number and anchor */
        margin-right: 60px !important;
    }

    /* -------------------------------------------------- */
    /* anchors plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anchor button */
        .anchor {
            opacity: 0;
            margin-left: 5px;
        }

        /* anchor buttons within <h2>'s */
        h2 .anchor {
            margin-left: 10px;
        }

        /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
        *:hover > .anchor,
        .anchor:hover,
        .anchor:focus {
            opacity: 1;
        }

        /* anchor button when hovered */
        .anchor:hover {
            cursor: pointer;
        }
    }

    /* always show anchor button on devices with no mouse/hover ability */
    @media (hover: none) {
        .anchor {
            opacity: 1;
        }
    }

    /* always hide anchor button on print */
    @media only print {
        .anchor {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* accordion plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* accordion arrow button */
        .accordion_arrow {
            margin-right: 10px;
        }

        /* arrow icon when <h2> data-collapsed attribute true */
        h2[data-collapsed="true"] > .accordion_arrow > svg {
            transform: rotate(-90deg);
        }

        /* all elements (except <h2>'s) when data-collapsed attribute true */
        *:not(h2)[data-collapsed="true"] {
            display: none;
        }

        /* accordion arrow button when hovered and <h2>'s when hovered */
        .accordion_arrow:hover,
        h2[data-collapsed="true"]:hover,
        h2[data-collapsed="false"]:hover {
            cursor: pointer;
        }
    }

    /* always hide accordion arrow button on print */
    @media only print {
        .accordion_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* tooltips plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* tooltip container */
        #tooltip {
            position: absolute;
            width: 50%;
            min-width: 240px;
            max-width: 75%;
            z-index: 1;
        }

        /* tooltip content */
        #tooltip_content {
            margin-bottom: 5px;
            padding: 20px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
            overflow-wrap: break-word;
        }

        /* tooltip copy of paragraphs and figures */
        #tooltip_content > p,
        #tooltip_content > figure {
            margin: 0;
            max-height: 320px;
            overflow-y: auto;
        }

        /* tooltip copy of <img> */
        #tooltip_content > figure > img {
            max-height: 260px;
        }

        /* navigation bar */
        #tooltip_nav_bar {
            margin-top: 10px;
            text-align: center;
        }

        /* navigation bar previous/next buton */
        #tooltip_nav_bar > .icon_button {
            position: relative;
            top: 3px;
        }

        /* navigation bar previous button */
        #tooltip_nav_bar > .icon_button:first-of-type {
            margin-right: 5px;
        }

        /* navigation bar next button */
        #tooltip_nav_bar > .icon_button:last-of-type {
            margin-left: 5px;
        }
    }

    /* always hide tooltip on print */
    @media only print {
        #tooltip {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* jump to first plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* jump button */
        .jump_arrow {
            position: relative;
            top: 0.125em;
            margin-right: 5px;
        }
    }

    /* always hide jump button on print */
    @media only print {
        .jump_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* link highlight plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anything with data-highlighted attribute true */
        [data-highlighted="true"] {
            background: #ffeb3b;
        }

        /* anything with data-selected attribute true */
        [data-selected="true"] {
            background: #ff8a65 !important;
        }

        /* animation definition for glow */
        @keyframes highlight_glow {
            0% {
                background: none;
            }
            10% {
                background: #bbdefb;
            }
            100% {
                background: none;
            }
        }

        /* anything with data-glow attribute true */
        [data-glow="true"] {
            animation: highlight_glow 2s;
        }
    }

    /* -------------------------------------------------- */
    /* table of contents plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* toc panel */
        #toc_panel {
            box-sizing: border-box;
            position: fixed;
            top: 0;
            left: 0;
            background: #ffffff;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            z-index: 2;
        }

        /* toc panel when closed */
        #toc_panel[data-open="false"] {
            min-width: 60px;
            width: 60px;
            height: 60px;
            border-right: solid 1px #bdbdbd;
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc panel when open */
        #toc_panel[data-open="true"] {
            min-width: 260px;
            max-width: 480px;
            /* keep panel edge consistent distance away from "page" edge */
            width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
            bottom: 0;
            border-right: solid 1px #bdbdbd;
        }

        /* toc panel header */
        #toc_header {
            box-sizing: border-box;
            display: flex;
            flex-direction: row;
            align-items: center;
            height: 60px;
            margin: 0;
            padding: 20px;
        }

        /* toc panel header when hovered */
        #toc_header:hover {
            cursor: pointer;
        }

        /* toc panel header when panel open */
        #toc_panel[data-open="true"] > #toc_header {
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc open/close header button */
        #toc_button {
            margin-right: 20px;
        }

        /* hide toc list and header text when closed */
        #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
        #toc_panel[data-open="false"] > #toc_list {
            display: none;
        }

        /* toc list of entries */
        #toc_list {
            box-sizing: border-box;
            width: 100%;
            padding: 20px;
            position: absolute;
            top: calc(60px + 1px);
            bottom: 0;
            overflow: auto;
        }

        /* toc entry, link to section in document */
        .toc_link {
            display: block;
            padding: 5px;
            position: relative;
            font-weight: 600;
            text-decoration: none;
        }

        /* toc entry when hovered or when "viewed" */
        .toc_link:hover,
        .toc_link[data-viewing="true"] {
            background: #f5f5f5;
        }

        /* toc entry, level 1 indentation */
        .toc_link[data-level="1"] {
            margin-left: 0;
        }

        /* toc entry, level 2 indentation */
        .toc_link[data-level="2"] {
            margin-left: 20px;
        }

        /* toc entry, level 3 indentation */
        .toc_link[data-level="3"] {
            margin-left: 40px;
        }

        /* toc entry, level 4 indentation */
        .toc_link[data-level="4"] {
            margin-left: 60px;
        }

        /* toc entry bullets */
        #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
            position: absolute;
            left: -15px;
            top: -1px;
            font-size: 1.5em;
        }

        /* toc entry, level 2 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
            content: "\2022";
        }

        /* toc entry, level 3 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
            content: "\25AB";
        }

        /* toc entry, level 4 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
            content: "-";
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* push <body> ("page") element down to make room for toc icon */
        .toc_body_nudge {
            padding-top: 60px;
        }

        /* toc icon when panel closed and not hovered */
        #toc_panel[data-open="false"]:not(:hover) {
            background: rgba(255, 255, 255, 0.75);
        }
    }

    /* always hide toc panel on print */
    @media only print {
        #toc_panel {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* lightbox plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* regular <img> in document when hovered */
        .lightbox_document_img:hover {
            cursor: pointer;
        }

        .body_no_scroll {
            overflow: hidden !important;
        }

        /* screen overlay */
        #lightbox_overlay {
            display: flex;
            flex-direction: column;
            position: fixed;
            left: 0;
            top: 0;
            right: 0;
            bottom: 0;
            background: rgba(0, 0, 0, 0.75);
            z-index: 3;
        }

        /* middle area containing lightbox image */
        #lightbox_image_container {
            flex-grow: 1;
            display: flex;
            justify-content: center;
            align-items: center;
            overflow: hidden;
            position: relative;
            padding: 20px;
        }

        /* bottom area containing caption */
        #lightbox_bottom_container {
            display: flex;
            justify-content: center;
            align-items: center;
            height: 100px;
            min-height: 100px;
            max-height: 100px;
            background: rgba(0, 0, 0, 0.5);
        }

        /* image number info text box */
        #lightbox_number_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            left: 2px;
            top: 0;
            z-index: 4;
        }

        /* zoom info text box */
        #lightbox_zoom_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            right: 2px;
            top: 0;
            z-index: 4;
        }

        /* copy of image caption */
        #lightbox_caption {
            box-sizing: border-box;
            display: inline-block;
            width: 100%;
            max-height: 100%;
            padding: 10px 0;
            text-align: center;
            overflow-y: auto;
            color: #ffffff;
        }

        /* navigation previous/next button */
        .lightbox_button {
            width: 100px;
            height: 100%;
            min-width: 100px;
            min-height: 100%;
            color: #ffffff;
        }

        /* navigation previous/next button when hovered */
        .lightbox_button:hover {
            background: none !important;
        }

        /* navigation button icon */
        .lightbox_button > svg {
            height: 25px;
        }

        /* figure auto-number */
        #lightbox_caption > span:first-of-type {
            font-weight: bold;
            margin-right: 5px;
        }

        /* lightbox image when hovered */
        #lightbox_img:hover {
            cursor: grab;
        }

        /* lightbox image when grabbed */
        #lightbox_img:active {
            cursor: grabbing;
        }
    }

    /* when on screen < 480px wide */
    @media only screen and (max-width: 480px) {
        /* make navigation buttons skinnier on small screens to make more room for caption text */
        .lightbox_button {
            width: 50px;
            min-width: 50px;
        }
    }

    /* always hide lightbox on print */
    @media only print {
        #lightbox_overlay {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* hypothesis (annotations) plugin */
    /* -------------------------------------------------- */

    /* side panel */
    .annotator-frame {
        width: 280px !important;
        z-index: 0 !important;
    }

    /* match highlight color to rest of theme */
    .annotator-highlights-always-on .annotator-hl {
        background-color: #ffeb3b !important;
    }

    /* match focused color to rest of theme */
    .annotator-hl.annotator-hl-focused {
        background-color: #ff8a65 !important;
    }

    /* match bucket bar color to rest of theme */
    .annotator-bucket-bar {
        background: #f5f5f5 !important;
    }

    /* always hide toolbar and tooltip on print */
    @media only print {
        .annotator-frame {
            display: none !important;
        }

        hypothesis-adder {
            display: none !important;
        }
    }
</style>
<!-- anchors plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin adds an anchor next to each of a certain type
        // of element that provides a human-readable url to that specific
        // item/position in the document (eg "manuscript.html#abstract"). It
        // also makes it such that scrolling out of view of a target removes
        // its identifier from the url.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'anchors';

        // default plugin options
        const options = {
            // which types of elements to add anchors next to, in
            // "document.querySelector" format
            typesQuery: 'h1, h2, h3, [id^="fig:"], [id^="tbl:"], [id^="eq:"]',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // add anchor to each element of specified types
            const elements = document.querySelectorAll(options.typesQuery);
            for (const element of elements)
                addAnchor(element);

            // attach scroll listener to window
            window.addEventListener('scroll', onScroll);
        }

        // when window is scrolled
        function onScroll() {
            // if url has hash and user has scrolled out of view of hash
            // target, remove hash from url
            const tolerance = 100;
            const target = getHashTarget();
            if (target) {
                if (
                    target.getBoundingClientRect().top >
                        window.innerHeight + tolerance ||
                    target.getBoundingClientRect().bottom < 0 - tolerance
                )
                    history.pushState(null, null, ' ');
            }
        }

        // add anchor to element
        function addAnchor(element) {
            let addTo; // element to add anchor button to

            // if figure or table, modify withId and addTo to get expected
            // elements
            if (element.id.indexOf('fig:') === 0) {
                addTo = element.querySelector('figcaption');
            } else if (element.id.indexOf('tbl:') === 0) {
                addTo = element.querySelector('caption');
            } else if (element.id.indexOf('eq:') === 0) {
                addTo = element.querySelector('.eqnos-number');
            }

            addTo = addTo || element;
            const id = element.id || null;

            // do not add anchor if element doesn't have assigned id.
            // id is generated by pandoc and is assumed to be unique and
            // human-readable
            if (!id)
                return;

            // create anchor button
            const anchor = document.createElement('a');
            anchor.innerHTML = document.querySelector('.icon_link').innerHTML;
            anchor.title = 'Link to this part of the document';
            anchor.classList.add('icon_button', 'anchor');
            anchor.dataset.ignore = 'true';
            anchor.href = '#' + id;
            addTo.appendChild(anchor);
        }

        // get element that is target of link or url hash
        function getHashTarget() {
            const hash = window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (id.indexOf('fig:') === 0)
                target = target.querySelector('figure');
            if (id.indexOf('tbl:') === 0)
                target = target.querySelector('table');

            return target;
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- link icon -->

<template class="icon_link">
    <!-- modified from: https://fontawesome.com/icons/link -->
    <svg width="16" height="16" viewBox="0 0 512 512">
        <path
            fill="currentColor"
            d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
        ></path>
    </svg>
</template>
<!-- accordion plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin allows sections of content under <h2> headings
        // to be collapsible.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'accordion';

        // default plugin options
        const options = {
            // whether to always start expanded ('false'), always start
            // collapsed ('true'), or start collapsed when screen small ('auto')
            startCollapsed: 'auto',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // run through each <h2> heading
            const headings = document.querySelectorAll('h2');
            for (const heading of headings) {
                addArrow(heading);

                // start expanded/collapsed based on option
                if (
                    options.startCollapsed === 'true' ||
                    (options.startCollapsed === 'auto' && isSmallScreen())
                )
                    collapseHeading(heading);
                else
                    expandHeading(heading);
            }

            // attach hash change listener to window
            window.addEventListener('hashchange', onHashChange);
        }

        // when hash (eg manuscript.html#introduction) changes
        function onHashChange() {
            const target = getHashTarget();
            if (target)
                goToElement(target);
        }

        // add arrow to heading
        function addArrow(heading) {
            // add arrow button
            const arrow = document.createElement('button');
            arrow.innerHTML = document.querySelector(
                '.icon_angle_down'
            ).innerHTML;
            arrow.classList.add('icon_button', 'accordion_arrow');
            heading.insertBefore(arrow, heading.firstChild);

            // attach click listener to heading and button
            heading.addEventListener('click', onHeadingClick);
            arrow.addEventListener('click', onArrowClick);
        }

        // determine if on mobile-like device with small screen
        function isSmallScreen() {
            return Math.min(window.innerWidth, window.innerHeight) < 480;
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // get element that is target of hash
        function getHashTarget(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (id.indexOf('fig:') === 0)
                target = target.querySelector('figure');
            if (id.indexOf('tbl:') === 0)
                target = target.querySelector('table');

            return target;
        }

        // when <h2> heading is clicked
        function onHeadingClick(event) {
            // only collapse if <h2> itself is target of click (eg, user did
            // not click on anchor within <h2>)
            if (event.target === this)
                toggleCollapse(this);
        }

        // when arrow button is clicked
        function onArrowClick() {
            toggleCollapse(this.parentNode);
        }

        // collapse section if expanded, expand if collapsed
        function toggleCollapse(heading) {
            if (heading.dataset.collapsed === 'false')
                collapseHeading(heading);
            else
                expandHeading(heading);
        }

        // elements to exclude from collapse, such as table of contents panel,
        // hypothesis panel, etc
        const exclude = '#toc_panel, div.annotator-frame, #lightbox_overlay';

        // collapse section
        function collapseHeading(heading) {
            heading.setAttribute('data-collapsed', 'true');
            const children = getChildren(heading);
            for (const child of children)
                child.setAttribute('data-collapsed', 'true');
        }

        // expand section
        function expandHeading(heading) {
            heading.setAttribute('data-collapsed', 'false');
            const children = getChildren(heading);
            for (const child of children)
                child.setAttribute('data-collapsed', 'false');
        }

        // get list of elements between this <h2> and next <h2> or <h1>
        // ("children" of the <h2> section)
        function getChildren(heading) {
            return nextUntil(heading, 'h2, h1', exclude);
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get list of elements after a start element up to element matching
        // query
        function nextUntil(element, query, exclude) {
            const elements = [];
            while (element = element.nextElementSibling, element) {
                if (element.matches(query))
                    break;
                if (!element.matches(exclude))
                    elements.push(element);
            }
            return elements;
        }

        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
    <!-- modified from: https://fontawesome.com/icons/angle-down -->
    <svg width="16" height="16" viewBox="0 0 448 512">
        <path
            fill="currentColor"
            d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
        ></path>
    </svg>
</template>
<!-- tooltips plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when the user hovers or
        // focuses a link to a citation or figure, a tooltip appears with a
        // preview of the reference content, along with arrows to navigate
        // between instances of the same reference in the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'tooltips';

        // default plugin options
        const options = {
            // whether user must click off to close tooltip instead of just
            // un-hovering
            clickClose: 'false',
            // delay (in ms) between opening and closing tooltip
            delay: '100',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            const links = getLinks();
            for (const link of links) {
                // attach hover and focus listeners to link
                link.addEventListener('mouseover', onLinkHover);
                link.addEventListener('mouseleave', onLinkUnhover);
                link.addEventListener('focus', onLinkFocus);
                link.addEventListener('touchend', onLinkTouch);
            }

            // attach mouse, key, and resize listeners to window
            window.addEventListener('mousedown', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('keyup', onKeyUp);
            window.addEventListener('resize', onResize);
        }

        // when link is hovered
        function onLinkHover() {
            // function to open tooltip
            const delayOpenTooltip = function() {
                openTooltip(this);
            }.bind(this);

            // run open function after delay
            this.openTooltipTimer = window.setTimeout(
                delayOpenTooltip,
                options.delay
            );
        }

        // when mouse leaves link
        function onLinkUnhover() {
            // cancel opening tooltip
            window.clearTimeout(this.openTooltipTimer);

            // don't close on unhover if option specifies
            if (options.clickClose === 'true')
                return;

            // function to close tooltip
            const delayCloseTooltip = function() {
                // if tooltip open and if mouse isn't over tooltip, close
                const tooltip = document.getElementById('tooltip');
                if (tooltip && !tooltip.matches(':hover'))
                    closeTooltip();
            };

            // run close function after delay
            this.closeTooltipTimer = window.setTimeout(
                delayCloseTooltip,
                options.delay
            );
        }

        // when link is focused (tabbed to)
        function onLinkFocus(event) {
            openTooltip(this);
        }

        // when link is touched on touch screen
        function onLinkTouch(event) {
            // attempt to force hover state on first tap always, and trigger
            // regular link click (and navigation) on second tap
            if (event.target === document.activeElement)
                event.target.click();
            else {
                document.activeElement.blur();
                event.target.focus();
            }
            if (event.cancelable)
                event.preventDefault();
            event.stopPropagation();
            return false;
        }

        // when mouse is clicked anywhere in window
        function onClick(event) {
            closeTooltip();
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            switch (event.key) {
                // trigger click of prev button
                case 'ArrowLeft':
                    const prevButton = document.getElementById(
                        'tooltip_prev_button'
                    );
                    if (prevButton)
                        prevButton.click();
                    break;
                // trigger click of next button
                case 'ArrowRight':
                    const nextButton = document.getElementById(
                        'tooltip_next_button'
                    );
                    if (nextButton)
                        nextButton.click();
                    break;
                // close on esc
                case 'Escape':
                    closeTooltip();
                    break;
            }
        }

        // when window is resized or zoomed
        function onResize() {
            closeTooltip();
        }

        // get all links of types we wish to handle
        function getLinks() {
            const queries = [];
            // exclude buttons, anchor links, toc links, etc
            const exclude =
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            queries.push('a[href^="#ref-"]' + exclude); // citation links
            queries.push('a[href^="#fig:"]' + exclude); // figure links
            const query = queries.join(', ');
            return document.querySelectorAll(query);
        }

        // get links with same target, get index of link in set, get total
        // same links
        function getSameLinks(link) {
            const sameLinks = [];
            const links = getLinks();
            for (const otherLink of links) {
                if (
                    otherLink.getAttribute('href') === link.getAttribute('href')
                )
                    sameLinks.push(otherLink);
            }

            return {
                elements: sameLinks,
                index: sameLinks.indexOf(link),
                total: sameLinks.length
            };
        }

        // open tooltip
        function openTooltip(link) {
            // delete tooltip if it exists, start fresh
            closeTooltip();

            // make tooltip element
            const tooltip = makeTooltip(link);

            // if source couldn't be found and tooltip not made, exit
            if (!tooltip)
                return;

            // make navbar elements
            const navBar = makeNavBar(link);
            if (navBar)
                tooltip.firstElementChild.appendChild(navBar);

            // attach tooltip to page
            document.body.appendChild(tooltip);

            // position tooltip
            const position = function() {
                positionTooltip(link);
            };
            position();

            // if tooltip contains images, position again after they've loaded
            const imgs = tooltip.querySelectorAll('img');
            for (const img of imgs)
                img.addEventListener('load', position);
        }

        // close (delete) tooltip
        function closeTooltip() {
            const tooltip = document.getElementById('tooltip');
            if (tooltip)
                tooltip.remove();
        }

        // make tooltip
        function makeTooltip(link) {
            // get target element that link points to
            const source = getSource(link);

            // if source can't be found, exit
            if (!source)
                return;

            // create new tooltip
            const tooltip = document.createElement('div');
            tooltip.id = 'tooltip';
            const tooltipContent = document.createElement('div');
            tooltipContent.id = 'tooltip_content';
            tooltip.appendChild(tooltipContent);

            // make copy of source node and put in tooltip
            const sourceCopy = makeCopy(source);
            tooltipContent.appendChild(sourceCopy);

            // attach mouse event listeners
            tooltip.addEventListener('click', onTooltipClick);
            tooltip.addEventListener('mousedown', onTooltipClick);
            tooltip.addEventListener('touchstart', onTooltipClick);
            tooltip.addEventListener('mouseleave', onTooltipUnhover);

            // (for interaction with lightbox plugin)
            // transfer click on tooltip copied img to original img
            const sourceImg = source.querySelector('img');
            const sourceCopyImg = sourceCopy.querySelector('img');
            if (sourceImg && sourceCopyImg) {
                const clickImg = function() {
                    sourceImg.click();
                    closeTooltip();
                };
                sourceCopyImg.addEventListener('click', clickImg);
            }

            return tooltip;
        }

        // make carbon copy of html dom element
        function makeCopy(source) {
            const sourceCopy = source.cloneNode(true);

            // delete elements marked with ignore (eg anchor and jump buttons)
            const deleteFromCopy = sourceCopy.querySelectorAll(
                '[data-ignore="true"]'
            );
            for (const element of deleteFromCopy)
                element.remove();

            // delete certain element attributes
            const attributes = [
                'id',
                'data-collapsed',
                'data-selected',
                'data-highlighted',
                'data-glow'
            ];
            for (const attribute of attributes) {
                sourceCopy.removeAttribute(attribute);
                const elements = sourceCopy.querySelectorAll(
                    '[' + attribute + ']'
                );
                for (const element of elements)
                    element.removeAttribute(attribute);
            }

            return sourceCopy;
        }

        // when tooltip is clicked
        function onTooltipClick(event) {
            // when user clicks on tooltip, stop click from transferring
            // outside of tooltip (eg, click off to close tooltip, or eg click
            // off to unhighlight same refs)
            event.stopPropagation();
        }

        // when tooltip is unhovered
        function onTooltipUnhover(event) {
            if (options.clickClose === 'true')
                return;

            // make sure new mouse/touch/focus no longer over tooltip or any
            // element within it
            const tooltip = document.getElementById('tooltip');
            if (!tooltip)
                return;
            if (this.contains(event.relatedTarget))
                return;

            closeTooltip();
        }

        // make nav bar to go betwen prev/next instances of same reference
        function makeNavBar(link) {
            // find other links to the same source
            const sameLinks = getSameLinks(link);

            // don't show nav bar when singular reference
            if (sameLinks.total <= 1)
                return;

            // find prev/next links with same target
            const prevLink = getPrevLink(link, sameLinks);
            const nextLink = getNextLink(link, sameLinks);

            // create nav bar
            const navBar = document.createElement('div');
            navBar.id = 'tooltip_nav_bar';
            const text = sameLinks.index + 1 + ' of ' + sameLinks.total;

            // create nav bar prev/next buttons
            const prevButton = document.createElement('button');
            const nextButton = document.createElement('button');
            prevButton.id = 'tooltip_prev_button';
            nextButton.id = 'tooltip_next_button';
            prevButton.title =
                'Jump to the previous occurence of this item in the document [‚Üê]';
            nextButton.title =
                'Jump to the next occurence of this item in the document [‚Üí]';
            prevButton.classList.add('icon_button');
            nextButton.classList.add('icon_button');
            prevButton.innerHTML = document.querySelector(
                '.icon_caret_left'
            ).innerHTML;
            nextButton.innerHTML = document.querySelector(
                '.icon_caret_right'
            ).innerHTML;
            navBar.appendChild(prevButton);
            navBar.appendChild(document.createTextNode(text));
            navBar.appendChild(nextButton);

            // attach click listeners to buttons
            prevButton.addEventListener('click', function() {
                onPrevNextClick(link, prevLink);
            });
            nextButton.addEventListener('click', function() {
                onPrevNextClick(link, nextLink);
            });

            return navBar;
        }

        // get previous link with same target
        function getPrevLink(link, sameLinks) {
            if (!sameLinks)
                sameLinks = getSameLinks(link);
            // wrap index to other side if < 1
            let index;
            if (sameLinks.index - 1 >= 0)
                index = sameLinks.index - 1;
            else
                index = sameLinks.total - 1;
            return sameLinks.elements[index];
        }

        // get next link with same target
        function getNextLink(link, sameLinks) {
            if (!sameLinks)
                sameLinks = getSameLinks(link);
            // wrap index to other side if > total
            let index;
            if (sameLinks.index + 1 <= sameLinks.total - 1)
                index = sameLinks.index + 1;
            else
                index = 0;
            return sameLinks.elements[index];
        }

        // get element that is target of link or url hash
        function getSource(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            // if ref or figure, modify target to get expected element
            if (id.indexOf('ref-') === 0)
                target = target.querySelector('p');
            else if (id.indexOf('fig:') === 0)
                target = target.querySelector('figure');

            return target;
        }

        // when prev/next arrow button is clicked
        function onPrevNextClick(link, prevNextLink) {
            if (link && prevNextLink)
                goToElement(prevNextLink, window.innerHeight * 0.5);
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // determine position to place tooltip based on link position in
        // viewport and tooltip size
        function positionTooltip(link, left, top) {
            const tooltipElement = document.getElementById('tooltip');
            if (!tooltipElement)
                return;

            // get convenient vars for position/dimensions of
            // link/tooltip/page/view
            link = getRectInPage(link);
            const tooltip = getRectInPage(tooltipElement);
            const view = getRectInPage();

            // horizontal positioning
            if (left)
                // use explicit value
                left = left;
            else if (link.left + tooltip.width < view.right)
                // fit tooltip to right of link
                left = link.left;
            else if (link.right - tooltip.width > view.left)
                // fit tooltip to left of link
                left = link.right - tooltip.width;
            // center tooltip in view
            else
                left = (view.right - view.left) / 2 - tooltip.width / 2;

            // vertical positioning
            if (top)
                // use explicit value
                top = top;
            else if (link.top - tooltip.height > view.top)
                // fit tooltip above link
                top = link.top - tooltip.height;
            else if (link.bottom + tooltip.height < view.bottom)
                // fit tooltip below link
                top = link.bottom;
            else {
                // center tooltip in view
                top = view.top + view.height / 2 - tooltip.height / 2;
                // nudge off of link to left/right if possible
                if (link.right + tooltip.width < view.right)
                    left = link.right;
                else if (link.left - tooltip.width > view.left)
                    left = link.left - tooltip.width;
            }

            tooltipElement.style.left = left + 'px';
            tooltipElement.style.top = top + 'px';
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get position of element relative to page
        function getRectInPage(element) {
            const rect = getRectInView(element);
            const body = getRectInView(document.body);

            const newRect = {};
            newRect.left = rect.left - body.left;
            newRect.top = rect.top - body.top;
            newRect.right = rect.right - body.left;
            newRect.bottom = rect.bottom - body.top;
            newRect.width = rect.width;
            newRect.height = rect.height;

            return newRect;
        }

        // (for interaction with accordion plugin)
        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // (for interaction with accordion plugin)
        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // (for interaction with accordion plugin)
        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
    <!-- modified from: https://fontawesome.com/icons/caret-left -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
        ></path>
    </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
    <!-- modified from: https://fontawesome.com/icons/caret-right -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
        ></path>
    </svg>
</template>
<!-- jump to first plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin adds a button next to each reference entry,
        // figure, and table that jumps the page to the first occurrence of a
        // link to that item in the manuscript.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'jumpToFirst';

        // default plugin options
        const options = {
            // whether to add buttons next to reference entries
            references: 'true',
            // whether to add buttons next to figures
            figures: 'true',
            // whether to add buttons next to tables
            tables: 'true',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            if (options.references !== 'false')
                makeReferenceButtons();
            if (options.figures !== 'false')
                makeFigureButtons();
            if (options.tables !== 'false')
                makeTableButtons();
        }

        // when jump button clicked
        function onButtonClick() {
            const first = getFirstOccurrence(this.dataset.id);
            if (!first)
                return;

            // update url hash so navigating "back" in history will return
            // user to jump button
            window.location.hash = this.dataset.id;
            // scroll to link
            window.setTimeout(function() {
                goToElement(first, window.innerHeight * 0.5);
            }, 0);
        }

        // get first occurence of link to item in document
        function getFirstOccurrence(id) {
            let query = 'a';
            query += '[href="#' + id + '"]';
            // exclude buttons, anchor links, toc links, etc
            query +=
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            return document.querySelector(query);
        }

        // add button next to each reference entry
        function makeReferenceButtons() {
            const references = document.querySelectorAll('div[id^="ref-"]');
            for (const reference of references) {
                // get reference id and element to add button to
                const id = reference.id;
                const container = reference.firstElementChild;
                const first = getFirstOccurrence(id);

                // if can't find link to reference, ignore
                if (!first)
                    continue;

                // make jump button
                let button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this reference in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.innerHTML = button.outerHTML + container.innerHTML;
                button = container.firstElementChild;
                button.addEventListener('click', onButtonClick);
            }
        }

        // add button next to each figure
        function makeFigureButtons() {
            const figures = document.querySelectorAll('[id^="fig:"]');
            for (const figure of figures) {
                // get figure id and element to add button to
                const id = figure.id;
                const container = figure.querySelector('figcaption') || figure;
                const first = getFirstOccurrence(id);

                // if can't find link to figure, ignore
                if (!first)
                    continue;

                // make jump button
                const button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this figure in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.insertBefore(button, container.firstElementChild);
                button.addEventListener('click', onButtonClick);
            }
        }

        // add button next to each figure
        function makeTableButtons() {
            const tables = document.querySelectorAll('[id^="tbl:"]');
            for (const table of tables) {
                // get ref id and element to add button to
                const id = table.id;
                const container = table.querySelector('caption') || table;
                const first = getFirstOccurrence(id);

                // if can't find link to table, ignore
                if (!first)
                    continue;

                // make jump button
                const button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this table in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.insertBefore(button, container.firstElementChild);
                button.addEventListener('click', onButtonClick);
            }
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // (for interaction with accordion plugin)
        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
    <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
    <svg width="16" height="16" viewBox="0 0 320 512">
        <path
            fill="currentColor"
            d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
        ></path>
    </svg>
</template>
<!-- link highlight plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when a user hovers or
        // focuses a link, other links that have the same target will be
        // highlighted. It also makes it such that when clicking a link, the
        // target of the link (eg reference, figure, table) is briefly
        // highlighted.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'linkHighlight';

        // default plugin options
        const options = {
            // whether to also highlight links that go to external urls
            externalLinks: 'false',
            // whether user must click off to unhighlight instead of just
            // un-hovering
            clickUnhighlight: 'false',
            // whether to also highlight links that are unique
            highlightUnique: 'true',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            const links = getLinks();
            for (const link of links) {
                // attach mouse and focus listeners to link
                link.addEventListener('mouseenter', onLinkFocus);
                link.addEventListener('focus', onLinkFocus);
                link.addEventListener('mouseleave', onLinkUnhover);
            }

            // attach click and hash change listeners to window
            window.addEventListener('click', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('hashchange', onHashChange);

            // run hash change on window load in case user has navigated
            // directly to hash
            onHashChange();
        }

        // when link is focused (tabbed to) or hovered
        function onLinkFocus() {
            highlight(this);
        }

        // when link is unhovered
        function onLinkUnhover() {
            if (options.clickUnhighlight !== 'true')
                unhighlightAll();
        }

        // when the mouse is clicked anywhere in window
        function onClick(event) {
            unhighlightAll();
        }

        // when hash (eg manuscript.html#introduction) changes
        function onHashChange() {
            const target = getHashTarget();
            if (target)
                glowElement(target);
        }

        // get element that is target of link or url hash
        function getHashTarget(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            return target;
        }

        // start glow sequence on an element
        function glowElement(element) {
            const startGlow = function() {
                onGlowEnd();
                element.dataset.glow = 'true';
                element.addEventListener('animationend', onGlowEnd);
            };
            const onGlowEnd = function() {
                element.removeAttribute('data-glow');
                element.removeEventListener('animationend', onGlowEnd);
            };
            startGlow();
        }

        // highlight link and all others with same target
        function highlight(link) {
            // force unhighlight all to start fresh
            unhighlightAll();

            // get links with same target
            if (!link)
                return;
            const sameLinks = getSameLinks(link);

            // if link unique and option is off, exit and don't highlight
            if (sameLinks.length <= 1 && options.highlightUnique !== 'true')
                return;

            // highlight all same links, and "select" (special highlight) this
            // one
            for (const sameLink of sameLinks) {
                if (sameLink === link)
                    sameLink.setAttribute('data-selected', 'true');
                else
                    sameLink.setAttribute('data-highlighted', 'true');
            }
        }

        // unhighlight all links
        function unhighlightAll() {
            const links = getLinks();
            for (const link of links) {
                link.setAttribute('data-selected', 'false');
                link.setAttribute('data-highlighted', 'false');
            }
        }

        // get links with same target
        function getSameLinks(link) {
            const results = [];
            const links = getLinks();
            for (const otherLink of links) {
                if (
                    otherLink.getAttribute('href') === link.getAttribute('href')
                )
                    results.push(otherLink);
            }
            return results;
        }

        // get all links of types we wish to handle
        function getLinks() {
            let query = 'a';
            if (options.externalLinks !== 'true')
                query += '[href^="#"]';
            // exclude buttons, anchor links, toc links, etc
            query +=
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            return document.querySelectorAll(query);
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>
<!-- table of contents plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin provides a "table of contents" (toc) panel on
        // the side of the document that allows the user to conveniently
        // navigate between sections of the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'tableOfContents';

        // default plugin options
        const options = {
            // which types of elements to add links for, in
            // "document.querySelector" format
            typesQuery: 'h1, h2, h3',
            // whether default behavior is to be closed ('false'), open
            // ('true'), or only open when screen wide enough to fit panel
            // ('auto'). note: still always starts closed when page loads.
            open: 'auto',
            // if list item is more than this many characters, text will be
            // truncated
            charLimit: '50',
            // whether or not to show bullets next to each toc item
            bullets: 'false',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // make toc panel and populate with entries (links to document
            // sections)
            const panel = makePanel();
            if (!panel)
                return;
            makeEntries(panel);
            document.body.insertBefore(panel, document.body.firstChild);

            closePanel();

            // attach click, scroll, and hash change listeners to window
            window.addEventListener('click', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('scroll', onScroll);
            window.addEventListener('hashchange', onScroll);
            window.addEventListener('keyup', onKeyUp);
            onScroll();

            // add class to push document body down out of way of toc button
            document.body.classList.add('toc_body_nudge');
        }

        // determine if screen wide enough to fit toc panel
        function isSmallScreen() {
            // in default theme:
            // 816px = 8.5in = width of "page" (<body>) element
            // 260px = min width of toc panel (*2 for both sides of <body>)
            return window.innerWidth < 816 + 260 * 2;
        }

        // open/close panel based on option and screen size
        function openOrClosePanel() {
            if (
                options.open === 'true' ||
                (options.open === 'auto' && !isSmallScreen())
            )
                openPanel();
            else
                closePanel();
        }

        // when mouse is clicked anywhere in window
        function onClick() {
            const panel = document.getElementById('toc_panel');
            if (!panel)
                return;

            if (panel.dataset.open === 'true')
                openOrClosePanel();
        }

        // when window is scrolled or hash changed
        function onScroll() {
            highlightViewed();
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            // close on esc
            if (event.key === 'Escape')
                closePanel();
        }

        // find entry of currently viewed document section in toc and highlight
        function highlightViewed() {
            const firstId = getFirstInView(options.typesQuery);

            // get toc entries (links), unhighlight all, then highlight viewed
            const list = document.getElementById('toc_list');
            if (!firstId || !list)
                return;
            const links = list.querySelectorAll('a');
            for (const link of links)
                link.dataset.viewing = 'false';
            const link = list.querySelector('a[href="#' + firstId + '"]');
            if (!link)
                return;
            link.dataset.viewing = 'true';
        }

        // get first or previous toc listed element in top half of view
        function getFirstInView(query) {
            // get all elements matching query and with id
            const elements = document.querySelectorAll(query);
            const elementsWithIds = [];
            for (const element of elements) {
                if (element.id)
                    elementsWithIds.push(element);
            }


            // get first or previous element in top half of view
            for (let i = 0; i < elementsWithIds.length; i++) {
                const element = elementsWithIds[i];
                const prevElement = elementsWithIds[Math.max(0, i - 1)];
                if (element.getBoundingClientRect().top >= 0) {
                    if (
                        element.getBoundingClientRect().top <
                        window.innerHeight / 2
                    )
                        return element.id;
                    else
                        return prevElement.id;
                }
            }
        }

        // make panel
        function makePanel() {
            // create panel
            const panel = document.createElement('div');
            panel.id = 'toc_panel';
            if (options.bullets === 'true')
                panel.dataset.bullets = 'true';

            // create header
            const header = document.createElement('div');
            header.id = 'toc_header';

            // create toc button
            const button = document.createElement('button');
            button.id = 'toc_button';
            button.innerHTML = document.querySelector(
                '.icon_th_list'
            ).innerHTML;
            button.classList.add('icon_button');

            // create header text
            const text = document.createElement('h3');
            text.innerHTML = 'Table of Contents';

            // create container for toc list
            const list = document.createElement('div');
            list.id = 'toc_list';

            // attach click listeners
            panel.addEventListener('click', onPanelClick);
            header.addEventListener('click', onHeaderClick);
            button.addEventListener('click', onButtonClick);

            // attach elements
            header.appendChild(button);
            header.appendChild(text);
            panel.appendChild(header);
            panel.appendChild(list);

            return panel;
        }

        // create toc entries (links) to each element of the specified types
        function makeEntries(panel) {
            const elements = document.querySelectorAll(options.typesQuery);
            for (const element of elements) {
                // do not add link if element doesn't have assigned id
                if (!element.id)
                    continue;

                // create link/list item
                const link = document.createElement('a');
                link.classList.add('toc_link');
                switch (element.tagName.toLowerCase()) {
                    case 'h1':
                        link.dataset.level = '1';
                        break;
                    case 'h2':
                        link.dataset.level = '2';
                        break;
                    case 'h3':
                        link.dataset.level = '3';
                        break;
                    case 'h4':
                        link.dataset.level = '4';
                        break;
                }
                link.title = element.innerText;
                let text = element.innerText;
                if (text.length > options.charLimit)
                    text = text.slice(0, options.charLimit) + '...';
                link.innerHTML = text;
                link.href = '#' + element.id;
                link.addEventListener('click', onLinkClick);

                // attach link
                panel.querySelector('#toc_list').appendChild(link);
            }
        }

        // when panel is clicked
        function onPanelClick(event) {
            // stop click from propagating to window/document and closing panel
            event.stopPropagation();
        }

        // when header itself is clicked
        function onHeaderClick(event) {
            togglePanel();
        }

        // when button is clicked
        function onButtonClick(event) {
            togglePanel();
            // stop header underneath button from also being clicked
            event.stopPropagation();
        }

        // when link is clicked
        function onLinkClick() {
            openOrClosePanel();
        }

        // open panel if closed, close if opened
        function togglePanel() {
            const panel = document.getElementById('toc_panel');
            if (!panel)
                return;

            if (panel.dataset.open === 'true')
                closePanel();
            else
                openPanel();
        }

        // open panel
        function openPanel() {
            const panel = document.getElementById('toc_panel');
            if (panel)
                panel.dataset.open = 'true';
        }

        // close panel
        function closePanel() {
            const panel = document.getElementById('toc_panel');
            if (panel)
                panel.dataset.open = 'false';
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- th list icon -->

<template class="icon_th_list">
    <!-- modified from: https://fontawesome.com/icons/th-list -->
    <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
        <path
            fill="currentColor"
            d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
            tabindex="-1"
        ></path>
    </svg>
</template>
<!-- lightbox plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when a user clicks on an
        // image, the image fills the screen and the user can pan/drag/zoom
        // the image and navigate between other images in the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'lightbox';

        // default plugin options
        const options = {
            // list of possible zoom/scale factors
            zoomSteps:
                '0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1,' +
                '1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8',
            // whether to fit image to view ('fit'), display at 100% and shrink
            // if necessary ('shrink'), or always display at 100% ('100')
            defaultZoom: 'fit',
            // whether to zoom in/out toward center of view ('true') or mouse
            // ('false')
            centerZoom: 'false',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // run through each <img> element
            const imgs = document.querySelectorAll('figure > img');
            let count = 1;
            for (const img of imgs) {
                img.classList.add('lightbox_document_img');
                img.dataset.number = count;
                img.dataset.total = imgs.length;
                img.addEventListener('click', openLightbox);
                count++;
            }

            // attach mouse and key listeners to window
            window.addEventListener('mousemove', onWindowMouseMove);
            window.addEventListener('keyup', onKeyUp);
        }

        // when mouse is moved anywhere in window
        function onWindowMouseMove(event) {
            window.mouseX = event.clientX;
            window.mouseY = event.clientY;
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            switch (event.key) {
                // trigger click of prev button
                case 'ArrowLeft':
                    const prevButton = document.getElementById(
                        'lightbox_prev_button'
                    );
                    if (prevButton)
                        prevButton.click();
                    break;
                // trigger click of next button
                case 'ArrowRight':
                    const nextButton = document.getElementById(
                        'lightbox_next_button'
                    );
                    if (nextButton)
                        nextButton.click();
                    break;
                // close on esc
                case 'Escape':
                    closeLightbox();
                    break;
            }
        }

        // open lightbox
        function openLightbox() {
            const lightbox = makeLightbox(this);
            if (!lightbox)
                return;

            blurBody(lightbox);
            document.body.appendChild(lightbox);
        }

        // make lightbox
        function makeLightbox(img) {
            // delete lightbox if it exists, start fresh
            closeLightbox();

            // create screen overlay containing lightbox
            const overlay = document.createElement('div');
            overlay.id = 'lightbox_overlay';

            // create image info boxes
            const numberInfo = document.createElement('div');
            const zoomInfo = document.createElement('div');
            numberInfo.id = 'lightbox_number_info';
            zoomInfo.id = 'lightbox_zoom_info';

            // create container for image
            const imageContainer = document.createElement('div');
            imageContainer.id = 'lightbox_image_container';
            const lightboxImg = makeLightboxImg(
                img,
                imageContainer,
                numberInfo,
                zoomInfo
            );
            imageContainer.appendChild(lightboxImg);

            // create bottom container for caption and navigation buttons
            const bottomContainer = document.createElement('div');
            bottomContainer.id = 'lightbox_bottom_container';
            const caption = makeCaption(img);
            const prevButton = makePrevButton(img);
            const nextButton = makeNextButton(img);
            bottomContainer.appendChild(prevButton);
            bottomContainer.appendChild(caption);
            bottomContainer.appendChild(nextButton);

            // attach top middle and bottom to overlay
            overlay.appendChild(numberInfo);
            overlay.appendChild(zoomInfo);
            overlay.appendChild(imageContainer);
            overlay.appendChild(bottomContainer);

            return overlay;
        }

        // make <img> object that is intuitively draggable and zoomable
        function makeLightboxImg(
            sourceImg,
            container,
            numberInfoBox,
            zoomInfoBox
        ) {
            // create copy of source <img>
            const img = sourceImg.cloneNode(true);
            img.classList.remove('lightbox_document_img');
            img.removeAttribute('id');
            img.removeAttribute('width');
            img.removeAttribute('height');
            img.style.position = 'unset';
            img.style.margin = '0';
            img.style.padding = '0';
            img.style.width = '';
            img.style.height = '';
            img.style.minWidth = '';
            img.style.minHeight = '';
            img.style.maxWidth = '';
            img.style.maxHeight = '';
            img.id = 'lightbox_img';

            // build sorted list of unique zoomSteps, always including a 100%
            let zoomSteps = [];
            const optionsZooms = options.zoomSteps.split(/[^0-9.]/);
            for (const optionZoom of optionsZooms) {
                const newZoom = parseFloat(optionZoom);
                if (newZoom && !zoomSteps.includes(newZoom))
                    zoomSteps.push(newZoom);
            }
            if (!zoomSteps.includes(1))
                zoomSteps.push(1);
            zoomSteps = zoomSteps.sort(function sortNumber(a, b) {
                return a - b;
            });

            // <img> object property variables
            let zoom = 1;
            let translateX = 0;
            let translateY = 0;
            let clickMouseX = undefined;
            let clickMouseY = undefined;
            let clickTranslateX = undefined;
            let clickTranslateY = undefined;

            updateNumberInfo();

            // update image numbers displayed in info box
            function updateNumberInfo() {
                numberInfoBox.innerHTML =
                    sourceImg.dataset.number + ' of ' + sourceImg.dataset.total;
            }

            // update zoom displayed in info box
            function updateZoomInfo() {
                let zoomInfo = zoom * 100;
                if (!Number.isInteger(zoomInfo))
                    zoomInfo = zoomInfo.toFixed(2);
                zoomInfoBox.innerHTML = zoomInfo + '%';
            }

            // move to closest zoom step above current zoom
            const zoomIn = function() {
                for (const zoomStep of zoomSteps) {
                    if (zoomStep > zoom) {
                        zoom = zoomStep;
                        break;
                    }
                }
                updateTransform();
            };

            // move to closest zoom step above current zoom
            const zoomOut = function() {
                zoomSteps.reverse();
                for (const zoomStep of zoomSteps) {
                    if (zoomStep < zoom) {
                        zoom = zoomStep;
                        break;
                    }
                }
                zoomSteps.reverse();

                updateTransform();
            };

            // update display of <img> based on scale/translate properties
            const updateTransform = function() {
                // set transform
                img.style.transform =
                    'translate(' +
                    (translateX || 0) +
                    'px,' +
                    (translateY || 0) +
                    'px) scale(' +
                    (zoom || 1) +
                    ')';

                // get new width/height after scale
                const rect = img.getBoundingClientRect();
                // limit translate
                translateX = Math.max(translateX, -rect.width / 2);
                translateX = Math.min(translateX, rect.width / 2);
                translateY = Math.max(translateY, -rect.height / 2);
                translateY = Math.min(translateY, rect.height / 2);

                // set transform
                img.style.transform =
                    'translate(' +
                    (translateX || 0) +
                    'px,' +
                    (translateY || 0) +
                    'px) scale(' +
                    (zoom || 1) +
                    ')';

                updateZoomInfo();
            };

            // fit <img> to container
            const fit = function() {
                // no x/y offset, 100% zoom by default
                translateX = 0;
                translateY = 0;
                zoom = 1;

                // widths of <img> and container
                const imgWidth = img.naturalWidth;
                const imgHeight = img.naturalHeight;
                const containerWidth = parseFloat(
                    window.getComputedStyle(container).width
                );
                const containerHeight = parseFloat(
                    window.getComputedStyle(container).height
                );

                // how much zooming is needed to fit <img> to container
                const xRatio = imgWidth / containerWidth;
                const yRatio = imgHeight / containerHeight;
                const maxRatio = Math.max(xRatio, yRatio);
                const newZoom = 1 / maxRatio;

                // fit <img> to container according to option
                if (options.defaultZoom === 'shrink') {
                    if (maxRatio > 1)
                        zoom = newZoom;
                } else if (options.defaultZoom === 'fit')
                    zoom = newZoom;

                updateTransform();
            };

            // when mouse wheel is rolled anywhere in container
            const onContainerWheel = function(event) {
                if (!event)
                    return;

                // let ctrl + mouse wheel to zoom behave as normal
                if (event.ctrlKey)
                    return;

                // prevent normal scroll behavior
                event.preventDefault();
                event.stopPropagation();

                // point around which to scale img
                const viewRect = container.getBoundingClientRect();
                const viewX = (viewRect.left + viewRect.right) / 2;
                const viewY = (viewRect.top + viewRect.bottom) / 2;
                const originX = options.centerZoom === 'true' ? viewX : mouseX;
                const originY = options.centerZoom === 'true' ? viewY : mouseY;

                // get point on image under origin
                const oldRect = img.getBoundingClientRect();
                const oldPercentX = (originX - oldRect.left) / oldRect.width;
                const oldPercentY = (originY - oldRect.top) / oldRect.height;

                // increment/decrement zoom
                if (event.deltaY < 0)
                    zoomIn();
                if (event.deltaY > 0)
                    zoomOut();

                // get offset between previous image point and origin
                const newRect = img.getBoundingClientRect();
                const offsetX =
                    originX - (newRect.left + newRect.width * oldPercentX);
                const offsetY =
                    originY - (newRect.top + newRect.height * oldPercentY);

                // translate image to keep image point under origin
                translateX += offsetX;
                translateY += offsetY;

                // perform translate
                updateTransform();
            };

            // when container is clicked
            function onContainerClick(event) {
                // if container itself is target of click, and not other
                // element above it
                if (event.target === this)
                    closeLightbox();
            }

            // when mouse button is pressed on image
            const onImageMouseDown = function(event) {
                // store original mouse position relative to image
                clickMouseX = window.mouseX;
                clickMouseY = window.mouseY;
                clickTranslateX = translateX;
                clickTranslateY = translateY;
                event.stopPropagation();
                event.preventDefault();
            };

            // when mouse button is released anywhere in window
            const onWindowMouseUp = function(event) {
                // reset original mouse position
                clickMouseX = undefined;
                clickMouseY = undefined;
                clickTranslateX = undefined;
                clickTranslateY = undefined;

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('mouseup', onWindowMouseUp);
            };

            // when mouse is moved anywhere in window
            const onWindowMouseMove = function(event) {
                if (
                    clickMouseX === undefined ||
                    clickMouseY === undefined ||
                    clickTranslateX === undefined ||
                    clickTranslateY === undefined
                )
                    return;

                // offset image based on original and current mouse position
                translateX = clickTranslateX + window.mouseX - clickMouseX;
                translateY = clickTranslateY + window.mouseY - clickMouseY;
                updateTransform();
                event.preventDefault();

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('mousemove', onWindowMouseMove);
            };

            // when window is resized
            const onWindowResize = function(event) {
                fit();

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('resize', onWindowResize);
            };

            // attach the necessary event listeners
            img.addEventListener('dblclick', fit);
            img.addEventListener('mousedown', onImageMouseDown);
            container.addEventListener('wheel', onContainerWheel);
            container.addEventListener('mousedown', onContainerClick);
            container.addEventListener('touchstart', onContainerClick);
            window.addEventListener('mouseup', onWindowMouseUp);
            window.addEventListener('mousemove', onWindowMouseMove);
            window.addEventListener('resize', onWindowResize);

            // run fit() after lightbox atttached to document and <img> Loaded
            // so needed container and img dimensions available
            img.addEventListener('load', fit);

            return img;
        }

        // make caption
        function makeCaption(img) {
            const caption = document.createElement('div');
            caption.id = 'lightbox_caption';
            const captionSource = img.nextElementSibling;
            if (captionSource.tagName.toLowerCase() === 'figcaption') {
                const captionCopy = makeCopy(captionSource);
                caption.innerHTML = captionCopy.innerHTML;
            }

            caption.addEventListener('touchstart', function(event) {
                event.stopPropagation();
            });

            return caption;
        }

        // make carbon copy of html dom element
        function makeCopy(source) {
            const sourceCopy = source.cloneNode(true);

            // delete elements marked with ignore (eg anchor and jump buttons)
            const deleteFromCopy = sourceCopy.querySelectorAll(
                '[data-ignore="true"]'
            );
            for (const element of deleteFromCopy)
                element.remove();

            // delete certain element attributes
            const attributes = [
                'id',
                'data-collapsed',
                'data-selected',
                'data-highlighted',
                'data-glow'
            ];
            for (const attribute of attributes) {
                sourceCopy.removeAttribute(attribute);
                const elements = sourceCopy.querySelectorAll(
                    '[' + attribute + ']'
                );
                for (const element of elements)
                    element.removeAttribute(attribute);
            }

            return sourceCopy;
        }

        // make button to jump to previous image in document
        function makePrevButton(img) {
            const prevButton = document.createElement('button');
            prevButton.id = 'lightbox_prev_button';
            prevButton.title = 'Jump to the previous image in the document [‚Üê]';
            prevButton.classList.add('icon_button', 'lightbox_button');
            prevButton.innerHTML = document.querySelector(
                '.icon_caret_left'
            ).innerHTML;

            // attach click listeners to button
            prevButton.addEventListener('click', function() {
                getPrevImg(img).click();
            });

            return prevButton;
        }

        // make button to jump to next image in document
        function makeNextButton(img) {
            const nextButton = document.createElement('button');
            nextButton.id = 'lightbox_next_button';
            nextButton.title = 'Jump to the next image in the document [‚Üí]';
            nextButton.classList.add('icon_button', 'lightbox_button');
            nextButton.innerHTML = document.querySelector(
                '.icon_caret_right'
            ).innerHTML;

            // attach click listeners to button
            nextButton.addEventListener('click', function() {
                getNextImg(img).click();
            });

            return nextButton;
        }

        // get previous image in document
        function getPrevImg(img) {
            const imgs = document.querySelectorAll('.lightbox_document_img');

            // find index of provided img
            let index;
            for (index = 0; index < imgs.length; index++) {
                if (imgs[index] === img)
                    break;
            }


            // wrap index to other side if < 1
            if (index - 1 >= 0)
                index--;
            else
                index = imgs.length - 1;
            return imgs[index];
        }

        // get next image in document
        function getNextImg(img) {
            const imgs = document.querySelectorAll('.lightbox_document_img');

            // find index of provided img
            let index;
            for (index = 0; index < imgs.length; index++) {
                if (imgs[index] === img)
                    break;
            }


            // wrap index to other side if > total
            if (index + 1 <= imgs.length - 1)
                index++;
            else
                index = 0;
            return imgs[index];
        }

        // close lightbox
        function closeLightbox() {
            focusBody();

            const lightbox = document.getElementById('lightbox_overlay');
            if (lightbox)
                lightbox.remove();
        }

        // make all elements behind lightbox non-focusable
        function blurBody(overlay) {
            const all = document.querySelectorAll('*');
            for (const element of all)
                element.tabIndex = -1;
            document.body.classList.add('body_no_scroll');
        }

        // make all elements focusable again
        function focusBody() {
            const all = document.querySelectorAll('*');
            for (const element of all)
                element.removeAttribute('tabIndex');
            document.body.classList.remove('body_no_scroll');
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
    <!-- modified from: https://fontawesome.com/icons/caret-left -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
        ></path>
    </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
    <!-- modified from: https://fontawesome.com/icons/caret-right -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
        ></path>
    </svg>
</template>
<!-- attributes plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin allows arbitrary HTML attributes to be attached
        // to (almost) any element. Place an HTML comment inside or next to the
        // desired element in the format <!-- $attribute="value" -->

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'attributes';

        // default plugin options
        const options = {
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // get list of comments in document
            const comments = findComments();

            for(const comment of comments)
                if (comment.parentElement)
                    addAttributes(
                        comment.parentElement,
                        comment.nodeValue.trim()
                    );
        }

        // add html attributes to specified element based on string of 
        // html attributes and values
        function addAttributes(element, text) {
            // regex's for finding attribute/value pairs in the format of
            // attribute="value" or attribute='value
            const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
            const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

            // loop through attribute/value pairs
            let match;
            while(match = text.match(regex2) || text.match(regex1)) {
                // get attribute and value from regex capture groups
                let attribute = match[1];
                let value = match[2];

                // remove from string
                text = text.substring(match.index + match[0].length);

                if (!attribute || !value)
                    break;

                // set attribute of parent element
                try {
                    element.setAttribute(attribute, value);
                } catch(error) {
                    console.log(error);
                }

                // special case for colspan
                if (attribute === 'colspan')
                    removeTableCells(element, value);
            }
        }

        // get list of comment elements in document
        function findComments() {
            const comments = [];

            // iterate over comment nodes in document
            function acceptNode(node) {
                return NodeFilter.FILTER_ACCEPT;
            }
            const iterator = document.createNodeIterator(
                document.body,
                NodeFilter.SHOW_COMMENT,
                acceptNode
            );
            let node;
            while(node = iterator.nextNode())
                comments.push(node);

            return comments;
        }

        // remove certain number of cells after specified cell
        function removeTableCells(cell, number) {
            number = parseInt(number);
            if (!number)
                return;

            // remove elements
            for(; number > 1; number--) {
                if (cell.nextElementSibling)
                    cell.nextElementSibling.remove();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>
<!-- math plugin configuration -->

<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
        "CommonHTML": { linebreaks: { automatic: true } },
        "HTML-CSS": { linebreaks: { automatic: true } },
        "SVG": { linebreaks: { automatic: true } },
        "fast-preview": { disabled: true }
    });
</script>

<!-- math plugin -->

<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML'>
    // /////////////////////////
    // DESCRIPTION
    // /////////////////////////

    // This third-party plugin 'MathJax' allows the proper rendering of
    // math/equations written in LaTeX.

    // https://www.mathjax.org/
</script>
<!-- annotations plugin configuration -->

<script>
    window.hypothesisConfig = function() {
        return {
            branding: {
                accentColor: '#2196f3',
                appBackgroundColor: '#f8f8f8',
                ctaBackgroundColor: '#f8f8f8',
                ctaTextColor: '#000000',
                selectionFontFamily: 'Open Sans, Helvetica, sans serif',
                annotationFontFamily: 'Open Sans, Helvetica, sans serif'
            }
        };
    };
</script>

<!-- annotations plugin -->

<script src='https://hypothes.is/embed.js'>
    // /////////////////////////
    // DESCRIPTION
    // /////////////////////////

    // This third-party plugin 'Hypothesis' allows public annotation of the
    // manuscript.

    // https://web.hypothes.is/
</script>
<!-- analytics plugin -->

<!-- copy and paste code from Google Analytics or similar service here -->
</body>
</html>
